<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>AR: E2DHT Vs. E2 Compared To ER: E2DHT Vs. E2</title>

<script src="site_libs/header-attrs-2.22/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/sandstone.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/core-js-2.5.3/shim.min.js"></script>
<script src="site_libs/react-17.0.0/react.min.js"></script>
<script src="site_libs/react-17.0.0/react-dom.min.js"></script>
<script src="site_libs/reactwidget-1.0.0/react-tools.js"></script>
<script src="site_libs/htmlwidgets-1.6.2/htmlwidgets.js"></script>
<link href="site_libs/reactable-0.4.4/reactable.css" rel="stylesheet" />
<script src="site_libs/reactable-binding-0.4.4/reactable.js"></script>
<link href="site_libs/font-awesome-6.4.0/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.0/css/v4-shims.min.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>





<style type="text/css">
/* for pandoc --citeproc since 2.11 */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="gravi.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">GRAVI</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="annotation_description.html">Annotations</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    MACS2 Peak Calling
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="AR_macs2_summary.html">AR</a>
    </li>
    <li>
      <a href="ER_macs2_summary.html">ER</a>
    </li>
    <li>
      <a href="H3K27ac_macs2_summary.html">H3K27ac</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Differential Binding
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">AR</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="AR_E2_E2DHT_differential_binding.html">E2DHT Vs. E2</a>
        </li>
      </ul>
    </li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">ER</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="ER_E2_E2DHT_differential_binding.html">E2DHT Vs. E2</a>
        </li>
      </ul>
    </li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">H3K27ac</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="H3K27ac_E2_E2DHT_differential_binding.html">E2DHT Vs. E2</a>
        </li>
      </ul>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Pairwise Comparisons
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">AR-ER</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="AR_E2_E2DHT_ER_E2_E2DHT_pairwise_comparison.html">E2DHT Vs. E2 / E2DHT Vs. E2</a>
        </li>
      </ul>
    </li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">AR-H3K27ac</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="AR_E2_E2DHT_H3K27ac_E2_E2DHT_pairwise_comparison.html">E2DHT Vs. E2 / E2DHT Vs. E2</a>
        </li>
      </ul>
    </li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">ER-H3K27ac</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="ER_E2_E2DHT_H3K27ac_E2_E2DHT_pairwise_comparison.html">E2DHT Vs. E2 / E2DHT Vs. E2</a>
        </li>
      </ul>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/steveped/GRAVI">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">AR: E2DHT Vs. E2 Compared To ER: E2DHT Vs.
E2</h1>
<h4 class="date">02 July, 2023</h4>

</div>


<pre class="r"><code>threads &lt;- 4
pairs &lt;- list(
  &quot;AR&quot; = c(&quot;E2&quot;, &quot;E2DHT&quot;),
  &quot;ER&quot; = c(&quot;E2&quot;, &quot;E2DHT&quot;)
)</code></pre>
<pre class="r"><code>library(tidyverse)
library(glue)
library(pander)
library(yaml)
library(reactable)
library(plyranges)
library(rtracklayer)
library(VennDiagram)
library(UpSetR)
library(magrittr)
library(scales)
library(ggrepel)
library(rlang)
library(ggside)
library(msigdbr)
library(htmltools)
library(goseq)
library(parallel)
library(GenomicInteractions)
library(extraChIPs)
library(ggraph)
library(tidygraph)
source(here::here(&quot;workflow&quot;, &quot;scripts&quot;, &quot;custom_functions.R&quot;))</code></pre>
<pre class="r"><code>panderOptions(&quot;big.mark&quot;, &quot;,&quot;)
panderOptions(&quot;missing&quot;, &quot;&quot;)
panderOptions(&quot;table.split.table&quot;, Inf)
theme_set(
  theme_bw() +
    theme(
      plot.title = element_text(hjust = 0.5)
    )
)</code></pre>
<pre class="r"><code>config &lt;- read_yaml(here::here(&quot;config&quot;, &quot;config.yml&quot;))
params &lt;- read_yaml(here::here(&quot;config&quot;, &quot;params.yml&quot;))
targets &lt;- names(pairs)
macs2_path &lt;- here::here(&quot;output&quot;, &quot;macs2&quot;, targets)
fdr_alpha &lt;- config$comparisons$fdr
comps &lt;- targets
if (length(unique(targets)) == 1) {
  comps &lt;- seq_along(pairs) %&gt;%
    vapply(
      function(x) paste(
        # names(pairs)[[x]], 
        paste(rev(pairs[[x]]), collapse = &quot; Vs. &quot;)#, 
        # sep = &quot;: &quot;
      ), 
      character(1)
    )
}
n_targets = length(unique(names(pairs)))
samples &lt;- file.path(macs2_path, glue(&quot;{targets}_qc_samples.tsv&quot;)) %&gt;% 
  lapply(read_tsv) %&gt;% 
  lapply(mutate_all, as.character) %&gt;% 
  bind_rows() %&gt;% 
  dplyr::filter(
    qc == &quot;pass&quot;,
    (target == names(pairs)[[1]] &amp; treat %in% pairs[[1]]) |
      (target == names(pairs)[[2]] &amp; treat %in% pairs[[2]])
  ) %&gt;% 
  unite(label, target, label, remove = FALSE) %&gt;% 
  mutate(
    target = factor(target, levels = unique(targets)),
    treat = factor(treat, levels = unique(unlist(pairs)))
  ) %&gt;% 
  dplyr::select(-qc) %&gt;% 
  droplevels()
stopifnot(nrow(samples) &gt; 0)
rep_col &lt;- setdiff(
  colnames(samples), c(&quot;sample&quot;, &quot;treat&quot;, &quot;target&quot;, &quot;input&quot;, &quot;label&quot;, &quot;qc&quot;)
)
samples[[rep_col]] &lt;- as.factor(samples[[rep_col]])</code></pre>
<pre class="r"><code>annotation_path &lt;- here::here(&quot;output&quot;, &quot;annotations&quot;)
colours &lt;- read_rds(file.path(annotation_path, &quot;colours.rds&quot;))
treat_colours &lt;- unlist(colours$treat[levels(samples$treat)])
combs &lt;- paste(
  rep(comps, each = 4), c(&quot;Up&quot;, &quot;Down&quot;, &quot;Unchanged&quot;, &quot;Undetected&quot;)
) %&gt;% 
  combn(2) %&gt;% 
  t() %&gt;% 
  set_colnames(c(&quot;S1&quot;, &quot;S2&quot;)) %&gt;% 
  as_tibble() %&gt;%
  mutate(
    C1 = str_remove(S1, &quot; (Up|Down|Unchanged|Undetected)&quot;),
    C2 = str_remove(S2, &quot; (Up|Down|Unchanged|Undetected)&quot;),
  ) %&gt;%
  dplyr::filter(C1 != C2) %&gt;%
  unite(combn, S1, S2, sep = &quot; - &quot;) %&gt;%
  pull(&quot;combn&quot;)
direction_colours &lt;- c(
  &#39;#e6194B&#39;, &#39;#3cb44b&#39;, &#39;#ffe119&#39;, &#39;#4363d8&#39;, &#39;#f58231&#39;, &#39;#42d4f4&#39;, &#39;#f032e6&#39;, 
  &#39;#fabed4&#39;, &#39;#469990&#39;, &#39;#dcbeff&#39;, &#39;#9A6324&#39;, &#39;#fffac8&#39;, &#39;#800000&#39;, &#39;#aaffc3&#39;, 
  &#39;#000075&#39;, &#39;#a9a9a9&#39;
) %&gt;% # https://sashamaps.net/docs/resources/20-colors/
  setNames(combs) %&gt;% 
  c(&quot;Other&quot; = &quot;#4d4d4d&quot;)
comp_cols &lt;- setNames(hcl.colors(3, &quot;Zissou1&quot;, rev = TRUE), c(comps, &quot;Both&quot;))</code></pre>
<pre class="r"><code>cb &lt;- config$genome$build %&gt;% 
  str_to_lower() %&gt;% 
  paste0(&quot;.cytobands&quot;) 
data(list = cb)
bands_df &lt;- get(cb)
sq &lt;- read_rds(file.path(annotation_path, &quot;seqinfo.rds&quot;))
blacklist &lt;- import.bed(here::here(config$external$blacklist), seqinfo = sq)

gtf_gene &lt;- read_rds(file.path(annotation_path, &quot;gtf_gene.rds&quot;))
gtf_transcript &lt;- read_rds(file.path(annotation_path, &quot;gtf_transcript.rds&quot;))
gtf_exon &lt;- read_rds(file.path(annotation_path, &quot;gtf_exon.rds&quot;))
id2gene &lt;- setNames(gtf_gene$gene_name, gtf_gene$gene_id)
trans_models &lt;- file.path(annotation_path, &quot;trans_models.rds&quot;) %&gt;% 
  read_rds() 
tss &lt;- file.path(annotation_path, &quot;tss.rds&quot;) %&gt;% 
  read_rds()

external_features &lt;- c()
has_features &lt;- FALSE
if (!is.null(config$external$features)) {
  external_features &lt;- suppressWarnings(
    import.gff(here::here(config$external$features), genome = sq)
  )
  keep_cols &lt;- !vapply(
    mcols(external_features), function(x) all(is.na(x)), logical(1)
  )
  mcols(external_features) &lt;- mcols(external_features)[keep_cols]
  has_features &lt;- TRUE
  feature_colours &lt;- colours$features %&gt;% 
    setNames(str_sep_to_title(names(.)))
}
gene_regions &lt;- read_rds(file.path(annotation_path, &quot;gene_regions.rds&quot;))
regions &lt;- vapply(gene_regions, function(x) unique(x$region), character(1))
region_colours &lt;-  unlist(colours$regions) %&gt;% setNames(regions[names(.)])</code></pre>
<pre class="r"><code>rna_path &lt;- here::here(config$external$rnaseq)
rnaseq &lt;- tibble(gene_id = character(), gene_name = character())
if (length(rna_path) &gt; 0) {
  stopifnot(file.exists(rna_path))
  if (str_detect(rna_path, &quot;tsv$&quot;)) rnaseq &lt;- read_tsv(rna_path)
  if (str_detect(rna_path, &quot;csv$&quot;)) rnaseq &lt;- read_csv(rna_path)
  if (!&quot;gene_id&quot; %in% colnames(rnaseq)) stop(&quot;Supplied RNA-Seq data must contain the column &#39;gene_id&#39;&quot;)
  gtf_gene &lt;- subset(gtf_gene, gene_id %in% rnaseq$gene_id)
}
has_rnaseq &lt;- as.logical(nrow(rnaseq))
tx_col &lt;- intersect(c(&quot;tx_id&quot;, &quot;transcript_id&quot;), colnames(rnaseq))
rna_gr_col &lt;- ifelse(length(tx_col) &gt; 0, &quot;transcript_id&quot;, &quot;gene_id&quot;)
rna_col &lt;- c(tx_col, &quot;gene_id&quot;)[[1]]</code></pre>
<pre class="r"><code>hic &lt;- GInteractions()
hic_path &lt;- here::here(config$external$hic)
has_hic &lt;- FALSE
if (length(hic_path) &gt; 0)
  if (file.exists(hic_path)) {
    has_hic &lt;- TRUE
    hic &lt;- makeGenomicInteractionsFromFile(hic_path, type = &quot;bedpe&quot;)
    reg_combs &lt;- expand.grid(regions, regions) %&gt;% 
      as.matrix() %&gt;% 
      apply(
        MARGIN = 1, 
        function(x) {
          x &lt;- sort(factor(x, levels = regions))
          paste(as.character(x), collapse = &quot; - &quot;)
        }
      ) %&gt;% 
      unique()
    hic$regions &lt;- anchors(hic) %&gt;% 
      vapply(
        bestOverlap,
        y = GRangesList(lapply(gene_regions, granges)),
        character(length(hic))
      ) %&gt;% 
      apply(MARGIN = 2, function(x) regions[x]) %&gt;% 
      apply(
        MARGIN = 1, 
        function(x) {
          x &lt;- sort(factor(x, levels = regions))
          paste(as.character(x), collapse = &quot; - &quot;)
        }
      ) %&gt;% 
      factor(levels = reg_combs) %&gt;%
      fct_relabel(
        str_replace_all,
        pattern = &quot;Promoter \\([0-9kbp/\\+-]+\\)&quot;, replacement = &quot;Promoter&quot;
      )
    if (has_features) {
      feat_combs &lt;- expand.grid(names(feature_colours), names(feature_colours)) %&gt;% 
        as.matrix() %&gt;% 
        apply(
          MARGIN = 1, 
          function(x) {
            x &lt;- sort(factor(x, levels = names(feature_colours)))
            paste(as.character(x), collapse = &quot; - &quot;)
          }
        ) %&gt;% 
        unique()
      hic$features &lt;- vapply(
        anchors(hic),
        function(x) bestOverlap(
          x, external_features, var = &quot;feature&quot;, missing = &quot;no_feature&quot;
        ),
        character(length(hic))
      )  %&gt;% 
        apply(
          MARGIN = 1, 
          function(x) {
            x &lt;- sort(factor(x, levels = names(feature_colours)))
            paste(as.character(x), collapse = &quot; - &quot;)
          }
        ) %&gt;% 
        factor(levels = feat_combs) %&gt;%
        fct_relabel(str_sep_to_title, pattern = &quot;_&quot;)
    }
  }
stopifnot(is(hic, &quot;GInteractions&quot;))
seqlevels(hic) &lt;- seqlevels(sq)
seqinfo(hic) &lt;- sq
hic &lt;- hic[!overlapsAny(hic, blacklist)]</code></pre>
<pre class="r"><code>treatment_peaks &lt;- macs2_path %&gt;%
  file.path(glue(&quot;{targets}_treatment_peaks.rds&quot;)) %&gt;%
  unique() %&gt;% 
  sapply(read_rds) %&gt;% 
  setNames(unique(basename(macs2_path)))
union_peaks &lt;- macs2_path %&gt;%
  file.path(glue(&quot;{targets}_union_peaks.bed&quot;)) %&gt;%
  lapply(import.bed, seqinfo = sq) %&gt;%
  setNames(basename(macs2_path))
all_union &lt;- union_peaks %&gt;% 
  GRangesList() %&gt;% 
  unlist() %&gt;% 
  GenomicRanges::reduce() %&gt;% 
  sort() 
db_results &lt;- seq_along(pairs) %&gt;% 
  lapply(
    function(x) {
      tg &lt;- names(pairs)[[x]]
      here::here(
        &quot;output&quot;, &quot;differential_binding&quot;, tg, 
        glue(&quot;{tg}_{pairs[[x]][[1]]}_{pairs[[x]][[2]]}-differential_binding.rds&quot;)
      )
    }
  ) %&gt;% 
  lapply(read_rds) %&gt;% 
  lapply(mutate, fdr_mu0 = p.adjust(p_mu0, &quot;fdr&quot;)) %&gt;% 
  setNames(comps)
fdr_column &lt;- ifelse(
  &quot;fdr_ihw&quot; %in% colnames(mcols(db_results[[1]])), 
  &quot;fdr_ihw&quot;, 
  str_subset(&quot;fdr&quot;, colnames(mcols(db_results[[1]])))
)</code></pre>
<pre class="r"><code>up_col &lt;- function(x) {
  if (is.na(x) | is.nan(x)) return(&quot;#ffffff&quot;)
  rgb(
    colorRamp(c(&quot;#ffffff&quot;, colours$direction[[&quot;up&quot;]]))(x), maxColorValue = 255
  )
}
down_col &lt;- function(x) {
  if (is.na(x) | is.nan(x)) return(&quot;#ffffff&quot;)
  rgb(
    colorRamp(c(&quot;#ffffff&quot;, colours$direction[[&quot;down&quot;]]))(x), maxColorValue = 255
  )
}
unch_col &lt;- function(x) {
  if (is.na(x) | is.nan(x)) return(&quot;#ffffff&quot;)
  rgb(
    colorRamp(c(&quot;#ffffff&quot;, colours$direction[[&quot;unchanged&quot;]]))(x), 
    maxColorValue = 255
  )
}
lfc_col &lt;- function(x){
  if (is.na(x) | is.nan(x)) return(&quot;#ffffff&quot;)
  rgb(
    colorRamp(c(colours$direction[[&quot;down&quot;]], &quot;#ffffff&quot;, colours$direction[[&quot;up&quot;]]))(x), 
    maxColorValue = 255
  )
}
expr_col &lt;- function(x){
  if (is.na(x) | is.nan(x)) return(&quot;#ffffff&quot;)
  rgb(colorRamp(hcl.colors(9, &quot;TealRose&quot;))(x), maxColorValue = 255)
}
bar_style &lt;- function(width = 1, fill = &quot;#e6e6e6&quot;, height = &quot;75%&quot;, align = c(&quot;left&quot;, &quot;right&quot;), color = NULL) {
  align &lt;- match.arg(align)
  if (align == &quot;left&quot;) {
    position &lt;- paste0(width * 100, &quot;%&quot;)
    image &lt;- sprintf(&quot;linear-gradient(90deg, %1$s %2$s, transparent %2$s)&quot;, fill, position)
  } else {
    position &lt;- paste0(100 - width * 100, &quot;%&quot;)
    image &lt;- sprintf(&quot;linear-gradient(90deg, transparent %1$s, %2$s %1$s)&quot;, position, fill)
  }
  list(
    backgroundImage = image,
    backgroundSize = paste(&quot;100%&quot;, height),
    backgroundRepeat = &quot;no-repeat&quot;,
    backgroundPosition = &quot;center&quot;,
    color = color
  )
}
with_tooltip &lt;- function(value, width = 30) {
  htmltools::tags$span(title = value, str_trunc(value, width))
}</code></pre>
<pre class="r"><code>comp_name &lt;- glue(
  &quot;{targets[[1]]}_{pairs[[1]][[1]]}_{pairs[[1]][[2]]}-&quot;, 
  &quot;{targets[[2]]}_{pairs[[2]][[1]]}_{pairs[[2]][[2]]}&quot;
)
fig_path &lt;- here::here(&quot;docs&quot;, &quot;assets&quot;, comp_name)
if (!dir.exists(fig_path)) dir.create(fig_path, recursive = TRUE)
fig_dev &lt;- knitr::opts_chunk$get(&quot;dev&quot;)
fig_type &lt;- fig_dev[[1]]
if (is.null(fig_type)) stop(&quot;Couldn&#39;t detect figure type&quot;)
fig_fun &lt;- match.fun(fig_type)
if (fig_type %in% c(&quot;bmp&quot;, &quot;jpeg&quot;, &quot;png&quot;, &quot;tiff&quot;)) {
  ## These figure types require dpi &amp; resetting to be inches
  formals(fig_fun)$units &lt;- &quot;in&quot;
  formals(fig_fun)$res &lt;- 300
}

out_path &lt;- here::here(
  &quot;output&quot;, &quot;pairwise_comparisons&quot;, paste(targets, collapse = &quot;_&quot;)
)
if (!dir.exists(out_path)) dir.create(out_path, recursive = TRUE)
all_out &lt;- list(
  csv = file.path(
    out_path, glue(comp_name, &quot;-pairwise_comparison.csv.gz&quot;
    )
  ),
  de_genes = file.path(
    out_path, 
    glue(&quot;{comp_name}-de_genes.csv&quot;)
  ),
  goseq = file.path(
    out_path,
    glue(&quot;{comp_name}-enrichment.csv&quot;)
  ),
  results = file.path(
    out_path, glue(comp_name, &quot;-all_windows.rds&quot;)
  ),
  rna_enrichment = file.path(
    out_path,
    glue(&quot;{comp_name}-rnaseq_enrichment.csv&quot;)
  ),
  renv = here::here(
    &quot;output&quot;, &quot;envs&quot;,
    glue(comp_name, &quot;-pairwise_comparison.RData&quot;)
  )
)
## Empty objects for when RNA-Seq is absent
de_genes_both_comps &lt;- tibble()
cmn_res &lt;- list(tibble(gs_name = character(), leadingEdge = list()))</code></pre>
<div id="outline" class="section level1">
<h1>Outline</h1>
<p>This stage of the GRAVI workflow takes the results from two
individual differential binding analyses and compares the results, by
finding peaks which directly overlap and considering the joint
behaviour. In this specific analyses the differential binding response
for AR when comparing E2DHT Vs E2, will be integrated with the
differential binding response for ER comparing E2DHT Vs E2. Data will
first be described from the perspective of <em>union peaks</em> and
<em>treatment peaks</em>, before moving onto the differentially bound
windows obtained in previous steps.</p>
<div id="integration-of-differential-binding-results"
class="section level2">
<h2>Integration Of Differential Binding Results</h2>
<p>The initial steps of differential binding analysis detect regions
with clear changes in binding patterns, however the integration of
datasets instead focusses on the correct classification of a given
range/window. Differential binding analyses generally focus on
controlling the Type II errors (i.e. false discoveries), which comes
with a commensurate increase in Type I errors (false negatives). When
seeking to correctly classify a window across two datasets, the
detection as changed can be taken as a robust finding, whilst the
consideration as unchanged may be less robust. When comparing a window
across two datasets, a detection as changed within one dataset is taken
as primary evidence of ‘something interesting’ occurring, with the
secondary consideration being the correct description of the combined
binding patterns. For this reason, and to minimise issues with hard
thresholding, once a window is considered to be of interest, the p-value
used in the second comparison is that obtained (and FDR-adjusted) using
the test for <span class="math inline">\(H_0: \mu = 0\)</span> instead
of the initial <span class="math inline">\(H_0: 0 &lt; |\mu| &lt;
\lambda\)</span>. The insistence on the more stringent initial inclusion
criteria still protects against false discoveries, but this approach
instead allows <em>improved classification</em> across both
comparisons.</p>
<p>For <em>classification steps within each window</em>, each target is
given the status 1) Up, 2) Down, 3) Unchanged, or 4) Undetected. Across
two comparisons, this gives 15 possible classifications for each region
found with at least one target bound (e.g. Up-Up, Up-Down, Up-Unchanged,
Up-Undetected etc.), given that any potential Undetected-Undetected
windows will not be included for obvious reasons.</p>
</div>
<div id="enrichment-testing" class="section level2">
<h2>Enrichment Testing</h2>
<p>After direct comparison of binding patterns, enrichment analysis is
performed in a similar manner to for individual analyses. If RNA-Seq
data is provided, genes will be restricted to those present within the
RNA-Seq dataset, as representative of detected genes. Enrichment testing
is performed on:</p>
<ol style="list-style-type: decimal">
<li>All genes mapped to any bound region against those not mapped to a
bound region</li>
<li>All genes mapped to both targets against genes not mapped</li>
</ol>
</div>
<div id="comparison-to-rna-seq" class="section level2">
<h2>Comparison To RNA-Seq</h2>
<p>If RNA-Seq data is supplied, the sets of genes associated with all
binding patterns (e.g. Up-Up, Up-Down etc) were treated as gene-sets and
Gene Set Enrichment Analysis (GSEA) <span class="citation">(<a
href="#ref-SubramanianGsea2005" role="doc-biblioref">Subramanian et al.
2005</a>)</span> was used to determine enrichment and leading-edge genes
associated with altered expression. Genes will be ranked to incorporate
direction of change (i.e. Directional GSEA) or significance only (i.e
Non-Directional GSEA). The results from enrichment analysis across ChIP
target and RNA-Seq data will also be combined to reveal pathways under
the regulatory influence in either comparison, for which we also have
evidence of changed gene expression.</p>
</div>
</div>
<div id="peak-comparison" class="section level1 tabset">
<h1 class="tabset">Peak Comparison</h1>
<p>Data was loaded for treatment-agnostic <em>union peaks</em> for AR
and ER with treatment-specific <em>treatment peaks</em> (i.e. based on
reproducibility across treatment-specific replicates). The sets of
macs2-derived peaks for both AR and ER were first compared using
treatment-agnostic peaks for each target. The sets of treatment-specific
<em>treatment-peaks</em> were then compared across the combined
treatment groups and targets.</p>
<div id="union-peaks" class="section level2">
<h2>Union Peaks</h2>
<pre class="r"><code>fig_name &lt;- targets %&gt;% 
  paste(collapse = &quot;_&quot;) %&gt;% 
  paste0(&quot;_common_peaks.&quot;, fig_type)
lapply(
  union_peaks,
  function(x) as.character(subsetByOverlaps(all_union, x))
)%&gt;% 
  venn.diagram(
    file.path(fig_path, fig_name),
    imagetype = fig_type,
    units = &quot;in&quot;,
    cat.cex = 1.4,
    height = 9, 
    width = 10,
    fill = comp_cols[comps], 
    alpha = 0.3
  )
file.remove(list.files(fig_path, pattern = &quot;log$&quot;, full.names = TRUE))</code></pre>
<div class="figure">
<img src="assets/AR_E2_E2DHT-ER_E2_E2DHT/AR_ER_common_peaks.png"
alt="" />
<p class="caption"><em>Union of peaks found in AR and ER, and which of
the two targets each peak is found in. Peaks are based on the union of
both sets of union peaks, which are themselves independent of treatment
group.</em></p>
</div>
</div>
<div id="treatment-specific-peaks" class="section level2">
<h2>Treatment Specific Peaks</h2>
<pre class="r"><code>bar_cols &lt;- treat_colours[rev(unlist(pairs))]
if (length(unique(targets)) == 1)
  bar_cols &lt;- treat_colours[
    pairs %&gt;% 
      unlist() %&gt;% 
      unique() %&gt;% 
      rev()
  ]
samples %&gt;% 
  distinct(target, treat) %&gt;% 
  mutate_all(as.character) %&gt;% 
  unite(group, target, treat, remove = FALSE) %&gt;% 
  split(.$group) %&gt;% 
  lapply(
    function(x) {
      gr &lt;- subsetByOverlaps(all_union, treatment_peaks[[x$target]][[x$treat]])
      as.character(gr)
    }
  ) %&gt;% 
  setNames(
    str_replace_all(names(.), &quot;(.+)_(.+)&quot;, &quot;\\1 (\\2)&quot;)
  ) %&gt;% 
  fromList() %&gt;% 
  upset(
    sets = samples %&gt;%
      distinct(target, treat) %&gt;%
      mutate_all(as.character) %&gt;%
      mutate(grp = glue(&quot;{target} ({treat})&quot;)) %&gt;%
      pull(&quot;grp&quot;) %&gt;%
      as.character() %&gt;%
      rev(),
    keep.order = TRUE,
    order.by = &quot;freq&quot;,
    set_size.show = TRUE, 
    set_size.scale_max = treatment_peaks %&gt;% 
      lapply(function(x) lapply(x, length)) %&gt;% 
      unlist() %&gt;% 
      max() %&gt;% 
      multiply_by(1.15),
    sets.bar.color = bar_cols
  )</code></pre>
<div class="figure" style="text-align: center">
<img src="AR_E2_E2DHT_ER_E2_E2DHT_pairwise_comparison_files/figure-html/treatment-peaks-upset-1.png" alt="*Macs2 peaks detected for AR and ER in the treatment groups E2 or E2DHT.*" width="960" />
<p class="caption">
<em>Macs2 peaks detected for AR and ER in the treatment groups E2 or
E2DHT.</em>
</p>
</div>
</div>
</div>
<div id="differentially-bound-windows" class="section level1 tabset">
<h1 class="tabset">Differentially Bound Windows</h1>
<pre class="r"><code>lambda &lt;- log2(config$comparisons$fc)
c1_status &lt;- glue(&quot;{comps[[1]]}_status&quot;)
c1_fdr &lt;- glue(&quot;{comps[[1]]}_fdr&quot;)
c1_logfc &lt;- glue(&quot;{comps[[1]]}_logFC&quot;)
c1_logcpm &lt;- glue(&quot;{comps[[1]]}_AveExpr&quot;)
c1_centre &lt;- glue(&quot;{comps[[1]]}_centre&quot;)
c2_status &lt;- glue(&quot;{comps[[2]]}_status&quot;)
c2_fdr &lt;- glue(&quot;{comps[[2]]}_fdr&quot;)
c2_logfc &lt;- glue(&quot;{comps[[2]]}_logFC&quot;)
c2_logcpm &lt;- glue(&quot;{comps[[2]]}_AveExpr&quot;)
c2_centre &lt;- glue(&quot;{comps[[2]]}_centre&quot;)
all_windows &lt;- db_results %&gt;%
  lapply(granges) %&gt;% 
  GRangesList() %&gt;% 
  unlist()%&gt;%
  sort() %&gt;% 
  GenomicRanges::reduce() %&gt;% 
  mutate(
    region = regions[
      bestOverlap(., GRangesList(lapply(gene_regions, granges)))
    ] %&gt;% 
      factor(levels = regions)
  ) %&gt;%
  join_overlap_left(
    db_results[[comps[[1]]]] %&gt;% 
      mutate(
        status = as.character(status),
        peak_centre = start(
          resize(keyval_range, width = 1, fix = &quot;center&quot;)
        )
      ) %&gt;% 
      select(
        !!sym(c1_logcpm) := AveExpr, 
        !!sym(c1_logfc) := logFC, 
        !!sym(c1_status) := status, 
        !!sym(c1_fdr) := !!sym(fdr_column),
        !!sym(glue(&quot;{comps[[1]]}_fdr_mu0&quot;)) := fdr_mu0,
        !!sym(glue(&quot;{comps[[1]]}_centre&quot;)) := peak_centre
      )
  ) %&gt;%
  join_overlap_left(
    db_results[[comps[[2]]]] %&gt;% 
      mutate(
        status = as.character(status),
        peak_centre = start(
          resize(keyval_range, width = 1, fix = &quot;center&quot;)
        )
      ) %&gt;% 
      select(
        !!sym(c2_logcpm) := AveExpr, 
        !!sym(c2_logfc) := logFC, 
        !!sym(c2_status) := status,
        !!sym(c2_fdr) := !!sym(fdr_column),
        !!sym(glue(&quot;{comps[[2]]}_fdr_mu0&quot;)) := fdr_mu0,
        !!sym(glue(&quot;{comps[[2]]}_centre&quot;)) := peak_centre
      )
  ) %&gt;% 
  ## Remove duplicate mappings to each range, just selecting the one with the
  ## lowest FDR
  arrange(!!sym(c2_fdr)) %&gt;% 
  distinctMC(!!sym(c1_fdr), .keep_all = TRUE) %&gt;% 
  arrange(!!sym(c1_fdr)) %&gt;% 
  sort() 
## plyranges::mutate changes the column names using `make.names()`. Avoid!!
## Reclassify windows from the first comparison using data from the second
## to reduce false negatives. If the 2nd comparison contains a significant
## change in binding, reclassify the first using the fdr for H0: mu[0] = 0
## but only where |logFC[est]| &gt; lambda
switch_up &lt;- mcols(all_windows)[[c2_fdr]] &lt; fdr_alpha &amp;
  mcols(all_windows)[[glue(&quot;{comps[[1]]}_fdr_mu0&quot;)]] &lt; fdr_alpha &amp;
  mcols(all_windows)[[c1_logfc]] &gt; lambda
mcols(all_windows)[[c1_status]][switch_up] &lt;- &quot;Up&quot;
switch_down &lt;- mcols(all_windows)[[c2_fdr]] &lt; fdr_alpha &amp;
  mcols(all_windows)[[glue(&quot;{comps[[1]]}_fdr_mu0&quot;)]] &lt; fdr_alpha &amp;
  mcols(all_windows)[[c1_logfc]] &lt; (-1) * lambda
mcols(all_windows)[[c1_status]][switch_down] &lt;- &quot;Down&quot;
mcols(all_windows)[[c1_status]] &lt;- mcols(all_windows) %&gt;% 
  .[[c1_status]] %&gt;% 
  factor(levels = str_sep_to_title(names(colours$direction))) %&gt;% 
  fct_explicit_na(&quot;Undetected&quot;) %&gt;% 
  fct_relabel(function(x) paste(comps[[1]], x))  
## Now reclassify the second comparison using changed windows in the first
switch_up &lt;- mcols(all_windows)[[c1_fdr]] &lt; fdr_alpha &amp;
  mcols(all_windows)[[glue(&quot;{comps[[2]]}_fdr_mu0&quot;)]] &lt; fdr_alpha &amp;
  mcols(all_windows)[[c2_logfc]] &gt; lambda
mcols(all_windows)[[c2_status]][switch_up] &lt;- &quot;Up&quot;
switch_down &lt;- mcols(all_windows)[[c1_fdr]] &lt; fdr_alpha &amp;
  mcols(all_windows)[[glue(&quot;{comps[[2]]}_fdr_mu0&quot;)]] &lt; fdr_alpha &amp;
  mcols(all_windows)[[c2_logfc]] &lt; (-1) * lambda
mcols(all_windows)[[c2_status]][switch_down] &lt;- &quot;Down&quot;
mcols(all_windows)[[c2_status]] &lt;- mcols(all_windows) %&gt;% 
  .[[c2_status]] %&gt;% 
  factor(levels = str_sep_to_title(names(colours$direction))) %&gt;% 
  fct_explicit_na(&quot;Undetected&quot;) %&gt;% 
  fct_relabel(function(x) paste(comps[[2]], x))  
mcols(all_windows)[[&quot;status&quot;]] &lt;- paste(
  mcols(all_windows)[[c1_status]],
  mcols(all_windows)[[c2_status]],
  sep = &quot; - &quot;
) %&gt;% 
  factor(levels = combs) %&gt;% 
  droplevels()
mcols(all_windows)[[&quot;distance&quot;]] &lt;- abs(
  mcols(all_windows)[[glue(&quot;{comps[[1]]}_centre&quot;)]] - mcols(all_windows)[[glue(&quot;{comps[[2]]}_centre&quot;)]]
)
## Remove any with an &#39;ambiguous&#39; status
all_windows &lt;- subset(all_windows, !is.na(status))

if (has_features) 
  all_windows$feature &lt;- bestOverlap(
    all_windows, external_features, var = &quot;feature&quot;, missing = &quot;no_feature&quot;
  ) %&gt;% 
  str_sep_to_title()

all_windows$hic &lt;- NA
if (has_hic) all_windows$hic &lt;- overlapsAny(all_windows, hic)

## Define promoters for mapping
feat_prom &lt;- feat_enh &lt;- GRanges()
if (has_features) {
  if (&quot;feature&quot; %in% colnames(mcols(external_features))) {
    feat_prom &lt;- external_features %&gt;%
      subset(str_detect(feature, &quot;[Pp]rom&quot;)) %&gt;%
      granges()
    feat_enh &lt;- external_features %&gt;%
      subset(str_detect(feature, &quot;[Ee]nhanc&quot;)) %&gt;%
      granges()
  }
}
prom4mapping &lt;- c(feat_prom, granges(gene_regions$promoter)) %&gt;%
  GenomicRanges::reduce()

all_windows &lt;- all_windows %&gt;% 
  select(any_of(c(&quot;region&quot;, &quot;feature&quot;, &quot;hic&quot;)), everything()) %&gt;% 
  mapByFeature(
    gtf_gene, prom = prom4mapping, enh = feat_enh, gi = hic,
    gr2gene = params$mapping$gr2gene, 
    prom2gene = params$mapping$prom2gene,
    enh2gene = params$mapping$enh2gene, gi2gene = params$mapping$gi2gene
  ) 
all_windows$detected &lt;- all_windows$gene_name
if (has_rnaseq) all_windows$detected &lt;- endoapply(
  all_windows$detected, intersect, rnaseq$gene_name
) 
n_mapped &lt;- all_windows %&gt;%
  select(status, gene_id) %&gt;% 
  as_tibble() %&gt;% 
  unnest(everything()) %&gt;% 
  distinct(status, gene_id) %&gt;% 
  group_by(status) %&gt;% 
  summarise(mapped = dplyr::n()) %&gt;% 
  arrange(desc(mapped)) </code></pre>
<p>A set of 18,401 common windows was formed by obtaining the union of
windows produced during the analysis of <em>AR</em> and <em>ER</em>
individually. These common windows were formed using any overlap between
target-specific windows. The window-to-gene mappings and the best
overlap with genomic regions was also reformed. The best overlap with
external features was also reformed.</p>
<p>In the original datasets, 14,648 windows were included for AR, whilst
6,108 windows were included for ER. The distance between each pair of
windows was found by taking the centre of the original sliding window
used for statistical analysis, and finding the difference. As windows
were detected in an unstranded manner, all distance values were set to
be positive. Taking the complete set of 18,401 windows, 2,353 (12.8%)
were considered as being detected in both datasets. The median distance
between windows found in both comparisons was 35bp. 0 peaks (0.0%) were
found to directly overlap.</p>
<p>Merged windows were compared across both targets and in order to
obtain the best classification for each window, the initial values of an
FDR-adjusted p-value &lt; 0.05 from either comparison were used to
consider a window as showing changed binding in at least one comparison.
A secondary step was then introduced for each window such that if one
target was significant and the other target had 1) received an
FDR-adjusted p-value &lt; 0.05 <em>using a point-based</em> <span
class="math inline">\(H_0\)</span>, and 2) had an estimated logFC beyond
the range specified for the range-based null hypothesis (<span
class="math inline">\(H_0: 0 &lt; |\mu| &lt; \lambda\)</span> for <span
class="math inline">\(\lambda = 0.26\)</span>), these windows were then
considered significant in the second comparison. This was to help
<em>reduce the number of windows incorrectly classified as
unchanged</em> whilst the alternative ChIP target was defined as
changed.</p>
<p>Whilst summary tables and figures are presented below, <strong>the
full set of windows and mapped genes were exported</strong> as
output/pairwise_comparisons/AR_ER/AR_E2_E2DHT-ER_E2_E2DHT-pairwise_comparison.csv.gz</p>
<pre class="r"><code>cp &lt;- htmltools::tags$em(
  glue(
    &quot;Summary of combined binding windows for &quot;,
    glue_collapse(comps, sep = &quot; and &quot;),
    &quot;. Windows were classified as described above. &quot;,
    &quot;The number of windows showing each combined pattern are given, along &quot;,
    &quot;with the number of unique genes mapped to each set of combined windows, &quot;,
    &quot;noting that some genes will be mapped to multiple sets of windows. &quot;,
    &quot;The % of all genes with a window mapped from either target is given in &quot;,
    &quot;the final column.&quot;
  )
)
tbl &lt;- all_windows %&gt;% 
  as_tibble() %&gt;% 
  rename_all(
    str_replace_all, pattern = &quot;\\.Vs\\.\\.&quot;, replacement = &quot; Vs. &quot;
  ) %&gt;% 
  rename_all(
    str_replace_all, pattern = &quot;\\.\\.&quot;, replacement = &quot;: &quot;
  ) %&gt;% 
  group_by(across(ends_with(&quot;_status&quot;))) %&gt;% 
  summarise(
    windows = dplyr::n(),
    mapped = length(unique(unlist(gene_id))),
    .groups = &quot;drop&quot;
  ) %&gt;% 
  mutate(
    `% genes` = mapped / length(unique(unlist(all_windows$gene_id)))
  ) %&gt;% 
  reactable(
    filterable = TRUE,
    showPageSizeOptions = TRUE,
    columns = list2(
      !!sym(c1_status) := colDef(
        name = comps[[1]],
        cell = function(value) {
          value &lt;- str_extract(value, &quot;Up|Down|Unchanged|Undetected&quot;)
          if (str_detect(value, &quot;Up&quot;)) value &lt;- paste(value, &quot;\u21E7&quot;)
          if (str_detect(value, &quot;Down&quot;)) value &lt;- paste(value, &quot;\u21E9&quot;)
          paste(targets[[1]], value)
        },
        style = function(value) {
          colour &lt;- case_when(
            str_detect(value, &quot;Up&quot;) ~ colours$direction[[&quot;up&quot;]],
            str_detect(value, &quot;Down&quot;) ~ colours$direction[[&quot;down&quot;]],
            str_detect(value, &quot;Unchanged&quot;) ~ colours$direction[[&quot;unchanged&quot;]],
            str_detect(value, &quot;Undetected&quot;) ~ colours$direction[[&quot;undetected&quot;]]
          )
          list(color = colour)
        },
        footer = htmltools::tags$b(&quot;Total&quot;)
      ),
      !!sym(c2_status) := colDef(
        name = comps[[2]],
        cell = function(value) {
          value &lt;- str_extract(value, &quot;Up|Down|Unchanged|Undetected&quot;)
          if (str_detect(value, &quot;Up&quot;)) value &lt;- paste(value, &quot;\u21E7&quot;)
          if (str_detect(value, &quot;Down&quot;)) value &lt;- paste(value, &quot;\u21E9&quot;)
          paste(targets[[2]], value)
        },
        style = function(value) {
          colour &lt;- case_when(
            str_detect(value, &quot;Up&quot;) ~ colours$direction[[&quot;up&quot;]],
            str_detect(value, &quot;Down&quot;) ~ colours$direction[[&quot;down&quot;]],
            str_detect(value, &quot;Unchanged&quot;) ~ colours$direction[[&quot;unchanged&quot;]],
            str_detect(value, &quot;Undetected&quot;) ~ colours$direction[[&quot;undetected&quot;]]
          )
          list(color = colour)
        }
      ),
      windows = colDef(
        name = &quot;Number of Windows&quot;, 
        cell = function(value) comma(value, 1),
        style = function(value) {
          bar_style(width = 0.9*value / length(all_windows), fill = &quot;#B3B3B3&quot;)
        },
        footer = htmltools::tags$b(comma(length(all_windows), 1))
      ),
      mapped = colDef(
        name = &quot;Mapped Genes&quot;, 
        cell = function(value) comma(value, 1),
        style = function(value) {
          bar_style(width = 0.9*value /length(unique(unlist(all_windows$gene_id))), fill = &quot;#B3B3B3&quot;)
        },
        footer = htmltools::tags$b(comma(length(unique(unlist(all_windows$gene_id))), 1))
      ),
      `% genes` = colDef(
        name = &quot;% Of All Mapped Genes&quot;,
        cell = function(x) percent(x, 0.1),
        style = function(value) {
          bar_style(width = 0.9*value, fill = &quot;#B3B3B3&quot;)
        }
      )
    )
  )
div(class = &quot;table&quot;,
    div(class = &quot;table-header&quot;,
        div(class = &quot;caption&quot;, cp),
        tbl
    )
)</code></pre>
<div class="table">
<div class="table-header">
<div class="caption">
<em>Summary of combined binding windows for AR and ER. Windows were classified as described above. The number of windows showing each combined pattern are given, along with the number of unique genes mapped to each set of combined windows, noting that some genes will be mapped to multiple sets of windows. The % of all genes with a window mapped from either target is given in the final column.</em>
</div>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-a69ae56ef5a536e67b08" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-a69ae56ef5a536e67b08">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"AR_status":["AR Up","AR Up","AR Up","AR Up","AR Down","AR Down","AR Unchanged","AR Unchanged","AR Unchanged","AR Undetected","AR Undetected"],"ER_status":["ER Up","ER Down","ER Unchanged","ER Undetected","ER Unchanged","ER Undetected","ER Up","ER Unchanged","ER Undetected","ER Up","ER Unchanged"],"windows":[240,2,1158,8942,1,17,1,951,3336,19,3734],"mapped":[260,1,1237,4621,1,6,0,899,2467,15,3397],"% genes":[0.0340715502555366,0.000131044424059756,0.162101952561918,0.605556283580134,0.000131044424059756,0.000786266544358538,0,0.117808937229721,0.323286594155419,0.00196566636089634,0.445157908530992]},"columns":[{"id":"AR_status","name":"AR","type":"factor","cell":["AR Up ⇧","AR Up ⇧","AR Up ⇧","AR Up ⇧","AR Down ⇩","AR Down ⇩","AR Unchanged","AR Unchanged","AR Unchanged","AR Undetected","AR Undetected"],"footer":{"name":"b","attribs":{},"children":["Total"]},"style":[{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#0571B0"},{"color":"#0571B0"},{"color":"#7F7F7F"},{"color":"#7F7F7F"},{"color":"#7F7F7F"},{"color":"#E5E5E5"},{"color":"#E5E5E5"}]},{"id":"ER_status","name":"ER","type":"factor","cell":["ER Up ⇧","ER Down ⇩","ER Unchanged","ER Undetected","ER Unchanged","ER Undetected","ER Up ⇧","ER Unchanged","ER Undetected","ER Up ⇧","ER Unchanged"],"style":[{"color":"#CA0020"},{"color":"#0571B0"},{"color":"#7F7F7F"},{"color":"#E5E5E5"},{"color":"#7F7F7F"},{"color":"#E5E5E5"},{"color":"#CA0020"},{"color":"#7F7F7F"},{"color":"#E5E5E5"},{"color":"#CA0020"},{"color":"#7F7F7F"}]},{"id":"windows","name":"Number of Windows","type":"numeric","cell":["240","2","1,158","8,942","1","17","1","951","3,336","19","3,734"],"footer":{"name":"b","attribs":{},"children":["18,401"]},"style":[{"backgroundImage":"linear-gradient(90deg, #B3B3B3 1.17384924732352%, transparent 1.17384924732352%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 0.00978207706102929%, transparent 0.00978207706102929%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 5.66382261833596%, transparent 5.66382261833596%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 43.735666539862%, transparent 43.735666539862%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 0.00489103853051465%, transparent 0.00489103853051465%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 0.083147655018749%, transparent 0.083147655018749%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 0.00489103853051465%, transparent 0.00489103853051465%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 4.65137764251943%, transparent 4.65137764251943%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 16.3165045377969%, transparent 16.3165045377969%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 0.0929297320797783%, transparent 0.0929297320797783%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 18.2631378729417%, transparent 18.2631378729417%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null}]},{"id":"mapped","name":"Mapped Genes","type":"numeric","cell":["260","1","1,237","4,621","1","6","0","899","2,467","15","3,397"],"footer":{"name":"b","attribs":{},"children":["7,631"]},"style":[{"backgroundImage":"linear-gradient(90deg, #B3B3B3 3.0664395229983%, transparent 3.0664395229983%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 0.0117939981653781%, transparent 0.0117939981653781%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 14.5891757305727%, transparent 14.5891757305727%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 54.500065522212%, transparent 54.500065522212%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 0.0117939981653781%, transparent 0.0117939981653781%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 0.0707639889922684%, transparent 0.0707639889922684%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 0%, transparent 0%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 10.6028043506749%, transparent 10.6028043506749%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 29.0957934739877%, transparent 29.0957934739877%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 0.176909972480671%, transparent 0.176909972480671%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 40.0642117677893%, transparent 40.0642117677893%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null}]},{"id":"% genes","name":"% Of All Mapped Genes","type":"numeric","cell":["3.4%","0.0%","16.2%","60.6%","0.0%","0.1%","0.0%","11.8%","32.3%","0.2%","44.5%"],"style":[{"backgroundImage":"linear-gradient(90deg, #B3B3B3 3.0664395229983%, transparent 3.0664395229983%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 0.0117939981653781%, transparent 0.0117939981653781%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 14.5891757305727%, transparent 14.5891757305727%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 54.500065522212%, transparent 54.500065522212%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 0.0117939981653781%, transparent 0.0117939981653781%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 0.0707639889922684%, transparent 0.0707639889922684%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 0%, transparent 0%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 10.6028043506749%, transparent 10.6028043506749%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 29.0957934739877%, transparent 29.0957934739877%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 0.176909972480671%, transparent 0.176909972480671%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, #B3B3B3 40.0642117677893%, transparent 40.0642117677893%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null}]}],"filterable":true,"showPageSizeOptions":true,"dataKey":"364213244579071c95ae27c6f72517fa"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div id="pairwise-differential-binding" class="section level2">
<h2>Pairwise Differential Binding</h2>
<pre class="r"><code>df &lt;- all_windows %&gt;% 
  as_tibble() %&gt;% 
  rename_all(
    str_replace_all, pattern = &quot;\\.Vs\\.\\.&quot;, replacement = &quot; Vs. &quot;
  ) %&gt;% 
  rename_all(
    str_replace_all, pattern = &quot;\\.\\.&quot;, replacement = &quot;: &quot;
  ) %&gt;% 
  dplyr::filter(!str_detect(status, &quot;Undetected&quot;)) %&gt;% 
  droplevels()
df %&gt;% 
  ggplot(
    aes(
      !!sym(c1_logfc), !!sym(c2_logfc),
      colour = status
    )
  ) +
  geom_point(alpha = 0.6) +
  geom_hline(yintercept = 0) +
  geom_hline(
    yintercept = c(1, -1) * lambda, 
    linetype = 2, colour = &quot;grey&quot;
  ) +
  geom_vline(xintercept = 0) +
  geom_vline(
    xintercept = c(1, -1)* lambda,
    linetype = 2, colour = &quot;grey&quot;
  ) +
  geom_text_repel(
    aes(label = lab),
    data = . %&gt;% 
      dplyr::filter(
        !str_detect(status, &quot;Unchanged.+Unchanged&quot;),
        vapply(detected, length, integer(1)) &gt; 0
      ) %&gt;% 
      mutate(
        total_lfc = abs(!!sym(c1_logfc)) + abs(!!sym(c2_logfc)) 
      ) %&gt;% 
      group_by(status) %&gt;%
      dplyr::filter(total_lfc == max(total_lfc)) %&gt;% 
      mutate(
        detected = vapply(detected, paste, character(1), collapse = &quot;; &quot;),
        lab = paste(status, detected, sep = &quot;: &quot;) %&gt;% str_wrap(25)
      ),
    size = 4, 
    bg.r = 0.001, bg.color = &quot;grey70&quot;,
    show.legend = FALSE
  ) +
  geom_text(
    aes(x, y, label = lab),
    data = . %&gt;%
      mutate(
        x = 0.9*max((!!sym(c1_logfc))),
        y = min(!!sym(c2_logfc))
      ) %&gt;% 
      summarise(
        x = unique(x), 
        y = unique(y),
        n = dplyr::n(),
        lab = paste(&quot;n =&quot;, comma(n, 1)),
        .groups = &quot;drop&quot;
      ),
    inherit.aes = FALSE
  ) +
  scale_colour_manual(values = direction_colours) +
  labs(
    x = paste(comps[[1]], &quot;logFC&quot;),
    y = paste(comps[[2]], &quot;logFC&quot;),
    colour = &quot;Status&quot;
  ) +
  theme(legend.position = &quot;top&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="AR_E2_E2DHT_ER_E2_E2DHT_pairwise_comparison_files/figure-html/plot-dbwin-1.png" alt="*Comparative changes in both AR and ER. The window with most extreme combined change in binding is shown with any mapped genes labelled. The range around zero used for range-based hypothesis testing is indicated with grey dashed lines.*" width="960" />
<p class="caption">
<em>Comparative changes in both AR and ER. The window with most extreme
combined change in binding is shown with any mapped genes labelled. The
range around zero used for range-based hypothesis testing is indicated
with grey dashed lines.</em>
</p>
</div>
</div>
<div id="logfc-by-region" class="section level2">
<h2>logFC By Region</h2>
<pre class="r"><code>df &lt;- all_windows %&gt;% 
  as_tibble() %&gt;% 
  rename_all(
    str_replace_all, pattern = &quot;\\.Vs\\.\\.&quot;, replacement = &quot; Vs. &quot;
  ) %&gt;% 
  rename_all(
    str_replace_all, pattern = &quot;\\.\\.&quot;, replacement = &quot;: &quot;
  ) %&gt;% 
  dplyr::filter(!str_detect(status, &quot;Undetected&quot;)) %&gt;% 
  droplevels() 
df %&gt;% 
  ggplot(
    aes(
      !!sym(c1_logfc), !!sym(c2_logfc),
      colour = status
    )
  ) +
  geom_point(alpha = 0.6) +
  geom_hline(yintercept = 0) +
  geom_hline(
    yintercept = c(1, -1) * lambda,
    linetype = 2, colour = &quot;grey&quot;
  ) +
  geom_vline(xintercept = 0) +
  geom_vline(
    xintercept = c(1, -1) * lambda, 
    linetype = 2, colour = &quot;grey&quot;
  ) +
  geom_text_repel(
    aes(label = detected),
    data = . %&gt;% 
      dplyr::filter(
        !str_detect(status, &quot;Unchanged.+Unchanged&quot;),
        vapply(detected, length, integer(1)) &gt; 0
      ) %&gt;% 
      mutate(
        total_lfc = abs(!!sym(c1_logfc)) + abs(!!sym(c2_logfc)) 
      ) %&gt;% 
      group_by(region) %&gt;% 
      dplyr::filter(total_lfc == max(total_lfc)) %&gt;% 
      mutate(
        detected = vapply(detected, paste, character(1), collapse = &quot;; &quot;) %&gt;% 
          str_wrap(25)
      ),
    size = 3, bg.color = &quot;grey70&quot;, bg.r = 0.001,
    show.legend = FALSE
  ) +
  geom_text(
    aes(x, y, label = lab),
    data = . %&gt;%
      mutate(
        x = 0.9*max(!!sym(c1_logfc)),
        y = min(!!sym(c2_logfc))
      ) %&gt;% 
      group_by(region) %&gt;%
      summarise(
        x = unique(x), 
        y = unique(y),
        n = dplyr::n(),
        lab = paste(&quot;n =&quot;, comma(n, 1)),
        .groups = &quot;drop&quot;
      ),
    inherit.aes = FALSE
  ) +
  facet_wrap(~region) +
  scale_colour_manual(values = direction_colours) +
  labs(
    x = paste(comps[[1]], &quot;logFC&quot;),
    y = paste(comps[[2]], &quot;logFC&quot;),
    colour = &quot;Status&quot;
  ) +
  theme(legend.position = &quot;top&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="AR_E2_E2DHT_ER_E2_E2DHT_pairwise_comparison_files/figure-html/plot-dbwin-by-region-1.png" alt="*Comparative changes in both AR and ER. The window with most extreme combined change in binding for each separate region is shown with any mapped genes labelled. The range around zero used for range-based hypothesis testing is indicated with grey dashed lines.*" width="960" />
<p class="caption">
<em>Comparative changes in both AR and ER. The window with most extreme
combined change in binding for each separate region is shown with any
mapped genes labelled. The range around zero used for range-based
hypothesis testing is indicated with grey dashed lines.</em>
</p>
</div>
</div>
<div id="logfc-by-feature" class="section level2">
<h2>logFC By Feature</h2>
<pre class="r"><code>df &lt;- all_windows %&gt;% 
  as_tibble() %&gt;% 
  rename_all(
    str_replace_all, pattern = &quot;\\.Vs\\.\\.&quot;, replacement = &quot; Vs. &quot;
  ) %&gt;% 
  rename_all(
    str_replace_all, pattern = &quot;\\.\\.&quot;, replacement = &quot;: &quot;
  ) %&gt;% 
  dplyr::filter(!str_detect(status, &quot;Undetected&quot;)) %&gt;% 
  droplevels() 
df %&gt;% 
  ggplot(
    aes(
      !!sym(c1_logfc), !!sym(c2_logfc),
      colour = status
    )
  ) +
  geom_point(alpha = 0.6) +
  geom_hline(yintercept = 0) +
  geom_hline(
    yintercept = c(1, -1) * lambda, 
    linetype = 2, colour = &quot;grey&quot;
  ) +
  geom_vline(xintercept = 0) +
  geom_vline(
    xintercept = c(1, -1) * lambda,
    linetype = 2, colour = &quot;grey&quot;
  ) +
  geom_text_repel(
    aes(label = detected),
    data = . %&gt;% 
      dplyr::filter(!str_detect(status, &quot;Unchanged.+Unchanged&quot;)) %&gt;% 
      mutate(
        total_lfc = abs(!!sym(c1_logfc)) + abs(!!sym(c2_logfc)) 
      ) %&gt;% 
      group_by(feature) %&gt;% 
      dplyr::filter(total_lfc == max(total_lfc)) %&gt;% 
      mutate(
        detected = vapply(detected, paste, character(1), collapse = &quot;; &quot;) %&gt;% 
          str_wrap(25)
      ) %&gt;% 
      dplyr::filter(detected != &quot;&quot;),
    size = 3, bg.color = &quot;grey70&quot;, bg.r = 0.001,
    show.legend = FALSE
  ) +
  geom_text(
    aes(x, y, label = lab),
    data = . %&gt;%
      mutate(
        x = 0.9 * max(!!sym(c1_logfc)),
        y = min(!!sym(c2_logfc))
      ) %&gt;% 
      group_by(feature) %&gt;%
      summarise(
        x = unique(x), 
        y = unique(y),
        n = dplyr::n(),
        lab = paste(&quot;n =&quot;, comma(n, 1)),
        .groups = &quot;drop&quot;
      ),
    inherit.aes = FALSE
  ) +
  facet_wrap(~feature) +
  scale_colour_manual(values = direction_colours[levels(df$status)]) +
  labs(
    x = paste(comps[[1]], &quot;logFC&quot;),
    y = paste(comps[[2]], &quot;logFC&quot;),
    colour = &quot;Status&quot;
  ) +
  theme(legend.position = &quot;top&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="AR_E2_E2DHT_ER_E2_E2DHT_pairwise_comparison_files/figure-html/plot-dbwin-by-feature-1.png" alt="*Comparative changes in both AR and ER. The window with most extreme combined change in binding for each separate feature is shown with any mapped genes labelled. The range around zero used for range-based hypothesis testing is indicated with grey dashed lines.*" width="960" />
<p class="caption">
<em>Comparative changes in both AR and ER. The window with most extreme
combined change in binding for each separate feature is shown with any
mapped genes labelled. The range around zero used for range-based
hypothesis testing is indicated with grey dashed lines.</em>
</p>
</div>
</div>
<div id="ar-by-er-status" class="section level2">
<h2>AR By ER Status</h2>
<pre class="r"><code>all_windows %&gt;%
  as_tibble() %&gt;% 
  rename_all(
    str_replace_all, pattern = &quot;\\.Vs\\.\\.&quot;, replacement = &quot; Vs. &quot;
  ) %&gt;% 
  rename_all(
    str_replace_all, pattern = &quot;\\.\\.&quot;, replacement = &quot;: &quot;
  ) %&gt;% 
  dplyr::filter(
    !!sym(c1_status) != paste(comps[[1]], &quot;Undetected&quot;)
  ) %&gt;% 
  mutate(
    !!sym(c1_status) := fct_relabel(
      !!sym(c1_status), 
      str_extract, 
      pattern = &quot;Up|Down|Unchanged|Undetected&quot;
    ),
    !!sym(c2_status) := fct_relabel(
      !!sym(c2_status), 
      str_extract, 
      pattern = &quot;Up|Down|Unchanged|Undetected&quot;
    )
  ) %&gt;% 
  droplevels() %&gt;% 
  ggplot(
    aes(
      !!sym(c1_logcpm), 
      !!sym(c1_logfc),
      colour = !!sym(c1_status), 
      fill = !!sym(c2_status)
    )
  ) +
  geom_point(alpha = 0.6) +
  geom_xsideboxplot(
    aes(y = !!sym(c2_status)),
    orientation = &quot;y&quot;, colour = &quot;black&quot;
  ) +
  geom_ysideboxplot(
    aes(x = !!sym(c2_status)),
    orientation = &quot;x&quot;, colour = &quot;black&quot;
  ) +
  geom_ysideline(
    aes(x, y),
    data = tibble(x = seq(0, 5), y = 0),
    inherit.aes = FALSE
  ) +
  geom_hline(yintercept = 0) +
  geom_hline(yintercept = c(1, -1) * lambda, linetype = 2, colour = &quot;grey30&quot;) +
  scale_xsidey_discrete() +
  scale_ysidex_discrete(guide = guide_axis(angle = 90)) +
  scale_colour_manual(
    values = colours$direction %&gt;%
      setNames(str_to_title(names(.))) %&gt;% 
      .[!str_detect(names(.), &quot;Undetected&quot;)]
  ) +
  scale_fill_manual(
    values = colours$direction %&gt;%
      setNames(str_to_title(names(.)))
  ) +
  labs(
    x = paste(comps[[1]], &quot;logCPM&quot;),
    y = paste(comps[[1]], &quot;logFC&quot;),
    colour = str_replace(comps[[1]], &quot;: &quot;, &quot;\n&quot;),
    fill = str_replace_all(comps[[2]], &quot;: &quot;, &quot;\n&quot;)
  ) +
  theme(
    ggside.panel.scale.y = .2,
    ggside.panel.scale.x = .25,
    axis.title.x = element_text(hjust = 0.4, vjust = 1),
    axis.title.y = element_text(hjust = 0.4, vjust = 1)
  )</code></pre>
<div class="figure" style="text-align: center">
<img src="AR_E2_E2DHT_ER_E2_E2DHT_pairwise_comparison_files/figure-html/comps1-by2-1.png" alt="*Differential binding patterns of AR by ER status. AR values are shown as points coloured by their individual classification as Up, Down or Unchanged, with the distributions of these points shown as boxplots by ER status in the side panels.*" width="960" />
<p class="caption">
<em>Differential binding patterns of AR by ER status. AR values are
shown as points coloured by their individual classification as Up, Down
or Unchanged, with the distributions of these points shown as boxplots
by ER status in the side panels.</em>
</p>
</div>
</div>
<div id="er-by-ar-status" class="section level2">
<h2>ER By AR Status</h2>
<pre class="r"><code>all_windows %&gt;%
  as_tibble() %&gt;% 
  rename_all(
    str_replace_all, pattern = &quot;\\.Vs\\.\\.&quot;, replacement = &quot; Vs. &quot;
  ) %&gt;% 
  rename_all(
    str_replace_all, pattern = &quot;\\.\\.&quot;, replacement = &quot;: &quot;
  ) %&gt;% 
  as_tibble() %&gt;% 
  dplyr::filter(
    !!sym(c2_status) != paste(comps[[2]], &quot;Undetected&quot;)
  ) %&gt;% 
  mutate(
    !!sym(c1_status) := fct_relabel(
      !!sym(c1_status), 
      str_extract, 
      pattern = &quot;Up|Down|Unchanged|Undetected&quot;
    ),
    !!sym(c2_status) := fct_relabel(
      !!sym(c2_status), 
      str_extract, 
      pattern = &quot;Up|Down|Unchanged|Undetected&quot;
    )
  ) %&gt;% 
  droplevels() %&gt;% 
  ggplot(
    aes(
      !!sym(c2_logcpm), 
      !!sym(c2_logfc),
      colour = !!sym(c2_status), 
      fill = !!sym(c1_status)
    )
  ) +
  geom_point(alpha = 0.6) +
  geom_xsideboxplot(
    aes(y = !!sym(c1_status)),
    orientation = &quot;y&quot;, colour = &quot;black&quot;
  ) +
  geom_ysideboxplot(
    aes(x = !!sym(c1_status)),
    orientation = &quot;x&quot;, colour = &quot;black&quot;
  ) +
  geom_ysideline(
    aes(x, y),
    data = tibble(x = seq(0, 5), y = 0),
    inherit.aes = FALSE
  ) +
  geom_hline(yintercept = 0) +
  geom_hline(yintercept = c(1, -1) * lambda, linetype = 2, colour = &quot;grey30&quot;) +
  scale_xsidey_discrete() +
  scale_ysidex_discrete(guide = guide_axis(angle = 90)) +
  scale_colour_manual(
    values = colours$direction %&gt;%
      setNames(str_to_title(names(.))) %&gt;% 
      .[!str_detect(names(.), &quot;Undetected&quot;)]
  ) +
  scale_fill_manual(
    values = colours$direction %&gt;%
      setNames(str_to_title(names(.)))
  ) +
  labs(
    x = paste(comps[[2]], &quot;logCPM&quot;),
    y = paste(comps[[2]], &quot;logFC&quot;),
    colour = str_replace(comps[[2]], &quot;: &quot;, &quot;\n&quot;),
    fill = str_replace_all(comps[[1]], &quot;: &quot;, &quot;\n&quot;)
  ) +
  theme(
    ggside.panel.scale.y = .2,
    ggside.panel.scale.x = .25,
    axis.title.x = element_text(hjust = 0.4, vjust = 1),
    axis.title.y = element_text(hjust = 0.4, vjust = 1)
  )</code></pre>
<div class="figure" style="text-align: center">
<img src="AR_E2_E2DHT_ER_E2_E2DHT_pairwise_comparison_files/figure-html/comps2-by1-1.png" alt="*Differential binding patterns of ER by AR status. ER values are shown as points coloured by their individual classification as Up, Down or Unchanged, with the distributions of these shown as boxplots by AR status in the side panels.*" width="960" />
<p class="caption">
<em>Differential binding patterns of ER by AR status. ER values are
shown as points coloured by their individual classification as Up, Down
or Unchanged, with the distributions of these shown as boxplots by AR
status in the side panels.</em>
</p>
</div>
</div>
<div id="distribution-by-region" class="section level2">
<h2>Distribution By Region</h2>
<pre class="r"><code>all_windows$grp &lt;- paste(
  mcols(all_windows)[[c1_status]], 
  mcols(all_windows)[[c2_status]], 
  all_windows$region
)
df &lt;- all_windows %&gt;% 
  split(.$grp) %&gt;% 
  lapply(
    function(x) {
      tibble(
        &quot;{c1_status}&quot; := unique(mcols(x)[[c1_status]]),
        &quot;{c2_status}&quot; := unique(mcols(x)[[c2_status]]),
        region = unique(mcols(x)[[&quot;region&quot;]]),
        n = length(x)
      )
    }
  ) %&gt;% 
  bind_rows() %&gt;% 
  dplyr::filter(
    str_detect(!!sym(c1_status), &quot;Up|Down&quot;) | str_detect(!!sym(c2_status), &quot;Up|Down&quot;)
  ) %&gt;% 
  group_by(!!sym(c1_status), !!sym(c2_status)) %&gt;% 
  mutate(total = sum(n)) %&gt;% 
  ungroup() %&gt;% 
  arrange(!!sym(c1_status), !!sym(c2_status), region) %&gt;% 
  mutate(
    x0 = as.integer(!!sym(c1_status)),
    y0 = as.integer(!!sym(c2_status)),
    r = total / sum(total),
    r =  0.5* r / max(r),
    !!sym(c1_status) := fct_relabel(!!sym(c1_status), str_extract, pattern = &quot;Up|Down|Un.+&quot;),
    !!sym(c2_status) := fct_relabel(!!sym(c2_status), str_extract, pattern = &quot;Up|Down|Un.+&quot;)
  )
df %&gt;% 
  ggplot() +
  ggforce::stat_pie(
    aes(x0 = x0, y0 = y0, r0 = 0, r = r, fill = region, amount = n)
  ) +
  geom_label(
    aes(x0, y0, label = comma(total)),
    data = . %&gt;% 
      distinct(x0, y0, total) %&gt;% 
      dplyr::filter(total &gt; 0.05 * sum(total)),
    alpha = 0.8, size = 4
  ) +
  coord_equal() +
  scale_fill_manual(values = region_colours) +
  scale_x_continuous(
    breaks = seq_along(levels(df[[c1_status]])), 
    labels = levels(df[[c1_status]])
  ) +
  scale_y_continuous(
    breaks = seq_along(levels(df[[c2_status]])), 
    labels = levels(df[[c2_status]])
  ) +
  labs(
    x = paste(comps[[1]], &quot;Status&quot;), 
    y = paste(comps[[2]], &quot;Status&quot;), 
    fill = &quot;Region&quot;
  ) +
  theme(
    panel.grid = element_blank()
  )</code></pre>
<div class="figure" style="text-align: center">
<img src="AR_E2_E2DHT_ER_E2_E2DHT_pairwise_comparison_files/figure-html/status-by-region-1.png" alt="*Distribution of binding patterns by genomic regions as defined in earlier sections. Windows are only shown if change was detected in one of the comparisons.*" width="960" />
<p class="caption">
<em>Distribution of binding patterns by genomic regions as defined in
earlier sections. Windows are only shown if change was detected in one
of the comparisons.</em>
</p>
</div>
</div>
<div id="distribution-by-feature" class="section level2">
<h2>Distribution By Feature</h2>
<pre class="r"><code>all_windows$grp &lt;- paste(
  mcols(all_windows)[[c1_status]], 
  mcols(all_windows)[[c2_status]], 
  all_windows$feature
)
df &lt;- all_windows %&gt;% 
  split(.$grp) %&gt;% 
  lapply(
    function(x) {
      tibble(
        &quot;{c1_status}&quot; := unique(mcols(x)[[c1_status]]),
        &quot;{c2_status}&quot; := unique(mcols(x)[[c2_status]]),
        feature = unique(mcols(x)[[&quot;feature&quot;]]),
        n = length(x)
      )
    }
  ) %&gt;% 
  bind_rows() %&gt;% 
  dplyr::filter(
    str_detect(!!sym(c1_status), &quot;Up|Down&quot;) | str_detect(!!sym(c2_status), &quot;Up|Down&quot;)
  ) %&gt;% 
  group_by(!!sym(c1_status), !!sym(c2_status)) %&gt;% 
  mutate(total = sum(n)) %&gt;% 
  ungroup() %&gt;% 
  arrange(!!sym(c1_status), !!sym(c2_status), feature) %&gt;% 
  mutate(
    x0 = as.integer(!!sym(c1_status)),
    y0 = as.integer(!!sym(c2_status)),
    r = total / sum(total),
    r =  0.5* r / max(r),
    !!sym(c1_status) := fct_relabel(!!sym(c1_status), str_extract, pattern = &quot;Up|Down|Un.+&quot;),
    !!sym(c2_status) := fct_relabel(!!sym(c2_status), str_extract, pattern = &quot;Up|Down|Un.+&quot;)
  )
df %&gt;% 
  ggplot() +
  ggforce::stat_pie(
    aes(x0 = x0, y0 = y0, r0 = 0, r = r, fill = feature, amount = n)
  ) +
  geom_label(
    aes(x0, y0, label = comma(total)),
    data = . %&gt;% 
      distinct(x0, y0, total) %&gt;% 
      dplyr::filter(total &gt; 0.05 * sum(total)),
    alpha = 0.8, size = 4
  ) +
  coord_equal() +
  scale_fill_manual(
    values = feature_colours %&gt;% 
      setNames(str_sep_to_title(names(.))) %&gt;% 
      .[unique(df$feature)]
  ) +
  scale_x_continuous(
    breaks = seq_along(levels(df[[c1_status]])), 
    labels = levels(df[[c1_status]])
  ) +
  scale_y_continuous(
    breaks = seq_along(levels(df[[c2_status]])), 
    labels = levels(df[[c2_status]])
  ) +
  labs(
    x = paste(comps[[1]], &quot;Status&quot;), 
    y = paste(comps[[2]], &quot;Status&quot;), 
    fill = &quot;External Feature&quot;
  ) +
  theme(
    panel.grid = element_blank()
  )</code></pre>
<div class="figure" style="text-align: center">
<img src="AR_E2_E2DHT_ER_E2_E2DHT_pairwise_comparison_files/figure-html/status-by-feature-1.png" alt="*Distribution of binding patterns by external features as defined in earlier sections. Windows are only shown if change was detected in one of the comparisons.*" width="960" />
<p class="caption">
<em>Distribution of binding patterns by external features as defined in
earlier sections. Windows are only shown if change was detected in one
of the comparisons.</em>
</p>
</div>
</div>
<div id="distance-by-status" class="section level2">
<h2>Distance By Status</h2>
<pre class="r"><code>all_windows %&gt;% 
  as_tibble() %&gt;% 
  dplyr::filter(!is.na(distance)) %&gt;% 
  arrange(status, distance) %&gt;% 
  group_by(status) %&gt;% 
  mutate(
    n = dplyr::n(),
    q = seq_along(status) / n
  ) %&gt;% 
  dplyr::filter(distance &lt; 1e3, !str_detect(status, &quot;Unchanged.+Unchanged&quot;)) %&gt;% 
  ggplot(aes(distance, q, colour = status)) +
  geom_line(linewidth = 1) +
  scale_x_continuous(breaks = seq(0, 10, by = 2) * 100) +
  scale_y_continuous(
    labels = percent, expand = expansion(c(0, 0.05)), 
    breaks = seq(0, 1, by = 0.2)
  ) +
  scale_colour_manual(
    values = direction_colours
  ) +
  labs(
    x = &quot;Distance Between Window Centres (bp)&quot;,
    y = &quot;Percentile&quot;,
    colour = &quot;Combined Status&quot;
  )</code></pre>
<div class="figure" style="text-align: center">
<img src="AR_E2_E2DHT_ER_E2_E2DHT_pairwise_comparison_files/figure-html/distances-bystatus-1.png" alt="*Distances between windows where maximal signal was detected for each target. Windows are only shown if change was detected in one or more comparisons. The x-axis is truncated at 1kb*" width="960" />
<p class="caption">
<em>Distances between windows where maximal signal was detected for each
target. Windows are only shown if change was detected in one or more
comparisons. The x-axis is truncated at 1kb</em>
</p>
</div>
</div>
<div id="distance-by-region" class="section level2">
<h2>Distance by Region</h2>
<pre class="r"><code>all_windows %&gt;% 
  as_tibble() %&gt;% 
  dplyr::filter(!is.na(distance)) %&gt;% 
  arrange(region, distance) %&gt;% 
  group_by(region) %&gt;% 
  mutate(
    n = dplyr::n(),
    q = seq_along(region) / n
  ) %&gt;% 
  dplyr::filter(distance &lt; 1e3) %&gt;% 
  ggplot(aes(distance, q, colour = region)) +
  geom_line(linewidth = 1) +
  scale_x_continuous(breaks = seq(0, 10, by = 2) * 100) +
  scale_y_continuous(
    labels = percent, expand = expansion(c(0, 0.05)), 
    breaks = seq(0, 1, by = 0.2)
  ) +
  scale_colour_manual(values = region_colours) +
  labs(
    x = &quot;Distance Between Window Centres (bp)&quot;,
    y = &quot;Percentile&quot;,
    colour = &quot;Genomic Region&quot;
  )</code></pre>
<div class="figure" style="text-align: center">
<img src="AR_E2_E2DHT_ER_E2_E2DHT_pairwise_comparison_files/figure-html/distance-by-region-1.png" alt="*Distribution of distances between peaks separated by genomic region. All sites are included regardless of changed binding. The x-axis is truncated 1kb.*" width="768" />
<p class="caption">
<em>Distribution of distances between peaks separated by genomic region.
All sites are included regardless of changed binding. The x-axis is
truncated 1kb.</em>
</p>
</div>
</div>
<div id="distance-by-feature" class="section level2">
<h2>Distance By Feature</h2>
<pre class="r"><code>all_windows %&gt;% 
  as_tibble() %&gt;% 
  dplyr::filter(!is.na(distance)) %&gt;% 
  mutate(feature = factor(feature, levels = names(feature_colours))) %&gt;% 
  arrange(feature, distance) %&gt;% 
  group_by(feature) %&gt;% 
  mutate(
    n = dplyr::n(),
    q = seq_along(feature) / n
  ) %&gt;% 
  dplyr::filter(distance &lt; 1e3) %&gt;% 
  droplevels() %&gt;% 
  ggplot(aes(distance, q, colour = feature)) +
  geom_line(linewidth = 1) +
  scale_x_continuous(breaks = seq(0, 10, by = 2) * 100) +
  scale_y_continuous(
    labels = percent, expand = expansion(c(0, 0.05)), 
    breaks = seq(0, 1, by = 0.2)
  ) +
  scale_colour_manual(values = unlist(feature_colours)) +
  labs(
    x = &quot;Distance Between Window Centres (bp)&quot;,
    y = &quot;Percentile&quot;,
    colour = &quot;External Feature&quot;
  )</code></pre>
<div class="figure" style="text-align: center">
<img src="AR_E2_E2DHT_ER_E2_E2DHT_pairwise_comparison_files/figure-html/distance-by-feature-1.png" alt="*Distribution of distances between peaks separated by external feature. All sites are included regardless of changed binding. The x-axis is truncated 1kb.*" width="768" />
<p class="caption">
<em>Distribution of distances between peaks separated by external
feature. All sites are included regardless of changed binding. The
x-axis is truncated 1kb.</em>
</p>
</div>
</div>
</div>
<div id="combined-binding" class="section level1">
<h1>Combined Binding</h1>
<div id="heatmaps" class="section level2 tabset">
<h2 class="tabset">Heatmaps</h2>
<p>A series of profile heatmaps were generated for any set of pairwise
comparisons with more than 25 ranges, and where binding was detected as
changed in at least one of the comparisons. If comparing two targets,
peaks were centred at the mid-point between the ranges of individual
maximal signal. If only one target is detected within a range, peaks are
centred at the range with maximal signal. Scales for fill and average
signal (both logCPM) are held constant across all plots for easier
comparison between groups.</p>
<pre class="r"><code>## Set the BWFL as a single object incorporating target &amp; treatment names
fl &lt;- seq_along(pairs) %&gt;% 
  lapply(
    function(x) {
      file.path(
        macs2_path[[x]],
         glue(&quot;{names(pairs)[x]}_{pairs[[x]]}_merged_treat_pileup.bw&quot;)
      ) %&gt;% 
        setNames(paste(names(pairs)[[x]], pairs[[x]]))
    } 
  )%&gt;% 
  unlist() %&gt;% 
  .[!duplicated(.)]
bwfl &lt;- setNames(BigWigFileList(fl), names(fl))
grl_to_plot &lt;- all_windows %&gt;% 
  split(.$status) %&gt;% 
  .[str_detect(names(.),&quot;Up|Down&quot;)] %&gt;% 
  .[vapply(., length, integer(1)) &gt; 25] %&gt;% # Only show groups with &gt; 10 ranges
  lapply(
    mutate, 
    centre = rowMeans(cbind(!!sym(c1_centre), !!sym(c2_centre)), na.rm = TRUE),
    rng = paste0(seqnames, &quot;:&quot;, as.integer(centre))
  ) %&gt;% 
  lapply(function(x) GRanges(x$rng, seqinfo = sq)) %&gt;% 
  GRangesList()
profile_width &lt;- 5e3
x_lab &lt;- c(
  seq(0, floor(0.5 * profile_width / 1e3), length.out = 3),
  seq(-floor(0.5 * profile_width / 1e3), 0, length.out = 3)
) %&gt;% 
  sort() %&gt;% 
  unique()
n_bins &lt;- 100
sig_profiles &lt;- mclapply(
  grl_to_plot, 
  function(x) getProfileData(
    bwfl, x, upstream = profile_width / 2, bins = n_bins
  ),
  mc.cores = min(length(grl_to_plot), threads)
)
profile_heatmaps &lt;- sig_profiles %&gt;% 
  mclapply(
    plotProfileHeatmap,
    profileCol = &quot;profile_data&quot;, 
    xLab = &quot;Distance from Centre (bp)&quot;,
    fillLab = &quot;logCPM&quot;,
    colour = &quot;name&quot;,
    mc.cores = min(length(grl_to_plot), threads)
  ) 
fill_range &lt;- profile_heatmaps %&gt;% 
  lapply(function(x) x$data[,&quot;score&quot;]) %&gt;% 
  unlist() %&gt;%
  c(0) %&gt;% 
  range()
sidey_range &lt;- profile_heatmaps %&gt;% 
  lapply(function(x) x$layers[[3]]$data$y) %&gt;% 
  unlist() %&gt;% 
  range()
profile_heatmaps &lt;- profile_heatmaps %&gt;% 
  lapply(
    function(x) {
      x + 
        scale_x_continuous(
          breaks = x_lab * 1e3, labels = x_lab, expand = expansion(0)
        ) +
        scale_xsidey_continuous(limits = sidey_range) +
        scale_fill_gradientn(colours = colours$heatmaps, limits = fill_range) +
        scale_colour_manual(
          values = lapply(
            seq_along(pairs), 
            function(i) setNames(
              treat_colours[pairs[[i]]], 
              paste(names(pairs)[[i]], pairs[[i]])
            )
          ) %&gt;%
            unlist() %&gt;% 
            .[!duplicated(names(.))]
        ) +
        labs(
          x = &quot;Distance from Centre (kb)&quot;,
          fill = &quot;logCPM&quot;, linetype = &quot;Union\nPeak\nOverlap&quot;,
          colour = &quot;Sample&quot;
        ) 
    }
  )</code></pre>
<pre class="r"><code>htmltools::tagList(
  mclapply(
    seq_along(profile_heatmaps),
    function(x) {
      ## Export the image
      pw &lt;- names(profile_heatmaps)[[x]] 
      img_out &lt;- file.path(
        fig_path,
        pw %&gt;% 
          str_remove_all(&quot;(:|Vs. |- )&quot;) %&gt;% 
          str_replace_all(&quot; &quot;, &quot;_&quot;) %&gt;% 
          str_replace_all(&quot;_-_&quot;, &quot;-&quot;) %&gt;% 
          paste0(&quot;_profile_heatmap.&quot;, fig_type)
      )
      n_ranges &lt;- length(unique(profile_heatmaps[[x]]$data$range))
      n_samples&lt;- length(unique(profile_heatmaps[[x]]$data$name))
      fig_fun(
        filename = img_out,
        width = knitr::opts_current$get(&quot;fig.width&quot;) * n_samples / 4, 
        height = min(
          3.5 + knitr::opts_current$get(&quot;fig.height&quot;) * n_ranges / 1.5e3, 
          10
        )
      )
      print(profile_heatmaps[[x]])
      dev.off()
      ## Create html tags
      fig_link &lt;- str_extract(img_out, &quot;assets.+&quot;)
      cp &lt;- htmltools::tags$em(
        glue(
          &quot;
        All {n_ranges} ranges showing the pattern of {pw}. Ranges are centered 
        at the midpoint between ranges identified as the maximal signal range.
        &quot;
        )
      )
      htmltools::div(
        htmltools::div(
          id = img_out %&gt;% 
            basename() %&gt;% 
            str_remove_all(glue(&quot;.{fig_type}$&quot;)) %&gt;% 
            str_to_lower() %&gt;% 
            str_replace_all(&quot;_&quot;, &quot;-&quot;),
          class=&quot;section level3&quot;,
          htmltools::h3(pw),
          htmltools::div(
            class = &quot;figure&quot;, style = &quot;text-align: center&quot;,
            htmltools::img(src = fig_link, width = &quot;100%&quot;),
            htmltools::tags$caption(cp)
          )
        )
      )
    },
    mc.cores = threads
  )
)</code></pre>
<div>
<div id="ar-up-er-up-profile-heatmap" class="section level3">
<h3>AR Up - ER Up</h3>
<div class="figure" style="text-align: center">
<img src="assets/AR_E2_E2DHT-ER_E2_E2DHT/AR_Up_ER_Up_profile_heatmap.png" width="100%"/>
<caption>
<em>All 240 ranges showing the pattern of AR Up - ER Up. Ranges are centered 
at the midpoint between ranges identified as the maximal signal range.</em>
</caption>
</div>
</div>
</div>
<div>
<div id="ar-up-er-unchanged-profile-heatmap" class="section level3">
<h3>AR Up - ER Unchanged</h3>
<div class="figure" style="text-align: center">
<img src="assets/AR_E2_E2DHT-ER_E2_E2DHT/AR_Up_ER_Unchanged_profile_heatmap.png" width="100%"/>
<caption>
<em>All 1158 ranges showing the pattern of AR Up - ER Unchanged. Ranges are centered 
at the midpoint between ranges identified as the maximal signal range.</em>
</caption>
</div>
</div>
</div>
<div>
<div id="ar-up-er-undetected-profile-heatmap" class="section level3">
<h3>AR Up - ER Undetected</h3>
<div class="figure" style="text-align: center">
<img src="assets/AR_E2_E2DHT-ER_E2_E2DHT/AR_Up_ER_Undetected_profile_heatmap.png" width="100%"/>
<caption>
<em>All 8942 ranges showing the pattern of AR Up - ER Undetected. Ranges are centered 
at the midpoint between ranges identified as the maximal signal range.</em>
</caption>
</div>
</div>
</div>
</div>
<div id="windows-with-the-strongest-signal"
class="section level2 tabset">
<h2 class="tabset">Windows With The Strongest Signal</h2>
<p>For the initial set of visualisations, the sets of windows with the
strongest signal were selected from within each group. Groups were
restricted to those where differential binding was seen in at least
comparison.</p>
<pre class="r"><code>grl_to_plot &lt;- all_windows %&gt;% 
  filter(
    !str_detect(status, &quot;Un.+Un&quot;),
    vapply(detected, length, integer(1)) &gt; 0
  ) %&gt;% 
  as_tibble() %&gt;% 
  rename_all(
    str_replace_all, pattern = &quot;\\.Vs\\.\\.&quot;, replacement = &quot; Vs. &quot;
  ) %&gt;% 
  rename_all(
    str_replace_all, pattern = &quot;\\.\\.&quot;, replacement = &quot;: &quot;
  ) %&gt;% 
  nest(
    AveExpr = ends_with(&quot;AveExpr&quot;)
  ) %&gt;% 
  mutate(
    max_signal = vapply(AveExpr, function(x) max(unlist(x)), numeric(1))
  ) %&gt;% 
  group_by(status) %&gt;% 
  filter(max_signal == max(max_signal)) %&gt;% 
  dplyr::select(
    range, status, ends_with(&quot;logFC&quot;)
  ) %&gt;% 
  droplevels() %&gt;% 
  split(.$status) %&gt;% 
  lapply(pull, &quot;range&quot;) %&gt;% 
  lapply(function(x) x[1]) %&gt;% 
  lapply(GRanges, seqinfo = sq) %&gt;% 
  GRangesList()</code></pre>
<pre class="r"><code>## The coverage
bwfl &lt;- seq_along(pairs) %&gt;% 
  lapply(
    function(x) {
      file.path(
        macs2_path[[x]], 
        glue(&quot;{names(pairs)[x]}_{pairs[[x]]}_merged_treat_pileup.bw&quot;)
      ) %&gt;% 
        BigWigFileList() %&gt;% 
        setNames(pairs[[x]])
    } 
  ) %&gt;% 
  setNames(comps)
line_col &lt;- lapply(bwfl, function(x) treat_colours[names(x)])

## Coverage annotations
annot &lt;- comps %&gt;% 
  lapply(
    function(x) {
      col &lt;- glue(&quot;{x}_status&quot;)
      select(all_windows, all_of(col)) %&gt;% 
        mutate(
          status = str_extract(!!sym(col), &quot;Down|Up|Unchanged|Undetected&quot;)
        ) %&gt;% 
        splitAsList(.$status) %&gt;% 
        lapply(granges) %&gt;% 
        lapply(unique) %&gt;% 
        GRangesList()
    }
  ) %&gt;% 
  setNames(comps)
annot_col &lt;- unlist(colours$direction) %&gt;% 
  setNames(str_to_title(names(.)))



## Coverage y-limits
gr &lt;- unlist(grl_to_plot)
y_lim &lt;- lapply(
  bwfl,
  function(x) {
    cov &lt;- lapply(x, import.bw, which = gr)
    unlist(lapply(cov, function(y) max(y$score))) %&gt;% 
      c(0) %&gt;% 
      range()
  }
)

## The features track
feat_gr &lt;- gene_regions %&gt;% 
  lapply(granges) %&gt;% 
  GRangesList()
feature_colours &lt;- colours$regions
if (has_features) {
  feat_gr &lt;- list(Regions = feat_gr)
  feat_gr$Features &lt;- splitAsList(external_features, external_features$feature)
  feature_colours &lt;- list(
    Regions = unlist(colours$regions),
    Features = unlist(colours$features)
  )
}

## The genes track
hfgc_genes &lt;- trans_models
gene_col &lt;- &quot;grey&quot;
if (has_rnaseq) {
  rna_lfc_col &lt;- colnames(rnaseq)[str_detect(str_to_lower(colnames(rnaseq)), &quot;logfc&quot;)][1]
  rna_fdr_col &lt;- colnames(rnaseq)[str_detect(str_to_lower(colnames(rnaseq)), &quot;fdr|adjp&quot;)][1]
  if (!is.na(rna_lfc_col) &amp; !is.na(rna_fdr_col)) {
    hfgc_genes &lt;- trans_models %&gt;% 
      mutate(
        status = case_when(
          !gene %in% rnaseq$gene_id ~ &quot;Undetected&quot;,
          gene %in% dplyr::filter(
            rnaseq, !!sym(rna_lfc_col) &gt; 0, !!sym(rna_fdr_col) &lt; fdr_alpha
          )$gene_id ~ &quot;Up&quot;,
          gene %in% dplyr::filter(
            rnaseq, !!sym(rna_lfc_col) &lt; 0, !!sym(rna_fdr_col) &lt; fdr_alpha
          )$gene_id ~ &quot;Down&quot;,
          gene %in% dplyr::filter(
            rnaseq, !!sym(rna_fdr_col) &gt;= fdr_alpha
          )$gene_id ~ &quot;Unchanged&quot;,
        )
      ) %&gt;% 
      splitAsList(.$status) %&gt;% 
      lapply(select, -status) %&gt;% 
      GRangesList()
    gene_col &lt;- colours$direction %&gt;% 
      setNames(str_to_title(names(.)))
  }
}

## External Coverage (Optional)
if (!is.null(config$external$coverage)) {
  ext_cov_path &lt;- config$external$coverage %&gt;% 
    lapply(unlist) %&gt;% 
    lapply(function(x) setNames(here::here(x), names(x)))
  bwfl &lt;- c(
    bwfl[comps],
    lapply(ext_cov_path, function(x) BigWigFileList(x) %&gt;% setNames(names(x)))
  )
  line_col &lt;- c(
    line_col[comps],
    ext_cov_path %&gt;% 
      lapply(
        function(x) {
          missing &lt;- setdiff(names(x), names(treat_colours))
          cmn &lt;- intersect(names(x), names(treat_colours))
          col &lt;- setNames(character(length(names(x))), names(x))
          if (length(cmn) &gt; 0) col[cmn] &lt;- treat_colours[cmn]
          if (length(missing) &gt; 0) 
            col[missing] &lt;- hcl.colors(
              max(5, length(missing)), &quot;Zissou 1&quot;)[seq_along(missing)]
          col
        }
      )
  )
  
  y_ranges &lt;- grl_to_plot %&gt;% 
    unlist() %&gt;% 
    granges() %&gt;% 
    resize(w = 10 * width(.), fix = &#39;center&#39;)
  
  y_lim &lt;- c(
    y_lim[comps],
    bwfl[names(config$external$coverage)] %&gt;% 
      lapply(
        function(x) {
          GRangesList(lapply(x, import.bw, which = y_ranges)) %&gt;% 
            unlist() %&gt;% 
            filter(score == max(score)) %&gt;% 
            mcols() %&gt;% 
            .[[&quot;score&quot;]] %&gt;% 
            c(0) %&gt;% 
            range()
        }
      )
  )
  
}</code></pre>
<pre class="r"><code>htmltools::tagList(
  mclapply(
    seq_along(grl_to_plot),
    function(x) {
      ## Export the image
      img_out &lt;- file.path(
        fig_path,
        names(grl_to_plot)[[x]] %&gt;% 
          str_remove_all(&quot;(:|Vs. |- )&quot;) %&gt;% 
          str_replace_all(&quot; &quot;, &quot;_&quot;) %&gt;% 
          paste0(&quot;_AveExpr.&quot;, fig_type)
      )
      fig_fun(
        filename = img_out, width = 10, height = 8
      )
      ct &lt;- FALSE
      if (length(subsetByOverlaps(gtf_transcript, grl_to_plot[[x]])) &gt; 10)
        ct &lt;- &quot;meta&quot;
      plotHFGC(
        grl_to_plot[[x]],
        features = feat_gr, featcol = feature_colours, 
        featsize = 1 + has_features,
        genes = hfgc_genes, genecol = gene_col,
        coverage = bwfl, linecol = line_col,
        annotation = annot, annotcol = annot_col,
        cytobands = bands_df,
        rotation.title = 90,
        zoom = 10,
        ylim = y_lim,
        collapseTranscripts = ct,
        col.title = &quot;black&quot;, background.title = &quot;white&quot;, showAxis = FALSE
      )
      dev.off()
      ## Create html tags
      fig_link &lt;- str_extract(img_out, &quot;assets.+&quot;)
      gn &lt;- unlist(subsetByOverlaps(all_windows, grl_to_plot[[x]])$detected)
      cp &lt;- htmltools::tags$em(
        glue(
          &quot;
        Highlighted region corresponds to the highest signal within the group 
        {names(grl_to_plot)[[x]]}. The most likely target 
        {ifelse(length(gn) == 1, &#39;gene is &#39;, &#39;genes are &#39;)}
        {collapseGenes(gn, format = &#39;&#39;)}
        &quot;
        )
      )
      htmltools::div(
        htmltools::div(
          id = names(grl_to_plot)[[x]] %&gt;% 
            str_remove_all(&quot;(:|Vs. |- )&quot;) %&gt;% 
            str_replace_all(&quot; &quot;, &quot;-&quot;) %&gt;% 
            str_to_lower() %&gt;% 
            paste0(&quot;-aveexpr&quot;),
          class=&quot;section level3&quot;,
          htmltools::h3(names(grl_to_plot)[[x]]),
          htmltools::div(
            class = &quot;figure&quot;, style = &quot;text-align: center&quot;,
            htmltools::img(src = fig_link, width = 960),
            htmltools::p(
              class = &quot;caption&quot;, htmltools::tags$em(cp)
            )
          )
        )
      )
    },
    mc.cores = threads
  )
)  </code></pre>
<div>
<div id="ar-up-er-up-aveexpr" class="section level3">
<h3>AR Up - ER Up</h3>
<div class="figure" style="text-align: center">
<img src="assets/AR_E2_E2DHT-ER_E2_E2DHT/AR_Up_ER_Up_AveExpr.png" width="960"/>
<p class="caption">
<em>
<em>Highlighted region corresponds to the highest signal within the group 
AR Up - ER Up. The most likely target 
gene is 
SGK3</em>
</em>
</p>
</div>
</div>
</div>
<div>
<div id="ar-up-er-down-aveexpr" class="section level3">
<h3>AR Up - ER Down</h3>
<div class="figure" style="text-align: center">
<img src="assets/AR_E2_E2DHT-ER_E2_E2DHT/AR_Up_ER_Down_AveExpr.png" width="960"/>
<p class="caption">
<em>
<em>Highlighted region corresponds to the highest signal within the group 
AR Up - ER Down. The most likely target 
gene is 
HNMT</em>
</em>
</p>
</div>
</div>
</div>
<div>
<div id="ar-up-er-unchanged-aveexpr" class="section level3">
<h3>AR Up - ER Unchanged</h3>
<div class="figure" style="text-align: center">
<img src="assets/AR_E2_E2DHT-ER_E2_E2DHT/AR_Up_ER_Unchanged_AveExpr.png" width="960"/>
<p class="caption">
<em>
<em>Highlighted region corresponds to the highest signal within the group 
AR Up - ER Unchanged. The most likely target 
genes are 
IGFBP4, RARA and TOP2A</em>
</em>
</p>
</div>
</div>
</div>
<div>
<div id="ar-down-er-unchanged-aveexpr" class="section level3">
<h3>AR Down - ER Unchanged</h3>
<div class="figure" style="text-align: center">
<img src="assets/AR_E2_E2DHT-ER_E2_E2DHT/AR_Down_ER_Unchanged_AveExpr.png" width="960"/>
<p class="caption">
<em>
<em>Highlighted region corresponds to the highest signal within the group 
AR Down - ER Unchanged. The most likely target 
gene is 
HSD17B7</em>
</em>
</p>
</div>
</div>
</div>
</div>
<div id="windows-with-the-largest-change" class="section level2 tabset">
<h2 class="tabset">Windows With The Largest Change</h2>
<p>For the initial set of visualisations, the sets of windows with the
strongest signal were selected from within each group. Groups were
restricted to those where differential binding was seen in at least
comparison.</p>
<pre class="r"><code>grl_to_plot &lt;- all_windows %&gt;% 
  filter(
    !str_detect(status, &quot;Un.+Un&quot;),
    vapply(detected, length, integer(1)) &gt; 0
  ) %&gt;% 
  as_tibble() %&gt;% 
  rename_all(
    str_replace_all, pattern = &quot;\\.Vs\\.\\.&quot;, replacement = &quot; Vs. &quot;
  ) %&gt;% 
  rename_all(
    str_replace_all, pattern = &quot;\\.\\.&quot;, replacement = &quot;: &quot;
  ) %&gt;% 
  nest(
    logFC = ends_with(&quot;logFC&quot;)
  ) %&gt;% 
  mutate(
    max_logFC = vapply(logFC, function(x) max(abs(unlist(x))), numeric(1))
  ) %&gt;% 
  group_by(status) %&gt;% 
  filter(max_logFC == max(max_logFC)) %&gt;% 
  dplyr::select(
    range, status, ends_with(&quot;logFC&quot;)
  ) %&gt;% 
  droplevels() %&gt;% 
  split(.$status) %&gt;% 
  lapply(pull, &quot;range&quot;) %&gt;% 
  lapply(GRanges, seqinfo = sq) %&gt;% 
  GRangesList()</code></pre>
<pre class="r"><code>## Coverage y-limits
gr &lt;- unlist(grl_to_plot)
y_lim &lt;- lapply(
  bwfl,
  function(x) {
    cov &lt;- lapply(x, import.bw, which = gr)
    unlist(lapply(cov, function(y) max(y$score))) %&gt;% 
      c(0) %&gt;% 
      range()
  }
)

## External Coverage (Optional)
if (!is.null(config$external$coverage)) {
  y_ranges &lt;- grl_to_plot %&gt;% 
    unlist() %&gt;% 
    granges() %&gt;% 
    resize(w = 10 * width(.), fix = &#39;center&#39;)
  
  y_lim &lt;- c(
    y_lim[comps],
    bwfl[names(config$external$coverage)] %&gt;% 
      lapply(
        function(x) {
          GRangesList(lapply(x, import.bw, which = y_ranges)) %&gt;% 
            unlist() %&gt;% 
            filter(score == max(score)) %&gt;% 
            mcols() %&gt;% 
            .[[&quot;score&quot;]] %&gt;% 
            c(0) %&gt;% 
            range()
        }
      )
  )
  
}</code></pre>
<pre class="r"><code>htmltools::tagList(
  mclapply(
    seq_along(grl_to_plot),
    function(x) {
      ## Export the image
      img_out &lt;- file.path(
        fig_path,
        names(grl_to_plot)[[x]] %&gt;% 
          str_remove_all(&quot;(:|Vs. |- )&quot;) %&gt;% 
          str_replace_all(&quot; &quot;, &quot;_&quot;) %&gt;% 
          paste0(&quot;_logFC.&quot;, fig_type)
      )
      fig_fun(
        filename = img_out, width = 10, height = 8
      )
      ct &lt;- FALSE
      if (length(subsetByOverlaps(gtf_transcript, grl_to_plot[[x]])) &gt; 10)
        ct &lt;- &quot;meta&quot;
      plotHFGC(
        grl_to_plot[[x]],
        features = feat_gr, featcol = feature_colours,
        featsize = 1 + has_features,
        genes = hfgc_genes, genecol = gene_col,
        coverage = bwfl, linecol = line_col,
        annotation = annot, annotcol = annot_col,
        cytobands = bands_df,
        rotation.title = 90,
        zoom = 10,
        ylim = y_lim,
        collapseTranscripts = ct,
        col.title = &quot;black&quot;, background.title = &quot;white&quot;, showAxis = FALSE
      )
      dev.off()
      ## Create html tags
      fig_link &lt;- str_extract(img_out, &quot;assets.+&quot;)
      gn &lt;- unlist(subsetByOverlaps(all_windows, grl_to_plot[[x]])$detected)
      cp &lt;- htmltools::tags$em(
        glue(
          &quot;
        Highlighted region corresponds to the largest change within the group 
        {names(grl_to_plot)[[x]]}. The most likely target 
        {ifelse(length(gn) == 1, &#39;gene is &#39;, &#39;genes are &#39;)}
        {collapseGenes(gn, format = &#39;&#39;)}
        &quot;
        )
      )
      htmltools::div(
        htmltools::div(
          id = names(grl_to_plot)[[x]] %&gt;% 
            str_remove_all(&quot;(:|Vs. |- )&quot;) %&gt;% 
            str_replace_all(&quot; &quot;, &quot;-&quot;) %&gt;% 
            str_to_lower() %&gt;% 
            paste0(&quot;-logfc&quot;),
          class=&quot;section level3&quot;,
          htmltools::h3(names(grl_to_plot)[[x]]),
          htmltools::div(
            class = &quot;figure&quot;, style = &quot;text-align: center&quot;,
            htmltools::img(src = fig_link, width = 960),
            htmltools::p(
              class = &quot;caption&quot;, htmltools::tags$em(cp)
            )
          )
        )
      )
    },
    mc.cores = threads
  )
)  </code></pre>
<div>
<div id="ar-up-er-up-logfc" class="section level3">
<h3>AR Up - ER Up</h3>
<div class="figure" style="text-align: center">
<img src="assets/AR_E2_E2DHT-ER_E2_E2DHT/AR_Up_ER_Up_logFC.png" width="960"/>
<p class="caption">
<em>
<em>Highlighted region corresponds to the largest change within the group 
AR Up - ER Up. The most likely target 
genes are 
ATG101, KRT80 and NR4A1</em>
</em>
</p>
</div>
</div>
</div>
<div>
<div id="ar-up-er-down-logfc" class="section level3">
<h3>AR Up - ER Down</h3>
<div class="figure" style="text-align: center">
<img src="assets/AR_E2_E2DHT-ER_E2_E2DHT/AR_Up_ER_Down_logFC.png" width="960"/>
<p class="caption">
<em>
<em>Highlighted region corresponds to the largest change within the group 
AR Up - ER Down. The most likely target 
gene is 
HNMT</em>
</em>
</p>
</div>
</div>
</div>
<div>
<div id="ar-up-er-unchanged-logfc" class="section level3">
<h3>AR Up - ER Unchanged</h3>
<div class="figure" style="text-align: center">
<img src="assets/AR_E2_E2DHT-ER_E2_E2DHT/AR_Up_ER_Unchanged_logFC.png" width="960"/>
<p class="caption">
<em>
<em>Highlighted region corresponds to the largest change within the group 
AR Up - ER Unchanged. The most likely target 
genes are 
MCM2, PODXL2 and TPRA1</em>
</em>
</p>
</div>
</div>
</div>
<div>
<div id="ar-down-er-unchanged-logfc" class="section level3">
<h3>AR Down - ER Unchanged</h3>
<div class="figure" style="text-align: center">
<img src="assets/AR_E2_E2DHT-ER_E2_E2DHT/AR_Down_ER_Unchanged_logFC.png" width="960"/>
<p class="caption">
<em>
<em>Highlighted region corresponds to the largest change within the group 
AR Down - ER Unchanged. The most likely target 
gene is 
HSD17B7</em>
</em>
</p>
</div>
</div>
</div>
</div>
</div>
<div id="enrichment-analysis" class="section level1">
<h1>Enrichment Analysis</h1>
<pre class="r"><code>min_gs_size &lt;- params$enrichment$min_size
if (is.null(min_gs_size) | is.na(min_gs_size)) min_gs_size &lt;- 0
max_gs_size &lt;- params$enrichment$max_size
if (is.null(max_gs_size) | is.na(max_gs_size)) max_gs_size &lt;- Inf
min_sig &lt;- params$enrichment$min_sig
if (is.null(min_sig) | is.na(min_sig)) min_sig &lt;- 1

all_ids &lt;- gtf_gene %&gt;%
  mutate(w = width) %&gt;% 
  as_tibble() %&gt;% 
  dplyr::select(gene_id, w) %&gt;% 
  arrange(desc(w)) %&gt;% 
  distinct(gene_id, w) %&gt;% 
  arrange(gene_id) %&gt;% 
  left_join(
    all_windows$gene_id %&gt;%
      unlist() %&gt;%
      table() %&gt;%
      enframe(name = &quot;gene_id&quot;, value = &quot;n_windows&quot;),
    by = &quot;gene_id&quot;
  ) %&gt;% 
  dplyr::filter(gene_id %in% gtf_gene$gene_id) %&gt;% 
  mutate(
    n_windows = as.integer(ifelse(is.na(n_windows), 0, n_windows))
  )
msigdb &lt;- msigdbr(species = &quot;Homo sapiens&quot;) %&gt;% 
  dplyr::filter(
    gs_cat %in% unlist(params$enrichment$msigdb$gs_cat) |
      gs_subcat %in% unlist(params$enrichment$msigdb$gs_subcat),
    str_detect(ensembl_gene, &quot;^E&quot;)
  ) %&gt;% 
  dplyr::rename(gene_id = ensembl_gene, gene_name = gene_symbol) %&gt;% # For easier integration 
  dplyr::select(-starts_with(&quot;human&quot;), -contains(&quot;entrez&quot;)) %&gt;% 
  dplyr::filter(gene_id %in% all_ids$gene_id) %&gt;% 
  group_by(gs_name) %&gt;% 
  mutate(n = dplyr::n()) %&gt;% 
  ungroup() %&gt;% 
  dplyr::filter(n &gt;= min_gs_size, n &lt;= max_gs_size) 
gs_by_gsid &lt;- msigdb %&gt;% 
  split(.$gs_name) %&gt;% 
  mclapply(pull, &quot;gene_id&quot;, mc.cores = threads) %&gt;% 
  mclapply(unique, mc.cores = threads)
gs_by_geneid &lt;- msigdb %&gt;% 
  split(.$gene_id) %&gt;% 
  mclapply(pull, &quot;gs_name&quot;, mc.cores = threads) %&gt;% 
  mclapply(unique, mc.cores = threads)
gs_url &lt;- msigdb %&gt;% 
  distinct(gs_name, gs_url) %&gt;%
  mutate(
    gs_url = ifelse(
      gs_url == &quot;&quot;, &quot;
      http://www.gsea-msigdb.org/gsea/msigdb/collections.jsp&quot;,
      gs_url
    )
  ) %&gt;% 
  with(setNames(gs_url, gs_name))

min_network_size &lt;- params$networks$min_size
if (is.null(min_network_size) | is.na(min_network_size))
  min_network_size &lt;- 2

max_network_size &lt;- params$networks$max_size
if (is.null(max_network_size) | is.na(max_network_size))
  max_network_size &lt;- Inf

max_network_dist &lt;- params$networks$max_distance
if (is.null(max_network_dist) | is.na(max_network_dist))
  max_network_dist &lt;- 1

net_layout &lt;- params$networks$layout
enrich_alpha &lt;- params$enrichment$alpha
adj_method &lt;- match.arg(params$enrichment$adj, p.adjust.methods)
adj_desc &lt;- case_when(
  p.adjust.methods %in% c(&quot;fdr&quot;, &quot;BH&quot;) ~ &quot;the Benjamini-Hochberg FDR&quot;,
  p.adjust.methods %in% c(&quot;BY&quot;) ~ &quot;the Benjamini-Yekutieli FDR&quot;,
  p.adjust.methods %in% c(&quot;bonferroni&quot;) ~ &quot;the Bonferroni&quot;,
  p.adjust.methods %in% c(&quot;holm&quot;) ~ &quot;Holm&#39;s&quot;,
  p.adjust.methods %in% c(&quot;hommel&quot;) ~ &quot;Hommel&#39;s&quot;,
  p.adjust.methods %in% c(&quot;hochberg&quot;) ~ &quot;Hochberg&#39;s&quot;,
  p.adjust.methods %in% c(&quot;none&quot;) ~ &quot;no&quot;
) %&gt;% 
  setNames(p.adjust.methods)
comp_ids &lt;- comps %&gt;% 
  lapply(
    function(x) {
      subset(all_windows, !str_detect(status, paste(x, &quot;Undetected&quot;)))$gene_id
    }
  ) %&gt;% 
  lapply(unlist) %&gt;% 
  lapply(unique) %&gt;% 
  lapply(intersect, gtf_gene$gene_id) %&gt;% 
  setNames(comps)
mapped_ids &lt;- comp_ids %&gt;% 
  unlist() %&gt;% 
  unique()
gene_lengths &lt;- structure(width(gtf_gene), names = gtf_gene$gene_id)[mapped_ids]
group_ids &lt;- all_windows %&gt;% 
  split(f = .$status) %&gt;% 
  mclapply(
    function(x) unique(unlist(x$gene_id)),
    mc.cores = threads
  ) </code></pre>
<div id="comparison-of-genes-mapped-to-targets" class="section level2">
<h2>Comparison of Genes Mapped To Targets</h2>
<p>The first level of enrichment testing performed was to simply check
the genes mapped to a binding site in either comparison, with no
consideration being paid to the dynamics of binding in either
comparison. This will capture and describe the overall biological
activity being jointly regulated.</p>
<p>The genes in each section and intersection in the following Venn
Diagram will then be analysed against the complete set of 7,631 genes
mapped to either AR or ER. Firstly, all 7,631 genes mapped to a binding
site were tested for enrichment in comparison to the set of 12,990
detected genes. For this analysis, gene width was used as a potential
source of sampling bias and Wallenius’ Non-Central Hypergeomteric was
used as implemented in <code>goseq</code> <span class="citation">(<a
href="#ref-YoungGoseq2010" role="doc-biblioref">Young et al.
2010</a>)</span>.</p>
<p>The sets of genes mapped to each section of the Venn Diagram were
then tested for enrichment in comparison to the larger set of 7,631
mapped genes. The standard hypergeometric distribution was used for
these analyses.</p>
<p>If 4 or more gene-sets were considered to be enriched, a network plot
was produced using the same methodology as for results from differential
binding for individual comparisons.</p>
<pre class="r"><code>venn_params &lt;- list(
  area1 = length(comp_ids[[1]]),
  area2 = length(comp_ids[[2]]),
  cross.area = sum(duplicated(unlist(comp_ids))),
  category = comps,
  fill = comp_cols[comps],
  cex = 1.2, cat.cex = 1.2
)
grid.newpage()
do.call(&quot;draw.pairwise.venn&quot;, venn_params)</code></pre>
<div class="figure" style="text-align: center">
<img src="AR_E2_E2DHT_ER_E2_E2DHT_pairwise_comparison_files/figure-html/venn-shared-genes-1.png" alt="*Venn Diagram showing __genes__ mapped to a binding region associated with AR or ER. Of the 12,990 genes under consideration, a total of 7,631 genes were mapped to a binding region across both comparisons, leaving 5,359 genes unmapped to a binding region across either comparison.*" width="960" />
<p class="caption">
<em>Venn Diagram showing <strong>genes</strong> mapped to a binding
region associated with AR or ER. Of the 12,990 genes under
consideration, a total of 7,631 genes were mapped to a binding region
across both comparisons, leaving 5,359 genes unmapped to a binding
region across either comparison.</em>
</p>
</div>
<div id="result-tables" class="section level3 tabset">
<h3 class="tabset">Result Tables</h3>
<div id="mapped-genes-to-either-ar-or-er" class="section level4">
<h4>Mapped Genes To Either AR or ER</h4>
<pre class="r"><code>pwf_mapped &lt;- gtf_gene %&gt;%
  mutate(w = width) %&gt;%
  as_tibble() %&gt;% 
  arrange(gene_id, desc(w)) %&gt;%
  distinct(gene_id, w) %&gt;% 
  mutate(mapped = gene_id %in% mapped_ids) %&gt;% 
  with(nullp(setNames(mapped, gene_id), bias.data = log10(w)))</code></pre>
<pre class="r"><code>goseq_mapped_res &lt;- pwf_mapped %&gt;% 
  goseq(gene2cat = gs_by_geneid) %&gt;% 
  dplyr::filter(numDEInCat &gt; 0) %&gt;% 
  as_tibble() %&gt;% 
  dplyr::select(
    gs_name = category, pval = over_represented_pvalue, starts_with(&quot;num&quot;)
  ) %&gt;% 
  mutate(
    adj_p = p.adjust(pval, adj_method),
    enriched = adj_p &lt; enrich_alpha
  ) %&gt;% 
  dplyr::filter(numDEInCat &gt;= min_sig) %&gt;% 
  left_join(
    dplyr::filter(msigdb, gene_id %in% mapped_ids)[c(&quot;gs_name&quot;, &quot;gene_name&quot;, &quot;gs_url&quot;, &quot;gs_description&quot;)],
    by = &quot;gs_name&quot;
  ) %&gt;% 
  chop(gene_name) %&gt;% 
  mutate(
    gene_name = vapply(gene_name, paste, character(1), collapse = &quot;; &quot;)
  )
any_goseq_mapped &lt;- sum(goseq_mapped_res$adj_p &lt; enrich_alpha) &gt; 0
tg_mapped &lt;- make_tbl_graph(
  goseq_mapped_res, 
  gs = gs_by_gsid %&gt;% 
    lapply(intersect, rownames(subset(pwf_mapped, DEgenes))) %&gt;% 
    .[vapply(., length, integer(1)) &gt; 0]
)
plot_network_mapped &lt;- length(tg_mapped) &gt;= min_network_size</code></pre>
<pre class="r"><code>tbl &lt;- goseq_mapped_res %&gt;%
  dplyr::filter(enriched) %&gt;% 
  dplyr::select(
    gs_name, gs_description, pval, adj_p, starts_with(&quot;num&quot;), gene_name
  ) %&gt;% 
  reactable(
    filterable = TRUE,
    showPageSizeOptions = TRUE,
    columns = list2(
      gs_name = colDef(
        name = &quot;Gene Set&quot;,
        cell = function(value) htmltools::tags$a(
          href = gs_url[[value]], 
          target = &quot;_blank&quot;, 
          str_replace_all(value, &quot;_&quot;, &quot; &quot;)
        ),
        html = TRUE,
        maxWidth = 150
      ),
      gs_description = colDef(
        name = &quot;Description&quot;,
        cell = function(value) str_trunc(value, width = 150)
      ),
      pval = colDef(
        name = &quot;P-Value&quot;,
        cell = function(value) ifelse(
          value &lt; 0.001,
          sprintf(&quot;%.2e&quot;, value),
          sprintf(&quot;%.3f&quot;, value)
        ),
        maxWidth = 80
      ),
      adj_p = colDef(
        name = &quot;p&lt;sub&gt;adj&lt;/sub&gt;&quot;, html = TRUE,
        cell = function(value) ifelse(
          value &lt; 0.001,
          sprintf(&quot;%.2e&quot;, value),
          sprintf(&quot;%.3f&quot;, value)
        ),
        maxWidth = 80
      ),
      numDEInCat = colDef(
        name = &quot;Mapped Genes&quot;,
        cell = function(value) comma(value, 1),
        maxWidth = 80
      ),
      numInCat = colDef(
        name = &quot;Genes in Gene-Set&quot;,
        cell = function(value) comma(value, 1),
        maxWidth = 80
      ),
      gene_name = colDef(
        name = &quot;Gene Names&quot;,
        cell = function(value) with_tooltip(value, width = 150)
      )
    )
  )
cp &lt;- glue(
  &quot;
  All {comma(length(mapped_ids))} genes with a mapped binding window from 
  either comparison were compared to the complete set of 
  {comma(length(unique(gtf_gene$gene_id)))} annotated genes, in order to 
  ascertain which key gene-sets and pathways were likely to be targeted in either analysis. 
  &quot;,
  ifelse(
    has_rnaseq,
    &quot;Genes were restricted to those in the set of detected genes from the provided RNA-Seq dataset. &quot;, 
    &quot;&quot;
  ),
  &quot;
  Gene-length was included as the source of any potential sampling bias for a 
  gene being mapped to any target-bound range. This analysis provides an overall 
  viewpoint as to which pathways are regulated across either dataset.
  &quot;
)
div(class = &quot;table&quot;,
    div(class = &quot;table-header&quot;, div(class = &quot;caption&quot;, cp)),
    tbl
)</code></pre>
<div class="table">
<div class="table-header">
<div class="caption">All 7,631 genes with a mapped binding window from 
either comparison were compared to the complete set of 
12,990 annotated genes, in order to 
ascertain which key gene-sets and pathways were likely to be targeted in either analysis. 
Genes were restricted to those in the set of detected genes from the provided RNA-Seq dataset. 
Gene-length was included as the source of any potential sampling bias for a 
gene being mapped to any target-bound range. This analysis provides an overall 
viewpoint as to which pathways are regulated across either dataset.</div>
</div>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-92e6638f33ed43d15717" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-92e6638f33ed43d15717">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"gs_name":["HALLMARK_ESTROGEN_RESPONSE_EARLY","GREB1_TARGET_GENES","ZNF766_TARGET_GENES","HALLMARK_ESTROGEN_RESPONSE_LATE","ZNF197_TARGET_GENES","REACTOME_FATTY_ACID_METABOLISM","HALLMARK_HYPOXIA","HES2_TARGET_GENES","ZNF175_TARGET_GENES","WP_CCL18_SIGNALING_PATHWAY","BPTF_TARGET_GENES","WP_NUCLEAR_RECEPTORS_METAPATHWAY","E2F5_TARGET_GENES","DLX2_TARGET_GENES","HALLMARK_UV_RESPONSE_DN","FOXJ2_TARGET_GENES","WP_PROSTAGLANDIN_SYNTHESIS_AND_REGULATION"],"gs_description":["Genes defining early response to estrogen.","Genes containing one or more binding sites for UniProt:Q4ZG55 (GREB1) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Genes containing one or more binding sites for UniProt:Q5HY98 (ZNF766) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Genes defining late response to estrogen.","Genes containing one or more binding sites for UniProt:O14709 (ZNF197) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Fatty acid metabolism","Genes up-regulated in response to low oxygen levels (hypoxia).","Genes containing one or more binding sites for UniProt:Q9Y543 (HES2) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Genes containing one or more binding sites for UniProt:Q9Y473 (ZNF175) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","CCL18 signaling pathway","Genes containing one or more binding sites for UniProt:Q12830 (BPTF) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Nuclear receptors meta-pathway","Genes containing one or more binding sites for UniProt:Q15329 (E2F5) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Genes containing one or more binding sites for UniProt:Q07687 (DLX2) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Genes down-regulated in response to ultraviolet (UV) radiation.","Genes containing one or more binding sites for UniProt:Q9P0K8 (FOXJ2) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Prostaglandin synthesis and regulation"],"pval":[4.45965685781061e-19,3.96137620720512e-15,2.9570233006084e-11,8.17093083391232e-10,9.29137163109975e-10,2.08412467672867e-09,1.71574947729703e-08,4.8183132330111e-07,1.8686092373949e-05,2.59891254747354e-05,3.91140779642279e-05,4.42570947770545e-05,4.65096759747705e-05,0.000235661824828712,0.000238204684800955,0.000243588126137703,0.000262400837169274],"adj_p":[1.18225503300559e-15,5.25080416265038e-12,2.61302292330429e-08,4.92628523880909e-07,4.92628523880909e-07,9.20835753001286e-07,6.49778837759203e-06,0.000159666854758905,0.00550409232037099,0.00688971716335234,0.00942649278937891,0.00948439623147052,0.00948439623147052,0.0403595076494407,0.0403595076494407,0.0403595076494407,0.0409190952550438],"numDEInCat":[163,497,620,136,346,99,119,626,406,26,375,136,639,147,100,291,21],"numInCat":[180,663,881,167,471,116,147,904,593,27,538,184,971,194,122,426,22],"gene_name":["ABAT; ABCA3; ABHD2; ABLIM1; ADCY1; ADCY9; ADD3; AFF1; AKAP1; AMFR; ANXA9; AQP3; AR; AREG; ARL3; ASB13; B4GALT1; BAG1; BCL11B; BCL2; BHLHE40; BLVRB; CA12; CBFA2T3; CCN5; CD44; CELSR1; CELSR2; CHPT1; DEPTOR; DHCR7; DHRS2; DHRS3; DLC1; DYNLT3; ELF3; ELOVL2; ELOVL5; ENDOD1; ESRP2; FAM102A; FARP1; FASN; FDFT1; FHL2; FKBP5; FLNB; FOS; FOXC1; FRK; GAB2; GFRA1; GLA; GREB1; HES1; HR; HSPB8; IGF1R; IGFBP4; IL17RB; IL6ST; INHBB; INPP5F; ISG20L2; ITPK1; JAK2; KAZN; KCNK15; KCNK5; KDM4B; KLF10; KLF4; KRT15; KRT18; KRT19; KRT8; LAD1; LRIG1; MAPT; MAST4; MED13L; MICB; MLPH; MREG; MUC1; MYB; MYBL1; MYC; MYOF; NADSYN1; NBL1; NCOR2; NRIP1; OPN3; OVOL2; P2RY2; PAPSS2; PDZK1; PEX11A; PGR; PLAAT3; PMAIP1; PODXL; PPIF; PTGES; RAB17; RAB31; RAPGEFL1; RARA; RASGRP1; RBBP8; REEP1; RETREG1; RHOBTB3; RHOD; RPS6KA2; RRP12; SCARB1; SCNN1A; SEC14L2; SFN; SH3BP5; SIAH2; SLC16A1; SLC19A2; SLC1A1; SLC1A4; SLC22A5; SLC24A3; SLC26A2; SLC27A2; SLC2A1; SLC39A6; SLC7A2; SLC7A5; SLC9A3R1; SNX24; SULT2B1; SVIL; SYBU; SYNGR1; SYT12; TBC1D30; TFAP2C; TFF1; TFF3; TGIF2; THSD4; TIAM1; TIPARP; TJP3; TMEM164; TOB1; TPBG; TPD52L1; TSKU; TTC39A; TUBB2B; UGCG; UNC119; WFS1; WWC1; XBP1","ABCA3; ABCC11; ABHD17C; ABHD2; ADAP1; ADGRG1; ADIRF-AS1; AFAP1; AFF1; AGAP2; AHCYL2; AKAP1; AKAP1-DT; AKAP8L; AKAP9; ALKBH7; AMOTL2; AMZ1; ANAPC2; ANKFY1; ANKRD11; ANKRD22; ANP32E; ANXA9; AP5B1; ARHGEF1; ARID1B; ARNTL; ARRDC1; ARRDC4; ASCC2; ASCL2; ATP2A2; ATP5MC1; ATP6V0A4; ATP8B1; ATXN7L1; B3GALNT1; BAIAP3; BATF; BBOF1; BCAR1; BCAR3; BCAS3; BCL3; BCL9L; BICDL1; BLVRB; C1QTNF6; C1orf21; C5AR2; C9orf152; CA12; CAB39L; CAMTA1; CARD10; CASP7; CCDC159; CCDC40; CCDC88C; CCN5; CCT7; CD276; CD59; CDC34; CDS1; CELSR1; CEP89; CFAP45; CHD3; CHD6; CHD8; CHPT1; CHST12; CIC; CITED2; CLN8; CMTR2; CNOT3; COMT; COMTD1; COQ8A; CORO2A; CRACR2A; CREBRF; CRKL; CRNDE; CRPPA; CSNK1G2; CTSB; CYB561; CYLD; CYP1B1; CYP1B1-AS1; DAAM1; DDA1; DDX47; DENND2D; DGAT2; DLC1; DLG5; DMXL1; DNAH11; DNASE1; DNMBP; DNPEP; DOCK4; DOK7; DOP1B; DRAIC; DSCAM-AS1; DYNLT3; ECH1; EHD1; EHF; EHMT1; ELAPOR1; EML6; ENC1; ENSA; ENSG00000224934; ENSG00000267288; ENSG00000268858; ENSG00000275765; EPB41L1; EPG5; EPHA1; EPHB4; EPN2; EPN3; EPS8; ERBB2; ERCC1; ERGIC1; ESRP2; ETNK2; EVI5L; EVL; EXOSC10; FAAP24; FAF1; FAM107B; FAM117A; FAM126A; FAM174B; FAM234B; FAM86EP; FBRS; FBXO38; FHDC1; FMN1; FNIP2; FOXN3; FREM2; FRMD4B; FRMD6; FRYL; G3BP1; GALNT10; GATAD2A; GATAD2B; GCLM; GCOM1; GIGYF2; GLRX5; GOLPH3L; GOT1; GPNMB; GPR160; GPRC5A; GRAMD1A; GREB1; GRHL1; GRHL2; GSPT1; GTPBP3; HEBP2; HERC1; HK1; HMGCS1; HNRNPC; HNRNPL; HOMEZ; HSH2D; HSPB8; ID1; IDE; IFITM10; IFRD1; IFT27; IGF1R; IGSF3; IGSF9; IKBKG; IKZF2; INPP4B; IQCD; IQSEC1; ISG20; JUND; JUP; KAT7; KATNB1; KCNK1; KCTD11; KDM4B; KDM5A; KHSRP; KIAA0100; KIAA0895; KIAA1217; KIF21A; KLF6; KLHL25; KRT7; KRT8; KRT80; LAD1; LAMB1; LCOR; LDLRAD4; LENG1; LFNG; LIMA1; LINC00511; LINC00963; LINC01588; LLGL2; LMCD1; LNC-LBCS; LPP; LRIG2; LRIG3; LRRFIP1; MACC1; MAF1; MANEAL; MAP3K3; MAP4K3-DT; MAPK10; MAPKAP1; MCCC2; MCF2L; MCRIP2; MEF2A; MGRN1; MICOS10; MIDEAS; MIDN; MIOS; MIR9-3HG; MIR99AHG; MIX23; MLLT3; MPHOSPH10; MSMO1; MTA1; MTMR12; MTMR4; MTUS1; MUC5B; NAALADL2; NADK; NAGK; NAIF1; NCAM2; NCOA1; NCOA4; NDRG1; NDUFAF8; NFATC2; NFKBIA; NFKBIZ; NFYB; NINL; NIPSNAP1; NIPSNAP2; NLK; NMT1; NOP14-AS1; NR1D1; NRP1; NT5DC3; NUDT16; NUMA1; OSBPL10; OSBPL2; OSBPL8; OVOL2; OXR1; P4HB; PABPN1; PAIP2B; PAK4; PAM; PARD3; PBX1; PCBP1-AS1; PCDH1; PCTP; PDGFB; PDGFD; PDZK1; PGP; PHTF2; PIK3C2G; PKIB; PKP4-AS1; PLA2G6; PLEKHF2; PNKD; PNPLA7; POLR3K; POU2F3; PPARD; PPM1H; PPM1L; PRADC1; PREX1; PRKCD; PRKCH; PROSER1; PROSER2; PROSER3; PRR15L; PRSS22; PRSS27; PSCA; PSMG3; PTPRH; PUM2; PWWP2A; PXDN; PXMP4; PYM1; PYROXD2; RAB26; RAB5B; RABL3; RAD51B; RANBP2; RAPGEFL1; RARA; RBBP8; RBBP8NL; RBL2; RBM20; RBM47; RC3H1; RECQL5; RFFL; RFX1; RGMB; RGMB-AS1; RGS3; RHOBTB3; RHOC; RNF19A; RNF207; RNF224; RPL27A; RPS6KA2; RPS6KL1; RTN4; S100A10; S100A16; S100A4; SAMD8; SCAMP4; SCGB1D2; SCIN; SCNN1A; SCRIB; SEC14L2; SEPTIN9; SERPINA1; SGMS2; SHARPIN; SIDT2; SKA2; SKIL; SLC16A3; SLC1A5; SLC25A25; SLC25A36; SLC27A2; SLC29A1; SLC35E1; SLC38A1; SLC38A2; SLC39A1; SLC44A2; SLC45A4; SLC6A6; SLC7A2; SLC9A8; SMAD3; SMARCD2; SNX9; SPECC1L; SPNS2; SPRED2; SPRYD3; SPTBN4; SPTLC2; SPTSSB; SRI; SRPK1; SRRM2-AS1; SSNA1; ST6GAL1; ST6GALNAC4; STARD10; STAT3; SUCLA2; SULT1A1; SYBU; SYT8; TACC2; TBC1D16; TBC1D8; TBCD; TBX2; TEDC1; TEPSIN; TFAP2C; TFF1; TGFBI; THADA; THSD4; TIPARP; TJP1; TLE1; TMBIM1; TMC1; TMCO3; TMED3; TMEM120A; TMEM160; TMEM184B; TMEM229B; TMEM241; TMEM60; TMEM94; TMPRSS4; TMTC2; TNK2; TNS1; TOM1L2; TOP2A; TPD52L1; TPD52L2; TPRN; TRAF7; TRAPPC9; TRIB1; TRIM29; TSPAN9; TTC39A-AS1; TTLL5; TUBA1A; TUBA1C; TUBB4B; UBE2C; UNC13D; UNKL; UPF1; URB1; USF3; UTRN; VAMP5; VAV3; VEPH1; VIRMA; VMP1; VPS26C; WDR90; WIPI1; WWC1; WWP1; XRCC3; YIF1A; ZDHHC1; ZFYVE21; ZFYVE28; ZKSCAN1; ZMYM2; ZMYND8; ZNF444; ZNF552; ZNF669; ZNF687; ZNF689; ZNF750; ZNF823; ZNRF1; ZSWIM1","AAMP; ACP2; ACTB; ACTG1; ACTR1A; ACTR3; ACTR3B; ADAM9; ADAMTSL4; AHCTF1; AHCYL2; AHSA2P; AKAP8; AKIRIN2; ALDH5A1; ALMS1; AMMECR1; AMN1; AMOTL1; ANKMY1; ANKS6; ANTXR2; ANXA2; AP1S2; AP2M1; AP2S1; APOO; AREG; ARFRP1; ARHGAP5; ARHGAP5-AS1; ARHGEF12; ARID4B; ARL4A; ARNTL; ARPC1A; ARV1; ASAP3; ASNSD1; ASXL2; ATG10; ATG4B; ATP5PF; ATXN7; AUP1; B4GALT2; B4GAT1; BABAM1; BANP; BATF; BCKDK; BICD2; BIRC5; BMERB1; BOD1; BORCS7; BTN3A2; BTRC; BUB3; C11orf58; C15orf61; C17orf58; C19orf44; C1orf159; CABIN1; CACNA1D; CALCOCO1; CAMP; CAP1; CAPG; CAST; CBFA2T3; CBX7; CCAR1; CCDC12; CCDC137; CCDC71; CCDC90B; CCP110; CDC73; CDCA3; CDH23; CDK5; CDK8; CDYL; CENPT; CEP290; CFL1; CHCHD2; CHD7; CHFR; CHIC2; CHMP4B; CIC; CITED2; CLEC16A; CLPB; CLSTN1; CLTC; CLUAP1; CNPY2; COMMD3; COPS7B; CORO1A; COX7A2L; COX7C; CPE; CRADD; CRAMP1; CRTC2; CRYM; CSNK2B; CSRNP2; CSRP1; CTDP1; CTSB; CYB561D2; CYB5D1; DCK; DDX23; DEDD; DENND1A; DFFA; DGAT1; DGAT2; DGCR8; DHRS3; DIP2A; DMD; DNAJA3; DNAJC17; DNALI1; DNM1L; DST; DUS1L; DUS3L; DUSP4; DYNC1I2; DYNC2I2; ECHDC2; EFNA3; EHD1; EHMT2; ELOC; EML2; EML6; ENSG00000233834; ENSG00000255468; ENY2; EPB41L5; EPC1; ERH; ERI3; ERMAP; ERP44; ERVK3-1; EVL; EXOC6; EYA3; FAF2; FAM117A; FAM13A; FAM168B; FAM193A; FAM222A; FASN; FASTKD1; FBXL16; FBXL17; FBXL19; FBXL3; FBXO15; FCSK; FGF13; FHL2; FLYWCH1; FN3K; FOXP1; FTO; FYCO1; G3BP2; GAB2; GALNT7; GAPVD1; GATAD2A; GATB; GDAP1; GDPD5; GFER; GFUS; GGPS1; GLUD1; GNG4; GPANK1; GPAT4; GPBP1; GPBP1L1; GPKOW; GPRC5C; GPT2; GRAMD4; GRK3; GRK4; GSPT1; GTF2I; GTF2IRD1; GTPBP2; GUSB; H4C8; H6PD; HAUS6; HECTD2; HERC1; HGD; HIKESHI; HM13; HMG20A; HSD11B1L; HSPA13; HSPB11; HSPH1; HTRA2; IFRD1; IFRD2; IFT140; IFT20; IFTAP; IGF1R; IKZF2; IL1R1; ILF3; ILF3-DT; IMPACT; ING3; INTS6; INVS; IQCH-AS1; IRF2BP2; KANSL3; KAT7; KBTBD6; KBTBD7; KCNB1; KCTD20; KCTD21; KDM7A; KDM7A-DT; KHDC4; KIF15; KLC1; KLHDC8B; KLHL21; KMT2B; LAMTOR2; LARP1; LARS1; LCORL; LEMD2; LEPROTL1; LIFR; LIG1; LINC-PINT; LINC00504; LINC01521; LLGL2; LMNB2; LNX1; LPXN; LSM10; LTA4H; LUC7L; LYPLA1; LYSMD1; MACC1; MAD2L1BP; MAF1; MALSU1; MAML3; MAN2A2; MANBAL; MAP4K3; MAP4K3-DT; MAT2A; MAX; MDH1; MED15; MFAP3L; MGME1; MIAT; MICOS13; MIDEAS; MIDN; MITF; MKKS; MLLT10; MMACHC; MPI; MPZL3; MR1; MRFAP1; MRGBP; MRPL24; MTIF2; MTSS1; MUS81; MUTYH; MYB; MYCL; MYO5B; MYO9A; NAA38; NAPA; NDUFAB1; NEAT1; NEDD4; NEK6; NF2; NFASC; NFE2L1; NFIX; NFKBIA; NFKBIB; NKAIN1; NOP14; NOP53; NORAD; NOSIP; NPEPPS; NPEPPSP1; NPL; NPRL2; NRDE2; NSMCE1-DT; NTAN1; NUCB1; NUDCD1; NUDT13; NUMA1; NUP205; NUP43; NUTF2; OPLAH; ORMDL3; OTUD7A; PA2G4; PABPN1; PACS1; PACS2; PACSIN2; PAFAH2; PANK4; PAPSS1; PATZ1; PCBP1-AS1; PCGF5; PCSK6; PDCD10; PDE3B; PDE6D; PDS5B; PEAK1; PEMT; PEX14; PEX26; PFDN6; PGBD5; PGM2L1; PGP; PHB; PHF12; PHF13; PISD; PLA2G6; PLCG2; PLEKHF2; PNKD; PNPLA6; PODXL2; POLDIP3; POLR1G; PPP1R12C; PPP1R13L; PPP1R15A; PPRC1; PRKCA; PRMT3; PROSER2; PRPF31; PRPSAP1; PRR5; PRRT2; PTCD3; PTK2; QRICH1; RAB1A; RAB27B; RAB7A; RABGAP1L; RALBP1; RANBP10; RANGAP1; RBBP4; RBM14; RBM25; RCAN1; RCBTB2; RERE; REXO2; RHOBTB3; RIC8B; RIMKLB; RIN3; RMC1; RNF145; RNFT2; RPGRIP1L; RPIA; RPL36; RPP40; RPRD1B; RPS13; RXRA; RYK; S100P; SAP30; SBK1; SCAF1; SCAMP4; SCNM1; SCRIB; SEC11C; SEC24C; SEC61A1; SENP8; SERPINI1; SETD5; SETMAR; SFMBT1; SFSWAP; SFT2D2; SFXN1; SHARPIN; SHLD2; SIPA1L3; SIRT2; SKIL; SLC16A1; SLC25A12; SLC25A6; SLC2A4RG; SLC35B4; SLC38A2; SLC39A11; SLC39A9; SLC44A1; SLC45A4; SLC48A1; SLC4A2; SLC9A3-AS1; SLK; SLMAP; SLX4IP; SMAD3; SMAP1; SMIM13; SMPD3; SNX10; SNX19; SNX5; SPEN; SPRING1; SQOR; SRCIN1; SRGAP3; ST3GAL2; STAG2; STAT5B; STAU1; STAU2; STK25; STK40; STMN3; STMP1; STPG1; STRIP1; STRN3; STXBP5; SULT1A2; SUMO3; SYMPK; SYNGR1; TANC1; TARBP2; TBC1D4; TBC1D8B; TBX6; TCEA2; TCERG1; TENT2; TFAP2A; TFEB; TFPT; THADA; THRAP3; THUMPD2; TIAL1; TIMM21; TJP3; TM7SF3; TM9SF4; TMBIM1; TMCC1; TMCC1-DT; TMCO1; TMCO1-AS1; TMEM11; TMEM115; TMEM160; TMEM181; TMEM245; TMEM69; TMEM70; TMTC1; TMTC3; TNFAIP1; TNS1; TOE1; TP53BP2; TPD52L1; TPGS1; TRAM2-AS1; TRIM29; TRIT1; TRMT12; TRMT44; TRPM7; TSC22D1; TSC22D2; TSSC4; TTC1; TTC13; TTC33; TTLL4; TUFT1; TXNDC15; UBE2E2; UBE2E3; UBE2Q2; UBE3C; UBQLN4; UBR2; UBR3; UBXN2A; UCP2; UNC45A; UPF1; UROS; USF1; USF2; USP31; USP32; USP35; USP48; USP6NL; VAPA; VDAC2; VMP1; VPS35L; VPS50; VRK1; WARS1; WARS2-AS1; WBP1; WBP2; WDPCP; WDR1; WDR24; WDR25; WDR46; WNK1; WSB1; WWP1; XIST; XKR9; XYLT2; YAP1; YEATS2; YIF1B; YIPF6; ZC3H7A; ZCCHC8; ZDHHC18; ZFP91; ZFYVE1; ZFYVE19; ZFYVE28; ZGPAT; ZMYM4; ZMYND8; ZNF181; ZNF207; ZNF217; ZNF221; ZNF223; ZNF225; ZNF32; ZNF331; ZNF337; ZNF37A; ZNF37BP; ZNF436; ZNF444; ZNF529; ZNF586; ZNF605; ZNF644; ZNF674-AS1; ZNF688; ZNF776; ZSCAN22","ABCA3; ABHD2; ACOX2; ADD3; AFF1; AGR2; AMFR; ANXA9; AREG; ARL3; BAG1; BATF; BCL2; BLVRB; CA12; CA2; CACNA2D2; CCN5; CD44; CD9; CDC6; CDH1; CELSR2; CHPT1; CHST8; COX6C; CPE; DCXR; DHCR7; DHRS2; DLG5; DNAJC1; DYNLT3; ELOVL5; EMP2; ETFB; FABP5; FAM102A; FARP1; FDFT1; FGFR3; FKBP5; FLNB; FOS; FOXC1; FRK; GALE; GFUS; GINS2; GLA; HMGCS2; HOMER2; HPRT1; HR; HSPB8; IDH2; IGFBP4; IL17RB; IL6ST; IMPA2; ISG20; ITPK1; JAK1; JAK2; KCNK5; KIF20A; KLF4; KRT19; LAMC2; LARGE1; LLGL2; LSR; MAPK13; MAPT; MICB; MOCS2; MYB; MYOF; NBL1; NCOR2; NRIP1; OPN3; OVOL2; PAPSS2; PDCD4; PDZK1; PGR; PLAAT3; PPIF; PRKAR2B; PRLR; PTGES; RAB31; RABEP1; RAPGEFL1; RBBP8; RPS6KA2; S100A9; SCARB1; SCNN1A; SCUBE2; SERPINA1; SFN; SIAH2; SLC16A1; SLC1A4; SLC22A5; SLC24A3; SLC26A2; SLC27A2; SLC29A1; SLC7A5; SLC9A3R1; SNX10; SORD; ST14; ST6GALNAC2; SULT2B1; TFAP2C; TFF1; TFF3; TFPI2; TIAM1; TJP3; TOB1; TOP2A; TPBG; TPD52L1; TRIM29; TSPAN13; UGDH; UNC13B; WFS1; XBP1; XRCC3; ZFP36","AAMP; ABCA5; ABLIM3; ACACA; ACAD10; ACAD8; ACP2; ACSL3; ACTG1; ADAM9; ADCY9; ADNP; AHCYL2; AHI1; AHSA1; AKIRIN2; AMMECR1; ANKS6; ANKZF1; AP2S1; ARAP1; ARHGAP5; ARHGAP5-AS1; ARNTL; ASAP3; ATG4B; ATG9A; AUP1; B4GALT2; BABAM1; BANP; BICDL1; BLOC1S4; BOD1; BSG; BTBD9; BTRC; BUB1B; BZW2; C17orf58; C1orf159; C6orf89; CABIN1; CALCOCO1; CAMKK2; CAMKMT; CAPG; CBFA2T3; CBX7; CCAR1; CCDC137; CCDC18-AS1; CCDC90B; CCP110; CDC40; CDCA3; CDH23; CDK5; CDKN2AIPNL; CEP290; CFL1; CHD7; CHFR; CHMP4B; CLEC16A; CLSTN1; CLTC; CNPY2; COMMD3; CORO1A; COX7A2L; COX7C; CPEB4; CPQ; CRADD; CRCP; CREBL2; CRTC2; CRYBG2; CYB561D2; CYB5R3; DCK; DDX23; DEDD; DENND1A; DENND5B; DFFA; DGCR8; DNAAF5; DR1; DYNC1I2; ECH1; EFNA3; ELOC; ENSG00000233834; ENY2; EPC1; ERMAP; ERVK3-1; EXOC6; FAM117A; FANCC; FASN; FASTKD1; FBXL16; FBXL19; FBXL3; FBXO15; FBXO38; FCHSD2; FCSK; FLYWCH1; FYCO1; GAB2; GFER; GFM1; GFUS; GNG4; GP6-AS1; GPAT4; GPKOW; GRHL1; GRK3; GSPT1; GTF2I; H3-3B; HDAC8; HERC1; HIF1A; HIKESHI; HM13; HTRA2; IFRD2; IFTAP; IKZF2; IL1R1; IMPACT; ING3; IRF2BP2; KAT7; KBTBD6; KBTBD7; KCNC3; KCNK1; KDM4C; KHK; LARP1; LARS1; LEMD2; LEPROTL1; LIG1; LINC01521; LSM10; LYPLA1; LYSMD1; MACC1; MAN2A2; MANEAL; MAPT; MARS1; MAX; MDH1; MITF; MKKS; MLLT10; MMACHC; MRFAP1; MRPL24; MRPS22; MUS81; MYB; MYCL; MYO1D; NDRG3; NEK6; NFASC; NFIX; NOP53; NOSIP; NPEPPS; NPEPPSP1; NPRL2; NRDE2; NSD2; NUDCD1; NUDT13; OPLAH; OTUD7A; PA2G4; PACS2; PAFAH2; PALS2; PANK4; PATZ1; PCBP1-AS1; PCSK6; PGBD5; PGP; PHB; PHF12; PHF13; PHLPP1; PISD; PLEKHA1; PLEKHF2; PNKD; PODXL2; PPM1E; PPOX; PPP1R12C; PPP1R15A; PPP2R2C; PREPL; PRKCA; PRMT3; PROSER2; PRPF31; PRPSAP1; PRSS27; PTDSS2; PTK2; PTPRN2; PUM3; RAB7A; RABAC1; RABGAP1L; RANBP10; RANGAP1; RBL1; RBM6; RHOBTB3; RIN3; RNASEH2C; RNF144A; RPP40; RXRA; RYBP; SAP30; SBK1; SCAF1; SCAMP4; SCAMP5; SCNM1; SEC24C; SEC61A1; SEL1L; SEM1; SEPHS2; SETMAR; SFI1; SFXN1; SGPL1; SIAH1; SIPA1L3; SKIL; SLC4A2; SLC9A3-AS1; SLF2; SLK; SLMAP; SLX4IP; SMARCAL1; SMPD3; SNAPC3; SPTBN4; SRCIN1; SRRM3; ST3GAL2; ST7; STARD10; STAT3; STAT5B; STAU1; STAU2; STK25; STRIP1; SUMO3; SYNGR1; TADA2A; TBX6; TCEA2; TCERG1; TCP11L1; TFPT; THADA; THUMPD2; THYN1; TIMM21; TM9SF4; TMCC1; TMCC1-DT; TMED10; TMEM11; TMEM115; TMEM160; TMEM181; TMEM70; TMTC3; TPGS1; TRABD; TRIM33; TRMT12; TSC22D1; TXNDC11; TXNDC15; UBE2E2; UBE2Q2; UBE3C; UBXN2A; UCKL1; UNK; UROS; USE1; USF2; USP31; USP6NL; VCPIP1; VIPAS39; WASF1; WDPCP; WDR82; WNK1; WSB1; WWP1; XIST; XKR9; XYLT2; YEATS2; ZBTB38; ZBTB46; ZC3H7A; ZFYVE1; ZFYVE28; ZMAT1; ZNF221; ZNF225; ZNF236; ZNF280D; ZNF32; ZNF331; ZNF37BP; ZNF417; ZNF444; ZNF605; ZNF688; ZNF776","ABCC1; ACAA1; ACACA; ACACB; ACAD10; ACADM; ACBD4; ACBD6; ACBD7; ACLY; ACOT1; ACOT13; ACOT2; ACOT4; ACOT7; ACOT8; ACOX1; ACOX2; ACOX3; ACSF2; ACSL1; ACSL3; ACSL4; AKR1C3; ALOX5; AMACR; CBR4; CPT1A; CPT2; CRAT; CROT; CYP1B1; CYP2J2; CYP2U1; CYP4B1; DBI; DECR1; DECR2; ECI1; ECI2; EHHADH; ELOVL2; ELOVL4; ELOVL5; ELOVL6; ELOVL7; FAAH; FAAH2; FADS2; FASN; GPX4; HACD1; HACD2; HACD3; HACD4; HACL1; HADH; HADHA; HADHB; HPGD; HSD17B12; HSD17B4; LTA4H; MCEE; MECR; MID1IP1; MLYCD; MMAA; MMUT; MORC2; NDUFAB1; NUDT19; PCCA; PCCB; PCTP; PECR; PLA2G4A; PON2; PPARD; PPT1; PRKAA2; PRKAB2; PRKAG2; PRXL2B; PTGES; PTGES2; PTGR1; PTGR2; RXRA; SCD; SCD5; SCP2; SLC22A5; SLC25A1; SLC25A17; SLC25A20; SLC27A2; SLC27A3; THEM4","ACKR3; AK4; ALDOA; AMPD3; ANGPTL4; ANKZF1; ANXA2; ATP7A; B4GALNT2; BCL2; BHLHE40; BNIP3L; BTG1; CA12; CCN1; CCN5; CCNG2; CDKN1B; CHST3; CITED2; COL5A1; CXCR4; DDIT4; DTNA; DUSP1; EDN2; EFNA1; EFNA3; EGFR; ENO1; ERO1A; ERRFI1; EXT1; F3; FAM162A; FBP1; FOS; FOSL2; FOXO3; GAA; GALK1; GAPDH; GBE1; GCNT2; GLRX; GPI; GRHPR; GYS1; HK1; HK2; HMOX1; HSPA5; IDS; IER3; IGFBP3; ILVBL; IRS2; ISG20; JMJD6; JUN; KDM3A; KLF6; KLF7; KLHL24; LARGE1; LDHA; MAFF; MAP3K1; MT2A; MXI1; NAGK; NDRG1; NDST1; NEDD4L; NOCT; P4HA1; P4HA2; PAM; PDGFB; PDK1; PFKFB3; PFKP; PGK1; PGM2; PIM1; PLAUR; PNRC1; PPP1R15A; PRKCA; RBPJ; RORA; RRAGD; S100A4; SAP30; SCARB1; SDC2; SDC3; SDC4; SELENBP1; SIAH2; SLC25A1; SLC2A1; SLC6A6; STC1; SULT2B1; TES; TGFBI; TIPARP; TPBG; TPD52; TPI1; TPST2; UGP2; VEGFA; VLDLR; WSB1; XPNPEP1; ZFP36; ZNF292","ABCA5; ABCC3; ABCG1; ABHD11; ABHD12; ABLIM1; ABLIM2; ACOT13; ACOT7; ACOX2; ACTN1; ADAM9; ADAP1; ADAR; ADD3; ADIPOR2; AFDN; AGK; AGPS; AGTPBP1; AHCYL1; AJUBA; AKAP6; AKR1C3; ALAS1; ALG10; ALG13; ALPK1; AMBRA1; ANAPC5; ANK3; ANKEF1; ANKRD11; ANKRD40; ANP32B; ANXA1; ANXA11; ANXA2; AOPEP; AP1G1; APBA3; APOLD1; AQP3; AREG; ARFIP1; ARHGAP21; ARHGAP24; ARHGAP26; ARID5B; ASCC2; ASNSD1; ATAD1; ATF6; ATG10; ATP2C1; ATP5MC2; ATP9B; ATXN10; ATXN7L1; AVL9; B3GNTL1; B4GAT1; BAZ2B; BCL10; BCL6; BCOR; BCR; BDP1; BHLHE40; BMERB1; BMF; BMP1; BMPR1B; BOLA1; BPHL; BRD4; BUD31; BZW2; C11orf68; C1QTNF6; C1orf21; C1orf226; C2orf76; CADM2; CALCOCO2; CBX4; CCDC159; CCDC171; CCDC57; CCDC9B; CCND3; CCNG2; CD109; CD59; CDC40; CDC42SE1; CDC7; CDCA2; CDK5RAP1; CDK5RAP2; CECR2; CELSR1; CEP112; CFAP418; CFDP1; CHCHD5; CHD1; CHD2; CHEK2; CHP1; CIART; CIPC; CLASP1; CLGN; CLK4; COL12A1; CPD; CRKL; CRYBG1; CSAD; CSGALNACT1; CSTA; CTR9; CTSB; CTSC; CTSD; CUEDC1; CUL2; CUX1; CXADR; CYB5A; CYP4B1; CYTH1; DAAM1; DAGLB; DBI; DCBLD2; DDB2; DDIAS; DDIT4; DDX10; DDX47; DDX60L; DENND10; DENND2B; DIO2; DLC1; DLEU2; DLG5; DLGAP1-AS1; DMAP1; DNAH11; DPH6; DPY19L1; DRAIC; DRAM1; DRAP1; DTNA; DTWD1; DUSP1; E2F2; E4F1; ECHDC3; EDEM1; EFCAB2; EFNA1; EFNA5; EHD1; EHF; EHMT1; EID1; EIF2S3; EIF4E; EIF5A2; ELOVL6; EML2; ENAH; ENO1; ENSG00000215154; ENSG00000255468; ENSG00000268858; EPB41; EPHB4; EPS8; ERCC4; ERGIC1; ERI1; ETNK2; ETV6; EVL; EXD3; EXOSC7; EZR; FAM107B; FAM13A; FAM167A; FAM219B; FBXL18; FBXL5; FBXO38; FERMT2; FGD4; FGFR1; FLNB; FLOT1; FMN1; FNDC3A; FOS; FOSL2; FOXN2; FTL; GALNT10; GALNT18; GARS1; GARS1-DT; GBE1; GCC2; GCLC; GMCL1; GNAL; GON4L; GPAT3; GPD1L; GPRC5A; GREB1; GTF2F2; GTF3C3; GTPBP2; GUSBP1; GZF1; H2BC21; H4C3; HBP1; HBS1L; HDAC11; HDAC5; HERPUD2; HERPUD2-AS1; HES4; HGD; HIVEP2; HK1; HMG20A; HMGCL; HMOX1; HNMT; HNRNPF; HOXC-AS2; HSPD1; HSPE1; HTR1D; IFNGR1; IFT122; IGF1R; IGF2R; IL1R1; IL1RAP; IL6R; INAVA; INTU; IQGAP2; IRF2; IRS2; ISG15; ISG20; ITGB1; ITGB3BP; ITGB5; ITPRIPL2; JDP2; JPX; JRK; KAT2B; KATNBL1; KCNK1; KDM3A; KDSR; KIAA0232; KIAA0319L; KIFC3; KLF13; KLF5; KLF7; KLHDC4; KLHL24; KMT2A; KRT7; KRT8; LDLRAD4; LGALSL; LIMCH1; LINC-PINT; LINC00511; LINC00886; LINC00963; LINC02615; LMO4; LMO7; LMX1B; LPIN2; LRBA; LRIG3; LRRC27; LRRFIP2; LTBP1; LTBR; LZTFL1; MAFK; MAP2K3; MAP3K20; MAP3K8; MAPK15; MAPK3; MARCHF3; MARCHF7; MBD4; MBNL1; MBP; MCCC1; MCCC2; MCL1; MCTP1; MDH1; MDH2; ME2; MECP2; MED23; MED28; MEIS1; METTL15; MFSD4B; MFSD6; MIB1; MID1IP1; MIDEAS; MIDN; MIOS-DT; MIR99AHG; MITF; MIX23; MLPH; MPP3; MRC2; MRPL46; MRPL54; MRPS15; MRPS23; MRPS35; MRPS5; MT2A; MTHFD1L; MTHFD2L; MTMR12; MTMR9; MUC1; MYO10; NBL1; NCOA7; NDRG1; NDUFA12; NDUFB11; NDUFS4; NEAT1; NEDD4; NEDD4L; NFE2L2; NFIA; NFIB; NFIC; NFKBIA; NFKBIZ; NFX1; NGLY1; NHEJ1; NIBAN1; NIBAN2; NIPAL2; NISCH; NKILA; NOL11; NR4A1; NRG1; NRP1; NTN4; NUF2; NUMB; NXN; OIP5; OPN3; OSBPL1A; OSMR; P4HA2; P4HB; PAFAH2; PALMD; PARD3; PARP8; PCBP1-AS1; PCID2; PDAP1; PDCD6; PDE4D; PDGFC; PDK1; PDLIM5; PEAK1; PECR; PER1; PER2; PHACTR4; PHF20; PIGT; PIK3C3; PKP4-AS1; PLCE1; PLD1; PLEKHG2; PLOD2; PMEPA1; PNRC1; POC1A; POLG; PPARG; PRCP; PRDM15; PRDM6; PRKCH; PRMT5; PSMG3; PTCH1; PTPN12; PTPRG; PTPRM; PVT1; PYCR1; RAB31; RAB3D; RAB5IF; RABEPK; RBM39; RCAN1; RCL1; RFESD; RFFL; RHOBTB2; RHOBTB3; RHOT2; RMDN3; RNF13; RNF145; RPH3AL; RPL24; RPL34; RPL6; RPS6KB1; RPS7; RRBP1; RSU1; RUSC2; RUVBL1; RXRA; S100A10; S100A6; S100A8; S100A9; S100P; S100PBP; SBNO2; SCN8A; SCNN1A; SCUBE2; SEC62; SEH1L; SEL1L3; SEMA4C; SEPTIN2; SEPTIN9; SETD5; SGK3; SGPP1; SH3BP4; SH3TC1; SHANK2; SIAH1; SKIL; SLC16A1; SLC1A5; SLC25A28; SLC2A4RG; SLC33A1; SLC38A1; SLC38A2; SLC44A1; SLC45A4; SLC7A6; SLC9A1; SLPI; SMAD7; SMARCA2; SMARCD2; SMIM14; SMIM20; SNHG16; SNX18; SNX27; SNX8; SP100; SPATS2L; SPIDR; SPIN1; SPPL2A; SPRED2; SPRY4; SQOR; SRBD1; SRCIN1; SRI; SRRM3; SSBP2; STAT1; STAT3; STAU1; STC1; STK40; STX1A; SUB1; SUMF2; SUSD1; SYBU; SYNC; SYNPO; SYT7; TAGLN2; TBC1D16; TBPL1; TCF7L2; TDP2; TEAD1; TET2; TEX2; TFE3; TFEB; TFRC; TGFBI; TGFBR1; TGFBR2; TGIF1; THADA; THRA; TIGD2; TIMM13; TM4SF1; TMBIM1; TMC1; TMED10; TMED2; TMEM143; TMEM205; TMEM260; TMEM33; TMEM40; TMEM9; TMSB4X; TNFRSF1A; TNIP1; TNPO1; TNS1; TPD52L2; TPM4; TPRA1; TPT1; TPT1-AS1; TRAPPC3; TRAPPC5; TRIB1; TRIM29; TRPC4AP; TRPM7; TSEN15; TSFM; TTC7A; TTC7B; TUBA1C; TUT1; TXNDC11; TXNRD1; UBC; UBE2B; USP3; USP54; USP9X; UTRN; VAV3; VMP1; VWA8; WASF1; WASL; WDPCP; WDR7; WIZ; WWTR1; XDH; YAP1; YARS1; ZBTB20; ZBTB38; ZBTB44; ZBTB7A; ZC3H12A; ZC3H7A; ZDHHC18; ZDHHC3; ZFAND5; ZFP36L2; ZFR; ZMAT1; ZMIZ1; ZNF14; ZNF217; ZNF221; ZNF354B; ZNF529; ZNF580; ZNF581; ZNF594; ZNF706","AAMP; ABCA3; ABCC3; ACP2; ACTB; ACTG1; ADAM15; ADAM9; ADD1; ADNP; AGPS; AHCYL2; AHI1; AHNAK; AIMP2; ALKBH7; ALKBH8; ALPK1; AMBRA1; AMMECR1; ANKRD17; ANKRD42; ANKS6; ANKZF1; ANO10; ANP32A; ANXA11; AP2S1; APH1B; APP; ARAP1; ARHGAP24; ARHGDIA; ARHGEF12; ARL4A; ARNTL; ARPC2; ARPC5; ASAP3; ATG9A; ATP5F1D; ATP5MC1; BABAM1; BANP; BCL7C; BIRC5; BLOC1S4; BOD1; BRD2; C17orf58; C19orf44; C1orf159; C2CD3; C2CD5; C6orf62; CABIN1; CACTIN; CALCOCO1; CAMKK2; CAMTA1; CAND1; CAPN15; CBX4; CCAR1; CCDC137; CCDC159; CCDC25; CCNL1; CCT6B; CDC40; CDCA3; CDH23; CDK5RAP2; CDYL; CEP290; CEPT1; CFL1; CHCHD2; CHD2; CHMP3; CHMP4B; CLASP1; CLEC16A; CLPB; CLSTN1; CLTC; CNOT10; COG3; COX7C; CPEB2; CRTC2; CRYBG2; CUL5; CWC22; DDX23; DDX3X; DFFA; DGCR8; DHX58; DNAAF5; DRAM2; DYNC1I2; EBAG9; EEF2K; EIF3B; EPC1; ERI1; ERP44; EZR; FAM168B; FASN; FBXO15; FBXO38; FBXW11; FCHSD2; FEM1A; FGGY; FLYWCH1; GAB2; GALE; GAPVD1; GFER; GFUS; GLB1; GNG4; GON4L; GPATCH8; GPBP1L1; GRB10; GSPT1; GTF2IRD1; H4C8; HERC1; HES4; HIKESHI; HM13; HMBOX1; HMGB1; HMOX2; IFRD2; IFT20; IFTAP; IGF1R; IKZF2; IL1R1; IMPACT; INTS9; INVS; IRF2BP2; ISG15; ISG20; ITGAL; JMJD1C; JOSD2; KBTBD6; KBTBD7; KCNK1; KCTD20; KDM7A; KDM7A-DT; KHSRP; KIAA0100; KIF15; KMT5A; KNTC1; LARS1; LCOR; LEMD2; LEPROTL1; LINC00235; LINC01521; LRBA; LRP6; LSM10; LUC7L; LYPLA1; LYSMD1; MACROH2A1; MAD2L1BP; MAN2A2; MANEAL; MAPK13; MARS1; MATN1-AS1; MAU2; MAZ; MDH1; MFAP3L; MGRN1; MKKS; MLLT10; MRFAP1L1; MRPL52; MTMR4; MTMR9; MTR; MUC20-OT1; MUS81; MXD1; MYCL; MYL12B; MYO5B; NADK2; NCOA3; NDC1; NFE2L1; NFE2L2; NFIX; NMRAL1; NOSIP; NPEPPS; NPEPPSP1; NPL; NR4A1; NTPCR; NUMA1; NUP210; OIP5-AS1; OPLAH; OTUD7A; P4HB; PAFAH2; PAK1; PANK4; PATZ1; PCBP1-AS1; PCSK6; PDE7A; PEDS1; PEMT; PGP; PHB; PHF12; PHF8; PITHD1; PLAUR; PLEKHF2; PNKD; PNPLA6; POLDIP3; PPA2; PPM1E; PPP1R12C; PPP1R15A; PPP1R3E; PPRC1; PRDM5; PRKAB2; PRKCE; PRKCH; PROSER2; PRPSAP1; PRR5; PSEN1; PTDSS2; PTK2; PUM3; PVT1; RAB1A; RAB30; RAB30-DT; RAB4A; RAB5IF; RAB7A; RABGAP1L; RAD51C; RANBP10; RANGAP1; RBBP4; REXO2; RFFL; RFX1; RIC8B; RIN3; RIPOR3; RPIA; RPP14; RPRD1B; SAP30; SCAF1; SCAMP5; SCNM1; SCOC; SCRIB; SEC14L1; SEC24C; SEMA4B; SFSWAP; SFT2D2; SH3TC1; SIAH1; SKA2; SKIL; SLC25A12; SLC25A6; SLC48A1; SLC4A11; SLC4A2; SLC5A6; SLC7A5; SLC9A1; SLK; SLMAP; SLX4IP; SMIM13; SNF8; SNX10; SNX19; SOSTDC1; SPTBN4; SRP19; SRSF1; ST3GAL2; ST6GALNAC4; STARD10; STAT5B; STAU1; STAU2; STMN1; STRIP1; STX4; SUGP1; SULT2B1; SUMO3; SYNPO; TANC1; TARBP2; TASOR2; TBX6; TCEA2; TCERG1; TEAD1; TEDC1; TELO2; TERF2; TFEB; THADA; TIMM21; TIPRL; TKT; TM9SF4; TMEM11; TMEM115; TMEM209; TMEM245; TMEM69; TMPPE; TMTC3; TNFAIP1; TNKS; TP53RK; TPGS1; TPM4; TRIM29; TRMT12; TSC22D1; TSSC4; TTC1; TUBB2A; TUFT1; TXNDC15; UBALD2; UBC; UBE2E2; UBE2E3; UBE3C; UCKL1; UFL1; USF1; USP10; USP31; USP33; USP9X; VDAC2; VPS35L; VPS37B; VPS41; VRK1; WASF1; WBP1; WBP2; WDPCP; WDR1; WDR46; WSB1; WWP1; WWTR1; XIST; XKR9; XYLT2; YEATS2; YIF1B; ZBED9; ZBTB38; ZC3H7A; ZCCHC8; ZFYVE1; ZFYVE28; ZMYND8; ZNF217; ZNF225; ZNF280C; ZNF280D; ZNF337; ZNF444; ZYX","AK4; BCYRN1; CD44; CDH1; EP300; FN1; FZD4; HIF1A; HOTAIR; KIF5B; LDHA; LINC01089; MIAT; MTDH; MTERF4; MYC; PTEN; PTK2B; RAD51-AS1; SLC2A1; STAT1; TEAD4; TFAP2A; TWIST1; VEGFC; VMP1","ABCB9; ABCC5; ABLIM2; ACAD10; ACP2; ADAM9; AFF1; AKR1C3; AMBRA1; ANK3; ANKRD17; ANLN; ANP32E; AP2B1; ARFGEF1; ARHGEF2; ARHGEF26-AS1; ARID5B; ARRDC1; ASNSD1; ASRGL1; ATAD1; ATG7; ATP11B; ATP13A4; ATP2A2; B4GALT5; BANP; BCR; BICD1; BRD2; BRD3OS; BTBD11; BUD31; BZW2; C1orf52; C2orf76; C3orf33; CAND1; CANX; CASP2; CBX4; CCDC57; CCDC59; CD82; CDC14A; CDC42SE1; CDK13; CDK5RAP1; CEP350; CFAP45; CHD2; CLASP1; CMAHP; CNOT8; COL18A1; COQ8A; COX5A; CPD; CTDSP2; CTNNA1; CUEDC1; CYB5D1; DAAM1; DBI; DDHD1; DDX55; DECR2; DENND1B; DGLUCY; DHRS2; DHX29; DLG1; DNAJC19; DNAJC7; DNAJC9; DNASE1; DRAM1; DYNLL1; ECT2; EEF1AKMT3; EHHADH; EID1; EIF2AK4; EIF2S3; EIF3E; EIF4A2; EIF4G3; ELMO1; ENSG00000215154; ENSG00000224934; ENSG00000275765; ERAL1; ERGIC2; ETFA; EXOC3-AS1; EXPH5; FAHD1; FAM107B; FAM20B; FAR2; FBXL18; FBXL2; FNDC3B; FNTA; FRMD3; FRMD4A; FRS2; FXR1; G3BP1; GARS1; GDAP1; GNA12; GOLPH3L; GOT1; GPHN; GPS1; GPX3; GPX8; GRAMD1B; GREB1; GSE1; GZF1; H3-3B; HAGH; HECTD1; HEG1; HERC6; HNRNPF; HOXC10; HSPB1; IFIT5; IGF2BP2; INTS6; IPO5; IYD; JUP; KCNB1; KDM4A-AS1; KIAA0895; KIAA1109; KIF23; KIFAP3; KLF10; KMT2A; KMT2E; KMT5A; KRT19; KRT81; KRT8P46; KSR1; LCOR; LDHB; LIMCH1; LIN7C; LINC01089; LINC01137; LINC01588; LINC02303; LMNA; LPP; LRRC26; LRRC45; LRRCC1; LUC7L; LYRM9; MAGI2; MAP3K12; MAP3K4; MAPK6; MAPRE2; MARS1; MBNL1; MBNL2; MDM2; MEF2A; METTL1; METTL25; MEX3A; MFSD11; MGA; MIDEAS; MIOS; MOB4; MORF4L1; MRPL13; MRPS15; MRPS22; MRPS35; MTA1; MTHFD2L; MTREX; MTRR; MXRA7; MXRA8; NAA38; NCALD; NCAM2; NCL; NDE1; NDUFAF6; NDUFS4; NFE2L2; NFX1; NIT1; NOL3; NR1D1; NR2C1; NR6A1; NSMCE1; NSMCE2; NSMF; NUBPL; NUDT9; NUF2; NUPR1; OPA1; ORAI1; ORAI3; OSBPL8; OVCH1-AS1; OXR1; PARP14; PBXIP1; PDAP1; PEX14; PHACTR4; PHF14; PHF19; PHF20; PHF20L1; PIK3CB; PITRM1; PLA2G6; PLD1; PMM2; POC1B; POFUT1; PPM1L; PPP2CA; PPP2CA-DT; PPP2R5C; PPP3CB; PPP3CB-AS1; PRKAB2; PRLR; PROSER2; PROSER3; PTK2; PWWP2A; PYCR1; R3HDM2; RAB31; RAB5B; RAB5IF; RABEPK; RAD23B; RAD51; RAD51-AS1; RALA; RALGAPB; RAPGEF3; RAPGEF5; RAPGEF6; RFC4; RFC5; RGMB-AS1; RGS9; RIC1; RNF157; RNF216; RNLS; RPA1; RPL6; RRP12; RSU1; SARS2; SC5D; SCAMP5; SEC11A; SENP2; SENP5; SETD5; SH3BP5; SKP1; SLC1A5; SLC2A4RG; SLC33A1; SLC35B1; SLC48A1; SLC6A6; SMARCD2; SMARCE1; SMIM14; SMURF2; SNRPA1; SNX1; SNX2; SNX27; SPAG9; SPECC1; SPIDR; SPRYD3; SPSB1; SQOR; SRCAP; SRGAP1; SRPRA; SRPRB; SSR3; ST3GAL1; ST3GAL4; STARD3; STAT3; STRBP; SUPT4H1; SYNE1; SYTL2; TAPT1; TARBP2; TARDBP; TARS1; TBC1D16; TBL1XR1; TCEA2; TFAP2A; TFDP2; THRA; TIGD6; TIMM44; TMEM106C; TMEM167A; TMEM186; TMEM192; TMEM25; TMEM41A; TMEM44; TMEM9; TMEM94; TNIK; TNK2; TNS2; TOR1AIP1; TOR3A; TPM4; TRAPPC9; TRDMT1; TRIM2; TRIQK; TSEN15; TSNAX; TSTD1; TTC21A; TXNIP; TXNRD1; ULK2; UNC13D; VCPKMT; VDR; VPS13B; VPS13B-DT; VPS26B; VPS41; VPS45; WBP1L; WIPF2; WWP2; YEATS4; YTHDF3; ZC3H12A; ZC3H7A; ZCCHC24; ZFAND6; ZNF131; ZNF621","ABCC2; ABCC3; ABCC4; ABCC5; ABHD2; ACAA1; ACADM; ACKR3; ACOX1; AHR; AKAP13; ALAS1; AMIGO2; ANGPTL4; BAX; BHLHE40; BIRC2; BLVRB; CAP2; CBR3; CDC37; CDC42EP3; CDK1; CDK4; CDKN1B; CES2; CPEB4; CPT1A; CPT2; CYP1B1; DBI; DNAJB1; DNAJC15; DNAJC7; EDN2; EGFR; EHHADH; ENC1; EP300; EPB41L4B; ESR1; ETNK2; FASN; FGD4; FGF13; FKBP5; FOXO1; FTL; G6PD; GADD45B; GCC1; GCLC; GCLM; GPAM; GPR153; GPX3; GSR; GSTA4; GSTM3; GSTM4; HES1; HMOX1; HSP90AA1; IL12A; IRS2; JUN; JUND; KAT2B; KEAP1; KTN1; LRRC8A; MAFF; MAFG; MGST3; MYC; MYOF; NCOA1; NCOA2; NCOA3; NFE2L2; NQO1; NRG1; NRIP1; PDE4B; PDGFB; POLK; PPARA; PPARD; PRDX1; PRDX6; PSMC5; PTGR1; RGS2; RXRA; S100P; SCD; SCNN1A; SCP2; SEC14L1; SERPINA1; SERPINB2; SERTAD2; SLC19A2; SLC26A2; SLC2A1; SLC2A10; SLC2A13; SLC2A6; SLC39A1; SLC39A10; SLC39A11; SLC39A14; SLC39A6; SLC39A9; SLC5A12; SLC5A6; SLC6A11; SLC6A17; SLC6A6; SLC6A9; SLC7A5; SMARCA1; SMC1A; SP1; SPRY1; SRXN1; STAT3; STOM; SULT1A1; TGFA; TGFBR2; TGFBR3; TSC22D3; TXN; TXNRD1; VDR","AAMP; ABCA5; ABRAXAS2; ACAD8; ACOX2; ACP2; ACTG1; ACTN4; ACTR1A; ADAM9; ADCY9; ADD1; ADNP; ADO; ADSS2; AFDN; AFMID; AGPS; AHCTF1; AHCYL2; AHI1; AHSA1; AKIRIN2; AKT2; ALDH5A1; ALDOA; ALG10; ALKBH8; AMMECR1; ANKMY1; ANKRD11; ANKS6; ANKZF1; ANP32E; ANXA5; AP2M1; AP2S1; AP4M1; APLF; ARHGAP5; ARHGAP5-AS1; ARHGDIA; ARHGEF2; ARMH3; ARNTL; ARPC2; ARV1; ASAP3; ASPH; ATAD2; ATG4B; ATG9A; ATP5MF; ATP7A; AUP1; B4GALT2; B4GALT3; BABAM1; BABAM2; BANP; BICDL1; BIRC5; BLM; BOD1; BRD3; BRD4; BRD9; BSG; BTBD1; BTBD3; BTRC; BUB1B; BUB3; C11orf58; C18orf54; C19orf44; C1orf109; C1orf159; C2orf69; CA12; CABIN1; CACTIN; CALCOCO1; CAMKK2; CASP8AP2; CBFA2T3; CCDC115; CCDC136; CCDC159; CCDC191; CCDC71; CCNA2; CCNB2; CCNG2; CCP110; CDC40; CDCA2; CDCA3; CDCA8; CDH23; CDK11A; CDK5; CDK5RAP2; CDYL; CECR2; CENPA; CENPO; CENPT; CEP128; CEP290; CEP295; CEP89; CFL1; CHCHD2; CHCHD6; CHD7; CHEK2; CHFR; CHIC2; CHMP4B; CHPT1; CIAO2A; CIDECP1; CLEC16A; CLINT1; CLSTN1; CLTC; CMIP; CMSS1; CNPY2; COMMD3; COPS7B; COX7B; COX7C; CRADD; CRAMP1; CRTC2; CSGALNACT2; CSNK2B; CTBP1; CTCF; CTDP1; CWC22; CYB561D2; DCAKD; DCK; DDIAS; DDX23; DEDD; DENND1A; DGCR8; DIAPH3; DLGAP4; DMXL2; DNAJC17; DNAL1; DPYSL2; DUS1L; DUSP14; DYNC1I2; E2F1; E2F2; E2F3; EEF2K; EFNA3; EHD1; ELOC; EMC3; EME1; EML6; ENSG00000215154; ENSG00000233834; ENSG00000234432; ENY2; EPC1; ERP44; ERVK3-1; EXOC6; FAAP24; FAM110A; FAM117A; FAM168B; FAM193A; FAM222A; FANCC; FANCD2; FANCI; FASN; FASTKD1; FBF1; FBRSL1; FBXL16; FBXL20; FBXL5; FBXO15; FBXO38; FBXO46; FBXO48; FBXO5; FBXW11; FCHSD2; FEM1A; FGFR1; FHL2; FN3K; FTO; FYCO1; GAB2; GATB; GBAP1; GDAP1; GFER; GFUS; GINS2; GLUD1; GNG4; GP6-AS1; GPANK1; GPAT4; GPBP1L1; GPT2; GRAMD4; GRHL1; GRK3; GRK4; GSPT1; GTF2IRD1; GTPBP2; GUCD1; H2AC11; H4C8; HAUS8; HDAC5; HECTD2; HERC1; HIF1A; HIKESHI; HINT3; HM13; HMG20A; HMGB1; HNRNPF; HOOK3; HSCB; HSPD1; HSPE1; HTRA2; IFRD1; IFRD2; IFT140; IFT80; IGF1R; IKZF2; IL1R1; IMPACT; ING3; INTS1; INVS; IPO13; IQGAP3; IRF2BP2; KAT7; KBTBD6; KBTBD7; KCNB1; KCNC3; KCNK1; KCTD20; KCTD9; KDM7A; KDM7A-DT; KIAA0232; KIF15; KIF18B; KIF20B; KIF22; KLF13; KMT2B; KPNA3; LAMTOR2; LARS1; LEMD2; LEPROTL1; LGALSL; LIFR; LIN9; LINC01521; LMNB1; LMNB2; LPIN2; LSM10; LUC7L; LUC7L2; LYSMD1; MACC1; MAD2L1BP; MAGOHB; MAN2A2; MANEAL; MAPT; MARS1; MAT2A; MAU2; MAX; MBTPS2; MCM2; MCM5; MCM7; MDH1; MDM2; MED15; MEGF11; METTL15; METTL4; MFAP3L; MGME1; MIDEAS; MIS18BP1; MITF; MKI67; MKKS; MMACHC; MOB4; MPI; MRGBP; MRPL27; MRPL52; MRPS25; MTMR4; MUS81; MYB; MYCL; MYO6; MYO9B; MZT1; NADK2; NCAPH; NCOA5; NDC80; NEDD4; NEK6; NF2; NFASC; NFE2L2; NFXL1; NKAIN1; NMT1; NOP14; NOP53; NOSIP; NPEPPS; NPEPPSP1; NPRL2; NRDE2; NSD2; NSMCE1-DT; NTAN1; NTHL1; NUCB1; NUDCD1; NUDT13; NUF2; NUMA1; NUTF2; OBSL1; OPLAH; ORMDL3; OSTF1; OTUD7A; PACS2; PAFAH2; PAN2; PANK4; PATZ1; PBK; PCBP1-AS1; PCED1B; PCSK6; PDE3B; PDE6D; PDIK1L; PDS5B; PEAK1; PEX26; PGP; PHB; PHF12; PHF13; PHF21A; PHLPP1; PIAS1; PLEKHF2; PNKD; PNPLA6; PODXL2; POLE2; PPCDC; PPM1E; PPP1R12C; PPP1R15A; PRCP; PRKCA; PRKCE; PRKD1; PRKDC; PRMT3; PRR5; PTCD1; PTDSS2; PTMS; PTPRN2; PTRHD1; QRICH1; QTRT1; QTRT2; RAB30; RAB30-DT; RAB7A; RABEP1; RABGAP1L; RAD51; RAD51-AS1; RAD51B; RALBP1; RANBP10; RANGAP1; RBBP4; RBKS; RBL1; RBM6; RCCD1; RERE; REXO2; RFC3; RFC4; RHOBTB3; RIC8B; RIN3; RMDN3; RNF170; RNF213; RNF227; RPF1; RPGRIP1L; RPH3AL; RPIA; RPL23A; RPP40; RPS13; RPS20; RRM1; RTEL1; RTEL1-TNFRSF6B; RTKN2; RXRA; RYK; SAP30; SASS6; SAV1; SBK1; SCAF1; SCAMP5; SCNM1; SCRIB; SDK1; SEC14L1; SEC24C; SEC61A1; SEL1L; SETMAR; SGPL1; SHLD2; SIAH1; SIK3; SLC35A1; SLC35F2; SLC38A1; SLC44A1; SLC45A4; SLC48A1; SLC4A2; SLC6A6; SLK; SLX4IP; SMARCAL1; SMC6; SMIM30; SMPD3; SMUG1; SNAPC3; SNIP1; SNRPD3; SNX1; SNX5; SPC25; SPG11; SPTBN4; SRCIN1; SRRM3; SRSF1; ST3GAL2; ST7; STAT1; STAT3; STAT5B; STAU1; STK25; STMN1; STMN3; STN1; STON2; STRADB; STRIP1; SUGP1; SUMO3; SYMPK; SYNGR1; TACC1; TBC1D31; TBC1D4; TBC1D8B; TBX6; TCEA2; TCP11L1; TEDC2; TFAP2A; TFEB; TGFBR3; THAP1; THAP12; THUMPD2; THYN1; TIAL1; TIMM21; TIMM22; TK1; TKT; TM9SF4; TMBIM1; TMEM11; TMEM115; TMEM131; TMEM150A; TMEM160; TMEM181; TMEM33; TMEM69; TMEM70; TMTC3; TNRC6B; TOP2A; TOR2A; TP53; TP53BP2; TPGS1; TPRA1; TPX2; TRAK2; TRAPPC3; TRIM29; TRIP13; TSC2; TTC13; TTC33; TTLL4; TUBGCP3; TXNDC15; UBE2E2; UBE2E3; UBE2G2; UBE2Q1; UBE2Q2; UBE3C; UBQLN4; UBR2; UBR3; UBXN2A; UCKL1; UNG; UPF1; USF2; USP28; USP3; USP31; USP39; USP48; USP6NL; VAPA; VCPIP1; VDAC2; VIPAS39; VKORC1; VMP1; VPS35L; VPS50; VRK1; VRK3; WASF1; WBP2; WDPCP; WDR13; WNK1; WRAP53; WT1-AS; WWP1; XIST; XRCC3; XYLT2; YEATS2; ZBTB11; ZBTB46; ZFX; ZFYVE1; ZFYVE19; ZFYVE21; ZFYVE28; ZNF133; ZNF155; ZNF217; ZNF225; ZNF280C; ZNF280D; ZNF32; ZNF331; ZNF337; ZNF354A; ZNF37A; ZNF436; ZNF444; ZNF451; ZNF473; ZNF573; ZNF589; ZNF605; ZNF687; ZNF785; ZW10; ZWINT; ZYG11A","ACAA1; ADD3; AGPS; AGR3; AHR; AKAP13; AKAP9; ANAPC7; ANK3; AP4E1; ARHGAP26; ARHGAP32; ARID5B; ATP2C1; C17orf67; C1orf21; C1orf226; CAP1; CASD1; CCDC115; CCDC159; CEP120; CFDP1; CIB1; CTDSPL; DMXL2; DNAH11; DNMBP; DNTTIP2; DRAIC; EFCAB14; ELP3; EML6; ENC1; EPS8; FERMT2; GARNL3; GDPGP1; GLRX; GNAL; GPR160; H2AC7; H2BC5; HEG1; HINT2; HMOX1; HNMT; HOXC6; HPGD; ICA1; IFT46; ILRUN; INO80; IRS1; ITGB5; JMJD1C; JMY; KAZN; KCNK1; KHDC4; LAMA3; LEKR1; LIMA1; LINC00511; LINC01588; LMO7; LRRC1; LTBP1; LUC7L2; MALSU1; MAPK6; MBP; MLLT3; MPZL1; MTMR11; MYO10; MYOF; NAALADL2; NASP; NFKBIZ; NOD1; NPM1; NXN; OAS1; PANK3; PDE4D; PDHX; PGM3; PHF12; PIGC; PIM1; PLCB1; POMP; PPFIBP2; PPP2R2A; PRKCI; PRLR; RAPGEF6; RBM47; REPIN1; REXO2; RIOK2; RND3; RNF125; RNF144A; RNF145; RPF1; RPS27A; RPS27L; S100A16; SCOC; SELENOP; SEMA3C; SEPTIN8; SETD5; SGO2; SLC9A1; SMAD3; SMIM31; SNAP23; SP110; SPATS2L; SPRED2; SPSB1; SRI; STARD4; SUCLA2; SUMF2; SYNE2; TBL1XR1; TDP2; TERF2; TGS1; THADA; TMCC1; TMEM8B; TMOD3; TMPRSS4; TNRC18; TRIO; UTRN; VCPIP1; VMP1; VWA8; XPNPEP1; YAP1; ZNF217","ABCC1; ACVR2A; ADD3; ADGRL2; AKT3; ANXA2; ANXA4; APBB2; ATP2B1; ATP2C1; ATRN; ATRX; ATXN1; BCKDHB; BHLHE40; BMPR1A; CAP2; CCN1; CDC42BPA; CDK13; CDKN1B; CDON; CITED2; COL5A2; DBP; DDAH1; DLC1; DLG1; DUSP1; DYRK1A; EFEMP1; ERBB2; F3; FHL2; GCNT1; ICA1; ID1; IGF1R; IGFBP5; INPP4B; IRS1; KALRN; KCNMA1; LAMC1; LPAR1; LTBP1; MAGI2; MAP1B; MAP2K5; MAPK14; MGLL; MIOS; MMP16; MTA1; MYC; NFIB; NFKB1; NIPBL; NOTCH2; NR1D2; NRP1; PDLIM5; PEX14; PHF3; PIAS3; PIK3R3; PLCB4; PLPP3; PMP22; PPARG; PRDM2; PRKAR2B; PRKCA; PRKCE; PTEN; PTPN21; PTPRM; RASA2; RBPMS; RND3; RUNX1; RXRA; SCN8A; SDC2; SFMBT1; SIPA1L1; SMAD3; SMAD7; SPOP; SRI; SYNE1; SYNJ2; TGFBR2; TGFBR3; TJP1; TOGARAM1; VAV2; VLDLR; YTHDC1; ZMIZ1","AACS; ADAT2; ADGRG1; ALDOA; ANKRD42; ANKRD42-DT; AP2S1; ARID4B; ARL4A; ARNTL; ASB3; ASB7; ATP13A4; ATXN7L3B; AZI2; BABAM1; BDP1; BRMS1; BSG; C19orf44; CABIN1; CAMK2D; CASP9; CBLL1; CBX8; CCDC157; CCDC191; CCDC40; CCDC91; CDH23; CFAP45; CFDP1; CHRNA7; CHST11; CIC; CITED2; CLGN; CLPB; CMTM6; CNIH2; COPE; COPS2; CREBL2; CRIP2; CRTC2; CSF1; CTU1; DCAF6; DDX23; DDX3X; DDX49; DHX8; DNAH7; DNAJC16; DNAL4; DNALI1; DTWD1; DUS3L; DUSP16; DYNC1I2; DYRK1A; DYRK1B; DYRK2; ECT2; EFCAB11; EFCAB2; EFHC1; EHD1; ELAPOR1; EMC3; ENY2; EPHB6; ERCC1; ERG28; ERLEC1; ESYT1; ETFRF1; ETV1; FAM13A; FAM229B; FBXO32; FEM1A; FGGY; FOXO6; GALK2; GEMIN6; GET1; GGPS1; GIHCG; GRK4; GSE1; H4C8; HBP1; HECTD1; HIPK1; HM13; HMBOX1; HNRNPF; HSP90B1; IDS; IFRD2; IFT122; IFT20; IK; ILF3; ILF3-DT; ING3; INTS1; INTS9; IPO13; IQGAP2; IRF2BPL; JMJD1C; JUND; KAZN; KLF13; KLF4; KLHL18; KLHL24; KRT8; LAMTOR2; LCORL; LEMD2; LINC00630; LINC01876; LINS1; LMNA; LRRC23; LRRC27; LYPLA2; MAFK; MAML3; MAP4K3; MARCHF8; MBD4; MBTPS1; MBTPS2; MDH1; MED31; MFSD4B; MLF1; MLLT1; MNS1; MOK; MPC2; MRPL24; MRPS21; MTX1; NAA20; NBAS; NCOA5; NCOR2; NDRG4; NDUFA2; NEK11; NFATC2; NFIA; NLK; NOD2; NOP14; NR4A2; NRM; NUBPL; NUDCD1; NUDT13; NUF2; NUP62CL; NUPR1; OAZ2; OPA1; OSR2; P4HB; PANK4; PBX1-AS1; PCGF5; PCSK6; PER1; PES1; PFDN6; PHF8; PIBF1; PIFO; PIM1; PLEKHF2; PPOX; PPP1R12C; PPP1R15A; PPP1R3E; PRORP; PSTPIP2; PTCH1; QTRT2; RABGAP1L; RARA; RBM23; RERE; RFC5; RFX1; RFX3; RGL2; RMND5A; RPRD1B; RPS6KA5; RRM2B; RRP12; RSRC1; SARAF; SELENOP; SETD5; SF3A1; SIK3; SIRT1; SLC16A1; SLC38A2; SLC44A1; SMC1A; SNX29; SRSF7; SSBP1; ST7; STAG2; STAU2; STK32C; STRIP1; SUMO3; SUN1; SYNE2; TAGLN2; TAPBP; TARS1; TBC1D16; TCEA2; TDP1; TENT5C; TFAP4; TGIF1; THADA; THBS3; TIMM44; TIPARP; TMED10; TMEM115; TMEM139; TNFAIP1; TOM1L2; TOR2A; TRAM1; TTLL5; TUBB4B; TUBE1; TUFT1; TXNDC11; TXNIP; UACA; UBE2Z; UBL7; UBL7-DT; UBQLN4; UBXN11; UFSP2; VRK3; VTA1; WBP1; WBP2; WDPCP; WDR46; WDR77; WHRN; YBX3; YIPF4; YLPM1; YTHDC1; ZC2HC1C; ZC3H10; ZDHHC1; ZDHHC14; ZFP36L1; ZFP36L2; ZFR; ZFYVE28; ZMYND10; ZNF131; ZNF3; ZNF37BP; ZNF473; ZNF487; ZNF497; ZNF510; ZNF584; ZNF837; ZSWIM8","ABCC4; AKR1B1; AKR1C3; ANXA1; ANXA2; ANXA4; ANXA5; ANXA6; EDN1; HPGD; HSD11B2; MITF; PLA2G4A; PPARG; PPARGC1B; PTGER4; PTGES; PTGFRN; S100A10; S100A6; SOX9"]},"columns":[{"id":"gs_name","name":"Gene Set","type":"character","cell":[{"name":"a","attribs":{"href":"\n      http://www.gsea-msigdb.org/gsea/msigdb/collections.jsp","target":"_blank"},"children":["HALLMARK ESTROGEN RESPONSE EARLY"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["GREB1 TARGET GENES"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["ZNF766 TARGET GENES"]},{"name":"a","attribs":{"href":"\n      http://www.gsea-msigdb.org/gsea/msigdb/collections.jsp","target":"_blank"},"children":["HALLMARK ESTROGEN RESPONSE LATE"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["ZNF197 TARGET GENES"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-8978868|https://reactome.org/PathwayBrowser/#/R-HSA-8978868","target":"_blank"},"children":["REACTOME FATTY ACID METABOLISM"]},{"name":"a","attribs":{"href":"\n      http://www.gsea-msigdb.org/gsea/msigdb/collections.jsp","target":"_blank"},"children":["HALLMARK HYPOXIA"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["HES2 TARGET GENES"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["ZNF175 TARGET GENES"]},{"name":"a","attribs":{"href":"http://www.wikipathways.org/instance/WP5097_r119075","target":"_blank"},"children":["WP CCL18 SIGNALING PATHWAY"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["BPTF TARGET GENES"]},{"name":"a","attribs":{"href":"http://www.wikipathways.org/instance/WP2882_r117808","target":"_blank"},"children":["WP NUCLEAR RECEPTORS METAPATHWAY"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["E2F5 TARGET GENES"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["DLX2 TARGET GENES"]},{"name":"a","attribs":{"href":"\n      http://www.gsea-msigdb.org/gsea/msigdb/collections.jsp","target":"_blank"},"children":["HALLMARK UV RESPONSE DN"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["FOXJ2 TARGET GENES"]},{"name":"a","attribs":{"href":"http://www.wikipathways.org/instance/WP98_r117172","target":"_blank"},"children":["WP PROSTAGLANDIN SYNTHESIS AND REGULATION"]}],"html":true,"maxWidth":150},{"id":"gs_description","name":"Description","type":"character","cell":["Genes defining early response to estrogen.","Genes containing one or more binding sites for UniProt:Q4ZG55 (GREB1) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 2...","Genes containing one or more binding sites for UniProt:Q5HY98 (ZNF766) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version ...","Genes defining late response to estrogen.","Genes containing one or more binding sites for UniProt:O14709 (ZNF197) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version ...","Fatty acid metabolism","Genes up-regulated in response to low oxygen levels (hypoxia).","Genes containing one or more binding sites for UniProt:Q9Y543 (HES2) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20...","Genes containing one or more binding sites for UniProt:Q9Y473 (ZNF175) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version ...","CCL18 signaling pathway","Genes containing one or more binding sites for UniProt:Q12830 (BPTF) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20...","Nuclear receptors meta-pathway","Genes containing one or more binding sites for UniProt:Q15329 (E2F5) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20...","Genes containing one or more binding sites for UniProt:Q07687 (DLX2) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20...","Genes down-regulated in response to ultraviolet (UV) radiation.","Genes containing one or more binding sites for UniProt:Q9P0K8 (FOXJ2) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 2...","Prostaglandin synthesis and regulation"]},{"id":"pval","name":"P-Value","type":"numeric","cell":["4.46e-19","3.96e-15","2.96e-11","8.17e-10","9.29e-10","2.08e-09","1.72e-08","4.82e-07","1.87e-05","2.60e-05","3.91e-05","4.43e-05","4.65e-05","2.36e-04","2.38e-04","2.44e-04","2.62e-04"],"maxWidth":80},{"id":"adj_p","name":"p<sub>adj<\/sub>","type":"numeric","cell":["1.18e-15","5.25e-12","2.61e-08","4.93e-07","4.93e-07","9.21e-07","6.50e-06","1.60e-04","0.006","0.007","0.009","0.009","0.009","0.040","0.040","0.040","0.041"],"html":true,"maxWidth":80},{"id":"numDEInCat","name":"Mapped Genes","type":"numeric","cell":["163","497","620","136","346","99","119","626","406","26","375","136","639","147","100","291","21"],"maxWidth":80},{"id":"numInCat","name":"Genes in Gene-Set","type":"numeric","cell":["180","663","881","167","471","116","147","904","593","27","538","184","971","194","122","426","22"],"maxWidth":80},{"id":"gene_name","name":"Gene Names","type":"character","cell":[{"name":"span","attribs":{"title":"ABAT; ABCA3; ABHD2; ABLIM1; ADCY1; ADCY9; ADD3; AFF1; AKAP1; AMFR; ANXA9; AQP3; AR; AREG; ARL3; ASB13; B4GALT1; BAG1; BCL11B; BCL2; BHLHE40; BLVRB; CA12; CBFA2T3; CCN5; CD44; CELSR1; CELSR2; CHPT1; DEPTOR; DHCR7; DHRS2; DHRS3; DLC1; DYNLT3; ELF3; ELOVL2; ELOVL5; ENDOD1; ESRP2; FAM102A; FARP1; FASN; FDFT1; FHL2; FKBP5; FLNB; FOS; FOXC1; FRK; GAB2; GFRA1; GLA; GREB1; HES1; HR; HSPB8; IGF1R; IGFBP4; IL17RB; IL6ST; INHBB; INPP5F; ISG20L2; ITPK1; JAK2; KAZN; KCNK15; KCNK5; KDM4B; KLF10; KLF4; KRT15; KRT18; KRT19; KRT8; LAD1; LRIG1; MAPT; MAST4; MED13L; MICB; MLPH; MREG; MUC1; MYB; MYBL1; MYC; MYOF; NADSYN1; NBL1; NCOR2; NRIP1; OPN3; OVOL2; P2RY2; PAPSS2; PDZK1; PEX11A; PGR; PLAAT3; PMAIP1; PODXL; PPIF; PTGES; RAB17; RAB31; RAPGEFL1; RARA; RASGRP1; RBBP8; REEP1; RETREG1; RHOBTB3; RHOD; RPS6KA2; RRP12; SCARB1; SCNN1A; SEC14L2; SFN; SH3BP5; SIAH2; SLC16A1; SLC19A2; SLC1A1; SLC1A4; SLC22A5; SLC24A3; SLC26A2; SLC27A2; SLC2A1; SLC39A6; SLC7A2; SLC7A5; SLC9A3R1; SNX24; SULT2B1; SVIL; SYBU; SYNGR1; SYT12; TBC1D30; TFAP2C; TFF1; TFF3; TGIF2; THSD4; TIAM1; TIPARP; TJP3; TMEM164; TOB1; TPBG; TPD52L1; TSKU; TTC39A; TUBB2B; UGCG; UNC119; WFS1; WWC1; XBP1"},"children":["ABAT; ABCA3; ABHD2; ABLIM1; ADCY1; ADCY9; ADD3; AFF1; AKAP1; AMFR; ANXA9; AQP3; AR; AREG; ARL3; ASB13; B4GALT1; BAG1; BCL11B; BCL2; BHLHE40; BLVRB;..."]},{"name":"span","attribs":{"title":"ABCA3; ABCC11; ABHD17C; ABHD2; ADAP1; ADGRG1; ADIRF-AS1; AFAP1; AFF1; AGAP2; AHCYL2; AKAP1; AKAP1-DT; AKAP8L; AKAP9; ALKBH7; AMOTL2; AMZ1; ANAPC2; ANKFY1; ANKRD11; ANKRD22; ANP32E; ANXA9; AP5B1; ARHGEF1; ARID1B; ARNTL; ARRDC1; ARRDC4; ASCC2; ASCL2; ATP2A2; ATP5MC1; ATP6V0A4; ATP8B1; ATXN7L1; B3GALNT1; BAIAP3; BATF; BBOF1; BCAR1; BCAR3; BCAS3; BCL3; BCL9L; BICDL1; BLVRB; C1QTNF6; C1orf21; C5AR2; C9orf152; CA12; CAB39L; CAMTA1; CARD10; CASP7; CCDC159; CCDC40; CCDC88C; CCN5; CCT7; CD276; CD59; CDC34; CDS1; CELSR1; CEP89; CFAP45; CHD3; CHD6; CHD8; CHPT1; CHST12; CIC; CITED2; CLN8; CMTR2; CNOT3; COMT; COMTD1; COQ8A; CORO2A; CRACR2A; CREBRF; CRKL; CRNDE; CRPPA; CSNK1G2; CTSB; CYB561; CYLD; CYP1B1; CYP1B1-AS1; DAAM1; DDA1; DDX47; DENND2D; DGAT2; DLC1; DLG5; DMXL1; DNAH11; DNASE1; DNMBP; DNPEP; DOCK4; DOK7; DOP1B; DRAIC; DSCAM-AS1; DYNLT3; ECH1; EHD1; EHF; EHMT1; ELAPOR1; EML6; ENC1; ENSA; ENSG00000224934; ENSG00000267288; ENSG00000268858; ENSG00000275765; EPB41L1; EPG5; EPHA1; EPHB4; EPN2; EPN3; EPS8; ERBB2; ERCC1; ERGIC1; ESRP2; ETNK2; EVI5L; EVL; EXOSC10; FAAP24; FAF1; FAM107B; FAM117A; FAM126A; FAM174B; FAM234B; FAM86EP; FBRS; FBXO38; FHDC1; FMN1; FNIP2; FOXN3; FREM2; FRMD4B; FRMD6; FRYL; G3BP1; GALNT10; GATAD2A; GATAD2B; GCLM; GCOM1; GIGYF2; GLRX5; GOLPH3L; GOT1; GPNMB; GPR160; GPRC5A; GRAMD1A; GREB1; GRHL1; GRHL2; GSPT1; GTPBP3; HEBP2; HERC1; HK1; HMGCS1; HNRNPC; HNRNPL; HOMEZ; HSH2D; HSPB8; ID1; IDE; IFITM10; IFRD1; IFT27; IGF1R; IGSF3; IGSF9; IKBKG; IKZF2; INPP4B; IQCD; IQSEC1; ISG20; JUND; JUP; KAT7; KATNB1; KCNK1; KCTD11; KDM4B; KDM5A; KHSRP; KIAA0100; KIAA0895; KIAA1217; KIF21A; KLF6; KLHL25; KRT7; KRT8; KRT80; LAD1; LAMB1; LCOR; LDLRAD4; LENG1; LFNG; LIMA1; LINC00511; LINC00963; LINC01588; LLGL2; LMCD1; LNC-LBCS; LPP; LRIG2; LRIG3; LRRFIP1; MACC1; MAF1; MANEAL; MAP3K3; MAP4K3-DT; MAPK10; MAPKAP1; MCCC2; MCF2L; MCRIP2; MEF2A; MGRN1; MICOS10; MIDEAS; MIDN; MIOS; MIR9-3HG; MIR99AHG; MIX23; MLLT3; MPHOSPH10; MSMO1; MTA1; MTMR12; MTMR4; MTUS1; MUC5B; NAALADL2; NADK; NAGK; NAIF1; NCAM2; NCOA1; NCOA4; NDRG1; NDUFAF8; NFATC2; NFKBIA; NFKBIZ; NFYB; NINL; NIPSNAP1; NIPSNAP2; NLK; NMT1; NOP14-AS1; NR1D1; NRP1; NT5DC3; NUDT16; NUMA1; OSBPL10; OSBPL2; OSBPL8; OVOL2; OXR1; P4HB; PABPN1; PAIP2B; PAK4; PAM; PARD3; PBX1; PCBP1-AS1; PCDH1; PCTP; PDGFB; PDGFD; PDZK1; PGP; PHTF2; PIK3C2G; PKIB; PKP4-AS1; PLA2G6; PLEKHF2; PNKD; PNPLA7; POLR3K; POU2F3; PPARD; PPM1H; PPM1L; PRADC1; PREX1; PRKCD; PRKCH; PROSER1; PROSER2; PROSER3; PRR15L; PRSS22; PRSS27; PSCA; PSMG3; PTPRH; PUM2; PWWP2A; PXDN; PXMP4; PYM1; PYROXD2; RAB26; RAB5B; RABL3; RAD51B; RANBP2; RAPGEFL1; RARA; RBBP8; RBBP8NL; RBL2; RBM20; RBM47; RC3H1; RECQL5; RFFL; RFX1; RGMB; RGMB-AS1; RGS3; RHOBTB3; RHOC; RNF19A; RNF207; RNF224; RPL27A; RPS6KA2; RPS6KL1; RTN4; S100A10; S100A16; S100A4; SAMD8; SCAMP4; SCGB1D2; SCIN; SCNN1A; SCRIB; SEC14L2; SEPTIN9; SERPINA1; SGMS2; SHARPIN; SIDT2; SKA2; SKIL; SLC16A3; SLC1A5; SLC25A25; SLC25A36; SLC27A2; SLC29A1; SLC35E1; SLC38A1; SLC38A2; SLC39A1; SLC44A2; SLC45A4; SLC6A6; SLC7A2; SLC9A8; SMAD3; SMARCD2; SNX9; SPECC1L; SPNS2; SPRED2; SPRYD3; SPTBN4; SPTLC2; SPTSSB; SRI; SRPK1; SRRM2-AS1; SSNA1; ST6GAL1; ST6GALNAC4; STARD10; STAT3; SUCLA2; SULT1A1; SYBU; SYT8; TACC2; TBC1D16; TBC1D8; TBCD; TBX2; TEDC1; TEPSIN; TFAP2C; TFF1; TGFBI; THADA; THSD4; TIPARP; TJP1; TLE1; TMBIM1; TMC1; TMCO3; TMED3; TMEM120A; TMEM160; TMEM184B; TMEM229B; TMEM241; TMEM60; TMEM94; TMPRSS4; TMTC2; TNK2; TNS1; TOM1L2; TOP2A; TPD52L1; TPD52L2; TPRN; TRAF7; TRAPPC9; TRIB1; TRIM29; TSPAN9; TTC39A-AS1; TTLL5; TUBA1A; TUBA1C; TUBB4B; UBE2C; UNC13D; UNKL; UPF1; URB1; USF3; UTRN; VAMP5; VAV3; VEPH1; VIRMA; VMP1; VPS26C; WDR90; WIPI1; WWC1; WWP1; XRCC3; YIF1A; ZDHHC1; ZFYVE21; ZFYVE28; ZKSCAN1; ZMYM2; ZMYND8; ZNF444; ZNF552; ZNF669; ZNF687; ZNF689; ZNF750; ZNF823; ZNRF1; ZSWIM1"},"children":["ABCA3; ABCC11; ABHD17C; ABHD2; ADAP1; ADGRG1; ADIRF-AS1; AFAP1; AFF1; AGAP2; AHCYL2; AKAP1; AKAP1-DT; AKAP8L; AKAP9; ALKBH7; AMOTL2; AMZ1; ANAPC2; ..."]},{"name":"span","attribs":{"title":"AAMP; ACP2; ACTB; ACTG1; ACTR1A; ACTR3; ACTR3B; ADAM9; ADAMTSL4; AHCTF1; AHCYL2; AHSA2P; AKAP8; AKIRIN2; ALDH5A1; ALMS1; AMMECR1; AMN1; AMOTL1; ANKMY1; ANKS6; ANTXR2; ANXA2; AP1S2; AP2M1; AP2S1; APOO; AREG; ARFRP1; ARHGAP5; ARHGAP5-AS1; ARHGEF12; ARID4B; ARL4A; ARNTL; ARPC1A; ARV1; ASAP3; ASNSD1; ASXL2; ATG10; ATG4B; ATP5PF; ATXN7; AUP1; B4GALT2; B4GAT1; BABAM1; BANP; BATF; BCKDK; BICD2; BIRC5; BMERB1; BOD1; BORCS7; BTN3A2; BTRC; BUB3; C11orf58; C15orf61; C17orf58; C19orf44; C1orf159; CABIN1; CACNA1D; CALCOCO1; CAMP; CAP1; CAPG; CAST; CBFA2T3; CBX7; CCAR1; CCDC12; CCDC137; CCDC71; CCDC90B; CCP110; CDC73; CDCA3; CDH23; CDK5; CDK8; CDYL; CENPT; CEP290; CFL1; CHCHD2; CHD7; CHFR; CHIC2; CHMP4B; CIC; CITED2; CLEC16A; CLPB; CLSTN1; CLTC; CLUAP1; CNPY2; COMMD3; COPS7B; CORO1A; COX7A2L; COX7C; CPE; CRADD; CRAMP1; CRTC2; CRYM; CSNK2B; CSRNP2; CSRP1; CTDP1; CTSB; CYB561D2; CYB5D1; DCK; DDX23; DEDD; DENND1A; DFFA; DGAT1; DGAT2; DGCR8; DHRS3; DIP2A; DMD; DNAJA3; DNAJC17; DNALI1; DNM1L; DST; DUS1L; DUS3L; DUSP4; DYNC1I2; DYNC2I2; ECHDC2; EFNA3; EHD1; EHMT2; ELOC; EML2; EML6; ENSG00000233834; ENSG00000255468; ENY2; EPB41L5; EPC1; ERH; ERI3; ERMAP; ERP44; ERVK3-1; EVL; EXOC6; EYA3; FAF2; FAM117A; FAM13A; FAM168B; FAM193A; FAM222A; FASN; FASTKD1; FBXL16; FBXL17; FBXL19; FBXL3; FBXO15; FCSK; FGF13; FHL2; FLYWCH1; FN3K; FOXP1; FTO; FYCO1; G3BP2; GAB2; GALNT7; GAPVD1; GATAD2A; GATB; GDAP1; GDPD5; GFER; GFUS; GGPS1; GLUD1; GNG4; GPANK1; GPAT4; GPBP1; GPBP1L1; GPKOW; GPRC5C; GPT2; GRAMD4; GRK3; GRK4; GSPT1; GTF2I; GTF2IRD1; GTPBP2; GUSB; H4C8; H6PD; HAUS6; HECTD2; HERC1; HGD; HIKESHI; HM13; HMG20A; HSD11B1L; HSPA13; HSPB11; HSPH1; HTRA2; IFRD1; IFRD2; IFT140; IFT20; IFTAP; IGF1R; IKZF2; IL1R1; ILF3; ILF3-DT; IMPACT; ING3; INTS6; INVS; IQCH-AS1; IRF2BP2; KANSL3; KAT7; KBTBD6; KBTBD7; KCNB1; KCTD20; KCTD21; KDM7A; KDM7A-DT; KHDC4; KIF15; KLC1; KLHDC8B; KLHL21; KMT2B; LAMTOR2; LARP1; LARS1; LCORL; LEMD2; LEPROTL1; LIFR; LIG1; LINC-PINT; LINC00504; LINC01521; LLGL2; LMNB2; LNX1; LPXN; LSM10; LTA4H; LUC7L; LYPLA1; LYSMD1; MACC1; MAD2L1BP; MAF1; MALSU1; MAML3; MAN2A2; MANBAL; MAP4K3; MAP4K3-DT; MAT2A; MAX; MDH1; MED15; MFAP3L; MGME1; MIAT; MICOS13; MIDEAS; MIDN; MITF; MKKS; MLLT10; MMACHC; MPI; MPZL3; MR1; MRFAP1; MRGBP; MRPL24; MTIF2; MTSS1; MUS81; MUTYH; MYB; MYCL; MYO5B; MYO9A; NAA38; NAPA; NDUFAB1; NEAT1; NEDD4; NEK6; NF2; NFASC; NFE2L1; NFIX; NFKBIA; NFKBIB; NKAIN1; NOP14; NOP53; NORAD; NOSIP; NPEPPS; NPEPPSP1; NPL; NPRL2; NRDE2; NSMCE1-DT; NTAN1; NUCB1; NUDCD1; NUDT13; NUMA1; NUP205; NUP43; NUTF2; OPLAH; ORMDL3; OTUD7A; PA2G4; PABPN1; PACS1; PACS2; PACSIN2; PAFAH2; PANK4; PAPSS1; PATZ1; PCBP1-AS1; PCGF5; PCSK6; PDCD10; PDE3B; PDE6D; PDS5B; PEAK1; PEMT; PEX14; PEX26; PFDN6; PGBD5; PGM2L1; PGP; PHB; PHF12; PHF13; PISD; PLA2G6; PLCG2; PLEKHF2; PNKD; PNPLA6; PODXL2; POLDIP3; POLR1G; PPP1R12C; PPP1R13L; PPP1R15A; PPRC1; PRKCA; PRMT3; PROSER2; PRPF31; PRPSAP1; PRR5; PRRT2; PTCD3; PTK2; QRICH1; RAB1A; RAB27B; RAB7A; RABGAP1L; RALBP1; RANBP10; RANGAP1; RBBP4; RBM14; RBM25; RCAN1; RCBTB2; RERE; REXO2; RHOBTB3; RIC8B; RIMKLB; RIN3; RMC1; RNF145; RNFT2; RPGRIP1L; RPIA; RPL36; RPP40; RPRD1B; RPS13; RXRA; RYK; S100P; SAP30; SBK1; SCAF1; SCAMP4; SCNM1; SCRIB; SEC11C; SEC24C; SEC61A1; SENP8; SERPINI1; SETD5; SETMAR; SFMBT1; SFSWAP; SFT2D2; SFXN1; SHARPIN; SHLD2; SIPA1L3; SIRT2; SKIL; SLC16A1; SLC25A12; SLC25A6; SLC2A4RG; SLC35B4; SLC38A2; SLC39A11; SLC39A9; SLC44A1; SLC45A4; SLC48A1; SLC4A2; SLC9A3-AS1; SLK; SLMAP; SLX4IP; SMAD3; SMAP1; SMIM13; SMPD3; SNX10; SNX19; SNX5; SPEN; SPRING1; SQOR; SRCIN1; SRGAP3; ST3GAL2; STAG2; STAT5B; STAU1; STAU2; STK25; STK40; STMN3; STMP1; STPG1; STRIP1; STRN3; STXBP5; SULT1A2; SUMO3; SYMPK; SYNGR1; TANC1; TARBP2; TBC1D4; TBC1D8B; TBX6; TCEA2; TCERG1; TENT2; TFAP2A; TFEB; TFPT; THADA; THRAP3; THUMPD2; TIAL1; TIMM21; TJP3; TM7SF3; TM9SF4; TMBIM1; TMCC1; TMCC1-DT; TMCO1; TMCO1-AS1; TMEM11; TMEM115; TMEM160; TMEM181; TMEM245; TMEM69; TMEM70; TMTC1; TMTC3; TNFAIP1; TNS1; TOE1; TP53BP2; TPD52L1; TPGS1; TRAM2-AS1; TRIM29; TRIT1; TRMT12; TRMT44; TRPM7; TSC22D1; TSC22D2; TSSC4; TTC1; TTC13; TTC33; TTLL4; TUFT1; TXNDC15; UBE2E2; UBE2E3; UBE2Q2; UBE3C; UBQLN4; UBR2; UBR3; UBXN2A; UCP2; UNC45A; UPF1; UROS; USF1; USF2; USP31; USP32; USP35; USP48; USP6NL; VAPA; VDAC2; VMP1; VPS35L; VPS50; VRK1; WARS1; WARS2-AS1; WBP1; WBP2; WDPCP; WDR1; WDR24; WDR25; WDR46; WNK1; WSB1; WWP1; XIST; XKR9; XYLT2; YAP1; YEATS2; YIF1B; YIPF6; ZC3H7A; ZCCHC8; ZDHHC18; ZFP91; ZFYVE1; ZFYVE19; ZFYVE28; ZGPAT; ZMYM4; ZMYND8; ZNF181; ZNF207; ZNF217; ZNF221; ZNF223; ZNF225; ZNF32; ZNF331; ZNF337; ZNF37A; ZNF37BP; ZNF436; ZNF444; ZNF529; ZNF586; ZNF605; ZNF644; ZNF674-AS1; ZNF688; ZNF776; ZSCAN22"},"children":["AAMP; ACP2; ACTB; ACTG1; ACTR1A; ACTR3; ACTR3B; ADAM9; ADAMTSL4; AHCTF1; AHCYL2; AHSA2P; AKAP8; AKIRIN2; ALDH5A1; ALMS1; AMMECR1; AMN1; AMOTL1; ANK..."]},{"name":"span","attribs":{"title":"ABCA3; ABHD2; ACOX2; ADD3; AFF1; AGR2; AMFR; ANXA9; AREG; ARL3; BAG1; BATF; BCL2; BLVRB; CA12; CA2; CACNA2D2; CCN5; CD44; CD9; CDC6; CDH1; CELSR2; CHPT1; CHST8; COX6C; CPE; DCXR; DHCR7; DHRS2; DLG5; DNAJC1; DYNLT3; ELOVL5; EMP2; ETFB; FABP5; FAM102A; FARP1; FDFT1; FGFR3; FKBP5; FLNB; FOS; FOXC1; FRK; GALE; GFUS; GINS2; GLA; HMGCS2; HOMER2; HPRT1; HR; HSPB8; IDH2; IGFBP4; IL17RB; IL6ST; IMPA2; ISG20; ITPK1; JAK1; JAK2; KCNK5; KIF20A; KLF4; KRT19; LAMC2; LARGE1; LLGL2; LSR; MAPK13; MAPT; MICB; MOCS2; MYB; MYOF; NBL1; NCOR2; NRIP1; OPN3; OVOL2; PAPSS2; PDCD4; PDZK1; PGR; PLAAT3; PPIF; PRKAR2B; PRLR; PTGES; RAB31; RABEP1; RAPGEFL1; RBBP8; RPS6KA2; S100A9; SCARB1; SCNN1A; SCUBE2; SERPINA1; SFN; SIAH2; SLC16A1; SLC1A4; SLC22A5; SLC24A3; SLC26A2; SLC27A2; SLC29A1; SLC7A5; SLC9A3R1; SNX10; SORD; ST14; ST6GALNAC2; SULT2B1; TFAP2C; TFF1; TFF3; TFPI2; TIAM1; TJP3; TOB1; TOP2A; TPBG; TPD52L1; TRIM29; TSPAN13; UGDH; UNC13B; WFS1; XBP1; XRCC3; ZFP36"},"children":["ABCA3; ABHD2; ACOX2; ADD3; AFF1; AGR2; AMFR; ANXA9; AREG; ARL3; BAG1; BATF; BCL2; BLVRB; CA12; CA2; CACNA2D2; CCN5; CD44; CD9; CDC6; CDH1; CELSR2; ..."]},{"name":"span","attribs":{"title":"AAMP; ABCA5; ABLIM3; ACACA; ACAD10; ACAD8; ACP2; ACSL3; ACTG1; ADAM9; ADCY9; ADNP; AHCYL2; AHI1; AHSA1; AKIRIN2; AMMECR1; ANKS6; ANKZF1; AP2S1; ARAP1; ARHGAP5; ARHGAP5-AS1; ARNTL; ASAP3; ATG4B; ATG9A; AUP1; B4GALT2; BABAM1; BANP; BICDL1; BLOC1S4; BOD1; BSG; BTBD9; BTRC; BUB1B; BZW2; C17orf58; C1orf159; C6orf89; CABIN1; CALCOCO1; CAMKK2; CAMKMT; CAPG; CBFA2T3; CBX7; CCAR1; CCDC137; CCDC18-AS1; CCDC90B; CCP110; CDC40; CDCA3; CDH23; CDK5; CDKN2AIPNL; CEP290; CFL1; CHD7; CHFR; CHMP4B; CLEC16A; CLSTN1; CLTC; CNPY2; COMMD3; CORO1A; COX7A2L; COX7C; CPEB4; CPQ; CRADD; CRCP; CREBL2; CRTC2; CRYBG2; CYB561D2; CYB5R3; DCK; DDX23; DEDD; DENND1A; DENND5B; DFFA; DGCR8; DNAAF5; DR1; DYNC1I2; ECH1; EFNA3; ELOC; ENSG00000233834; ENY2; EPC1; ERMAP; ERVK3-1; EXOC6; FAM117A; FANCC; FASN; FASTKD1; FBXL16; FBXL19; FBXL3; FBXO15; FBXO38; FCHSD2; FCSK; FLYWCH1; FYCO1; GAB2; GFER; GFM1; GFUS; GNG4; GP6-AS1; GPAT4; GPKOW; GRHL1; GRK3; GSPT1; GTF2I; H3-3B; HDAC8; HERC1; HIF1A; HIKESHI; HM13; HTRA2; IFRD2; IFTAP; IKZF2; IL1R1; IMPACT; ING3; IRF2BP2; KAT7; KBTBD6; KBTBD7; KCNC3; KCNK1; KDM4C; KHK; LARP1; LARS1; LEMD2; LEPROTL1; LIG1; LINC01521; LSM10; LYPLA1; LYSMD1; MACC1; MAN2A2; MANEAL; MAPT; MARS1; MAX; MDH1; MITF; MKKS; MLLT10; MMACHC; MRFAP1; MRPL24; MRPS22; MUS81; MYB; MYCL; MYO1D; NDRG3; NEK6; NFASC; NFIX; NOP53; NOSIP; NPEPPS; NPEPPSP1; NPRL2; NRDE2; NSD2; NUDCD1; NUDT13; OPLAH; OTUD7A; PA2G4; PACS2; PAFAH2; PALS2; PANK4; PATZ1; PCBP1-AS1; PCSK6; PGBD5; PGP; PHB; PHF12; PHF13; PHLPP1; PISD; PLEKHA1; PLEKHF2; PNKD; PODXL2; PPM1E; PPOX; PPP1R12C; PPP1R15A; PPP2R2C; PREPL; PRKCA; PRMT3; PROSER2; PRPF31; PRPSAP1; PRSS27; PTDSS2; PTK2; PTPRN2; PUM3; RAB7A; RABAC1; RABGAP1L; RANBP10; RANGAP1; RBL1; RBM6; RHOBTB3; RIN3; RNASEH2C; RNF144A; RPP40; RXRA; RYBP; SAP30; SBK1; SCAF1; SCAMP4; SCAMP5; SCNM1; SEC24C; SEC61A1; SEL1L; SEM1; SEPHS2; SETMAR; SFI1; SFXN1; SGPL1; SIAH1; SIPA1L3; SKIL; SLC4A2; SLC9A3-AS1; SLF2; SLK; SLMAP; SLX4IP; SMARCAL1; SMPD3; SNAPC3; SPTBN4; SRCIN1; SRRM3; ST3GAL2; ST7; STARD10; STAT3; STAT5B; STAU1; STAU2; STK25; STRIP1; SUMO3; SYNGR1; TADA2A; TBX6; TCEA2; TCERG1; TCP11L1; TFPT; THADA; THUMPD2; THYN1; TIMM21; TM9SF4; TMCC1; TMCC1-DT; TMED10; TMEM11; TMEM115; TMEM160; TMEM181; TMEM70; TMTC3; TPGS1; TRABD; TRIM33; TRMT12; TSC22D1; TXNDC11; TXNDC15; UBE2E2; UBE2Q2; UBE3C; UBXN2A; UCKL1; UNK; UROS; USE1; USF2; USP31; USP6NL; VCPIP1; VIPAS39; WASF1; WDPCP; WDR82; WNK1; WSB1; WWP1; XIST; XKR9; XYLT2; YEATS2; ZBTB38; ZBTB46; ZC3H7A; ZFYVE1; ZFYVE28; ZMAT1; ZNF221; ZNF225; ZNF236; ZNF280D; ZNF32; ZNF331; ZNF37BP; ZNF417; ZNF444; ZNF605; ZNF688; ZNF776"},"children":["AAMP; ABCA5; ABLIM3; ACACA; ACAD10; ACAD8; ACP2; ACSL3; ACTG1; ADAM9; ADCY9; ADNP; AHCYL2; AHI1; AHSA1; AKIRIN2; AMMECR1; ANKS6; ANKZF1; AP2S1; ARA..."]},{"name":"span","attribs":{"title":"ABCC1; ACAA1; ACACA; ACACB; ACAD10; ACADM; ACBD4; ACBD6; ACBD7; ACLY; ACOT1; ACOT13; ACOT2; ACOT4; ACOT7; ACOT8; ACOX1; ACOX2; ACOX3; ACSF2; ACSL1; ACSL3; ACSL4; AKR1C3; ALOX5; AMACR; CBR4; CPT1A; CPT2; CRAT; CROT; CYP1B1; CYP2J2; CYP2U1; CYP4B1; DBI; DECR1; DECR2; ECI1; ECI2; EHHADH; ELOVL2; ELOVL4; ELOVL5; ELOVL6; ELOVL7; FAAH; FAAH2; FADS2; FASN; GPX4; HACD1; HACD2; HACD3; HACD4; HACL1; HADH; HADHA; HADHB; HPGD; HSD17B12; HSD17B4; LTA4H; MCEE; MECR; MID1IP1; MLYCD; MMAA; MMUT; MORC2; NDUFAB1; NUDT19; PCCA; PCCB; PCTP; PECR; PLA2G4A; PON2; PPARD; PPT1; PRKAA2; PRKAB2; PRKAG2; PRXL2B; PTGES; PTGES2; PTGR1; PTGR2; RXRA; SCD; SCD5; SCP2; SLC22A5; SLC25A1; SLC25A17; SLC25A20; SLC27A2; SLC27A3; THEM4"},"children":["ABCC1; ACAA1; ACACA; ACACB; ACAD10; ACADM; ACBD4; ACBD6; ACBD7; ACLY; ACOT1; ACOT13; ACOT2; ACOT4; ACOT7; ACOT8; ACOX1; ACOX2; ACOX3; ACSF2; ACSL1;..."]},{"name":"span","attribs":{"title":"ACKR3; AK4; ALDOA; AMPD3; ANGPTL4; ANKZF1; ANXA2; ATP7A; B4GALNT2; BCL2; BHLHE40; BNIP3L; BTG1; CA12; CCN1; CCN5; CCNG2; CDKN1B; CHST3; CITED2; COL5A1; CXCR4; DDIT4; DTNA; DUSP1; EDN2; EFNA1; EFNA3; EGFR; ENO1; ERO1A; ERRFI1; EXT1; F3; FAM162A; FBP1; FOS; FOSL2; FOXO3; GAA; GALK1; GAPDH; GBE1; GCNT2; GLRX; GPI; GRHPR; GYS1; HK1; HK2; HMOX1; HSPA5; IDS; IER3; IGFBP3; ILVBL; IRS2; ISG20; JMJD6; JUN; KDM3A; KLF6; KLF7; KLHL24; LARGE1; LDHA; MAFF; MAP3K1; MT2A; MXI1; NAGK; NDRG1; NDST1; NEDD4L; NOCT; P4HA1; P4HA2; PAM; PDGFB; PDK1; PFKFB3; PFKP; PGK1; PGM2; PIM1; PLAUR; PNRC1; PPP1R15A; PRKCA; RBPJ; RORA; RRAGD; S100A4; SAP30; SCARB1; SDC2; SDC3; SDC4; SELENBP1; SIAH2; SLC25A1; SLC2A1; SLC6A6; STC1; SULT2B1; TES; TGFBI; TIPARP; TPBG; TPD52; TPI1; TPST2; UGP2; VEGFA; VLDLR; WSB1; XPNPEP1; ZFP36; ZNF292"},"children":["ACKR3; AK4; ALDOA; AMPD3; ANGPTL4; ANKZF1; ANXA2; ATP7A; B4GALNT2; BCL2; BHLHE40; BNIP3L; BTG1; CA12; CCN1; CCN5; CCNG2; CDKN1B; CHST3; CITED2; COL..."]},{"name":"span","attribs":{"title":"ABCA5; ABCC3; ABCG1; ABHD11; ABHD12; ABLIM1; ABLIM2; ACOT13; ACOT7; ACOX2; ACTN1; ADAM9; ADAP1; ADAR; ADD3; ADIPOR2; AFDN; AGK; AGPS; AGTPBP1; AHCYL1; AJUBA; AKAP6; AKR1C3; ALAS1; ALG10; ALG13; ALPK1; AMBRA1; ANAPC5; ANK3; ANKEF1; ANKRD11; ANKRD40; ANP32B; ANXA1; ANXA11; ANXA2; AOPEP; AP1G1; APBA3; APOLD1; AQP3; AREG; ARFIP1; ARHGAP21; ARHGAP24; ARHGAP26; ARID5B; ASCC2; ASNSD1; ATAD1; ATF6; ATG10; ATP2C1; ATP5MC2; ATP9B; ATXN10; ATXN7L1; AVL9; B3GNTL1; B4GAT1; BAZ2B; BCL10; BCL6; BCOR; BCR; BDP1; BHLHE40; BMERB1; BMF; BMP1; BMPR1B; BOLA1; BPHL; BRD4; BUD31; BZW2; C11orf68; C1QTNF6; C1orf21; C1orf226; C2orf76; CADM2; CALCOCO2; CBX4; CCDC159; CCDC171; CCDC57; CCDC9B; CCND3; CCNG2; CD109; CD59; CDC40; CDC42SE1; CDC7; CDCA2; CDK5RAP1; CDK5RAP2; CECR2; CELSR1; CEP112; CFAP418; CFDP1; CHCHD5; CHD1; CHD2; CHEK2; CHP1; CIART; CIPC; CLASP1; CLGN; CLK4; COL12A1; CPD; CRKL; CRYBG1; CSAD; CSGALNACT1; CSTA; CTR9; CTSB; CTSC; CTSD; CUEDC1; CUL2; CUX1; CXADR; CYB5A; CYP4B1; CYTH1; DAAM1; DAGLB; DBI; DCBLD2; DDB2; DDIAS; DDIT4; DDX10; DDX47; DDX60L; DENND10; DENND2B; DIO2; DLC1; DLEU2; DLG5; DLGAP1-AS1; DMAP1; DNAH11; DPH6; DPY19L1; DRAIC; DRAM1; DRAP1; DTNA; DTWD1; DUSP1; E2F2; E4F1; ECHDC3; EDEM1; EFCAB2; EFNA1; EFNA5; EHD1; EHF; EHMT1; EID1; EIF2S3; EIF4E; EIF5A2; ELOVL6; EML2; ENAH; ENO1; ENSG00000215154; ENSG00000255468; ENSG00000268858; EPB41; EPHB4; EPS8; ERCC4; ERGIC1; ERI1; ETNK2; ETV6; EVL; EXD3; EXOSC7; EZR; FAM107B; FAM13A; FAM167A; FAM219B; FBXL18; FBXL5; FBXO38; FERMT2; FGD4; FGFR1; FLNB; FLOT1; FMN1; FNDC3A; FOS; FOSL2; FOXN2; FTL; GALNT10; GALNT18; GARS1; GARS1-DT; GBE1; GCC2; GCLC; GMCL1; GNAL; GON4L; GPAT3; GPD1L; GPRC5A; GREB1; GTF2F2; GTF3C3; GTPBP2; GUSBP1; GZF1; H2BC21; H4C3; HBP1; HBS1L; HDAC11; HDAC5; HERPUD2; HERPUD2-AS1; HES4; HGD; HIVEP2; HK1; HMG20A; HMGCL; HMOX1; HNMT; HNRNPF; HOXC-AS2; HSPD1; HSPE1; HTR1D; IFNGR1; IFT122; IGF1R; IGF2R; IL1R1; IL1RAP; IL6R; INAVA; INTU; IQGAP2; IRF2; IRS2; ISG15; ISG20; ITGB1; ITGB3BP; ITGB5; ITPRIPL2; JDP2; JPX; JRK; KAT2B; KATNBL1; KCNK1; KDM3A; KDSR; KIAA0232; KIAA0319L; KIFC3; KLF13; KLF5; KLF7; KLHDC4; KLHL24; KMT2A; KRT7; KRT8; LDLRAD4; LGALSL; LIMCH1; LINC-PINT; LINC00511; LINC00886; LINC00963; LINC02615; LMO4; LMO7; LMX1B; LPIN2; LRBA; LRIG3; LRRC27; LRRFIP2; LTBP1; LTBR; LZTFL1; MAFK; MAP2K3; MAP3K20; MAP3K8; MAPK15; MAPK3; MARCHF3; MARCHF7; MBD4; MBNL1; MBP; MCCC1; MCCC2; MCL1; MCTP1; MDH1; MDH2; ME2; MECP2; MED23; MED28; MEIS1; METTL15; MFSD4B; MFSD6; MIB1; MID1IP1; MIDEAS; MIDN; MIOS-DT; MIR99AHG; MITF; MIX23; MLPH; MPP3; MRC2; MRPL46; MRPL54; MRPS15; MRPS23; MRPS35; MRPS5; MT2A; MTHFD1L; MTHFD2L; MTMR12; MTMR9; MUC1; MYO10; NBL1; NCOA7; NDRG1; NDUFA12; NDUFB11; NDUFS4; NEAT1; NEDD4; NEDD4L; NFE2L2; NFIA; NFIB; NFIC; NFKBIA; NFKBIZ; NFX1; NGLY1; NHEJ1; NIBAN1; NIBAN2; NIPAL2; NISCH; NKILA; NOL11; NR4A1; NRG1; NRP1; NTN4; NUF2; NUMB; NXN; OIP5; OPN3; OSBPL1A; OSMR; P4HA2; P4HB; PAFAH2; PALMD; PARD3; PARP8; PCBP1-AS1; PCID2; PDAP1; PDCD6; PDE4D; PDGFC; PDK1; PDLIM5; PEAK1; PECR; PER1; PER2; PHACTR4; PHF20; PIGT; PIK3C3; PKP4-AS1; PLCE1; PLD1; PLEKHG2; PLOD2; PMEPA1; PNRC1; POC1A; POLG; PPARG; PRCP; PRDM15; PRDM6; PRKCH; PRMT5; PSMG3; PTCH1; PTPN12; PTPRG; PTPRM; PVT1; PYCR1; RAB31; RAB3D; RAB5IF; RABEPK; RBM39; RCAN1; RCL1; RFESD; RFFL; RHOBTB2; RHOBTB3; RHOT2; RMDN3; RNF13; RNF145; RPH3AL; RPL24; RPL34; RPL6; RPS6KB1; RPS7; RRBP1; RSU1; RUSC2; RUVBL1; RXRA; S100A10; S100A6; S100A8; S100A9; S100P; S100PBP; SBNO2; SCN8A; SCNN1A; SCUBE2; SEC62; SEH1L; SEL1L3; SEMA4C; SEPTIN2; SEPTIN9; SETD5; SGK3; SGPP1; SH3BP4; SH3TC1; SHANK2; SIAH1; SKIL; SLC16A1; SLC1A5; SLC25A28; SLC2A4RG; SLC33A1; SLC38A1; SLC38A2; SLC44A1; SLC45A4; SLC7A6; SLC9A1; SLPI; SMAD7; SMARCA2; SMARCD2; SMIM14; SMIM20; SNHG16; SNX18; SNX27; SNX8; SP100; SPATS2L; SPIDR; SPIN1; SPPL2A; SPRED2; SPRY4; SQOR; SRBD1; SRCIN1; SRI; SRRM3; SSBP2; STAT1; STAT3; STAU1; STC1; STK40; STX1A; SUB1; SUMF2; SUSD1; SYBU; SYNC; SYNPO; SYT7; TAGLN2; TBC1D16; TBPL1; TCF7L2; TDP2; TEAD1; TET2; TEX2; TFE3; TFEB; TFRC; TGFBI; TGFBR1; TGFBR2; TGIF1; THADA; THRA; TIGD2; TIMM13; TM4SF1; TMBIM1; TMC1; TMED10; TMED2; TMEM143; TMEM205; TMEM260; TMEM33; TMEM40; TMEM9; TMSB4X; TNFRSF1A; TNIP1; TNPO1; TNS1; TPD52L2; TPM4; TPRA1; TPT1; TPT1-AS1; TRAPPC3; TRAPPC5; TRIB1; TRIM29; TRPC4AP; TRPM7; TSEN15; TSFM; TTC7A; TTC7B; TUBA1C; TUT1; TXNDC11; TXNRD1; UBC; UBE2B; USP3; USP54; USP9X; UTRN; VAV3; VMP1; VWA8; WASF1; WASL; WDPCP; WDR7; WIZ; WWTR1; XDH; YAP1; YARS1; ZBTB20; ZBTB38; ZBTB44; ZBTB7A; ZC3H12A; ZC3H7A; ZDHHC18; ZDHHC3; ZFAND5; ZFP36L2; ZFR; ZMAT1; ZMIZ1; ZNF14; ZNF217; ZNF221; ZNF354B; ZNF529; ZNF580; ZNF581; ZNF594; ZNF706"},"children":["ABCA5; ABCC3; ABCG1; ABHD11; ABHD12; ABLIM1; ABLIM2; ACOT13; ACOT7; ACOX2; ACTN1; ADAM9; ADAP1; ADAR; ADD3; ADIPOR2; AFDN; AGK; AGPS; AGTPBP1; AHCY..."]},{"name":"span","attribs":{"title":"AAMP; ABCA3; ABCC3; ACP2; ACTB; ACTG1; ADAM15; ADAM9; ADD1; ADNP; AGPS; AHCYL2; AHI1; AHNAK; AIMP2; ALKBH7; ALKBH8; ALPK1; AMBRA1; AMMECR1; ANKRD17; ANKRD42; ANKS6; ANKZF1; ANO10; ANP32A; ANXA11; AP2S1; APH1B; APP; ARAP1; ARHGAP24; ARHGDIA; ARHGEF12; ARL4A; ARNTL; ARPC2; ARPC5; ASAP3; ATG9A; ATP5F1D; ATP5MC1; BABAM1; BANP; BCL7C; BIRC5; BLOC1S4; BOD1; BRD2; C17orf58; C19orf44; C1orf159; C2CD3; C2CD5; C6orf62; CABIN1; CACTIN; CALCOCO1; CAMKK2; CAMTA1; CAND1; CAPN15; CBX4; CCAR1; CCDC137; CCDC159; CCDC25; CCNL1; CCT6B; CDC40; CDCA3; CDH23; CDK5RAP2; CDYL; CEP290; CEPT1; CFL1; CHCHD2; CHD2; CHMP3; CHMP4B; CLASP1; CLEC16A; CLPB; CLSTN1; CLTC; CNOT10; COG3; COX7C; CPEB2; CRTC2; CRYBG2; CUL5; CWC22; DDX23; DDX3X; DFFA; DGCR8; DHX58; DNAAF5; DRAM2; DYNC1I2; EBAG9; EEF2K; EIF3B; EPC1; ERI1; ERP44; EZR; FAM168B; FASN; FBXO15; FBXO38; FBXW11; FCHSD2; FEM1A; FGGY; FLYWCH1; GAB2; GALE; GAPVD1; GFER; GFUS; GLB1; GNG4; GON4L; GPATCH8; GPBP1L1; GRB10; GSPT1; GTF2IRD1; H4C8; HERC1; HES4; HIKESHI; HM13; HMBOX1; HMGB1; HMOX2; IFRD2; IFT20; IFTAP; IGF1R; IKZF2; IL1R1; IMPACT; INTS9; INVS; IRF2BP2; ISG15; ISG20; ITGAL; JMJD1C; JOSD2; KBTBD6; KBTBD7; KCNK1; KCTD20; KDM7A; KDM7A-DT; KHSRP; KIAA0100; KIF15; KMT5A; KNTC1; LARS1; LCOR; LEMD2; LEPROTL1; LINC00235; LINC01521; LRBA; LRP6; LSM10; LUC7L; LYPLA1; LYSMD1; MACROH2A1; MAD2L1BP; MAN2A2; MANEAL; MAPK13; MARS1; MATN1-AS1; MAU2; MAZ; MDH1; MFAP3L; MGRN1; MKKS; MLLT10; MRFAP1L1; MRPL52; MTMR4; MTMR9; MTR; MUC20-OT1; MUS81; MXD1; MYCL; MYL12B; MYO5B; NADK2; NCOA3; NDC1; NFE2L1; NFE2L2; NFIX; NMRAL1; NOSIP; NPEPPS; NPEPPSP1; NPL; NR4A1; NTPCR; NUMA1; NUP210; OIP5-AS1; OPLAH; OTUD7A; P4HB; PAFAH2; PAK1; PANK4; PATZ1; PCBP1-AS1; PCSK6; PDE7A; PEDS1; PEMT; PGP; PHB; PHF12; PHF8; PITHD1; PLAUR; PLEKHF2; PNKD; PNPLA6; POLDIP3; PPA2; PPM1E; PPP1R12C; PPP1R15A; PPP1R3E; PPRC1; PRDM5; PRKAB2; PRKCE; PRKCH; PROSER2; PRPSAP1; PRR5; PSEN1; PTDSS2; PTK2; PUM3; PVT1; RAB1A; RAB30; RAB30-DT; RAB4A; RAB5IF; RAB7A; RABGAP1L; RAD51C; RANBP10; RANGAP1; RBBP4; REXO2; RFFL; RFX1; RIC8B; RIN3; RIPOR3; RPIA; RPP14; RPRD1B; SAP30; SCAF1; SCAMP5; SCNM1; SCOC; SCRIB; SEC14L1; SEC24C; SEMA4B; SFSWAP; SFT2D2; SH3TC1; SIAH1; SKA2; SKIL; SLC25A12; SLC25A6; SLC48A1; SLC4A11; SLC4A2; SLC5A6; SLC7A5; SLC9A1; SLK; SLMAP; SLX4IP; SMIM13; SNF8; SNX10; SNX19; SOSTDC1; SPTBN4; SRP19; SRSF1; ST3GAL2; ST6GALNAC4; STARD10; STAT5B; STAU1; STAU2; STMN1; STRIP1; STX4; SUGP1; SULT2B1; SUMO3; SYNPO; TANC1; TARBP2; TASOR2; TBX6; TCEA2; TCERG1; TEAD1; TEDC1; TELO2; TERF2; TFEB; THADA; TIMM21; TIPRL; TKT; TM9SF4; TMEM11; TMEM115; TMEM209; TMEM245; TMEM69; TMPPE; TMTC3; TNFAIP1; TNKS; TP53RK; TPGS1; TPM4; TRIM29; TRMT12; TSC22D1; TSSC4; TTC1; TUBB2A; TUFT1; TXNDC15; UBALD2; UBC; UBE2E2; UBE2E3; UBE3C; UCKL1; UFL1; USF1; USP10; USP31; USP33; USP9X; VDAC2; VPS35L; VPS37B; VPS41; VRK1; WASF1; WBP1; WBP2; WDPCP; WDR1; WDR46; WSB1; WWP1; WWTR1; XIST; XKR9; XYLT2; YEATS2; YIF1B; ZBED9; ZBTB38; ZC3H7A; ZCCHC8; ZFYVE1; ZFYVE28; ZMYND8; ZNF217; ZNF225; ZNF280C; ZNF280D; ZNF337; ZNF444; ZYX"},"children":["AAMP; ABCA3; ABCC3; ACP2; ACTB; ACTG1; ADAM15; ADAM9; ADD1; ADNP; AGPS; AHCYL2; AHI1; AHNAK; AIMP2; ALKBH7; ALKBH8; ALPK1; AMBRA1; AMMECR1; ANKRD17..."]},{"name":"span","attribs":{"title":"AK4; BCYRN1; CD44; CDH1; EP300; FN1; FZD4; HIF1A; HOTAIR; KIF5B; LDHA; LINC01089; MIAT; MTDH; MTERF4; MYC; PTEN; PTK2B; RAD51-AS1; SLC2A1; STAT1; TEAD4; TFAP2A; TWIST1; VEGFC; VMP1"},"children":["AK4; BCYRN1; CD44; CDH1; EP300; FN1; FZD4; HIF1A; HOTAIR; KIF5B; LDHA; LINC01089; MIAT; MTDH; MTERF4; MYC; PTEN; PTK2B; RAD51-AS1; SLC2A1; STAT1; T..."]},{"name":"span","attribs":{"title":"ABCB9; ABCC5; ABLIM2; ACAD10; ACP2; ADAM9; AFF1; AKR1C3; AMBRA1; ANK3; ANKRD17; ANLN; ANP32E; AP2B1; ARFGEF1; ARHGEF2; ARHGEF26-AS1; ARID5B; ARRDC1; ASNSD1; ASRGL1; ATAD1; ATG7; ATP11B; ATP13A4; ATP2A2; B4GALT5; BANP; BCR; BICD1; BRD2; BRD3OS; BTBD11; BUD31; BZW2; C1orf52; C2orf76; C3orf33; CAND1; CANX; CASP2; CBX4; CCDC57; CCDC59; CD82; CDC14A; CDC42SE1; CDK13; CDK5RAP1; CEP350; CFAP45; CHD2; CLASP1; CMAHP; CNOT8; COL18A1; COQ8A; COX5A; CPD; CTDSP2; CTNNA1; CUEDC1; CYB5D1; DAAM1; DBI; DDHD1; DDX55; DECR2; DENND1B; DGLUCY; DHRS2; DHX29; DLG1; DNAJC19; DNAJC7; DNAJC9; DNASE1; DRAM1; DYNLL1; ECT2; EEF1AKMT3; EHHADH; EID1; EIF2AK4; EIF2S3; EIF3E; EIF4A2; EIF4G3; ELMO1; ENSG00000215154; ENSG00000224934; ENSG00000275765; ERAL1; ERGIC2; ETFA; EXOC3-AS1; EXPH5; FAHD1; FAM107B; FAM20B; FAR2; FBXL18; FBXL2; FNDC3B; FNTA; FRMD3; FRMD4A; FRS2; FXR1; G3BP1; GARS1; GDAP1; GNA12; GOLPH3L; GOT1; GPHN; GPS1; GPX3; GPX8; GRAMD1B; GREB1; GSE1; GZF1; H3-3B; HAGH; HECTD1; HEG1; HERC6; HNRNPF; HOXC10; HSPB1; IFIT5; IGF2BP2; INTS6; IPO5; IYD; JUP; KCNB1; KDM4A-AS1; KIAA0895; KIAA1109; KIF23; KIFAP3; KLF10; KMT2A; KMT2E; KMT5A; KRT19; KRT81; KRT8P46; KSR1; LCOR; LDHB; LIMCH1; LIN7C; LINC01089; LINC01137; LINC01588; LINC02303; LMNA; LPP; LRRC26; LRRC45; LRRCC1; LUC7L; LYRM9; MAGI2; MAP3K12; MAP3K4; MAPK6; MAPRE2; MARS1; MBNL1; MBNL2; MDM2; MEF2A; METTL1; METTL25; MEX3A; MFSD11; MGA; MIDEAS; MIOS; MOB4; MORF4L1; MRPL13; MRPS15; MRPS22; MRPS35; MTA1; MTHFD2L; MTREX; MTRR; MXRA7; MXRA8; NAA38; NCALD; NCAM2; NCL; NDE1; NDUFAF6; NDUFS4; NFE2L2; NFX1; NIT1; NOL3; NR1D1; NR2C1; NR6A1; NSMCE1; NSMCE2; NSMF; NUBPL; NUDT9; NUF2; NUPR1; OPA1; ORAI1; ORAI3; OSBPL8; OVCH1-AS1; OXR1; PARP14; PBXIP1; PDAP1; PEX14; PHACTR4; PHF14; PHF19; PHF20; PHF20L1; PIK3CB; PITRM1; PLA2G6; PLD1; PMM2; POC1B; POFUT1; PPM1L; PPP2CA; PPP2CA-DT; PPP2R5C; PPP3CB; PPP3CB-AS1; PRKAB2; PRLR; PROSER2; PROSER3; PTK2; PWWP2A; PYCR1; R3HDM2; RAB31; RAB5B; RAB5IF; RABEPK; RAD23B; RAD51; RAD51-AS1; RALA; RALGAPB; RAPGEF3; RAPGEF5; RAPGEF6; RFC4; RFC5; RGMB-AS1; RGS9; RIC1; RNF157; RNF216; RNLS; RPA1; RPL6; RRP12; RSU1; SARS2; SC5D; SCAMP5; SEC11A; SENP2; SENP5; SETD5; SH3BP5; SKP1; SLC1A5; SLC2A4RG; SLC33A1; SLC35B1; SLC48A1; SLC6A6; SMARCD2; SMARCE1; SMIM14; SMURF2; SNRPA1; SNX1; SNX2; SNX27; SPAG9; SPECC1; SPIDR; SPRYD3; SPSB1; SQOR; SRCAP; SRGAP1; SRPRA; SRPRB; SSR3; ST3GAL1; ST3GAL4; STARD3; STAT3; STRBP; SUPT4H1; SYNE1; SYTL2; TAPT1; TARBP2; TARDBP; TARS1; TBC1D16; TBL1XR1; TCEA2; TFAP2A; TFDP2; THRA; TIGD6; TIMM44; TMEM106C; TMEM167A; TMEM186; TMEM192; TMEM25; TMEM41A; TMEM44; TMEM9; TMEM94; TNIK; TNK2; TNS2; TOR1AIP1; TOR3A; TPM4; TRAPPC9; TRDMT1; TRIM2; TRIQK; TSEN15; TSNAX; TSTD1; TTC21A; TXNIP; TXNRD1; ULK2; UNC13D; VCPKMT; VDR; VPS13B; VPS13B-DT; VPS26B; VPS41; VPS45; WBP1L; WIPF2; WWP2; YEATS4; YTHDF3; ZC3H12A; ZC3H7A; ZCCHC24; ZFAND6; ZNF131; ZNF621"},"children":["ABCB9; ABCC5; ABLIM2; ACAD10; ACP2; ADAM9; AFF1; AKR1C3; AMBRA1; ANK3; ANKRD17; ANLN; ANP32E; AP2B1; ARFGEF1; ARHGEF2; ARHGEF26-AS1; ARID5B; ARRDC1..."]},{"name":"span","attribs":{"title":"ABCC2; ABCC3; ABCC4; ABCC5; ABHD2; ACAA1; ACADM; ACKR3; ACOX1; AHR; AKAP13; ALAS1; AMIGO2; ANGPTL4; BAX; BHLHE40; BIRC2; BLVRB; CAP2; CBR3; CDC37; CDC42EP3; CDK1; CDK4; CDKN1B; CES2; CPEB4; CPT1A; CPT2; CYP1B1; DBI; DNAJB1; DNAJC15; DNAJC7; EDN2; EGFR; EHHADH; ENC1; EP300; EPB41L4B; ESR1; ETNK2; FASN; FGD4; FGF13; FKBP5; FOXO1; FTL; G6PD; GADD45B; GCC1; GCLC; GCLM; GPAM; GPR153; GPX3; GSR; GSTA4; GSTM3; GSTM4; HES1; HMOX1; HSP90AA1; IL12A; IRS2; JUN; JUND; KAT2B; KEAP1; KTN1; LRRC8A; MAFF; MAFG; MGST3; MYC; MYOF; NCOA1; NCOA2; NCOA3; NFE2L2; NQO1; NRG1; NRIP1; PDE4B; PDGFB; POLK; PPARA; PPARD; PRDX1; PRDX6; PSMC5; PTGR1; RGS2; RXRA; S100P; SCD; SCNN1A; SCP2; SEC14L1; SERPINA1; SERPINB2; SERTAD2; SLC19A2; SLC26A2; SLC2A1; SLC2A10; SLC2A13; SLC2A6; SLC39A1; SLC39A10; SLC39A11; SLC39A14; SLC39A6; SLC39A9; SLC5A12; SLC5A6; SLC6A11; SLC6A17; SLC6A6; SLC6A9; SLC7A5; SMARCA1; SMC1A; SP1; SPRY1; SRXN1; STAT3; STOM; SULT1A1; TGFA; TGFBR2; TGFBR3; TSC22D3; TXN; TXNRD1; VDR"},"children":["ABCC2; ABCC3; ABCC4; ABCC5; ABHD2; ACAA1; ACADM; ACKR3; ACOX1; AHR; AKAP13; ALAS1; AMIGO2; ANGPTL4; BAX; BHLHE40; BIRC2; BLVRB; CAP2; CBR3; CDC37; ..."]},{"name":"span","attribs":{"title":"AAMP; ABCA5; ABRAXAS2; ACAD8; ACOX2; ACP2; ACTG1; ACTN4; ACTR1A; ADAM9; ADCY9; ADD1; ADNP; ADO; ADSS2; AFDN; AFMID; AGPS; AHCTF1; AHCYL2; AHI1; AHSA1; AKIRIN2; AKT2; ALDH5A1; ALDOA; ALG10; ALKBH8; AMMECR1; ANKMY1; ANKRD11; ANKS6; ANKZF1; ANP32E; ANXA5; AP2M1; AP2S1; AP4M1; APLF; ARHGAP5; ARHGAP5-AS1; ARHGDIA; ARHGEF2; ARMH3; ARNTL; ARPC2; ARV1; ASAP3; ASPH; ATAD2; ATG4B; ATG9A; ATP5MF; ATP7A; AUP1; B4GALT2; B4GALT3; BABAM1; BABAM2; BANP; BICDL1; BIRC5; BLM; BOD1; BRD3; BRD4; BRD9; BSG; BTBD1; BTBD3; BTRC; BUB1B; BUB3; C11orf58; C18orf54; C19orf44; C1orf109; C1orf159; C2orf69; CA12; CABIN1; CACTIN; CALCOCO1; CAMKK2; CASP8AP2; CBFA2T3; CCDC115; CCDC136; CCDC159; CCDC191; CCDC71; CCNA2; CCNB2; CCNG2; CCP110; CDC40; CDCA2; CDCA3; CDCA8; CDH23; CDK11A; CDK5; CDK5RAP2; CDYL; CECR2; CENPA; CENPO; CENPT; CEP128; CEP290; CEP295; CEP89; CFL1; CHCHD2; CHCHD6; CHD7; CHEK2; CHFR; CHIC2; CHMP4B; CHPT1; CIAO2A; CIDECP1; CLEC16A; CLINT1; CLSTN1; CLTC; CMIP; CMSS1; CNPY2; COMMD3; COPS7B; COX7B; COX7C; CRADD; CRAMP1; CRTC2; CSGALNACT2; CSNK2B; CTBP1; CTCF; CTDP1; CWC22; CYB561D2; DCAKD; DCK; DDIAS; DDX23; DEDD; DENND1A; DGCR8; DIAPH3; DLGAP4; DMXL2; DNAJC17; DNAL1; DPYSL2; DUS1L; DUSP14; DYNC1I2; E2F1; E2F2; E2F3; EEF2K; EFNA3; EHD1; ELOC; EMC3; EME1; EML6; ENSG00000215154; ENSG00000233834; ENSG00000234432; ENY2; EPC1; ERP44; ERVK3-1; EXOC6; FAAP24; FAM110A; FAM117A; FAM168B; FAM193A; FAM222A; FANCC; FANCD2; FANCI; FASN; FASTKD1; FBF1; FBRSL1; FBXL16; FBXL20; FBXL5; FBXO15; FBXO38; FBXO46; FBXO48; FBXO5; FBXW11; FCHSD2; FEM1A; FGFR1; FHL2; FN3K; FTO; FYCO1; GAB2; GATB; GBAP1; GDAP1; GFER; GFUS; GINS2; GLUD1; GNG4; GP6-AS1; GPANK1; GPAT4; GPBP1L1; GPT2; GRAMD4; GRHL1; GRK3; GRK4; GSPT1; GTF2IRD1; GTPBP2; GUCD1; H2AC11; H4C8; HAUS8; HDAC5; HECTD2; HERC1; HIF1A; HIKESHI; HINT3; HM13; HMG20A; HMGB1; HNRNPF; HOOK3; HSCB; HSPD1; HSPE1; HTRA2; IFRD1; IFRD2; IFT140; IFT80; IGF1R; IKZF2; IL1R1; IMPACT; ING3; INTS1; INVS; IPO13; IQGAP3; IRF2BP2; KAT7; KBTBD6; KBTBD7; KCNB1; KCNC3; KCNK1; KCTD20; KCTD9; KDM7A; KDM7A-DT; KIAA0232; KIF15; KIF18B; KIF20B; KIF22; KLF13; KMT2B; KPNA3; LAMTOR2; LARS1; LEMD2; LEPROTL1; LGALSL; LIFR; LIN9; LINC01521; LMNB1; LMNB2; LPIN2; LSM10; LUC7L; LUC7L2; LYSMD1; MACC1; MAD2L1BP; MAGOHB; MAN2A2; MANEAL; MAPT; MARS1; MAT2A; MAU2; MAX; MBTPS2; MCM2; MCM5; MCM7; MDH1; MDM2; MED15; MEGF11; METTL15; METTL4; MFAP3L; MGME1; MIDEAS; MIS18BP1; MITF; MKI67; MKKS; MMACHC; MOB4; MPI; MRGBP; MRPL27; MRPL52; MRPS25; MTMR4; MUS81; MYB; MYCL; MYO6; MYO9B; MZT1; NADK2; NCAPH; NCOA5; NDC80; NEDD4; NEK6; NF2; NFASC; NFE2L2; NFXL1; NKAIN1; NMT1; NOP14; NOP53; NOSIP; NPEPPS; NPEPPSP1; NPRL2; NRDE2; NSD2; NSMCE1-DT; NTAN1; NTHL1; NUCB1; NUDCD1; NUDT13; NUF2; NUMA1; NUTF2; OBSL1; OPLAH; ORMDL3; OSTF1; OTUD7A; PACS2; PAFAH2; PAN2; PANK4; PATZ1; PBK; PCBP1-AS1; PCED1B; PCSK6; PDE3B; PDE6D; PDIK1L; PDS5B; PEAK1; PEX26; PGP; PHB; PHF12; PHF13; PHF21A; PHLPP1; PIAS1; PLEKHF2; PNKD; PNPLA6; PODXL2; POLE2; PPCDC; PPM1E; PPP1R12C; PPP1R15A; PRCP; PRKCA; PRKCE; PRKD1; PRKDC; PRMT3; PRR5; PTCD1; PTDSS2; PTMS; PTPRN2; PTRHD1; QRICH1; QTRT1; QTRT2; RAB30; RAB30-DT; RAB7A; RABEP1; RABGAP1L; RAD51; RAD51-AS1; RAD51B; RALBP1; RANBP10; RANGAP1; RBBP4; RBKS; RBL1; RBM6; RCCD1; RERE; REXO2; RFC3; RFC4; RHOBTB3; RIC8B; RIN3; RMDN3; RNF170; RNF213; RNF227; RPF1; RPGRIP1L; RPH3AL; RPIA; RPL23A; RPP40; RPS13; RPS20; RRM1; RTEL1; RTEL1-TNFRSF6B; RTKN2; RXRA; RYK; SAP30; SASS6; SAV1; SBK1; SCAF1; SCAMP5; SCNM1; SCRIB; SDK1; SEC14L1; SEC24C; SEC61A1; SEL1L; SETMAR; SGPL1; SHLD2; SIAH1; SIK3; SLC35A1; SLC35F2; SLC38A1; SLC44A1; SLC45A4; SLC48A1; SLC4A2; SLC6A6; SLK; SLX4IP; SMARCAL1; SMC6; SMIM30; SMPD3; SMUG1; SNAPC3; SNIP1; SNRPD3; SNX1; SNX5; SPC25; SPG11; SPTBN4; SRCIN1; SRRM3; SRSF1; ST3GAL2; ST7; STAT1; STAT3; STAT5B; STAU1; STK25; STMN1; STMN3; STN1; STON2; STRADB; STRIP1; SUGP1; SUMO3; SYMPK; SYNGR1; TACC1; TBC1D31; TBC1D4; TBC1D8B; TBX6; TCEA2; TCP11L1; TEDC2; TFAP2A; TFEB; TGFBR3; THAP1; THAP12; THUMPD2; THYN1; TIAL1; TIMM21; TIMM22; TK1; TKT; TM9SF4; TMBIM1; TMEM11; TMEM115; TMEM131; TMEM150A; TMEM160; TMEM181; TMEM33; TMEM69; TMEM70; TMTC3; TNRC6B; TOP2A; TOR2A; TP53; TP53BP2; TPGS1; TPRA1; TPX2; TRAK2; TRAPPC3; TRIM29; TRIP13; TSC2; TTC13; TTC33; TTLL4; TUBGCP3; TXNDC15; UBE2E2; UBE2E3; UBE2G2; UBE2Q1; UBE2Q2; UBE3C; UBQLN4; UBR2; UBR3; UBXN2A; UCKL1; UNG; UPF1; USF2; USP28; USP3; USP31; USP39; USP48; USP6NL; VAPA; VCPIP1; VDAC2; VIPAS39; VKORC1; VMP1; VPS35L; VPS50; VRK1; VRK3; WASF1; WBP2; WDPCP; WDR13; WNK1; WRAP53; WT1-AS; WWP1; XIST; XRCC3; XYLT2; YEATS2; ZBTB11; ZBTB46; ZFX; ZFYVE1; ZFYVE19; ZFYVE21; ZFYVE28; ZNF133; ZNF155; ZNF217; ZNF225; ZNF280C; ZNF280D; ZNF32; ZNF331; ZNF337; ZNF354A; ZNF37A; ZNF436; ZNF444; ZNF451; ZNF473; ZNF573; ZNF589; ZNF605; ZNF687; ZNF785; ZW10; ZWINT; ZYG11A"},"children":["AAMP; ABCA5; ABRAXAS2; ACAD8; ACOX2; ACP2; ACTG1; ACTN4; ACTR1A; ADAM9; ADCY9; ADD1; ADNP; ADO; ADSS2; AFDN; AFMID; AGPS; AHCTF1; AHCYL2; AHI1; AHS..."]},{"name":"span","attribs":{"title":"ACAA1; ADD3; AGPS; AGR3; AHR; AKAP13; AKAP9; ANAPC7; ANK3; AP4E1; ARHGAP26; ARHGAP32; ARID5B; ATP2C1; C17orf67; C1orf21; C1orf226; CAP1; CASD1; CCDC115; CCDC159; CEP120; CFDP1; CIB1; CTDSPL; DMXL2; DNAH11; DNMBP; DNTTIP2; DRAIC; EFCAB14; ELP3; EML6; ENC1; EPS8; FERMT2; GARNL3; GDPGP1; GLRX; GNAL; GPR160; H2AC7; H2BC5; HEG1; HINT2; HMOX1; HNMT; HOXC6; HPGD; ICA1; IFT46; ILRUN; INO80; IRS1; ITGB5; JMJD1C; JMY; KAZN; KCNK1; KHDC4; LAMA3; LEKR1; LIMA1; LINC00511; LINC01588; LMO7; LRRC1; LTBP1; LUC7L2; MALSU1; MAPK6; MBP; MLLT3; MPZL1; MTMR11; MYO10; MYOF; NAALADL2; NASP; NFKBIZ; NOD1; NPM1; NXN; OAS1; PANK3; PDE4D; PDHX; PGM3; PHF12; PIGC; PIM1; PLCB1; POMP; PPFIBP2; PPP2R2A; PRKCI; PRLR; RAPGEF6; RBM47; REPIN1; REXO2; RIOK2; RND3; RNF125; RNF144A; RNF145; RPF1; RPS27A; RPS27L; S100A16; SCOC; SELENOP; SEMA3C; SEPTIN8; SETD5; SGO2; SLC9A1; SMAD3; SMIM31; SNAP23; SP110; SPATS2L; SPRED2; SPSB1; SRI; STARD4; SUCLA2; SUMF2; SYNE2; TBL1XR1; TDP2; TERF2; TGS1; THADA; TMCC1; TMEM8B; TMOD3; TMPRSS4; TNRC18; TRIO; UTRN; VCPIP1; VMP1; VWA8; XPNPEP1; YAP1; ZNF217"},"children":["ACAA1; ADD3; AGPS; AGR3; AHR; AKAP13; AKAP9; ANAPC7; ANK3; AP4E1; ARHGAP26; ARHGAP32; ARID5B; ATP2C1; C17orf67; C1orf21; C1orf226; CAP1; CASD1; CCD..."]},{"name":"span","attribs":{"title":"ABCC1; ACVR2A; ADD3; ADGRL2; AKT3; ANXA2; ANXA4; APBB2; ATP2B1; ATP2C1; ATRN; ATRX; ATXN1; BCKDHB; BHLHE40; BMPR1A; CAP2; CCN1; CDC42BPA; CDK13; CDKN1B; CDON; CITED2; COL5A2; DBP; DDAH1; DLC1; DLG1; DUSP1; DYRK1A; EFEMP1; ERBB2; F3; FHL2; GCNT1; ICA1; ID1; IGF1R; IGFBP5; INPP4B; IRS1; KALRN; KCNMA1; LAMC1; LPAR1; LTBP1; MAGI2; MAP1B; MAP2K5; MAPK14; MGLL; MIOS; MMP16; MTA1; MYC; NFIB; NFKB1; NIPBL; NOTCH2; NR1D2; NRP1; PDLIM5; PEX14; PHF3; PIAS3; PIK3R3; PLCB4; PLPP3; PMP22; PPARG; PRDM2; PRKAR2B; PRKCA; PRKCE; PTEN; PTPN21; PTPRM; RASA2; RBPMS; RND3; RUNX1; RXRA; SCN8A; SDC2; SFMBT1; SIPA1L1; SMAD3; SMAD7; SPOP; SRI; SYNE1; SYNJ2; TGFBR2; TGFBR3; TJP1; TOGARAM1; VAV2; VLDLR; YTHDC1; ZMIZ1"},"children":["ABCC1; ACVR2A; ADD3; ADGRL2; AKT3; ANXA2; ANXA4; APBB2; ATP2B1; ATP2C1; ATRN; ATRX; ATXN1; BCKDHB; BHLHE40; BMPR1A; CAP2; CCN1; CDC42BPA; CDK13; CD..."]},{"name":"span","attribs":{"title":"AACS; ADAT2; ADGRG1; ALDOA; ANKRD42; ANKRD42-DT; AP2S1; ARID4B; ARL4A; ARNTL; ASB3; ASB7; ATP13A4; ATXN7L3B; AZI2; BABAM1; BDP1; BRMS1; BSG; C19orf44; CABIN1; CAMK2D; CASP9; CBLL1; CBX8; CCDC157; CCDC191; CCDC40; CCDC91; CDH23; CFAP45; CFDP1; CHRNA7; CHST11; CIC; CITED2; CLGN; CLPB; CMTM6; CNIH2; COPE; COPS2; CREBL2; CRIP2; CRTC2; CSF1; CTU1; DCAF6; DDX23; DDX3X; DDX49; DHX8; DNAH7; DNAJC16; DNAL4; DNALI1; DTWD1; DUS3L; DUSP16; DYNC1I2; DYRK1A; DYRK1B; DYRK2; ECT2; EFCAB11; EFCAB2; EFHC1; EHD1; ELAPOR1; EMC3; ENY2; EPHB6; ERCC1; ERG28; ERLEC1; ESYT1; ETFRF1; ETV1; FAM13A; FAM229B; FBXO32; FEM1A; FGGY; FOXO6; GALK2; GEMIN6; GET1; GGPS1; GIHCG; GRK4; GSE1; H4C8; HBP1; HECTD1; HIPK1; HM13; HMBOX1; HNRNPF; HSP90B1; IDS; IFRD2; IFT122; IFT20; IK; ILF3; ILF3-DT; ING3; INTS1; INTS9; IPO13; IQGAP2; IRF2BPL; JMJD1C; JUND; KAZN; KLF13; KLF4; KLHL18; KLHL24; KRT8; LAMTOR2; LCORL; LEMD2; LINC00630; LINC01876; LINS1; LMNA; LRRC23; LRRC27; LYPLA2; MAFK; MAML3; MAP4K3; MARCHF8; MBD4; MBTPS1; MBTPS2; MDH1; MED31; MFSD4B; MLF1; MLLT1; MNS1; MOK; MPC2; MRPL24; MRPS21; MTX1; NAA20; NBAS; NCOA5; NCOR2; NDRG4; NDUFA2; NEK11; NFATC2; NFIA; NLK; NOD2; NOP14; NR4A2; NRM; NUBPL; NUDCD1; NUDT13; NUF2; NUP62CL; NUPR1; OAZ2; OPA1; OSR2; P4HB; PANK4; PBX1-AS1; PCGF5; PCSK6; PER1; PES1; PFDN6; PHF8; PIBF1; PIFO; PIM1; PLEKHF2; PPOX; PPP1R12C; PPP1R15A; PPP1R3E; PRORP; PSTPIP2; PTCH1; QTRT2; RABGAP1L; RARA; RBM23; RERE; RFC5; RFX1; RFX3; RGL2; RMND5A; RPRD1B; RPS6KA5; RRM2B; RRP12; RSRC1; SARAF; SELENOP; SETD5; SF3A1; SIK3; SIRT1; SLC16A1; SLC38A2; SLC44A1; SMC1A; SNX29; SRSF7; SSBP1; ST7; STAG2; STAU2; STK32C; STRIP1; SUMO3; SUN1; SYNE2; TAGLN2; TAPBP; TARS1; TBC1D16; TCEA2; TDP1; TENT5C; TFAP4; TGIF1; THADA; THBS3; TIMM44; TIPARP; TMED10; TMEM115; TMEM139; TNFAIP1; TOM1L2; TOR2A; TRAM1; TTLL5; TUBB4B; TUBE1; TUFT1; TXNDC11; TXNIP; UACA; UBE2Z; UBL7; UBL7-DT; UBQLN4; UBXN11; UFSP2; VRK3; VTA1; WBP1; WBP2; WDPCP; WDR46; WDR77; WHRN; YBX3; YIPF4; YLPM1; YTHDC1; ZC2HC1C; ZC3H10; ZDHHC1; ZDHHC14; ZFP36L1; ZFP36L2; ZFR; ZFYVE28; ZMYND10; ZNF131; ZNF3; ZNF37BP; ZNF473; ZNF487; ZNF497; ZNF510; ZNF584; ZNF837; ZSWIM8"},"children":["AACS; ADAT2; ADGRG1; ALDOA; ANKRD42; ANKRD42-DT; AP2S1; ARID4B; ARL4A; ARNTL; ASB3; ASB7; ATP13A4; ATXN7L3B; AZI2; BABAM1; BDP1; BRMS1; BSG; C19orf..."]},{"name":"span","attribs":{"title":"ABCC4; AKR1B1; AKR1C3; ANXA1; ANXA2; ANXA4; ANXA5; ANXA6; EDN1; HPGD; HSD11B2; MITF; PLA2G4A; PPARG; PPARGC1B; PTGER4; PTGES; PTGFRN; S100A10; S100A6; SOX9"},"children":["ABCC4; AKR1B1; AKR1C3; ANXA1; ANXA2; ANXA4; ANXA5; ANXA6; EDN1; HPGD; HSD11B2; MITF; PLA2G4A; PPARG; PPARGC1B; PTGER4; PTGES; PTGFRN; S100A10; S100..."]}]}],"filterable":true,"showPageSizeOptions":true,"dataKey":"8e57841b826bfaa43404fc9e43d7c56d"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div id="genes-mapped-to-both-ar-and-er" class="section level4">
<h4>Genes Mapped to Both AR and ER</h4>
<pre class="r"><code>pwf_t1t2 &lt;- nullp(
  vec_t1t2,
  bias.data = log10(setNames(all_ids$w, all_ids$gene_id)[mapped_ids]), 
  plot.fit = TRUE
) </code></pre>
<pre class="r"><code>if (sum(pwf_t1t2$DEgenes) &gt; 0) {
  goseq_t1t2_res &lt;- pwf_t1t2 %&gt;% 
    goseq(gene2cat = gs_by_geneid, method = gs_method) %&gt;% 
    dplyr::filter(numDEInCat &gt; 0) %&gt;% 
    as_tibble() %&gt;% 
    dplyr::select(
      gs_name = category, pval = over_represented_pvalue, starts_with(&quot;num&quot;)
    ) %&gt;% 
    mutate(
      adj_p = p.adjust(pval, adj_method),
      enriched = adj_p &lt; enrich_alpha
    ) %&gt;% 
    dplyr::filter(numDEInCat &gt;= min_sig) %&gt;% 
    left_join(
      dplyr::filter(msigdb, gene_id %in% mapped_ids)[c(&quot;gs_name&quot;, &quot;gene_name&quot;, &quot;gs_url&quot;, &quot;gs_description&quot;)],
      by = &quot;gs_name&quot;
    ) %&gt;% 
    chop(gene_name) %&gt;% 
    mutate(gene_name = vapply(gene_name, paste, character(1), collapse = &quot;; &quot;))
}
any_goseq_t1t2 &lt;- sum(goseq_t1t2_res$adj_p &lt; enrich_alpha) &gt; 0
tg_t1t2 &lt;- make_tbl_graph(
  goseq_t1t2_res, 
  gs = gs_by_gsid %&gt;% 
    lapply(intersect, rownames(subset(pwf_t1t2, DEgenes))) %&gt;% 
    .[vapply(., length, integer(1)) &gt; 0]
)
plot_network_t1t2 &lt;- length(tg_t1t2) &gt;= min_network_size
if (!any_goseq_t1t2) para &lt;- glue(para, &quot; No enrichment was found.&quot;)</code></pre>
<pre class="r"><code>tbl &lt;- goseq_t1t2_res %&gt;%
  dplyr::filter(enriched) %&gt;% 
  dplyr::select(
    gs_name, gs_description, pval, adj_p, starts_with(&quot;num&quot;), gene_name
  ) %&gt;% 
  reactable(
    filterable = TRUE,
    showPageSizeOptions = TRUE,
    columns = list2(
      gs_name = colDef(
        name = &quot;Gene Set&quot;,
        cell = function(value) htmltools::tags$a(
          href = gs_url[[value]], 
          target = &quot;_blank&quot;, 
          str_replace_all(value, &quot;_&quot;, &quot; &quot;)
        ),
        html = TRUE,
        maxWidth = 150
      ),
      gs_description = colDef(
        name = &quot;Description&quot;,
        cell = function(value) str_trunc(value, width = 150)
      ),
      pval = colDef(
        name = &quot;P-Value&quot;,
        cell = function(value) ifelse(
          value &lt; 0.001,
          sprintf(&quot;%.2e&quot;, value),
          sprintf(&quot;%.3f&quot;, value)
        ),
        maxWidth = 80
      ),
      adj_p = colDef(
        name = &quot;p&lt;sub&gt;adj&lt;/sub&gt;&quot;, html = TRUE,
        cell = function(value) ifelse(
          value &lt; 0.001,
          sprintf(&quot;%.2e&quot;, value),
          sprintf(&quot;%.3f&quot;, value)
        ),
        maxWidth = 80
      ),
      numDEInCat = colDef(
        name = &quot;Genes Mapped to Both&quot;,
        cell = function(value) comma(value, 1),
        maxWidth = 80
      ),
      numInCat = colDef(
        name = &quot;Genes Mapped to Either&quot;,
        cell = function(value) comma(value, 1),
        maxWidth = 80
      ),
      gene_name = colDef(
        name = &quot;Gene Names&quot;,
        cell = function(value) with_tooltip(value, width = 150)
      )
    )
  )
div(class = &quot;table&quot;,
    div(class = &quot;table-header&quot;, div(class = &quot;caption&quot;, para)),
    tbl
)</code></pre>
<div class="table">
<div class="table-header">
<div class="caption">All 3,217 genes with a 
mapped binding window to both AR and ER were compared to 
the 7,631 genes mapped to either, 
in order to ascertain whether any key gene-sets were likely to be targeted by 
both factors in combination. No sampling bias offset was incorporated, instead 
using a simple Hypergeometric model for enrichment testing.</div>
</div>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-99cb6d8377c832f661cb" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-99cb6d8377c832f661cb">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"gs_name":["HALLMARK_ESTROGEN_RESPONSE_EARLY","GREB1_TARGET_GENES","HALLMARK_ESTROGEN_RESPONSE_LATE","IGLV5_37_TARGET_GENES"],"gs_description":["Genes defining early response to estrogen.","Genes containing one or more binding sites for UniProt:Q4ZG55 (GREB1) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Genes defining late response to estrogen.","Genes containing one or more binding sites for UniProt:A0A075B6J1 (IGLV5_37) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization."],"pval":[1.27763540353069e-13,2.71062213556352e-12,4.43364411839946e-10,2.52104329930578e-05],"adj_p":[3.288633528688e-10,3.48857068847024e-09,3.80406665358674e-07,0.0162229136310327],"numDEInCat":[115,284,93,132],"numInCat":[163,497,136,239],"gene_name":["ABAT; ABCA3; ABHD2; ABLIM1; ADCY1; ADCY9; ADD3; AFF1; AKAP1; AMFR; ANXA9; AQP3; AR; AREG; ARL3; ASB13; B4GALT1; BAG1; BCL11B; BCL2; BHLHE40; BLVRB; CA12; CBFA2T3; CCN5; CD44; CELSR1; CELSR2; CHPT1; DEPTOR; DHCR7; DHRS2; DHRS3; DLC1; DYNLT3; ELF3; ELOVL2; ELOVL5; ENDOD1; ESRP2; FAM102A; FARP1; FASN; FDFT1; FHL2; FKBP5; FLNB; FOS; FOXC1; FRK; GAB2; GFRA1; GLA; GREB1; HES1; HR; HSPB8; IGF1R; IGFBP4; IL17RB; IL6ST; INHBB; INPP5F; ISG20L2; ITPK1; JAK2; KAZN; KCNK15; KCNK5; KDM4B; KLF10; KLF4; KRT15; KRT18; KRT19; KRT8; LAD1; LRIG1; MAPT; MAST4; MED13L; MICB; MLPH; MREG; MUC1; MYB; MYBL1; MYC; MYOF; NADSYN1; NBL1; NCOR2; NRIP1; OPN3; OVOL2; P2RY2; PAPSS2; PDZK1; PEX11A; PGR; PLAAT3; PMAIP1; PODXL; PPIF; PTGES; RAB17; RAB31; RAPGEFL1; RARA; RASGRP1; RBBP8; REEP1; RETREG1; RHOBTB3; RHOD; RPS6KA2; RRP12; SCARB1; SCNN1A; SEC14L2; SFN; SH3BP5; SIAH2; SLC16A1; SLC19A2; SLC1A1; SLC1A4; SLC22A5; SLC24A3; SLC26A2; SLC27A2; SLC2A1; SLC39A6; SLC7A2; SLC7A5; SLC9A3R1; SNX24; SULT2B1; SVIL; SYBU; SYNGR1; SYT12; TBC1D30; TFAP2C; TFF1; TFF3; TGIF2; THSD4; TIAM1; TIPARP; TJP3; TMEM164; TOB1; TPBG; TPD52L1; TSKU; TTC39A; TUBB2B; UGCG; UNC119; WFS1; WWC1; XBP1","ABCA3; ABCC11; ABHD17C; ABHD2; ADAP1; ADGRG1; ADIRF-AS1; AFAP1; AFF1; AGAP2; AHCYL2; AKAP1; AKAP1-DT; AKAP8L; AKAP9; ALKBH7; AMOTL2; AMZ1; ANAPC2; ANKFY1; ANKRD11; ANKRD22; ANP32E; ANXA9; AP5B1; ARHGEF1; ARID1B; ARNTL; ARRDC1; ARRDC4; ASCC2; ASCL2; ATP2A2; ATP5MC1; ATP6V0A4; ATP8B1; ATXN7L1; B3GALNT1; BAIAP3; BATF; BBOF1; BCAR1; BCAR3; BCAS3; BCL3; BCL9L; BICDL1; BLVRB; C1QTNF6; C1orf21; C5AR2; C9orf152; CA12; CAB39L; CAMTA1; CARD10; CASP7; CCDC159; CCDC40; CCDC88C; CCN5; CCT7; CD276; CD59; CDC34; CDS1; CELSR1; CEP89; CFAP45; CHD3; CHD6; CHD8; CHPT1; CHST12; CIC; CITED2; CLN8; CMTR2; CNOT3; COMT; COMTD1; COQ8A; CORO2A; CRACR2A; CREBRF; CRKL; CRNDE; CRPPA; CSNK1G2; CTSB; CYB561; CYLD; CYP1B1; CYP1B1-AS1; DAAM1; DDA1; DDX47; DENND2D; DGAT2; DLC1; DLG5; DMXL1; DNAH11; DNASE1; DNMBP; DNPEP; DOCK4; DOK7; DOP1B; DRAIC; DSCAM-AS1; DYNLT3; ECH1; EHD1; EHF; EHMT1; ELAPOR1; EML6; ENC1; ENSA; ENSG00000224934; ENSG00000267288; ENSG00000268858; ENSG00000275765; EPB41L1; EPG5; EPHA1; EPHB4; EPN2; EPN3; EPS8; ERBB2; ERCC1; ERGIC1; ESRP2; ETNK2; EVI5L; EVL; EXOSC10; FAAP24; FAF1; FAM107B; FAM117A; FAM126A; FAM174B; FAM234B; FAM86EP; FBRS; FBXO38; FHDC1; FMN1; FNIP2; FOXN3; FREM2; FRMD4B; FRMD6; FRYL; G3BP1; GALNT10; GATAD2A; GATAD2B; GCLM; GCOM1; GIGYF2; GLRX5; GOLPH3L; GOT1; GPNMB; GPR160; GPRC5A; GRAMD1A; GREB1; GRHL1; GRHL2; GSPT1; GTPBP3; HEBP2; HERC1; HK1; HMGCS1; HNRNPC; HNRNPL; HOMEZ; HSH2D; HSPB8; ID1; IDE; IFITM10; IFRD1; IFT27; IGF1R; IGSF3; IGSF9; IKBKG; IKZF2; INPP4B; IQCD; IQSEC1; ISG20; JUND; JUP; KAT7; KATNB1; KCNK1; KCTD11; KDM4B; KDM5A; KHSRP; KIAA0100; KIAA0895; KIAA1217; KIF21A; KLF6; KLHL25; KRT7; KRT8; KRT80; LAD1; LAMB1; LCOR; LDLRAD4; LENG1; LFNG; LIMA1; LINC00511; LINC00963; LINC01588; LLGL2; LMCD1; LNC-LBCS; LPP; LRIG2; LRIG3; LRRFIP1; MACC1; MAF1; MANEAL; MAP3K3; MAP4K3-DT; MAPK10; MAPKAP1; MCCC2; MCF2L; MCRIP2; MEF2A; MGRN1; MICOS10; MIDEAS; MIDN; MIOS; MIR9-3HG; MIR99AHG; MIX23; MLLT3; MPHOSPH10; MSMO1; MTA1; MTMR12; MTMR4; MTUS1; MUC5B; NAALADL2; NADK; NAGK; NAIF1; NCAM2; NCOA1; NCOA4; NDRG1; NDUFAF8; NFATC2; NFKBIA; NFKBIZ; NFYB; NINL; NIPSNAP1; NIPSNAP2; NLK; NMT1; NOP14-AS1; NR1D1; NRP1; NT5DC3; NUDT16; NUMA1; OSBPL10; OSBPL2; OSBPL8; OVOL2; OXR1; P4HB; PABPN1; PAIP2B; PAK4; PAM; PARD3; PBX1; PCBP1-AS1; PCDH1; PCTP; PDGFB; PDGFD; PDZK1; PGP; PHTF2; PIK3C2G; PKIB; PKP4-AS1; PLA2G6; PLEKHF2; PNKD; PNPLA7; POLR3K; POU2F3; PPARD; PPM1H; PPM1L; PRADC1; PREX1; PRKCD; PRKCH; PROSER1; PROSER2; PROSER3; PRR15L; PRSS22; PRSS27; PSCA; PSMG3; PTPRH; PUM2; PWWP2A; PXDN; PXMP4; PYM1; PYROXD2; RAB26; RAB5B; RABL3; RAD51B; RANBP2; RAPGEFL1; RARA; RBBP8; RBBP8NL; RBL2; RBM20; RBM47; RC3H1; RECQL5; RFFL; RFX1; RGMB; RGMB-AS1; RGS3; RHOBTB3; RHOC; RNF19A; RNF207; RNF224; RPL27A; RPS6KA2; RPS6KL1; RTN4; S100A10; S100A16; S100A4; SAMD8; SCAMP4; SCGB1D2; SCIN; SCNN1A; SCRIB; SEC14L2; SEPTIN9; SERPINA1; SGMS2; SHARPIN; SIDT2; SKA2; SKIL; SLC16A3; SLC1A5; SLC25A25; SLC25A36; SLC27A2; SLC29A1; SLC35E1; SLC38A1; SLC38A2; SLC39A1; SLC44A2; SLC45A4; SLC6A6; SLC7A2; SLC9A8; SMAD3; SMARCD2; SNX9; SPECC1L; SPNS2; SPRED2; SPRYD3; SPTBN4; SPTLC2; SPTSSB; SRI; SRPK1; SRRM2-AS1; SSNA1; ST6GAL1; ST6GALNAC4; STARD10; STAT3; SUCLA2; SULT1A1; SYBU; SYT8; TACC2; TBC1D16; TBC1D8; TBCD; TBX2; TEDC1; TEPSIN; TFAP2C; TFF1; TGFBI; THADA; THSD4; TIPARP; TJP1; TLE1; TMBIM1; TMC1; TMCO3; TMED3; TMEM120A; TMEM160; TMEM184B; TMEM229B; TMEM241; TMEM60; TMEM94; TMPRSS4; TMTC2; TNK2; TNS1; TOM1L2; TOP2A; TPD52L1; TPD52L2; TPRN; TRAF7; TRAPPC9; TRIB1; TRIM29; TSPAN9; TTC39A-AS1; TTLL5; TUBA1A; TUBA1C; TUBB4B; UBE2C; UNC13D; UNKL; UPF1; URB1; USF3; UTRN; VAMP5; VAV3; VEPH1; VIRMA; VMP1; VPS26C; WDR90; WIPI1; WWC1; WWP1; XRCC3; YIF1A; ZDHHC1; ZFYVE21; ZFYVE28; ZKSCAN1; ZMYM2; ZMYND8; ZNF444; ZNF552; ZNF669; ZNF687; ZNF689; ZNF750; ZNF823; ZNRF1; ZSWIM1","ABCA3; ABHD2; ACOX2; ADD3; AFF1; AGR2; AMFR; ANXA9; AREG; ARL3; BAG1; BATF; BCL2; BLVRB; CA12; CA2; CACNA2D2; CCN5; CD44; CD9; CDC6; CDH1; CELSR2; CHPT1; CHST8; COX6C; CPE; DCXR; DHCR7; DHRS2; DLG5; DNAJC1; DYNLT3; ELOVL5; EMP2; ETFB; FABP5; FAM102A; FARP1; FDFT1; FGFR3; FKBP5; FLNB; FOS; FOXC1; FRK; GALE; GFUS; GINS2; GLA; HMGCS2; HOMER2; HPRT1; HR; HSPB8; IDH2; IGFBP4; IL17RB; IL6ST; IMPA2; ISG20; ITPK1; JAK1; JAK2; KCNK5; KIF20A; KLF4; KRT19; LAMC2; LARGE1; LLGL2; LSR; MAPK13; MAPT; MICB; MOCS2; MYB; MYOF; NBL1; NCOR2; NRIP1; OPN3; OVOL2; PAPSS2; PDCD4; PDZK1; PGR; PLAAT3; PPIF; PRKAR2B; PRLR; PTGES; RAB31; RABEP1; RAPGEFL1; RBBP8; RPS6KA2; S100A9; SCARB1; SCNN1A; SCUBE2; SERPINA1; SFN; SIAH2; SLC16A1; SLC1A4; SLC22A5; SLC24A3; SLC26A2; SLC27A2; SLC29A1; SLC7A5; SLC9A3R1; SNX10; SORD; ST14; ST6GALNAC2; SULT2B1; TFAP2C; TFF1; TFF3; TFPI2; TIAM1; TJP3; TOB1; TOP2A; TPBG; TPD52L1; TRIM29; TSPAN13; UGDH; UNC13B; WFS1; XBP1; XRCC3; ZFP36","ABCC12; ABHD12; ACTG1; ADAT2; ADSS1; AFAP1; AFDN; AGFG1; AGRN; AHCY; AHNAK; ALDOA; ALG10; ALKBH2; AMPD3; ANKAR; ANKRD40; ANXA11; AP4E1; AP5B1; APBA2; ARHGEF1; ARL8B; ATP8B3; BAHCC1; BAIAP3; BCL2; BCOR; BCR; BMP1; BTBD11; CACNA1D; CALM1; CAMK2B; CASKIN2; CASP7; CCDC159; CCDC59; CCDC88C; CDC42EP3; CDCA2; CDK5RAP3; CELSR1; CFDP1; CHCHD4; CHEK2; CIDECP1; CLASP1; CLEC16A; CNNM2; CNTNAP2; COG2; COL9A2; CPLX1; CTBP2; CTNNA1; CTSB; CUEDC1; CYB561; DDX47; DLG3; DMAP1; DMTN; DRAIC; DUSP5; E2F6; EFHD1; EGFR; EHD1; ELAVL2; EML1; ENC1; ENSG00000215154; ENTPD8; EPHB6; ETV4; EXD3; FAM189A2; FAM83F; FANK1; FBXL8; FBXO27; FCSK; FGFR1; FGFR3; FOXA1; FOXP2; FRA10AC1; FRMD4B; FZD1; GFM2; GNG4; GPNMB; GRB10; GRHL3; GRIN1; GSE1; GSTCD; HECW2; HIVEP2; HIVEP3; HMG20A; HNMT; HOXC6; HOXC9; IKBKG; INTS12; IRS1; JAG1; KANSL3; KEAP1; KIF12; LFNG; LHX4; LIG1; LIMCH1; LMNA; LPIN1; LRATD1; LRIG3; LTBP4; MAILR; MAML3; MAP2; MBOAT7; MCF2L; METTL25; MID1IP1; MIR9-3HG; MRPL1; MRPL13; MRPL39; MTBP; MYC; MYO5B; MYOF; NDST1; NEDD4; NET1; NF1; NFKBIA; NIBAN1; NIBAN2; NIPAL1; NOXA1; NRARP; NSA2; NT5DC3; NTRK3; NUAK1; NUDT9; NUMA1; OSGIN1; OSMR; PAFAH2; PBX1; PDE5A; PDHX; PDZK1; PLEKHG4B; PLOD2; PNRC1; POLR2B; POU2F3; PPCDC; PPP2CB; PRICKLE2; PRKAB2; PRKCE; PTCH1; PTPRH; PTPRM; PWWP2B; RAP1GAP; RARA; RARS1; RCAN1; RFFL; RHOBTB3; RHOD; RHOT1; RIPOR3; RNF166; ROR2; RPS7; RRAGD; RUNX1; S100A14; SEC62; SGK3; SH3TC1; SHANK2; SLC22A5; SLC2A4RG; SLC45A4; SMAD7; SPRED2; SPRED3; SRCIN1; SRRM3; SSBP1; STARD10; STAT1; STAU2; STXBP1; STXBP6; SUCLG1; SYNE2; TBC1D1; TBL3; TBPL1; TCF7; TFDP2; TFPI2; TGFBI; THADA; THRB; TMEM248; TMTC2; TNNI2; TNS3; TPD52; TRAF4; TRAPPC3; TRIB2; TRIM47; TTC6; TXNRD2; UBE2E2; USP3; WNT5A; YAP1; ZBTB10; ZC3H12A; ZMIZ1; ZNF3; ZNF337; ZNF609; ZNF827"]},"columns":[{"id":"gs_name","name":"Gene Set","type":"character","cell":[{"name":"a","attribs":{"href":"\n      http://www.gsea-msigdb.org/gsea/msigdb/collections.jsp","target":"_blank"},"children":["HALLMARK ESTROGEN RESPONSE EARLY"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["GREB1 TARGET GENES"]},{"name":"a","attribs":{"href":"\n      http://www.gsea-msigdb.org/gsea/msigdb/collections.jsp","target":"_blank"},"children":["HALLMARK ESTROGEN RESPONSE LATE"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["IGLV5 37 TARGET GENES"]}],"html":true,"maxWidth":150},{"id":"gs_description","name":"Description","type":"character","cell":["Genes defining early response to estrogen.","Genes containing one or more binding sites for UniProt:Q4ZG55 (GREB1) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 2...","Genes defining late response to estrogen.","Genes containing one or more binding sites for UniProt:A0A075B6J1 (IGLV5_37) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD ve..."]},{"id":"pval","name":"P-Value","type":"numeric","cell":["1.28e-13","2.71e-12","4.43e-10","2.52e-05"],"maxWidth":80},{"id":"adj_p","name":"p<sub>adj<\/sub>","type":"numeric","cell":["3.29e-10","3.49e-09","3.80e-07","0.016"],"html":true,"maxWidth":80},{"id":"numDEInCat","name":"Genes Mapped to Both","type":"numeric","cell":["115","284","93","132"],"maxWidth":80},{"id":"numInCat","name":"Genes Mapped to Either","type":"numeric","cell":["163","497","136","239"],"maxWidth":80},{"id":"gene_name","name":"Gene Names","type":"character","cell":[{"name":"span","attribs":{"title":"ABAT; ABCA3; ABHD2; ABLIM1; ADCY1; ADCY9; ADD3; AFF1; AKAP1; AMFR; ANXA9; AQP3; AR; AREG; ARL3; ASB13; B4GALT1; BAG1; BCL11B; BCL2; BHLHE40; BLVRB; CA12; CBFA2T3; CCN5; CD44; CELSR1; CELSR2; CHPT1; DEPTOR; DHCR7; DHRS2; DHRS3; DLC1; DYNLT3; ELF3; ELOVL2; ELOVL5; ENDOD1; ESRP2; FAM102A; FARP1; FASN; FDFT1; FHL2; FKBP5; FLNB; FOS; FOXC1; FRK; GAB2; GFRA1; GLA; GREB1; HES1; HR; HSPB8; IGF1R; IGFBP4; IL17RB; IL6ST; INHBB; INPP5F; ISG20L2; ITPK1; JAK2; KAZN; KCNK15; KCNK5; KDM4B; KLF10; KLF4; KRT15; KRT18; KRT19; KRT8; LAD1; LRIG1; MAPT; MAST4; MED13L; MICB; MLPH; MREG; MUC1; MYB; MYBL1; MYC; MYOF; NADSYN1; NBL1; NCOR2; NRIP1; OPN3; OVOL2; P2RY2; PAPSS2; PDZK1; PEX11A; PGR; PLAAT3; PMAIP1; PODXL; PPIF; PTGES; RAB17; RAB31; RAPGEFL1; RARA; RASGRP1; RBBP8; REEP1; RETREG1; RHOBTB3; RHOD; RPS6KA2; RRP12; SCARB1; SCNN1A; SEC14L2; SFN; SH3BP5; SIAH2; SLC16A1; SLC19A2; SLC1A1; SLC1A4; SLC22A5; SLC24A3; SLC26A2; SLC27A2; SLC2A1; SLC39A6; SLC7A2; SLC7A5; SLC9A3R1; SNX24; SULT2B1; SVIL; SYBU; SYNGR1; SYT12; TBC1D30; TFAP2C; TFF1; TFF3; TGIF2; THSD4; TIAM1; TIPARP; TJP3; TMEM164; TOB1; TPBG; TPD52L1; TSKU; TTC39A; TUBB2B; UGCG; UNC119; WFS1; WWC1; XBP1"},"children":["ABAT; ABCA3; ABHD2; ABLIM1; ADCY1; ADCY9; ADD3; AFF1; AKAP1; AMFR; ANXA9; AQP3; AR; AREG; ARL3; ASB13; B4GALT1; BAG1; BCL11B; BCL2; BHLHE40; BLVRB;..."]},{"name":"span","attribs":{"title":"ABCA3; ABCC11; ABHD17C; ABHD2; ADAP1; ADGRG1; ADIRF-AS1; AFAP1; AFF1; AGAP2; AHCYL2; AKAP1; AKAP1-DT; AKAP8L; AKAP9; ALKBH7; AMOTL2; AMZ1; ANAPC2; ANKFY1; ANKRD11; ANKRD22; ANP32E; ANXA9; AP5B1; ARHGEF1; ARID1B; ARNTL; ARRDC1; ARRDC4; ASCC2; ASCL2; ATP2A2; ATP5MC1; ATP6V0A4; ATP8B1; ATXN7L1; B3GALNT1; BAIAP3; BATF; BBOF1; BCAR1; BCAR3; BCAS3; BCL3; BCL9L; BICDL1; BLVRB; C1QTNF6; C1orf21; C5AR2; C9orf152; CA12; CAB39L; CAMTA1; CARD10; CASP7; CCDC159; CCDC40; CCDC88C; CCN5; CCT7; CD276; CD59; CDC34; CDS1; CELSR1; CEP89; CFAP45; CHD3; CHD6; CHD8; CHPT1; CHST12; CIC; CITED2; CLN8; CMTR2; CNOT3; COMT; COMTD1; COQ8A; CORO2A; CRACR2A; CREBRF; CRKL; CRNDE; CRPPA; CSNK1G2; CTSB; CYB561; CYLD; CYP1B1; CYP1B1-AS1; DAAM1; DDA1; DDX47; DENND2D; DGAT2; DLC1; DLG5; DMXL1; DNAH11; DNASE1; DNMBP; DNPEP; DOCK4; DOK7; DOP1B; DRAIC; DSCAM-AS1; DYNLT3; ECH1; EHD1; EHF; EHMT1; ELAPOR1; EML6; ENC1; ENSA; ENSG00000224934; ENSG00000267288; ENSG00000268858; ENSG00000275765; EPB41L1; EPG5; EPHA1; EPHB4; EPN2; EPN3; EPS8; ERBB2; ERCC1; ERGIC1; ESRP2; ETNK2; EVI5L; EVL; EXOSC10; FAAP24; FAF1; FAM107B; FAM117A; FAM126A; FAM174B; FAM234B; FAM86EP; FBRS; FBXO38; FHDC1; FMN1; FNIP2; FOXN3; FREM2; FRMD4B; FRMD6; FRYL; G3BP1; GALNT10; GATAD2A; GATAD2B; GCLM; GCOM1; GIGYF2; GLRX5; GOLPH3L; GOT1; GPNMB; GPR160; GPRC5A; GRAMD1A; GREB1; GRHL1; GRHL2; GSPT1; GTPBP3; HEBP2; HERC1; HK1; HMGCS1; HNRNPC; HNRNPL; HOMEZ; HSH2D; HSPB8; ID1; IDE; IFITM10; IFRD1; IFT27; IGF1R; IGSF3; IGSF9; IKBKG; IKZF2; INPP4B; IQCD; IQSEC1; ISG20; JUND; JUP; KAT7; KATNB1; KCNK1; KCTD11; KDM4B; KDM5A; KHSRP; KIAA0100; KIAA0895; KIAA1217; KIF21A; KLF6; KLHL25; KRT7; KRT8; KRT80; LAD1; LAMB1; LCOR; LDLRAD4; LENG1; LFNG; LIMA1; LINC00511; LINC00963; LINC01588; LLGL2; LMCD1; LNC-LBCS; LPP; LRIG2; LRIG3; LRRFIP1; MACC1; MAF1; MANEAL; MAP3K3; MAP4K3-DT; MAPK10; MAPKAP1; MCCC2; MCF2L; MCRIP2; MEF2A; MGRN1; MICOS10; MIDEAS; MIDN; MIOS; MIR9-3HG; MIR99AHG; MIX23; MLLT3; MPHOSPH10; MSMO1; MTA1; MTMR12; MTMR4; MTUS1; MUC5B; NAALADL2; NADK; NAGK; NAIF1; NCAM2; NCOA1; NCOA4; NDRG1; NDUFAF8; NFATC2; NFKBIA; NFKBIZ; NFYB; NINL; NIPSNAP1; NIPSNAP2; NLK; NMT1; NOP14-AS1; NR1D1; NRP1; NT5DC3; NUDT16; NUMA1; OSBPL10; OSBPL2; OSBPL8; OVOL2; OXR1; P4HB; PABPN1; PAIP2B; PAK4; PAM; PARD3; PBX1; PCBP1-AS1; PCDH1; PCTP; PDGFB; PDGFD; PDZK1; PGP; PHTF2; PIK3C2G; PKIB; PKP4-AS1; PLA2G6; PLEKHF2; PNKD; PNPLA7; POLR3K; POU2F3; PPARD; PPM1H; PPM1L; PRADC1; PREX1; PRKCD; PRKCH; PROSER1; PROSER2; PROSER3; PRR15L; PRSS22; PRSS27; PSCA; PSMG3; PTPRH; PUM2; PWWP2A; PXDN; PXMP4; PYM1; PYROXD2; RAB26; RAB5B; RABL3; RAD51B; RANBP2; RAPGEFL1; RARA; RBBP8; RBBP8NL; RBL2; RBM20; RBM47; RC3H1; RECQL5; RFFL; RFX1; RGMB; RGMB-AS1; RGS3; RHOBTB3; RHOC; RNF19A; RNF207; RNF224; RPL27A; RPS6KA2; RPS6KL1; RTN4; S100A10; S100A16; S100A4; SAMD8; SCAMP4; SCGB1D2; SCIN; SCNN1A; SCRIB; SEC14L2; SEPTIN9; SERPINA1; SGMS2; SHARPIN; SIDT2; SKA2; SKIL; SLC16A3; SLC1A5; SLC25A25; SLC25A36; SLC27A2; SLC29A1; SLC35E1; SLC38A1; SLC38A2; SLC39A1; SLC44A2; SLC45A4; SLC6A6; SLC7A2; SLC9A8; SMAD3; SMARCD2; SNX9; SPECC1L; SPNS2; SPRED2; SPRYD3; SPTBN4; SPTLC2; SPTSSB; SRI; SRPK1; SRRM2-AS1; SSNA1; ST6GAL1; ST6GALNAC4; STARD10; STAT3; SUCLA2; SULT1A1; SYBU; SYT8; TACC2; TBC1D16; TBC1D8; TBCD; TBX2; TEDC1; TEPSIN; TFAP2C; TFF1; TGFBI; THADA; THSD4; TIPARP; TJP1; TLE1; TMBIM1; TMC1; TMCO3; TMED3; TMEM120A; TMEM160; TMEM184B; TMEM229B; TMEM241; TMEM60; TMEM94; TMPRSS4; TMTC2; TNK2; TNS1; TOM1L2; TOP2A; TPD52L1; TPD52L2; TPRN; TRAF7; TRAPPC9; TRIB1; TRIM29; TSPAN9; TTC39A-AS1; TTLL5; TUBA1A; TUBA1C; TUBB4B; UBE2C; UNC13D; UNKL; UPF1; URB1; USF3; UTRN; VAMP5; VAV3; VEPH1; VIRMA; VMP1; VPS26C; WDR90; WIPI1; WWC1; WWP1; XRCC3; YIF1A; ZDHHC1; ZFYVE21; ZFYVE28; ZKSCAN1; ZMYM2; ZMYND8; ZNF444; ZNF552; ZNF669; ZNF687; ZNF689; ZNF750; ZNF823; ZNRF1; ZSWIM1"},"children":["ABCA3; ABCC11; ABHD17C; ABHD2; ADAP1; ADGRG1; ADIRF-AS1; AFAP1; AFF1; AGAP2; AHCYL2; AKAP1; AKAP1-DT; AKAP8L; AKAP9; ALKBH7; AMOTL2; AMZ1; ANAPC2; ..."]},{"name":"span","attribs":{"title":"ABCA3; ABHD2; ACOX2; ADD3; AFF1; AGR2; AMFR; ANXA9; AREG; ARL3; BAG1; BATF; BCL2; BLVRB; CA12; CA2; CACNA2D2; CCN5; CD44; CD9; CDC6; CDH1; CELSR2; CHPT1; CHST8; COX6C; CPE; DCXR; DHCR7; DHRS2; DLG5; DNAJC1; DYNLT3; ELOVL5; EMP2; ETFB; FABP5; FAM102A; FARP1; FDFT1; FGFR3; FKBP5; FLNB; FOS; FOXC1; FRK; GALE; GFUS; GINS2; GLA; HMGCS2; HOMER2; HPRT1; HR; HSPB8; IDH2; IGFBP4; IL17RB; IL6ST; IMPA2; ISG20; ITPK1; JAK1; JAK2; KCNK5; KIF20A; KLF4; KRT19; LAMC2; LARGE1; LLGL2; LSR; MAPK13; MAPT; MICB; MOCS2; MYB; MYOF; NBL1; NCOR2; NRIP1; OPN3; OVOL2; PAPSS2; PDCD4; PDZK1; PGR; PLAAT3; PPIF; PRKAR2B; PRLR; PTGES; RAB31; RABEP1; RAPGEFL1; RBBP8; RPS6KA2; S100A9; SCARB1; SCNN1A; SCUBE2; SERPINA1; SFN; SIAH2; SLC16A1; SLC1A4; SLC22A5; SLC24A3; SLC26A2; SLC27A2; SLC29A1; SLC7A5; SLC9A3R1; SNX10; SORD; ST14; ST6GALNAC2; SULT2B1; TFAP2C; TFF1; TFF3; TFPI2; TIAM1; TJP3; TOB1; TOP2A; TPBG; TPD52L1; TRIM29; TSPAN13; UGDH; UNC13B; WFS1; XBP1; XRCC3; ZFP36"},"children":["ABCA3; ABHD2; ACOX2; ADD3; AFF1; AGR2; AMFR; ANXA9; AREG; ARL3; BAG1; BATF; BCL2; BLVRB; CA12; CA2; CACNA2D2; CCN5; CD44; CD9; CDC6; CDH1; CELSR2; ..."]},{"name":"span","attribs":{"title":"ABCC12; ABHD12; ACTG1; ADAT2; ADSS1; AFAP1; AFDN; AGFG1; AGRN; AHCY; AHNAK; ALDOA; ALG10; ALKBH2; AMPD3; ANKAR; ANKRD40; ANXA11; AP4E1; AP5B1; APBA2; ARHGEF1; ARL8B; ATP8B3; BAHCC1; BAIAP3; BCL2; BCOR; BCR; BMP1; BTBD11; CACNA1D; CALM1; CAMK2B; CASKIN2; CASP7; CCDC159; CCDC59; CCDC88C; CDC42EP3; CDCA2; CDK5RAP3; CELSR1; CFDP1; CHCHD4; CHEK2; CIDECP1; CLASP1; CLEC16A; CNNM2; CNTNAP2; COG2; COL9A2; CPLX1; CTBP2; CTNNA1; CTSB; CUEDC1; CYB561; DDX47; DLG3; DMAP1; DMTN; DRAIC; DUSP5; E2F6; EFHD1; EGFR; EHD1; ELAVL2; EML1; ENC1; ENSG00000215154; ENTPD8; EPHB6; ETV4; EXD3; FAM189A2; FAM83F; FANK1; FBXL8; FBXO27; FCSK; FGFR1; FGFR3; FOXA1; FOXP2; FRA10AC1; FRMD4B; FZD1; GFM2; GNG4; GPNMB; GRB10; GRHL3; GRIN1; GSE1; GSTCD; HECW2; HIVEP2; HIVEP3; HMG20A; HNMT; HOXC6; HOXC9; IKBKG; INTS12; IRS1; JAG1; KANSL3; KEAP1; KIF12; LFNG; LHX4; LIG1; LIMCH1; LMNA; LPIN1; LRATD1; LRIG3; LTBP4; MAILR; MAML3; MAP2; MBOAT7; MCF2L; METTL25; MID1IP1; MIR9-3HG; MRPL1; MRPL13; MRPL39; MTBP; MYC; MYO5B; MYOF; NDST1; NEDD4; NET1; NF1; NFKBIA; NIBAN1; NIBAN2; NIPAL1; NOXA1; NRARP; NSA2; NT5DC3; NTRK3; NUAK1; NUDT9; NUMA1; OSGIN1; OSMR; PAFAH2; PBX1; PDE5A; PDHX; PDZK1; PLEKHG4B; PLOD2; PNRC1; POLR2B; POU2F3; PPCDC; PPP2CB; PRICKLE2; PRKAB2; PRKCE; PTCH1; PTPRH; PTPRM; PWWP2B; RAP1GAP; RARA; RARS1; RCAN1; RFFL; RHOBTB3; RHOD; RHOT1; RIPOR3; RNF166; ROR2; RPS7; RRAGD; RUNX1; S100A14; SEC62; SGK3; SH3TC1; SHANK2; SLC22A5; SLC2A4RG; SLC45A4; SMAD7; SPRED2; SPRED3; SRCIN1; SRRM3; SSBP1; STARD10; STAT1; STAU2; STXBP1; STXBP6; SUCLG1; SYNE2; TBC1D1; TBL3; TBPL1; TCF7; TFDP2; TFPI2; TGFBI; THADA; THRB; TMEM248; TMTC2; TNNI2; TNS3; TPD52; TRAF4; TRAPPC3; TRIB2; TRIM47; TTC6; TXNRD2; UBE2E2; USP3; WNT5A; YAP1; ZBTB10; ZC3H12A; ZMIZ1; ZNF3; ZNF337; ZNF609; ZNF827"},"children":["ABCC12; ABHD12; ACTG1; ADAT2; ADSS1; AFAP1; AFDN; AGFG1; AGRN; AHCY; AHNAK; ALDOA; ALG10; ALKBH2; AMPD3; ANKAR; ANKRD40; ANXA11; AP4E1; AP5B1; APBA..."]}]}],"filterable":true,"showPageSizeOptions":true,"dataKey":"54400197a9f64e173b1d8d08ed78fc0c"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div id="genes-mapped-to-ar-only" class="section level4">
<h4>Genes Mapped to AR Only</h4>
<pre class="r"><code>goseq_t1only_res &lt;- tibble(
  gs_name = character(), mapped = numeric(), enriched = logical(),
  adj_p = numeric()
)
vec_t1only &lt;- structure(
  mapped_ids %in% setdiff(comp_ids[[1]], comp_ids[[2]]),
  names = mapped_ids
)
gs_method &lt;- &quot;Hypergeometric&quot;
any_goseq_t1only &lt;- plot_network_t1only &lt;- FALSE
para &lt;- glue(
  &quot;
  All {comma(length(setdiff(comp_ids[[1]], comp_ids[[2]])))} genes with a 
  mapped binding window to only {comps[[1]]} were compared to 
  the {comma(length(mapped_ids))} genes mapped to either
  {glue_collapse(comps, last = &#39; or &#39;)}, in order to ascertain whether any gene-sets 
  were likely to be targeted exclusively. No sampling bias offset was 
  incorporated, instead using a simple Hypergeometric model for enrichment testing.
  &quot;
)</code></pre>
<pre class="r"><code>pwf_t1only &lt;- nullp(
  vec_t1only,
  bias.data = log10(setNames(all_ids$w, all_ids$gene_id)[mapped_ids]), 
  plot.fit = TRUE
) </code></pre>
<pre class="r"><code>if (sum(pwf_t1only$DEgenes) &gt; 0) {
  goseq_t1only_res &lt;- pwf_t1only %&gt;% 
    goseq(gene2cat = gs_by_geneid, method = gs_method) %&gt;% 
    dplyr::filter(numDEInCat &gt; 0) %&gt;% 
    as_tibble() %&gt;% 
    dplyr::select(
      gs_name = category, pval = over_represented_pvalue, starts_with(&quot;num&quot;)
    ) %&gt;% 
    mutate(
      adj_p = p.adjust(pval, adj_method),
      enriched = adj_p &lt; enrich_alpha
    ) %&gt;% 
    dplyr::filter(numDEInCat &gt;= min_sig) %&gt;% 
    left_join(
      dplyr::filter(msigdb, gene_id %in% mapped_ids)[c(&quot;gs_name&quot;, &quot;gene_name&quot;, &quot;gs_url&quot;, &quot;gs_description&quot;)],
      by = &quot;gs_name&quot;
    ) %&gt;% 
    chop(gene_name) %&gt;% 
    mutate(gene_name = vapply(gene_name, paste, character(1), collapse = &quot;; &quot;))
}
any_goseq_t1only &lt;- sum(goseq_t1only_res$adj_p &lt; enrich_alpha) &gt; 0
tg_t1only &lt;- make_tbl_graph(
  goseq_t1only_res, 
  gs = gs_by_gsid %&gt;% 
    lapply(intersect, rownames(subset(pwf_t1only, DEgenes))) %&gt;% 
    .[vapply(., length, integer(1)) &gt; 0]
)
plot_network_t1only &lt;- length(tg_t1only) &gt;= min_network_size
if (!any_goseq_t1only) para &lt;- glue(para, &quot; No enrichment was found.&quot;)</code></pre>
<p>All 3,087 genes with a mapped binding window to only AR were compared
to the 7,631 genes mapped to either AR or ER, in order to ascertain
whether any gene-sets were likely to be targeted exclusively. No
sampling bias offset was incorporated, instead using a simple
Hypergeometric model for enrichment testing. No enrichment was
found.</p>
</div>
<div id="genes-mapped-to-er-only" class="section level4">
<h4>Genes Mapped to ER Only</h4>
<pre class="r"><code>goseq_t2only_res &lt;- tibble(
  gs_name = character(), mapped = numeric(), enriched = logical(),
  adj_p = numeric()
)
vec_t2only &lt;- structure(
  mapped_ids %in% setdiff(comp_ids[[2]], comp_ids[[1]]),
  names = mapped_ids
)
gs_method &lt;- &quot;Hypergeometric&quot;
para &lt;- glue(
  &quot;
  All {comma(length(setdiff(comp_ids[[2]], comp_ids[[1]])))} genes with a 
  mapped binding window to only {comps[[2]]} were compared to 
  the {comma(length(mapped_ids))} genes mapped to either 
  {glue_collapse(comps, last = &#39; or &#39;)}, in order to ascertain whether any gene-sets 
  were likely to be targeted exclusively. No sampling bias offset was 
  incorporated, instead using a simple Hypergeometric model for enrichment testing.
  &quot;
)
any_goseq_t2only &lt;- plot_network_t2only &lt;- FALSE</code></pre>
<pre class="r"><code>pwf_t2only &lt;- nullp(
  vec_t2only,
  bias.data = log10(setNames(all_ids$w, all_ids$gene_id)[mapped_ids]), 
  plot.fit = TRUE
) </code></pre>
<pre class="r"><code>if (sum(pwf_t2only$DEgenes) &gt; 0) {
  goseq_t2only_res &lt;- pwf_t2only %&gt;% 
    goseq(gene2cat = gs_by_geneid, method = gs_method) %&gt;% 
    dplyr::filter(numDEInCat &gt; 0) %&gt;% 
    as_tibble() %&gt;% 
    dplyr::select(
      gs_name = category, pval = over_represented_pvalue, starts_with(&quot;num&quot;)
    ) %&gt;% 
    mutate(
      adj_p = p.adjust(pval, adj_method),
      enriched = adj_p &lt; enrich_alpha
    ) %&gt;% 
    dplyr::filter(numDEInCat &gt;= min_sig) %&gt;% 
    left_join(
      dplyr::filter(msigdb, gene_id %in% mapped_ids)[c(&quot;gs_name&quot;, &quot;gene_name&quot;, &quot;gs_url&quot;, &quot;gs_description&quot;)],
      by = &quot;gs_name&quot;
    ) %&gt;% 
    chop(gene_name) %&gt;% 
    mutate(gene_name = vapply(gene_name, paste, character(1), collapse = &quot;; &quot;))
}
any_goseq_t2only &lt;- sum(goseq_t2only_res$adj_p &lt; enrich_alpha) &gt; 0
tg_t2only &lt;- make_tbl_graph(
  goseq_t2only_res, 
  gs = gs_by_gsid %&gt;% 
    lapply(intersect, rownames(subset(pwf_t2only, DEgenes))) %&gt;% 
    .[vapply(., length, integer(1)) &gt; 0]
)
plot_network_t2only &lt;- length(tg_t2only) &gt;= min_network_size
if (!any_goseq_t2only) para &lt;- glue(para, &quot; No enrichment was found.&quot;)</code></pre>
<pre class="r"><code>tbl &lt;- goseq_t2only_res %&gt;%
  dplyr::filter(enriched) %&gt;% 
  dplyr::select(
    gs_name, gs_description, pval, adj_p, starts_with(&quot;num&quot;), gene_name
  ) %&gt;% 
  reactable(
    filterable = TRUE,
    showPageSizeOptions = TRUE,
    columns = list2(
      gs_name = colDef(
        name = &quot;Gene Set&quot;,
        cell = function(value) htmltools::tags$a(
          href = gs_url[[value]], 
          target = &quot;_blank&quot;, 
          str_replace_all(value, &quot;_&quot;, &quot; &quot;)
        ),
        html = TRUE,
        maxWidth = 150
      ),
      gs_description = colDef(
        name = &quot;Description&quot;,
        cell = function(value) str_trunc(value, width = 150)
      ),
      pval = colDef(
        name = &quot;P-Value&quot;,
        cell = function(value) ifelse(
          value &lt; 0.001,
          sprintf(&quot;%.2e&quot;, value),
          sprintf(&quot;%.3f&quot;, value)
        ),
        maxWidth = 80
      ),
      adj_p = colDef(
        name = &quot;p&lt;sub&gt;adj&lt;/sub&gt;&quot;, html = TRUE,
        cell = function(value) ifelse(
          value &lt; 0.001,
          sprintf(&quot;%.2e&quot;, value),
          sprintf(&quot;%.3f&quot;, value)
        ),
        maxWidth = 80
      ),
      numDEInCat = colDef(
        name = glue(&quot;Genes Mapped to {comps[[2]]} Only&quot;),
        cell = function(value) comma(value, 1),
        maxWidth = 80
      ),
      numInCat = colDef(
        name = &quot;Genes Mapped to Both&quot;,
        cell = function(value) comma(value, 1),
        maxWidth = 80
      ),
      gene_name = colDef(
        name = &quot;Gene Names&quot;,
        cell = function(value) with_tooltip(value, width = 150)
      )
    )
  )
div(class = &quot;table&quot;,
    div(class = &quot;table-header&quot;, div(class = &quot;caption&quot;, para)),
    tbl
)</code></pre>
<div class="table">
<div class="table-header">
<div class="caption">All 1,327 genes with a 
mapped binding window to only ER were compared to 
the 7,631 genes mapped to either 
AR or ER, in order to ascertain whether any gene-sets 
were likely to be targeted exclusively. No sampling bias offset was 
incorporated, instead using a simple Hypergeometric model for enrichment testing.</div>
</div>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-269363b56334f380e5d2" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-269363b56334f380e5d2">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"gs_name":["KAT2A_TARGET_GENES","ZNF766_TARGET_GENES","E2F5_TARGET_GENES","SETD7_TARGET_GENES","ZNF197_TARGET_GENES"],"gs_description":["Genes containing one or more binding sites for UniProt:Q92830 (KAT2A) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Genes containing one or more binding sites for UniProt:Q5HY98 (ZNF766) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Genes containing one or more binding sites for UniProt:Q15329 (E2F5) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Genes containing one or more binding sites for UniProt:Q8WTS6 (SETD7) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Genes containing one or more binding sites for UniProt:O14709 (ZNF197) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization."],"pval":[2.01935016435482e-08,3.24798698602883e-07,8.61851662452235e-07,1.88770345557329e-06,2.09982154761343e-06],"adj_p":[4.63238927702996e-05,0.000372544107297506,0.000659029237888476,0.000963398126045043,0.000963398126045043],"numDEInCat":[159,155,157,106,94],"numInCat":[615,620,639,401,346],"gene_name":["A2M-AS1; AAGAB; ABHD2; ACP2; ACTG1; ADAM17; ADAT2; ADNP; AFF4; AFMID; AGK; AGPAT2; AGPS; AHCYL2; AKAP8L; AKIRIN2; ALAS1; ALDH7A1; ALDOA; ALG10; ALG8; AMBRA1; ANAPC2; ANAPC5; ANG; ANO10; AP4M1; APTX; ARF6; ARHGEF1; ARID4B; ARL3; ARMC6; ARV1; ATAD2; ATP1A1-AS1; ATP2B1; ATP5MF; ATPAF1; ATXN10; AUNIP; B3GALT4; B4GAT1; BCAR3; BCAT2; BLOC1S6; BOD1; BOLA1; BRD2; BRD3OS; BSG; C11orf58; C11orf68; C14orf93; C16orf91; C17orf75; C18orf25; C19orf44; C19orf48; C1orf159; CAB39; CALCOCO1; CALM1; CAP1; CARS2; CASP7; CCDC47; CCDC57; CCDC59; CCDC6; CCNG2; CCNL1; CCT7; CDC123; CDC42EP4; CDC45; CDC7; CDCA2; CDCA3; CDK4; CDK5; CDK5RAP2; CDK8; CEP192; CEP290; CEP95; CEPT1; CFAP418; CHCHD7; CHD8; CHERP; CHMP4B; CHSY1; CHTOP; CIAO2A; CIAO3; CIPC; CLASP1; CLCN7; CLEC16A; CLPB; CLPTM1L; CLTC; CLUH; CNOT4; COA8; COMT; COPS2; COX19; CRTC2; CSNK2B; CSRNP2; CTNNA1; CTNNBL1; CWC27; CYB561; CYB5D1; CYB5D2; CYCS; CYLD; CYP51A1; DCAF8; DCAKD; DCLRE1A; DCP1A; DCTN6; DDX23; DDX47; DDX5; DFFA; DHDDS; DHX8; DHX9; DICER1; DICER1-AS1; DNAAF5; DNAH7; DNAJA1; DNAJB6; DNAJC17; DNAJC19; DPH6; DPP8; DRAM2; DRAP1; DTWD1; DUS1L; DVL3; E2F3; E2F6; EHD1; EIF1B; EIF1B-AS1; EIF2B1; EIF3B; EIF3E; EIF3K; EIF4A2; EIF4E; EIF4E2; EIF4ENIF1; EIF4G3; ELAC1; ELMOD3; ELP3; EMC3; ENAH; ENSG00000254837; ENSG00000255468; ENSG00000268403; ENY2; EPB41L4A-AS1; EPCAM; ERAL1; ERF; ERRFI1; ERVK3-1; ESYT1; EVI5L; EXOSC3; FAAP100; FAF1; FAM168A; FAM222B; FAM229B; FANCI; FBF1; FBXL19; FBXL19-AS1; FBXO15; FBXO22; FBXO36; FBXO5; FBXW11; FEM1A; FGD5-AS1; FIZ1; FLOT1; FMO5; FOXK2; FUS; GAA; GABARAP; GABPB1; GALE; GALK2; GATB; GCFC2; GCN1; GDI2; GFM2; GGA1; GGCX; GGPS1; GLO1; GLT8D1; GNL3; GORASP2; GPANK1; GPBP1L1; GRK4; GSPT1; GSTCD; GTF2H3; GTF2I; GTPBP3; GUCD1; GUSB; H2AC20; H2AC7; H2BC21; H2BC5; H3-3B; H4C8; HAUS6; HEBP2; HIF1AN; HLA-DMA; HNRNPA0; HNRNPDL; HNRNPLL; HP1BP3; HSD11B2; HSP90AA1; HSP90B1; HSPD1; HSPE1; HUS1; ICMT; ID2-AS1; IER3; IER5; IFRD2; IFT20; IFTAP; IGHMBP2; ILF2; IMP3; INTS12; IPO13; IQCK; ISG20L2; ITGB3BP; JMJD1C; JMJD4; KATNBL1; KBTBD7; KCTD5; KCTD9; KITLG; KMT2A; LAMTOR2; LAMTOR5; LAMTOR5-AS1; LARS1; LDHA; LIAS; LIMS1; LINC00337; LINC00886; LINC01137; LIPT2; LPXN; LRPAP1; LYSMD1; LZIC; MAGOHB; MAN1A2; MANBAL; MAP3K3; MARK4; MAX; MAZ; MBTPS2; MCEE; MCL1; MCM3AP; MCM7; MCMBP; MCOLN1; MCU; MDH1; MED15; MED22; MED23; MED4; METTL15; METTL25; METTL25B; METTL9; MIB2; MIDN; MMACHC; MNAT1; MORF4L1; MPHOSPH10; MRFAP1; MRFAP1L1; MRPL19; MRPL38; MRPL39; MRPS12; MRPS27; MRPS28; MRPS33; MRTO4; MTCH1; MTERF4; MTIF2; MTMR9; MTOR; N4BP3; NAA38; NAIF1; NCOA7; NDC1; NDUFA12; NDUFA3; NDUFC2; NEAT1; NELFB; NFAT5; NFE2L2; NFYB; NHLRC2; NMNAT1; NMT1; NOP14; NOP16; NPLOC4; NSA2; NSUN2; NTAN1; NUDCD1; NUDT9; NUMB; NUP107; ORMDL3; OTUD7B; PABPC1; PABPN1; PANK4; PARP4; PBRM1; PCID2; PCLAF; PCYT1A; PDCD10; PDCD6IP; PDE4D; PDIA5; PEMT; PEX2; PFN2; PGS1; PHAF1; PHB; PHF12; PHF3; PIGQ; PIM3; PITHD1; PKM; PLEKHA8; POLA1; POLD3; POLDIP3; POLG; POLR3E; PPIL3; PPP1R12C; PPP2CB; PPP2R2A; PPP2R5B; PPP4R3A; PPP5C; PPRC1; PRKAB2; PRKDC; PRMT5; PRORP; PROSER1; PRPSAP1; PRR34-AS1; PSMD12; PTCD1; PUM2; RAD52; RALBP1; RANBP10; RANGAP1; RBL1; RBM26; RBM26-AS1; RBM28; RCC1; RETSAT; REXO1; RIC8B; RNASEH1; RNASEH1-AS1; RNF121; RNF135; RNF213-AS1; RNF216; RNF216P1; RNF220; RNPS1; RPIA; RPL18; RPL3; RPL31; RPL34; RPL36; RPL38; RPL41; RPL6; RPL7A; RPL9; RPP14; RPS12; RPS13; RPS2; RPS20; RPS3; RPS7; RRP7A; RSAD1; S100PBP; SARS2; SBNO2; SCAF1; SCNM1; SCRIB; SDC4; SEC22C; SENP5; SEPTIN7; SEPTIN9; SERP1; SERPINI1; SERTAD2; SF1; SF3B4; SFXN2; SIL1; SKP1; SLC25A25; SLC35A3; SLC35G1; SLC38A10; SLC44A1; SLC4A2; SLC9A1; SLC9A3R1; SMAD5; SMARCC2; SMARCD2; SMG7; SNAP29; SNAP47; SNHG3; SNHG9; SNORD3A; SNRNP27; SNRPD3; SNX1; SNX19; SNX27; SNX8; SPCS1; SPHK2; SPRED1; SRD5A1; SRI; SS18L2; SSNA1; ST3GAL2; STAT3; STAU1; STK17A; STMP1; STRIP1; SUGP2; SYPL1; TAMM41; TBC1D30; TBC1D31; TBC1D8; TBCB; TCEA2; TDRD3; TEDC1; THADA; THRAP3; TIGD1; TIMM21; TIMM23; TIMM23B; TIMM44; TK1; TM2D1; TMA16; TMED10; TMED2; TMEM11; TMEM147-AS1; TMEM198B; TMEM69; TMSB10; TMTC3; TNFAIP1; TOR1AIP1; TRABD; TRAM2; TRAM2-AS1; TRAP1; TRERF1; TRIM33; TRIP11; TRIP12; TSEN15; TSN; TSSC4; TTC13; TTC33; TUBA1C; TUBB; TUBE1; TUBGCP3; TXNDC11; TXNRD2; TYW5; UBAC2; UBE2I; UBQLN4; UBR2; UCKL1; UFD1; UHMK1; UNK; UPF1; USP21; USP24; USP30; USP39; VAMP4; VAMP8; VCPIP1; VEGFA; VEZF1; VMP1; VPS13B; VPS13B-DT; VTI1A; WAC; WAC-AS1; WARS1; WDFY1; WDPCP; WDR25; WDR27; WNK1; WRAP53; XPC; YARS1; YBEY; YTHDF1; ZBTB1; ZBTB5; ZC3H10; ZC3H12A; ZC3H15; ZCCHC8; ZDHHC6; ZFAND3; ZFP91; ZFYVE1; ZFYVE19; ZMPSTE24; ZMPSTE24-DT; ZMYM4; ZMYND11; ZNF143; ZNF195; ZNF3; ZNF337; ZNF37A; ZNF524; ZNF581; ZNF691; ZNF706; ZNF76; ZNHIT6; ZZEF1","AAMP; ACP2; ACTB; ACTG1; ACTR1A; ACTR3; ACTR3B; ADAM9; ADAMTSL4; AHCTF1; AHCYL2; AHSA2P; AKAP8; AKIRIN2; ALDH5A1; ALMS1; AMMECR1; AMN1; AMOTL1; ANKMY1; ANKS6; ANTXR2; ANXA2; AP1S2; AP2M1; AP2S1; APOO; AREG; ARFRP1; ARHGAP5; ARHGAP5-AS1; ARHGEF12; ARID4B; ARL4A; ARNTL; ARPC1A; ARV1; ASAP3; ASNSD1; ASXL2; ATG10; ATG4B; ATP5PF; ATXN7; AUP1; B4GALT2; B4GAT1; BABAM1; BANP; BATF; BCKDK; BICD2; BIRC5; BMERB1; BOD1; BORCS7; BTN3A2; BTRC; BUB3; C11orf58; C15orf61; C17orf58; C19orf44; C1orf159; CABIN1; CACNA1D; CALCOCO1; CAMP; CAP1; CAPG; CAST; CBFA2T3; CBX7; CCAR1; CCDC12; CCDC137; CCDC71; CCDC90B; CCP110; CDC73; CDCA3; CDH23; CDK5; CDK8; CDYL; CENPT; CEP290; CFL1; CHCHD2; CHD7; CHFR; CHIC2; CHMP4B; CIC; CITED2; CLEC16A; CLPB; CLSTN1; CLTC; CLUAP1; CNPY2; COMMD3; COPS7B; CORO1A; COX7A2L; COX7C; CPE; CRADD; CRAMP1; CRTC2; CRYM; CSNK2B; CSRNP2; CSRP1; CTDP1; CTSB; CYB561D2; CYB5D1; DCK; DDX23; DEDD; DENND1A; DFFA; DGAT1; DGAT2; DGCR8; DHRS3; DIP2A; DMD; DNAJA3; DNAJC17; DNALI1; DNM1L; DST; DUS1L; DUS3L; DUSP4; DYNC1I2; DYNC2I2; ECHDC2; EFNA3; EHD1; EHMT2; ELOC; EML2; EML6; ENSG00000233834; ENSG00000255468; ENY2; EPB41L5; EPC1; ERH; ERI3; ERMAP; ERP44; ERVK3-1; EVL; EXOC6; EYA3; FAF2; FAM117A; FAM13A; FAM168B; FAM193A; FAM222A; FASN; FASTKD1; FBXL16; FBXL17; FBXL19; FBXL3; FBXO15; FCSK; FGF13; FHL2; FLYWCH1; FN3K; FOXP1; FTO; FYCO1; G3BP2; GAB2; GALNT7; GAPVD1; GATAD2A; GATB; GDAP1; GDPD5; GFER; GFUS; GGPS1; GLUD1; GNG4; GPANK1; GPAT4; GPBP1; GPBP1L1; GPKOW; GPRC5C; GPT2; GRAMD4; GRK3; GRK4; GSPT1; GTF2I; GTF2IRD1; GTPBP2; GUSB; H4C8; H6PD; HAUS6; HECTD2; HERC1; HGD; HIKESHI; HM13; HMG20A; HSD11B1L; HSPA13; HSPB11; HSPH1; HTRA2; IFRD1; IFRD2; IFT140; IFT20; IFTAP; IGF1R; IKZF2; IL1R1; ILF3; ILF3-DT; IMPACT; ING3; INTS6; INVS; IQCH-AS1; IRF2BP2; KANSL3; KAT7; KBTBD6; KBTBD7; KCNB1; KCTD20; KCTD21; KDM7A; KDM7A-DT; KHDC4; KIF15; KLC1; KLHDC8B; KLHL21; KMT2B; LAMTOR2; LARP1; LARS1; LCORL; LEMD2; LEPROTL1; LIFR; LIG1; LINC-PINT; LINC00504; LINC01521; LLGL2; LMNB2; LNX1; LPXN; LSM10; LTA4H; LUC7L; LYPLA1; LYSMD1; MACC1; MAD2L1BP; MAF1; MALSU1; MAML3; MAN2A2; MANBAL; MAP4K3; MAP4K3-DT; MAT2A; MAX; MDH1; MED15; MFAP3L; MGME1; MIAT; MICOS13; MIDEAS; MIDN; MITF; MKKS; MLLT10; MMACHC; MPI; MPZL3; MR1; MRFAP1; MRGBP; MRPL24; MTIF2; MTSS1; MUS81; MUTYH; MYB; MYCL; MYO5B; MYO9A; NAA38; NAPA; NDUFAB1; NEAT1; NEDD4; NEK6; NF2; NFASC; NFE2L1; NFIX; NFKBIA; NFKBIB; NKAIN1; NOP14; NOP53; NORAD; NOSIP; NPEPPS; NPEPPSP1; NPL; NPRL2; NRDE2; NSMCE1-DT; NTAN1; NUCB1; NUDCD1; NUDT13; NUMA1; NUP205; NUP43; NUTF2; OPLAH; ORMDL3; OTUD7A; PA2G4; PABPN1; PACS1; PACS2; PACSIN2; PAFAH2; PANK4; PAPSS1; PATZ1; PCBP1-AS1; PCGF5; PCSK6; PDCD10; PDE3B; PDE6D; PDS5B; PEAK1; PEMT; PEX14; PEX26; PFDN6; PGBD5; PGM2L1; PGP; PHB; PHF12; PHF13; PISD; PLA2G6; PLCG2; PLEKHF2; PNKD; PNPLA6; PODXL2; POLDIP3; POLR1G; PPP1R12C; PPP1R13L; PPP1R15A; PPRC1; PRKCA; PRMT3; PROSER2; PRPF31; PRPSAP1; PRR5; PRRT2; PTCD3; PTK2; QRICH1; RAB1A; RAB27B; RAB7A; RABGAP1L; RALBP1; RANBP10; RANGAP1; RBBP4; RBM14; RBM25; RCAN1; RCBTB2; RERE; REXO2; RHOBTB3; RIC8B; RIMKLB; RIN3; RMC1; RNF145; RNFT2; RPGRIP1L; RPIA; RPL36; RPP40; RPRD1B; RPS13; RXRA; RYK; S100P; SAP30; SBK1; SCAF1; SCAMP4; SCNM1; SCRIB; SEC11C; SEC24C; SEC61A1; SENP8; SERPINI1; SETD5; SETMAR; SFMBT1; SFSWAP; SFT2D2; SFXN1; SHARPIN; SHLD2; SIPA1L3; SIRT2; SKIL; SLC16A1; SLC25A12; SLC25A6; SLC2A4RG; SLC35B4; SLC38A2; SLC39A11; SLC39A9; SLC44A1; SLC45A4; SLC48A1; SLC4A2; SLC9A3-AS1; SLK; SLMAP; SLX4IP; SMAD3; SMAP1; SMIM13; SMPD3; SNX10; SNX19; SNX5; SPEN; SPRING1; SQOR; SRCIN1; SRGAP3; ST3GAL2; STAG2; STAT5B; STAU1; STAU2; STK25; STK40; STMN3; STMP1; STPG1; STRIP1; STRN3; STXBP5; SULT1A2; SUMO3; SYMPK; SYNGR1; TANC1; TARBP2; TBC1D4; TBC1D8B; TBX6; TCEA2; TCERG1; TENT2; TFAP2A; TFEB; TFPT; THADA; THRAP3; THUMPD2; TIAL1; TIMM21; TJP3; TM7SF3; TM9SF4; TMBIM1; TMCC1; TMCC1-DT; TMCO1; TMCO1-AS1; TMEM11; TMEM115; TMEM160; TMEM181; TMEM245; TMEM69; TMEM70; TMTC1; TMTC3; TNFAIP1; TNS1; TOE1; TP53BP2; TPD52L1; TPGS1; TRAM2-AS1; TRIM29; TRIT1; TRMT12; TRMT44; TRPM7; TSC22D1; TSC22D2; TSSC4; TTC1; TTC13; TTC33; TTLL4; TUFT1; TXNDC15; UBE2E2; UBE2E3; UBE2Q2; UBE3C; UBQLN4; UBR2; UBR3; UBXN2A; UCP2; UNC45A; UPF1; UROS; USF1; USF2; USP31; USP32; USP35; USP48; USP6NL; VAPA; VDAC2; VMP1; VPS35L; VPS50; VRK1; WARS1; WARS2-AS1; WBP1; WBP2; WDPCP; WDR1; WDR24; WDR25; WDR46; WNK1; WSB1; WWP1; XIST; XKR9; XYLT2; YAP1; YEATS2; YIF1B; YIPF6; ZC3H7A; ZCCHC8; ZDHHC18; ZFP91; ZFYVE1; ZFYVE19; ZFYVE28; ZGPAT; ZMYM4; ZMYND8; ZNF181; ZNF207; ZNF217; ZNF221; ZNF223; ZNF225; ZNF32; ZNF331; ZNF337; ZNF37A; ZNF37BP; ZNF436; ZNF444; ZNF529; ZNF586; ZNF605; ZNF644; ZNF674-AS1; ZNF688; ZNF776; ZSCAN22","AAMP; ABCA5; ABRAXAS2; ACAD8; ACOX2; ACP2; ACTG1; ACTN4; ACTR1A; ADAM9; ADCY9; ADD1; ADNP; ADO; ADSS2; AFDN; AFMID; AGPS; AHCTF1; AHCYL2; AHI1; AHSA1; AKIRIN2; AKT2; ALDH5A1; ALDOA; ALG10; ALKBH8; AMMECR1; ANKMY1; ANKRD11; ANKS6; ANKZF1; ANP32E; ANXA5; AP2M1; AP2S1; AP4M1; APLF; ARHGAP5; ARHGAP5-AS1; ARHGDIA; ARHGEF2; ARMH3; ARNTL; ARPC2; ARV1; ASAP3; ASPH; ATAD2; ATG4B; ATG9A; ATP5MF; ATP7A; AUP1; B4GALT2; B4GALT3; BABAM1; BABAM2; BANP; BICDL1; BIRC5; BLM; BOD1; BRD3; BRD4; BRD9; BSG; BTBD1; BTBD3; BTRC; BUB1B; BUB3; C11orf58; C18orf54; C19orf44; C1orf109; C1orf159; C2orf69; CA12; CABIN1; CACTIN; CALCOCO1; CAMKK2; CASP8AP2; CBFA2T3; CCDC115; CCDC136; CCDC159; CCDC191; CCDC71; CCNA2; CCNB2; CCNG2; CCP110; CDC40; CDCA2; CDCA3; CDCA8; CDH23; CDK11A; CDK5; CDK5RAP2; CDYL; CECR2; CENPA; CENPO; CENPT; CEP128; CEP290; CEP295; CEP89; CFL1; CHCHD2; CHCHD6; CHD7; CHEK2; CHFR; CHIC2; CHMP4B; CHPT1; CIAO2A; CIDECP1; CLEC16A; CLINT1; CLSTN1; CLTC; CMIP; CMSS1; CNPY2; COMMD3; COPS7B; COX7B; COX7C; CRADD; CRAMP1; CRTC2; CSGALNACT2; CSNK2B; CTBP1; CTCF; CTDP1; CWC22; CYB561D2; DCAKD; DCK; DDIAS; DDX23; DEDD; DENND1A; DGCR8; DIAPH3; DLGAP4; DMXL2; DNAJC17; DNAL1; DPYSL2; DUS1L; DUSP14; DYNC1I2; E2F1; E2F2; E2F3; EEF2K; EFNA3; EHD1; ELOC; EMC3; EME1; EML6; ENSG00000215154; ENSG00000233834; ENSG00000234432; ENY2; EPC1; ERP44; ERVK3-1; EXOC6; FAAP24; FAM110A; FAM117A; FAM168B; FAM193A; FAM222A; FANCC; FANCD2; FANCI; FASN; FASTKD1; FBF1; FBRSL1; FBXL16; FBXL20; FBXL5; FBXO15; FBXO38; FBXO46; FBXO48; FBXO5; FBXW11; FCHSD2; FEM1A; FGFR1; FHL2; FN3K; FTO; FYCO1; GAB2; GATB; GBAP1; GDAP1; GFER; GFUS; GINS2; GLUD1; GNG4; GP6-AS1; GPANK1; GPAT4; GPBP1L1; GPT2; GRAMD4; GRHL1; GRK3; GRK4; GSPT1; GTF2IRD1; GTPBP2; GUCD1; H2AC11; H4C8; HAUS8; HDAC5; HECTD2; HERC1; HIF1A; HIKESHI; HINT3; HM13; HMG20A; HMGB1; HNRNPF; HOOK3; HSCB; HSPD1; HSPE1; HTRA2; IFRD1; IFRD2; IFT140; IFT80; IGF1R; IKZF2; IL1R1; IMPACT; ING3; INTS1; INVS; IPO13; IQGAP3; IRF2BP2; KAT7; KBTBD6; KBTBD7; KCNB1; KCNC3; KCNK1; KCTD20; KCTD9; KDM7A; KDM7A-DT; KIAA0232; KIF15; KIF18B; KIF20B; KIF22; KLF13; KMT2B; KPNA3; LAMTOR2; LARS1; LEMD2; LEPROTL1; LGALSL; LIFR; LIN9; LINC01521; LMNB1; LMNB2; LPIN2; LSM10; LUC7L; LUC7L2; LYSMD1; MACC1; MAD2L1BP; MAGOHB; MAN2A2; MANEAL; MAPT; MARS1; MAT2A; MAU2; MAX; MBTPS2; MCM2; MCM5; MCM7; MDH1; MDM2; MED15; MEGF11; METTL15; METTL4; MFAP3L; MGME1; MIDEAS; MIS18BP1; MITF; MKI67; MKKS; MMACHC; MOB4; MPI; MRGBP; MRPL27; MRPL52; MRPS25; MTMR4; MUS81; MYB; MYCL; MYO6; MYO9B; MZT1; NADK2; NCAPH; NCOA5; NDC80; NEDD4; NEK6; NF2; NFASC; NFE2L2; NFXL1; NKAIN1; NMT1; NOP14; NOP53; NOSIP; NPEPPS; NPEPPSP1; NPRL2; NRDE2; NSD2; NSMCE1-DT; NTAN1; NTHL1; NUCB1; NUDCD1; NUDT13; NUF2; NUMA1; NUTF2; OBSL1; OPLAH; ORMDL3; OSTF1; OTUD7A; PACS2; PAFAH2; PAN2; PANK4; PATZ1; PBK; PCBP1-AS1; PCED1B; PCSK6; PDE3B; PDE6D; PDIK1L; PDS5B; PEAK1; PEX26; PGP; PHB; PHF12; PHF13; PHF21A; PHLPP1; PIAS1; PLEKHF2; PNKD; PNPLA6; PODXL2; POLE2; PPCDC; PPM1E; PPP1R12C; PPP1R15A; PRCP; PRKCA; PRKCE; PRKD1; PRKDC; PRMT3; PRR5; PTCD1; PTDSS2; PTMS; PTPRN2; PTRHD1; QRICH1; QTRT1; QTRT2; RAB30; RAB30-DT; RAB7A; RABEP1; RABGAP1L; RAD51; RAD51-AS1; RAD51B; RALBP1; RANBP10; RANGAP1; RBBP4; RBKS; RBL1; RBM6; RCCD1; RERE; REXO2; RFC3; RFC4; RHOBTB3; RIC8B; RIN3; RMDN3; RNF170; RNF213; RNF227; RPF1; RPGRIP1L; RPH3AL; RPIA; RPL23A; RPP40; RPS13; RPS20; RRM1; RTEL1; RTEL1-TNFRSF6B; RTKN2; RXRA; RYK; SAP30; SASS6; SAV1; SBK1; SCAF1; SCAMP5; SCNM1; SCRIB; SDK1; SEC14L1; SEC24C; SEC61A1; SEL1L; SETMAR; SGPL1; SHLD2; SIAH1; SIK3; SLC35A1; SLC35F2; SLC38A1; SLC44A1; SLC45A4; SLC48A1; SLC4A2; SLC6A6; SLK; SLX4IP; SMARCAL1; SMC6; SMIM30; SMPD3; SMUG1; SNAPC3; SNIP1; SNRPD3; SNX1; SNX5; SPC25; SPG11; SPTBN4; SRCIN1; SRRM3; SRSF1; ST3GAL2; ST7; STAT1; STAT3; STAT5B; STAU1; STK25; STMN1; STMN3; STN1; STON2; STRADB; STRIP1; SUGP1; SUMO3; SYMPK; SYNGR1; TACC1; TBC1D31; TBC1D4; TBC1D8B; TBX6; TCEA2; TCP11L1; TEDC2; TFAP2A; TFEB; TGFBR3; THAP1; THAP12; THUMPD2; THYN1; TIAL1; TIMM21; TIMM22; TK1; TKT; TM9SF4; TMBIM1; TMEM11; TMEM115; TMEM131; TMEM150A; TMEM160; TMEM181; TMEM33; TMEM69; TMEM70; TMTC3; TNRC6B; TOP2A; TOR2A; TP53; TP53BP2; TPGS1; TPRA1; TPX2; TRAK2; TRAPPC3; TRIM29; TRIP13; TSC2; TTC13; TTC33; TTLL4; TUBGCP3; TXNDC15; UBE2E2; UBE2E3; UBE2G2; UBE2Q1; UBE2Q2; UBE3C; UBQLN4; UBR2; UBR3; UBXN2A; UCKL1; UNG; UPF1; USF2; USP28; USP3; USP31; USP39; USP48; USP6NL; VAPA; VCPIP1; VDAC2; VIPAS39; VKORC1; VMP1; VPS35L; VPS50; VRK1; VRK3; WASF1; WBP2; WDPCP; WDR13; WNK1; WRAP53; WT1-AS; WWP1; XIST; XRCC3; XYLT2; YEATS2; ZBTB11; ZBTB46; ZFX; ZFYVE1; ZFYVE19; ZFYVE21; ZFYVE28; ZNF133; ZNF155; ZNF217; ZNF225; ZNF280C; ZNF280D; ZNF32; ZNF331; ZNF337; ZNF354A; ZNF37A; ZNF436; ZNF444; ZNF451; ZNF473; ZNF573; ZNF589; ZNF605; ZNF687; ZNF785; ZW10; ZWINT; ZYG11A","ACOT13; ACSL3; ACTB; ACTG1; ACTN4; ACYP2; ADAP1; ADI1; AHCY; AKAP8; ALDOA; AMZ2; ANP32B; AP4M1; ARFGAP2; ARID1B; ARPC1A; ARPC1B; ATL2; ATP1A1; ATP5MC3; AVIL; AXIN2; B3GALT4; B3GALT9; B4GAT1; BCL2L1; BCL2L2; BCL9L; BHLHE40; BICRA; BOLA1; BRD2; BRICD5; BSG; BTF3; BUB1; C14orf93; C19orf48; CAPN15; CBX3; CCDC106; CCDC34; CCNL1; CDC42EP3; CDC6; CENPJ; CERS5; CFL1; CHASERR; CHD2; CHD8; CHERP; CIRBP; CLASP1; CLTC; CNOT3; CNP; COX19; CSK; CSNK1G1; CSNK2B; CSRNP2; CYB5R2; CYRIB; DANCR; DAZAP2; DCP1A; DCP1B; DDIT4; DDX3X; DECR2; DENND2B; DEPDC5; DHX9; DNASE1; DPY19L3; DPYSL2; DYNLL1; DYNLL2; DYRK1B; EDC4; EEF1D; EFCAB2; EFHC1; EIF1; EIF4A2; EIF4A3; EIF5; ELF2; EME2; ENSG00000215154; ENSG00000242622; ENSG00000255468; ENSG00000268403; EPHA1; EPM2AIP1; EPN1; ERCC1; ESYT1; FAM174C; FAM200A; FAM20A; FASN; FBXL14; FBXW2; FER; FGFR1; FLAD1; FLOT1; FTSJ3; FTX; FUT11; GABARAP; GABPB1; GALE; GAPDH; GARS1-DT; GCN1; GDPD3; GEMIN6; GLUL; GPANK1; GPATCH4; GPX4; GRB2; GRWD1; GSK3B; H1-4; H2AC11; H2AC12; H2AC20; H2AC21; H2AC6; H2AC7; H2AC8; H2AZ1; H2BC10; H2BC18; H2BC21; H2BC4; H3-3B; H3C6; H4C8; HDAC4; HERC1; HEXIM1; HEXIM2; HIF1A; HMGA1; HMGB1; HNRNPA2B1; HNRNPC; HNRNPF; HSP90AA1; HSPA5; HUS1; HUWE1; IAH1; ID4; IER5L; IL17RD; ING1; ING5; INO80; INTS10; ISG20L2; JMJD6; JMJD8; KAT8; KCNC3; KCTD15; KDM8; KDSR; KIF22; KLC1; KLF7; KLHL35; KPNA2; LAMA5; LETM2; LINC01089; LINC01521; LRP2BP; LRP6; LSR; LUC7L3; LYRM2; MAP3K12; MAT2A; MAZ; MCL1; MCM7; MEIS1; METTL23; METTL25B; MFSD10; MFSD11; MIB2; MIDN; MLH1; MLLT10; MMACHC; MPHOSPH9; MRPL28; MRPS27; MRPS28; MRPS34; MRPS36; MTA1; MTHFS; MTRF1L; MYL12A; MYL6; NAA50; NCL; NEK8; NEU3; NFE2L3; NFKBIA; NINL; NIT1; NKIRAS1; NOP14; NOPCHAP1; NPM1; NR1H2; NT5C; NUDT17; OAZ2; OGA; OIP5-AS1; P4HA1; PACS1; PANX1; PATZ1; PCBP1-AS1; PCGF3; PCID2; PCIF1; PCNT; PDHB; PFDN2; PFKFB3; PFKP; PHACTR4; PHLDB1; PKM; PKP2; PLEKHG2; PLEKHN1; PNRC1; POLDIP3; POLG; POLG2; POLR2C; POU2F1; PPP4R3A; PRKAR1A; PRKD2; PRKRIP1; PRPF31; PSMB10; PSMC5; PSMD12; PUM2; PYGO2; QSER1; RAB1B; RAB26; RAB33B; RABGEF1; RBL1; RBM25; RBPMS; RCC1; RCC2; REEP3; REEP4; RFC4; RFX3; RHOF; RMI1; RND3; RPL14; RPL15; RPL22L1; RPL27A; RPL3; RPS14; RPS19; RPS2; RPS3A; RPS9; RSRC1; SCRIB; SEC31A; SELENOW; SETD5; SF1; SGF29; SH3D21; SIL1; SIRT7; SIX5; SLC25A13; SLC25A25-AS1; SLC25A36; SLC26A6; SLC35E2B; SLC38A2; SLC39A10; SLF2; SMAD7; SMG1P3; SMIM10L1; SNHG12; SNHG8; SNHG9; SP3; SPATS2L; SPNS2; SRCAP; SRRM2; SRRM2-AS1; SRSF2; SRSF6; SRSF7; SS18; SSU72; STK26; STMN1; STUB1; SUMF2; SYAP1; TAF15; TCAF1; TDP2; TERF1; THBS3; TIGD5; TLCD1; TMEM33; TMEM63A; TMSB4X; TOB1-AS1; TP53I11; TPI1; TPM1; TPM4; TPT1; TPT1-AS1; TRAF4; TRIM65; TRIOBP; TSC22D1; TSSC4; TTN; TWF2; TXNIP; TYMS; UBA2; UBAC2; UBC; UBE2D3; ULK3; UNG; USF2; USP32; USP34; USPL1; VCL; VDAC2; VEZF1; WDR1; WDR27; XBP1; XRN1; YY1; ZBTB10; ZBTB9; ZC3H10; ZDHHC14; ZFAND5; ZFP36L1; ZFP36L2; ZFYVE21; ZNF143; ZNF213; ZNF213-AS1; ZNF292; ZNF33B; ZNF34; ZNF417; ZNF580; ZNF581; ZNF587; ZNF598; ZNF655; ZNF668; ZNF687","AAMP; ABCA5; ABLIM3; ACACA; ACAD10; ACAD8; ACP2; ACSL3; ACTG1; ADAM9; ADCY9; ADNP; AHCYL2; AHI1; AHSA1; AKIRIN2; AMMECR1; ANKS6; ANKZF1; AP2S1; ARAP1; ARHGAP5; ARHGAP5-AS1; ARNTL; ASAP3; ATG4B; ATG9A; AUP1; B4GALT2; BABAM1; BANP; BICDL1; BLOC1S4; BOD1; BSG; BTBD9; BTRC; BUB1B; BZW2; C17orf58; C1orf159; C6orf89; CABIN1; CALCOCO1; CAMKK2; CAMKMT; CAPG; CBFA2T3; CBX7; CCAR1; CCDC137; CCDC18-AS1; CCDC90B; CCP110; CDC40; CDCA3; CDH23; CDK5; CDKN2AIPNL; CEP290; CFL1; CHD7; CHFR; CHMP4B; CLEC16A; CLSTN1; CLTC; CNPY2; COMMD3; CORO1A; COX7A2L; COX7C; CPEB4; CPQ; CRADD; CRCP; CREBL2; CRTC2; CRYBG2; CYB561D2; CYB5R3; DCK; DDX23; DEDD; DENND1A; DENND5B; DFFA; DGCR8; DNAAF5; DR1; DYNC1I2; ECH1; EFNA3; ELOC; ENSG00000233834; ENY2; EPC1; ERMAP; ERVK3-1; EXOC6; FAM117A; FANCC; FASN; FASTKD1; FBXL16; FBXL19; FBXL3; FBXO15; FBXO38; FCHSD2; FCSK; FLYWCH1; FYCO1; GAB2; GFER; GFM1; GFUS; GNG4; GP6-AS1; GPAT4; GPKOW; GRHL1; GRK3; GSPT1; GTF2I; H3-3B; HDAC8; HERC1; HIF1A; HIKESHI; HM13; HTRA2; IFRD2; IFTAP; IKZF2; IL1R1; IMPACT; ING3; IRF2BP2; KAT7; KBTBD6; KBTBD7; KCNC3; KCNK1; KDM4C; KHK; LARP1; LARS1; LEMD2; LEPROTL1; LIG1; LINC01521; LSM10; LYPLA1; LYSMD1; MACC1; MAN2A2; MANEAL; MAPT; MARS1; MAX; MDH1; MITF; MKKS; MLLT10; MMACHC; MRFAP1; MRPL24; MRPS22; MUS81; MYB; MYCL; MYO1D; NDRG3; NEK6; NFASC; NFIX; NOP53; NOSIP; NPEPPS; NPEPPSP1; NPRL2; NRDE2; NSD2; NUDCD1; NUDT13; OPLAH; OTUD7A; PA2G4; PACS2; PAFAH2; PALS2; PANK4; PATZ1; PCBP1-AS1; PCSK6; PGBD5; PGP; PHB; PHF12; PHF13; PHLPP1; PISD; PLEKHA1; PLEKHF2; PNKD; PODXL2; PPM1E; PPOX; PPP1R12C; PPP1R15A; PPP2R2C; PREPL; PRKCA; PRMT3; PROSER2; PRPF31; PRPSAP1; PRSS27; PTDSS2; PTK2; PTPRN2; PUM3; RAB7A; RABAC1; RABGAP1L; RANBP10; RANGAP1; RBL1; RBM6; RHOBTB3; RIN3; RNASEH2C; RNF144A; RPP40; RXRA; RYBP; SAP30; SBK1; SCAF1; SCAMP4; SCAMP5; SCNM1; SEC24C; SEC61A1; SEL1L; SEM1; SEPHS2; SETMAR; SFI1; SFXN1; SGPL1; SIAH1; SIPA1L3; SKIL; SLC4A2; SLC9A3-AS1; SLF2; SLK; SLMAP; SLX4IP; SMARCAL1; SMPD3; SNAPC3; SPTBN4; SRCIN1; SRRM3; ST3GAL2; ST7; STARD10; STAT3; STAT5B; STAU1; STAU2; STK25; STRIP1; SUMO3; SYNGR1; TADA2A; TBX6; TCEA2; TCERG1; TCP11L1; TFPT; THADA; THUMPD2; THYN1; TIMM21; TM9SF4; TMCC1; TMCC1-DT; TMED10; TMEM11; TMEM115; TMEM160; TMEM181; TMEM70; TMTC3; TPGS1; TRABD; TRIM33; TRMT12; TSC22D1; TXNDC11; TXNDC15; UBE2E2; UBE2Q2; UBE3C; UBXN2A; UCKL1; UNK; UROS; USE1; USF2; USP31; USP6NL; VCPIP1; VIPAS39; WASF1; WDPCP; WDR82; WNK1; WSB1; WWP1; XIST; XKR9; XYLT2; YEATS2; ZBTB38; ZBTB46; ZC3H7A; ZFYVE1; ZFYVE28; ZMAT1; ZNF221; ZNF225; ZNF236; ZNF280D; ZNF32; ZNF331; ZNF37BP; ZNF417; ZNF444; ZNF605; ZNF688; ZNF776"]},"columns":[{"id":"gs_name","name":"Gene Set","type":"character","cell":[{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["KAT2A TARGET GENES"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["ZNF766 TARGET GENES"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["E2F5 TARGET GENES"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["SETD7 TARGET GENES"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["ZNF197 TARGET GENES"]}],"html":true,"maxWidth":150},{"id":"gs_description","name":"Description","type":"character","cell":["Genes containing one or more binding sites for UniProt:Q92830 (KAT2A) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 2...","Genes containing one or more binding sites for UniProt:Q5HY98 (ZNF766) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version ...","Genes containing one or more binding sites for UniProt:Q15329 (E2F5) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20...","Genes containing one or more binding sites for UniProt:Q8WTS6 (SETD7) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 2...","Genes containing one or more binding sites for UniProt:O14709 (ZNF197) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version ..."]},{"id":"pval","name":"P-Value","type":"numeric","cell":["2.02e-08","3.25e-07","8.62e-07","1.89e-06","2.10e-06"],"maxWidth":80},{"id":"adj_p","name":"p<sub>adj<\/sub>","type":"numeric","cell":["4.63e-05","3.73e-04","6.59e-04","9.63e-04","9.63e-04"],"html":true,"maxWidth":80},{"id":"numDEInCat","name":"Genes Mapped to ER Only","type":"numeric","cell":["159","155","157","106","94"],"maxWidth":80},{"id":"numInCat","name":"Genes Mapped to Both","type":"numeric","cell":["615","620","639","401","346"],"maxWidth":80},{"id":"gene_name","name":"Gene Names","type":"character","cell":[{"name":"span","attribs":{"title":"A2M-AS1; AAGAB; ABHD2; ACP2; ACTG1; ADAM17; ADAT2; ADNP; AFF4; AFMID; AGK; AGPAT2; AGPS; AHCYL2; AKAP8L; AKIRIN2; ALAS1; ALDH7A1; ALDOA; ALG10; ALG8; AMBRA1; ANAPC2; ANAPC5; ANG; ANO10; AP4M1; APTX; ARF6; ARHGEF1; ARID4B; ARL3; ARMC6; ARV1; ATAD2; ATP1A1-AS1; ATP2B1; ATP5MF; ATPAF1; ATXN10; AUNIP; B3GALT4; B4GAT1; BCAR3; BCAT2; BLOC1S6; BOD1; BOLA1; BRD2; BRD3OS; BSG; C11orf58; C11orf68; C14orf93; C16orf91; C17orf75; C18orf25; C19orf44; C19orf48; C1orf159; CAB39; CALCOCO1; CALM1; CAP1; CARS2; CASP7; CCDC47; CCDC57; CCDC59; CCDC6; CCNG2; CCNL1; CCT7; CDC123; CDC42EP4; CDC45; CDC7; CDCA2; CDCA3; CDK4; CDK5; CDK5RAP2; CDK8; CEP192; CEP290; CEP95; CEPT1; CFAP418; CHCHD7; CHD8; CHERP; CHMP4B; CHSY1; CHTOP; CIAO2A; CIAO3; CIPC; CLASP1; CLCN7; CLEC16A; CLPB; CLPTM1L; CLTC; CLUH; CNOT4; COA8; COMT; COPS2; COX19; CRTC2; CSNK2B; CSRNP2; CTNNA1; CTNNBL1; CWC27; CYB561; CYB5D1; CYB5D2; CYCS; CYLD; CYP51A1; DCAF8; DCAKD; DCLRE1A; DCP1A; DCTN6; DDX23; DDX47; DDX5; DFFA; DHDDS; DHX8; DHX9; DICER1; DICER1-AS1; DNAAF5; DNAH7; DNAJA1; DNAJB6; DNAJC17; DNAJC19; DPH6; DPP8; DRAM2; DRAP1; DTWD1; DUS1L; DVL3; E2F3; E2F6; EHD1; EIF1B; EIF1B-AS1; EIF2B1; EIF3B; EIF3E; EIF3K; EIF4A2; EIF4E; EIF4E2; EIF4ENIF1; EIF4G3; ELAC1; ELMOD3; ELP3; EMC3; ENAH; ENSG00000254837; ENSG00000255468; ENSG00000268403; ENY2; EPB41L4A-AS1; EPCAM; ERAL1; ERF; ERRFI1; ERVK3-1; ESYT1; EVI5L; EXOSC3; FAAP100; FAF1; FAM168A; FAM222B; FAM229B; FANCI; FBF1; FBXL19; FBXL19-AS1; FBXO15; FBXO22; FBXO36; FBXO5; FBXW11; FEM1A; FGD5-AS1; FIZ1; FLOT1; FMO5; FOXK2; FUS; GAA; GABARAP; GABPB1; GALE; GALK2; GATB; GCFC2; GCN1; GDI2; GFM2; GGA1; GGCX; GGPS1; GLO1; GLT8D1; GNL3; GORASP2; GPANK1; GPBP1L1; GRK4; GSPT1; GSTCD; GTF2H3; GTF2I; GTPBP3; GUCD1; GUSB; H2AC20; H2AC7; H2BC21; H2BC5; H3-3B; H4C8; HAUS6; HEBP2; HIF1AN; HLA-DMA; HNRNPA0; HNRNPDL; HNRNPLL; HP1BP3; HSD11B2; HSP90AA1; HSP90B1; HSPD1; HSPE1; HUS1; ICMT; ID2-AS1; IER3; IER5; IFRD2; IFT20; IFTAP; IGHMBP2; ILF2; IMP3; INTS12; IPO13; IQCK; ISG20L2; ITGB3BP; JMJD1C; JMJD4; KATNBL1; KBTBD7; KCTD5; KCTD9; KITLG; KMT2A; LAMTOR2; LAMTOR5; LAMTOR5-AS1; LARS1; LDHA; LIAS; LIMS1; LINC00337; LINC00886; LINC01137; LIPT2; LPXN; LRPAP1; LYSMD1; LZIC; MAGOHB; MAN1A2; MANBAL; MAP3K3; MARK4; MAX; MAZ; MBTPS2; MCEE; MCL1; MCM3AP; MCM7; MCMBP; MCOLN1; MCU; MDH1; MED15; MED22; MED23; MED4; METTL15; METTL25; METTL25B; METTL9; MIB2; MIDN; MMACHC; MNAT1; MORF4L1; MPHOSPH10; MRFAP1; MRFAP1L1; MRPL19; MRPL38; MRPL39; MRPS12; MRPS27; MRPS28; MRPS33; MRTO4; MTCH1; MTERF4; MTIF2; MTMR9; MTOR; N4BP3; NAA38; NAIF1; NCOA7; NDC1; NDUFA12; NDUFA3; NDUFC2; NEAT1; NELFB; NFAT5; NFE2L2; NFYB; NHLRC2; NMNAT1; NMT1; NOP14; NOP16; NPLOC4; NSA2; NSUN2; NTAN1; NUDCD1; NUDT9; NUMB; NUP107; ORMDL3; OTUD7B; PABPC1; PABPN1; PANK4; PARP4; PBRM1; PCID2; PCLAF; PCYT1A; PDCD10; PDCD6IP; PDE4D; PDIA5; PEMT; PEX2; PFN2; PGS1; PHAF1; PHB; PHF12; PHF3; PIGQ; PIM3; PITHD1; PKM; PLEKHA8; POLA1; POLD3; POLDIP3; POLG; POLR3E; PPIL3; PPP1R12C; PPP2CB; PPP2R2A; PPP2R5B; PPP4R3A; PPP5C; PPRC1; PRKAB2; PRKDC; PRMT5; PRORP; PROSER1; PRPSAP1; PRR34-AS1; PSMD12; PTCD1; PUM2; RAD52; RALBP1; RANBP10; RANGAP1; RBL1; RBM26; RBM26-AS1; RBM28; RCC1; RETSAT; REXO1; RIC8B; RNASEH1; RNASEH1-AS1; RNF121; RNF135; RNF213-AS1; RNF216; RNF216P1; RNF220; RNPS1; RPIA; RPL18; RPL3; RPL31; RPL34; RPL36; RPL38; RPL41; RPL6; RPL7A; RPL9; RPP14; RPS12; RPS13; RPS2; RPS20; RPS3; RPS7; RRP7A; RSAD1; S100PBP; SARS2; SBNO2; SCAF1; SCNM1; SCRIB; SDC4; SEC22C; SENP5; SEPTIN7; SEPTIN9; SERP1; SERPINI1; SERTAD2; SF1; SF3B4; SFXN2; SIL1; SKP1; SLC25A25; SLC35A3; SLC35G1; SLC38A10; SLC44A1; SLC4A2; SLC9A1; SLC9A3R1; SMAD5; SMARCC2; SMARCD2; SMG7; SNAP29; SNAP47; SNHG3; SNHG9; SNORD3A; SNRNP27; SNRPD3; SNX1; SNX19; SNX27; SNX8; SPCS1; SPHK2; SPRED1; SRD5A1; SRI; SS18L2; SSNA1; ST3GAL2; STAT3; STAU1; STK17A; STMP1; STRIP1; SUGP2; SYPL1; TAMM41; TBC1D30; TBC1D31; TBC1D8; TBCB; TCEA2; TDRD3; TEDC1; THADA; THRAP3; TIGD1; TIMM21; TIMM23; TIMM23B; TIMM44; TK1; TM2D1; TMA16; TMED10; TMED2; TMEM11; TMEM147-AS1; TMEM198B; TMEM69; TMSB10; TMTC3; TNFAIP1; TOR1AIP1; TRABD; TRAM2; TRAM2-AS1; TRAP1; TRERF1; TRIM33; TRIP11; TRIP12; TSEN15; TSN; TSSC4; TTC13; TTC33; TUBA1C; TUBB; TUBE1; TUBGCP3; TXNDC11; TXNRD2; TYW5; UBAC2; UBE2I; UBQLN4; UBR2; UCKL1; UFD1; UHMK1; UNK; UPF1; USP21; USP24; USP30; USP39; VAMP4; VAMP8; VCPIP1; VEGFA; VEZF1; VMP1; VPS13B; VPS13B-DT; VTI1A; WAC; WAC-AS1; WARS1; WDFY1; WDPCP; WDR25; WDR27; WNK1; WRAP53; XPC; YARS1; YBEY; YTHDF1; ZBTB1; ZBTB5; ZC3H10; ZC3H12A; ZC3H15; ZCCHC8; ZDHHC6; ZFAND3; ZFP91; ZFYVE1; ZFYVE19; ZMPSTE24; ZMPSTE24-DT; ZMYM4; ZMYND11; ZNF143; ZNF195; ZNF3; ZNF337; ZNF37A; ZNF524; ZNF581; ZNF691; ZNF706; ZNF76; ZNHIT6; ZZEF1"},"children":["A2M-AS1; AAGAB; ABHD2; ACP2; ACTG1; ADAM17; ADAT2; ADNP; AFF4; AFMID; AGK; AGPAT2; AGPS; AHCYL2; AKAP8L; AKIRIN2; ALAS1; ALDH7A1; ALDOA; ALG10; ALG..."]},{"name":"span","attribs":{"title":"AAMP; ACP2; ACTB; ACTG1; ACTR1A; ACTR3; ACTR3B; ADAM9; ADAMTSL4; AHCTF1; AHCYL2; AHSA2P; AKAP8; AKIRIN2; ALDH5A1; ALMS1; AMMECR1; AMN1; AMOTL1; ANKMY1; ANKS6; ANTXR2; ANXA2; AP1S2; AP2M1; AP2S1; APOO; AREG; ARFRP1; ARHGAP5; ARHGAP5-AS1; ARHGEF12; ARID4B; ARL4A; ARNTL; ARPC1A; ARV1; ASAP3; ASNSD1; ASXL2; ATG10; ATG4B; ATP5PF; ATXN7; AUP1; B4GALT2; B4GAT1; BABAM1; BANP; BATF; BCKDK; BICD2; BIRC5; BMERB1; BOD1; BORCS7; BTN3A2; BTRC; BUB3; C11orf58; C15orf61; C17orf58; C19orf44; C1orf159; CABIN1; CACNA1D; CALCOCO1; CAMP; CAP1; CAPG; CAST; CBFA2T3; CBX7; CCAR1; CCDC12; CCDC137; CCDC71; CCDC90B; CCP110; CDC73; CDCA3; CDH23; CDK5; CDK8; CDYL; CENPT; CEP290; CFL1; CHCHD2; CHD7; CHFR; CHIC2; CHMP4B; CIC; CITED2; CLEC16A; CLPB; CLSTN1; CLTC; CLUAP1; CNPY2; COMMD3; COPS7B; CORO1A; COX7A2L; COX7C; CPE; CRADD; CRAMP1; CRTC2; CRYM; CSNK2B; CSRNP2; CSRP1; CTDP1; CTSB; CYB561D2; CYB5D1; DCK; DDX23; DEDD; DENND1A; DFFA; DGAT1; DGAT2; DGCR8; DHRS3; DIP2A; DMD; DNAJA3; DNAJC17; DNALI1; DNM1L; DST; DUS1L; DUS3L; DUSP4; DYNC1I2; DYNC2I2; ECHDC2; EFNA3; EHD1; EHMT2; ELOC; EML2; EML6; ENSG00000233834; ENSG00000255468; ENY2; EPB41L5; EPC1; ERH; ERI3; ERMAP; ERP44; ERVK3-1; EVL; EXOC6; EYA3; FAF2; FAM117A; FAM13A; FAM168B; FAM193A; FAM222A; FASN; FASTKD1; FBXL16; FBXL17; FBXL19; FBXL3; FBXO15; FCSK; FGF13; FHL2; FLYWCH1; FN3K; FOXP1; FTO; FYCO1; G3BP2; GAB2; GALNT7; GAPVD1; GATAD2A; GATB; GDAP1; GDPD5; GFER; GFUS; GGPS1; GLUD1; GNG4; GPANK1; GPAT4; GPBP1; GPBP1L1; GPKOW; GPRC5C; GPT2; GRAMD4; GRK3; GRK4; GSPT1; GTF2I; GTF2IRD1; GTPBP2; GUSB; H4C8; H6PD; HAUS6; HECTD2; HERC1; HGD; HIKESHI; HM13; HMG20A; HSD11B1L; HSPA13; HSPB11; HSPH1; HTRA2; IFRD1; IFRD2; IFT140; IFT20; IFTAP; IGF1R; IKZF2; IL1R1; ILF3; ILF3-DT; IMPACT; ING3; INTS6; INVS; IQCH-AS1; IRF2BP2; KANSL3; KAT7; KBTBD6; KBTBD7; KCNB1; KCTD20; KCTD21; KDM7A; KDM7A-DT; KHDC4; KIF15; KLC1; KLHDC8B; KLHL21; KMT2B; LAMTOR2; LARP1; LARS1; LCORL; LEMD2; LEPROTL1; LIFR; LIG1; LINC-PINT; LINC00504; LINC01521; LLGL2; LMNB2; LNX1; LPXN; LSM10; LTA4H; LUC7L; LYPLA1; LYSMD1; MACC1; MAD2L1BP; MAF1; MALSU1; MAML3; MAN2A2; MANBAL; MAP4K3; MAP4K3-DT; MAT2A; MAX; MDH1; MED15; MFAP3L; MGME1; MIAT; MICOS13; MIDEAS; MIDN; MITF; MKKS; MLLT10; MMACHC; MPI; MPZL3; MR1; MRFAP1; MRGBP; MRPL24; MTIF2; MTSS1; MUS81; MUTYH; MYB; MYCL; MYO5B; MYO9A; NAA38; NAPA; NDUFAB1; NEAT1; NEDD4; NEK6; NF2; NFASC; NFE2L1; NFIX; NFKBIA; NFKBIB; NKAIN1; NOP14; NOP53; NORAD; NOSIP; NPEPPS; NPEPPSP1; NPL; NPRL2; NRDE2; NSMCE1-DT; NTAN1; NUCB1; NUDCD1; NUDT13; NUMA1; NUP205; NUP43; NUTF2; OPLAH; ORMDL3; OTUD7A; PA2G4; PABPN1; PACS1; PACS2; PACSIN2; PAFAH2; PANK4; PAPSS1; PATZ1; PCBP1-AS1; PCGF5; PCSK6; PDCD10; PDE3B; PDE6D; PDS5B; PEAK1; PEMT; PEX14; PEX26; PFDN6; PGBD5; PGM2L1; PGP; PHB; PHF12; PHF13; PISD; PLA2G6; PLCG2; PLEKHF2; PNKD; PNPLA6; PODXL2; POLDIP3; POLR1G; PPP1R12C; PPP1R13L; PPP1R15A; PPRC1; PRKCA; PRMT3; PROSER2; PRPF31; PRPSAP1; PRR5; PRRT2; PTCD3; PTK2; QRICH1; RAB1A; RAB27B; RAB7A; RABGAP1L; RALBP1; RANBP10; RANGAP1; RBBP4; RBM14; RBM25; RCAN1; RCBTB2; RERE; REXO2; RHOBTB3; RIC8B; RIMKLB; RIN3; RMC1; RNF145; RNFT2; RPGRIP1L; RPIA; RPL36; RPP40; RPRD1B; RPS13; RXRA; RYK; S100P; SAP30; SBK1; SCAF1; SCAMP4; SCNM1; SCRIB; SEC11C; SEC24C; SEC61A1; SENP8; SERPINI1; SETD5; SETMAR; SFMBT1; SFSWAP; SFT2D2; SFXN1; SHARPIN; SHLD2; SIPA1L3; SIRT2; SKIL; SLC16A1; SLC25A12; SLC25A6; SLC2A4RG; SLC35B4; SLC38A2; SLC39A11; SLC39A9; SLC44A1; SLC45A4; SLC48A1; SLC4A2; SLC9A3-AS1; SLK; SLMAP; SLX4IP; SMAD3; SMAP1; SMIM13; SMPD3; SNX10; SNX19; SNX5; SPEN; SPRING1; SQOR; SRCIN1; SRGAP3; ST3GAL2; STAG2; STAT5B; STAU1; STAU2; STK25; STK40; STMN3; STMP1; STPG1; STRIP1; STRN3; STXBP5; SULT1A2; SUMO3; SYMPK; SYNGR1; TANC1; TARBP2; TBC1D4; TBC1D8B; TBX6; TCEA2; TCERG1; TENT2; TFAP2A; TFEB; TFPT; THADA; THRAP3; THUMPD2; TIAL1; TIMM21; TJP3; TM7SF3; TM9SF4; TMBIM1; TMCC1; TMCC1-DT; TMCO1; TMCO1-AS1; TMEM11; TMEM115; TMEM160; TMEM181; TMEM245; TMEM69; TMEM70; TMTC1; TMTC3; TNFAIP1; TNS1; TOE1; TP53BP2; TPD52L1; TPGS1; TRAM2-AS1; TRIM29; TRIT1; TRMT12; TRMT44; TRPM7; TSC22D1; TSC22D2; TSSC4; TTC1; TTC13; TTC33; TTLL4; TUFT1; TXNDC15; UBE2E2; UBE2E3; UBE2Q2; UBE3C; UBQLN4; UBR2; UBR3; UBXN2A; UCP2; UNC45A; UPF1; UROS; USF1; USF2; USP31; USP32; USP35; USP48; USP6NL; VAPA; VDAC2; VMP1; VPS35L; VPS50; VRK1; WARS1; WARS2-AS1; WBP1; WBP2; WDPCP; WDR1; WDR24; WDR25; WDR46; WNK1; WSB1; WWP1; XIST; XKR9; XYLT2; YAP1; YEATS2; YIF1B; YIPF6; ZC3H7A; ZCCHC8; ZDHHC18; ZFP91; ZFYVE1; ZFYVE19; ZFYVE28; ZGPAT; ZMYM4; ZMYND8; ZNF181; ZNF207; ZNF217; ZNF221; ZNF223; ZNF225; ZNF32; ZNF331; ZNF337; ZNF37A; ZNF37BP; ZNF436; ZNF444; ZNF529; ZNF586; ZNF605; ZNF644; ZNF674-AS1; ZNF688; ZNF776; ZSCAN22"},"children":["AAMP; ACP2; ACTB; ACTG1; ACTR1A; ACTR3; ACTR3B; ADAM9; ADAMTSL4; AHCTF1; AHCYL2; AHSA2P; AKAP8; AKIRIN2; ALDH5A1; ALMS1; AMMECR1; AMN1; AMOTL1; ANK..."]},{"name":"span","attribs":{"title":"AAMP; ABCA5; ABRAXAS2; ACAD8; ACOX2; ACP2; ACTG1; ACTN4; ACTR1A; ADAM9; ADCY9; ADD1; ADNP; ADO; ADSS2; AFDN; AFMID; AGPS; AHCTF1; AHCYL2; AHI1; AHSA1; AKIRIN2; AKT2; ALDH5A1; ALDOA; ALG10; ALKBH8; AMMECR1; ANKMY1; ANKRD11; ANKS6; ANKZF1; ANP32E; ANXA5; AP2M1; AP2S1; AP4M1; APLF; ARHGAP5; ARHGAP5-AS1; ARHGDIA; ARHGEF2; ARMH3; ARNTL; ARPC2; ARV1; ASAP3; ASPH; ATAD2; ATG4B; ATG9A; ATP5MF; ATP7A; AUP1; B4GALT2; B4GALT3; BABAM1; BABAM2; BANP; BICDL1; BIRC5; BLM; BOD1; BRD3; BRD4; BRD9; BSG; BTBD1; BTBD3; BTRC; BUB1B; BUB3; C11orf58; C18orf54; C19orf44; C1orf109; C1orf159; C2orf69; CA12; CABIN1; CACTIN; CALCOCO1; CAMKK2; CASP8AP2; CBFA2T3; CCDC115; CCDC136; CCDC159; CCDC191; CCDC71; CCNA2; CCNB2; CCNG2; CCP110; CDC40; CDCA2; CDCA3; CDCA8; CDH23; CDK11A; CDK5; CDK5RAP2; CDYL; CECR2; CENPA; CENPO; CENPT; CEP128; CEP290; CEP295; CEP89; CFL1; CHCHD2; CHCHD6; CHD7; CHEK2; CHFR; CHIC2; CHMP4B; CHPT1; CIAO2A; CIDECP1; CLEC16A; CLINT1; CLSTN1; CLTC; CMIP; CMSS1; CNPY2; COMMD3; COPS7B; COX7B; COX7C; CRADD; CRAMP1; CRTC2; CSGALNACT2; CSNK2B; CTBP1; CTCF; CTDP1; CWC22; CYB561D2; DCAKD; DCK; DDIAS; DDX23; DEDD; DENND1A; DGCR8; DIAPH3; DLGAP4; DMXL2; DNAJC17; DNAL1; DPYSL2; DUS1L; DUSP14; DYNC1I2; E2F1; E2F2; E2F3; EEF2K; EFNA3; EHD1; ELOC; EMC3; EME1; EML6; ENSG00000215154; ENSG00000233834; ENSG00000234432; ENY2; EPC1; ERP44; ERVK3-1; EXOC6; FAAP24; FAM110A; FAM117A; FAM168B; FAM193A; FAM222A; FANCC; FANCD2; FANCI; FASN; FASTKD1; FBF1; FBRSL1; FBXL16; FBXL20; FBXL5; FBXO15; FBXO38; FBXO46; FBXO48; FBXO5; FBXW11; FCHSD2; FEM1A; FGFR1; FHL2; FN3K; FTO; FYCO1; GAB2; GATB; GBAP1; GDAP1; GFER; GFUS; GINS2; GLUD1; GNG4; GP6-AS1; GPANK1; GPAT4; GPBP1L1; GPT2; GRAMD4; GRHL1; GRK3; GRK4; GSPT1; GTF2IRD1; GTPBP2; GUCD1; H2AC11; H4C8; HAUS8; HDAC5; HECTD2; HERC1; HIF1A; HIKESHI; HINT3; HM13; HMG20A; HMGB1; HNRNPF; HOOK3; HSCB; HSPD1; HSPE1; HTRA2; IFRD1; IFRD2; IFT140; IFT80; IGF1R; IKZF2; IL1R1; IMPACT; ING3; INTS1; INVS; IPO13; IQGAP3; IRF2BP2; KAT7; KBTBD6; KBTBD7; KCNB1; KCNC3; KCNK1; KCTD20; KCTD9; KDM7A; KDM7A-DT; KIAA0232; KIF15; KIF18B; KIF20B; KIF22; KLF13; KMT2B; KPNA3; LAMTOR2; LARS1; LEMD2; LEPROTL1; LGALSL; LIFR; LIN9; LINC01521; LMNB1; LMNB2; LPIN2; LSM10; LUC7L; LUC7L2; LYSMD1; MACC1; MAD2L1BP; MAGOHB; MAN2A2; MANEAL; MAPT; MARS1; MAT2A; MAU2; MAX; MBTPS2; MCM2; MCM5; MCM7; MDH1; MDM2; MED15; MEGF11; METTL15; METTL4; MFAP3L; MGME1; MIDEAS; MIS18BP1; MITF; MKI67; MKKS; MMACHC; MOB4; MPI; MRGBP; MRPL27; MRPL52; MRPS25; MTMR4; MUS81; MYB; MYCL; MYO6; MYO9B; MZT1; NADK2; NCAPH; NCOA5; NDC80; NEDD4; NEK6; NF2; NFASC; NFE2L2; NFXL1; NKAIN1; NMT1; NOP14; NOP53; NOSIP; NPEPPS; NPEPPSP1; NPRL2; NRDE2; NSD2; NSMCE1-DT; NTAN1; NTHL1; NUCB1; NUDCD1; NUDT13; NUF2; NUMA1; NUTF2; OBSL1; OPLAH; ORMDL3; OSTF1; OTUD7A; PACS2; PAFAH2; PAN2; PANK4; PATZ1; PBK; PCBP1-AS1; PCED1B; PCSK6; PDE3B; PDE6D; PDIK1L; PDS5B; PEAK1; PEX26; PGP; PHB; PHF12; PHF13; PHF21A; PHLPP1; PIAS1; PLEKHF2; PNKD; PNPLA6; PODXL2; POLE2; PPCDC; PPM1E; PPP1R12C; PPP1R15A; PRCP; PRKCA; PRKCE; PRKD1; PRKDC; PRMT3; PRR5; PTCD1; PTDSS2; PTMS; PTPRN2; PTRHD1; QRICH1; QTRT1; QTRT2; RAB30; RAB30-DT; RAB7A; RABEP1; RABGAP1L; RAD51; RAD51-AS1; RAD51B; RALBP1; RANBP10; RANGAP1; RBBP4; RBKS; RBL1; RBM6; RCCD1; RERE; REXO2; RFC3; RFC4; RHOBTB3; RIC8B; RIN3; RMDN3; RNF170; RNF213; RNF227; RPF1; RPGRIP1L; RPH3AL; RPIA; RPL23A; RPP40; RPS13; RPS20; RRM1; RTEL1; RTEL1-TNFRSF6B; RTKN2; RXRA; RYK; SAP30; SASS6; SAV1; SBK1; SCAF1; SCAMP5; SCNM1; SCRIB; SDK1; SEC14L1; SEC24C; SEC61A1; SEL1L; SETMAR; SGPL1; SHLD2; SIAH1; SIK3; SLC35A1; SLC35F2; SLC38A1; SLC44A1; SLC45A4; SLC48A1; SLC4A2; SLC6A6; SLK; SLX4IP; SMARCAL1; SMC6; SMIM30; SMPD3; SMUG1; SNAPC3; SNIP1; SNRPD3; SNX1; SNX5; SPC25; SPG11; SPTBN4; SRCIN1; SRRM3; SRSF1; ST3GAL2; ST7; STAT1; STAT3; STAT5B; STAU1; STK25; STMN1; STMN3; STN1; STON2; STRADB; STRIP1; SUGP1; SUMO3; SYMPK; SYNGR1; TACC1; TBC1D31; TBC1D4; TBC1D8B; TBX6; TCEA2; TCP11L1; TEDC2; TFAP2A; TFEB; TGFBR3; THAP1; THAP12; THUMPD2; THYN1; TIAL1; TIMM21; TIMM22; TK1; TKT; TM9SF4; TMBIM1; TMEM11; TMEM115; TMEM131; TMEM150A; TMEM160; TMEM181; TMEM33; TMEM69; TMEM70; TMTC3; TNRC6B; TOP2A; TOR2A; TP53; TP53BP2; TPGS1; TPRA1; TPX2; TRAK2; TRAPPC3; TRIM29; TRIP13; TSC2; TTC13; TTC33; TTLL4; TUBGCP3; TXNDC15; UBE2E2; UBE2E3; UBE2G2; UBE2Q1; UBE2Q2; UBE3C; UBQLN4; UBR2; UBR3; UBXN2A; UCKL1; UNG; UPF1; USF2; USP28; USP3; USP31; USP39; USP48; USP6NL; VAPA; VCPIP1; VDAC2; VIPAS39; VKORC1; VMP1; VPS35L; VPS50; VRK1; VRK3; WASF1; WBP2; WDPCP; WDR13; WNK1; WRAP53; WT1-AS; WWP1; XIST; XRCC3; XYLT2; YEATS2; ZBTB11; ZBTB46; ZFX; ZFYVE1; ZFYVE19; ZFYVE21; ZFYVE28; ZNF133; ZNF155; ZNF217; ZNF225; ZNF280C; ZNF280D; ZNF32; ZNF331; ZNF337; ZNF354A; ZNF37A; ZNF436; ZNF444; ZNF451; ZNF473; ZNF573; ZNF589; ZNF605; ZNF687; ZNF785; ZW10; ZWINT; ZYG11A"},"children":["AAMP; ABCA5; ABRAXAS2; ACAD8; ACOX2; ACP2; ACTG1; ACTN4; ACTR1A; ADAM9; ADCY9; ADD1; ADNP; ADO; ADSS2; AFDN; AFMID; AGPS; AHCTF1; AHCYL2; AHI1; AHS..."]},{"name":"span","attribs":{"title":"ACOT13; ACSL3; ACTB; ACTG1; ACTN4; ACYP2; ADAP1; ADI1; AHCY; AKAP8; ALDOA; AMZ2; ANP32B; AP4M1; ARFGAP2; ARID1B; ARPC1A; ARPC1B; ATL2; ATP1A1; ATP5MC3; AVIL; AXIN2; B3GALT4; B3GALT9; B4GAT1; BCL2L1; BCL2L2; BCL9L; BHLHE40; BICRA; BOLA1; BRD2; BRICD5; BSG; BTF3; BUB1; C14orf93; C19orf48; CAPN15; CBX3; CCDC106; CCDC34; CCNL1; CDC42EP3; CDC6; CENPJ; CERS5; CFL1; CHASERR; CHD2; CHD8; CHERP; CIRBP; CLASP1; CLTC; CNOT3; CNP; COX19; CSK; CSNK1G1; CSNK2B; CSRNP2; CYB5R2; CYRIB; DANCR; DAZAP2; DCP1A; DCP1B; DDIT4; DDX3X; DECR2; DENND2B; DEPDC5; DHX9; DNASE1; DPY19L3; DPYSL2; DYNLL1; DYNLL2; DYRK1B; EDC4; EEF1D; EFCAB2; EFHC1; EIF1; EIF4A2; EIF4A3; EIF5; ELF2; EME2; ENSG00000215154; ENSG00000242622; ENSG00000255468; ENSG00000268403; EPHA1; EPM2AIP1; EPN1; ERCC1; ESYT1; FAM174C; FAM200A; FAM20A; FASN; FBXL14; FBXW2; FER; FGFR1; FLAD1; FLOT1; FTSJ3; FTX; FUT11; GABARAP; GABPB1; GALE; GAPDH; GARS1-DT; GCN1; GDPD3; GEMIN6; GLUL; GPANK1; GPATCH4; GPX4; GRB2; GRWD1; GSK3B; H1-4; H2AC11; H2AC12; H2AC20; H2AC21; H2AC6; H2AC7; H2AC8; H2AZ1; H2BC10; H2BC18; H2BC21; H2BC4; H3-3B; H3C6; H4C8; HDAC4; HERC1; HEXIM1; HEXIM2; HIF1A; HMGA1; HMGB1; HNRNPA2B1; HNRNPC; HNRNPF; HSP90AA1; HSPA5; HUS1; HUWE1; IAH1; ID4; IER5L; IL17RD; ING1; ING5; INO80; INTS10; ISG20L2; JMJD6; JMJD8; KAT8; KCNC3; KCTD15; KDM8; KDSR; KIF22; KLC1; KLF7; KLHL35; KPNA2; LAMA5; LETM2; LINC01089; LINC01521; LRP2BP; LRP6; LSR; LUC7L3; LYRM2; MAP3K12; MAT2A; MAZ; MCL1; MCM7; MEIS1; METTL23; METTL25B; MFSD10; MFSD11; MIB2; MIDN; MLH1; MLLT10; MMACHC; MPHOSPH9; MRPL28; MRPS27; MRPS28; MRPS34; MRPS36; MTA1; MTHFS; MTRF1L; MYL12A; MYL6; NAA50; NCL; NEK8; NEU3; NFE2L3; NFKBIA; NINL; NIT1; NKIRAS1; NOP14; NOPCHAP1; NPM1; NR1H2; NT5C; NUDT17; OAZ2; OGA; OIP5-AS1; P4HA1; PACS1; PANX1; PATZ1; PCBP1-AS1; PCGF3; PCID2; PCIF1; PCNT; PDHB; PFDN2; PFKFB3; PFKP; PHACTR4; PHLDB1; PKM; PKP2; PLEKHG2; PLEKHN1; PNRC1; POLDIP3; POLG; POLG2; POLR2C; POU2F1; PPP4R3A; PRKAR1A; PRKD2; PRKRIP1; PRPF31; PSMB10; PSMC5; PSMD12; PUM2; PYGO2; QSER1; RAB1B; RAB26; RAB33B; RABGEF1; RBL1; RBM25; RBPMS; RCC1; RCC2; REEP3; REEP4; RFC4; RFX3; RHOF; RMI1; RND3; RPL14; RPL15; RPL22L1; RPL27A; RPL3; RPS14; RPS19; RPS2; RPS3A; RPS9; RSRC1; SCRIB; SEC31A; SELENOW; SETD5; SF1; SGF29; SH3D21; SIL1; SIRT7; SIX5; SLC25A13; SLC25A25-AS1; SLC25A36; SLC26A6; SLC35E2B; SLC38A2; SLC39A10; SLF2; SMAD7; SMG1P3; SMIM10L1; SNHG12; SNHG8; SNHG9; SP3; SPATS2L; SPNS2; SRCAP; SRRM2; SRRM2-AS1; SRSF2; SRSF6; SRSF7; SS18; SSU72; STK26; STMN1; STUB1; SUMF2; SYAP1; TAF15; TCAF1; TDP2; TERF1; THBS3; TIGD5; TLCD1; TMEM33; TMEM63A; TMSB4X; TOB1-AS1; TP53I11; TPI1; TPM1; TPM4; TPT1; TPT1-AS1; TRAF4; TRIM65; TRIOBP; TSC22D1; TSSC4; TTN; TWF2; TXNIP; TYMS; UBA2; UBAC2; UBC; UBE2D3; ULK3; UNG; USF2; USP32; USP34; USPL1; VCL; VDAC2; VEZF1; WDR1; WDR27; XBP1; XRN1; YY1; ZBTB10; ZBTB9; ZC3H10; ZDHHC14; ZFAND5; ZFP36L1; ZFP36L2; ZFYVE21; ZNF143; ZNF213; ZNF213-AS1; ZNF292; ZNF33B; ZNF34; ZNF417; ZNF580; ZNF581; ZNF587; ZNF598; ZNF655; ZNF668; ZNF687"},"children":["ACOT13; ACSL3; ACTB; ACTG1; ACTN4; ACYP2; ADAP1; ADI1; AHCY; AKAP8; ALDOA; AMZ2; ANP32B; AP4M1; ARFGAP2; ARID1B; ARPC1A; ARPC1B; ATL2; ATP1A1; ATP5..."]},{"name":"span","attribs":{"title":"AAMP; ABCA5; ABLIM3; ACACA; ACAD10; ACAD8; ACP2; ACSL3; ACTG1; ADAM9; ADCY9; ADNP; AHCYL2; AHI1; AHSA1; AKIRIN2; AMMECR1; ANKS6; ANKZF1; AP2S1; ARAP1; ARHGAP5; ARHGAP5-AS1; ARNTL; ASAP3; ATG4B; ATG9A; AUP1; B4GALT2; BABAM1; BANP; BICDL1; BLOC1S4; BOD1; BSG; BTBD9; BTRC; BUB1B; BZW2; C17orf58; C1orf159; C6orf89; CABIN1; CALCOCO1; CAMKK2; CAMKMT; CAPG; CBFA2T3; CBX7; CCAR1; CCDC137; CCDC18-AS1; CCDC90B; CCP110; CDC40; CDCA3; CDH23; CDK5; CDKN2AIPNL; CEP290; CFL1; CHD7; CHFR; CHMP4B; CLEC16A; CLSTN1; CLTC; CNPY2; COMMD3; CORO1A; COX7A2L; COX7C; CPEB4; CPQ; CRADD; CRCP; CREBL2; CRTC2; CRYBG2; CYB561D2; CYB5R3; DCK; DDX23; DEDD; DENND1A; DENND5B; DFFA; DGCR8; DNAAF5; DR1; DYNC1I2; ECH1; EFNA3; ELOC; ENSG00000233834; ENY2; EPC1; ERMAP; ERVK3-1; EXOC6; FAM117A; FANCC; FASN; FASTKD1; FBXL16; FBXL19; FBXL3; FBXO15; FBXO38; FCHSD2; FCSK; FLYWCH1; FYCO1; GAB2; GFER; GFM1; GFUS; GNG4; GP6-AS1; GPAT4; GPKOW; GRHL1; GRK3; GSPT1; GTF2I; H3-3B; HDAC8; HERC1; HIF1A; HIKESHI; HM13; HTRA2; IFRD2; IFTAP; IKZF2; IL1R1; IMPACT; ING3; IRF2BP2; KAT7; KBTBD6; KBTBD7; KCNC3; KCNK1; KDM4C; KHK; LARP1; LARS1; LEMD2; LEPROTL1; LIG1; LINC01521; LSM10; LYPLA1; LYSMD1; MACC1; MAN2A2; MANEAL; MAPT; MARS1; MAX; MDH1; MITF; MKKS; MLLT10; MMACHC; MRFAP1; MRPL24; MRPS22; MUS81; MYB; MYCL; MYO1D; NDRG3; NEK6; NFASC; NFIX; NOP53; NOSIP; NPEPPS; NPEPPSP1; NPRL2; NRDE2; NSD2; NUDCD1; NUDT13; OPLAH; OTUD7A; PA2G4; PACS2; PAFAH2; PALS2; PANK4; PATZ1; PCBP1-AS1; PCSK6; PGBD5; PGP; PHB; PHF12; PHF13; PHLPP1; PISD; PLEKHA1; PLEKHF2; PNKD; PODXL2; PPM1E; PPOX; PPP1R12C; PPP1R15A; PPP2R2C; PREPL; PRKCA; PRMT3; PROSER2; PRPF31; PRPSAP1; PRSS27; PTDSS2; PTK2; PTPRN2; PUM3; RAB7A; RABAC1; RABGAP1L; RANBP10; RANGAP1; RBL1; RBM6; RHOBTB3; RIN3; RNASEH2C; RNF144A; RPP40; RXRA; RYBP; SAP30; SBK1; SCAF1; SCAMP4; SCAMP5; SCNM1; SEC24C; SEC61A1; SEL1L; SEM1; SEPHS2; SETMAR; SFI1; SFXN1; SGPL1; SIAH1; SIPA1L3; SKIL; SLC4A2; SLC9A3-AS1; SLF2; SLK; SLMAP; SLX4IP; SMARCAL1; SMPD3; SNAPC3; SPTBN4; SRCIN1; SRRM3; ST3GAL2; ST7; STARD10; STAT3; STAT5B; STAU1; STAU2; STK25; STRIP1; SUMO3; SYNGR1; TADA2A; TBX6; TCEA2; TCERG1; TCP11L1; TFPT; THADA; THUMPD2; THYN1; TIMM21; TM9SF4; TMCC1; TMCC1-DT; TMED10; TMEM11; TMEM115; TMEM160; TMEM181; TMEM70; TMTC3; TPGS1; TRABD; TRIM33; TRMT12; TSC22D1; TXNDC11; TXNDC15; UBE2E2; UBE2Q2; UBE3C; UBXN2A; UCKL1; UNK; UROS; USE1; USF2; USP31; USP6NL; VCPIP1; VIPAS39; WASF1; WDPCP; WDR82; WNK1; WSB1; WWP1; XIST; XKR9; XYLT2; YEATS2; ZBTB38; ZBTB46; ZC3H7A; ZFYVE1; ZFYVE28; ZMAT1; ZNF221; ZNF225; ZNF236; ZNF280D; ZNF32; ZNF331; ZNF37BP; ZNF417; ZNF444; ZNF605; ZNF688; ZNF776"},"children":["AAMP; ABCA5; ABLIM3; ACACA; ACAD10; ACAD8; ACP2; ACSL3; ACTG1; ADAM9; ADCY9; ADNP; AHCYL2; AHI1; AHSA1; AKIRIN2; AMMECR1; ANKS6; ANKZF1; AP2S1; ARA..."]}]}],"filterable":true,"showPageSizeOptions":true,"dataKey":"1fb59c7445f4b32ba5f8b9edffc6a17c"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
<div id="network-plots" class="section level3 tabset">
<h3 class="tabset">Network Plots</h3>
<div id="all-mapped-genes" class="section level4">
<h4>All Mapped Genes</h4>
<pre class="r"><code>tg_mapped %&gt;% 
  ggraph(layout = net_layout, weights = oc^2) +
  geom_edge_link(aes(width = oc^2, alpha = oc^2)) +
  geom_node_point(
    aes(fill = -log10(pval), size = numDEInCat),
    shape = 21
  ) +
  geom_node_text(
    aes(label = label),
    colour = &quot;black&quot;, size = 3, 
    data = . %&gt;%
      mutate(
        label = str_replace_all(label, &quot;_&quot;, &quot; &quot;) %&gt;% str_trunc(60) %&gt;% str_wrap(width = 18)
      ),
    repel = TRUE, max.overlaps = max(10, round(length(tg_mapped) / 4, 0)),
    bg.color = &quot;white&quot;, bg.r = 0.1, 
  ) +
  scale_x_continuous(expand = expansion(c(0.1, 0.1))) +
  scale_y_continuous(expand = expansion(c(0.1, 0.1))) +
  scale_fill_viridis_c(option = &quot;inferno&quot;, begin = 0.25) +
  scale_size_continuous(range = c(1, 10)) +
  scale_edge_width(range = c(1, 6), limits = c(0, 1)) +
  scale_edge_alpha(range = c(0.1, 0.4), limits = c(0, 1)) +
  guides(edge_alpha= &quot;none&quot;) +
  labs(size = &quot;Mapped Genes&quot;, edge_width = expr(paste(OC^2))) +
  theme_void() </code></pre>
<div class="figure" style="text-align: center">
<img src="AR_E2_E2DHT_ER_E2_E2DHT_pairwise_comparison_files/figure-html/plot-network-mapped-1.png" alt="*Network plot showing  gene-sets enriched amongst the overall set of sites with a binding site from __either__ comparison.*" width="960" />
<p class="caption">
<em>Network plot showing gene-sets enriched amongst the overall set of
sites with a binding site from <strong>either</strong> comparison.</em>
</p>
</div>
</div>
<div id="genes-mapped-to-both-ar-and-er-1" class="section level4">
<h4>Genes Mapped to Both AR and ER</h4>
<pre class="r"><code>tg_t1t2 %&gt;% 
  ggraph(layout = net_layout, weights = oc^2) +
  geom_edge_link(aes(width = oc^2, alpha = oc^2)) +
  geom_node_point(
    aes(fill = -log10(pval), size = numDEInCat),
    shape = 21
  ) +
  geom_node_text(
    aes(label = label),
    colour = &quot;black&quot;, size = 3, 
    data = . %&gt;%
      mutate(
        label = str_replace_all(label, &quot;_&quot;, &quot; &quot;) %&gt;% str_trunc(60) %&gt;% str_wrap(width = 18)
      ),
    repel = TRUE, max.overlaps = max(10, round(length(tg_t1t2) / 4, 0)),
    bg.color = &quot;white&quot;, bg.r = 0.1, 
  ) +
  scale_x_continuous(expand = expansion(c(0.1, 0.1))) +
  scale_y_continuous(expand = expansion(c(0.1, 0.1))) +
  scale_fill_viridis_c(option = &quot;inferno&quot;, begin = 0.25) +
  scale_size_continuous(range = c(1, 10)) +
  scale_edge_width(range = c(1, 6), limits = c(0, 1)) +
  scale_edge_alpha(range = c(0.1, 0.4), limits = c(0, 1)) +
  guides(edge_alpha= &quot;none&quot;) +
  labs(size = &quot;Mapped Genes&quot;, edge_width = expr(paste(OC^2))) +
  theme_void() </code></pre>
<div class="figure" style="text-align: center">
<img src="AR_E2_E2DHT_ER_E2_E2DHT_pairwise_comparison_files/figure-html/plot-network-t1t2-1.png" alt="*Network plot showing gene-sets enriched amongst the overall set of sites with a binding site for __both__ AR and ER.*" width="960" />
<p class="caption">
<em>Network plot showing gene-sets enriched amongst the overall set of
sites with a binding site for <strong>both</strong> AR and ER.</em>
</p>
</div>
</div>
<div id="genes-mapped-to-er-only-1" class="section level4">
<h4>Genes Mapped to ER Only</h4>
<pre class="r"><code>tg_t2only %&gt;% 
  ggraph(layout = net_layout, weights = oc^2) +
  geom_edge_link(aes(width = oc^2, alpha = oc^2)) +
  geom_node_point(
    aes(fill = -log10(pval), size = numDEInCat),
    shape = 21
  ) +
  geom_node_text(
    aes(label = label),
    colour = &quot;black&quot;, size = 3, 
    data = . %&gt;%
      mutate(
        label = str_replace_all(label, &quot;_&quot;, &quot; &quot;) %&gt;% str_trunc(60) %&gt;% str_wrap(width = 18)
      ),
    repel = TRUE, max.overlaps = max(10, round(length(tg_t2only) / 4, 0)),
    bg.color = &quot;white&quot;, bg.r = 0.1, 
  ) +
  scale_x_continuous(expand = expansion(c(0.1, 0.1))) +
  scale_y_continuous(expand = expansion(c(0.1, 0.1))) +
  scale_fill_viridis_c(option = &quot;inferno&quot;, begin = 0.25) +
  scale_size_continuous(range = c(1, 10)) +
  scale_edge_width(range = c(1, 6), limits = c(0, 1)) +
  scale_edge_alpha(range = c(0.1, 0.4), limits = c(0, 1)) +
  guides(edge_alpha= &quot;none&quot;) +
  labs(size = &quot;Mapped Genes&quot;, edge_width = expr(paste(OC^2))) +
  theme_void() </code></pre>
<div class="figure" style="text-align: center">
<img src="AR_E2_E2DHT_ER_E2_E2DHT_pairwise_comparison_files/figure-html/plot-network-t2only-1.png" alt="*Network plot showing all gene-sets enriched amongst the overall set of sites with a binding site for __ER only__.*" width="960" />
<p class="caption">
<em>Network plot showing all gene-sets enriched amongst the overall set
of sites with a binding site for <strong>ER only</strong>.</em>
</p>
</div>
</div>
</div>
</div>
<div id="comparison-of-differentially-bound-windows"
class="section level2">
<h2>Comparison of Differentially Bound Windows</h2>
<pre class="r"><code>n_peaks &lt;- structure(all_ids$n_windows, names = all_ids$gene_id)[mapped_ids]
goseq_group_res &lt;- group_ids %&gt;% 
  .[str_detect(names(.), &quot;Up|Down&quot;)] %&gt;% 
  mclapply(
    function(x) {
      vec &lt;- structure(mapped_ids %in% x, names = mapped_ids)
      pwf &lt;- nullp(vec, bias.data = n_peaks, plot.fit = FALSE)
      res &lt;- tibble(adj_p = numeric())
      gs_method &lt;- &quot;Hypergeometric&quot;
      if (sum(pwf$DEgenes) &gt; 0) {
        res &lt;- goseq(pwf, gene2cat = gs_by_geneid, method = gs_method) %&gt;% 
          as_tibble() %&gt;% 
          dplyr::filter(numDEInCat &gt; 0) %&gt;% 
          mutate(adj_p = p.adjust(over_represented_pvalue, adj_method)) %&gt;% 
          dplyr::filter(numDEInCat &gt;= min_sig) %&gt;% 
          dplyr::select(
            gs_name = category, pval = over_represented_pvalue, adj_p, 
            ends_with(&quot;InCat&quot;)
          ) %&gt;% 
          left_join(
            dplyr::filter(msigdb, gene_id %in% x)[c(&quot;gs_name&quot;, &quot;gene_name&quot;, &quot;gs_url&quot;, &quot;gs_description&quot;)],
            by = &quot;gs_name&quot;
          ) %&gt;% 
          chop(gene_name) %&gt;% 
          mutate(
            gene_name = vapply(gene_name, paste, character(1), collapse = &quot;; &quot;)
          )
      }
      res
    },
    mc.cores = threads
  ) %&gt;% 
  ## This step at the end basically removes any groups with no results
  lapply(list) %&gt;% 
  as_tibble() %&gt;% 
  pivot_longer(everything(), names_to = &quot;group&quot;) %&gt;% 
  unnest(everything()) %&gt;% 
  split(.$group)
goseq_group_sig &lt;- goseq_group_res %&gt;% 
  lapply(dplyr::filter, adj_p &lt; enrich_alpha) %&gt;% 
  .[vapply(., nrow, integer(1)) &gt; 0]
tg_group_res &lt;- names(goseq_group_sig) %&gt;% 
  lapply(
    function(x) {
      make_tbl_graph(
        goseq_group_sig[[x]],
        gs_by_gsid[goseq_group_sig[[x]]$gs_name] %&gt;%
          lapply(intersect, group_ids[[x]])
      )
    }
  ) %&gt;% 
  setNames(names(goseq_group_sig)) %&gt;% 
  .[vapply(., length, integer(1)) &gt;= min_network_size]</code></pre>
<p>The same MSigDB gene sets were used for associating any pathways with
the combined changes in both <em>AR</em> and <em>ER</em>. The ‘universe’
of genes was defined as all 7,631 gene ids with a binding window mapped
from either target. No sampling bias was included, with test results
using a simple Hypergeometric Distribution.</p>
<div id="results" class="section level3 tabset">
<h3 class="tabset">Results</h3>
<pre class="r"><code>htmltools::tagList(
  names(goseq_group_sig) %&gt;% 
    mclapply(
      function(x) {
        cp &lt;- glue(
          &quot;
          All {comma(length(group_ids[[x]]), 1)} genes mapped to binding windows
           with the pattern {x} were compared to the complete set of 
          {comma(length(mapped_ids), 1)} genes mapped to at least one binding
          window. {nrow(goseq_group_sig[[x]])} gene-sets were considered as 
          enriched using a threshold of {enrich_alpha} for adjusted-p-values.
          &quot;
        )
        htmltools::div(
          htmltools::div(
            id = x %&gt;% 
              str_to_lower %&gt;% 
              str_replace_all(&quot; &quot;, &quot;-&quot;) %&gt;% 
              str_replace_all(&quot;-+&quot;, &quot;-&quot;),
            class=&quot;section level4&quot;,
            htmltools::h4(class = &quot;tabset&quot;, x),
            htmltools::tags$em(cp),
            goseq_group_sig[[x]] %&gt;% 
              separate(group, into = comps, sep = &quot; - &quot;) %&gt;% 
              dplyr::select(
                all_of(comps), gs_name, gs_description, pval, adj_p, everything()
              ) %&gt;% 
              reactable(
                filterable = TRUE,
                showPageSizeOptions = TRUE,
                columns = list2(
                  &quot;{comps[[1]]}&quot; := colDef(
                    cell = function(value) {
                      value &lt;- str_extract(value, &quot;Up|Down|Unchanged|Undetected&quot;)
                      if (str_detect(value, &quot;Up&quot;)) value &lt;- paste(value, &quot;\u21E7&quot;)
                      if (str_detect(value, &quot;Down&quot;)) value &lt;- paste(value, &quot;\u21E9&quot;)
                      paste(targets[[1]], value)
                    },
                    style = function(value) {
                      colour &lt;- case_when(
                        str_detect(value, &quot;Up&quot;) ~ colours$direction[[&quot;up&quot;]],
                        str_detect(value, &quot;Down&quot;) ~ colours$direction[[&quot;down&quot;]],
                        str_detect(value, &quot;Unchanged&quot;) ~ colours$direction[[&quot;unchanged&quot;]],
                        str_detect(value, &quot;Undetected&quot;) ~ colours$direction[[&quot;undetected&quot;]]
                      )
                      list(color = colour)
                    },
                    maxWidth = 120
                  ),
                  &quot;{comps[[2]]}&quot; := colDef(
                    cell = function(value) {
                      value &lt;- str_extract(value, &quot;Up|Down|Unchanged|Undetected&quot;)
                      if (str_detect(value, &quot;Up&quot;)) value &lt;- paste(value, &quot;\u21E7&quot;)
                      if (str_detect(value, &quot;Down&quot;)) value &lt;- paste(value, &quot;\u21E9&quot;)
                      paste(targets[[2]], value)
                    },
                    style = function(value) {
                      colour &lt;- case_when(
                        str_detect(value, &quot;Up&quot;) ~ colours$direction[[&quot;up&quot;]],
                        str_detect(value, &quot;Down&quot;) ~ colours$direction[[&quot;down&quot;]],
                        str_detect(value, &quot;Unchanged&quot;) ~ colours$direction[[&quot;unchanged&quot;]],
                        str_detect(value, &quot;Undetected&quot;) ~ colours$direction[[&quot;undetected&quot;]]
                      )
                      list(color = colour)
                    },
                    maxWidth = 120
                  ),
                  gs_name = colDef(
                    name = &quot;Gene Set&quot;,
                    cell = function(value) {
                      htmltools::tags$a(
                        href = gs_url[[value]], target = &quot;_blank&quot;,
                        str_replace_all(value, &quot;_&quot;, &quot; &quot;)
                      )
                    },
                    html = TRUE,
                    maxWidth = 150
                  ),
                  gs_description = colDef(
                    name = &quot;Description&quot;,
                    cell = function(value) str_trunc(value, width = 150)
                  ),
                  gs_url = colDef(show = FALSE),
                  pval = colDef(
                    name = &quot;P-Value&quot;,
                    cell = function(value) ifelse(
                      value &lt; 0.001,
                      sprintf(&quot;%.2e&quot;, value),
                      sprintf(&quot;%.3f&quot;, value)
                    ),
                    maxWidth = 80
                  ),
                  adj_p = colDef(
                    name = &quot;p&lt;sub&gt;adj&lt;/sub&gt;&quot;, html = TRUE,
                    cell = function(value) ifelse(
                      value &lt; 0.001,
                      sprintf(&quot;%.2e&quot;, value),
                      sprintf(&quot;%.3f&quot;, value)
                    ),
                    maxWidth = 80
                  ),
                  numDEInCat = colDef(
                    name = &quot;Mapped Genes&quot;,
                    cell = function(value) comma(value, 1),
                    maxWidth = 80
                  ),
                  numInCat = colDef(
                    name = &quot;Genes in Gene-Set&quot;,
                    cell = function(value) comma(value, 1),
                    maxWidth = 80
                  ),
                  gene_name = colDef(
                    name = &quot;Gene Names&quot;,
                    cell = function(value) with_tooltip(value, width = 150)
                  )
                )
              )
          )
        )
      },
      mc.cores = threads
    )
)</code></pre>
<div>
<div id="ar-up-er-unchanged" class="section level4">
<h4 class="tabset">AR Up - ER Unchanged</h4>
<em>All 1,237 genes mapped to binding windows
 with the pattern AR Up - ER Unchanged were compared to the complete set of 
7,631 genes mapped to at least one binding
window. 3 gene-sets were considered as 
enriched using a threshold of 0.05 for adjusted-p-values.</em>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-1c782854d027327fbcd2" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-1c782854d027327fbcd2">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"AR":["AR Up","AR Up","AR Up"],"ER":["ER Unchanged","ER Unchanged","ER Unchanged"],"gs_name":["HALLMARK_ESTROGEN_RESPONSE_EARLY","GREB1_TARGET_GENES","HALLMARK_ESTROGEN_RESPONSE_LATE"],"gs_description":["Genes defining early response to estrogen.","Genes containing one or more binding sites for UniProt:Q4ZG55 (GREB1) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Genes defining late response to estrogen."],"pval":[2.38287135615251e-13,2.19226600096158e-10,1.26947355103052e-09],"adj_p":[5.39482075032928e-10,2.48164511308851e-07,9.58029373177701e-07],"numDEInCat":[65,134,51],"numInCat":[163,497,136],"gs_url":["","http://gtrd.biouml.org/",""],"gene_name":["ABHD2; ADCY9; ADD3; AFF1; AMFR; AQP3; AREG; ASB13; BCL2; CA12; CCN5; CHPT1; DHRS2; DHRS3; FAM102A; FARP1; FKBP5; FLNB; GAB2; GFRA1; GREB1; HES1; IGF1R; IGFBP4; IL6ST; INHBB; ITPK1; KLF10; KLF4; KRT15; KRT18; KRT19; KRT8; LRIG1; MAST4; MED13L; MLPH; MYB; MYBL1; MYC; NCOR2; P2RY2; PAPSS2; PLAAT3; RAB17; RAB31; RARA; RETREG1; RHOD; SIAH2; SLC22A5; SLC24A3; SLC26A2; SLC39A6; SLC9A3R1; SYBU; SYT12; TFAP2C; THSD4; TIPARP; TMEM164; TPD52L1; TSKU; WWC1; XBP1","ABCC11; ABHD2; ADAP1; AFAP1; AFF1; ANAPC2; ANKFY1; ARID1B; ARRDC1; ASCC2; ATP8B1; BAIAP3; BCAR3; C1QTNF6; CA12; CCN5; CEP89; CHD6; CHPT1; CITED2; CORO2A; CYP1B1; CYP1B1-AS1; DDA1; DLG5; DMXL1; DNPEP; DOP1B; DRAIC; DSCAM-AS1; ECH1; EHF; EHMT1; EML6; EPG5; EPHA1; ERBB2; ETNK2; FAAP24; FAM107B; FAM117A; FAM174B; FBXO38; FMN1; GPRC5A; GREB1; GRHL2; GTPBP3; HNRNPL; IFRD1; IGF1R; IGSF3; IKBKG; JUP; KAT7; KIAA1217; KRT7; KRT8; KRT80; LDLRAD4; LINC00511; LINC01588; LLGL2; MAP4K3-DT; MAPKAP1; MCCC2; MCF2L; MICOS10; MIDEAS; MTMR4; NAALADL2; NAIF1; NCAM2; NFKBIA; NFYB; NMT1; PBX1; PCBP1-AS1; PLEKHF2; PNKD; PNPLA7; PRKCD; PRKCH; PUM2; PXMP4; RABL3; RAD51B; RARA; RBM20; RECQL5; RGMB; RGMB-AS1; RNF224; RTN4; SERPINA1; SIDT2; SKIL; SLC1A5; SLC25A25; SLC38A1; SLC38A2; SPECC1L; SPRED2; SPRYD3; SPTLC2; SPTSSB; SRPK1; SSNA1; SYBU; SYT8; TBC1D16; TBC1D8; TBX2; TFAP2C; THADA; THSD4; TIPARP; TLE1; TMBIM1; TMEM184B; TMEM241; TMEM94; TMTC2; TOP2A; TPD52L1; TPRN; TUBB4B; UNKL; VAMP5; VMP1; WWC1; ZKSCAN1; ZNF552; ZNRF1","ABHD2; ADD3; AFF1; AGR2; AMFR; AREG; BCL2; CA12; CCN5; CD9; CHPT1; DHRS2; DLG5; DNAJC1; EMP2; FAM102A; FARP1; FKBP5; FLNB; GALE; GINS2; HMGCS2; IGFBP4; IL6ST; ITPK1; KLF4; KRT19; LARGE1; LLGL2; MYB; NCOR2; PAPSS2; PDCD4; PLAAT3; RAB31; S100A9; SCUBE2; SERPINA1; SIAH2; SLC22A5; SLC24A3; SLC26A2; SLC9A3R1; TFAP2C; TFPI2; TOP2A; TPD52L1; TSPAN13; UGDH; XBP1; ZFP36"]},"columns":[{"id":"AR","name":"AR","type":"character","cell":["AR Up ⇧","AR Up ⇧","AR Up ⇧"],"maxWidth":120,"style":[{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"}]},{"id":"ER","name":"ER","type":"character","cell":["ER Unchanged","ER Unchanged","ER Unchanged"],"maxWidth":120,"style":[{"color":"#7F7F7F"},{"color":"#7F7F7F"},{"color":"#7F7F7F"}]},{"id":"gs_name","name":"Gene Set","type":"character","cell":[{"name":"a","attribs":{"href":"\n      http://www.gsea-msigdb.org/gsea/msigdb/collections.jsp","target":"_blank"},"children":["HALLMARK ESTROGEN RESPONSE EARLY"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["GREB1 TARGET GENES"]},{"name":"a","attribs":{"href":"\n      http://www.gsea-msigdb.org/gsea/msigdb/collections.jsp","target":"_blank"},"children":["HALLMARK ESTROGEN RESPONSE LATE"]}],"html":true,"maxWidth":150},{"id":"gs_description","name":"Description","type":"character","cell":["Genes defining early response to estrogen.","Genes containing one or more binding sites for UniProt:Q4ZG55 (GREB1) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 2...","Genes defining late response to estrogen."]},{"id":"pval","name":"P-Value","type":"numeric","cell":["2.38e-13","2.19e-10","1.27e-09"],"maxWidth":80},{"id":"adj_p","name":"p<sub>adj<\/sub>","type":"numeric","cell":["5.39e-10","2.48e-07","9.58e-07"],"html":true,"maxWidth":80},{"id":"numDEInCat","name":"Mapped Genes","type":"numeric","cell":["65","134","51"],"maxWidth":80},{"id":"numInCat","name":"Genes in Gene-Set","type":"numeric","cell":["163","497","136"],"maxWidth":80},{"id":"gs_url","name":"gs_url","type":"character","show":false},{"id":"gene_name","name":"Gene Names","type":"character","cell":[{"name":"span","attribs":{"title":"ABHD2; ADCY9; ADD3; AFF1; AMFR; AQP3; AREG; ASB13; BCL2; CA12; CCN5; CHPT1; DHRS2; DHRS3; FAM102A; FARP1; FKBP5; FLNB; GAB2; GFRA1; GREB1; HES1; IGF1R; IGFBP4; IL6ST; INHBB; ITPK1; KLF10; KLF4; KRT15; KRT18; KRT19; KRT8; LRIG1; MAST4; MED13L; MLPH; MYB; MYBL1; MYC; NCOR2; P2RY2; PAPSS2; PLAAT3; RAB17; RAB31; RARA; RETREG1; RHOD; SIAH2; SLC22A5; SLC24A3; SLC26A2; SLC39A6; SLC9A3R1; SYBU; SYT12; TFAP2C; THSD4; TIPARP; TMEM164; TPD52L1; TSKU; WWC1; XBP1"},"children":["ABHD2; ADCY9; ADD3; AFF1; AMFR; AQP3; AREG; ASB13; BCL2; CA12; CCN5; CHPT1; DHRS2; DHRS3; FAM102A; FARP1; FKBP5; FLNB; GAB2; GFRA1; GREB1; HES1; IG..."]},{"name":"span","attribs":{"title":"ABCC11; ABHD2; ADAP1; AFAP1; AFF1; ANAPC2; ANKFY1; ARID1B; ARRDC1; ASCC2; ATP8B1; BAIAP3; BCAR3; C1QTNF6; CA12; CCN5; CEP89; CHD6; CHPT1; CITED2; CORO2A; CYP1B1; CYP1B1-AS1; DDA1; DLG5; DMXL1; DNPEP; DOP1B; DRAIC; DSCAM-AS1; ECH1; EHF; EHMT1; EML6; EPG5; EPHA1; ERBB2; ETNK2; FAAP24; FAM107B; FAM117A; FAM174B; FBXO38; FMN1; GPRC5A; GREB1; GRHL2; GTPBP3; HNRNPL; IFRD1; IGF1R; IGSF3; IKBKG; JUP; KAT7; KIAA1217; KRT7; KRT8; KRT80; LDLRAD4; LINC00511; LINC01588; LLGL2; MAP4K3-DT; MAPKAP1; MCCC2; MCF2L; MICOS10; MIDEAS; MTMR4; NAALADL2; NAIF1; NCAM2; NFKBIA; NFYB; NMT1; PBX1; PCBP1-AS1; PLEKHF2; PNKD; PNPLA7; PRKCD; PRKCH; PUM2; PXMP4; RABL3; RAD51B; RARA; RBM20; RECQL5; RGMB; RGMB-AS1; RNF224; RTN4; SERPINA1; SIDT2; SKIL; SLC1A5; SLC25A25; SLC38A1; SLC38A2; SPECC1L; SPRED2; SPRYD3; SPTLC2; SPTSSB; SRPK1; SSNA1; SYBU; SYT8; TBC1D16; TBC1D8; TBX2; TFAP2C; THADA; THSD4; TIPARP; TLE1; TMBIM1; TMEM184B; TMEM241; TMEM94; TMTC2; TOP2A; TPD52L1; TPRN; TUBB4B; UNKL; VAMP5; VMP1; WWC1; ZKSCAN1; ZNF552; ZNRF1"},"children":["ABCC11; ABHD2; ADAP1; AFAP1; AFF1; ANAPC2; ANKFY1; ARID1B; ARRDC1; ASCC2; ATP8B1; BAIAP3; BCAR3; C1QTNF6; CA12; CCN5; CEP89; CHD6; CHPT1; CITED2; C..."]},{"name":"span","attribs":{"title":"ABHD2; ADD3; AFF1; AGR2; AMFR; AREG; BCL2; CA12; CCN5; CD9; CHPT1; DHRS2; DLG5; DNAJC1; EMP2; FAM102A; FARP1; FKBP5; FLNB; GALE; GINS2; HMGCS2; IGFBP4; IL6ST; ITPK1; KLF4; KRT19; LARGE1; LLGL2; MYB; NCOR2; PAPSS2; PDCD4; PLAAT3; RAB31; S100A9; SCUBE2; SERPINA1; SIAH2; SLC22A5; SLC24A3; SLC26A2; SLC9A3R1; TFAP2C; TFPI2; TOP2A; TPD52L1; TSPAN13; UGDH; XBP1; ZFP36"},"children":["ABHD2; ADD3; AFF1; AGR2; AMFR; AREG; BCL2; CA12; CCN5; CD9; CHPT1; DHRS2; DLG5; DNAJC1; EMP2; FAM102A; FARP1; FKBP5; FLNB; GALE; GINS2; HMGCS2; IGF..."]}]}],"filterable":true,"showPageSizeOptions":true,"dataKey":"eb544a9049bbb189f5313638c8fbc859"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div>
<div id="ar-up-er-up" class="section level4">
<h4 class="tabset">AR Up - ER Up</h4>
<em>All 260 genes mapped to binding windows
 with the pattern AR Up - ER Up were compared to the complete set of 
7,631 genes mapped to at least one binding
window. 24 gene-sets were considered as 
enriched using a threshold of 0.05 for adjusted-p-values.</em>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-102e3d8a24534432b344" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-102e3d8a24534432b344">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"AR":["AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up"],"ER":["ER Up","ER Up","ER Up","ER Up","ER Up","ER Up","ER Up","ER Up","ER Up","ER Up","ER Up","ER Up","ER Up","ER Up","ER Up","ER Up","ER Up","ER Up","ER Up","ER Up","ER Up","ER Up","ER Up","ER Up"],"gs_name":["HALLMARK_ESTROGEN_RESPONSE_LATE","REACTOME_DNA_METHYLATION","REACTOME_RNA_POLYMERASE_I_PROMOTER_ESCAPE","REACTOME_B_WICH_COMPLEX_POSITIVELY_REGULATES_RRNA_EXPRESSION","REACTOME_NEGATIVE_EPIGENETIC_REGULATION_OF_RRNA_EXPRESSION","REACTOME_AMYLOID_FIBER_FORMATION","HALLMARK_ESTROGEN_RESPONSE_EARLY","REACTOME_ASSEMBLY_OF_THE_ORC_COMPLEX_AT_THE_ORIGIN_OF_REPLICATION","REACTOME_ACTIVATED_PKN1_STIMULATES_TRANSCRIPTION_OF_AR_ANDROGEN_RECEPTOR_REGULATED_GENES_KLK2_AND_KLK3","REACTOME_SIRT1_NEGATIVELY_REGULATES_RRNA_EXPRESSION","REACTOME_ACTIVATION_OF_ANTERIOR_HOX_GENES_IN_HINDBRAIN_DEVELOPMENT_DURING_EARLY_EMBRYOGENESIS","REACTOME_POSITIVE_EPIGENETIC_REGULATION_OF_RRNA_EXPRESSION","KEGG_SYSTEMIC_LUPUS_ERYTHEMATOSUS","REACTOME_RNA_POLYMERASE_I_TRANSCRIPTION","REACTOME_CONDENSATION_OF_PROPHASE_CHROMOSOMES","REACTOME_PRC2_METHYLATES_HISTONES_AND_DNA","REACTOME_TRANSCRIPTIONAL_REGULATION_OF_GRANULOPOIESIS","REACTOME_EPIGENETIC_REGULATION_OF_GENE_EXPRESSION","REACTOME_ERCC6_CSB_AND_EHMT2_G9A_POSITIVELY_REGULATE_RRNA_EXPRESSION","REACTOME_MEIOTIC_RECOMBINATION","REACTOME_HDMS_DEMETHYLATE_HISTONES","REACTOME_RECOGNITION_AND_ASSOCIATION_OF_DNA_GLYCOSYLASE_WITH_SITE_CONTAINING_AN_AFFECTED_PURINE","REACTOME_PHASE_II_CONJUGATION_OF_COMPOUNDS","REACTOME_RHO_GTPASES_ACTIVATE_PKNS"],"gs_description":["Genes defining late response to estrogen.","DNA methylation","RNA Polymerase I Promoter Escape","B-WICH complex positively regulates rRNA expression","Negative epigenetic regulation of rRNA expression","Amyloid fiber formation","Genes defining early response to estrogen.","Assembly of the ORC complex at the origin of replication","Activated PKN1 stimulates transcription of AR (androgen receptor) regulated genes KLK2 and KLK3","SIRT1 negatively regulates rRNA expression","Activation of anterior HOX genes in hindbrain development during early embryogenesis","Positive epigenetic regulation of rRNA expression","Systemic lupus erythematosus","RNA Polymerase I Transcription","Condensation of Prophase Chromosomes","PRC2 methylates histones and DNA","Transcriptional regulation of granulopoiesis","Epigenetic regulation of gene expression","ERCC6 (CSB) and EHMT2 (G9a) positively regulate rRNA expression","Meiotic recombination","HDMs demethylate histones","Recognition and association of DNA glycosylase with site containing an affected purine","Phase II - Conjugation of compounds","RHO GTPases activate PKNs"],"pval":[5.34526808823124e-07,6.7118605744031e-06,1.89164569136411e-05,2.34446490623713e-05,7.40022471265124e-05,0.000103816639201785,0.000104101302560605,0.000107429084691108,0.000137430111205126,0.000137430111205126,0.000142939513264566,0.000142939513264566,0.000157459867762025,0.000193482023027096,0.000217138849641385,0.000217138849641385,0.000224832573905997,0.000318977684642437,0.000577687912291428,0.000577687912291428,0.000666934152818929,0.000666934152818929,0.000810645738320013,0.00111014150012834],"adj_p":[0.000570874631823097,0.00358413354673126,0.00625972129965314,0.00625972129965314,0.0127216166805464,0.0127216166805464,0.0127216166805464,0.0127216166805464,0.0127216166805464,0.0127216166805464,0.0127216166805464,0.0127216166805464,0.0129359337515263,0.0141247758195062,0.0141247758195062,0.0141247758195062,0.0141247758195062,0.0189260092887846,0.0308485345163622,0.0308485345163622,0.0323766216004826,0.0323766216004826,0.0376421586315554,0.0494012967557113],"numDEInCat":[18,7,8,8,8,8,16,6,6,6,8,8,7,8,6,6,7,9,6,6,5,5,6,6],"numInCat":[136,23,36,37,43,45,163,24,25,25,47,47,36,49,27,27,38,66,32,32,22,22,34,36],"gs_url":["","https://www.reactome.org/content/detail/R-HSA-5334118|https://reactome.org/PathwayBrowser/#/R-HSA-5334118","https://www.reactome.org/content/detail/R-HSA-73772|https://reactome.org/PathwayBrowser/#/R-HSA-73772","https://www.reactome.org/content/detail/R-HSA-5250924|https://reactome.org/PathwayBrowser/#/R-HSA-5250924","https://www.reactome.org/content/detail/R-HSA-5250941|https://reactome.org/PathwayBrowser/#/R-HSA-5250941","https://www.reactome.org/content/detail/R-HSA-977225|https://reactome.org/PathwayBrowser/#/R-HSA-977225","","https://www.reactome.org/content/detail/R-HSA-68616|https://reactome.org/PathwayBrowser/#/R-HSA-68616","https://www.reactome.org/content/detail/R-HSA-5625886|https://reactome.org/PathwayBrowser/#/R-HSA-5625886","https://www.reactome.org/content/detail/R-HSA-427359|https://reactome.org/PathwayBrowser/#/R-HSA-427359","https://www.reactome.org/content/detail/R-HSA-5617472|https://reactome.org/PathwayBrowser/#/R-HSA-5617472","https://www.reactome.org/content/detail/R-HSA-5250913|https://reactome.org/PathwayBrowser/#/R-HSA-5250913","http://www.genome.jp/kegg/pathway/hsa/hsa05322.html","https://www.reactome.org/content/detail/R-HSA-73864|https://reactome.org/PathwayBrowser/#/R-HSA-73864","https://www.reactome.org/content/detail/R-HSA-2299718|https://reactome.org/PathwayBrowser/#/R-HSA-2299718","https://www.reactome.org/content/detail/R-HSA-212300|https://reactome.org/PathwayBrowser/#/R-HSA-212300","https://www.reactome.org/content/detail/R-HSA-9616222|https://reactome.org/PathwayBrowser/#/R-HSA-9616222","https://www.reactome.org/content/detail/R-HSA-212165|https://reactome.org/PathwayBrowser/#/R-HSA-212165","https://www.reactome.org/content/detail/R-HSA-427389|https://reactome.org/PathwayBrowser/#/R-HSA-427389","https://www.reactome.org/content/detail/R-HSA-912446|https://reactome.org/PathwayBrowser/#/R-HSA-912446","https://www.reactome.org/content/detail/R-HSA-3214842|https://reactome.org/PathwayBrowser/#/R-HSA-3214842","https://www.reactome.org/content/detail/R-HSA-110330|https://reactome.org/PathwayBrowser/#/R-HSA-110330","https://www.reactome.org/content/detail/R-HSA-156580|https://reactome.org/PathwayBrowser/#/R-HSA-156580","https://www.reactome.org/content/detail/R-HSA-5625740|https://reactome.org/PathwayBrowser/#/R-HSA-5625740"],"gene_name":["ABHD2; AGR2; AREG; DLG5; HOMER2; HPRT1; MAPT; MYB; PAPSS2; PPIF; PRLR; SIAH2; SLC1A4; SLC24A3; SLC9A3R1; TOB1; TSPAN13; UGDH","H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; UHRF1","H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; POLR1A; POLR1B","H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; POLR1A; POLR1B","H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; POLR1A; POLR1B","GGA2; H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; SIAH2","ABHD2; AREG; INPP5F; KDM4B; MAPT; MREG; MYB; PAPSS2; PPIF; SEC14L2; SIAH2; SLC1A4; SLC24A3; SLC9A3R1; SYBU; TOB1","H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8","H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8","H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8","CTCF; H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; PBX1","H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; POLR1A; POLR1B","C5; H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8","H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; POLR1A; POLR1B","H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8","H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8","H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; MYB","H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; POLR1A; POLR1B; UHRF1","H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8","H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8","ARID5B; H3C6; H4C5; H4C8; KDM4B","H2AC7; H2AC8; H2BC10; H4C5; H4C8","ACSM1; GCLC; GSTM3; GSTM4; PAPSS2; UGDH","H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8"]},"columns":[{"id":"AR","name":"AR","type":"character","cell":["AR Up ⇧","AR Up ⇧","AR Up ⇧","AR Up ⇧","AR Up ⇧","AR Up ⇧","AR Up ⇧","AR Up ⇧","AR Up ⇧","AR Up ⇧","AR Up ⇧","AR Up ⇧","AR Up ⇧","AR Up ⇧","AR Up ⇧","AR Up ⇧","AR Up ⇧","AR Up ⇧","AR Up ⇧","AR Up ⇧","AR Up ⇧","AR Up ⇧","AR Up ⇧","AR Up ⇧"],"maxWidth":120,"style":[{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"}]},{"id":"ER","name":"ER","type":"character","cell":["ER Up ⇧","ER Up ⇧","ER Up ⇧","ER Up ⇧","ER Up ⇧","ER Up ⇧","ER Up ⇧","ER Up ⇧","ER Up ⇧","ER Up ⇧","ER Up ⇧","ER Up ⇧","ER Up ⇧","ER Up ⇧","ER Up ⇧","ER Up ⇧","ER Up ⇧","ER Up ⇧","ER Up ⇧","ER Up ⇧","ER Up ⇧","ER Up ⇧","ER Up ⇧","ER Up ⇧"],"maxWidth":120,"style":[{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"}]},{"id":"gs_name","name":"Gene Set","type":"character","cell":[{"name":"a","attribs":{"href":"\n      http://www.gsea-msigdb.org/gsea/msigdb/collections.jsp","target":"_blank"},"children":["HALLMARK ESTROGEN RESPONSE LATE"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-5334118|https://reactome.org/PathwayBrowser/#/R-HSA-5334118","target":"_blank"},"children":["REACTOME DNA METHYLATION"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-73772|https://reactome.org/PathwayBrowser/#/R-HSA-73772","target":"_blank"},"children":["REACTOME RNA POLYMERASE I PROMOTER ESCAPE"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-5250924|https://reactome.org/PathwayBrowser/#/R-HSA-5250924","target":"_blank"},"children":["REACTOME B WICH COMPLEX POSITIVELY REGULATES RRNA EXPRESSION"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-5250941|https://reactome.org/PathwayBrowser/#/R-HSA-5250941","target":"_blank"},"children":["REACTOME NEGATIVE EPIGENETIC REGULATION OF RRNA EXPRESSION"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-977225|https://reactome.org/PathwayBrowser/#/R-HSA-977225","target":"_blank"},"children":["REACTOME AMYLOID FIBER FORMATION"]},{"name":"a","attribs":{"href":"\n      http://www.gsea-msigdb.org/gsea/msigdb/collections.jsp","target":"_blank"},"children":["HALLMARK ESTROGEN RESPONSE EARLY"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-68616|https://reactome.org/PathwayBrowser/#/R-HSA-68616","target":"_blank"},"children":["REACTOME ASSEMBLY OF THE ORC COMPLEX AT THE ORIGIN OF REPLICATION"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-5625886|https://reactome.org/PathwayBrowser/#/R-HSA-5625886","target":"_blank"},"children":["REACTOME ACTIVATED PKN1 STIMULATES TRANSCRIPTION OF AR ANDROGEN RECEPTOR REGULATED GENES KLK2 AND KLK3"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-427359|https://reactome.org/PathwayBrowser/#/R-HSA-427359","target":"_blank"},"children":["REACTOME SIRT1 NEGATIVELY REGULATES RRNA EXPRESSION"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-5617472|https://reactome.org/PathwayBrowser/#/R-HSA-5617472","target":"_blank"},"children":["REACTOME ACTIVATION OF ANTERIOR HOX GENES IN HINDBRAIN DEVELOPMENT DURING EARLY EMBRYOGENESIS"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-5250913|https://reactome.org/PathwayBrowser/#/R-HSA-5250913","target":"_blank"},"children":["REACTOME POSITIVE EPIGENETIC REGULATION OF RRNA EXPRESSION"]},{"name":"a","attribs":{"href":"http://www.genome.jp/kegg/pathway/hsa/hsa05322.html","target":"_blank"},"children":["KEGG SYSTEMIC LUPUS ERYTHEMATOSUS"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-73864|https://reactome.org/PathwayBrowser/#/R-HSA-73864","target":"_blank"},"children":["REACTOME RNA POLYMERASE I TRANSCRIPTION"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-2299718|https://reactome.org/PathwayBrowser/#/R-HSA-2299718","target":"_blank"},"children":["REACTOME CONDENSATION OF PROPHASE CHROMOSOMES"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-212300|https://reactome.org/PathwayBrowser/#/R-HSA-212300","target":"_blank"},"children":["REACTOME PRC2 METHYLATES HISTONES AND DNA"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-9616222|https://reactome.org/PathwayBrowser/#/R-HSA-9616222","target":"_blank"},"children":["REACTOME TRANSCRIPTIONAL REGULATION OF GRANULOPOIESIS"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-212165|https://reactome.org/PathwayBrowser/#/R-HSA-212165","target":"_blank"},"children":["REACTOME EPIGENETIC REGULATION OF GENE EXPRESSION"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-427389|https://reactome.org/PathwayBrowser/#/R-HSA-427389","target":"_blank"},"children":["REACTOME ERCC6 CSB AND EHMT2 G9A POSITIVELY REGULATE RRNA EXPRESSION"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-912446|https://reactome.org/PathwayBrowser/#/R-HSA-912446","target":"_blank"},"children":["REACTOME MEIOTIC RECOMBINATION"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-3214842|https://reactome.org/PathwayBrowser/#/R-HSA-3214842","target":"_blank"},"children":["REACTOME HDMS DEMETHYLATE HISTONES"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-110330|https://reactome.org/PathwayBrowser/#/R-HSA-110330","target":"_blank"},"children":["REACTOME RECOGNITION AND ASSOCIATION OF DNA GLYCOSYLASE WITH SITE CONTAINING AN AFFECTED PURINE"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-156580|https://reactome.org/PathwayBrowser/#/R-HSA-156580","target":"_blank"},"children":["REACTOME PHASE II CONJUGATION OF COMPOUNDS"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-5625740|https://reactome.org/PathwayBrowser/#/R-HSA-5625740","target":"_blank"},"children":["REACTOME RHO GTPASES ACTIVATE PKNS"]}],"html":true,"maxWidth":150},{"id":"gs_description","name":"Description","type":"character","cell":["Genes defining late response to estrogen.","DNA methylation","RNA Polymerase I Promoter Escape","B-WICH complex positively regulates rRNA expression","Negative epigenetic regulation of rRNA expression","Amyloid fiber formation","Genes defining early response to estrogen.","Assembly of the ORC complex at the origin of replication","Activated PKN1 stimulates transcription of AR (androgen receptor) regulated genes KLK2 and KLK3","SIRT1 negatively regulates rRNA expression","Activation of anterior HOX genes in hindbrain development during early embryogenesis","Positive epigenetic regulation of rRNA expression","Systemic lupus erythematosus","RNA Polymerase I Transcription","Condensation of Prophase Chromosomes","PRC2 methylates histones and DNA","Transcriptional regulation of granulopoiesis","Epigenetic regulation of gene expression","ERCC6 (CSB) and EHMT2 (G9a) positively regulate rRNA expression","Meiotic recombination","HDMs demethylate histones","Recognition and association of DNA glycosylase with site containing an affected purine","Phase II - Conjugation of compounds","RHO GTPases activate PKNs"]},{"id":"pval","name":"P-Value","type":"numeric","cell":["5.35e-07","6.71e-06","1.89e-05","2.34e-05","7.40e-05","1.04e-04","1.04e-04","1.07e-04","1.37e-04","1.37e-04","1.43e-04","1.43e-04","1.57e-04","1.93e-04","2.17e-04","2.17e-04","2.25e-04","3.19e-04","5.78e-04","5.78e-04","6.67e-04","6.67e-04","8.11e-04","0.001"],"maxWidth":80},{"id":"adj_p","name":"p<sub>adj<\/sub>","type":"numeric","cell":["5.71e-04","0.004","0.006","0.006","0.013","0.013","0.013","0.013","0.013","0.013","0.013","0.013","0.013","0.014","0.014","0.014","0.014","0.019","0.031","0.031","0.032","0.032","0.038","0.049"],"html":true,"maxWidth":80},{"id":"numDEInCat","name":"Mapped Genes","type":"numeric","cell":["18","7","8","8","8","8","16","6","6","6","8","8","7","8","6","6","7","9","6","6","5","5","6","6"],"maxWidth":80},{"id":"numInCat","name":"Genes in Gene-Set","type":"numeric","cell":["136","23","36","37","43","45","163","24","25","25","47","47","36","49","27","27","38","66","32","32","22","22","34","36"],"maxWidth":80},{"id":"gs_url","name":"gs_url","type":"character","show":false},{"id":"gene_name","name":"Gene Names","type":"character","cell":[{"name":"span","attribs":{"title":"ABHD2; AGR2; AREG; DLG5; HOMER2; HPRT1; MAPT; MYB; PAPSS2; PPIF; PRLR; SIAH2; SLC1A4; SLC24A3; SLC9A3R1; TOB1; TSPAN13; UGDH"},"children":["ABHD2; AGR2; AREG; DLG5; HOMER2; HPRT1; MAPT; MYB; PAPSS2; PPIF; PRLR; SIAH2; SLC1A4; SLC24A3; SLC9A3R1; TOB1; TSPAN13; UGDH"]},{"name":"span","attribs":{"title":"H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; UHRF1"},"children":["H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; UHRF1"]},{"name":"span","attribs":{"title":"H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; POLR1A; POLR1B"},"children":["H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; POLR1A; POLR1B"]},{"name":"span","attribs":{"title":"H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; POLR1A; POLR1B"},"children":["H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; POLR1A; POLR1B"]},{"name":"span","attribs":{"title":"H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; POLR1A; POLR1B"},"children":["H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; POLR1A; POLR1B"]},{"name":"span","attribs":{"title":"GGA2; H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; SIAH2"},"children":["GGA2; H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; SIAH2"]},{"name":"span","attribs":{"title":"ABHD2; AREG; INPP5F; KDM4B; MAPT; MREG; MYB; PAPSS2; PPIF; SEC14L2; SIAH2; SLC1A4; SLC24A3; SLC9A3R1; SYBU; TOB1"},"children":["ABHD2; AREG; INPP5F; KDM4B; MAPT; MREG; MYB; PAPSS2; PPIF; SEC14L2; SIAH2; SLC1A4; SLC24A3; SLC9A3R1; SYBU; TOB1"]},{"name":"span","attribs":{"title":"H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8"},"children":["H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8"]},{"name":"span","attribs":{"title":"H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8"},"children":["H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8"]},{"name":"span","attribs":{"title":"H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8"},"children":["H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8"]},{"name":"span","attribs":{"title":"CTCF; H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; PBX1"},"children":["CTCF; H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; PBX1"]},{"name":"span","attribs":{"title":"H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; POLR1A; POLR1B"},"children":["H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; POLR1A; POLR1B"]},{"name":"span","attribs":{"title":"C5; H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8"},"children":["C5; H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8"]},{"name":"span","attribs":{"title":"H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; POLR1A; POLR1B"},"children":["H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; POLR1A; POLR1B"]},{"name":"span","attribs":{"title":"H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8"},"children":["H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8"]},{"name":"span","attribs":{"title":"H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8"},"children":["H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8"]},{"name":"span","attribs":{"title":"H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; MYB"},"children":["H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; MYB"]},{"name":"span","attribs":{"title":"H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; POLR1A; POLR1B; UHRF1"},"children":["H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8; POLR1A; POLR1B; UHRF1"]},{"name":"span","attribs":{"title":"H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8"},"children":["H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8"]},{"name":"span","attribs":{"title":"H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8"},"children":["H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8"]},{"name":"span","attribs":{"title":"ARID5B; H3C6; H4C5; H4C8; KDM4B"},"children":["ARID5B; H3C6; H4C5; H4C8; KDM4B"]},{"name":"span","attribs":{"title":"H2AC7; H2AC8; H2BC10; H4C5; H4C8"},"children":["H2AC7; H2AC8; H2BC10; H4C5; H4C8"]},{"name":"span","attribs":{"title":"ACSM1; GCLC; GSTM3; GSTM4; PAPSS2; UGDH"},"children":["ACSM1; GCLC; GSTM3; GSTM4; PAPSS2; UGDH"]},{"name":"span","attribs":{"title":"H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8"},"children":["H2AC7; H2AC8; H2BC10; H3C6; H4C5; H4C8"]}]}],"filterable":true,"showPageSizeOptions":true,"dataKey":"21eebdb428678d8ae04ab7ea27f00981"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div id="all-groups" class="section level4">
<h4>All Groups</h4>
<pre class="r"><code>goseq_group_sig %&gt;%  
  bind_rows() %&gt;% 
  arrange(pval) %&gt;% 
  left_join(n_mapped, by = c(&quot;group&quot; = &quot;status&quot;)) %&gt;% 
  mutate(
    gs_name = fct_inorder(gs_name) %&gt;% 
      fct_relabel(str_replace_all, &quot;_&quot;, &quot; &quot;) %&gt;% 
      fct_relabel(str_trunc, width = 60),
    prop = numDEInCat / mapped,
    group = factor(group, levels = n_mapped$status)
  ) %&gt;%
  droplevels() %&gt;% 
  ggplot(aes(group, fct_rev(gs_name))) + 
  geom_point(aes(size = prop, fill = -log10(pval)), shape = 21) +
  geom_ysidecol(
    aes(x = numInCat), data = . %&gt;% distinct(gs_name, numInCat),
  ) +
  geom_xsidecol(
    aes(y = mapped),
    data = n_mapped %&gt;% 
      dplyr::filter(status %in% names(goseq_group_sig)) %&gt;% 
      dplyr::rename(group = status),
    width = 0.5
  ) +
  scale_x_discrete(labels = label_wrap(10)) +
  scale_ysidex_continuous(expand = expansion(c(0, 0.15))) +
  scale_xsidey_continuous(
    expand = expansion(c(0, 0.15))
  ) +
  scale_fill_viridis_c(option = &quot;inferno&quot;, begin = 0.2) +
  scale_size(range = c(0, 5), labels = percent) + 
  labs(
    x = c(), y = c(), size = &quot;% Mapped\nGenes&quot;, fill = expr(paste(-log[10], &quot;p&quot;))
  ) +
  theme(
    panel.grid = element_blank(), 
    axis.text = element_text(size = 8),
    legend.position = &quot;right&quot;,
    ggside.panel.scale.x = 0.2,
    ggside.panel.scale.y = 0.3,
    ggside.axis.text.x.bottom = element_text(angle = 270, hjust = 0, vjust = 0.5)
  ) </code></pre>
<div class="figure" style="text-align: center">
<img src="AR_E2_E2DHT_ER_E2_E2DHT_pairwise_comparison_files/figure-html/plot-all-enriched-1.png" alt="*Combined enrichment across all groups, showing only significant results for enrichment. The top panel shows how many genes were mapped to sites for each group, whilst the right panel shows gene set size. Point sizes indicate the proportion of mapped genes which are from each pathway.*" width="672" />
<p class="caption">
<em>Combined enrichment across all groups, showing only significant
results for enrichment. The top panel shows how many genes were mapped
to sites for each group, whilst the right panel shows gene set size.
Point sizes indicate the proportion of mapped genes which are from each
pathway.</em>
</p>
</div>
</div>
</div>
<div id="network-plots-1" class="section level3 tabset">
<h3 class="tabset">Network Plots</h3>
<pre class="r"><code>htmltools::tagList(
  names(tg_group_res) %&gt;% 
    mclapply(
      function(x) {
        ## Export the image
        img_out &lt;- file.path(
          fig_path,
          x %&gt;% 
            str_replace_all(&quot; &quot;, &quot;_&quot;) %&gt;% 
            str_replace_all(&quot;_-_&quot;, &quot;-&quot;) %&gt;% 
            paste0(&quot;_network.&quot;, fig_type)
        )
        fig_fun(
          filename = img_out,
          width = knitr::opts_current$get(&quot;fig.width&quot;),
          height = knitr::opts_current$get(&quot;fig.height&quot;)
        )
        p &lt;- tg_group_res[[x]] %&gt;%
          ggraph(layout = net_layout, weights =oc^2) +
          geom_edge_link(aes(width = oc^2, alpha = oc^2)) +
          geom_node_point(
            aes(fill = -log10(pval), size = numDEInCat),
            shape = 21
          ) +
          geom_node_text(
            aes(label = label),
            colour = &quot;black&quot;, size = 3, 
            data = . %&gt;%
              mutate(
                label = str_replace_all(label, &quot;_&quot;, &quot; &quot;) %&gt;% str_trunc(60) %&gt;% str_wrap(width = 18)
              ),
            repel = TRUE,
            max.overlaps = max(10, round(length(tg_group_res[[x]]) / 4, 0)),
            bg.color = &quot;white&quot;, bg.r = 0.1
          ) +
          scale_x_continuous(expand = expansion(c(0.1, 0.1))) +
          scale_y_continuous(expand = expansion(c(0.1, 0.1))) +
          scale_fill_viridis_c(option = &quot;inferno&quot;, begin = 0.25) +
          scale_size_continuous(range = c(1, 10)) +
          scale_edge_width(range = c(1, 6), limits = c(0, 1)) +
          scale_edge_alpha(range = c(0.1, 0.4), limits = c(0, 1)) +
          guides(edge_alpha= &quot;none&quot;) +
          labs(size = &quot;Mapped Genes&quot;, edge_width = expr(paste(OC^2))) +
          theme_void()
        print(p)
        dev.off()
        ## Create html tags
        fig_link &lt;- str_extract(img_out, &quot;assets.+&quot;)
        cp &lt;- htmltools::tags$em(
          glue(
            &quot;
            Network plot showing enriched pathways mapped to genes 
            associated with {x}. 
            &quot;
          )
        )
        htmltools::div(
          htmltools::div(
            id = img_out %&gt;%
              basename() %&gt;%
              str_remove_all(glue(&quot;.{fig_type}$&quot;)) %&gt;%
              str_to_lower() %&gt;%
              str_replace_all(&quot;_&quot;, &quot;-&quot;),
            class=&quot;section level4&quot;,
            htmltools::h4(x),
            htmltools::div(
              class = &quot;figure&quot;, style = &quot;text-align: center&quot;,
              htmltools::img(src = fig_link, width = &quot;100%&quot;),
              htmltools::tags$caption(cp)
            )
          )
        )
      },
      mc.cores = threads
    )
)</code></pre>
<div>
<div id="ar-up-er-up-network" class="section level4">
<h4>AR Up - ER Up</h4>
<div class="figure" style="text-align: center">
<img src="assets/AR_E2_E2DHT-ER_E2_E2DHT/AR_Up-ER_Up_network.png" width="100%"/>
<caption>
<em>Network plot showing enriched pathways mapped to genes 
associated with AR Up - ER Up. </em>
</caption>
</div>
</div>
</div>
</div>
</div>
</div>
<div id="comparison-with-rna-seq" class="section level1">
<h1>Comparison With RNA Seq</h1>
<pre class="r"><code>library(fgsea)
library(cowplot)
library(BiocParallel)
library(metap)
register(MulticoreParam(workers = threads))
id2gene &lt;- structure(gtf_gene$gene_name, names = gtf_gene$gene_id)
rna_status &lt;- rnaseq %&gt;% 
  mutate(
    de_status = case_when(
      !!sym(rna_fdr_col) &lt; fdr_alpha &amp; !!sym(rna_lfc_col) &lt; 0 ~ &quot;Down&quot;,
      !!sym(rna_fdr_col) &lt; fdr_alpha &amp; !!sym(rna_lfc_col) &gt; 0 ~ &quot;Up&quot;,
      !!sym(rna_fdr_col) &gt;= fdr_alpha ~ &quot;Unchanged&quot;
    )
  ) %&gt;% 
  dplyr::select(gene_id, gene_name, de_status) %&gt;% 
  bind_rows(
    gtf_gene %&gt;% 
      subset(!gene_id %in% rnaseq$gene_id) %&gt;% 
      select(gene_id, gene_name) %&gt;% 
      mcols() %&gt;% 
      as_tibble() %&gt;% 
      mutate(de_status = &quot;Undetected&quot;)
  ) %&gt;% 
  mutate(
    de_status = factor(
      de_status, levels = str_to_title(names(colours$direction))
    )
  ) %&gt;% 
  arrange(gene_id)
rna_p_col &lt;- colnames(rnaseq)[
  str_detect(str_to_lower(colnames(rnaseq)), &quot;pval|p\\.val&quot;)
][1]
stopifnot(length(rna_p_col) == 1)
diff_windows &lt;- all_windows %&gt;% 
  as_tibble() 
diff_windows[[&quot;r1&quot;]] &lt;- rank(mcols(all_windows)[[c1_fdr]], ties.method = &quot;min&quot;)
diff_windows[[&quot;r2&quot;]] &lt;- rank(mcols(all_windows)[[c2_fdr]], ties.method = &quot;min&quot;)
diff_windows &lt;- diff_windows %&gt;% 
  dplyr::arrange(r1 + r2) %&gt;% 
  dplyr::select(gene_id, status, r1, r2) %&gt;% 
  unnest(everything()) %&gt;% 
  distinct(gene_id, .keep_all = TRUE) %&gt;% 
  droplevels() %&gt;% 
  split(.$status) %&gt;% 
  lapply(pull, &quot;gene_id&quot;) 
any_de &lt;- sum(rnaseq[[rna_fdr_col]] &lt; fdr_alpha) &gt; 0
n_mapped &lt;- all_windows %&gt;% 
  dplyr::select(status, detected) %&gt;% 
  as_tibble() %&gt;% 
  unnest(everything()) %&gt;% 
  distinct(status, detected) %&gt;% 
  group_by(status) %&gt;% 
  summarise(detected = dplyr::n()) %&gt;% 
  right_join(n_mapped, by = &quot;status&quot;)</code></pre>
<div id="volcano-plots" class="section level2">
<h2>Volcano Plots</h2>
<pre class="r"><code>diff_windows %&gt;% 
  lapply(function(x) tibble(gene_id = x)) %&gt;% 
  bind_rows(.id = &quot;binding_status&quot;) %&gt;% 
  mutate(
    binding_status = factor(binding_status, levels = combs) %&gt;% 
      fct_lump_prop(0.01)
  ) %&gt;% 
  full_join(rnaseq) %&gt;% 
  dplyr::filter(!is.na(gene_name)) %&gt;% 
  mutate(
    binding_status = fct_explicit_na(binding_status, na_level = &quot;No Targets&quot;),
    de_status = case_when(
      DE &amp; !!sym(rna_lfc_col) &gt; 0 ~ &quot;Up&quot;,
      DE &amp; !!sym(rna_lfc_col) &lt; 0 ~ &quot;Down&quot;,
      TRUE ~ &quot;Unchanged&quot;
    ) %&gt;% 
      factor(levels = c(&quot;Up&quot;, &quot;Down&quot;, &quot;Unchanged&quot;))
  ) %&gt;% 
  ggplot(
    aes(!!sym(rna_lfc_col), -log10(!!sym(rna_p_col)), colour = de_status)
  ) +
  geom_point() +
  geom_text_repel(
    aes(label = gene_name),
    data = . %&gt;% 
      arrange(!!sym(rna_p_col)) %&gt;% 
      dplyr::filter(DE) %&gt;% 
      split(.$binding_status) %&gt;% 
      lapply(dplyr::slice, 1:2) %&gt;% 
      bind_rows(),
    show.legend = FALSE
  ) +
  geom_text(
    aes(label = lab),
    data = . %&gt;% 
      group_by(binding_status) %&gt;% 
      summarise(n = dplyr::n()) %&gt;% 
      mutate(lab = glue(&quot;n = {comma(n)}&quot;)),
    x = min(rnaseq[[rna_lfc_col]]) + 0.9 * diff(range(rnaseq[[rna_lfc_col]])),
    y = 1,
    colour = &quot;black&quot;,
    inherit.aes = FALSE,
    show.legend = FALSE
  ) +
  facet_wrap(~binding_status) +
  scale_colour_manual(
    values = colours$direction[c(&quot;up&quot;, &quot;down&quot;, &quot;unchanged&quot;)] %&gt;% 
      unlist() %&gt;% 
      setNames(str_to_title(names(.))) 
  ) +
  labs(colour = &quot;DE Status&quot;) +
  theme(legend.position = &quot;right&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="AR_E2_E2DHT_ER_E2_E2DHT_pairwise_comparison_files/figure-html/plot-volcano-1.png" alt="*Volcano plots for differential expression separated by target binding patterns. Some genes will have multiple target binding regions mapped to them, and may appear in multiple panels. The two most highly ranked genes for differential expression are labelled in each panel. All DE genes are listed with full binding patterns in the table below.*" width="960" />
<p class="caption">
<em>Volcano plots for differential expression separated by target
binding patterns. Some genes will have multiple target binding regions
mapped to them, and may appear in multiple panels. The two most highly
ranked genes for differential expression are labelled in each panel. All
DE genes are listed with full binding patterns in the table below.</em>
</p>
</div>
</div>
<div id="de-genes-and-pairwise-binding" class="section level2">
<h2>DE Genes And Pairwise Binding</h2>
<pre class="r"><code>col_defs &lt;- list2(
      de = colDef(
        name = &quot;Gene&quot;, maxWidth = 120
      ),
      logFC = colDef(
        name = &quot;logFC&lt;br&gt;(RNA-Seq)&quot;,
        html = TRUE,
        aggregate = &quot;mean&quot;,
        maxWidth = 100,
        format = colFormat(digits = 2),
        style = JS(
            glue(
            &quot;function(rowInfo) {
              var value = rowInfo.row[&#39;logFC&#39;]
              if (value &lt; 0) {
                var color = &#39;{{colours$direction[[&#39;down&#39;]]}}&#39;
              } else if (value &gt; 0) {
                var color = &#39;{{colours$direction[[&#39;up&#39;]]}}&#39;
              } 
              return { color: color }
            }&quot;,
            .open = &quot;{{&quot;, .close = &quot;}}&quot;
            )
        )
      ),
      range = colDef(
        name = &quot;Range&quot;,
        aggregate = JS(
            &quot;function(values){
              var chrom = [];
              var rng = [];
              var start = [];
              var end = [];
              for (i = 0; i &lt; values.length; i++) {
                chrom[i] = values[i].split(&#39;:&#39;)[0];
                rng[i] = values[i].split(&#39;:&#39;)[1];
                start[i] = rng[i].split(&#39;-&#39;)[0];
                end[i] = rng[i].split(&#39;-&#39;)[1];
              }
              
              min_start = Math.min(...start);
              max_end = Math.max(...end);
              var ret_val = [...new Set(chrom)] + &#39;:&#39; + min_start.toString() + &#39;-&#39; + max_end.toString();
              
              return ret_val
              
            }&quot;
          )
      ),
      region = colDef(
        aggregate = &quot;unique&quot;,
        name = &quot;Region&quot;
      ),
      &quot;{c1_logfc}&quot; := colDef(
        name = comps[[1]],
        maxWidth = 100,
        format = colFormat(digits = 2),
        style = function(value) {
          cl &lt;- ifelse(
            value &gt; 0, colours$direction[[&quot;up&quot;]], colours$direction[[&quot;down&quot;]]
          )
          list(color = cl)
        }
      ),
      &quot;{c2_logfc}&quot; := colDef(
        name = comps[[2]],
        maxWidth = 100,
        format = colFormat(digits = 2),
        style = function(value) {
          cl &lt;- ifelse(
            value &gt; 0, colours$direction[[&quot;up&quot;]], colours$direction[[&quot;down&quot;]]
          )
          list(color = cl)
        }
      ),
      &quot;{c1_status}&quot; := colDef(
        aggregate = &quot;unique&quot;,
        name = comps[[1]],
        maxWidth = 150
      ),
      &quot;{c2_status}&quot; := colDef(
        aggregate = &quot;unique&quot;,
        name = comps[[2]],
        maxWidth = 150
      )
    ) 
if (has_features) col_defs$feature &lt;- colDef(
  name = &quot;Feature&quot;,
  aggregate = &quot;unique&quot;   
)
htmltools::tags$caption(
  htmltools::em(
    glue(
    &quot;
    All differentially expressed genes where binding was detected in both 
    comparisons.
    &quot;
    )
  )
)</code></pre>
<caption>
<em>All differentially expressed genes where binding was detected in both 
comparisons.</em>
</caption>
<pre class="r"><code>de_genes_both_comps &lt;- all_windows %&gt;% 
  as_tibble()
## Placeholder until conda-forge sort out R4.2.1
colnames(de_genes_both_comps) &lt;- c(&quot;range&quot;, colnames(mcols(all_windows)))
de_genes_both_comps &lt;- de_genes_both_comps %&gt;% 
  dplyr::select(
    any_of(c(&quot;range&quot;, &quot;region&quot;, &quot;feature&quot;)), ends_with(&quot;logFC&quot;), ends_with(&quot;status&quot;), 
    any_of(c(&quot;detected&quot;))
  ) %&gt;% 
  mutate(
    de = lapply(detected, intersect, dplyr::filter(rnaseq, !!sym(rna_fdr_col) &lt; fdr_alpha)$gene_name)
  ) %&gt;% 
  dplyr::filter(
    vapply(de, length, integer(1)) &gt; 0, 
    str_detect(status, &quot;Up|Down&quot;),
    !str_detect(status, &quot;Undetected&quot;)
  ) %&gt;% 
  unnest(de) %&gt;% 
  left_join(rnaseq, by = c(&quot;de&quot; = &quot;gene_name&quot;)) %&gt;% 
  mutate(direction = ifelse(logFC &gt; 0, &quot;Up&quot;, &quot;Down&quot;)) %&gt;% 
  droplevels() %&gt;% 
  dplyr::select(
    de, logFC, any_of(c(&quot;range&quot;, &quot;region&quot;, &quot;feature&quot;)), ends_with(&quot;logFC&quot;),
    ends_with(&quot;_status&quot;)
  ) 
de_genes_both_comps %&gt;% 
  reactable(
    filterable = TRUE,
    groupBy = &quot;de&quot;,
    columns = col_defs,
    columnGroups = list(
      colGroup(
        name = &quot;ChIP logFC&quot;, 
        columns = c(as.character(c1_logfc), as.character(c2_logfc))
      ),
      colGroup(
        name = &quot;ChIP Status&quot;,
        columns =  c(as.character(c1_status), as.character(c2_status))
      )
    ),
    theme = reactableTheme(style = list(fontSize = 14))
  )</code></pre>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-09ea66caf5511b802473" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-09ea66caf5511b802473">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"de":["TENT5C","TENT5C","GLUL","GREB1","IL1R1","PLCL1","CSRNP1","ARHGEF26","ARHGEF26","UGDH","AC021148.3","UGDH","AC021148.3","PARM1","LEF1","LEF1","ARHGEF37","SLC26A2","WWC1","AL033523.1","FKBP5","SASH1","PDE10A","PDE10A","TRGC1","TRGC1","TRGC1","TRGC1","TRGC1","SYBU","SYBU","MPDZ","AQP3","TBC1D2","ADD3","SCUBE2","SCUBE2","EHF","EHF","EHF","EHF","EHF","PTPRJ","CD9","GPRC5A","TMTC2","NFKBIA","HSD11B2","OSGIN1","KRT15","ZNF652","TBC1D16","TBC1D16","CABLES1","CABLES1","NEDD4L","NEDD4L","BCL2","KDM4B","SEC14L2","PISD","PISD","PISD","WWC3","WWC3","TMEM164"],"logFC":[0.453361442217992,0.453361442217992,0.49603258742251,0.672489104048776,-0.520904713359755,1.40224703303471,0.724290454212561,0.865417410958739,0.865417410958739,1.52195535860114,3.323799859993,1.52195535860114,3.323799859993,1.10554646171515,0.783184511974607,0.783184511974607,0.959466645703423,0.583264078166994,0.563530986380658,2.21378646833637,1.99899166960751,0.836022499007808,1.30727983464792,1.30727983464792,0.610727700262191,0.610727700262191,0.610727700262191,0.610727700262191,0.610727700262191,0.84070988458889,0.84070988458889,0.497098237600896,2.36942177842281,0.619224444495583,0.574358158492719,0.548685918463357,0.548685918463357,0.591050336429137,0.591050336429137,0.591050336429137,0.591050336429137,0.591050336429137,0.601234639862719,0.457093842888302,0.505022117715957,0.584928679933631,0.486175076389985,1.13830350246976,0.656529395236609,0.529285680056739,0.768462734396911,0.48074000543264,0.48074000543264,-0.466222240679165,-0.466222240679165,0.537173436697143,0.537173436697143,-0.423119324887046,0.738557156216351,2.65364804947665,1.09629114216671,1.09629114216671,1.09629114216671,0.592409511594803,0.592409511594803,0.430875244289631],"range":["chr1:118132831-118133250","chr1:118174861-118175560","chr1:182309671-182310180","chr2:11646181-11646600","chr2:102739441-102739800","chr2:198709521-198710080","chr3:39171841-39172520","chr3:153935701-153937050","chr3:153935701-153937050","chr4:39540961-39541400","chr4:39540961-39541400","chr4:39545521-39546160","chr4:39545521-39546160","chr4:76024591-76024840","chr4:108966811-108967240","chr4:108979761-108980320","chr5:148862281-148862840","chr5:149257651-149258240","chr5:167780081-167780880","chr6:3669961-3670560","chr6:35750121-35750640","chr6:148579801-148580480","chr6:165980761-165981090","chr6:166061241-166061640","chr7:38294401-38294960","chr7:38304121-38304680","chr7:38310481-38311080","chr7:38313081-38313600","chr7:38345441-38345880","chr8:110592321-110592720","chr8:110605401-110606000","chr9:13064081-13064480","chr9:33415521-33416000","chr9:100908561-100909160","chr10:111762241-111762960","chr11:9105541-9106080","chr11:9131491-9132030","chr11:34653241-34654020","chr11:34655131-34655600","chr11:34661441-34662280","chr11:34689641-34690240","chr11:34771641-34772320","chr11:48063321-48064040","chr12:6344321-6345120","chr12:13036121-13036880","chr12:83324801-83325640","chr14:35872921-35873520","chr16:67495481-67496280","chr16:83980261-83980980","chr17:39695441-39695920","chr17:47468041-47468960","chr17:77967001-77967600","chr17:78076721-78077080","chr18:20840601-20840920","chr18:20853541-20853870","chr18:55755151-55755600","chr18:55755721-55756360","chr18:60822841-60823360","chr19:4991361-4992560","chr22:30804721-30805120","chr22:31975721-31976280","chr22:31981401-31982730","chr22:31981401-31982730","chrX:9972041-9972480","chrX:10019761-10020440","chrX:109416331-109417000"],"region":["Intergenic (<10kb)","Intergenic (<10kb)","Intergenic (<10kb)","Intron","Intron","Intron","Promoter (-1500/+500)","Intron","Intron","Exon","Exon","Exon","Exon","Upstream Promoter (-5kb)","Intergenic (<10kb)","Intron","Intergenic (<10kb)","Intron","Intron","Intron","Upstream Promoter (-5kb)","Intron","Intron","Intron","Upstream Promoter (-5kb)","Intron","Upstream Promoter (-5kb)","Promoter (-1500/+500)","Intergenic (<10kb)","Promoter (-1500/+500)","Intron","Intergenic (>10kb)","Intergenic (<10kb)","Intron","Promoter (-1500/+500)","Intron","Intron","Promoter (-1500/+500)","Intron","Upstream Promoter (-5kb)","Intergenic (<10kb)","Intergenic (<10kb)","Upstream Promoter (-5kb)","Promoter (-1500/+500)","Intergenic (<10kb)","Intron","Promoter (-1500/+500)","Upstream Promoter (-5kb)","Upstream Promoter (-5kb)","Intergenic (>10kb)","Intron","Promoter (-1500/+500)","Intron","Promoter (-1500/+500)","Intron","Upstream Promoter (-5kb)","Upstream Promoter (-5kb)","Upstream Promoter (-5kb)","Intron","Promoter (-1500/+500)","Promoter (-1500/+500)","Intron","Intron","Intergenic (>10kb)","Intron","Exon"],"feature":["Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","No Feature","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","No Feature","No Feature","Enhancer","Enhancer","Enhancer","No Feature","No Feature","Enhancer","No Feature","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","No Feature","Enhancer","Enhancer","Enhancer","No Feature","Enhancer","Enhancer","No Feature","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","No Feature","Enhancer","No Feature","Enhancer","Enhancer","Enhancer","No Feature","Enhancer","No Feature"],"AR_logFC":[1.20069073339936,0.888753594392706,0.827558565800936,1.0198282880489,0.751517652702296,0.635024149662066,3.8790875950919,2.34518232721692,1.5984565983779,1.54063177656409,1.54063177656409,2.3554001283601,2.3554001283601,0.898173592768965,0.940590120860502,2.11794223928506,1.99837996311045,1.62886849557129,1.87945591283767,3.17403123993926,2.09041200930191,2.23240980353371,1.85675982641067,2.61696364796369,1.73585441759596,1.13598596402466,1.68549905284668,1.14182519286861,1.13315423693867,1.54716331337146,2.24438195173873,0.775979437903524,0.942550016207118,1.40852109677808,1.45377771829631,2.18896912173123,1.03350034578139,1.27361016931518,1.37724310534282,2.2590417855773,1.04089912441366,2.25154139180708,1.27371440057597,1.07156737848282,1.01671516349836,1.53673415060846,2.58169012960605,2.20363321919353,1.5335672109911,1.31722692574251,2.82559611455131,1.16883333626089,2.19547654288621,0.990126224932691,0.847130267054235,1.36670615247045,1.1643173338971,0.885915140173766,0.86323111921097,2.6107685864327,1.33832353430239,1.7422540488503,1.26685614934181,0.803640906586706,2.87306082531428,2.30899999002722],"ER_logFC":[0.258141275130583,0.0126081224223805,-0.044310029663596,-0.0397440915894326,-0.282901302898653,0.110818383218942,1.44603184900199,0.167933365698347,0.167933365698347,0.00389128120499152,0.00389128120499152,1.48462370450969,1.48462370450969,-0.23226044784898,0.00234989473434632,0.430690709313056,0.339357412595007,0.274269984634905,0.60177445266823,1.43572113279025,-0.0458675831740035,1.62683889186267,1.36534909243693,0.56038454921546,0.114421720753591,-0.00139680784055747,0.516948779323436,0.135575621207194,-0.0405155605077144,0.437538739717138,1.20645975892099,0.00516340583640056,-0.217479744783929,-0.219789349514639,-0.0703638112826583,0.283823005846383,-0.175755113860391,-0.256290591775664,0.15482037122092,0.616004102260624,0.212794525216262,1.29137546386539,-0.222972948175863,0.229101495234619,0.140284521417072,0.416007283763226,0.467651679942806,0.998649728105702,0.253457059677004,0.0538953632362705,1.6264950128196,0.197656508899513,1.31781411756384,-0.141839619175297,-0.261441795664313,0.729028673081445,0.63545254474465,-0.166075391198822,1.09745625118093,1.95160124538619,0.310735776510054,0.479549894255308,0.479549894255308,0.332375941728761,1.14500848777582,0.24825870074203],"AR_status":["AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up","AR Up"],"ER_status":["ER Unchanged","ER Unchanged","ER Unchanged","ER Unchanged","ER Unchanged","ER Unchanged","ER Up","ER Unchanged","ER Unchanged","ER Unchanged","ER Unchanged","ER Up","ER Up","ER Unchanged","ER Unchanged","ER Unchanged","ER Unchanged","ER Unchanged","ER Unchanged","ER Up","ER Unchanged","ER Up","ER Up","ER Unchanged","ER Unchanged","ER Unchanged","ER Unchanged","ER Unchanged","ER Unchanged","ER Unchanged","ER Up","ER Unchanged","ER Unchanged","ER Unchanged","ER Unchanged","ER Unchanged","ER Unchanged","ER Unchanged","ER Unchanged","ER Unchanged","ER Unchanged","ER Up","ER Unchanged","ER Unchanged","ER Unchanged","ER Unchanged","ER Unchanged","ER Up","ER Unchanged","ER Unchanged","ER Up","ER Unchanged","ER Up","ER Unchanged","ER Unchanged","ER Unchanged","ER Unchanged","ER Unchanged","ER Up","ER Up","ER Unchanged","ER Unchanged","ER Unchanged","ER Unchanged","ER Up","ER Unchanged"]},"columns":[{"id":"de","name":"Gene","type":"character","maxWidth":120},{"id":"logFC","name":"logFC<br>(RNA-Seq)","type":"numeric","aggregate":"mean","format":{"cell":{"digits":2},"aggregated":{"digits":2}},"html":true,"maxWidth":100,"style":"function(rowInfo) {\n  var value = rowInfo.row['logFC']\n  if (value < 0) {\n    var color = '#0571B0'\n  } else if (value > 0) {\n    var color = '#CA0020'\n  } \n  return { color: color }\n}"},{"id":"range","name":"Range","type":"character","aggregate":"function(values){\n              var chrom = [];\n              var rng = [];\n              var start = [];\n              var end = [];\n              for (i = 0; i < values.length; i++) {\n                chrom[i] = values[i].split(':')[0];\n                rng[i] = values[i].split(':')[1];\n                start[i] = rng[i].split('-')[0];\n                end[i] = rng[i].split('-')[1];\n              }\n              \n              min_start = Math.min(...start);\n              max_end = Math.max(...end);\n              var ret_val = [...new Set(chrom)] + ':' + min_start.toString() + '-' + max_end.toString();\n              \n              return ret_val\n              \n            }"},{"id":"region","name":"Region","type":"factor","aggregate":"unique"},{"id":"feature","name":"Feature","type":"character","aggregate":"unique"},{"id":"AR_logFC","name":"AR","type":"numeric","format":{"cell":{"digits":2},"aggregated":{"digits":2}},"maxWidth":100,"style":[{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"}]},{"id":"ER_logFC","name":"ER","type":"numeric","format":{"cell":{"digits":2},"aggregated":{"digits":2}},"maxWidth":100,"style":[{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#0571B0"},{"color":"#0571B0"},{"color":"#0571B0"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#0571B0"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#0571B0"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#0571B0"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#0571B0"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#0571B0"},{"color":"#0571B0"},{"color":"#0571B0"},{"color":"#CA0020"},{"color":"#0571B0"},{"color":"#0571B0"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#0571B0"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#0571B0"},{"color":"#0571B0"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#0571B0"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"}]},{"id":"AR_status","name":"AR","type":"factor","aggregate":"unique","maxWidth":150},{"id":"ER_status","name":"ER","type":"factor","aggregate":"unique","maxWidth":150}],"columnGroups":[{"name":"ChIP logFC","columns":["AR_logFC","ER_logFC"]},{"name":"ChIP Status","columns":["AR_status","ER_status"]}],"groupBy":["de"],"filterable":true,"theme":{"style":{"fontSize":14}},"dataKey":"9cf0f6b90892a1b2b3fd4d45d4a7bfb7"},"children":[]},"class":"reactR_markup"},"evals":["tag.attribs.columns.1.style","tag.attribs.columns.2.aggregate"],"jsHooks":[]}</script>
</div>
<div id="binding-pattern-gsea" class="section level2 tabset">
<h2 class="tabset">Binding Pattern GSEA</h2>
<pre class="r"><code>rna_dir_ranks &lt;- rnaseq %&gt;% 
  mutate(
    ranking_stat = -sign(!!sym(rna_lfc_col))*log10(!!sym(rna_p_col)) %&gt;% 
      setNames(gene_id)
  ) %&gt;% 
  arrange(ranking_stat) %&gt;% 
  dplyr::filter(!!sym(rna_p_col) &lt; 1) %&gt;% 
  pull(&quot;ranking_stat&quot;) 
rna_sig_ranks &lt;- rna_dir_ranks %&gt;% 
  abs() %&gt;% 
  sort()
status_gs &lt;- all_windows %&gt;% 
  select(status, starts_with(&quot;gene&quot;)) %&gt;% 
  as_tibble() %&gt;% 
  split(.$status) %&gt;% 
  lapply(pull, &quot;gene_id&quot;) %&gt;% 
  lapply(unlist) %&gt;% 
  lapply(unique) %&gt;% 
  .[vapply(., length, integer(1)) &gt; min_gs_size]</code></pre>
<p>GSEA analysis was performed taking the set of genes mapped to each
binding pattern. This was initially performed ranking the RNA-Seq
results in order of significance, with most significantly up-regulated
and one end and the most significantly down-regulated at the other,
i.e. Directional GSEA A secondary analysis was performed taking only
statistical significance into account, without including the direction
of any change in expression, i.e. Non_Directional GSEA. This second
analysis allows for detecting regulatory changes due to altered binding,
but which are spread between both up- and down-regulation, as may be
induced by the presence of other co-factors not under direct
investigation here.</p>
<div id="directional-gsea" class="section level3">
<h3>Directional GSEA</h3>
<pre class="r"><code>gsea_dir &lt;- fgsea(status_gs, rna_dir_ranks)
gsea_dir_sig &lt;- gsea_dir %&gt;%
  arrange(pval) %&gt;%
  mutate(padj = p.adjust(pval, adj_method)) %&gt;% 
  dplyr::filter(padj &lt; enrich_alpha, size &gt;= min_sig) %&gt;%
  as_tibble()
p &lt;- gsea_dir_sig %&gt;% 
  dplyr::slice(1:9) %&gt;% 
  pull(&quot;pathway&quot;) %&gt;% 
  lapply(
    function(x) {
      plotEnrichment(status_gs[[x]], rna_dir_ranks) +
        ggtitle(x) +
        theme(
          plot.title = element_text(hjust = 0.5, size = 10)
        )
    }
  )</code></pre>
<pre class="r"><code>cp &lt;-  htmltools::em(
  glue(
    &quot;Combined windows were mapped to genes, and their position amongst the &quot;,
    &quot;RNA-Seq results was assessed. {nrow(gsea_dir_sig)} sets of windows were &quot;,
    &quot;associated with changes in gene expression, using the sign of &quot;, 
    &quot;fold-change and ranking statistic to initially rank the &quot;, 
    &quot;{comma(nrow(rnaseq), 1)} genes considered as detected.&quot;
  )
)
tbl &lt;- gsea_dir_sig %&gt;%
  mutate(
    `Edge Size` = vapply(leadingEdge, length, integer(1)),
    leadingEdge = lapply(leadingEdge, function(x) id2gene[x]) %&gt;% 
      vapply(paste, character(1), collapse = &quot;; &quot;),
    Direction = ifelse(NES &gt; 0, &quot;\u21E7 Up-regulated&quot;, &quot;\u21E9 Down-regulated&quot;)
  ) %&gt;%
  separate(pathway, comps, sep = &quot; - &quot;) %&gt;% 
  dplyr::select(
    all_of(comps), Windows = size, Direction, 
    p = pval, padj, `Edge Size`, `Leading Edge` = leadingEdge
  ) %&gt;% 
  reactable(
    filterable = TRUE,
    columns = list2(
      &quot;{comps[[1]]}&quot; := colDef(
        maxWidth = 150,
        cell = function(value) {
          html_symbol &lt;- &quot;&quot;
          if (str_detect(value, &quot;Up&quot;)) html_symbol &lt;- &quot;\u21E7&quot;
          if (str_detect(value, &quot;Down&quot;)) html_symbol &lt;- &quot;\u21E9&quot;
          paste(html_symbol, value)
        },
        style = function(value) {
          colour &lt;- case_when(
            str_detect(value, &quot;Up&quot;) ~ colours$direction[[&quot;up&quot;]],
            str_detect(value, &quot;Down&quot;) ~ colours$direction[[&quot;down&quot;]],
            TRUE ~ colours$direction[[&quot;unchanged&quot;]]
          )
          list(color = colour)
        }
      ),
      &quot;{comps[[2]]}&quot; := colDef(
        maxWidth = 150,
        cell = function(value) {
          html_symbol &lt;- &quot;&quot;
          if (str_detect(value, &quot;Up&quot;)) html_symbol &lt;- &quot;\u21E7&quot;
          if (str_detect(value, &quot;Down&quot;)) html_symbol &lt;- &quot;\u21E9&quot;
          paste(html_symbol, value)
        },
        style = function(value) {
          colour &lt;- case_when(
            str_detect(value, &quot;Up&quot;) ~ colours$direction[[&quot;up&quot;]],
            str_detect(value, &quot;Down&quot;) ~ colours$direction[[&quot;down&quot;]],
            TRUE ~ colours$direction[[&quot;unchanged&quot;]]
          )
          list(color = colour)
        }
      ),
      Windows = colDef(maxWidth = 80),
      Direction = colDef(
        name = &quot;Gene Direction&quot;,
        maxWidth = 120,
        style = function(value) {
          colour &lt;- ifelse(
            str_detect(value, &quot;Up&quot;), 
            colours$direction[[&quot;up&quot;]], 
            colours$direction[[&quot;down&quot;]]
          )
          list(color = colour)
        },
      ),
      p = colDef(
        cell = function(value) ifelse(
          value &lt; 0.001,
          sprintf(&quot;%.2e&quot;, value),
          sprintf(&quot;%.3f&quot;, value)
        ), 
        maxWidth = 80
      ),
      padj = colDef(
        name = glue(&quot;p&lt;sub&gt;{adj_method}&lt;/sub&gt;&quot;), html = TRUE,
        cell = function(value) {
          ifelse(
            value &lt; 0.001, sprintf(&quot;%.2e&quot;, value), sprintf(&quot;%.3f&quot;, value)
          )
        }
      ),
      &quot;Edge Size&quot; = colDef(maxWidth = 80),
     &quot;Leading Edge&quot; = colDef(
       minWidth = 150,
       cell = function(value) with_tooltip(value, width = 50)
     ) 
    )
  )
div(class = &quot;table&quot;,
  div(class = &quot;table-header&quot;,
      div(class = &quot;caption&quot;, cp),
      tbl
  )
)</code></pre>
<div class="table">
<div class="table-header">
<div class="caption">
<em>Combined windows were mapped to genes, and their position amongst the RNA-Seq results was assessed. 4 sets of windows were associated with changes in gene expression, using the sign of fold-change and ranking statistic to initially rank the 13,043 genes considered as detected.</em>
</div>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-42852ecb743fa6a6a96a" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-42852ecb743fa6a6a96a">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"AR":["AR Up","AR Up","AR Up","AR Down"],"ER":["ER Undetected","ER Unchanged","ER Up","ER Undetected"],"Windows":[4621,1237,260,6],"Direction":["⇧ Up-regulated","⇧ Up-regulated","⇧ Up-regulated","⇩ Down-regulated"],"p":[3.76426850466288e-25,3.58294848541845e-11,6.33458851959355e-08,0.00178213922336727],"padj":[3.01141480373031e-24,1.43317939416738e-10,1.68922360522495e-07,0.00356427844673455],"Edge Size":[616,144,55,1],"Leading Edge":["AQP3; UGDH; FKBP5; SEC14L2; CLDN8; EAF2; STEAP2; RANBP3L; HSD11B2; AC021148.3; PISD; ZNF652; ZNF689; OTULINL; ANKRD22; ARHGEF26; AKR1C3; EHF; KRT4; PARM1; ARHGEF37; SERHL2; GLUL; THBS1; NRP1; EPGN; WWC3; CROT; GREB1; AL033523.1; PALMD; TTC12; PTPRJ; PWWP3B; ADD3; SLC16A9; SYBU; MYLIP; RAB3B; ARHGEF28; NEDD4L; CSRNP1; GPCPD1; GPRC5A; WWC1; SASH1; PRR15; CNTNAP2; TBC1D16; SCUBE2; SLC26A2; FRK; DIO2; SOWAHC; SERHL; PER1; AKAP13; SLC22A23; TBC1D2; CRYBG1; ASRGL1; LEF1; TMTC2; TENT5C; CD9; PDE10A; PLCL1; SPRED1; PPP3CA; VSTM2L; HS3ST3B1; IQGAP2; STON2; JUN; PRKAB2; NADK2; PMEPA1; MPDZ; NDRG1; TMEM164; IL20RA; NFKBIA; RRP7A; SLC16A6; MCCC2; GRHL2; AC007255.1; HEY2; RRP7BP; PLEKHD1; LNX2; KLF13; RASSF6; FHDC1; PARP14; PREP; FAM214A; AC092053.4; TGFBR2; CYP2B7P; ECI2; FGFR2; AL135905.2; TMCC3; SERPINI1; LMO3; ASB13; SSTR2; PARP9; SLC25A18; MTSS1; ADH1C; AL606500.1; ATP13A4; LRIG3; SOWAHA; STEAP4; FLRT3; CDC42EP3; MKLN1-AS; KLF9; PEA15; VLDLR; NXPE3; TPD52; INSYN2A; SGK3; EMP1; TBC1D8; MKRN2OS; EPN2; BCL2L1; RAB6C; SORD; ELL2; STARD13-AS; STARD13; MYCBP2; AL162377.1; KLF5; ZNF761; AL445309.1; DIPK2A; FAM107B; GNA12; TTC21A; WWTR1; DOCK4; ZBTB7C; F2R; TMEM17; TP53TG1; CCDC40; CNNM1; ALCAM; PID1; PAQR5; LRCH1; SEMA6D; TMEM131L; MTMR9; ATP10A; GNMT; SNX25; VWFP1; NUAK1; PDE9A; CRAT; HECTD3; NAMPT; PACSIN2; AK4; SPSB1; GASK1B; CHN1; CBX4; SLC1A1; TNFAIP8; AMACR; GPR137C; ABCC4; ARHGEF26-AS1; TESK2; TOE1; LRIG1; TPM3P9; ZNF442; RASSF3; NYNRIN; CCDC136; BCL6; UAP1; AC093866.1; FAH; HERC3; KCNB1; ACACA; SLC38A2; GLDC; CRISPLD1; DSCAM-AS1; ADCY2; TIPARP; PEX10; LYPD6; PXDC1; DOCK8; SLC2A1; ACAA1; HSPA2; ADIRF-AS1; PXYLP1; TGFBR3; SRMS; LPAR3; ADAT2; CA13; SLC16A1; FZD5; AC024896.1; RGL1; NSRP1; AC105285.1; SLC22A5; RN7SL674P; SPRY4; RWDD3; HMGCS1; CRYM; DLGAP1-AS1; TFPI2; FHL2; TTC32; C20orf144; CASD1; GET4; ATP7B; MRFAP1; DDR2; RFLNA; SLC44A5; PARTICL; AL158211.5; NAGK; SORD2P; PRR15L; HNRNPA1P48; TSPAN33; PRAG1; SMKR1; KIF21A; LDHB; DBI; KIAA0825; AC022137.4; HSP90AA2P; VMP1; PDIA5; DOK7; KRT10; LEKR1; RPH3AL; LINC01881; IQCB1; TES; PRKAA2; FAM198B-AS1; EGLN3; TMEM63C; VIPR1; SPRED2; LMO4; CHPT1; ZNRF1; VWDE; DEPTOR; NRARP; CLIP4; ID3; DUSP16; ZBTB10; SLC6A17; ZNF837; KLRG2; AC138028.4; AKAP7; AZGP1P1; PSMD6-AS2; STARD5; AC113404.3; HGD; ZMYND10; NEDD4; PGAM4; ZNF365; ZFX; CSF1; LOXL3; CD163L1; CEP112; PSCA; NOL6; ID1; SKAP1; COL5A2; GADD45B; ENPP1; ZNF674-AS1; AIFM2; MNS1; PLLP; CNOT2-DT; PPARGC1B; AL135905.1; MAP1B; IFI6; MFF-DT; KL; CCDC96; VAMP5; LIFR; UNC5B; CARMIL1; GRAMD1B; SLC20A1; ACSM1; AP001816.1; ANKRD44; RBM26-AS1; LHFPL6; TRIM36; MAFB; EPB41L2; TMEM62; LRRC26; ARL15; CCDC34; SMIM14; AC115223.1; TOR1AIP2; RFESD; CCDC85A; LRRC75A; HELZ2; DDX60; IGFBP3; APBA1; SMAD9; RHOBTB1; RRS1; G3BP2; TRIM35; TXNDC16; AC112777.1; COL12A1; CYTH1; RAB11B-AS1; ZFP36L2; STAT5A; NKX2-1; HSPBAP1; SLC6A11; SERTAD1; RMDN2; AC092171.5; CFAP45; HES1; UROD; HNRNPA3P6; TSFM; TSPAN5; AC007405.4; ERVK3-1; EML6; PRDM4; RASD2; ATP1A1; PTCH1; AADAT; BPIFA4P; PXMP4; NKILA; RNF166; QDPR; SLC39A14; EFNA4; PTCHD1; SLC16A4; HMGCS2; PLEKHG4B; MOB3C; RMI1; PPIAP29; LDHD; TFCP2L1; AHR; CA8; AC022137.3; BCCIP; BCAR3; MIR3936HG; MCOLN2; LINC02615; UGT8; AREG; PHKA1; SIAH2; SALL2; AC090517.5; POLA2; CACNA2D2; ZEB1-AS1; TSKU; EFNA3; PIP4P2; AC141930.1; STAMBPL1; AC138028.2; MTHFS; ZFAND4; TUT7; RPS2P55; C9orf106; LYSMD4; AL451165.2; RNF215; C1QTNF3; MPC2; KCNK1; FZD8; SPRYD7; BRD7P2; SMCO4; WDFY3-AS2; OPLAH; RGS2; AL121906.2; SIK1B; FMO5; FZD1; UXS1; SIK1; GAR1; PRIMA1; AC104825.1; ATP6V1B2; CASC2; FAM241B; ANKRD62P1; PPM1M; AC007743.1; FAM189A2; DNMBP; AL139385.1; SLC24A3; ANP32B; CACYBPP2; SCD5; EML5; MAD2L1BP; AC084082.1; RASGEF1A; ITGAV; NCOA1; SMPDL3A; PDE3B; ZDHHC8P1; PRELID2; RFLNB; FAM53A; GGT7; GPR37; ACTN2; CTU1; FAM81A; AC104109.2; CYP4Z2P; Z99572.1; PLEKHG5; POU2F1; EEPD1; RHOB; KCNMA1; FBXL16; CRY2; ELAC1; VPS37D; FKBP9P1; DHCR24; NTRK3; CAMKMT; TRIM68; FBXO48; OLMALINC; AC097523.1; ARHGAP5-AS1; SNORD3C; FOXO1; KDM4A-AS1; PRSS53; ZNF438; HOMER2; APIP; OCEL1; LARGE1; KCND3; UBE3D; ETHE1; KLHL8; POLE4; SRRM3; MAPT; RPL7P23; CPQ; RNF125; CSGALNACT1; HINT2; ENOX1; GTF2IP13; PPP1R37; RIPOR3; TSC22D1; PTPRB; METTL15P1; MIR17HG; IGSF8; AC091982.3; ZFPM1; CAPZB; CA2; CSPG4P12; CUEDC1; KCNJ11; LTBP4; MAOB; PAQR3; MBD3; UQCRHL; PRSS27; CIART; MBTPS1; DGLUCY; MED21; AL391684.1; PSMC5; ARID5B; SMOC1; TTYH2; ALKBH7; MTCL1; LINC02482; CASP9; ACYP2; ENTPD8; AC018521.5; SERGEF; AC011476.3; ANGPTL4; AC018797.3; AUH; PIK3CB; DBNDD2; ABCC12; AC004943.2; NEMP2; MAFF; CTU2; GIHCG; SIX1; CALML5; SESTD1; DHRS2; EFNA5; NEK5; ANKRD18B; AC015712.2; ZC3H12A; MHENCR; SPAG6; YBX1P1; NDFIP2; MRC2; TPO; KCNG1; KIF13B; ZBED3-AS1; TOB1-AS1; EIF4A3; AC005914.1; PANK1; TTLL1; FCRLB; KIAA1671; AL731571.1; TMC5; MAML2; PYDC1; DCST1-AS1; TPRN; STX18; ZSCAN21; PSD4; GFRA1; ACTR3B; EIF2AK4; PRKAR2B; NCEH1; SH3GL2; PART1; ACSF2; ZNF19; CTAGE9; AC007114.2; CORO2A; PRPS1; B4GALT6","AQP3; UGDH; FKBP5; AC021148.3; PISD; ARHGEF26; EHF; PARM1; ARHGEF37; GLUL; WWC3; GREB1; PTPRJ; ADD3; SYBU; NEDD4L; GPRC5A; WWC1; TBC1D16; SCUBE2; OSGIN1; SLC26A2; TBC1D2; KRT15; LEF1; TMTC2; TENT5C; CD9; PDE10A; PLCL1; TRGC1; MPDZ; TMEM164; NFKBIA; SLC16A6; MCCC2; GRHL2; AC007255.1; HEY2; FAM214A; CYP2B7P; ASB13; SSTR2; MTSS1; STEAP4; CSGALNACT2; GSTM3; TPD52; TBC1D8; STARD13-AS; STARD13; MYCBP2; KLF5; FAM107B; HMGN5; LRP2BP; YPEL3; FRMD3; ZBTB7C; PAQR5; TRGC2; SNX25; ZNF703; ABCC4; IGFBP4; ARHGEF26-AS1; ZNF552; LRIG1; BCL6; FAH; HERC3; AP000331.1; GDPD3; SLC38A2; DSCAM-AS1; ADCY2; TIPARP; LYPD6; DOCK8; SAMD12-AS1; RGL1; SLC22A5; RN7SL674P; HEG1; TFPI2; SEPHS2; CASD1; GET4; MRFAP1; RFLNA; PARTICL; PRAG1; CORO1A; SYT8; VMP1; ARHGAP40; LEKR1; SGPP1; TSC22D3; SPRED2; CHPT1; ZNRF1; RAD17P1; ARRB1; KRT81; DUSP16; AZGP1P1; HGD; NOL6; SKAP1; GADD45B; PPARGC1B; KL; MFSD2A; VAMP5; SLC20A1; HAP1; ACSM1; IRS2; AC007996.1; ENKD1; LRRC26; SMIM14; HERC6; GADD45G; RMDN2; AC092171.5; HES1; EML6; PRDM4; RASD2; ATP1A1; SPOCD1; PXMP4; SLC16A4; HMGCS2; LDHD; AHR; BCAR3; MIR3936HG; AREG; SIAH2; RCBTB2; TSKU","UGDH; SEC14L2; HSD11B2; AC021148.3; ZNF652; EHF; WWC3; AL033523.1; KDM4B; SYBU; CSRNP1; SASH1; TBC1D16; PDE10A; MCCC2; AC007255.1; FAM214A; AC092053.4; GSTM3; TPD52; SGK3; RAB6C; TTC21A; TMEM17; CCDC40; VWFP1; CRAT; BCL6; ACACA; PXDC1; H4C8; MRFAP1; PRR15L; BAG3; PRKAA2; VIPR1; ZNRF1; RAD17P1; VIPR1-AS1; ZFX; COL5A2; GADD45B; SLC20A1; ACSM1; SMIM14; CEMIP2; LDHD; AHR; AREG; SIAH2; STC1; RNF215; ANKRD62P1; ATG101; SLC24A3","DOCK2"]},"columns":[{"id":"AR","name":"AR","type":"character","cell":["⇧ AR Up","⇧ AR Up","⇧ AR Up","⇩ AR Down"],"maxWidth":150,"style":[{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#0571B0"}]},{"id":"ER","name":"ER","type":"character","cell":[" ER Undetected"," ER Unchanged","⇧ ER Up"," ER Undetected"],"maxWidth":150,"style":[{"color":"#7F7F7F"},{"color":"#7F7F7F"},{"color":"#CA0020"},{"color":"#7F7F7F"}]},{"id":"Windows","name":"Windows","type":"numeric","maxWidth":80},{"id":"Direction","name":"Gene Direction","type":"character","maxWidth":120,"style":[{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#0571B0"}]},{"id":"p","name":"p","type":"numeric","cell":["3.76e-25","3.58e-11","6.33e-08","0.002"],"maxWidth":80},{"id":"padj","name":"p<sub>fdr<\/sub>","type":"numeric","cell":["3.01e-24","1.43e-10","1.69e-07","0.004"],"html":true},{"id":"Edge Size","name":"Edge Size","type":"numeric","maxWidth":80},{"id":"Leading Edge","name":"Leading Edge","type":"character","cell":[{"name":"span","attribs":{"title":"AQP3; UGDH; FKBP5; SEC14L2; CLDN8; EAF2; STEAP2; RANBP3L; HSD11B2; AC021148.3; PISD; ZNF652; ZNF689; OTULINL; ANKRD22; ARHGEF26; AKR1C3; EHF; KRT4; PARM1; ARHGEF37; SERHL2; GLUL; THBS1; NRP1; EPGN; WWC3; CROT; GREB1; AL033523.1; PALMD; TTC12; PTPRJ; PWWP3B; ADD3; SLC16A9; SYBU; MYLIP; RAB3B; ARHGEF28; NEDD4L; CSRNP1; GPCPD1; GPRC5A; WWC1; SASH1; PRR15; CNTNAP2; TBC1D16; SCUBE2; SLC26A2; FRK; DIO2; SOWAHC; SERHL; PER1; AKAP13; SLC22A23; TBC1D2; CRYBG1; ASRGL1; LEF1; TMTC2; TENT5C; CD9; PDE10A; PLCL1; SPRED1; PPP3CA; VSTM2L; HS3ST3B1; IQGAP2; STON2; JUN; PRKAB2; NADK2; PMEPA1; MPDZ; NDRG1; TMEM164; IL20RA; NFKBIA; RRP7A; SLC16A6; MCCC2; GRHL2; AC007255.1; HEY2; RRP7BP; PLEKHD1; LNX2; KLF13; RASSF6; FHDC1; PARP14; PREP; FAM214A; AC092053.4; TGFBR2; CYP2B7P; ECI2; FGFR2; AL135905.2; TMCC3; SERPINI1; LMO3; ASB13; SSTR2; PARP9; SLC25A18; MTSS1; ADH1C; AL606500.1; ATP13A4; LRIG3; SOWAHA; STEAP4; FLRT3; CDC42EP3; MKLN1-AS; KLF9; PEA15; VLDLR; NXPE3; TPD52; INSYN2A; SGK3; EMP1; TBC1D8; MKRN2OS; EPN2; BCL2L1; RAB6C; SORD; ELL2; STARD13-AS; STARD13; MYCBP2; AL162377.1; KLF5; ZNF761; AL445309.1; DIPK2A; FAM107B; GNA12; TTC21A; WWTR1; DOCK4; ZBTB7C; F2R; TMEM17; TP53TG1; CCDC40; CNNM1; ALCAM; PID1; PAQR5; LRCH1; SEMA6D; TMEM131L; MTMR9; ATP10A; GNMT; SNX25; VWFP1; NUAK1; PDE9A; CRAT; HECTD3; NAMPT; PACSIN2; AK4; SPSB1; GASK1B; CHN1; CBX4; SLC1A1; TNFAIP8; AMACR; GPR137C; ABCC4; ARHGEF26-AS1; TESK2; TOE1; LRIG1; TPM3P9; ZNF442; RASSF3; NYNRIN; CCDC136; BCL6; UAP1; AC093866.1; FAH; HERC3; KCNB1; ACACA; SLC38A2; GLDC; CRISPLD1; DSCAM-AS1; ADCY2; TIPARP; PEX10; LYPD6; PXDC1; DOCK8; SLC2A1; ACAA1; HSPA2; ADIRF-AS1; PXYLP1; TGFBR3; SRMS; LPAR3; ADAT2; CA13; SLC16A1; FZD5; AC024896.1; RGL1; NSRP1; AC105285.1; SLC22A5; RN7SL674P; SPRY4; RWDD3; HMGCS1; CRYM; DLGAP1-AS1; TFPI2; FHL2; TTC32; C20orf144; CASD1; GET4; ATP7B; MRFAP1; DDR2; RFLNA; SLC44A5; PARTICL; AL158211.5; NAGK; SORD2P; PRR15L; HNRNPA1P48; TSPAN33; PRAG1; SMKR1; KIF21A; LDHB; DBI; KIAA0825; AC022137.4; HSP90AA2P; VMP1; PDIA5; DOK7; KRT10; LEKR1; RPH3AL; LINC01881; IQCB1; TES; PRKAA2; FAM198B-AS1; EGLN3; TMEM63C; VIPR1; SPRED2; LMO4; CHPT1; ZNRF1; VWDE; DEPTOR; NRARP; CLIP4; ID3; DUSP16; ZBTB10; SLC6A17; ZNF837; KLRG2; AC138028.4; AKAP7; AZGP1P1; PSMD6-AS2; STARD5; AC113404.3; HGD; ZMYND10; NEDD4; PGAM4; ZNF365; ZFX; CSF1; LOXL3; CD163L1; CEP112; PSCA; NOL6; ID1; SKAP1; COL5A2; GADD45B; ENPP1; ZNF674-AS1; AIFM2; MNS1; PLLP; CNOT2-DT; PPARGC1B; AL135905.1; MAP1B; IFI6; MFF-DT; KL; CCDC96; VAMP5; LIFR; UNC5B; CARMIL1; GRAMD1B; SLC20A1; ACSM1; AP001816.1; ANKRD44; RBM26-AS1; LHFPL6; TRIM36; MAFB; EPB41L2; TMEM62; LRRC26; ARL15; CCDC34; SMIM14; AC115223.1; TOR1AIP2; RFESD; CCDC85A; LRRC75A; HELZ2; DDX60; IGFBP3; APBA1; SMAD9; RHOBTB1; RRS1; G3BP2; TRIM35; TXNDC16; AC112777.1; COL12A1; CYTH1; RAB11B-AS1; ZFP36L2; STAT5A; NKX2-1; HSPBAP1; SLC6A11; SERTAD1; RMDN2; AC092171.5; CFAP45; HES1; UROD; HNRNPA3P6; TSFM; TSPAN5; AC007405.4; ERVK3-1; EML6; PRDM4; RASD2; ATP1A1; PTCH1; AADAT; BPIFA4P; PXMP4; NKILA; RNF166; QDPR; SLC39A14; EFNA4; PTCHD1; SLC16A4; HMGCS2; PLEKHG4B; MOB3C; RMI1; PPIAP29; LDHD; TFCP2L1; AHR; CA8; AC022137.3; BCCIP; BCAR3; MIR3936HG; MCOLN2; LINC02615; UGT8; AREG; PHKA1; SIAH2; SALL2; AC090517.5; POLA2; CACNA2D2; ZEB1-AS1; TSKU; EFNA3; PIP4P2; AC141930.1; STAMBPL1; AC138028.2; MTHFS; ZFAND4; TUT7; RPS2P55; C9orf106; LYSMD4; AL451165.2; RNF215; C1QTNF3; MPC2; KCNK1; FZD8; SPRYD7; BRD7P2; SMCO4; WDFY3-AS2; OPLAH; RGS2; AL121906.2; SIK1B; FMO5; FZD1; UXS1; SIK1; GAR1; PRIMA1; AC104825.1; ATP6V1B2; CASC2; FAM241B; ANKRD62P1; PPM1M; AC007743.1; FAM189A2; DNMBP; AL139385.1; SLC24A3; ANP32B; CACYBPP2; SCD5; EML5; MAD2L1BP; AC084082.1; RASGEF1A; ITGAV; NCOA1; SMPDL3A; PDE3B; ZDHHC8P1; PRELID2; RFLNB; FAM53A; GGT7; GPR37; ACTN2; CTU1; FAM81A; AC104109.2; CYP4Z2P; Z99572.1; PLEKHG5; POU2F1; EEPD1; RHOB; KCNMA1; FBXL16; CRY2; ELAC1; VPS37D; FKBP9P1; DHCR24; NTRK3; CAMKMT; TRIM68; FBXO48; OLMALINC; AC097523.1; ARHGAP5-AS1; SNORD3C; FOXO1; KDM4A-AS1; PRSS53; ZNF438; HOMER2; APIP; OCEL1; LARGE1; KCND3; UBE3D; ETHE1; KLHL8; POLE4; SRRM3; MAPT; RPL7P23; CPQ; RNF125; CSGALNACT1; HINT2; ENOX1; GTF2IP13; PPP1R37; RIPOR3; TSC22D1; PTPRB; METTL15P1; MIR17HG; IGSF8; AC091982.3; ZFPM1; CAPZB; CA2; CSPG4P12; CUEDC1; KCNJ11; LTBP4; MAOB; PAQR3; MBD3; UQCRHL; PRSS27; CIART; MBTPS1; DGLUCY; MED21; AL391684.1; PSMC5; ARID5B; SMOC1; TTYH2; ALKBH7; MTCL1; LINC02482; CASP9; ACYP2; ENTPD8; AC018521.5; SERGEF; AC011476.3; ANGPTL4; AC018797.3; AUH; PIK3CB; DBNDD2; ABCC12; AC004943.2; NEMP2; MAFF; CTU2; GIHCG; SIX1; CALML5; SESTD1; DHRS2; EFNA5; NEK5; ANKRD18B; AC015712.2; ZC3H12A; MHENCR; SPAG6; YBX1P1; NDFIP2; MRC2; TPO; KCNG1; KIF13B; ZBED3-AS1; TOB1-AS1; EIF4A3; AC005914.1; PANK1; TTLL1; FCRLB; KIAA1671; AL731571.1; TMC5; MAML2; PYDC1; DCST1-AS1; TPRN; STX18; ZSCAN21; PSD4; GFRA1; ACTR3B; EIF2AK4; PRKAR2B; NCEH1; SH3GL2; PART1; ACSF2; ZNF19; CTAGE9; AC007114.2; CORO2A; PRPS1; B4GALT6"},"children":["AQP3; UGDH; FKBP5; SEC14L2; CLDN8; EAF2; STEAP2..."]},{"name":"span","attribs":{"title":"AQP3; UGDH; FKBP5; AC021148.3; PISD; ARHGEF26; EHF; PARM1; ARHGEF37; GLUL; WWC3; GREB1; PTPRJ; ADD3; SYBU; NEDD4L; GPRC5A; WWC1; TBC1D16; SCUBE2; OSGIN1; SLC26A2; TBC1D2; KRT15; LEF1; TMTC2; TENT5C; CD9; PDE10A; PLCL1; TRGC1; MPDZ; TMEM164; NFKBIA; SLC16A6; MCCC2; GRHL2; AC007255.1; HEY2; FAM214A; CYP2B7P; ASB13; SSTR2; MTSS1; STEAP4; CSGALNACT2; GSTM3; TPD52; TBC1D8; STARD13-AS; STARD13; MYCBP2; KLF5; FAM107B; HMGN5; LRP2BP; YPEL3; FRMD3; ZBTB7C; PAQR5; TRGC2; SNX25; ZNF703; ABCC4; IGFBP4; ARHGEF26-AS1; ZNF552; LRIG1; BCL6; FAH; HERC3; AP000331.1; GDPD3; SLC38A2; DSCAM-AS1; ADCY2; TIPARP; LYPD6; DOCK8; SAMD12-AS1; RGL1; SLC22A5; RN7SL674P; HEG1; TFPI2; SEPHS2; CASD1; GET4; MRFAP1; RFLNA; PARTICL; PRAG1; CORO1A; SYT8; VMP1; ARHGAP40; LEKR1; SGPP1; TSC22D3; SPRED2; CHPT1; ZNRF1; RAD17P1; ARRB1; KRT81; DUSP16; AZGP1P1; HGD; NOL6; SKAP1; GADD45B; PPARGC1B; KL; MFSD2A; VAMP5; SLC20A1; HAP1; ACSM1; IRS2; AC007996.1; ENKD1; LRRC26; SMIM14; HERC6; GADD45G; RMDN2; AC092171.5; HES1; EML6; PRDM4; RASD2; ATP1A1; SPOCD1; PXMP4; SLC16A4; HMGCS2; LDHD; AHR; BCAR3; MIR3936HG; AREG; SIAH2; RCBTB2; TSKU"},"children":["AQP3; UGDH; FKBP5; AC021148.3; PISD; ARHGEF26; ..."]},{"name":"span","attribs":{"title":"UGDH; SEC14L2; HSD11B2; AC021148.3; ZNF652; EHF; WWC3; AL033523.1; KDM4B; SYBU; CSRNP1; SASH1; TBC1D16; PDE10A; MCCC2; AC007255.1; FAM214A; AC092053.4; GSTM3; TPD52; SGK3; RAB6C; TTC21A; TMEM17; CCDC40; VWFP1; CRAT; BCL6; ACACA; PXDC1; H4C8; MRFAP1; PRR15L; BAG3; PRKAA2; VIPR1; ZNRF1; RAD17P1; VIPR1-AS1; ZFX; COL5A2; GADD45B; SLC20A1; ACSM1; SMIM14; CEMIP2; LDHD; AHR; AREG; SIAH2; STC1; RNF215; ANKRD62P1; ATG101; SLC24A3"},"children":["UGDH; SEC14L2; HSD11B2; AC021148.3; ZNF652; EHF..."]},{"name":"span","attribs":{"title":"DOCK2"},"children":["DOCK2"]}],"minWidth":150}],"filterable":true,"dataKey":"15c4868301bd9c312b8662a9eca57e3c"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<pre class="r"><code>plot_grid(plotlist = p, ncol = min(3, length(p)))</code></pre>
<div class="figure" style="text-align: center">
<img src="AR_E2_E2DHT_ER_E2_E2DHT_pairwise_comparison_files/figure-html/barcode-gsea-dir-sig-1.png" alt="*Barcode plots for the top 4 sets of windows associated with __directional__ changes in gene expression.*" width="960" />
<p class="caption">
<em>Barcode plots for the top 4 sets of windows associated with
<strong>directional</strong> changes in gene expression.</em>
</p>
</div>
</div>
<div id="non-directional-gsea" class="section level3">
<h3>Non-Directional GSEA</h3>
<pre class="r"><code>gsea_nondir &lt;- suppressWarnings(
  fgsea(status_gs, rna_sig_ranks, scoreType = &quot;pos&quot;)
)
gsea_nondir_sig &lt;- gsea_nondir %&gt;% 
  arrange(pval) %&gt;% 
  mutate(padj = p.adjust(pval, adj_method)) %&gt;% 
  dplyr::filter(padj &lt; enrich_alpha, size &gt;= min_sig) %&gt;% 
  as_tibble() 
p &lt;- gsea_nondir_sig %&gt;% 
  dplyr::slice(1:9) %&gt;% 
  pull(&quot;pathway&quot;) %&gt;% 
  lapply(
    function(x) {
      plotEnrichment(status_gs[[x]], rna_sig_ranks) +
        ggtitle(x) +
        theme(plot.title = element_text(hjust = 0.5, size = 10))
    }
  )
txt &lt;- ifelse(
  length(p) &gt; 0,
  glue(&quot;&quot;),
  glue(&quot;No association was found between changed binding across {comps[[1]]} and {comps[[2]]} when combined with *overall* significance in differentially expressed genes.&quot;)
)</code></pre>
<pre class="r"><code>cp &lt;-  htmltools::em(
  glue(
    &quot;Combined windows were mapped to genes, and their position amongst the &quot;,
    &quot;RNA-Seq results was assessed. {nrow(gsea_nondir_sig)} sets of windows were &quot;,
    &quot;associated with changes in gene expression, using only the p-value to &quot;, 
    &quot;rank the {comma(nrow(rnaseq), 1)} genes considered as detected.&quot;
  )
)
tbl &lt;- gsea_nondir_sig %&gt;%
  mutate(
    `Edge Size` = vapply(leadingEdge, length, integer(1)),
    leadingEdge = lapply(leadingEdge, function(x) id2gene[x]) %&gt;% 
      vapply(paste, character(1), collapse = &quot;; &quot;)
  ) %&gt;% 
  separate(pathway, comps, sep = &quot; - &quot;) %&gt;% 
  dplyr::select(
    all_of(comps), Windows = size, 
    p = pval, padj, `Edge Size`, `Leading Edge` = leadingEdge
  ) %&gt;% 
  reactable(
    filterable = TRUE,
    columns = list2(
      &quot;{comps[[1]]}&quot; := colDef(
        maxWidth = 150,
        cell = function(value) {
          html_symbol &lt;- &quot;&quot;
          if (str_detect(value, &quot;Up&quot;)) html_symbol &lt;- &quot;\u21E7&quot;
          if (str_detect(value, &quot;Down&quot;)) html_symbol &lt;- &quot;\u21E9&quot;
          paste(html_symbol, value)
        },
        style = function(value) {
          colour &lt;- case_when(
            str_detect(value, &quot;Up&quot;) ~ colours$direction[[&quot;up&quot;]],
            str_detect(value, &quot;Down&quot;) ~ colours$direction[[&quot;down&quot;]],
            TRUE ~ colours$direction[[&quot;unchanged&quot;]]
          )
          list(color = colour)
        }
      ),
      &quot;{comps[[2]]}&quot; := colDef(
        maxWidth = 150,
        cell = function(value) {
          html_symbol &lt;- &quot;&quot;
          if (str_detect(value, &quot;Up&quot;)) html_symbol &lt;- &quot;\u21E7&quot;
          if (str_detect(value, &quot;Down&quot;)) html_symbol &lt;- &quot;\u21E9&quot;
          paste(html_symbol, value)
        },
        style = function(value) {
          colour &lt;- case_when(
            str_detect(value, &quot;Up&quot;) ~ colours$direction[[&quot;up&quot;]],
            str_detect(value, &quot;Down&quot;) ~ colours$direction[[&quot;down&quot;]],
            TRUE ~ colours$direction[[&quot;unchanged&quot;]]
          )
          list(color = colour)
        }
      ),
      Windows = colDef(maxWidth = 80),
      p = colDef(
        cell = function(value) ifelse(
          value &lt; 0.001,
          sprintf(&quot;%.2e&quot;, value),
          sprintf(&quot;%.3f&quot;, value)
        ), 
        maxWidth = 80
      ),
      padj = colDef(
        name = glue(&quot;p&lt;sub&gt;{adj_method}&lt;/sub&gt;&quot;), html = TRUE,
        cell = function(value) {
          ifelse(
            value &lt; 0.001, sprintf(&quot;%.2e&quot;, value), sprintf(&quot;%.3f&quot;, value)
          )
        }
      ),
      &quot;Edge Size&quot; = colDef(maxWidth = 80),
     &quot;Leading Edge&quot; = colDef(
       minWidth = 150,
       cell = function(value) with_tooltip(value, width = 50)
     ) 
    )
  )
div(class = &quot;table&quot;,
  div(class = &quot;table-header&quot;,
      div(class = &quot;caption&quot;, cp),
      tbl
  )
)</code></pre>
<div class="table">
<div class="table-header">
<div class="caption">
<em>Combined windows were mapped to genes, and their position amongst the RNA-Seq results was assessed. 6 sets of windows were associated with changes in gene expression, using only the p-value to rank the 13,043 genes considered as detected.</em>
</div>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-b386634747a62f8e31ec" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-b386634747a62f8e31ec">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"AR":["AR Up","AR Up","AR Unchanged","AR Up","AR Unchanged","AR Down"],"ER":["ER Undetected","ER Unchanged","ER Undetected","ER Up","ER Unchanged","ER Undetected"],"Windows":[4621,1237,2467,260,899,6],"p":[2.62680890485866e-34,5.11384704919106e-13,7.81262986013565e-08,9.59535333451162e-08,8.29128147995031e-05,0.0106862623010595],"padj":[2.10144712388693e-33,2.04553881967642e-12,1.91907066690232e-07,1.91907066690232e-07,0.000132660503679205,0.014248349734746],"Edge Size":[913,231,545,57,205,2],"Leading Edge":["AQP3; UGDH; FKBP5; SEC14L2; CLDN8; EAF2; STEAP2; RANBP3L; HSD11B2; AC021148.3; PISD; ZNF652; ZNF689; OTULINL; ANKRD22; ARHGEF26; AKR1C3; EHF; KRT4; AMIGO2; PARM1; ARHGEF37; SERHL2; GLUL; THBS1; NRP1; EPGN; WWC3; CROT; GREB1; AL033523.1; PALMD; TTC12; PTPRJ; PWWP3B; ADD3; SLC16A9; SYBU; AR; MYLIP; RAB3B; ARHGEF28; NEDD4L; CSRNP1; GPCPD1; GPRC5A; WWC1; SASH1; PRR15; CNTNAP2; TBC1D16; SCUBE2; SLC26A2; AL035661.1; FRK; DIO2; SOWAHC; SERHL; PER1; DOCK2; AKAP13; SLC22A23; TBC1D2; CRYBG1; ASRGL1; LEF1; GRIN2B; TMTC2; TENT5C; CD9; PDE10A; PLCL1; SPRED1; PPP3CA; MCC; VSTM2L; HS3ST3B1; IQGAP2; SOX9; FOXI1; STON2; JUN; SOSTDC1; PRKAB2; NADK2; PMEPA1; MPDZ; NDRG1; TMEM164; IL20RA; NFKBIA; RRP7A; PPFIBP2; SLC16A6; MCCC2; GRHL2; AC007255.1; HEY2; RRP7BP; PLEKHD1; LNX2; KLF13; RASSF6; TRIM2; FHDC1; BLNK; PARP14; PREP; FAM214A; AC092053.4; TGFBR2; PMP22; CYP2B7P; FAM167A; ECI2; FGFR2; AL135905.2; ESR1; TMCC3; DKK1; ACKR3; MIR99AHG; SERPINI1; LMO3; ASB13; SSTR2; PARP9; SLC25A18; ADGRV1; MTSS1; ADH1C; PREX1; AL606500.1; ATP13A4; LRIG3; GCNT2; SOWAHA; STEAP4; FLRT3; CDC42EP3; CXCR4; MKLN1-AS; ENC1; KLF9; PGR; PEA15; GRM4; PCED1B; VLDLR; RNF152; NXPE3; TPD52; INSYN2A; SGK3; EMP1; TBC1D8; MKRN2OS; EPN2; BCL2L1; COLCA1; RAB6C; AC005920.1; SORD; ELL2; STARD13-AS; STARD13; MYCBP2; AL162377.1; KLF5; ZNF761; OXGR1; AL445309.1; DIPK2A; FAM107B; GNA12; LFNG; GUCY1A1; TTC21A; WWTR1; CACNB4; DOCK4; ZBTB7C; ANK3; F2R; MUC19; AL157895.2; RARRES1; TMEM17; TP53TG1; CCDC40; CNNM1; ALCAM; PID1; SERPINB2; PAQR5; LRCH1; SEMA6D; TMEM131L; GUCY1A2; MTMR9; ATP10A; GNMT; MYRFL; SNX25; VWFP1; PRKCE; NUAK1; PDE9A; CRAT; HECTD3; NAMPT; NR3C2; SOX5; PACSIN2; AK4; SPSB1; GASK1B; LINC02303; CHN1; GCOM1; CBX4; SLC1A1; AC108488.3; FBXO32; TNFAIP8; DENND2B; GATA3; AMACR; GPR137C; IKZF2; ABCC4; ARHGEF26-AS1; TESK2; MIR181A1HG; Z83843.1; TOE1; ITGA5; GCNT1; RNF144B; LRIG1; TPM3P9; ZNF442; RASSF3; NYNRIN; IGSF9; C15orf61; CCDC136; DNM3; BCL6; UAP1; AC093866.1; FAH; HERC3; KCNB1; SCIN; ACACA; SLC38A2; GLDC; CRISPLD1; DSCAM-AS1; ADCY2; RAB11FIP1P1; TIPARP; PEX10; LYPD6; LIPH; AC022400.8; PXDC1; STX18-AS1; CMAHP; DOCK8; CHRNB2; DLG2; SLC2A1; ACAA1; HSPA2; ADIRF-AS1; PDE4B; PXYLP1; TGFBR3; SRMS; LPAR3; ADAT2; CA13; SLC16A1; FZD5; PIK3C2G; AC024896.1; RGL1; NSRP1; AC105285.1; SLC22A5; RN7SL674P; SPRY4; RWDD3; VAV3; HMGCS1; CRYM; DLGAP1-AS1; TFPI2; FHL2; TTC32; C20orf144; ARL10; CASD1; GET4; ATP7B; TMEM154; PLEKHH1; CACNA1D; AL157392.3; MRFAP1; DHRS3; DDR2; RFLNA; SLC44A5; PARTICL; AL158211.5; NAGK; QSOX2; CDK18; SORD2P; PRR15L; HNRNPA1P48; TSPAN33; PRAG1; SMKR1; KIF21A; HECW2; LDHB; KRT23; NIFK-AS1; DBI; KLF3-AS1; LRRC46; RAD51B; KIAA0825; AC022137.4; HSP90AA2P; VMP1; PDIA5; ACACB; DOK7; KRT10; LEKR1; MAGI2; RPH3AL; SRCIN1; SHANK2; ZMIZ1; LINC01881; IQCB1; OSR2; TES; PRKAA2; SMIM5; FAM198B-AS1; AL137782.1; EGLN3; CMTM8; TMEM63C; VIPR1; RAB30; SNORA37; SPRED2; HSPG2; FOXO6; LMO4; ITGAM; MEGF11; SLC27A2; PDE11A; FAM110B; CHPT1; AC118344.4; ZNRF1; AC083843.3; VWDE; DEPTOR; NRARP; IQCH-AS1; CLIP4; ID3; PHLDB1; BDNF-AS; SUSD2; DUSP16; ZBTB10; RBM20; LYST; TCF7L2; SLC6A17; ZNF837; AC026979.4; KLRG2; ST6GAL1; AC138028.4; AL133230.2; AKAP7; AZGP1P1; PSMD6-AS2; STARD5; AC113404.3; HGD; ZMYND10; NEDD4; C7orf31; COLCA2; PGAM4; ZNF365; ZFX; CSF1; LOXL3; BDKRB2; CD163L1; RNF43; CEP112; PSCA; NOL6; AC007285.1; REEP1; ID1; SKAP1; COL5A2; INPP5J; AC116533.1; GADD45B; ENPP1; ZNF674-AS1; AIFM2; PIFO; MNS1; PLLP; CNOT2-DT; PPARGC1B; SPRR3; AL135905.1; MYRIP; MAP1B; FGD6; IFI6; POTEKP; MFF-DT; CTSO; KL; CCDC96; ANKRD61; AP000873.2; AP005131.6; VAMP5; LIFR; PLCH1; SLC16A2; UNC5B; PDPR; CARMIL1; SEMA3C; SILC1; WNT5A; GRAMD1B; DRAIC; SLC20A1; ACSM1; NFATC2; MLLT3; DGAT2; AP001816.1; ANKRD44; AL391001.1; RBM26-AS1; LHFPL6; TRIM36; PRUNE2; MAFB; EPB41L2; NTN1; TMEM62; LRRC26; ARL15; CCDC34; TDO2; SMIM14; AC115223.1; TOR1AIP2; RFESD; FAM222B; CCDC85A; LRRC75A; HELZ2; DDX60; AC007637.1; IGFBP3; ZNF519; APBA1; SMAD9; MYB; RHOBTB1; GNB1L; RRS1; G3BP2; CXCR6; LINC00511; TRIM35; ADAMTSL4; TXNDC16; AC112777.1; ABCB6; RPL23AP2; COL12A1; CYTH1; FABP5; ANKRD10-IT1; ARHGEF3; ALPK1; RAB11B-AS1; ZFP36L2; STAT5A; LINC00265; NFKBIZ; NKX2-1; HSPBAP1; SLC6A11; SERTAD1; RMDN2; AC092171.5; CFAP45; PBX3; CATSPERB; BBOX1; MEIS1; HCAR1; OPN3; HES1; UROD; HNRNPA3P6; TSFM; TSPAN12; TSPAN5; AC007405.4; ERVK3-1; EDN2; EML6; PRDM4; RASD2; BX470102.2; ATP1A1; IGSF11; ENTPD3-AS1; PTCH1; AADAT; KCNE4; BPIFA4P; HOXC6; PXMP4; NKILA; FRMD4A; RAPGEF4; RNF166; AC124944.2; QDPR; SLC39A14; AC027097.2; EFNA4; PTCHD1; SLC16A4; HMGCS2; PLEKHG4B; MOB3C; RMI1; SLC16A10; PPIAP29; LDHD; FAM221A; TFCP2L1; AHR; CA8; SPIN3; AC022137.3; EVL; BCCIP; GPRIN3; BCAR3; RNASEH1-AS1; BCL2L11; MIR3936HG; MCOLN2; LINC02615; EPN3; UGT8; RPSAP47; LAMTOR5-AS1; TRIM45; AREG; PHKA1; SIAH2; BMF; SIDT1; RPL29P11; SALL2; BAHCC1; AC013394.1; AC090517.5; AC093642.2; AP4S1; POLA2; CACNA2D2; AC009318.2; AC020978.9; ZEB1-AS1; AC007114.1; AP000766.1; TSKU; EFNA3; PIP4P2; AL451123.1; AC141930.1; GRB7; CLIP2; STAMBPL1; KALRN; AC138028.2; MTHFS; ZFAND4; AL359715.1; ETV1; TUT7; CSTA; RPS2P55; C9orf106; LYSMD4; AL451165.2; ZSCAN12; SHROOM2; RNF215; C1QTNF3; MPC2; KCNK1; BHLHE41; FKRP; STXBP6; FZD8; SPRYD7; BRD7P2; CCDC7; SMCO4; WDFY3-AS2; ZBED3; OPLAH; RGS2; AL121906.2; RAP1GAP; SIK1B; FOXN3; MAP2K6; FMO5; TBCE; SRRM2; OAS1; GPR157; AL158163.1; NCOR2; PIK3IP1; FZD1; SELENON; UXS1; SIK1; FAM151B; GAR1; PRIMA1; AC104825.1; DICER1-AS1; ATP6V1B2; CASC2; C15orf62; EEFSEC; HLF; LINC01239; MAN1C1; FAM241B; TNFRSF11A; ANKRD62P1; PPM1M; RAB6B; AC007743.1; FAM189A2; DNMBP; ENSAP2; AC008895.1; AL157838.1; AL139385.1; ANG; YPEL1; HTR7P1; AC135050.6; SUFU; SLC24A3; SH3RF2; ANP32B; CACYBPP2; COL27A1; SCD5; EML5; MAD2L1BP; AC084082.1; C15orf41; RASGEF1A; ITGAV; PALM2AKAP2; NCOA1; SMPDL3A; PDE3B; ZDHHC8P1; AC097376.3; PRELID2; AL009174.1; SLC35C1; RFLNB; ATXN1; FAM53A; GGT7; LINC00624; GPR37; ACTN2; FTX; CTU1; FLNB-AS1; FAM81A; AC104109.2; CYP4Z2P; Z99572.1; NFIX; TNS1; AL049834.1; PLEKHG5; C3orf62; POU2F1; AC084824.5; CHST15; EEPD1; CLSTN2; RHOB; KCNMA1; FBXL16; CRY2; MATN1-AS1; BAALC-AS1; ELAC1; VPS37D; FKBP9P1; ZNF594; TMC8; DHCR24; NTRK3; CAMKMT; HDAC9; AL158212.3; AC009133.1; ALPK3; TRIM68; FBXO48; CLN3; OLMALINC; AC097523.1; ARHGAP5-AS1; RABEP1; SNORD3C; FOXO1; KDM4A-AS1; ZSWIM3; PRSS53; ZNF438; TTC23; LRRC49; NHEJ1; HOMER2; APIP; KIAA1211L; OCEL1; LARGE1; KCND3; UBE3D; SRCAP; C5orf63; ETHE1; KLHL8; POLE4; UBALD2; SRRM3; PLA2R1; MAPT; RPL7P23; HACD1; CPQ; ASAP1; RNF125; METAP1D; RAD51-AS1; CSGALNACT1; ELK3; TMEM8B; NOL4L; ZSCAN5A; KCNS3; HINT2; KLF3; CYSRT1; ACSS1; ENOX1; GTF2IP13; PPP1R37; RIPOR3; LINC00888; TSC22D1; LINC01504; PTPRB; LINC01521; METTL15P1; RFX1; TIMP2; MIR17HG; LCA5; IGSF8; AC091982.3; CPAMD8; ZFPM1; DLEU1; CAPZB; CA2; CSPG4P12; ROBO1; LRRC27; TTC13; CUEDC1; OTUD7A; KCNJ11; LTBP4; FOXD1; MAOB; PAQR3; MBD3; UQCRHL; PRSS27; USP3-AS1; CIART; MBTPS1; TUBD1; DGLUCY; MED21; AL391684.1; UNC5C; PSMC5; ARID5B; PARP8; BCAS3; SMOC1; CEP126; TTYH2; AL355488.1; SCUBE3; ALKBH7; MGAM; MTCL1; LINC02482; RPS26P6; MAP3K5; CASP9; ACYP2; ENTPD8; AC018521.5; PROM1; SERGEF; AC011476.3; ANGPTL4; PPARG; AC018797.3; CRISPLD2; AUH; IGF2BP2; PIK3CB; DNAH11; RTN4IP1; DBNDD2; ABCC12; AC004943.2; NOSTRIN; NEMP2; MAFF; EPS8; CTU2; GIHCG; PPP3CB-AS1; SIX1; CALML5; SESTD1; DHRS2; EFNA5; NEK5; PAX9; ANKRD18B; AC015712.2; ASH1L-AS1; MYC; ZC3H12A; MHENCR; SPAG6; TSEN2; YBX1P1; RERG; JMJD7-PLA2G4B; NDFIP2; MRC2; NHSL1; TPO","AQP3; UGDH; FKBP5; AC021148.3; PISD; ARHGEF26; EHF; PARM1; ARHGEF37; GLUL; WWC3; GREB1; PTPRJ; ADD3; SYBU; NEDD4L; GPRC5A; WWC1; IL1R1; TBC1D16; SCUBE2; OSGIN1; SLC26A2; TBC1D2; KRT15; LEF1; TMTC2; TENT5C; CD9; PDE10A; PLCL1; TRGC1; CABLES1; MPDZ; TMEM164; BCL2; NFKBIA; PPFIBP2; SLC16A6; MCCC2; GRHL2; AC007255.1; HEY2; FAM214A; CYP2B7P; ESR1; ASB13; SSTR2; MTSS1; STEAP4; CSGALNACT2; GSTM3; TPD52; TBC1D8; STARD13-AS; STARD13; MYCBP2; KLF5; FAM107B; HMGN5; LRP2BP; YPEL3; FRMD3; ZBTB7C; SOX13; PAQR5; TRGC2; GUCY1A2; SNX25; PRKCE; SLC16A14; ZNF703; GATA3; ABCC4; IGFBP4; ARHGEF26-AS1; ZNF552; LINC01341; GCNT1; LRIG1; BCL6; FAH; HERC3; AP000331.1; GDPD3; AC008268.1; SLC38A2; GIPR; DSCAM-AS1; ADCY2; TIPARP; LYPD6; DOCK8; DLG2; TAPT1; SAMD12-AS1; RGL1; SLC22A5; RN7SL674P; HEG1; TFPI2; SEPHS2; CASD1; GET4; CACNA1D; MRFAP1; DHRS3; RFLNA; PARTICL; PRAG1; CORO1A; SYT8; RAD51B; VMP1; ARHGAP40; LEKR1; SGPP1; PPP1R3D; TSC22D3; S100A8; SMIM5; RAB30; SPRED2; C14orf132; ITGAM; CHPT1; ZNRF1; RAD17P1; ARRB1; KRT81; SUSD2; DUSP16; RBM20; AZGP1P1; HGD; FAM117A; LRRC37A3; NOL6; SKAP1; GADD45B; PPARGC1B; MYRIP; KL; AP005131.6; MFSD2A; VAMP5; DRAIC; SLC20A1; HAP1; AC007191.1; ACSM1; IRS2; AL391001.1; AC007996.1; ENKD1; PRUNE2; LRRC26; SMIM14; GAB2; HERC6; GRIK3; MYB; LINC00511; RPL23AP2; ALPK1; CBR3-AS1; GADD45G; RMDN2; AC092171.5; HES1; EML6; PRDM4; RASD2; ATP1A1; SPOCD1; PXMP4; AC027097.2; SLC16A4; HMGCS2; LDHD; AHR; BCAR3; RNASEH1-AS1; MIR3936HG; GATA3-AS1; LAMTOR5-AS1; AC106782.1; AREG; SIAH2; BMF; TAPT1-AS1; RCBTB2; TSKU; GRB7; LCMT1-AS2; FKRP; CCDC7; AL158163.1; NCOR2; AC104667.2; FAM151B; USP35; DICER1-AS1; AL591848.4; CASC2; AC008895.1; AL157838.1; ANKRD34A; ZNF814; ZCCHC17; FEM1A; SLC24A3; AL157935.2; SEPTIN1; CLIC6; EML5; AC084082.1; PALM2AKAP2; PRELID2; ZNF688; CTU1; FLNB-AS1; PLAAT4; RTN4RL1; AL049834.1; DAPK2; CHST15; CLSTN2; RHOB; BICRA; KCNMA1","UGDH; RANBP3L; AC021148.3; OTULINL; ARHGEF26; EHF; KRT4; AMIGO2; GLUL; THBS1; NRP1; WWC3; GREB1; AL033523.1; PTPRJ; ADD3; SYBU; NEDD4L; SASH1; PRR15; CNTNAP2; IL1R1; TBC1D16; HPGD; SCUBE2; SLC26A2; DIO2; SLC22A23; GRIN2B; TMTC2; CD9; PPP3CA; VSTM2L; IQGAP2; SOX9; STON2; CABLES1; JUN; SOSTDC1; PRKAB2; MPDZ; SRGAP3; PPFIBP2; MCCC2; GRHL2; AC007255.1; HEY2; TRIM9; ECI2; AL135905.2; TMCC3; DKK1; SERPINI1; ASB13; PARP9; SLC25A18; ADGRV1; PREX1; PLEKHH2; POU2F3; LRIG3; GCNT2; CDC42EP3; ENC1; PCED1B; RNF152; NXPE3; TPD52; INSYN2A; TBC1D8; COLCA1; AC005920.1; ELL2; SNORD94; STARD13-AS; STARD13; MYCBP2; OXGR1; AL445309.1; CRYBG2; LFNG; AC005912.1; ZBTB7C; ANK3; MUC19; RARRES1; CNNM1; ALCAM; LRATD1; LRCH1; SEMA6D; TMEM131L; GUCY1A2; PRKCE; SLC16A14; SOX5; PACSIN2; LINC02303; DENND2B; SLC9A1; IKZF2; ABCC4; ARHGEF26-AS1; TESK2; SYT16; TOE1; LINC01341; GCNT1; SGSM1; RNF144B; RASSF3; RBBP8NL; IGSF9; DNM3; BCL6; AP000331.1; AC022007.1; KCNB1; SCIN; ANKH; AC008268.1; SLC38A2; GLDC; GIPR; CRISPLD1; ADCY2; TIPARP; LYPD6; PXDC1; ZDHHC11; AL603750.1; CMAHP; DLG2; PDE4B; LRATD2; CA13; FZD5; PIK3C2G; SAMD12-AS1; RGL1; AC105285.1; RN7SL674P; AC005083.1; VAV3; LINC01814; SEPHS2; C20orf144; GET4; MAP2; NRCAM; AL157392.3; DHRS3; SYDE1; RFLNA; KCNK5; AL158211.5; CDK18; PRR15L; GLCCI1; SMKR1; KIF21A; TRIM6; LDHB; TRERF1; KLF3-AS1; RAD51B; KIAA0825; VMP1; ARHGAP40; PCDH10; ACACB; LEKR1; MAGI2; SGPP1; MOK; SHANK2; ZMIZ1; PPP1R3D; ARID5A; IVL; PRKAA2; EGLN3; CMTM8; L3MBTL3; SPRED2; SPCS2P4; C14orf132; ADAM19; CXXC4; RNF150; SLC27A2; PDE11A; LYRM9; MUC16; CHPT1; SCYL3; RAD17P1; VWDE; SLC4A8; Y_RNA; AL357568.1; SOX11; LINC01176; HGD; C7orf31; COLCA2; SYT15; BDKRB2; CCDC82; CD163L1; RNF43; CEP112; PSCA; REEP1; ID1; COL5A2; INPP5J; AC116533.1; ENPP1; BCYRN1; PTAFR; IFNLR1; PPARGC1B; AL135905.1; MYRIP; MFSD2A; NFASC; PLCH1; PDPR; CARMIL1; SEMA3C; WNT5A; DRAIC; ACSM1; NFATC2; MLLT3; AP001816.1; ANKRD44; AL391001.1; RBM26-AS1; PRUNE2; TLK2P1; SMIM14; AC115223.1; ABCC3; METTL7B; NOVA1; AC110285.2; ETV4; CCDC85A; NSMCE3; RHOBTB1; GNB1L; G3BP2; LINC00511; CAMK4; DBP; IL17RE; ABCB6; ANKRD10-IT1; ARHGEF3; ALPK1; MAGI3; NAALAD2; LINC00265; NFKBIZ; AC004982.1; CFAP45; PBX3; CATSPERB; BBOX1; OPN3; HUS1; TSFM; LURAP1L; TSPAN5; AC007405.4; KIFC3; RASD2; BX470102.2; ENTPD3-AS1; FANCE; PXMP4; NKILA; FRMD4A; QDPR; NBL1; AC027097.2; EFNA4; AP002490.1; HMGCS2; HEBP1; SP140L; SNHG22; RNY3P1; FAM221A; RAPGEF3; AHR; CA8; CCDC167; EVL; BCCIP; BCAR3; RNASEH1-AS1; IL17RD; AC090772.5; LINC02615; DAPK1; NAV1; AL590714.1; EPN3; RNF128; FAM89A; NABP1; GPR162; CD302; BAHCC1; AC013394.1; AP4S1; AC018809.2; AC007114.1; MTUS2; EFNA3; AXIN2; GRB7; AC092910.3; KALRN; PPARD; FAM66C; ETV1; FO681492.1; SIRPA; LYSMD4; LCMT1-AS2; MS4A7; KCNK1; KLHL2; BHLHE41; FKRP; STXBP6; ZBED3; LINC-PINT; CMPK2; AL121906.2; JAG1; LIME1; SIK1B; FOXN3; POLR2J4; FMO5; OAS1; CRELD1; WHAMMP2; SIK1; AL591848.4; C15orf62; EEFSEC; PARD3B; RAB6B; AC007743.1; THAP7-AS1; HTR7P1; AC135050.6; SLC24A3; SPC25; AL157935.2; AC080013.1; MPP7; SCD5; EML5; RASGEF1A; ZNF784; SLC35C1; ATXN1; LINC00624; ACTN2; FAM81A; B3GNTL1; PITX1; AL049834.1; DAPK2; NSMCE1-DT; CHST15; CLSTN2; BICRA; KCNMA1; BAALC-AS1; ELAC1; AP1AR; KIF12; ZNF594; DHCR24; CAMKMT; HDAC9; AL158212.3; AC009133.1; AC097523.1; SNAPC1; RABEP1; ZSWIM3; PRSS53; GLCE; NHEJ1; APIP; KIAA1211L; LARGE1; KCND3; SRCAP; ABLIM3; ETHE1; UBALD2; PLA2R1; AC093627.4; CPQ; CENPQ; METAP1D; CSGALNACT1; ZDHHC18; KCNS3; KLF3; CCDC18-AS1; PTPRB; METTL15P1; MR1; ELAVL2; DLEU1; CAPZB; ROBO1; LRRC27; CUEDC1; CHST3; PAQR3; OSBPL6; DOCK3; AC005070.3; APBA2; TUBD1; DGLUCY; AL391684.1; RPL13P5; SERPINB5; UNC5C; ARID5B; PARP8; BCAS3; SMOC1; CEP126; RTEL1-TNFRSF6B; LINC00910; AC018521.5; SERGEF; DNAH11; DBNDD2; NOSTRIN; EPS8; GIHCG; CALML5; EFNA5; PAX9; ZNF428; ZC3H12A; MHENCR; AC010168.2; NDFIP2; NHSL1; ANAPC11; CTNND2; CRTC3; KIF13B; MED4; FBXO4; ZBED3-AS1; TOB1-AS1; ATF7IP; KIAA1522; TLE1; AC005914.1; NCALD; PANK1; TTLL1; SH3TC1; AC139149.1; KIAA1671; AL121839.2; TMC5; MAML2; PYDC1; DACH1; ARSJ; DCST1-AS1; RFX3; NCEH1; CTAGE9; AC007114.2; FAM241A; SLC9A3; ZNF580; HMOX1; ITPRIPL2; IQCK; RN7SKP255; ABTB2; Z93930.2; RTN1; PDCD6IPP2; CMYA5; SLC39A1; DDX12P; CCDC137; AC004982.2; PRDM11; C17orf67; COMTD1; INSIG2; RTEL1; KANSL1L; KHDRBS3; S100A1; B3GALT2; CCDC138; TM4SF1; NR4A2; RASGEF1B; AVIL; PCLO; GDPGP1; ARHGAP30; NHSL2; LETM2; FMNL2; SHLD2P1; ADNP-AS1; C9orf72; MBNL2; GHR; GPATCH3; AC093001.1; TLL1; MACROD2; SNHG9; ETFB; PAG1; CCDC148; HDAC5; TTC28","UGDH; SEC14L2; HSD11B2; AC021148.3; ZNF652; EHF; WWC3; AL033523.1; KDM4B; SYBU; CSRNP1; SASH1; TBC1D16; PDE10A; MCCC2; AC007255.1; FAM214A; AC092053.4; GSTM3; TPD52; SGK3; RAB6C; AC005920.1; TTC21A; TMEM17; CCDC40; VWFP1; CRAT; FBXO32; BCL6; ACACA; PXDC1; H4C8; MRFAP1; PRR15L; HECW2; LRRC46; RAD51B; BAG3; PRKAA2; VIPR1; ZNRF1; RAD17P1; VIPR1-AS1; RBM20; ZFX; COL5A2; GADD45B; SILC1; SLC20A1; ACSM1; AL391001.1; SMIM14; ABCC3; CEMIP2; MYB; AC009061.2","EHF; KRT4; ADD3; AR; GPRC5A; FAM49A; IL1R1; SOWAHC; DOCK2; KRT15; TENT5C; PLCL1; SOX9; FOXI1; CABLES1; PMEPA1; NDRG1; SRGAP3; BCL2; GRHL2; CYP2B7P; ESR1; SSTR2; ADGRV1; MTSS1; PREX1; TACR3; GRM4; EMP1; BCL2L1; COLCA1; AC005920.1; STARD13; LFNG; FSBP; ZBTB7C; CLEC3A; VWFP1; NR3C2; PXK; GCOM1; SLC1A1; AC004918.5; CEACAM6; SAP30L-AS1; LINC01341; SGSM1; RASSF3; HERC3; SCIN; ANKH; PGM5; SH3PXD2A; DLG2; RORC; SEMA3F-AS1; SLC22A5; GPR161; SPRY4; FHL2; TMEM154; PLEKHH1; AL157392.3; DHRS3; RFLNA; AC119673.3; PRAG1; GLCCI1; KCTD6; VMP1; ACACB; DOK7; LEKR1; CGNL1; SRCIN1; TSC22D3; TES; LIMD2; C14orf132; PDE11A; LYRM9; ZNRF1; HNRNPCP2; DEPTOR; KRT81; ZNF385A; DUSP16; ZBTB10; RBM20; AC026979.4; AL133230.2; LINC01176; COLCA2; BDKRB2; CEP112; ID1; COL5A2; FGD6; MFSD2A; SEMA3C; DRAIC; CRNDE; AL391001.1; AC007996.1; LRRC26; AC110285.2; HERC6; GRIK3; AC007637.1; CEMIP2; MYB; RHOBTB1; AC093904.4; LINC00511; TXNDC16; RAB11B-AS1; LINC00265; CBR3-AS1; SPRY1; HES1; LURAP1L; TSPAN5; RASD2; KCNE4; SPOCD1; FRMD4A; AC027097.2; PTCHD1; HMGCS2; SP140L; LINC01522; LDHD; AHR; EVL; GALNT10; MIR3936HG; PCDHA12; AC090772.5; RPSAP47; BAHCC1; AP000766.1; TSKU; AXIN2; KALRN; TUT7; TBC1D30; MB21D2; MS4A7; FKRP; CCDC7; ZBED3; PCDHAC1; OAS1; AL158163.1; AL591848.4; PARD3B; ANKRD62P1; SLC24A3; ANP32B; RASGEF1A; ITGAV; PALM2AKAP2; SMPDL3A; GSEC; LINC00624; MIR4453HG; CLSTN2; KCNMA1; ACOX2; HDAC9; TNS2; ZSWIM3; LARGE1; UBE3D; CPQ; ASAP1; TMEM8B; AC010536.1; ACSS1; RIPOR3; MR1; IGSF8; AC091982.3; HS6ST1; DLEU1; CUEDC1; TUBD1; DGLUCY; MED21; PSMC5; ARID5B; BCAS3; RBL2; ACYP2; PROM1; AC108134.2; RGS9BP; ABCC12; CALML5; TMEM186; JMJD7-PLA2G4B; RAB19; CTNND2; ZBED3-AS1; TOB1-AS1","DOCK2; RNF145"]},"columns":[{"id":"AR","name":"AR","type":"character","cell":["⇧ AR Up","⇧ AR Up"," AR Unchanged","⇧ AR Up"," AR Unchanged","⇩ AR Down"],"maxWidth":150,"style":[{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#7F7F7F"},{"color":"#CA0020"},{"color":"#7F7F7F"},{"color":"#0571B0"}]},{"id":"ER","name":"ER","type":"character","cell":[" ER Undetected"," ER Unchanged"," ER Undetected","⇧ ER Up"," ER Unchanged"," ER Undetected"],"maxWidth":150,"style":[{"color":"#7F7F7F"},{"color":"#7F7F7F"},{"color":"#7F7F7F"},{"color":"#CA0020"},{"color":"#7F7F7F"},{"color":"#7F7F7F"}]},{"id":"Windows","name":"Windows","type":"numeric","maxWidth":80},{"id":"p","name":"p","type":"numeric","cell":["2.63e-34","5.11e-13","7.81e-08","9.60e-08","8.29e-05","0.011"],"maxWidth":80},{"id":"padj","name":"p<sub>fdr<\/sub>","type":"numeric","cell":["2.10e-33","2.05e-12","1.92e-07","1.92e-07","1.33e-04","0.014"],"html":true},{"id":"Edge Size","name":"Edge Size","type":"numeric","maxWidth":80},{"id":"Leading Edge","name":"Leading Edge","type":"character","cell":[{"name":"span","attribs":{"title":"AQP3; UGDH; FKBP5; SEC14L2; CLDN8; EAF2; STEAP2; RANBP3L; HSD11B2; AC021148.3; PISD; ZNF652; ZNF689; OTULINL; ANKRD22; ARHGEF26; AKR1C3; EHF; KRT4; AMIGO2; PARM1; ARHGEF37; SERHL2; GLUL; THBS1; NRP1; EPGN; WWC3; CROT; GREB1; AL033523.1; PALMD; TTC12; PTPRJ; PWWP3B; ADD3; SLC16A9; SYBU; AR; MYLIP; RAB3B; ARHGEF28; NEDD4L; CSRNP1; GPCPD1; GPRC5A; WWC1; SASH1; PRR15; CNTNAP2; TBC1D16; SCUBE2; SLC26A2; AL035661.1; FRK; DIO2; SOWAHC; SERHL; PER1; DOCK2; AKAP13; SLC22A23; TBC1D2; CRYBG1; ASRGL1; LEF1; GRIN2B; TMTC2; TENT5C; CD9; PDE10A; PLCL1; SPRED1; PPP3CA; MCC; VSTM2L; HS3ST3B1; IQGAP2; SOX9; FOXI1; STON2; JUN; SOSTDC1; PRKAB2; NADK2; PMEPA1; MPDZ; NDRG1; TMEM164; IL20RA; NFKBIA; RRP7A; PPFIBP2; SLC16A6; MCCC2; GRHL2; AC007255.1; HEY2; RRP7BP; PLEKHD1; LNX2; KLF13; RASSF6; TRIM2; FHDC1; BLNK; PARP14; PREP; FAM214A; AC092053.4; TGFBR2; PMP22; CYP2B7P; FAM167A; ECI2; FGFR2; AL135905.2; ESR1; TMCC3; DKK1; ACKR3; MIR99AHG; SERPINI1; LMO3; ASB13; SSTR2; PARP9; SLC25A18; ADGRV1; MTSS1; ADH1C; PREX1; AL606500.1; ATP13A4; LRIG3; GCNT2; SOWAHA; STEAP4; FLRT3; CDC42EP3; CXCR4; MKLN1-AS; ENC1; KLF9; PGR; PEA15; GRM4; PCED1B; VLDLR; RNF152; NXPE3; TPD52; INSYN2A; SGK3; EMP1; TBC1D8; MKRN2OS; EPN2; BCL2L1; COLCA1; RAB6C; AC005920.1; SORD; ELL2; STARD13-AS; STARD13; MYCBP2; AL162377.1; KLF5; ZNF761; OXGR1; AL445309.1; DIPK2A; FAM107B; GNA12; LFNG; GUCY1A1; TTC21A; WWTR1; CACNB4; DOCK4; ZBTB7C; ANK3; F2R; MUC19; AL157895.2; RARRES1; TMEM17; TP53TG1; CCDC40; CNNM1; ALCAM; PID1; SERPINB2; PAQR5; LRCH1; SEMA6D; TMEM131L; GUCY1A2; MTMR9; ATP10A; GNMT; MYRFL; SNX25; VWFP1; PRKCE; NUAK1; PDE9A; CRAT; HECTD3; NAMPT; NR3C2; SOX5; PACSIN2; AK4; SPSB1; GASK1B; LINC02303; CHN1; GCOM1; CBX4; SLC1A1; AC108488.3; FBXO32; TNFAIP8; DENND2B; GATA3; AMACR; GPR137C; IKZF2; ABCC4; ARHGEF26-AS1; TESK2; MIR181A1HG; Z83843.1; TOE1; ITGA5; GCNT1; RNF144B; LRIG1; TPM3P9; ZNF442; RASSF3; NYNRIN; IGSF9; C15orf61; CCDC136; DNM3; BCL6; UAP1; AC093866.1; FAH; HERC3; KCNB1; SCIN; ACACA; SLC38A2; GLDC; CRISPLD1; DSCAM-AS1; ADCY2; RAB11FIP1P1; TIPARP; PEX10; LYPD6; LIPH; AC022400.8; PXDC1; STX18-AS1; CMAHP; DOCK8; CHRNB2; DLG2; SLC2A1; ACAA1; HSPA2; ADIRF-AS1; PDE4B; PXYLP1; TGFBR3; SRMS; LPAR3; ADAT2; CA13; SLC16A1; FZD5; PIK3C2G; AC024896.1; RGL1; NSRP1; AC105285.1; SLC22A5; RN7SL674P; SPRY4; RWDD3; VAV3; HMGCS1; CRYM; DLGAP1-AS1; TFPI2; FHL2; TTC32; C20orf144; ARL10; CASD1; GET4; ATP7B; TMEM154; PLEKHH1; CACNA1D; AL157392.3; MRFAP1; DHRS3; DDR2; RFLNA; SLC44A5; PARTICL; AL158211.5; NAGK; QSOX2; CDK18; SORD2P; PRR15L; HNRNPA1P48; TSPAN33; PRAG1; SMKR1; KIF21A; HECW2; LDHB; KRT23; NIFK-AS1; DBI; KLF3-AS1; LRRC46; RAD51B; KIAA0825; AC022137.4; HSP90AA2P; VMP1; PDIA5; ACACB; DOK7; KRT10; LEKR1; MAGI2; RPH3AL; SRCIN1; SHANK2; ZMIZ1; LINC01881; IQCB1; OSR2; TES; PRKAA2; SMIM5; FAM198B-AS1; AL137782.1; EGLN3; CMTM8; TMEM63C; VIPR1; RAB30; SNORA37; SPRED2; HSPG2; FOXO6; LMO4; ITGAM; MEGF11; SLC27A2; PDE11A; FAM110B; CHPT1; AC118344.4; ZNRF1; AC083843.3; VWDE; DEPTOR; NRARP; IQCH-AS1; CLIP4; ID3; PHLDB1; BDNF-AS; SUSD2; DUSP16; ZBTB10; RBM20; LYST; TCF7L2; SLC6A17; ZNF837; AC026979.4; KLRG2; ST6GAL1; AC138028.4; AL133230.2; AKAP7; AZGP1P1; PSMD6-AS2; STARD5; AC113404.3; HGD; ZMYND10; NEDD4; C7orf31; COLCA2; PGAM4; ZNF365; ZFX; CSF1; LOXL3; BDKRB2; CD163L1; RNF43; CEP112; PSCA; NOL6; AC007285.1; REEP1; ID1; SKAP1; COL5A2; INPP5J; AC116533.1; GADD45B; ENPP1; ZNF674-AS1; AIFM2; PIFO; MNS1; PLLP; CNOT2-DT; PPARGC1B; SPRR3; AL135905.1; MYRIP; MAP1B; FGD6; IFI6; POTEKP; MFF-DT; CTSO; KL; CCDC96; ANKRD61; AP000873.2; AP005131.6; VAMP5; LIFR; PLCH1; SLC16A2; UNC5B; PDPR; CARMIL1; SEMA3C; SILC1; WNT5A; GRAMD1B; DRAIC; SLC20A1; ACSM1; NFATC2; MLLT3; DGAT2; AP001816.1; ANKRD44; AL391001.1; RBM26-AS1; LHFPL6; TRIM36; PRUNE2; MAFB; EPB41L2; NTN1; TMEM62; LRRC26; ARL15; CCDC34; TDO2; SMIM14; AC115223.1; TOR1AIP2; RFESD; FAM222B; CCDC85A; LRRC75A; HELZ2; DDX60; AC007637.1; IGFBP3; ZNF519; APBA1; SMAD9; MYB; RHOBTB1; GNB1L; RRS1; G3BP2; CXCR6; LINC00511; TRIM35; ADAMTSL4; TXNDC16; AC112777.1; ABCB6; RPL23AP2; COL12A1; CYTH1; FABP5; ANKRD10-IT1; ARHGEF3; ALPK1; RAB11B-AS1; ZFP36L2; STAT5A; LINC00265; NFKBIZ; NKX2-1; HSPBAP1; SLC6A11; SERTAD1; RMDN2; AC092171.5; CFAP45; PBX3; CATSPERB; BBOX1; MEIS1; HCAR1; OPN3; HES1; UROD; HNRNPA3P6; TSFM; TSPAN12; TSPAN5; AC007405.4; ERVK3-1; EDN2; EML6; PRDM4; RASD2; BX470102.2; ATP1A1; IGSF11; ENTPD3-AS1; PTCH1; AADAT; KCNE4; BPIFA4P; HOXC6; PXMP4; NKILA; FRMD4A; RAPGEF4; RNF166; AC124944.2; QDPR; SLC39A14; AC027097.2; EFNA4; PTCHD1; SLC16A4; HMGCS2; PLEKHG4B; MOB3C; RMI1; SLC16A10; PPIAP29; LDHD; FAM221A; TFCP2L1; AHR; CA8; SPIN3; AC022137.3; EVL; BCCIP; GPRIN3; BCAR3; RNASEH1-AS1; BCL2L11; MIR3936HG; MCOLN2; LINC02615; EPN3; UGT8; RPSAP47; LAMTOR5-AS1; TRIM45; AREG; PHKA1; SIAH2; BMF; SIDT1; RPL29P11; SALL2; BAHCC1; AC013394.1; AC090517.5; AC093642.2; AP4S1; POLA2; CACNA2D2; AC009318.2; AC020978.9; ZEB1-AS1; AC007114.1; AP000766.1; TSKU; EFNA3; PIP4P2; AL451123.1; AC141930.1; GRB7; CLIP2; STAMBPL1; KALRN; AC138028.2; MTHFS; ZFAND4; AL359715.1; ETV1; TUT7; CSTA; RPS2P55; C9orf106; LYSMD4; AL451165.2; ZSCAN12; SHROOM2; RNF215; C1QTNF3; MPC2; KCNK1; BHLHE41; FKRP; STXBP6; FZD8; SPRYD7; BRD7P2; CCDC7; SMCO4; WDFY3-AS2; ZBED3; OPLAH; RGS2; AL121906.2; RAP1GAP; SIK1B; FOXN3; MAP2K6; FMO5; TBCE; SRRM2; OAS1; GPR157; AL158163.1; NCOR2; PIK3IP1; FZD1; SELENON; UXS1; SIK1; FAM151B; GAR1; PRIMA1; AC104825.1; DICER1-AS1; ATP6V1B2; CASC2; C15orf62; EEFSEC; HLF; LINC01239; MAN1C1; FAM241B; TNFRSF11A; ANKRD62P1; PPM1M; RAB6B; AC007743.1; FAM189A2; DNMBP; ENSAP2; AC008895.1; AL157838.1; AL139385.1; ANG; YPEL1; HTR7P1; AC135050.6; SUFU; SLC24A3; SH3RF2; ANP32B; CACYBPP2; COL27A1; SCD5; EML5; MAD2L1BP; AC084082.1; C15orf41; RASGEF1A; ITGAV; PALM2AKAP2; NCOA1; SMPDL3A; PDE3B; ZDHHC8P1; AC097376.3; PRELID2; AL009174.1; SLC35C1; RFLNB; ATXN1; FAM53A; GGT7; LINC00624; GPR37; ACTN2; FTX; CTU1; FLNB-AS1; FAM81A; AC104109.2; CYP4Z2P; Z99572.1; NFIX; TNS1; AL049834.1; PLEKHG5; C3orf62; POU2F1; AC084824.5; CHST15; EEPD1; CLSTN2; RHOB; KCNMA1; FBXL16; CRY2; MATN1-AS1; BAALC-AS1; ELAC1; VPS37D; FKBP9P1; ZNF594; TMC8; DHCR24; NTRK3; CAMKMT; HDAC9; AL158212.3; AC009133.1; ALPK3; TRIM68; FBXO48; CLN3; OLMALINC; AC097523.1; ARHGAP5-AS1; RABEP1; SNORD3C; FOXO1; KDM4A-AS1; ZSWIM3; PRSS53; ZNF438; TTC23; LRRC49; NHEJ1; HOMER2; APIP; KIAA1211L; OCEL1; LARGE1; KCND3; UBE3D; SRCAP; C5orf63; ETHE1; KLHL8; POLE4; UBALD2; SRRM3; PLA2R1; MAPT; RPL7P23; HACD1; CPQ; ASAP1; RNF125; METAP1D; RAD51-AS1; CSGALNACT1; ELK3; TMEM8B; NOL4L; ZSCAN5A; KCNS3; HINT2; KLF3; CYSRT1; ACSS1; ENOX1; GTF2IP13; PPP1R37; RIPOR3; LINC00888; TSC22D1; LINC01504; PTPRB; LINC01521; METTL15P1; RFX1; TIMP2; MIR17HG; LCA5; IGSF8; AC091982.3; CPAMD8; ZFPM1; DLEU1; CAPZB; CA2; CSPG4P12; ROBO1; LRRC27; TTC13; CUEDC1; OTUD7A; KCNJ11; LTBP4; FOXD1; MAOB; PAQR3; MBD3; UQCRHL; PRSS27; USP3-AS1; CIART; MBTPS1; TUBD1; DGLUCY; MED21; AL391684.1; UNC5C; PSMC5; ARID5B; PARP8; BCAS3; SMOC1; CEP126; TTYH2; AL355488.1; SCUBE3; ALKBH7; MGAM; MTCL1; LINC02482; RPS26P6; MAP3K5; CASP9; ACYP2; ENTPD8; AC018521.5; PROM1; SERGEF; AC011476.3; ANGPTL4; PPARG; AC018797.3; CRISPLD2; AUH; IGF2BP2; PIK3CB; DNAH11; RTN4IP1; DBNDD2; ABCC12; AC004943.2; NOSTRIN; NEMP2; MAFF; EPS8; CTU2; GIHCG; PPP3CB-AS1; SIX1; CALML5; SESTD1; DHRS2; EFNA5; NEK5; PAX9; ANKRD18B; AC015712.2; ASH1L-AS1; MYC; ZC3H12A; MHENCR; SPAG6; TSEN2; YBX1P1; RERG; JMJD7-PLA2G4B; NDFIP2; MRC2; NHSL1; TPO"},"children":["AQP3; UGDH; FKBP5; SEC14L2; CLDN8; EAF2; STEAP2..."]},{"name":"span","attribs":{"title":"AQP3; UGDH; FKBP5; AC021148.3; PISD; ARHGEF26; EHF; PARM1; ARHGEF37; GLUL; WWC3; GREB1; PTPRJ; ADD3; SYBU; NEDD4L; GPRC5A; WWC1; IL1R1; TBC1D16; SCUBE2; OSGIN1; SLC26A2; TBC1D2; KRT15; LEF1; TMTC2; TENT5C; CD9; PDE10A; PLCL1; TRGC1; CABLES1; MPDZ; TMEM164; BCL2; NFKBIA; PPFIBP2; SLC16A6; MCCC2; GRHL2; AC007255.1; HEY2; FAM214A; CYP2B7P; ESR1; ASB13; SSTR2; MTSS1; STEAP4; CSGALNACT2; GSTM3; TPD52; TBC1D8; STARD13-AS; STARD13; MYCBP2; KLF5; FAM107B; HMGN5; LRP2BP; YPEL3; FRMD3; ZBTB7C; SOX13; PAQR5; TRGC2; GUCY1A2; SNX25; PRKCE; SLC16A14; ZNF703; GATA3; ABCC4; IGFBP4; ARHGEF26-AS1; ZNF552; LINC01341; GCNT1; LRIG1; BCL6; FAH; HERC3; AP000331.1; GDPD3; AC008268.1; SLC38A2; GIPR; DSCAM-AS1; ADCY2; TIPARP; LYPD6; DOCK8; DLG2; TAPT1; SAMD12-AS1; RGL1; SLC22A5; RN7SL674P; HEG1; TFPI2; SEPHS2; CASD1; GET4; CACNA1D; MRFAP1; DHRS3; RFLNA; PARTICL; PRAG1; CORO1A; SYT8; RAD51B; VMP1; ARHGAP40; LEKR1; SGPP1; PPP1R3D; TSC22D3; S100A8; SMIM5; RAB30; SPRED2; C14orf132; ITGAM; CHPT1; ZNRF1; RAD17P1; ARRB1; KRT81; SUSD2; DUSP16; RBM20; AZGP1P1; HGD; FAM117A; LRRC37A3; NOL6; SKAP1; GADD45B; PPARGC1B; MYRIP; KL; AP005131.6; MFSD2A; VAMP5; DRAIC; SLC20A1; HAP1; AC007191.1; ACSM1; IRS2; AL391001.1; AC007996.1; ENKD1; PRUNE2; LRRC26; SMIM14; GAB2; HERC6; GRIK3; MYB; LINC00511; RPL23AP2; ALPK1; CBR3-AS1; GADD45G; RMDN2; AC092171.5; HES1; EML6; PRDM4; RASD2; ATP1A1; SPOCD1; PXMP4; AC027097.2; SLC16A4; HMGCS2; LDHD; AHR; BCAR3; RNASEH1-AS1; MIR3936HG; GATA3-AS1; LAMTOR5-AS1; AC106782.1; AREG; SIAH2; BMF; TAPT1-AS1; RCBTB2; TSKU; GRB7; LCMT1-AS2; FKRP; CCDC7; AL158163.1; NCOR2; AC104667.2; FAM151B; USP35; DICER1-AS1; AL591848.4; CASC2; AC008895.1; AL157838.1; ANKRD34A; ZNF814; ZCCHC17; FEM1A; SLC24A3; AL157935.2; SEPTIN1; CLIC6; EML5; AC084082.1; PALM2AKAP2; PRELID2; ZNF688; CTU1; FLNB-AS1; PLAAT4; RTN4RL1; AL049834.1; DAPK2; CHST15; CLSTN2; RHOB; BICRA; KCNMA1"},"children":["AQP3; UGDH; FKBP5; AC021148.3; PISD; ARHGEF26; ..."]},{"name":"span","attribs":{"title":"UGDH; RANBP3L; AC021148.3; OTULINL; ARHGEF26; EHF; KRT4; AMIGO2; GLUL; THBS1; NRP1; WWC3; GREB1; AL033523.1; PTPRJ; ADD3; SYBU; NEDD4L; SASH1; PRR15; CNTNAP2; IL1R1; TBC1D16; HPGD; SCUBE2; SLC26A2; DIO2; SLC22A23; GRIN2B; TMTC2; CD9; PPP3CA; VSTM2L; IQGAP2; SOX9; STON2; CABLES1; JUN; SOSTDC1; PRKAB2; MPDZ; SRGAP3; PPFIBP2; MCCC2; GRHL2; AC007255.1; HEY2; TRIM9; ECI2; AL135905.2; TMCC3; DKK1; SERPINI1; ASB13; PARP9; SLC25A18; ADGRV1; PREX1; PLEKHH2; POU2F3; LRIG3; GCNT2; CDC42EP3; ENC1; PCED1B; RNF152; NXPE3; TPD52; INSYN2A; TBC1D8; COLCA1; AC005920.1; ELL2; SNORD94; STARD13-AS; STARD13; MYCBP2; OXGR1; AL445309.1; CRYBG2; LFNG; AC005912.1; ZBTB7C; ANK3; MUC19; RARRES1; CNNM1; ALCAM; LRATD1; LRCH1; SEMA6D; TMEM131L; GUCY1A2; PRKCE; SLC16A14; SOX5; PACSIN2; LINC02303; DENND2B; SLC9A1; IKZF2; ABCC4; ARHGEF26-AS1; TESK2; SYT16; TOE1; LINC01341; GCNT1; SGSM1; RNF144B; RASSF3; RBBP8NL; IGSF9; DNM3; BCL6; AP000331.1; AC022007.1; KCNB1; SCIN; ANKH; AC008268.1; SLC38A2; GLDC; GIPR; CRISPLD1; ADCY2; TIPARP; LYPD6; PXDC1; ZDHHC11; AL603750.1; CMAHP; DLG2; PDE4B; LRATD2; CA13; FZD5; PIK3C2G; SAMD12-AS1; RGL1; AC105285.1; RN7SL674P; AC005083.1; VAV3; LINC01814; SEPHS2; C20orf144; GET4; MAP2; NRCAM; AL157392.3; DHRS3; SYDE1; RFLNA; KCNK5; AL158211.5; CDK18; PRR15L; GLCCI1; SMKR1; KIF21A; TRIM6; LDHB; TRERF1; KLF3-AS1; RAD51B; KIAA0825; VMP1; ARHGAP40; PCDH10; ACACB; LEKR1; MAGI2; SGPP1; MOK; SHANK2; ZMIZ1; PPP1R3D; ARID5A; IVL; PRKAA2; EGLN3; CMTM8; L3MBTL3; SPRED2; SPCS2P4; C14orf132; ADAM19; CXXC4; RNF150; SLC27A2; PDE11A; LYRM9; MUC16; CHPT1; SCYL3; RAD17P1; VWDE; SLC4A8; Y_RNA; AL357568.1; SOX11; LINC01176; HGD; C7orf31; COLCA2; SYT15; BDKRB2; CCDC82; CD163L1; RNF43; CEP112; PSCA; REEP1; ID1; COL5A2; INPP5J; AC116533.1; ENPP1; BCYRN1; PTAFR; IFNLR1; PPARGC1B; AL135905.1; MYRIP; MFSD2A; NFASC; PLCH1; PDPR; CARMIL1; SEMA3C; WNT5A; DRAIC; ACSM1; NFATC2; MLLT3; AP001816.1; ANKRD44; AL391001.1; RBM26-AS1; PRUNE2; TLK2P1; SMIM14; AC115223.1; ABCC3; METTL7B; NOVA1; AC110285.2; ETV4; CCDC85A; NSMCE3; RHOBTB1; GNB1L; G3BP2; LINC00511; CAMK4; DBP; IL17RE; ABCB6; ANKRD10-IT1; ARHGEF3; ALPK1; MAGI3; NAALAD2; LINC00265; NFKBIZ; AC004982.1; CFAP45; PBX3; CATSPERB; BBOX1; OPN3; HUS1; TSFM; LURAP1L; TSPAN5; AC007405.4; KIFC3; RASD2; BX470102.2; ENTPD3-AS1; FANCE; PXMP4; NKILA; FRMD4A; QDPR; NBL1; AC027097.2; EFNA4; AP002490.1; HMGCS2; HEBP1; SP140L; SNHG22; RNY3P1; FAM221A; RAPGEF3; AHR; CA8; CCDC167; EVL; BCCIP; BCAR3; RNASEH1-AS1; IL17RD; AC090772.5; LINC02615; DAPK1; NAV1; AL590714.1; EPN3; RNF128; FAM89A; NABP1; GPR162; CD302; BAHCC1; AC013394.1; AP4S1; AC018809.2; AC007114.1; MTUS2; EFNA3; AXIN2; GRB7; AC092910.3; KALRN; PPARD; FAM66C; ETV1; FO681492.1; SIRPA; LYSMD4; LCMT1-AS2; MS4A7; KCNK1; KLHL2; BHLHE41; FKRP; STXBP6; ZBED3; LINC-PINT; CMPK2; AL121906.2; JAG1; LIME1; SIK1B; FOXN3; POLR2J4; FMO5; OAS1; CRELD1; WHAMMP2; SIK1; AL591848.4; C15orf62; EEFSEC; PARD3B; RAB6B; AC007743.1; THAP7-AS1; HTR7P1; AC135050.6; SLC24A3; SPC25; AL157935.2; AC080013.1; MPP7; SCD5; EML5; RASGEF1A; ZNF784; SLC35C1; ATXN1; LINC00624; ACTN2; FAM81A; B3GNTL1; PITX1; AL049834.1; DAPK2; NSMCE1-DT; CHST15; CLSTN2; BICRA; KCNMA1; BAALC-AS1; ELAC1; AP1AR; KIF12; ZNF594; DHCR24; CAMKMT; HDAC9; AL158212.3; AC009133.1; AC097523.1; SNAPC1; RABEP1; ZSWIM3; PRSS53; GLCE; NHEJ1; APIP; KIAA1211L; LARGE1; KCND3; SRCAP; ABLIM3; ETHE1; UBALD2; PLA2R1; AC093627.4; CPQ; CENPQ; METAP1D; CSGALNACT1; ZDHHC18; KCNS3; KLF3; CCDC18-AS1; PTPRB; METTL15P1; MR1; ELAVL2; DLEU1; CAPZB; ROBO1; LRRC27; CUEDC1; CHST3; PAQR3; OSBPL6; DOCK3; AC005070.3; APBA2; TUBD1; DGLUCY; AL391684.1; RPL13P5; SERPINB5; UNC5C; ARID5B; PARP8; BCAS3; SMOC1; CEP126; RTEL1-TNFRSF6B; LINC00910; AC018521.5; SERGEF; DNAH11; DBNDD2; NOSTRIN; EPS8; GIHCG; CALML5; EFNA5; PAX9; ZNF428; ZC3H12A; MHENCR; AC010168.2; NDFIP2; NHSL1; ANAPC11; CTNND2; CRTC3; KIF13B; MED4; FBXO4; ZBED3-AS1; TOB1-AS1; ATF7IP; KIAA1522; TLE1; AC005914.1; NCALD; PANK1; TTLL1; SH3TC1; AC139149.1; KIAA1671; AL121839.2; TMC5; MAML2; PYDC1; DACH1; ARSJ; DCST1-AS1; RFX3; NCEH1; CTAGE9; AC007114.2; FAM241A; SLC9A3; ZNF580; HMOX1; ITPRIPL2; IQCK; RN7SKP255; ABTB2; Z93930.2; RTN1; PDCD6IPP2; CMYA5; SLC39A1; DDX12P; CCDC137; AC004982.2; PRDM11; C17orf67; COMTD1; INSIG2; RTEL1; KANSL1L; KHDRBS3; S100A1; B3GALT2; CCDC138; TM4SF1; NR4A2; RASGEF1B; AVIL; PCLO; GDPGP1; ARHGAP30; NHSL2; LETM2; FMNL2; SHLD2P1; ADNP-AS1; C9orf72; MBNL2; GHR; GPATCH3; AC093001.1; TLL1; MACROD2; SNHG9; ETFB; PAG1; CCDC148; HDAC5; TTC28"},"children":["UGDH; RANBP3L; AC021148.3; OTULINL; ARHGEF26; E..."]},{"name":"span","attribs":{"title":"UGDH; SEC14L2; HSD11B2; AC021148.3; ZNF652; EHF; WWC3; AL033523.1; KDM4B; SYBU; CSRNP1; SASH1; TBC1D16; PDE10A; MCCC2; AC007255.1; FAM214A; AC092053.4; GSTM3; TPD52; SGK3; RAB6C; AC005920.1; TTC21A; TMEM17; CCDC40; VWFP1; CRAT; FBXO32; BCL6; ACACA; PXDC1; H4C8; MRFAP1; PRR15L; HECW2; LRRC46; RAD51B; BAG3; PRKAA2; VIPR1; ZNRF1; RAD17P1; VIPR1-AS1; RBM20; ZFX; COL5A2; GADD45B; SILC1; SLC20A1; ACSM1; AL391001.1; SMIM14; ABCC3; CEMIP2; MYB; AC009061.2"},"children":["UGDH; SEC14L2; HSD11B2; AC021148.3; ZNF652; EHF..."]},{"name":"span","attribs":{"title":"EHF; KRT4; ADD3; AR; GPRC5A; FAM49A; IL1R1; SOWAHC; DOCK2; KRT15; TENT5C; PLCL1; SOX9; FOXI1; CABLES1; PMEPA1; NDRG1; SRGAP3; BCL2; GRHL2; CYP2B7P; ESR1; SSTR2; ADGRV1; MTSS1; PREX1; TACR3; GRM4; EMP1; BCL2L1; COLCA1; AC005920.1; STARD13; LFNG; FSBP; ZBTB7C; CLEC3A; VWFP1; NR3C2; PXK; GCOM1; SLC1A1; AC004918.5; CEACAM6; SAP30L-AS1; LINC01341; SGSM1; RASSF3; HERC3; SCIN; ANKH; PGM5; SH3PXD2A; DLG2; RORC; SEMA3F-AS1; SLC22A5; GPR161; SPRY4; FHL2; TMEM154; PLEKHH1; AL157392.3; DHRS3; RFLNA; AC119673.3; PRAG1; GLCCI1; KCTD6; VMP1; ACACB; DOK7; LEKR1; CGNL1; SRCIN1; TSC22D3; TES; LIMD2; C14orf132; PDE11A; LYRM9; ZNRF1; HNRNPCP2; DEPTOR; KRT81; ZNF385A; DUSP16; ZBTB10; RBM20; AC026979.4; AL133230.2; LINC01176; COLCA2; BDKRB2; CEP112; ID1; COL5A2; FGD6; MFSD2A; SEMA3C; DRAIC; CRNDE; AL391001.1; AC007996.1; LRRC26; AC110285.2; HERC6; GRIK3; AC007637.1; CEMIP2; MYB; RHOBTB1; AC093904.4; LINC00511; TXNDC16; RAB11B-AS1; LINC00265; CBR3-AS1; SPRY1; HES1; LURAP1L; TSPAN5; RASD2; KCNE4; SPOCD1; FRMD4A; AC027097.2; PTCHD1; HMGCS2; SP140L; LINC01522; LDHD; AHR; EVL; GALNT10; MIR3936HG; PCDHA12; AC090772.5; RPSAP47; BAHCC1; AP000766.1; TSKU; AXIN2; KALRN; TUT7; TBC1D30; MB21D2; MS4A7; FKRP; CCDC7; ZBED3; PCDHAC1; OAS1; AL158163.1; AL591848.4; PARD3B; ANKRD62P1; SLC24A3; ANP32B; RASGEF1A; ITGAV; PALM2AKAP2; SMPDL3A; GSEC; LINC00624; MIR4453HG; CLSTN2; KCNMA1; ACOX2; HDAC9; TNS2; ZSWIM3; LARGE1; UBE3D; CPQ; ASAP1; TMEM8B; AC010536.1; ACSS1; RIPOR3; MR1; IGSF8; AC091982.3; HS6ST1; DLEU1; CUEDC1; TUBD1; DGLUCY; MED21; PSMC5; ARID5B; BCAS3; RBL2; ACYP2; PROM1; AC108134.2; RGS9BP; ABCC12; CALML5; TMEM186; JMJD7-PLA2G4B; RAB19; CTNND2; ZBED3-AS1; TOB1-AS1"},"children":["EHF; KRT4; ADD3; AR; GPRC5A; FAM49A; IL1R1; SOW..."]},{"name":"span","attribs":{"title":"DOCK2; RNF145"},"children":["DOCK2; RNF145"]}],"minWidth":150}],"filterable":true,"dataKey":"2ab9324467e3d49173f26639baefb480"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<pre class="r"><code>plot_grid(plotlist = p, ncol = min(3, length(p)))</code></pre>
<div class="figure" style="text-align: center">
<img src="AR_E2_E2DHT_ER_E2_E2DHT_pairwise_comparison_files/figure-html/barcode-gsea-nondir-sig-1.png" alt="*Barcode plots for the top 6 sets of windows associated with __non-directional__ changes in gene expression.*" width="960" />
<p class="caption">
<em>Barcode plots for the top 6 sets of windows associated with
<strong>non-directional</strong> changes in gene expression.</em>
</p>
</div>
</div>
</div>
<div id="integration-of-rna-and-chip-enrichment-analyses"
class="section level2">
<h2>Integration of RNA and ChIP Enrichment Analyses</h2>
<p>Directional GSEA was performed on the set of RNA-Seq results and
given that enrichment for differentially bound target is an orthogonal
measure, enrichment for a set of potential pathways for each pairwise
set of regions was performed. The p-values from binding enrichment and
GSEA were combined using Wilkinson’s method for meta-analysis, taking
the maximum p-value and adjusting the resulting p-value using the same
strategy as earlier. Any pathways are annotated as being significant in
<em>both, either or neither</em> dataset when being analysed
individually.</p>
<p>Distances between nodes (gene-sets) for any network plots were
estimated by using the similarity of genes within the leading edge which
were also mapped to binding regions showing the respective pattern. If
the genes of interest from any two pathways were identical, only the
most highly-ranked pathway was included in the network, and similarly if
one gene-set was a subset of another, the most highly-ranked was
retained for visualisation.</p>
<pre class="r"><code>rnaseq_gsea &lt;- suppressWarnings(
  fgsea(
    pathways = gs_by_gsid,
    stats = rna_dir_ranks, 
    minSize = min_gs_size, 
    maxSize = max_gs_size
  )
) %&gt;% 
  dplyr::filter(!is.na(pval))
status_cols &lt;- setNames(
  hcl.colors(4, &quot;Viridis&quot;, rev = TRUE),
  c(&quot;Both&quot;, &quot;ChIP Only&quot;, &quot;RNA Only&quot;, &quot;Neither&quot;)
)
cmn_res &lt;- goseq_group_res %&gt;% 
  lapply(dplyr::select, group, gs_name, pval_goseq = pval, adj_p, starts_with(&quot;num&quot;)) %&gt;% 
  lapply(
    left_join, 
    dplyr::select(rnaseq_gsea, gs_name = pathway, NES, pval_gsea = pval, padj, leadingEdge), 
    by = &quot;gs_name&quot;
  ) %&gt;% 
  lapply(
    nest, p = starts_with(&quot;pval&quot;)
  ) %&gt;% 
  lapply(
    dplyr::rename, padj_chip = adj_p, padj_rna = padj
  ) %&gt;% 
  lapply(
    mutate, 
    pval = vapply(
      p, function(x) metap::maximump(unlist(x))$p, numeric(1)
    ),
    adj_p = p.adjust(pval, adj_method)
  ) %&gt;% 
  lapply(dplyr::filter, adj_p &lt; enrich_alpha) %&gt;% 
  lapply(
    mutate,
    status = case_when(
      padj_chip &lt; enrich_alpha &amp; padj_rna &lt; enrich_alpha ~ &quot;Both&quot;,
      !padj_chip &lt; enrich_alpha &amp; padj_rna &lt; enrich_alpha ~ &quot;RNA Only&quot;,
      padj_chip &lt; enrich_alpha &amp; !padj_rna &lt; enrich_alpha ~ &quot;ChIP Only&quot;,
      !padj_chip &lt; enrich_alpha &amp; !padj_rna &lt; enrich_alpha ~ &quot;Neither&quot;
    ) %&gt;% 
      factor(levels = names(status_cols)),
    leadingEdge = lapply(leadingEdge, intersect, group_ids[[unique(group)]]),
    numDEInCat = vapply(leadingEdge, length, integer(1)),
    genes = vapply(
      leadingEdge, function(x) paste(id2gene[x], collapse = &quot;; &quot;), character(1)
    )
  ) %&gt;% 
  lapply(dplyr::filter, numDEInCat &gt;= min_sig) %&gt;% 
  lapply(
    select,
    group, gs_name, adj_p, NES, status, starts_with(&quot;padj&quot;),
    starts_with(&quot;num&quot;), leadingEdge, genes
  ) %&gt;% 
  lapply(dplyr::filter, numDEInCat &gt; 0) %&gt;% 
  .[vapply(., nrow, integer(1)) &gt; 0] 
tg_cmn &lt;- names(cmn_res) %&gt;% 
  lapply(
    function(x) {
      make_tbl_graph(
        cmn_res[[x]],
        setNames(cmn_res[[x]]$leadingEdge, cmn_res[[x]]$gs_name)
      )
    }
  ) %&gt;% 
  setNames(names(cmn_res)) %&gt;% 
  .[vapply(., length, integer(1)) &gt;= min_network_size]
any_sig &lt;- length(cmn_res) &gt; 0
plot_net &lt;- length(tg_cmn) &gt; 0
txt &lt;- ifelse(any_sig, &quot;### Results {.tabset}&quot;, &quot;No shared enrichment was found&quot;)</code></pre>
<div id="results-1" class="section level3 tabset">
<h3 class="tabset">Results</h3>
<pre class="r"><code>htmltools::tagList(
  names(cmn_res) %&gt;% 
    lapply(
      function(x) {
        df &lt;- cmn_res[[x]] %&gt;% 
          dplyr::select(
            gs_name, adj_p, NES, status, starts_with(&quot;padj&quot;), numDEInCat, genes
          )
        tbl &lt;- df %&gt;% 
          arrange(adj_p) %&gt;% 
          reactable(
            filterable = TRUE, showPageSizeOptions = TRUE,
            columns = list(
              gs_name = colDef(
                name = &quot;Gene Set&quot;,
                cell = function(value) htmltools::tags$a(
                  href = gs_url[[value]], 
                  target = &quot;_blank&quot;, 
                  str_replace_all(value, &quot;_&quot;, &quot; &quot;)
                ),
                minWidth = 120
              ),
              NES = colDef(
                format = colFormat(digits = 2),
                style = function(value) {
                  col &lt;- ifelse(value &gt; 0, colours$direction$up, colours$direction$down)
                  list(color = col)
                },
                maxWidth = 80
              ),
              adj_p = colDef(
                name = &quot;p&lt;sub&gt;adj&lt;/sub&gt;&quot;, html = TRUE,
                cell = function(value) ifelse(
                  value &lt; 0.001,
                  sprintf(&quot;%.2e&quot;, value),
                  sprintf(&quot;%.3f&quot;, value)
                ),
                maxWidth = 80
              ),
              status = colDef(name = &quot;Individual Status&quot;, maxWidth = 100),
              padj_rna = colDef(
                name =  glue(&quot;p&lt;sub&gt;{params$enrichment$adj}&lt;/sub&gt; (RNA)&quot;), html = TRUE,
                cell = function(value) ifelse(
                  value &lt; 0.001,
                  sprintf(&quot;%.2e&quot;, value),
                  sprintf(&quot;%.3f&quot;, value)
                ),
                maxWidth = 80
              ),
              padj_chip = colDef(
                name =  glue(&quot;p&lt;sub&gt;{params$enrichment$adj}&lt;/sub&gt; (ChIP)&quot;), html = TRUE,
                cell = function(value) ifelse(
                  value &lt; 0.001,
                  sprintf(&quot;%.2e&quot;, value),
                  sprintf(&quot;%.3f&quot;, value)
                ),
                maxWidth = 80
              ),
              numDEInCat = colDef(name = &quot;# Targets in Leading Edge&quot;, maxWidth = 100),
              genes = colDef(
                name = &quot;Targets in Leading Edge&quot;,
                cell = function(value) with_tooltip(value, width = 50)
              )
            )
          )
        cp &lt;- glue(
          &quot;The {nrow(df)} Gene sets considered significant when combining GSEA 
          from RNA-Seq and enrichment anaylsis from binding sites corresponding 
          to {x}. The direction of the NES indicates the enriched direction in 
          the RNA-Seq data.
        &quot;
        )
        htmltools::div(
          htmltools::div(
            id = x %&gt;% 
              str_to_lower %&gt;% 
              str_replace_all(&quot; &quot;, &quot;-&quot;) %&gt;% 
              str_replace_all(&quot;-+&quot;, &quot;-&quot;) %&gt;% 
              paste(&quot;gsea&quot;, sep = &quot;-&quot;),
            class=&quot;section level4&quot;,
            htmltools::h4(class = &quot;tabset&quot;, x),
            htmltools::tags$em(cp),
            tbl
          )
        )
      }
    ) %&gt;% 
    setNames(names(cmn_res))
)</code></pre>
<div>
<div id="ar-up-er-unchanged-gsea" class="section level4">
<h4 class="tabset">AR Up - ER Unchanged</h4>
<em>The 4 Gene sets considered significant when combining GSEA 
  from RNA-Seq and enrichment anaylsis from binding sites corresponding 
  to AR Up - ER Unchanged. The direction of the NES indicates the enriched direction in 
  the RNA-Seq data.</em>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-6e74753cabae56fa6a84" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-6e74753cabae56fa6a84">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"gs_name":["HALLMARK_ANDROGEN_RESPONSE","HALLMARK_ESTROGEN_RESPONSE_EARLY","HES2_TARGET_GENES","REACTOME_PHASE_II_CONJUGATION_OF_COMPOUNDS"],"adj_p":[0.000279852076730022,0.00701757416451247,0.00701757416451247,0.0207168148331563],"NES":[1.6249595229107,1.46036118316238,1.35500007120358,1.47281975671626],"status":["Neither","ChIP Only","Neither","Neither"],"padj_chip":[0.203025723858998,5.39482075032928e-10,0.847225580493292,0.932523160090317],"padj_rna":[0.0509279866433805,0.191761967394594,0.0971397746224638,0.249800055163013],"numDEInCat":[7,19,22,4],"genes":["FKBP5; SLC26A2; STEAP4; TPD52; ABCC4; HERC3; SLC38A2","AQP3; FKBP5; GREB1; ADD3; SYBU; WWC1; SLC26A2; KRT15; TMEM164; ASB13; IGFBP4; LRIG1; TIPARP; SLC22A5; CHPT1; HES1; AREG; SIAH2; TSKU","AQP3; EHF; GREB1; ADD3; SYBU; NEDD4L; GPRC5A; TBC1D16; SCUBE2; NFKBIA; MCCC2; KLF5; FAM107B; BCL6; SLC38A2; VMP1; SGPP1; SPRED2; HGD; IRS2; SMIM14; AREG","UGDH; SLC26A2; GSTM3; ACSM1"]},"columns":[{"id":"gs_name","name":"Gene Set","type":"character","cell":[{"name":"a","attribs":{"href":"\n      http://www.gsea-msigdb.org/gsea/msigdb/collections.jsp","target":"_blank"},"children":["HALLMARK ANDROGEN RESPONSE"]},{"name":"a","attribs":{"href":"\n      http://www.gsea-msigdb.org/gsea/msigdb/collections.jsp","target":"_blank"},"children":["HALLMARK ESTROGEN RESPONSE EARLY"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["HES2 TARGET GENES"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-156580|https://reactome.org/PathwayBrowser/#/R-HSA-156580","target":"_blank"},"children":["REACTOME PHASE II CONJUGATION OF COMPOUNDS"]}],"minWidth":120},{"id":"adj_p","name":"p<sub>adj<\/sub>","type":"numeric","cell":["2.80e-04","0.007","0.007","0.021"],"html":true,"maxWidth":80},{"id":"NES","name":"NES","type":"numeric","format":{"cell":{"digits":2},"aggregated":{"digits":2}},"maxWidth":80,"style":[{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"}]},{"id":"status","name":"Individual Status","type":"factor","maxWidth":100},{"id":"padj_chip","name":"p<sub>fdr<\/sub> (ChIP)","type":"numeric","cell":["0.203","5.39e-10","0.847","0.933"],"html":true,"maxWidth":80},{"id":"padj_rna","name":"p<sub>fdr<\/sub> (RNA)","type":"numeric","cell":["0.051","0.192","0.097","0.250"],"html":true,"maxWidth":80},{"id":"numDEInCat","name":"# Targets in Leading Edge","type":"numeric","maxWidth":100},{"id":"genes","name":"Targets in Leading Edge","type":"character","cell":[{"name":"span","attribs":{"title":"FKBP5; SLC26A2; STEAP4; TPD52; ABCC4; HERC3; SLC38A2"},"children":["FKBP5; SLC26A2; STEAP4; TPD52; ABCC4; HERC3; SL..."]},{"name":"span","attribs":{"title":"AQP3; FKBP5; GREB1; ADD3; SYBU; WWC1; SLC26A2; KRT15; TMEM164; ASB13; IGFBP4; LRIG1; TIPARP; SLC22A5; CHPT1; HES1; AREG; SIAH2; TSKU"},"children":["AQP3; FKBP5; GREB1; ADD3; SYBU; WWC1; SLC26A2; ..."]},{"name":"span","attribs":{"title":"AQP3; EHF; GREB1; ADD3; SYBU; NEDD4L; GPRC5A; TBC1D16; SCUBE2; NFKBIA; MCCC2; KLF5; FAM107B; BCL6; SLC38A2; VMP1; SGPP1; SPRED2; HGD; IRS2; SMIM14; AREG"},"children":["AQP3; EHF; GREB1; ADD3; SYBU; NEDD4L; GPRC5A; T..."]},{"name":"span","attribs":{"title":"UGDH; SLC26A2; GSTM3; ACSM1"},"children":["UGDH; SLC26A2; GSTM3; ACSM1"]}]}],"filterable":true,"showPageSizeOptions":true,"dataKey":"cb3b6f0ca2bfbdb9ed61b4957fa10765"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div>
<div id="ar-up-er-undetected-gsea" class="section level4">
<h4 class="tabset">AR Up - ER Undetected</h4>
<em>The 3 Gene sets considered significant when combining GSEA 
  from RNA-Seq and enrichment anaylsis from binding sites corresponding 
  to AR Up - ER Undetected. The direction of the NES indicates the enriched direction in 
  the RNA-Seq data.</em>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-21bc23c4602c006b1ad0" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-21bc23c4602c006b1ad0">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"gs_name":["HALLMARK_ANDROGEN_RESPONSE","HES2_TARGET_GENES","HBZ_TARGET_GENES"],"adj_p":[5.71268482962184e-05,0.000416066862021432,0.0049526040837123],"NES":[1.6249595229107,1.35500007120358,1.3896538872197],"status":["Neither","Neither","Neither"],"padj_chip":[0.204074245115843,0.389434687640638,0.624988600824686],"padj_rna":[0.0509279866433805,0.0971397746224638,0.164688346461384],"numDEInCat":[18,52,20],"genes":["FKBP5; SLC26A2; IQGAP2; PMEPA1; NDRG1; STEAP4; TPD52; SORD; ELL2; TNFAIP8; ABCC4; UAP1; HERC3; SLC38A2; HMGCS1; DBI; ZBTB10; LIFR","AQP3; AKR1C3; EHF; NRP1; GREB1; PALMD; ADD3; SYBU; NEDD4L; GPRC5A; TBC1D16; SCUBE2; DIO2; PER1; CRYBG1; IQGAP2; PMEPA1; NDRG1; NFKBIA; MCCC2; KLF13; TGFBR2; LRIG3; SGK3; KLF5; FAM107B; WWTR1; MTMR9; CBX4; BCL6; SLC38A2; SLC16A1; SPRY4; DLGAP1-AS1; DBI; VMP1; RPH3AL; SPRED2; LMO4; HGD; NEDD4; CEP112; SMIM14; RFESD; COL12A1; CYTH1; ZFP36L2; TSFM; PTCH1; NKILA; LINC02615; AREG","AQP3; RANBP3L; NRP1; GREB1; SASH1; PPP3CA; MCCC2; PREP; EMP1; BCL2L1; FAM107B; WWTR1; SLC38A2; NSRP1; RN7SL674P; CRYM; AKAP7; NEDD4; LOXL3; EPB41L2"]},"columns":[{"id":"gs_name","name":"Gene Set","type":"character","cell":[{"name":"a","attribs":{"href":"\n      http://www.gsea-msigdb.org/gsea/msigdb/collections.jsp","target":"_blank"},"children":["HALLMARK ANDROGEN RESPONSE"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["HES2 TARGET GENES"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["HBZ TARGET GENES"]}],"minWidth":120},{"id":"adj_p","name":"p<sub>adj<\/sub>","type":"numeric","cell":["5.71e-05","4.16e-04","0.005"],"html":true,"maxWidth":80},{"id":"NES","name":"NES","type":"numeric","format":{"cell":{"digits":2},"aggregated":{"digits":2}},"maxWidth":80,"style":[{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"}]},{"id":"status","name":"Individual Status","type":"factor","maxWidth":100},{"id":"padj_chip","name":"p<sub>fdr<\/sub> (ChIP)","type":"numeric","cell":["0.204","0.389","0.625"],"html":true,"maxWidth":80},{"id":"padj_rna","name":"p<sub>fdr<\/sub> (RNA)","type":"numeric","cell":["0.051","0.097","0.165"],"html":true,"maxWidth":80},{"id":"numDEInCat","name":"# Targets in Leading Edge","type":"numeric","maxWidth":100},{"id":"genes","name":"Targets in Leading Edge","type":"character","cell":[{"name":"span","attribs":{"title":"FKBP5; SLC26A2; IQGAP2; PMEPA1; NDRG1; STEAP4; TPD52; SORD; ELL2; TNFAIP8; ABCC4; UAP1; HERC3; SLC38A2; HMGCS1; DBI; ZBTB10; LIFR"},"children":["FKBP5; SLC26A2; IQGAP2; PMEPA1; NDRG1; STEAP4; ..."]},{"name":"span","attribs":{"title":"AQP3; AKR1C3; EHF; NRP1; GREB1; PALMD; ADD3; SYBU; NEDD4L; GPRC5A; TBC1D16; SCUBE2; DIO2; PER1; CRYBG1; IQGAP2; PMEPA1; NDRG1; NFKBIA; MCCC2; KLF13; TGFBR2; LRIG3; SGK3; KLF5; FAM107B; WWTR1; MTMR9; CBX4; BCL6; SLC38A2; SLC16A1; SPRY4; DLGAP1-AS1; DBI; VMP1; RPH3AL; SPRED2; LMO4; HGD; NEDD4; CEP112; SMIM14; RFESD; COL12A1; CYTH1; ZFP36L2; TSFM; PTCH1; NKILA; LINC02615; AREG"},"children":["AQP3; AKR1C3; EHF; NRP1; GREB1; PALMD; ADD3; SY..."]},{"name":"span","attribs":{"title":"AQP3; RANBP3L; NRP1; GREB1; SASH1; PPP3CA; MCCC2; PREP; EMP1; BCL2L1; FAM107B; WWTR1; SLC38A2; NSRP1; RN7SL674P; CRYM; AKAP7; NEDD4; LOXL3; EPB41L2"},"children":["AQP3; RANBP3L; NRP1; GREB1; SASH1; PPP3CA; MCCC..."]}]}],"filterable":true,"showPageSizeOptions":true,"dataKey":"286ece9a854d5c95411cbcded86a7d64"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div>
<div id="ar-up-er-up-gsea" class="section level4">
<h4 class="tabset">AR Up - ER Up</h4>
<em>The 7 Gene sets considered significant when combining GSEA 
  from RNA-Seq and enrichment anaylsis from binding sites corresponding 
  to AR Up - ER Up. The direction of the NES indicates the enriched direction in 
  the RNA-Seq data.</em>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-4fe99a4ca6633b27f539" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-4fe99a4ca6633b27f539">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"gs_name":["HES2_TARGET_GENES","HALLMARK_ESTROGEN_RESPONSE_EARLY","REACTOME_PHASE_II_CONJUGATION_OF_COMPOUNDS","REACTOME_BIOLOGICAL_OXIDATIONS","HALLMARK_ESTROGEN_RESPONSE_LATE","REACTOME_HDMS_DEMETHYLATE_HISTONES","REACTOME_DNA_DOUBLE_STRAND_BREAK_RESPONSE"],"adj_p":[0.00114391002121867,0.00262687386798328,0.00748414361188775,0.00748414361188775,0.0333054156630815,0.0409358770028204,0.0409358770028204],"NES":[1.35500007120358,1.46036118316238,1.47281975671626,1.45799260570574,1.37829717139242,1.42225152994803,1.51816333092444],"status":["Neither","ChIP Only","ChIP Only","Neither","ChIP Only","ChIP Only","Neither"],"padj_chip":[0.0631736225648355,0.0127216166805464,0.0376421586315554,0.112702634304207,0.000570874631823097,0.0323766216004826,0.444088291657365],"padj_rna":[0.0971397746224638,0.191761967394594,0.249800055163013,0.263088056640676,0.394374559401878,0.429205918974404,0.191761967394594],"numDEInCat":[8,5,3,5,3,3,3],"genes":["EHF; SYBU; TBC1D16; MCCC2; SGK3; BCL6; SMIM14; AREG","SEC14L2; KDM4B; SYBU; AREG; SIAH2","UGDH; GSTM3; ACSM1","UGDH; GSTM3; ACSM1; AHR; GSTM4","UGDH; AREG; SIAH2","KDM4B; H4C8; ARID5B","KDM4B; H4C8; H2BC10"]},"columns":[{"id":"gs_name","name":"Gene Set","type":"character","cell":[{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["HES2 TARGET GENES"]},{"name":"a","attribs":{"href":"\n      http://www.gsea-msigdb.org/gsea/msigdb/collections.jsp","target":"_blank"},"children":["HALLMARK ESTROGEN RESPONSE EARLY"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-156580|https://reactome.org/PathwayBrowser/#/R-HSA-156580","target":"_blank"},"children":["REACTOME PHASE II CONJUGATION OF COMPOUNDS"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-211859|https://reactome.org/PathwayBrowser/#/R-HSA-211859","target":"_blank"},"children":["REACTOME BIOLOGICAL OXIDATIONS"]},{"name":"a","attribs":{"href":"\n      http://www.gsea-msigdb.org/gsea/msigdb/collections.jsp","target":"_blank"},"children":["HALLMARK ESTROGEN RESPONSE LATE"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-3214842|https://reactome.org/PathwayBrowser/#/R-HSA-3214842","target":"_blank"},"children":["REACTOME HDMS DEMETHYLATE HISTONES"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-5693606|https://reactome.org/PathwayBrowser/#/R-HSA-5693606","target":"_blank"},"children":["REACTOME DNA DOUBLE STRAND BREAK RESPONSE"]}],"minWidth":120},{"id":"adj_p","name":"p<sub>adj<\/sub>","type":"numeric","cell":["0.001","0.003","0.007","0.007","0.033","0.041","0.041"],"html":true,"maxWidth":80},{"id":"NES","name":"NES","type":"numeric","format":{"cell":{"digits":2},"aggregated":{"digits":2}},"maxWidth":80,"style":[{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"}]},{"id":"status","name":"Individual Status","type":"factor","maxWidth":100},{"id":"padj_chip","name":"p<sub>fdr<\/sub> (ChIP)","type":"numeric","cell":["0.063","0.013","0.038","0.113","5.71e-04","0.032","0.444"],"html":true,"maxWidth":80},{"id":"padj_rna","name":"p<sub>fdr<\/sub> (RNA)","type":"numeric","cell":["0.097","0.192","0.250","0.263","0.394","0.429","0.192"],"html":true,"maxWidth":80},{"id":"numDEInCat","name":"# Targets in Leading Edge","type":"numeric","maxWidth":100},{"id":"genes","name":"Targets in Leading Edge","type":"character","cell":[{"name":"span","attribs":{"title":"EHF; SYBU; TBC1D16; MCCC2; SGK3; BCL6; SMIM14; AREG"},"children":["EHF; SYBU; TBC1D16; MCCC2; SGK3; BCL6; SMIM14; ..."]},{"name":"span","attribs":{"title":"SEC14L2; KDM4B; SYBU; AREG; SIAH2"},"children":["SEC14L2; KDM4B; SYBU; AREG; SIAH2"]},{"name":"span","attribs":{"title":"UGDH; GSTM3; ACSM1"},"children":["UGDH; GSTM3; ACSM1"]},{"name":"span","attribs":{"title":"UGDH; GSTM3; ACSM1; AHR; GSTM4"},"children":["UGDH; GSTM3; ACSM1; AHR; GSTM4"]},{"name":"span","attribs":{"title":"UGDH; AREG; SIAH2"},"children":["UGDH; AREG; SIAH2"]},{"name":"span","attribs":{"title":"KDM4B; H4C8; ARID5B"},"children":["KDM4B; H4C8; ARID5B"]},{"name":"span","attribs":{"title":"KDM4B; H4C8; H2BC10"},"children":["KDM4B; H4C8; H2BC10"]}]}],"filterable":true,"showPageSizeOptions":true,"dataKey":"fa61c91e0d1a20507eef3228af8cc73a"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div id="all-groups-1" class="section level4">
<h4>All Groups</h4>
<pre class="r"><code>cmn_res %&gt;%  
  bind_rows() %&gt;% 
  arrange(adj_p) %&gt;% 
  left_join(n_mapped, by = c(&quot;group&quot; = &quot;status&quot;)) %&gt;% 
  mutate(
    gs_name = fct_inorder(gs_name) %&gt;% 
      fct_relabel(str_replace_all, &quot;_&quot;, &quot; &quot;) %&gt;% 
      fct_relabel(str_trunc, width = 60),
    prop = numDEInCat / detected,
    group = factor(group, levels = arrange(n_mapped, detected)$status)
  ) %&gt;%
  droplevels() %&gt;% 
  ggplot(aes(fct_rev(group), fct_rev(gs_name))) + 
  geom_point(aes(size = prop, fill = -log10(adj_p)), shape = 21) +
  geom_ysidecol(
    aes(x = numInCat), data = . %&gt;% distinct(gs_name, numInCat),
  ) +
  geom_xsidecol(
    aes(y = detected),
    data = n_mapped %&gt;% 
      dplyr::filter(status %in% names(cmn_res)) %&gt;% 
      dplyr::rename(group = status),
    width = 0.5
  ) +
  scale_x_discrete(labels = label_wrap(10)) +
  scale_ysidex_continuous(expand = expansion(c(0, 0.15))) +
  scale_xsidey_continuous(
    expand = expansion(c(0, 0.15))
  ) +
  scale_fill_viridis_c(option = &quot;inferno&quot;, begin = 0.2) +
  scale_size(range = c(0, 5), labels = percent) + 
  labs(
    x = c(), y = c(), size = &quot;% Mapped\n&amp; Detected\nGenes&quot;, 
    fill = expr(paste(-log[10], p[!!sym(adj_method)]))
  ) +
  theme(
    panel.grid = element_blank(), 
    axis.text = element_text(size = 8),
    legend.position = &quot;right&quot;,
    ggside.panel.scale.x = 0.2,
    ggside.panel.scale.y = 0.3,
    ggside.axis.text.x.bottom = element_text(angle = 270, hjust = 0, vjust = 0.5)
  ) </code></pre>
<div class="figure" style="text-align: center">
<img src="AR_E2_E2DHT_ER_E2_E2DHT_pairwise_comparison_files/figure-html/plot-all-enriched-rna-1.png" alt="*Combined enrichment across all groups, incorporating RNA expression data, showing only significant results for enrichment. The top panel shows how many detected genes were mapped to sites for each group, whilst the right panel shows gene set size, using detected genes only. Point sizes indicate the proportion of mapped and detected genes which are from each pathway.*" width="768" />
<p class="caption">
<em>Combined enrichment across all groups, incorporating RNA expression
data, showing only significant results for enrichment. The top panel
shows how many detected genes were mapped to sites for each group,
whilst the right panel shows gene set size, using detected genes only.
Point sizes indicate the proportion of mapped and detected genes which
are from each pathway.</em>
</p>
</div>
</div>
</div>
<div id="network-plots-2" class="section level3 tabset">
<h3 class="tabset">Network Plots</h3>
<pre class="r"><code>htmltools::tagList(
  names(tg_cmn) %&gt;% 
    mclapply(
      function(x) {
        ## Export the image
        img_out &lt;- file.path(
          fig_path,
          x %&gt;% 
            str_replace_all(&quot; &quot;, &quot;_&quot;) %&gt;% 
            str_replace_all(&quot;_-_&quot;, &quot;-&quot;) %&gt;% 
            paste0(&quot;_rnaseq_network.&quot;, fig_type)
        )
        fig_fun(
          filename = img_out,
          width = knitr::opts_current$get(&quot;fig.width&quot;),
          height = knitr::opts_current$get(&quot;fig.height&quot;)
        )
        p &lt;- tg_cmn[[x]] %&gt;%
          activate(nodes) %&gt;% 
          mutate(direction = ifelse(NES &gt; 0, &quot;Up&quot;, &quot;Down&quot;)) %&gt;% 
          ggraph(layout = net_layout, weights = oc^2) +
          geom_edge_link(aes(width = oc^2, alpha = oc^2)) +
          geom_node_point(
            aes(fill = status, size = numDEInCat),
            shape = 21
          ) +
          geom_node_text(
            aes(label = label, colour = direction),
            size = 3, 
            data = . %&gt;%
              mutate(
                label = str_replace_all(label, &quot;_&quot;, &quot; &quot;) %&gt;% str_trunc(60) %&gt;% str_wrap(width = 18)
              ),
            repel = TRUE,
            max.overlaps = max(10, round(length(tg_cmn[[x]]) / 4, 0)),
            bg.color = &quot;white&quot;, bg.r = 0.1
          ) +
          scale_x_continuous(expand = expansion(c(0.1, 0.1))) +
          scale_y_continuous(expand = expansion(c(0.1, 0.1))) +
          scale_fill_manual(values = status_cols) +
          scale_size_continuous(range = c(1, 10)) +
          scale_edge_width(range = c(1, 6), limits = c(0, 1)) +
          scale_edge_alpha(range = c(0.1, 0.4), limits = c(0, 1)) +
          scale_colour_manual(
            values = setNames(
              unlist(colours$direction), 
              str_to_title(names( unlist(colours$direction)))
            )
          ) +
          guides(edge_alpha= &quot;none&quot;) +
          labs(
            size = glue(&quot;{x} Targets In Leading Edge&quot;) %&gt;% str_wrap(15),
            colour = &quot;GSEA\nDirection&quot;,
            fill = &quot;Prior\nStatus&quot;,
            edge_width = expr(paste(OC^2))
          ) +
          theme_void()
        print(p)
        dev.off()
        ## Create html tags
        fig_link &lt;- str_extract(img_out, &quot;assets.+&quot;)
        cp &lt;- htmltools::tags$em(
          glue(
            &quot;
            Network plot showing enriched pathways mapped to genes 
            associated with {x} in combination with GSEA results from 
            the RNA-Seq data. 
            &quot;
          )
        )
        htmltools::div(
          htmltools::div(
            id = img_out %&gt;%
              basename() %&gt;%
              str_remove_all(glue(&quot;.{fig_type}$&quot;)) %&gt;%
              str_to_lower() %&gt;%
              str_replace_all(&quot;_&quot;, &quot;-&quot;),
            class=&quot;section level4&quot;,
            htmltools::h4(x),
            htmltools::div(
              class = &quot;figure&quot;, style = &quot;text-align: center&quot;,
              htmltools::img(src = fig_link, width = &quot;100%&quot;),
              htmltools::tags$caption(cp)
            )
          )
        )
      },
      mc.cores = threads
    )
)</code></pre>
<div>
<div id="ar-up-er-unchanged-rnaseq-network" class="section level4">
<h4>AR Up - ER Unchanged</h4>
<div class="figure" style="text-align: center">
<img src="assets/AR_E2_E2DHT-ER_E2_E2DHT/AR_Up-ER_Unchanged_rnaseq_network.png" width="100%"/>
<caption>
<em>Network plot showing enriched pathways mapped to genes 
associated with AR Up - ER Unchanged in combination with GSEA results from 
the RNA-Seq data. </em>
</caption>
</div>
</div>
</div>
<div>
<div id="ar-up-er-up-rnaseq-network" class="section level4">
<h4>AR Up - ER Up</h4>
<div class="figure" style="text-align: center">
<img src="assets/AR_E2_E2DHT-ER_E2_E2DHT/AR_Up-ER_Up_rnaseq_network.png" width="100%"/>
<caption>
<em>Network plot showing enriched pathways mapped to genes 
associated with AR Up - ER Up in combination with GSEA results from 
the RNA-Seq data. </em>
</caption>
</div>
</div>
</div>
</div>
</div>
</div>
<div id="data-export" class="section level1">
<h1>Data Export</h1>
<pre class="r"><code>write_rds(all_windows, all_out$results, compress = &quot;gz&quot;)
save.image(all_out$renv)</code></pre>
<pre class="r"><code>all_windows %&gt;% 
  as_tibble() %&gt;% 
  dplyr::select(-detected) %&gt;% 
  unnest(all_of(&quot;gene_id&quot;)) %&gt;% # This ensures correct id2gene mappings
  mutate(gene_name = id2gene[gene_id]) %&gt;% 
  dplyr::select(
    starts_with(&quot;gene&quot;), range, any_of(c(&quot;region&quot;, &quot;feature&quot;, &quot;hic&quot;)),
  ) %&gt;% 
  write_csv(
    gzfile(all_out$csv)
  )
write_csv(de_genes_both_comps, all_out$de_genes)
goseq_group_sig %&gt;% 
  bind_rows() %&gt;% 
  write_csv(all_out$goseq)
cmn_res %&gt;% 
  bind_rows() %&gt;% 
  mutate(
    leadingEdge = vapply(leadingEdge, paste, character(1), collapse = &quot;; &quot;),
    gs_url = gs_url[gs_name]
  ) %&gt;% 
  write_csv(all_out$rna_enrichment)</code></pre>
<p>During this workflow, the following files were exported:</p>
<ul>
<li><strong>csv</strong>:
output/pairwise_comparisons/AR_ER/AR_E2_E2DHT-ER_E2_E2DHT-pairwise_comparison.csv.gz</li>
<li><strong>de_genes</strong>:
output/pairwise_comparisons/AR_ER/AR_E2_E2DHT-ER_E2_E2DHT-de_genes.csv</li>
<li><strong>goseq</strong>:
output/pairwise_comparisons/AR_ER/AR_E2_E2DHT-ER_E2_E2DHT-enrichment.csv</li>
<li><strong>results</strong>:
output/pairwise_comparisons/AR_ER/AR_E2_E2DHT-ER_E2_E2DHT-all_windows.rds</li>
<li><strong>rna_enrichment</strong>:
output/pairwise_comparisons/AR_ER/AR_E2_E2DHT-ER_E2_E2DHT-rnaseq_enrichment.csv</li>
<li><strong>renv</strong>:
output/envs/AR_E2_E2DHT-ER_E2_E2DHT-pairwise_comparison.RData</li>
</ul>
<!-- end of list -->
<div id="references" class="section level2">
<h2>References</h2>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-SubramanianGsea2005" class="csl-entry">
Subramanian, Aravind, Pablo Tamayo, Vamsi K. Mootha, Sayan Mukherjee,
Benjamin L. Ebert, Michael A. Gillette, Amanda Paulovich, et al. 2005.
<span>“Gene Set Enrichment Analysis: A Knowledge-Based Approach for
Interpreting Genome-Wide Expression Profiles.”</span> <em>Proceedings of
the National Academy of Sciences</em> 102 (43): 15545–50. <a
href="https://doi.org/10.1073/pnas.0506580102">https://doi.org/10.1073/pnas.0506580102</a>.
</div>
<div id="ref-YoungGoseq2010" class="csl-entry">
Young, M. D., M. J. Wakefield, G. K. Smyth, and A. Oshlack. 2010.
<span>“<span class="nocase"><span>G</span>ene ontology analysis for
<span>R</span><span>N</span><span>A</span>-seq: accounting for selection
bias</span>.”</span> <em>Genome Biol</em> 11 (2): R14.
</div>
</div>
<p><br></p>
<button type="button" class="btn btn-default btn-sessioninfo" data-toggle="collapse" data-target="#sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="sessioninfo" class="collapse">
<p><strong>R version 4.2.3 (2023-03-15)</strong></p>
<p><strong>Platform:</strong> x86_64-conda-linux-gnu (64-bit)</p>
<p><strong>locale:</strong> <em>LC_CTYPE=en_AU.UTF-8</em>,
<em>LC_NUMERIC=C</em>, <em>LC_TIME=en_AU.UTF-8</em>,
<em>LC_COLLATE=en_AU.UTF-8</em>, <em>LC_MONETARY=en_AU.UTF-8</em>,
<em>LC_MESSAGES=en_AU.UTF-8</em>, <em>LC_PAPER=en_AU.UTF-8</em>,
<em>LC_NAME=C</em>, <em>LC_ADDRESS=C</em>, <em>LC_TELEPHONE=C</em>,
<em>LC_MEASUREMENT=en_AU.UTF-8</em> and <em>LC_IDENTIFICATION=C</em></p>
<p><strong>attached base packages:</strong> <em>parallel</em>,
<em>grid</em>, <em>stats4</em>, <em>stats</em>, <em>graphics</em>,
<em>grDevices</em>, <em>utils</em>, <em>datasets</em>, <em>methods</em>
and <em>base</em></p>
<p><strong>other attached packages:</strong> <em>metap(v.1.1)</em>,
<em>cowplot(v.1.1.1)</em>, <em>fgsea(v.1.24.0)</em>,
<em>tidygraph(v.1.2.3)</em>, <em>ggraph(v.2.1.0)</em>,
<em>extraChIPs(v.1.5.5)</em>, <em>BiocParallel(v.1.32.5)</em>,
<em>GenomicInteractions(v.1.32.0)</em>,
<em>InteractionSet(v.1.26.0)</em>,
<em>SummarizedExperiment(v.1.28.0)</em>, <em>Biobase(v.2.58.0)</em>,
<em>MatrixGenerics(v.1.10.0)</em>, <em>matrixStats(v.1.0.0)</em>,
<em>goseq(v.1.50.0)</em>, <em>geneLenDataBase(v.1.34.0)</em>,
<em>BiasedUrn(v.2.0.10)</em>, <em>htmltools(v.0.5.5)</em>,
<em>msigdbr(v.7.5.1)</em>, <em>ggside(v.0.2.2)</em>,
<em>rlang(v.1.1.1)</em>, <em>ggrepel(v.0.9.3)</em>,
<em>scales(v.1.2.1)</em>, <em>magrittr(v.2.0.3)</em>,
<em>UpSetR(v.1.4.0)</em>, <em>VennDiagram(v.1.7.3)</em>,
<em>futile.logger(v.1.4.3)</em>, <em>rtracklayer(v.1.58.0)</em>,
<em>plyranges(v.1.18.0)</em>, <em>GenomicRanges(v.1.50.0)</em>,
<em>GenomeInfoDb(v.1.34.9)</em>, <em>IRanges(v.2.32.0)</em>,
<em>S4Vectors(v.0.36.0)</em>, <em>BiocGenerics(v.0.44.0)</em>,
<em>reactable(v.0.4.4)</em>, <em>yaml(v.2.3.7)</em>,
<em>pander(v.0.6.5)</em>, <em>glue(v.1.6.2)</em>,
<em>lubridate(v.1.9.2)</em>, <em>forcats(v.1.0.0)</em>,
<em>stringr(v.1.5.0)</em>, <em>dplyr(v.1.1.2)</em>,
<em>purrr(v.1.0.1)</em>, <em>readr(v.2.1.4)</em>,
<em>tidyr(v.1.3.0)</em>, <em>tibble(v.3.2.1)</em>,
<em>ggplot2(v.3.4.2)</em> and <em>tidyverse(v.2.0.0)</em></p>
<p><strong>loaded via a namespace (and not attached):</strong>
<em>utf8(v.1.2.3)</em>, <em>tidyselect(v.1.2.0)</em>,
<em>RSQLite(v.2.3.1)</em>, <em>AnnotationDbi(v.1.60.0)</em>,
<em>htmlwidgets(v.1.6.2)</em>, <em>munsell(v.0.5.0)</em>,
<em>codetools(v.0.2-19)</em>, <em>interp(v.1.1-4)</em>,
<em>withr(v.2.5.0)</em>, <em>colorspace(v.2.1-0)</em>,
<em>filelock(v.1.0.2)</em>, <em>highr(v.0.10)</em>,
<em>knitr(v.1.43)</em>, <em>rstudioapi(v.0.14)</em>,
<em>Rdpack(v.2.4)</em>, <em>labeling(v.0.4.2)</em>,
<em>GenomeInfoDbData(v.1.2.9)</em>, <em>polyclip(v.1.10-4)</em>,
<em>farver(v.2.1.1)</em>, <em>bit64(v.4.0.5)</em>,
<em>rprojroot(v.2.0.3)</em>, <em>vctrs(v.0.6.3)</em>,
<em>generics(v.0.1.3)</em>, <em>lambda.r(v.1.2.4)</em>,
<em>xfun(v.0.39)</em>, <em>csaw(v.1.32.0)</em>,
<em>biovizBase(v.1.46.0)</em>, <em>timechange(v.0.2.0)</em>,
<em>BiocFileCache(v.2.6.0)</em>, <em>R6(v.2.5.1)</em>,
<em>doParallel(v.1.0.17)</em>, <em>graphlayouts(v.1.0.0)</em>,
<em>clue(v.0.3-64)</em>, <em>locfit(v.1.5-9.8)</em>,
<em>AnnotationFilter(v.1.22.0)</em>, <em>bitops(v.1.0-7)</em>,
<em>cachem(v.1.0.8)</em>, <em>DelayedArray(v.0.24.0)</em>,
<em>vroom(v.1.6.3)</em>, <em>BiocIO(v.1.8.0)</em>,
<em>nnet(v.7.3-19)</em>, <em>gtable(v.0.3.3)</em>,
<em>ensembldb(v.2.22.0)</em>, <em>GlobalOptions(v.0.1.2)</em>,
<em>splines(v.4.2.3)</em>, <em>lazyeval(v.0.2.2)</em>,
<em>dichromat(v.2.0-0.1)</em>, <em>broom(v.1.0.5)</em>,
<em>checkmate(v.2.2.0)</em>, <em>crosstalk(v.1.2.0)</em>,
<em>GenomicFeatures(v.1.50.2)</em>, <em>backports(v.1.4.1)</em>,
<em>Hmisc(v.5.1-0)</em>, <em>EnrichedHeatmap(v.1.27.2)</em>,
<em>tools(v.4.2.3)</em>, <em>ellipsis(v.0.3.2)</em>,
<em>jquerylib(v.0.1.4)</em>, <em>RColorBrewer(v.1.1-3)</em>,
<em>Rcpp(v.1.0.10)</em>, <em>plyr(v.1.8.8)</em>,
<em>base64enc(v.0.1-3)</em>, <em>progress(v.1.2.2)</em>,
<em>zlibbioc(v.1.44.0)</em>, <em>RCurl(v.1.98-1.12)</em>,
<em>prettyunits(v.1.1.1)</em>, <em>rpart(v.4.1.19)</em>,
<em>deldir(v.1.0-9)</em>, <em>viridis(v.0.6.3)</em>,
<em>GetoptLong(v.1.0.5)</em>, <em>reactR(v.0.4.4)</em>,
<em>cluster(v.2.1.4)</em>, <em>here(v.1.0.1)</em>,
<em>ComplexUpset(v.1.3.3)</em>, <em>data.table(v.1.14.8)</em>,
<em>futile.options(v.1.0.1)</em>, <em>circlize(v.0.4.15)</em>,
<em>ProtGenerics(v.1.30.0)</em>, <em>hms(v.1.1.3)</em>,
<em>patchwork(v.1.1.2)</em>, <em>evaluate(v.0.21)</em>,
<em>XML(v.3.99-0.14)</em>, <em>jpeg(v.0.1-10)</em>,
<em>shape(v.1.4.6)</em>, <em>gridExtra(v.2.3)</em>,
<em>compiler(v.4.2.3)</em>, <em>biomaRt(v.2.54.0)</em>,
<em>crayon(v.1.5.2)</em>, <em>mgcv(v.1.8-42)</em>,
<em>tzdb(v.0.4.0)</em>, <em>Formula(v.1.2-5)</em>,
<em>DBI(v.1.1.3)</em>, <em>tweenr(v.2.0.2)</em>,
<em>formatR(v.1.14)</em>, <em>dbplyr(v.2.3.2)</em>,
<em>ComplexHeatmap(v.2.14.0)</em>, <em>MASS(v.7.3-60)</em>,
<em>rappdirs(v.0.3.3)</em>, <em>babelgene(v.22.9)</em>,
<em>Matrix(v.1.5-4.1)</em>, <em>cli(v.3.6.1)</em>,
<em>rbibutils(v.2.2.13)</em>, <em>metapod(v.1.6.0)</em>,
<em>Gviz(v.1.42.0)</em>, <em>igraph(v.1.4.3)</em>,
<em>pkgconfig(v.2.0.3)</em>, <em>GenomicAlignments(v.1.34.0)</em>,
<em>foreign(v.0.8-84)</em>, <em>xml2(v.1.3.4)</em>,
<em>foreach(v.1.5.2)</em>, <em>bslib(v.0.5.0)</em>,
<em>XVector(v.0.38.0)</em>, <em>VariantAnnotation(v.1.44.0)</em>,
<em>digest(v.0.6.31)</em>, <em>Biostrings(v.2.66.0)</em>,
<em>fastmatch(v.1.1-3)</em>, <em>rmarkdown(v.2.22)</em>,
<em>htmlTable(v.2.4.1)</em>, <em>edgeR(v.3.40.0)</em>,
<em>restfulr(v.0.0.15)</em>, <em>curl(v.5.0.1)</em>,
<em>Rsamtools(v.2.14.0)</em>, <em>rjson(v.0.2.21)</em>,
<em>lifecycle(v.1.0.3)</em>, <em>nlme(v.3.1-162)</em>,
<em>jsonlite(v.1.8.7)</em>, <em>viridisLite(v.0.4.2)</em>,
<em>limma(v.3.54.0)</em>, <em>BSgenome(v.1.66.3)</em>,
<em>fansi(v.1.0.4)</em>, <em>pillar(v.1.9.0)</em>,
<em>lattice(v.0.21-8)</em>, <em>KEGGREST(v.1.38.0)</em>,
<em>fastmap(v.1.1.1)</em>, <em>httr(v.1.4.6)</em>,
<em>GO.db(v.3.16.0)</em>, <em>png(v.0.1-8)</em>,
<em>iterators(v.1.0.14)</em>, <em>bit(v.4.0.5)</em>,
<em>ggforce(v.0.4.1)</em>, <em>stringi(v.1.7.12)</em>,
<em>sass(v.0.4.6)</em>, <em>blob(v.1.2.4)</em>,
<em>latticeExtra(v.0.6-30)</em> and <em>memoise(v.2.0.1)</em></p>
</div>
</div>
</div>

<div class="footer">
  Workflow developed by:<br>
  Dr Stephen Pederson<br>
  Dame Roma Mitchell Cancer Research Laboratories<br>
  Adelaide Medical School, University of Adelaide, SA, Australia
</div>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
