<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>ER Differential Binding: E2DHT Vs. E2</title>

<script src="site_libs/header-attrs-2.22/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/sandstone.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/core-js-2.5.3/shim.min.js"></script>
<script src="site_libs/react-17.0.0/react.min.js"></script>
<script src="site_libs/react-17.0.0/react-dom.min.js"></script>
<script src="site_libs/reactwidget-1.0.0/react-tools.js"></script>
<script src="site_libs/htmlwidgets-1.6.2/htmlwidgets.js"></script>
<link href="site_libs/reactable-0.4.4/reactable.css" rel="stylesheet" />
<script src="site_libs/reactable-binding-0.4.4/reactable.js"></script>
<link href="site_libs/font-awesome-6.4.0/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.0/css/v4-shims.min.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>





<style type="text/css">
/* for pandoc --citeproc since 2.11 */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="gravi.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">GRAVI</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="annotation_description.html">Annotations</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    MACS2 Peak Calling
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="AR_macs2_summary.html">AR</a>
    </li>
    <li>
      <a href="ER_macs2_summary.html">ER</a>
    </li>
    <li>
      <a href="H3K27ac_macs2_summary.html">H3K27ac</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Differential Binding
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">AR</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="AR_E2_E2DHT_differential_binding.html">E2DHT Vs. E2</a>
        </li>
      </ul>
    </li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">ER</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="ER_E2_E2DHT_differential_binding.html">E2DHT Vs. E2</a>
        </li>
      </ul>
    </li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">H3K27ac</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="H3K27ac_E2_E2DHT_differential_binding.html">E2DHT Vs. E2</a>
        </li>
      </ul>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Pairwise Comparisons
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">AR-ER</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="AR_E2_E2DHT_ER_E2_E2DHT_pairwise_comparison.html">E2DHT Vs. E2 / E2DHT Vs. E2</a>
        </li>
      </ul>
    </li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">AR-H3K27ac</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="AR_E2_E2DHT_H3K27ac_E2_E2DHT_pairwise_comparison.html">E2DHT Vs. E2 / E2DHT Vs. E2</a>
        </li>
      </ul>
    </li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">ER-H3K27ac</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="ER_E2_E2DHT_H3K27ac_E2_E2DHT_pairwise_comparison.html">E2DHT Vs. E2 / E2DHT Vs. E2</a>
        </li>
      </ul>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/steveped/GRAVI">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">ER Differential Binding: E2DHT Vs. E2</h1>
<h4 class="date">27 June, 2023</h4>

</div>


<pre class="r"><code>target &lt;- &quot;ER&quot;
treat_levels &lt;- c(&quot;E2&quot;, &quot;E2DHT&quot;)
threads &lt;- 1</code></pre>
<pre class="r"><code>library(tidyverse)
library(glue)
library(pander)
library(BiocParallel)
library(yaml)
library(plyranges)
library(rtracklayer)
library(GenomeInfoDb)
library(Rsamtools)
library(cowplot)
library(ngsReports)
library(csaw)
library(edgeR)
library(scales)
library(magrittr)
library(statmod)
library(IHW)
library(ggrepel)
library(rlang)
library(ggside)
library(InteractionSet)
library(GenomicInteractions)
library(reactable)
library(htmltools)
library(msigdbr)
library(goseq)
library(extraChIPs)
library(tidygraph)
library(ggraph)
library(metap)</code></pre>
<pre class="r"><code>panderOptions(&quot;big.mark&quot;, &quot;,&quot;)
panderOptions(&quot;missing&quot;, &quot;&quot;)
panderOptions(&quot;table.split.table&quot;, Inf)
theme_set(
  theme_bw() +
  theme(
    plot.title = element_text(hjust = 0.5)
  )
)
register(MulticoreParam(workers = threads))
source(here::here(&quot;workflow&quot;, &quot;scripts&quot;, &quot;custom_functions.R&quot;))</code></pre>
<pre class="r"><code>params &lt;- read_yaml(here::here(&quot;config&quot;, &quot;params.yml&quot;))
config &lt;- read_yaml(here::here(&quot;config&quot;, &quot;config.yml&quot;))
macs2_path &lt;- here::here(&quot;output&quot;, &quot;macs2&quot;, target)
samples &lt;- file.path(macs2_path, &quot;qc_samples.tsv&quot;) %&gt;% 
  read_tsv() %&gt;% 
  dplyr::filter(
    qc == &quot;pass&quot;,
    treat %in% treat_levels
  ) %&gt;% 
  mutate(treat = factor(treat, levels = treat_levels)) %&gt;% 
  droplevels()
stopifnot(nrow(samples) &gt; 0)
rep_col &lt;- setdiff(
  colnames(samples), c(&quot;sample&quot;, &quot;treat&quot;, &quot;target&quot;, &quot;input&quot;, &quot;label&quot;, &quot;qc&quot;)
)
samples[[rep_col]] &lt;- as.factor(samples[[rep_col]])
fdr_alpha &lt;- config$comparisons$fdr
db_method &lt;- match.arg(config$comparisons$method, c(&quot;sq-lt&quot;, &quot;ls-ql&quot;))
ihw_opts &lt;- c(&quot;regions&quot;, &quot;features&quot;, &quot;targets&quot;)
ihw_method &lt;- c(
  intersect(str_to_lower(config$comparisons$ihw), ihw_opts), &quot;none&quot;
)
ihw_method &lt;- ihw_method[[1]]</code></pre>
<pre class="r"><code>bam_path &lt;- here::here(config$paths$bam)
stopifnot(dir.exists(bam_path))
annotation_path &lt;- here::here(&quot;output&quot;, &quot;annotations&quot;)
stopifnot(dir.exists(annotation_path))
out_path &lt;- here::here(&quot;output&quot;, &quot;differential_binding&quot;, target)
if (!dir.exists(out_path)) dir.create(out_path)</code></pre>
<pre class="r"><code>sq &lt;- file.path(annotation_path, &quot;seqinfo.rds&quot;) %&gt;% 
  read_rds()
blacklist &lt;- here::here(annotation_path, &quot;blacklist.bed.gz&quot;) %&gt;%
  import.bed(seqinfo = sq) %&gt;%
  sort()

cb &lt;- config$genome$build %&gt;% 
  str_to_lower() %&gt;% 
  paste0(&quot;.cytobands&quot;) 
data(list = cb)
bands_df &lt;- get(cb)
stopifnot(
  colnames(bands_df) == c(&quot;chrom&quot;, &quot;chromStart&quot;, &quot;chromEnd&quot;, &quot;name&quot;, &quot;gieStain&quot;)
)

all_gr &lt;- file.path(annotation_path, &quot;all_gr.rds&quot;) %&gt;% 
  read_rds() 
genes_gr &lt;- all_gr$gene
id2gene &lt;- structure(genes_gr$gene_name, names = genes_gr$gene_id)
trans_models &lt;- file.path(annotation_path, &quot;trans_models.rds&quot;) %&gt;% 
  read_rds() 

external_features &lt;- GRanges(seqinfo = sq)
if (!is.null(config$external$features)) {
  external_features &lt;- suppressWarnings(
    import.gff(here::here(config$external$features), genome = sq)
  )
  keep_cols &lt;- !vapply(
    mcols(external_features), function(x) all(is.na(x)), logical(1)
  )
  mcols(external_features) &lt;- mcols(external_features)[keep_cols]
}
has_features &lt;- length(external_features) &gt; 0
gene_regions &lt;- file.path(annotation_path, &quot;gene_regions.rds&quot;) %&gt;% 
  read_rds()
regions &lt;- vapply(gene_regions, function(x) unique(x$region), character(1))

rna_path &lt;- here::here(config$external$rnaseq)
rnaseq &lt;- tibble(gene_id = character())
if (length(rna_path) &gt; 0) {
  stopifnot(file.exists(rna_path))
  if (str_detect(rna_path, &quot;tsv$&quot;)) rnaseq &lt;- read_tsv(rna_path)
  if (str_detect(rna_path, &quot;csv$&quot;)) rnaseq &lt;- read_csv(rna_path)
  if (!&quot;gene_id&quot; %in% colnames(rnaseq)) stop(&quot;Supplied RNA-Seq data must contain the column &#39;gene_id&#39;&quot;)
  genes_gr &lt;- subset(genes_gr, gene_id %in% rnaseq$gene_id)
  rna_lfc_col &lt;- colnames(rnaseq)[str_detect(str_to_lower(colnames(rnaseq)), &quot;logfc&quot;)][1]
  rna_fdr_col &lt;- colnames(rnaseq)[str_detect(str_to_lower(colnames(rnaseq)), &quot;fdr|adjp&quot;)][1]
}
has_rnaseq &lt;- as.logical(nrow(rnaseq))
tx_col &lt;- intersect(c(&quot;tx_id&quot;, &quot;transcript_id&quot;), colnames(rnaseq))
rna_gr_col &lt;- ifelse(length(tx_col) &gt; 0, &quot;transcript_id&quot;, &quot;gene_id&quot;)
rna_col &lt;- c(tx_col, &quot;gene_id&quot;)[[1]]</code></pre>
<pre class="r"><code>colours &lt;- read_rds(here::here(&quot;output&quot;, &quot;annotations&quot;, &quot;colours.rds&quot;)) %&gt;% 
  lapply(unlist)
region_colours &lt;- setNames(colours$regions, regions[names(colours$regions)])
if (has_features)
  feature_colours &lt;- setNames(
    colours$features, str_sep_to_title(names(colours$features))
  )
direction_colours &lt;- colours$direction %&gt;% 
  setNames(str_to_title(names(.))) %&gt;% 
  .[names(.) %in% c(&quot;Up&quot;, &quot;Down&quot;, &quot;Unchanged&quot;, &quot;Ambiguous&quot;)]
treat_colours &lt;- colours$treat[treat_levels]
fig_dev &lt;- knitr::opts_chunk$get(&quot;dev&quot;)
fig_type &lt;- fig_dev[[1]]
if (is.null(fig_type)) stop(&quot;Couldn&#39;t detect figure type&quot;)</code></pre>
<pre class="r"><code>hic &lt;- GInteractions()
hic_path &lt;- here::here(config$external$hic)
if (length(hic_path) &gt; 0)
  if (file.exists(hic_path)) {
    has_hic &lt;- TRUE
    hic &lt;- makeGenomicInteractionsFromFile(hic_path, type = &quot;bedpe&quot;)
    reg_combs &lt;- expand.grid(regions, regions) %&gt;% 
      as.matrix() %&gt;% 
      apply(
        MARGIN = 1, 
        function(x) {
          x &lt;- sort(factor(x, levels = regions))
          paste(as.character(x), collapse = &quot; - &quot;)
        }
      ) %&gt;% 
      unique()
    hic$regions &lt;- anchors(hic) %&gt;% 
      vapply(
        bestOverlap,
        y = GRangesList(lapply(gene_regions, granges)),
        character(length(hic))
      ) %&gt;% 
      apply(MARGIN = 2, function(x) regions[x]) %&gt;% 
      apply(
        MARGIN = 1, 
        function(x) {
          x &lt;- sort(factor(x, levels = regions))
          paste(as.character(x), collapse = &quot; - &quot;)
        }
      ) %&gt;% 
      factor(levels = reg_combs) %&gt;%
      fct_relabel(
        str_replace_all,
        pattern = &quot;Promoter \\([0-9kbp/\\+-]+\\)&quot;, replacement = &quot;Promoter&quot;
      )
    if (has_features) {
      feat_combs &lt;- expand.grid(names(feature_colours), names(feature_colours)) %&gt;% 
        as.matrix() %&gt;% 
        apply(
          MARGIN = 1, 
          function(x) {
            x &lt;- sort(factor(x, levels = names(feature_colours)))
            paste(as.character(x), collapse = &quot; - &quot;)
          }
        ) %&gt;% 
        unique()
      hic$features &lt;- vapply(
        anchors(hic),
        function(x) bestOverlap(
          x, external_features, var = &quot;feature&quot;, missing = &quot;no_feature&quot;
        ),
        character(length(hic))
      )  %&gt;% 
        apply(
          MARGIN = 1, 
          function(x) {
            x &lt;- sort(factor(x, levels = names(feature_colours)))
            paste(as.character(x), collapse = &quot; - &quot;)
          }
        ) %&gt;% 
        factor(levels = feat_combs) %&gt;%
        fct_relabel(str_sep_to_title, pattern = &quot;_&quot;)
    }
  }
stopifnot(is(hic, &quot;GInteractions&quot;))
seqlevels(hic) &lt;- seqlevels(sq)
seqinfo(hic) &lt;- sq
hic &lt;- hic[!overlapsAny(hic, blacklist)]
has_hic &lt;- as.logical(length(hic))</code></pre>
<pre class="r"><code>facet_labeller &lt;- as_labeller(
  c(
    c(
      `TRUE` = &quot;Consensus Peak&quot;,
      `FALSE` = &quot;No Consensus Peak&quot;
    ),
    structure(treat_levels, names = treat_levels),
    c(
      Up = glue(&quot;Increased {target} Binding&quot;),
      Down = glue(&quot;Decreased {target} Binding&quot;)
    ),
    c(
      AveExpr = &quot;logCPM&quot;, logFC = &quot;logFC&quot;
    )
  )
)</code></pre>
<pre class="r"><code>all_out &lt;- list(
  results = file.path(
    out_path, 
    glue(&quot;{target}_{treat_levels[[1]]}_{treat_levels[[2]]}-differential_binding.rds&quot;)
  ),
  csv = file.path(
    out_path, 
    glue(&quot;{target}_{treat_levels[[1]]}_{treat_levels[[2]]}-differential_binding.csv.gz&quot;)
  ),
  windows  = file.path(
    out_path,
    glue(&quot;{target}_{treat_levels[[1]]}_{treat_levels[[2]]}-filtered_windows.rds&quot;)
  ),
  up_regions = file.path(
    out_path, glue(&quot;{target}_{treat_levels[[1]]}_{treat_levels[[2]]}-up.bed&quot;)
  ),
  down_regions = file.path(
    out_path, glue(&quot;{target}_{treat_levels[[1]]}_{treat_levels[[2]]}-down.bed&quot;)
  ),
  de_genes = file.path(
    out_path, glue(&quot;{target}_{treat_levels[1]}_{treat_levels[2]}-DE_genes.csv&quot;)
  ),
  enrichment = file.path(
    out_path, glue(&quot;{target}_{treat_levels[1]}_{treat_levels[2]}-enrichment.csv&quot;)
  ),
  rna_enrichment = file.path(
    out_path, 
    glue(&quot;{target}_{treat_levels[1]}_{treat_levels[2]}-rnaseq_enrichment.csv&quot;)
  ),
  renv = here::here(
    &quot;output/envs&quot;,
    glue(
      &quot;{target}_{treat_levels[[1]]}_{treat_levels[[2]]}-differential_binding.RData&quot;
    )
  )
)
## Initialise outputs from RNA-Seq
cmn_diff &lt;- cmn_up &lt;- cmn_down &lt;- tibble(leadingEdge = list())</code></pre>
<div id="outline" class="section level1">
<h1>Outline</h1>
<div id="differential-binding" class="section level2">
<h2>Differential Binding</h2>
<p>This step of the workflow uses a sliding window approach to
differential binding, as a logical extension to the methods suggested
and implemented in the <code>csaw</code> package <span
class="citation">(<a href="#ref-LunSmythF1000" role="doc-biblioref">A.
T. Lun and Smyth 2015</a>)</span>.</p>
<ol style="list-style-type: decimal">
<li>Consensus Peaks derived from <code>macs2 callpeak</code> <span
class="citation">(<a href="#ref-Zhang18798982"
role="doc-biblioref">Zhang et al. 2008</a>)</span> results are used to
determine inclusion/exclusion values for sliding windows</li>
<li>Smooth Quantile Normalisation <span class="citation">(<a
href="#ref-HicksSQN2017" role="doc-biblioref">Stephanie C. Hicks et al.
2017</a>)</span> is used on the set of logCPM values obtained from
retained sliding windows. This was chosen given that many transcription
factors can show vastly different cytoplasmic/nuclear distributions
across treatments</li>
<li>The <em>limma-trend</em> method <span class="citation">(<a
href="#ref-LawVoom2014" role="doc-biblioref">Law et al. 2014</a>)</span>
is used alongside <code>treat</code> <span class="citation">(<a
href="#ref-McCarthyTreat2009" role="doc-biblioref">McCarthy and Smyth
2009</a>)</span> for detection of differential binding. This should
correctly control the FDR, and allows the specification as a percentage
of a suitable threshold for the estimated change in binding, below which
we are not interested in any changed signal</li>
<li>Independent Hypothesis Weighting (IHW) <span class="citation">(<a
href="#ref-IgnatiadisIHW2016" role="doc-biblioref">Ignatiadis et al.
2016</a>)</span> is additionally used to improve the power of the
results. Under this strategy, p-values are partitioned based on the
presence/absence of any other ChIP targets under consideration in any
condition, which is considered here to be a statistically independent
variable.</li>
</ol>
<p>The workflow also depends heavily on the function implemented in the
Bioconductor package <a
href="https://bioconductor.org/packages/release/bioc/html/extraChIPs.html">extraChIPs</a></p>
</div>
<div id="enrichment-analysis" class="section level2">
<h2>Enrichment Analysis</h2>
<p>Beyond the simple analysis of differential binding, peaks are mapped
to genes and enrichment testing is performed on the following:</p>
<ol style="list-style-type: decimal">
<li>Genes mapped to any window with detected ER are compared to all
genes not mapped to any window</li>
<li>Genes mapped to <em>all differentially bound windows</em> are
compared to genes mapped to windows which are not differentially
bound</li>
<li>Genes mapped to windows with <em>increasing</em> ER binding are
compared to genes mapped to windows which are not differentially
bound</li>
<li>Genes mapped to windows with <em>decreasing</em> ER binding are
compared to genes mapped to windows which are not differentially
bound</li>
</ol>
<p>Enrichment testing is performed using <code>goseq</code> <span
class="citation">(<a href="#ref-YoungGoseq2010"
role="doc-biblioref">Young et al. 2010</a>)</span> with <em>no term
accounting for sampling bias</em>, except when comparing genes mapped to
any window. For this case <strong>only</strong>, gene width is used to
capture any sampling bias and Wallenius’ Non-Central Hypergeometric
Distribution is used. As RNA-seq data was not provided, the genes
considered for enrichment analysis are the 62,400 annotated genes.</p>
</div>
</div>
<div id="data-preparation" class="section level1">
<h1>Data Preparation</h1>
<pre class="r"><code>consensus_peaks &lt;- here::here(macs2_path, &quot;consensus_peaks.bed&quot;) %&gt;%
    import.bed(seqinfo = sq)</code></pre>
<pre class="r"><code>bfl &lt;- bam_path %&gt;%
  file.path(target, glue(&quot;{samples$sample}.bam&quot;)) %&gt;%
  c(
    file.path(bam_path, &quot;Input&quot;, glue(&quot;{unique(samples$input)}.bam&quot;))
  ) %&gt;% 
  BamFileList() %&gt;%
  setNames(c(samples$sample, unique(samples$input)))</code></pre>
<pre class="r"><code>ys &lt;- 1000
bfl_summary &lt;- bfl %&gt;%
    bplapply(
        function(x) {
            yieldSize(x) &lt;- ys
            sbp &lt;- ScanBamParam(what = c(&quot;qwidth&quot;, &quot;mapq&quot;))
            open(x)
            vals &lt;- scanBam(x, param = sbp)[[1]]
            close(x)
            list(as_tibble(vals))
        }
    ) %&gt;% 
    as_tibble() %&gt;%
    pivot_longer(everything(), names_to = &quot;sample&quot;) %&gt;%
    unnest(value) %&gt;% 
    pivot_longer(
        cols = c(&quot;qwidth&quot;, &quot;mapq&quot;),
        names_to = &quot;stat&quot;,
        values_to = &quot;value&quot;
    ) %&gt;%
    left_join(samples, by = &quot;sample&quot;) %&gt;% 
    mutate(
        across(
            all_of(c(&quot;target&quot;, &quot;label&quot;, &quot;treat&quot;)),
            str_replace_na, replacement = &quot;Input&quot;
        ),
        treat = factor(treat, levels = c(&quot;Input&quot;, treat_levels)),
        label = fct_inorder(label)
    ) %&gt;% 
    split(f = .$stat)
anyDups &lt;- bplapply(
    bfl,
    function(x) {
        sbp &lt;- ScanBamParam(
            flag = scanBamFlag(isDuplicate = TRUE),
            which = GRanges(sq)[which.min(seqlengths(sq))],
            what = &quot;qname&quot;
            )
        length(scanBam(x, param = sbp)[[1]]$qname)  &gt; 0
    }
) %&gt;%
  unlist()
anyPE &lt;- bplapply(
    bfl,
    function(x){
        yieldSize(x) &lt;- ys
        open(x)
        flag &lt;- scanBam(x, param=ScanBamParam(what=&quot;flag&quot;))[[1]]$flag
        close(x)
        any(bamFlagTest(flag, &quot;isPaired&quot;))
    }
) %&gt;% 
  unlist()</code></pre>
<p>Taking the first 1,000 alignments, a brief inspection of the Bam
Files revealed:</p>
<ul>
<li>Aligned read lengths ranged between 50 and 76. The median length was
75.</li>
<li>MAPQ scores of aligned reads ranged between 2 and 42. The median
score was 30.</li>
<li>All bam files have been de-duplicated</li>
<li>All alignments were single-end</li>
</ul>
<div id="sliding-window-counts" class="section level2">
<h2>Sliding Window Counts</h2>
<pre class="r"><code>greylist &lt;- file.path(
  annotation_path, glue(&quot;{unique(samples$input)}_greylist.bed&quot;)
  ) %&gt;%
  lapply(import.bed, seqinfo = sq) %&gt;%
  GRangesList() %&gt;%
  unlist() %&gt;%
  reduce()</code></pre>
<p>Prior to this workflow, high-signal regions were detected in any
input samples associated with ER libraries and grey-lists formed. For
these samples, this constituted 3,985 regions with total width of
82,913kb. Regions assigned to the greylist were added to the blacklisted
regions and excluded from all analyses.</p>
<pre class="r"><code>macs2_merged_logs &lt;- file.path(
  macs2_path, glue(&quot;{treat_levels}_merged_callpeak.log&quot;)
    ) %&gt;%
    importNgsLogs()
fl &lt;- max(macs2_merged_logs$fragment_length)
rp &lt;- readParam(
  pe = ifelse(any(anyPE), &quot;both&quot;, &quot;none&quot;),
  dedup = any(anyDups),
  restrict = seqnames(sq),
  discard = c(granges(blacklist), greylist)
)
win_step &lt;- 10*(1 + fl %/% 60)
win_size &lt;- 3*win_step
window_counts &lt;- windowCounts(
  bam.files = bfl,
  spacing = win_step,
  width = win_size,
  ext = fl,
  filter = length(bfl) - 1,
  param = rp,
  BPPARAM = bpparam()
)
colData(window_counts) &lt;- colData(window_counts) %&gt;% 
  as_tibble(rownames = &quot;sample&quot;) %&gt;% 
  dplyr::select(all_of(c(&quot;sample&quot;, &quot;bam.files&quot;, &quot;totals&quot;, &quot;ext&quot;, &quot;rlen&quot;))) %&gt;% 
  left_join(samples, by = &quot;sample&quot;) %&gt;% 
  mutate(
    treat = fct_explicit_na(treat, &quot;Input&quot;), 
    target = str_replace_na(target, &quot;Input&quot;)
  ) %&gt;% 
  as.data.frame() %&gt;% 
  column_to_rownames(&quot;sample&quot;) %&gt;% 
  DataFrame()
window_counts &lt;- sortSeqlevels(window_counts)
seqinfo(window_counts) &lt;- sq</code></pre>
<p>Using the macs2-estimated fragment length of 173nt, a set of genomic
sliding windows were defined using a window size of 90bp, sliding in
increments of 30bp. With the exclusion of black-listed and grey-listed
regions, all alignments within each window were counted for each
ER-associated sample, and all relevant input samples. Any windows with
fewer than 6 alignments (i.e.1 read/sample) across all samples were
discarded, leaving a total of 79,357,674 sliding windows, covering 80%
of the reference genome.</p>
<p>These windows were then filtered using the <code>dualFilter()</code>
function from <code>extraChIPs</code>, discarding windows with low
counts. Under this approach, two thresholds are determined above which
windows are retained, and with values chosen to return 60% of sliding
windows which overlap a macs2-derived consensus peak. These values are
set to filter based on 1) signal relative to input over an extended
range and, 2) overall signal level. Both filtering strategies rely on
the infrastructure provided by <code>csaw</code> <span
class="citation">(<a href="#ref-LunSmythCsaw2014"
role="doc-biblioref">A. T. L. Lun and Smyth 2014</a>)</span>.</p>
<pre class="r"><code>filtered_counts &lt;- dualFilter(
  x = window_counts[, samples$sample],
  bg = window_counts[, samples$input],
  ref = consensus_peaks,
  keep.totals = TRUE,
  q = config$comparisons$filter_q
)
colData(filtered_counts) &lt;- droplevels(colData(filtered_counts))
n_max &lt;- min(5e4, nrow(filtered_counts))
sample_labeller &lt;- as_labeller(
  setNames(colData(filtered_counts)$label, colnames(filtered_counts))
)</code></pre>
<pre class="r"><code>list(
  `Genomic Windows` = window_counts,
  `Retained Windows` = filtered_counts,
  `Consensus Peaks` = consensus_peaks
) %&gt;% 
  lapply(granges) %&gt;% 
  lapply(
    function(x) {
      tibble(
        N = comma(length(x)),
        `Total Width (kb)` = comma(sum(width(reduce(x))) / 1e3),
        `Median Width (bp)` = median(width(x))
      )
    }
  ) %&gt;% 
  lapply(list) %&gt;% 
  as_tibble() %&gt;% 
  pivot_longer(everything(), names_to = &quot;Dataset&quot;) %&gt;% 
  unnest(everything()) %&gt;% 
  left_join(
    tibble(
      Dataset = c(&quot;Retained Windows&quot;, &quot;Consensus Peaks&quot;),
      `Unique (kb)` = c(
        round(sum(width(setdiff(granges(filtered_counts), consensus_peaks))) / 1e3, 1),
        round(sum(width(setdiff(consensus_peaks, granges(filtered_counts)))) / 1e3, 1)
      ),
      `% Unique` = percent(
        1e3*`Unique (kb)` / c(
          sum(width(reduce(granges(filtered_counts)))),
          sum(width(consensus_peaks))
        ),
        0.1
      )
    )
  ) %&gt;% 
  pander(
    justify = &quot;lrrrrr&quot;,
    caption = paste(
      &quot;A dual filtering strategy was used based on retaining&quot;, 
      percent(config$comparisons$filter_q, 0.1), 
      &quot;of genomic windows overlapping the consensus peaks identified by &quot;, 
      &quot;`macs2 callpeak` on merged samples. This approach combined both &quot;, 
      &quot;1) expression percentiles, and 2) signal strength in relation to the input sample.&quot;,
      &quot;The complete set of sliding windows covered the majority of the genome,&quot;,
      &quot;whilst those retained after filtering were focussed on strong binding&quot;,
      &quot;signal. Consensus peaks were as identified by `macs2 callpeak` in a&quot;, 
      &quot;previous step of the workflow.&quot;,
      &quot;Importantly, consensus peaks are non-overlapping, whilst the other two&quot;,
      &quot;datasets are derived from overlapping sliding windows.&quot;,
      &quot;This strategy of filtering the set of initial sliding windows retained&quot;,
      percent(
        1 - sum(width(setdiff(consensus_peaks, rowRanges(filtered_counts)))) / sum(width(consensus_peaks))
      ), 
      &quot;of the genomic regions covered by consensus peaks, with&quot;, 
      percent(
        sum(width(setdiff(rowRanges(filtered_counts), consensus_peaks))) / sum(width(reduce(granges(filtered_counts)))))
      , 
      &quot;of the retained windows being outside genomic regions covered by consensus peaks.&quot;,
      ifelse(
        n_max == 5e4, 
        &quot;For subsequent density and RLE plots, a subsample of 50,000 genomic regions will be used for speed.&quot;, 
        &quot;&quot;
      )
    )
  )</code></pre>
<table>
<caption>A dual filtering strategy was used based on retaining 60.0% of
genomic windows overlapping the consensus peaks identified by
<code>macs2 callpeak</code> on merged samples. This approach combined
both 1) expression percentiles, and 2) signal strength in relation to
the input sample. The complete set of sliding windows covered the
majority of the genome, whilst those retained after filtering were
focussed on strong binding signal. Consensus peaks were as identified by
<code>macs2 callpeak</code> in a previous step of the workflow.
Importantly, consensus peaks are non-overlapping, whilst the other two
datasets are derived from overlapping sliding windows. This strategy of
filtering the set of initial sliding windows retained 81% of the genomic
regions covered by consensus peaks, with 16% of the retained windows
being outside genomic regions covered by consensus peaks. For subsequent
density and RLE plots, a subsample of 50,000 genomic regions will be
used for speed.</caption>
<colgroup>
<col width="19%" />
<col width="13%" />
<col width="19%" />
<col width="20%" />
<col width="14%" />
<col width="11%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Dataset</th>
<th align="right">N</th>
<th align="right">Total Width (kb)</th>
<th align="right">Median Width (bp)</th>
<th align="right">Unique (kb)</th>
<th align="right">% Unique</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Genomic Windows</td>
<td align="right">79,357,674</td>
<td align="right">2,489,591</td>
<td align="right">90</td>
<td align="right"></td>
<td align="right"></td>
</tr>
<tr class="even">
<td align="left">Retained Windows</td>
<td align="right">69,935</td>
<td align="right">2,556</td>
<td align="right">90</td>
<td align="right">416.7</td>
<td align="right">16.3%</td>
</tr>
<tr class="odd">
<td align="left">Consensus Peaks</td>
<td align="right">7,532</td>
<td align="right">2,632</td>
<td align="right">304</td>
<td align="right">493</td>
<td align="right">18.7%</td>
</tr>
</tbody>
</table>
</div>
<div id="normalisation" class="section level2">
<h2>Normalisation</h2>
<pre class="r"><code>library(quantro)
library(qsmooth)</code></pre>
<p>The <code>quantro</code> test was first applied <span
class="citation">(<a href="#ref-HicksQuantro2015"
role="doc-biblioref">S. C. Hicks and Irizarry 2015</a>)</span> to
determine if treatment-specific binding distributions were found in the
data. Whilst this may not always be the case, the robustness of
smooth-quantile normalisation (SQN) <span class="citation">(<a
href="#ref-HicksSQN2017" role="doc-biblioref">Stephanie C. Hicks et al.
2017</a>)</span> will be applicable if data is drawn from different or
near-identical distributions, and this method was applied grouping data
by treatment.</p>
<pre class="r"><code>qtest &lt;- quantro(
  assay(filtered_counts, &quot;logCPM&quot;),
  groupFactor = filtered_counts$treat
)
pander(
  anova(qtest), 
  caption = paste(
    &quot;*Results from qtest suggesting that the two treatment groups are drawn from&quot;,
    ifelse(
      qtest@anova$`Pr(&gt;F)`[[1]] &lt; 0.05,
      &quot;different distributions.*&quot;,
      &quot;the same distribution.*&quot;
    )
  )
)</code></pre>
<table style="width:96%;">
<caption><em>Results from qtest suggesting that the two treatment groups
are drawn from the same distribution.</em></caption>
<colgroup>
<col width="25%" />
<col width="6%" />
<col width="16%" />
<col width="16%" />
<col width="15%" />
<col width="15%" />
</colgroup>
<thead>
<tr class="header">
<th align="center"> </th>
<th align="center">Df</th>
<th align="center">Sum Sq</th>
<th align="center">Mean Sq</th>
<th align="center">F value</th>
<th align="center">Pr(&gt;F)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center"><strong>groupFactor</strong></td>
<td align="center">1</td>
<td align="center">0.0001461</td>
<td align="center">0.0001461</td>
<td align="center">0.001558</td>
<td align="center">0.9704</td>
</tr>
<tr class="even">
<td align="center"><strong>Residuals</strong></td>
<td align="center">4</td>
<td align="center">0.3749</td>
<td align="center">0.09374</td>
<td align="center"></td>
<td align="center"></td>
</tr>
</tbody>
</table>
<pre class="r"><code>qs &lt;-qsmooth(
  assay(filtered_counts, &quot;logCPM&quot;), group_factor = filtered_counts$treat
)
assay(filtered_counts, &quot;qsmooth&quot;) &lt;- qsmoothData(qs)</code></pre>
</div>
<div id="data-inspection" class="section level2 tabset">
<h2 class="tabset">Data Inspection</h2>
<div id="qsmooth-weights" class="section level3">
<h3>QSmooth Weights</h3>
<pre class="r"><code>qsmoothPlotWeights(qs, xLab = &quot;Quantiles&quot;, yLab = &quot;Weights&quot;, mainLab = &quot;QSmooth Weights&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="ER_E2_E2DHT_differential_binding_files/figure-html/plot-qs-weights-1.png" alt="*Quantile-specific weights used by the Smooth-Quantile normalisation. Low weights indicate signal quantiles which appear to be more specific within a group, whilst higher weights indicate similarity between groups.*" width="960" />
<p class="caption">
<em>Quantile-specific weights used by the Smooth-Quantile normalisation.
Low weights indicate signal quantiles which appear to be more specific
within a group, whilst higher weights indicate similarity between
groups.</em>
</p>
</div>
</div>
<div id="density-plot-all-sliding-windows" class="section level3">
<h3>Density Plot: All Sliding Windows</h3>
<pre class="r"><code>plotAssayDensities(
  window_counts, trans = &quot;log1p&quot;, n_max = n_max, colour = &quot;treat&quot;,
  linetype = &quot;target&quot;
) +
  scale_colour_manual(values = colours$treat) + 
  labs(
    x = &quot;log(counts + 1)&quot;, y = &quot;Density&quot;, colour = &quot;Treat&quot;, linetype = &quot;Target&quot;
  )</code></pre>
<div class="figure" style="text-align: center">
<img src="ER_E2_E2DHT_differential_binding_files/figure-html/plot-density-windows-1.png" alt="*Density plot for __all windows prior to the selection of windows__ more likely to contain true signal. Retained windows will be those at the upper end, whilst discarded windows will be at the lower end.*" width="768" />
<p class="caption">
<em>Density plot for <strong>all windows prior to the selection of
windows</strong> more likely to contain true signal. Retained windows
will be those at the upper end, whilst discarded windows will be at the
lower end.</em>
</p>
</div>
</div>
<div id="logcpm-densities" class="section level3">
<h3>logCPM Densities</h3>
<pre class="r"><code>a &lt;- filtered_counts %&gt;% 
  plotAssayDensities(&quot;logCPM&quot;, colour = &quot;treat&quot;, n_max = n_max) +
  scale_colour_manual(values = treat_colours) +
  labs(colour = &quot;Treat&quot;)
b &lt;- filtered_counts %&gt;% 
  plotAssayDensities(&quot;qsmooth&quot;, colour = &quot;treat&quot;, n_max = n_max) +
  scale_colour_manual(values = treat_colours) +
  labs(
    x = &quot;Normalised logCPM&quot;, colour = &quot;Treat&quot;
  )
plot_grid(
  a + theme(legend.position = &quot;none&quot;), 
  b + theme(legend.position = &quot;none&quot;),
  get_legend(b),
  nrow = 1,
  rel_widths = c(5, 5, 1),
  labels = c(&quot;A&quot;, &quot;B&quot;)
)</code></pre>
<div class="figure" style="text-align: center">
<img src="ER_E2_E2DHT_differential_binding_files/figure-html/plot-logcpm-densities-1.png" alt="*logCPM distributions for all retained windows A) prior to smooth quantile normalisation, and 2) after SQ normalisation.*" width="960" />
<p class="caption">
<em>logCPM distributions for all retained windows A) prior to smooth
quantile normalisation, and 2) after SQ normalisation.</em>
</p>
</div>
</div>
<div id="rle-pre-normalisation" class="section level3">
<h3>RLE (Pre-Normalisation)</h3>
<pre class="r"><code>filtered_counts %&gt;% 
  plotAssayRle(
  &quot;logCPM&quot;, fill = &quot;treat&quot;, rle_group = &quot;treat&quot;, n_max = n_max
) +
  geom_hline(yintercept = 0) + 
  facet_grid(.~treat, scales = &quot;free_x&quot;, space = &quot;free_x&quot;) +
  scale_x_discrete(labels = sample_labeller) +
  scale_fill_manual(values = treat_colours) +
  labs(x = &quot;Sample&quot;, fill = &quot;Treat&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="ER_E2_E2DHT_differential_binding_files/figure-html/plot-rle-1.png" alt="*RLE plot showing logCPM values. RLE values were calculated within each treatment group to account for the potentially different binding dynamics of ER.*" width="960" />
<p class="caption">
<em>RLE plot showing logCPM values. RLE values were calculated within
each treatment group to account for the potentially different binding
dynamics of ER.</em>
</p>
</div>
</div>
<div id="rle-post-normalisation" class="section level3">
<h3>RLE (Post-Normalisation)</h3>
<pre class="r"><code>filtered_counts %&gt;% 
  plotAssayRle(
  &quot;qsmooth&quot;, fill = &quot;treat&quot;, rle_group = &quot;treat&quot;, n_max = n_max
) +
  geom_hline(yintercept = 0) + 
  facet_grid(.~treat, scales = &quot;free_x&quot;, space = &quot;free_x&quot;) +
  scale_x_discrete(labels = sample_labeller) +
  scale_fill_manual(values = treat_colours) +
  labs(x = &quot;Sample&quot;, fill = &quot;Treat&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="ER_E2_E2DHT_differential_binding_files/figure-html/plot-rle-normalised-1.png" alt="*RLE plot showing normalised logCPM. RLE values were calculated within each treatment group to account for the potentially different binding dynamics of ER.*" width="960" />
<p class="caption">
<em>RLE plot showing normalised logCPM. RLE values were calculated
within each treatment group to account for the potentially different
binding dynamics of ER.</em>
</p>
</div>
</div>
<div id="pca" class="section level3">
<h3>PCA</h3>
<pre class="r"><code>a &lt;- plotAssayPCA(filtered_counts, &quot;logCPM&quot;, colour = &quot;treat&quot;, label = &quot;label&quot;) +
  scale_colour_manual(values = treat_colours) +
  ggtitle(&quot;Pre-Normalisation&quot;)
b &lt;- plotAssayPCA(filtered_counts, &quot;qsmooth&quot;, colour = &quot;treat&quot;, label = &quot;label&quot;) +
  scale_colour_manual(values = treat_colours) +
  ggtitle(&quot;Post-Normalisation&quot;)
plot_grid(
  a + theme(legend.position = &quot;none&quot;), b + theme(legend.position = &quot;none&quot;), 
  get_legend(b + labs(colour = &quot;Treat&quot;)), rel_widths = c(1, 1, 0.2), 
  nrow = 1, labels = c(&quot;A&quot;, &quot;B&quot;)
  )</code></pre>
<div class="figure" style="text-align: center">
<img src="ER_E2_E2DHT_differential_binding_files/figure-html/plot-pca-1.png" alt="*PCA plots for logCPM values A) before and B) after Smooth Quantile normalisation*" width="960" />
<p class="caption">
<em>PCA plots for logCPM values A) before and B) after Smooth Quantile
normalisation</em>
</p>
</div>
</div>
</div>
</div>
<div id="differential-binding-analysis" class="section level1">
<h1>Differential Binding Analysis</h1>
<div id="primary-analysis" class="section level2 tabset">
<h2 class="tabset">Primary Analysis</h2>
<pre class="r"><code>X &lt;- model.matrix(~treat, data = colData(filtered_counts)) %&gt;%
  set_colnames(str_remove(colnames(.), &quot;treat&quot;))
colData(filtered_counts)$design &lt;- X
paired_cors &lt;- block &lt;- txt &lt;- NULL
if (config$comparisons$paired) {
  block &lt;- colData(filtered_counts)[[rep_col]]
  set.seed(1e6)
  ind &lt;- sample.int(nrow(filtered_counts), n_max, replace = FALSE)
  paired_cors &lt;- duplicateCorrelation(
    object = assay(filtered_counts, &quot;qsmooth&quot;)[ind, ],
    design = X,
    block = block
  )$consensus.correlation
  txt &lt;- glue(
    &quot;Data were nested within {{rep_col}} as a potential source of correlation. &quot;,
    &quot;The estimated correlation within replicate samples was $\\hat{\\rho} = {{round(paired_cors, 3)}}$&quot;,
    .open = &quot;{{&quot;, .close = &quot;}}&quot;
  )
}
fc &lt;- ifelse(is.null(config$comparisons$fc), 1, config$comparisons$fc)
fit &lt;- assay(filtered_counts, &quot;qsmooth&quot;) %&gt;%
  lmFit(design = X, block = block, correlation = paired_cors) %&gt;% 
  treat(fc = fc, trend = TRUE, robust = FALSE)
fit_mu0 &lt;- assay(filtered_counts, &quot;qsmooth&quot;) %&gt;%
  lmFit(design = X, block = block, correlation = paired_cors) %&gt;% 
  treat(fc = 1, trend = TRUE, robust = FALSE)
res_cols &lt;- c(&quot;logFC&quot;, &quot;AveExpr&quot;, &quot;t&quot;, &quot;P.Value&quot;, &quot;fdr&quot;)
rowData(filtered_counts) &lt;- rowData(filtered_counts) %&gt;% 
  .[!colnames(.) %in% c(res_cols, &quot;p_mu0&quot;)] %&gt;% 
  cbind(
    fit %&gt;% 
      topTable(sort.by = &quot;none&quot;, number = Inf) %&gt;% 
      as.list %&gt;% 
      setNames(res_cols) %&gt;% 
      c(
        list(
          p_mu0 = topTable(fit_mu0, sort.by = &quot;none&quot;, number = Inf)$P.Value
        )
      )
  ) </code></pre>
<p>After SQ-normalisation of logCPM values, the <code>limma-trend</code>
method <span class="citation">(<a href="#ref-LawVoom2014"
role="doc-biblioref">Law et al. 2014</a>)</span> was applied to all
retained windows. A simple linear model was fitted taking E2 as the
baseline and estimating the effects of E2DHT on ER binding within each
sliding window, incorporating a trended prior-variance.</p>
<p>After model fitting, hypothesis testing was performed testing:</p>
<p><span class="math display">\[
H_0: -\lambda &lt; \mu &lt; \lambda
\]</span> against <span class="math display">\[
H_A: |\mu| &gt; \lambda
\]</span> where <span class="math inline">\(\mu\)</span> represents the
true mean change in ER binding. The value <span
class="math inline">\(\lambda =\)</span> 0.26 was chosen as this
corresponds to a 20% change in detected signal on the log<sub>2</sub>
scale. This is known as the <code>treat</code> method <span
class="citation">(<a href="#ref-McCarthyTreat2009"
role="doc-biblioref">McCarthy and Smyth 2009</a>)</span>, and p-values
were obtained for each initial window, before merging adjoining
windows.</p>
<p>For subsequent classification when combining across multiple
differential binding results, an additional p-value testing <span
class="math inline">\(\mu = 0\)</span> was added, however this is not
used for detection of sites showing evidence of altered ER binding.</p>
<pre class="r"><code>merged_results &lt;- mergeByCol(
  filtered_counts, col = &quot;AveExpr&quot;, merge_within = 2 * win_step,
  inc_cols = &quot;p_mu0&quot;
) %&gt;% 
  mutate(
    overlaps_ref = overlapsAny(., consensus_peaks),
    direction = case_when(
      n_up == 0 &amp; n_down == 0 ~ &quot;Unchanged&quot;,
      logFC &gt; 0 &amp; n_up &gt; n_down ~ &quot;Up&quot;,
      logFC &lt; 0 &amp; n_up &lt; n_down ~ &quot;Down&quot;,
      TRUE ~ &quot;Ambiguous&quot;
    ) %&gt;% 
      factor(levels = c(&quot;Up&quot;, &quot;Down&quot;, &quot;Ambiguous&quot;, &quot;Unchanged&quot;)) %&gt;% 
      droplevels()
  ) </code></pre>
<p>After selection of the 69,935 sliding windows (90bp), these were
merged into 7,264 genomic regions with size ranging from 90 to 1800bp,
with the median width being 330bp. Representative estimates of
differential binding (i.e. logFC) were taken from the sliding window
with the <em>highest average signal</em> across all samples, within each
set of windows to be merged. Similarly, p-values from the above tests
were also selected from the same sliding window as representative of the
merged region <span class="citation">(<a href="#ref-LunSmythF1000"
role="doc-biblioref">A. T. Lun and Smyth 2015</a>)</span>. The number of
windows showing increased or decreased binding within each merged region
were also included, by counting those within each set of windows with
p-values lower than the selected window. FDR-adjustment was performed
with 189 regions (2.6%) showing significance, based on an FDR-adjusted
p-value alone.</p>
</div>
<div id="independent-hypothesis-weighting"
class="section level2 tabset">
<h2 class="tabset">Independent Hypothesis Weighting</h2>
<pre class="r"><code>fdr_column &lt;- str_subset(names(mcols(merged_results)), &quot;fdr|FDR&quot;)
## Add status (direction) for the ihw = &quot;none&quot; possibility
merged_results &lt;- merged_results %&gt;% 
  mutate(
    status = case_when(
      !!sym(fdr_column) &lt; fdr_alpha ~ as.character(direction),
      !!sym(fdr_column) &gt;= fdr_alpha ~ &quot;Unchanged&quot;
    ) %&gt;% 
      factor(levels = c(levels(direction), &quot;Unchanged&quot;)) %&gt;% 
      droplevels()
  )</code></pre>
<pre class="r"><code>other_targets &lt;- here::here(config$samples$file) %&gt;% 
  read_tsv() %&gt;% 
  pull(&quot;target&quot;) %&gt;% 
  unique() %&gt;% 
  setdiff(target)
ihw_all &lt;- other_targets %&gt;% 
  sapply(
    function(x) {
      here::here(dirname(macs2_path), x, &quot;consensus_peaks.bed&quot;) %&gt;%
        import.bed(seqinfo = sq)
    },
    simplify = FALSE
  ) %&gt;% 
  lapply(granges) %&gt;% 
  lapply(
    function(x) overlapsAny(merged_results, x)
  ) %&gt;% 
  as_tibble() %&gt;% 
  mutate(
    range = as.character(merged_results), id = seq_along(range)
  ) %&gt;% 
  pivot_longer(
    cols = all_of(other_targets),
    names_to = &quot;targets&quot;, 
    values_to = &quot;overlap&quot;
  ) %&gt;% 
  group_by(id, range) %&gt;% 
  summarise(
    targets = ifelse(
      any(overlap), 
      paste(sort(targets[overlap]), collapse = &quot;+ &quot;),
      &quot;No Other Targets&quot;
    ),
    .groups = &quot;drop&quot;
  ) %&gt;% 
  arrange(id) %&gt;% 
  mutate(targets = fct_infreq(targets)) %&gt;% 
  .[[&quot;targets&quot;]]
n_levels &lt;- sum(table(ihw_all) &gt; 1e3) - 1
merged_results$ihw_covariate &lt;- fct_lump_n(ihw_all, n = n_levels)
ihw_proceed &lt;- length(levels(merged_results$ihw_covariate)) &gt; 1
if (ihw_proceed)  fdr_column &lt;- &quot;fdr_ihw&quot;
if (ihw_proceed) {
  ihw &lt;- ihw(
    pvalues = merged_results$P.Value,
    covariates = merged_results$ihw_covariate,
    alpha &lt;- fdr_alpha,
    covariate_type = &quot;nominal&quot;
  )
  merged_results &lt;- mutate(merged_results, fdr_ihw = adj_pvalues(ihw))
} 
merged_results &lt;- merged_results %&gt;% 
  mutate(
    status = case_when(
      !!sym(fdr_column) &lt; fdr_alpha ~ as.character(direction),
      !!sym(fdr_column) &gt;= fdr_alpha ~ &quot;Unchanged&quot;
    ) %&gt;% 
      factor(levels = c(levels(direction), &quot;Unchanged&quot;)) %&gt;% 
      droplevels()
  )</code></pre>
<p>Independent Hypothesis Weighting (IHW) <span class="citation">(<a
href="#ref-IgnatiadisIHW2016" role="doc-biblioref">Ignatiadis et al.
2016</a>)</span> was then used to partition the raw p-values for ER
differential binding by the detection of the other ChIP targets under
investigation in this workflow (AR and H3K27ac). The presence of AR and
H3K27ac was defined simply using the consensus peaks detected under any
treatment by <code>macs2 callpeak</code>, as determined <a
href="ER_macs2_summary.html">previously</a>. This allows recalculation
of the FDR using <em>weighted p-values</em> instead of <em>raw
p-values</em>. In order for IHW to be a viable strategy, partitions
should be greater than 1000. The provided consensus peaks were used as
initial partitions in combination, merging the smallest groups below
this size until all partitions were of a suitable size.</p>
<p>Windows were classified as overlapping a peak from a secondary ChIP
target if <em>any</em> section of the window overlapped the secondary
peak.</p>
<div id="summary-table" class="section level3">
<h3>Summary Table</h3>
<pre class="r"><code>ihw %&gt;%
  as.data.frame() %&gt;% 
  mutate(
    fdr = p.adjust(pvalue, &quot;fdr&quot;),
    status = case_when(
      fdr &lt; fdr_alpha &amp; adj_pvalue &gt; fdr_alpha ~ &quot;Lost&quot;,
      fdr &lt; fdr_alpha &amp; adj_pvalue &lt; fdr_alpha ~ &quot;Retained&quot;,
      fdr &gt; fdr_alpha &amp; adj_pvalue &lt; fdr_alpha ~ &quot;Gained&quot;,
      fdr &gt; fdr_alpha &amp; adj_pvalue &gt; fdr_alpha ~ &quot;Never Significant&quot;
    ) %&gt;% 
      factor(
        levels = c(&quot;Never Significant&quot;, &quot;Retained&quot;, &quot;Gained&quot;, &quot;Lost&quot;)
      )
  ) %&gt;% 
  as_tibble() %&gt;% 
  group_by(covariate, status) %&gt;% 
  tally() %&gt;% 
  ungroup() %&gt;% 
  complete(status = status, covariate = covariate, fill = list(n = 0)) %&gt;% 
  pivot_wider(
    names_from = status, values_from = n, values_fill = 0
  ) %&gt;% 
  mutate(
    `Nett Change` = Gained - Lost
  ) %&gt;% 
  bind_rows(
    summarise_if(., is.numeric, sum)
  ) %&gt;% 
  mutate(
    covariate = str_replace_na(covariate, &quot;Total&quot;),
    Significant = Retained + Gained
  ) %&gt;% 
  dplyr::select(
    Covariate = covariate, ends_with(&quot;Significant&quot;), Retained, everything()
  ) %&gt;% 
  pander(
    justify = &quot;lrrrrrr&quot;,
    caption = paste(
      &quot;*Summary of changes introduced by IHW for windows considered as being&quot;, 
      paste0(&quot;differentially bound by &quot;, target, &quot;.&quot;),
      &quot;This corresponds to a __nett change of&quot;, 
        dplyr::slice(., nrow(.)) %&gt;% 
        mutate(p = `Nett Change` / (Significant - `Nett Change`)) %&gt;% 
        .[[&quot;p&quot;]] %&gt;% 
        percent(0.1),
      &quot;from the initial list__.*&quot;
    ),
    emphasize.strong.rows = nrow(.)
  )</code></pre>
<table>
<caption><em>Summary of changes introduced by IHW for windows considered
as being differentially bound by ER. This corresponds to a <strong>nett
change of 4.2% from the initial list</strong>.</em></caption>
<colgroup>
<col width="15%" />
<col width="22%" />
<col width="15%" />
<col width="12%" />
<col width="10%" />
<col width="8%" />
<col width="15%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Covariate</th>
<th align="right">Never Significant</th>
<th align="right">Significant</th>
<th align="right">Retained</th>
<th align="right">Gained</th>
<th align="right">Lost</th>
<th align="right">Nett Change</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">H3K27ac</td>
<td align="right">2,899</td>
<td align="right">1</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="right">3</td>
<td align="right">-3</td>
</tr>
<tr class="even">
<td align="left">AR+ H3K27ac</td>
<td align="right">1,828</td>
<td align="right">116</td>
<td align="right">107</td>
<td align="right">9</td>
<td align="right">0</td>
<td align="right">9</td>
</tr>
<tr class="odd">
<td align="left">Other</td>
<td align="right">2,337</td>
<td align="right">80</td>
<td align="right">78</td>
<td align="right">2</td>
<td align="right">0</td>
<td align="right">2</td>
</tr>
<tr class="even">
<td align="left"><strong>Total</strong></td>
<td align="right"><strong>7,064</strong></td>
<td align="right"><strong>197</strong></td>
<td align="right"><strong>186</strong></td>
<td align="right"><strong>11</strong></td>
<td align="right"><strong>3</strong></td>
<td align="right"><strong>8</strong></td>
</tr>
</tbody>
</table>
</div>
<div id="initial-group-sizes" class="section level3">
<h3>Initial Group Sizes</h3>
<pre class="r"><code>tibble(ihw = ihw_all) %&gt;% 
  ggplot(
    aes(ihw, fill = ihw)
  ) +
  geom_bar() +
  geom_hline(yintercept = 1e3, colour = &quot;blue&quot;) +
  geom_text(
    aes(y = n + 0.025*max(n), label = comma(n, 1)),
    data = . %&gt;%
      group_by(ihw) %&gt;%
      summarise(n = dplyr::n(), .groups = &quot;drop&quot;) 
  ) +
  scale_y_continuous(expand = expansion(c(0, 0.05)), labels = comma) +
  labs(x = &quot;Peaks Overlapping Other ChIP Targets&quot;, y = &quot;Total&quot;) +
  theme(legend.position = &quot;none&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="ER_E2_E2DHT_differential_binding_files/figure-html/ihw-partitions-raw-1.png" alt="*Breakdown of all windows which overlapped peaks from additional ChIP targets. Any partitions with fewer than 1000 windows (indicated as the blue horizontal line) were combined into the next smallest partition consecutively, until all partitions contained &gt; 1000 windows.*" width="960" />
<p class="caption">
<em>Breakdown of all windows which overlapped peaks from additional ChIP
targets. Any partitions with fewer than 1000 windows (indicated as the
blue horizontal line) were combined into the next smallest partition
consecutively, until all partitions contained &gt; 1000 windows.</em>
</p>
</div>
</div>
<div id="p-value-distributions" class="section level3">
<h3>P-Value Distributions</h3>
<pre class="r"><code>y &lt;- ggplot_build(
  tibble(
    P.Value = merged_results$P.Value,
    ihw_covariate = merged_results$ihw_covariate
    ) %&gt;% 
    ggplot(aes(P.Value, stat(density))) + 
    geom_histogram(bins = 100) +
    facet_wrap(~ihw_covariate) 
) %&gt;% .[[&quot;data&quot;]] %&gt;% 
  .[[1]] %&gt;% 
  .[[&quot;y&quot;]] %&gt;% 
  max()
merged_results %&gt;% 
  select(P.Value, ihw_covariate) %&gt;% 
  mcols() %&gt;% 
  as_tibble() %&gt;% 
  ggplot(aes(P.Value, stat(density))) +
  geom_histogram(fill = &quot;grey&quot;, colour = &quot;black&quot;, bins = 100) +
  geom_text(
    aes(0.8, y, label = lab),
    data = . %&gt;% 
      group_by(ihw_covariate) %&gt;% 
      summarise(n = dplyr::n(), .groups = &quot;drop&quot;) %&gt;% 
      mutate(
        lab = glue(&quot;N = {comma(n, 1)}&quot;),
        y = y
      )
  ) +
  facet_wrap(~ihw_covariate) +
  scale_y_continuous(expand = expansion(c(0, 0.05))) +
  labs(x = &quot;P Value&quot;, y = &quot;Density&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="ER_E2_E2DHT_differential_binding_files/figure-html/plot-ihw-pvals-1.png" alt="*P-Value distributions within all final data partitions. The size of each partition is given within each panel.*" width="960" />
<p class="caption">
<em>P-Value distributions within all final data partitions. The size of
each partition is given within each panel.</em>
</p>
</div>
</div>
<div id="ihw-weights" class="section level3">
<h3>IHW Weights</h3>
<pre class="r"><code>plot(ihw) +
  geom_hline(yintercept = 1) +
  facet_wrap(~group) +
  scale_y_continuous(expand = expansion(c(0, 0.05))) +
  labs(x = &quot;Fold&quot;, y = &quot;Weight&quot;, fill = &quot;Fold&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="ER_E2_E2DHT_differential_binding_files/figure-html/plot-ihw-weights-1.png" alt="*Weights applied to p-values within each partition. 'Folds' represent random sub-partitions within each larger partition generated as part of the IHW process.*" width="960" />
<p class="caption">
<em>Weights applied to p-values within each partition. ‘Folds’ represent
random sub-partitions within each larger partition generated as part of
the IHW process.</em>
</p>
</div>
</div>
<div id="fdr-comparison" class="section level3">
<h3>FDR Comparison</h3>
<pre class="r"><code>ihw %&gt;%
  as.data.frame() %&gt;% 
  ggplot(
    aes(-log10(pvalue), -log10(weighted_pvalue), colour = fold)
  ) +
  geom_point(size = 0.4) +
  geom_hline(
    aes(yintercept = -log10(weighted_pvalue)),
    data = . %&gt;% 
      dplyr::select(-group) %&gt;% 
      dplyr::filter(adj_pvalue &lt; fdr_alpha) %&gt;% 
      dplyr::filter(weighted_pvalue == max(weighted_pvalue)),
    linetype = 2, colour = &quot;blue&quot;
  ) +
  geom_vline(
    aes(xintercept = -log10(pvalue)),
    data =. %&gt;% 
      dplyr::filter(p.adjust(pvalue, &quot;fdr&quot;) &lt; fdr_alpha) %&gt;% 
      dplyr::filter(pvalue == max(pvalue)) %&gt;% 
      dplyr::select(-group),
    linetype = 2, colour = &quot;blue&quot;
  ) +
  facet_wrap(~group, scales = &quot;free&quot;) +
  coord_cartesian(xlim = c(0, 5), ylim = c(0, 5)) +
  labs(
    x = expression(-log[10](p)),
    y = expression(-log[10](p[IHW])),
    colour = &quot;Fold&quot;
  )</code></pre>
<div class="figure" style="text-align: center">
<img src="ER_E2_E2DHT_differential_binding_files/figure-html/plot-fdr-comparison-1.png" alt="*Comparison of raw and weighted p-values for each partition. Blue dashed lines indicate FDR = 0.05 for each set of p-values. Those in the lower-right quadrant would no longer be considered significant after IHW, whilst those in the upper-left quadrant would only be considered as significant after the IHW process. Those in the upper-right quadrant would be considered as significant regardless of the methodology.*" width="960" />
<p class="caption">
<em>Comparison of raw and weighted p-values for each partition. Blue
dashed lines indicate FDR = 0.05 for each set of p-values. Those in the
lower-right quadrant would no longer be considered significant after
IHW, whilst those in the upper-left quadrant would only be considered as
significant after the IHW process. Those in the upper-right quadrant
would be considered as significant regardless of the methodology.</em>
</p>
</div>
</div>
</div>
<div id="mapping-windows" class="section level2">
<h2>Mapping Windows</h2>
<pre class="r"><code>merged_results &lt;- merged_results %&gt;% 
  mutate(
    region = bestOverlap(
      ., 
      gene_regions %&gt;% 
        lapply(select, region) %&gt;% 
        GRangesList() %&gt;% 
        unlist(),
      var = &quot;region&quot;
    ) %&gt;% 
      factor(levels = regions)
  )
if (has_features) {
  merged_results &lt;- merged_results %&gt;% 
    mutate(
      feature = bestOverlap(
        ., external_features, &quot;feature&quot;, missing = &quot;no_feature&quot;
      ) %&gt;% 
        factor(levels = names(colours$features)) %&gt;% 
        fct_relabel(str_sep_to_title)
    )
}
merged_results &lt;- merged_results %&gt;%
  mapByFeature(
    genes = genes_gr, 
    prom = GRangesList(
      ## Add any external features which appear to be promoters
      ifelse(
        &quot;feature&quot; %in% colnames(mcols(external_features)), 
        list(granges(subset(external_features, str_detect(feature, &quot;[Pp]rom&quot;)))), 
        list(GRanges())
      )[[1]], 
      granges(gene_regions$promoters)) %&gt;% 
      unlist() %&gt;% 
      reduce(),
    enh = ifelse(
      &quot;feature&quot; %in% colnames(mcols(external_features)), 
      list(granges(subset(external_features, str_detect(feature, &quot;[Ee]nhanc&quot;)))), 
      list(GRanges())
    )[[1]],
    gi = hic, 
    gr2gene = params$mapping$gr2gene,
    prom2gene = params$mapping$prom2gene,
    enh2gene = params$mapping$enh2gene,
    gi2gene = params$mapping$gi2gene
  )</code></pre>
<p>In addition to statistical analysis, merged windows were first mapped
to the gene-centric region with the largest overlap. , followed by
mapping to all annotated genes.</p>
<p>During mapping to genes, promoters were defined the union of all
regions defined based on transcript-level annotation, and any external
features which were defined as promoters.</p>
<p>These features were used to map merged windows to features using the
process defined in the function
<code>extraChIPs::mapByFeature()</code>:</p>
<ol style="list-style-type: decimal">
<li>Ranges <em>overlapping a promoter</em> are assigned to genes
directly overlapping that specific promoter</li>
<li>Ranges <em>overlapping an enhancer</em> are assigned to all genes
within 100 kb of the enhancer</li>
<li>Ranges <em>overlapping a long-range interaction</em> are assigned to
all genes directly overlapping either end of the interaction</li>
<li>Ranges with <em>no gene assignment from the previous steps</em> are
assigned to all overlapping genes, or the nearest gene within 100kb</li>
</ol>
<p>Notably, genes are only passed to step 4 if no gene assignment has
been made in steps 1, 2 or 3. For visualisation purposes, only genes
which were considered as detected in any provided RNA-Seq data will be
shown as the mapping targets.</p>
</div>
</div>
<div id="results" class="section level1">
<h1>Results</h1>
<div id="result-tables" class="section level2 tabset">
<h2 class="tabset">Result Tables</h2>
<pre class="r"><code>up_col &lt;- function(x) {
  if (is.na(x) | is.nan(x)) return(&quot;#ffffff&quot;)
  rgb(
    colorRamp(c(&quot;#ffffff&quot;, colours$direction[[&quot;up&quot;]]))(x), maxColorValue = 255
  )
}
down_col &lt;- function(x) {
  if (is.na(x) | is.nan(x)) return(&quot;#ffffff&quot;)
  rgb(
    colorRamp(c(&quot;#ffffff&quot;, colours$direction[[&quot;down&quot;]]))(x), maxColorValue = 255
  )
}
unch_col &lt;- function(x) {
  if (is.na(x) | is.nan(x)) return(&quot;#ffffff&quot;)
  rgb(
    colorRamp(c(&quot;#ffffff&quot;, colours$direction[[&quot;unchanged&quot;]]))(x), 
    maxColorValue = 255
  )
}
lfc_col &lt;- function(x){
  if (is.na(x) | is.nan(x)) return(&quot;#ffffff&quot;)
  rgb(
    colorRamp(c(colours$direction[[&quot;down&quot;]], &quot;#ffffff&quot;, colours$direction[[&quot;up&quot;]]))(x), 
    maxColorValue = 255
  )
}
expr_col &lt;- function(x){
  if (is.na(x) | is.nan(x)) return(&quot;#ffffff&quot;)
  rgb(colorRamp(hcl.colors(9, &quot;TealRose&quot;))(x), maxColorValue = 255)
}
bar_style &lt;- function(width = 1, fill = &quot;#e6e6e6&quot;, height = &quot;75%&quot;, align = c(&quot;left&quot;, &quot;right&quot;), color = NULL, fontSize = c()) {
  align &lt;- match.arg(align)
  if (align == &quot;left&quot;) {
    position &lt;- paste0(width * 100, &quot;%&quot;)
    image &lt;- sprintf(&quot;linear-gradient(90deg, %1$s %2$s, transparent %2$s)&quot;, fill, position)
  } else {
    position &lt;- paste0(100 - width * 100, &quot;%&quot;)
    image &lt;- sprintf(&quot;linear-gradient(90deg, transparent %1$s, %2$s %1$s)&quot;, position, fill)
  }
  styles &lt;- list(
    backgroundImage = image,
    backgroundSize = paste(&quot;100%&quot;, height),
    backgroundRepeat = &quot;no-repeat&quot;,
    backgroundPosition = &quot;center&quot;,
    color = color
  )
  if (!is.null(fontSize)) styles$fontSize = fontSize
  styles
}
with_tooltip &lt;- function(value, width = 30) {
  tags$span(title = value, str_trunc(value, width))
}
n_unch &lt;- sum(merged_results$status == &quot;Unchanged&quot;)
n_windows &lt;- length(merged_results)
tbl_cols &lt;- list(
  Region = colDef(
    footer = htmltools::tags$b(&quot;Total&quot;), minWidth = 200
  ),
  Feature = colDef(
    footer = htmltools::tags$b(&quot;Total&quot;), minWidth = 200
  ),
  Up = colDef(
    cell = function(value) comma(value, 1),
    style = function(value) {
      normalized &lt;- value / (n_windows - n_unch)
      color &lt;- up_col(normalized)
      list(background = color)
    },
    footer = function(values) htmltools::tags$b(comma(sum(values)))
  ),
  Down = colDef(
    cell = function(value) comma(value, 1),
    style = function(value) {
      normalized &lt;- value / (n_windows - n_unch)
      color &lt;- down_col(normalized)
      list(background = color)
    },
    footer = function(values) htmltools::tags$b(comma(sum(values)))
  ),
  Ambiguous = colDef(
    cell = function(value) comma(value, 1),
    style = function(value) {
      normalized &lt;- value / (n_windows - n_unch)
      color &lt;- unch_col(normalized)
      list(background = color)
    },
    footer = function(values) htmltools::tags$b(comma(sum(values)))
  ),
  Unchanged = colDef(
    cell = function(value) comma(value, 1),
    style = function(value) {
      normalized &lt;- value / n_windows
      color &lt;- unch_col(normalized)
      list(background = color)
    },
    footer = function(values) htmltools::tags$b(comma(sum(values)))
  ),
  `% Changed` = colDef(
    cell = function(value) percent(value, 0.1),
    style = function(value) bar_style(width = value, fill = &quot;#B3B3B3&quot;, align = &quot;right&quot;),
    align = &quot;right&quot;,
    footer = htmltools::tags$b(
      percent(sum(1 - n_unch/ n_windows), 0.1)
    )
  ),
  Total = colDef(
    cell = function(value) comma(value, 1),
    style = function(value) {
      bar_style(width = value / n_windows, fill = &quot;#B3B3B3&quot;, align = &quot;right&quot;)
    },
    align = &quot;right&quot;,
    footer = function(values) htmltools::tags$b(comma(sum(values)))
  ),
  `% Of All Windows` = colDef(
    style = function(value) bar_style(width = value, fill = &quot;#B3B3B3&quot;, align = &quot;right&quot;),
    format = colFormat(digits = 2, percent = TRUE)
  )
)</code></pre>
<div id="overall-results" class="section level3">
<h3>Overall Results</h3>
<pre class="r"><code>tbl &lt;- merged_results %&gt;% 
  mutate(w = width) %&gt;% 
  as_tibble() %&gt;% 
  group_by(status) %&gt;% 
  summarise(
    n = dplyr::n(), 
    width = median(w),
    AveExpr = median(AveExpr),
    .groups = &quot;drop&quot;
  ) %&gt;% 
  mutate(`%` = n / sum(n)) %&gt;% 
  dplyr::select(status, n, `%`, everything()) %&gt;% 
  reactable(
    searchable = FALSE, filterable = FALSE,
    columns = list(
      status = colDef(
        name = &quot;Status&quot;, maxWidth = 150,
        footer = htmltools::tags$b(&quot;Overall&quot;)
      ),
      n = colDef(
        name = &quot;Nbr of Windows&quot;, maxWidth = 150, format = colFormat(separators = TRUE),
        footer = htmltools::tags$b(comma(length(merged_results)))
      ),
      width = colDef(
        name = &quot;Median Width (bp)&quot;, format = colFormat(digits = 1),
        footer = htmltools::tags$b(round(median(width(merged_results)), 1)),
        maxWidth = 200
      ),
      `%` = colDef(
        name = &quot;% Total Windows&quot;, format = colFormat(percent = TRUE, digits = 1),
        style = function(value) bar_style(width = value, align = &quot;right&quot;),
        maxWidth = 150
      ),
      AveExpr = colDef(
        name = &quot;Median Signal (logCPM)&quot;,
        format = colFormat(digits = 3),
        maxWidth = 200,
        footer = htmltools::tags$b(
          round(median(merged_results$AveExpr), 3)
        )
      )
    )
  )
cp &lt;- htmltools::em(
  glue(
    &quot;Overall results for changed {target} binding in the &quot;,
    glue_collapse(rev(treat_levels), last = &quot; Vs. &quot;),
    &quot; comparison.&quot;
  )
)
div(class = &quot;table&quot;,
  div(class = &quot;table-header&quot;,
      div(class = &quot;caption&quot;, cp),
      tbl
  )
)</code></pre>
<div class="table">
<div class="table-header">
<div class="caption">
<em>Overall results for changed ER binding in the E2DHT Vs. E2 comparison.</em>
</div>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-88e4d9d817c96835cfff" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-88e4d9d817c96835cfff">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"status":["Up","Down","Unchanged"],"n":[195,2,7067],"%":[0.026844713656387666,0.00027533039647577095,0.97287995594713661],"width":[390,315,330],"AveExpr":[0.92934676735492272,0.30102573649244602,0.65314620701874737]},"columns":[{"id":"status","name":"Status","type":"factor","footer":{"name":"b","attribs":{},"children":["Overall"]},"maxWidth":150},{"id":"n","name":"Nbr of Windows","type":"numeric","format":{"cell":{"separators":true},"aggregated":{"separators":true}},"footer":{"name":"b","attribs":{},"children":["7,264"]},"maxWidth":150},{"id":"%","name":"% Total Windows","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":150,"style":[{"backgroundImage":"linear-gradient(90deg, transparent 97.3155286343612%, #e6e6e6 97.3155286343612%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 99.9724669603524%, #e6e6e6 99.9724669603524%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 2.71200440528634%, #e6e6e6 2.71200440528634%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null}]},{"id":"width","name":"Median Width (bp)","type":"numeric","format":{"cell":{"digits":1},"aggregated":{"digits":1}},"footer":{"name":"b","attribs":{},"children":["330"]},"maxWidth":200},{"id":"AveExpr","name":"Median Signal (logCPM)","type":"numeric","format":{"cell":{"digits":3},"aggregated":{"digits":3}},"footer":{"name":"b","attribs":{},"children":["0.66"]},"maxWidth":200}],"dataKey":"c1923ac3f0d9c793d62501de609650f2"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
<div id="summary-by-region" class="section level3">
<h3>Summary By Region</h3>
<pre class="r"><code>df &lt;- merged_results %&gt;% 
  select(
    status, all_of(fdr_column), region
  ) %&gt;% 
  as_tibble() %&gt;% 
  group_by(region, status) %&gt;% 
  summarise(n = dplyr::n(), .groups = &quot;drop&quot;) %&gt;% 
  complete(region, status, fill = list(n = 0)) %&gt;% 
  group_by(region) %&gt;% 
  mutate(Total = sum(n)) %&gt;% 
  ungroup() %&gt;% 
  pivot_wider(names_from = &quot;status&quot;, values_from = &quot;n&quot;) %&gt;% 
  mutate(
    `% Of All Windows` = Total / length(merged_results),
    `% Changed` = 1 - Unchanged / Total
  ) %&gt;% 
  arrange(region) %&gt;% 
  dplyr::select(
    Region = region,
    any_of(names(direction_colours)),
    `% Changed`, Total, `% Of All Windows`
  )
cp &lt;- htmltools::em(
  glue(
    &quot;Overall results for changed {target} binding in the &quot;,
    glue_collapse(rev(treat_levels), last = &quot; Vs. &quot;),
    &quot; comparison, broken down by which genomic region contains the largest &quot;, 
    &quot;overlap with each merged window.&quot;
  )
)
tbl &lt;- df %&gt;%
  reactable(
    columns = tbl_cols[colnames(df)],
    defaultColDef = colDef(
      footer = function(values) htmltools::tags$b(comma(sum(values)))
    ),
    fullWidth = TRUE
  )
div(class = &quot;table&quot;,
  div(class = &quot;table-header&quot;,
      div(class = &quot;caption&quot;, cp),
      tbl
  )
)</code></pre>
<div class="table">
<div class="table-header">
<div class="caption">
<em>Overall results for changed ER binding in the E2DHT Vs. E2 comparison, broken down by which genomic region contains the largest overlap with each merged window.</em>
</div>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-e19f252553237c324370" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-e19f252553237c324370">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"Region":["Promoter (-1500/+500bp)","Upstream Promoter (<5kb)","Exon","Intron","Intergenic (<10kb)","Intergenic (>10kb)"],"Up":[16,16,8,91,20,44],"Down":[0,0,0,2,0,0],"Unchanged":[1183,729,155,2967,577,1456],"% Changed":[0.01334445371142623,0.021476510067114041,0.049079754601227044,0.030392156862745101,0.03350083752093802,0.029333333333333322],"Total":[1199,745,163,3060,597,1500],"% Of All Windows":[0.16506057268722468,0.10256057268722467,0.022439427312775331,0.42125550660792954,0.082186123348017615,0.20649779735682819]},"columns":[{"id":"Region","name":"Region","type":"factor","footer":{"name":"b","attribs":{},"children":["Total"]},"minWidth":200},{"id":"Up","name":"Up","type":"numeric","footer":{"name":"b","attribs":{},"children":["195"]},"cell":["16","16","8","91","20","44"],"style":[{"background":"#FAEAEC"},{"background":"#FAEAEC"},{"background":"#FCF4F5"},{"background":"#E68997"},{"background":"#F9E5E8"},{"background":"#F3C6CD"}]},{"id":"Down","name":"Down","type":"numeric","footer":{"name":"b","attribs":{},"children":["2"]},"cell":["0","0","0","2","0","0"],"style":[{"background":"#FFFFFF"},{"background":"#FFFFFF"},{"background":"#FFFFFF"},{"background":"#FCFDFE"},{"background":"#FFFFFF"},{"background":"#FFFFFF"}]},{"id":"Unchanged","name":"Unchanged","type":"numeric","footer":{"name":"b","attribs":{},"children":["7,067"]},"cell":["1,183","729","155","2,967","577","1,456"],"style":[{"background":"#EAEAEA"},{"background":"#F2F2F2"},{"background":"#FCFCFC"},{"background":"#CACACA"},{"background":"#F4F4F4"},{"background":"#E5E5E5"}]},{"id":"% Changed","name":"% Changed","type":"numeric","footer":{"name":"b","attribs":{},"children":["2.7%"]},"cell":["1.3%","2.1%","4.9%","3.0%","3.4%","2.9%"],"align":"right","style":[{"backgroundImage":"linear-gradient(90deg, transparent 98.6655546288574%, #B3B3B3 98.6655546288574%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 97.8523489932886%, #B3B3B3 97.8523489932886%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 95.0920245398773%, #B3B3B3 95.0920245398773%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 96.9607843137255%, #B3B3B3 96.9607843137255%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 96.6499162479062%, #B3B3B3 96.6499162479062%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 97.0666666666667%, #B3B3B3 97.0666666666667%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null}]},{"id":"Total","name":"Total","type":"numeric","footer":{"name":"b","attribs":{},"children":["7,264"]},"cell":["1,199","745","163","3,060","597","1,500"],"align":"right","style":[{"backgroundImage":"linear-gradient(90deg, transparent 83.4939427312775%, #B3B3B3 83.4939427312775%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 89.7439427312775%, #B3B3B3 89.7439427312775%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 97.7560572687225%, #B3B3B3 97.7560572687225%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 57.874449339207%, #B3B3B3 57.874449339207%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 91.7813876651982%, #B3B3B3 91.7813876651982%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 79.3502202643172%, #B3B3B3 79.3502202643172%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null}]},{"id":"% Of All Windows","name":"% Of All Windows","type":"numeric","footer":{"name":"b","attribs":{},"children":["1"]},"format":{"cell":{"digits":2,"percent":true},"aggregated":{"digits":2,"percent":true}},"style":[{"backgroundImage":"linear-gradient(90deg, transparent 83.4939427312775%, #B3B3B3 83.4939427312775%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 89.7439427312775%, #B3B3B3 89.7439427312775%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 97.7560572687225%, #B3B3B3 97.7560572687225%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 57.874449339207%, #B3B3B3 57.874449339207%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 91.7813876651982%, #B3B3B3 91.7813876651982%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 79.3502202643172%, #B3B3B3 79.3502202643172%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null}]}],"dataKey":"fafed84bdf08c68c4fb042d9d77bf609"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
<div id="most-highly-ranked" class="section level3">
<h3>Most Highly Ranked</h3>
<pre class="r"><code>show_n &lt;- min(200, length(merged_results))
scaling_vals &lt;- list(
  logFC = c(-1, 1)*max(abs(merged_results$logFC)),
  AveExpr = range(merged_results$AveExpr),
  Width = max(width(merged_results))
)
cp &lt;- htmltools::em(
  glue(
    &quot;The {show_n} most highly-ranked windows by FDR, with &quot;,
    sum(merged_results$P.Value_fdr &lt; fdr_alpha), 
    &quot; showing evidence of changed {target} binding. &quot;,
    &quot;Regions were assigned based on which genomic region showed the largest &quot;,
    &quot;overlap with the final merged window. &quot;,
    ifelse(
      has_features, 
      glue(&quot;Features are as provided in the file {basename(config$external$features)}. &quot;),
      glue(&quot;&quot;)
    ),
    &quot;For windows mapped to large numbers of genes, hovering a mouse over the &quot;, 
    &quot;cell will reveal the full set of genes. &quot;,
    ifelse(
      has_rnaseq,
      &quot;Only genes considered as detected in the RNA-Seq data are shown. &quot;,
      &quot;&quot;
    ),
    &quot;The Macs2 Peak column is filterable using T/F values&quot;
  )
)
fs &lt;- 12
tbl &lt;- merged_results %&gt;%
  mutate(w = width) %&gt;% 
  arrange(!!sym(fdr_column)) %&gt;% 
  plyranges::slice(seq_len(show_n)) %&gt;% 
  select(
    w, AveExpr, logFC, FDR = !!sym(fdr_column), 
    overlaps_ref, Region = region, any_of(&quot;feature&quot;), Genes = gene_name
  ) %&gt;% 
  as_tibble() %&gt;% 
  dplyr::rename(
    Range = range, `Width (bp)` = w, `Macs2 Peak` = overlaps_ref
  ) %&gt;% 
  rename_all(str_replace_all, &quot;feature&quot;, &quot;Feature&quot;) %&gt;% 
  mutate(Genes = vapply(Genes, paste, character(1), collapse = &quot;; &quot;)) %&gt;% 
  reactable(
    filterable = TRUE,
    showPageSizeOptions = TRUE,
    pageSizeOptions = c(10, 20, 50, show_n), defaultPageSize = 10,
    borderless  = TRUE,
    columns = list(
      Range = colDef(
        minWidth = 10 * fs,
        cell = function(value) {
          str_replace_all(value, &quot;:&quot;, &quot;: &quot;)
        }
      ),
      `Width (bp)` = colDef(
        style = function(value) {
          x &lt;- value / scaling_vals$Width
          colour &lt;- expr_col(x)
          list(
            background = colour, 
            borderRight = &quot;1px solid rgba(0, 0, 0, 0.1)&quot;
          )
        },
        maxWidth = 5 * fs
      ),
      AveExpr = colDef(
        cell = function(value) round(value, 2),
        style = function(value) {
          x &lt;- (value - min(scaling_vals$AveExpr)) / diff(scaling_vals$AveExpr)
          colour = expr_col(x)
          list(background = colour)
        },
        maxWidth = 5.5 * fs
      ),
      logFC = colDef(
        cell = function(value) round(value, 2),
        style = function(value) {
          x &lt;- (value - min(scaling_vals$logFC)) / diff(scaling_vals$logFC)
          colour &lt;- lfc_col(x)
          list(background = colour)
        },
        maxWidth = 5.5 * fs
      ),
      FDR = colDef(
        cell = function(value) sprintf(&quot;%.2e&quot;, value),
        style = list(borderRight = &quot;1px solid rgba(0, 0, 0, 0.1)&quot;),
        maxWidth = 5.5 * fs
      ),
      `Macs2 Peak` = colDef(
        cell = function(value) ifelse(value, &quot;\u2714 Yes&quot;, &quot;\u2716 No&quot;),
        style = function(value) {
          color &lt;- ifelse(value, &quot;#008000&quot;, &quot;#e00000&quot;)
          list(color = color, borderRight = &quot;1px solid rgba(0, 0, 0, 0.1)&quot;)
        },
        maxWidth = 5 * fs
      ),
      Region = colDef(maxWidth = 150),
      Genes = colDef(
        cell = function(value) with_tooltip(value),
        minWidth = 11 * fs
      )
    ),
    theme = reactableTheme(style = list(fontSize = fs))
  )
div(class = &quot;table&quot;,
  div(class = &quot;table-header&quot;,
      div(class = &quot;caption&quot;, cp),
      tbl
  )
)</code></pre>
<div class="table">
<div class="table-header">
<div class="caption">
<em>The 200 most highly-ranked windows by FDR, with 189 showing evidence of changed ER binding. Regions were assigned based on which genomic region showed the largest overlap with the final merged window. For windows mapped to large numbers of genes, hovering a mouse over the cell will reveal the full set of genes. The Macs2 Peak column is filterable using T/F values</em>
</div>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-03d27108608e41b96dee" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-03d27108608e41b96dee">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"Range":["chr10:120116131-120116580","chr4:41079691-41080140","chr10:81102391-81102870","chr15:52948501-52948980","chr16:75087601-75088080","chr1:96354541-96354990","chr12:52520341-52520640","chr11:130299901-130300230","chr22:30804721-30805110","chr13:111490111-111490440","chrX:20489491-20490000","chr12:15426871-15427230","chr22:30788911-30789330","chr19:41360971-41361420","chr21:16959061-16959690","chr4:150780211-150780660","chr18:22748521-22749000","chr10:113150221-113150640","chr8:124417831-124418040","chr11:114050011-114050340","chr3:190525501-190526010","chr2:130703131-130703610","chr5:71044111-71044830","chr2:120095461-120095610","chr10:81101911-81102240","chr17:47468401-47468760","chr11:34844761-34845090","chr6:148579861-148580340","chr4:182878051-182878710","chr5:35205781-35206110","chrX:93855181-93855810","chr4:97953721-97954020","chr20:46540801-46541310","chr19:4991521-4992450","chr6:12844471-12845010","chr9:33437431-33437610","chr2:147629551-147629940","chr7:33872461-33873000","chr7:26598151-26598480","chr7:157413451-157414260","chr8:15525301-15525750","chr16:20599021-20599500","chr2:62652151-62652300","chr8:28005151-28005630","chr4:6634801-6635430","chr17:68267461-68268180","chr3:117850051-117850410","chr3:144174931-144175350","chr3:59527171-59527440","chr2:30827671-30828060","chr3:116952781-116953110","chr16:71627821-71628150","chrX:16579861-16580160","chr11:34771711-34772130","chr20:42930901-42931260","chr8:67626001-67626990","chr2:6050131-6050550","chr6:99658621-99659100","chr2:75548911-75549240","chr21:20939551-20939640","chr7:16920121-16920660","chr12:31612891-31613310","chr13:53559721-53560050","chr8:117010381-117010950","chr4:39545731-39546060","chrX:16607371-16607820","chr18:61835821-61836060","chr3:188000971-188001420","chr10:124987321-124987770","chr17:80071981-80072100","chr3:25302121-25302360","chr17:45983131-45983550","chr17:78076771-78077040","chr3:39172021-39172380","chr5:71010751-71011140","chr10:89407861-89407950","chr18:21183991-21184350","chrX:67583791-67584240","chr16:1680841-1681230","chr1:237301831-237302100","chr9:4866151-4866840","chr10:115681561-115682250","chr9:123748081-123748380","chr6:165980731-165981090","chr2:188851081-188851260","chr6:135467671-135468150","chr6:3670141-3670440","chr6:53333911-53334330","chr15:39187441-39187680","chr11:34847911-34848390","chr1:25802611-25803090","chr20:42509311-42509880","chr1:163586491-163587150","chr15:83588011-83588460","chr19:2576161-2576580","chr5:148398811-148398990","chr9:131869651-131869890","chr12:116931901-116932410","chr12:46887421-46887990","chr14:50437831-50438160","chr9:33431461-33431610","chr2:67392751-67393140","chr3:18531721-18531990","chr8:119395321-119395860","chr1:17641171-17641710","chr6:122138251-122138820","chr5:301981-302850","chr7:29495671-29496060","chr17:43978321-43978980","chr10:115998901-115999410","chr22:34433371-34433700","chr14:97127101-97127400","chrX:40378291-40378680","chr3:181019101-181019520","chr10:79629661-79630260","chr22:30212821-30213120","chr6:34566721-34566840","chr8:110605471-110605830","chr18:18909961-18910650","chr7:141701491-141701670","chr5:32094181-32094690","chr7:49007641-49008210","chr2:193302751-193303350","chr8:22592311-22592850","chr3:164019811-164020230","chr8:80990281-80990730","chr5:93739351-93739440","chr15:89672971-89673360","chr11:45007321-45007830","chr8:98361241-98361450","chr1:163596421-163596660","chr20:23576761-23577150","chr20:56563051-56563290","chr11:2214481-2214840","chr17:6589531-6589950","chr7:69278101-69278190","chr7:95179501-95179740","chrX:100000711-100001460","chr8:133911241-133911540","chr14:69036601-69037260","chr5:67830031-67830660","chr5:173193421-173193780","chr2:216830971-216831420","chr12:133042981-133043610","chr13:42355501-42355590","chr7:34275811-34276080","chr3:104330071-104330550","chr2:113346121-113346420","chr16:67628611-67628970","chr3:187634881-187635300","chr3:150454861-150455610","chr1:100951471-100951800","chr9:98044591-98044860","chr16:23373961-23374290","chr1:106819171-106819290","chr4:113204491-113204820","chr8:98278651-98278950","chr8:68722801-68723190","chr13:71037841-71038110","chr17:72754111-72754860","chr5:159713881-159714300","chr20:43968991-43969470","chrX:90632731-90633120","chr3:187498801-187499070","chr3:193640131-193640400","chr9:38993851-38994060","chr9:118088701-118089150","chr20:19110781-19111140","chr9:121766881-121767570","chr4:70274161-70274280","chr11:101049151-101049630","chr19:45622831-45623010","chr7:115318081-115318410","chrX:133704511-133704900","chr3:42586081-42586530","chr16:50163361-50163780","chr17:60803191-60803520","chr8:91013911-91014090","chr14:84427621-84427770","chr6:51877951-51878190","chr17:35633461-35633880","chrX:10020031-10020240","chr5:44589961-44590110","chr9:31545361-31545720","chr17:60795691-60795810","chr20:15421051-15421380","chr9:4859011-4859430","chr5:32563621-32564070","chr17:15658141-15658410","chr20:49459831-49460100","chr4:41477491-41477790","chr5:134544211-134544570","chr2:227203441-227204040","chr10:52233661-52234020","chr16:52790791-52791030","chr4:95481061-95481450","chr2:100684201-100684500","chr6:26296141-26296530","chr5:53951971-53952240","chr12:5186131-5186250"],"Width (bp)":[450,450,480,480,480,450,300,330,390,330,510,360,420,450,630,450,480,420,210,330,510,480,720,150,330,360,330,480,660,330,630,300,510,930,540,180,390,540,330,810,450,480,150,480,630,720,360,420,270,390,330,330,300,420,360,990,420,480,330,90,540,420,330,570,330,450,240,450,450,120,240,420,270,360,390,90,360,450,390,270,690,690,300,360,180,480,300,420,240,480,480,570,660,450,420,180,240,510,570,330,150,390,270,540,540,570,870,390,660,510,330,300,390,420,600,300,120,360,690,180,510,570,600,540,420,450,90,390,510,210,240,390,240,360,420,90,240,750,300,660,630,360,450,630,90,270,480,300,360,420,750,330,270,330,120,330,300,390,270,750,420,480,390,270,270,210,450,360,690,120,480,180,330,390,450,420,330,180,150,240,420,210,150,360,120,330,420,450,270,270,300,360,600,360,240,390,300,390,270,120],"AveExpr":[1.3854350762554835,1.2195520407764706,1.7595438225471902,2.3518010812719865,0.9588680533204611,0.44049391907349222,-0.0042603712187860117,1.2154611441580514,1.0569586607942754,0.10565556020708421,2.1482056777758638,1.4619463442471927,1.6070634068457779,1.2775129966700352,3.1310168986927747,1.9319743446631694,2.3059923315785573,0.93477602389551573,-0.23932189986288016,0.24771194285231857,0.42282368227561451,1.6141369370731451,3.1180508507214131,-0.072465098363314065,1.0469133163743387,0.91623811300619129,0.7604749122903347,0.99029450153381904,2.4515221894167114,0.63309825664018105,1.6745200509790086,-0.020990156841727892,1.8403324008608832,2.4765666145869254,2.5525138321471044,-0.37511947539575657,0.73811509721693758,2.5932365950688641,0.5813682609613805,2.8316342285542149,1.5367151106767569,1.8430669447578052,0.03021422067877183,1.8500711005149586,2.4043510861581314,3.4403920616661949,0.92934676735492272,1.6743890202993419,0.12101049000532405,0.48886375946924621,0.45581774959851795,0.066276580346751976,0.74632835179560386,1.0773096506741726,0.83970602824386542,3.9820677064049281,0.81053353207716872,1.3223066298942046,0.25357047013444034,-0.11016544715132513,3.1846663469197605,1.6178935624901012,0.59778297013123338,2.4198732571275543,0.33075004639244909,2.0306133321340551,-0.050505456688982361,1.0430014123405962,1.5932720884684184,-0.44464179671332515,0.13704686669448612,0.83193350319291004,0.80378224147752952,0.38868817680528583,1.354230989907411,0.10141855239687197,1.4264249460120553,2.1037819006501426,1.383016851520507,0.15457961084850522,3.2807779894041813,3.549350242663917,0.63499476090171469,0.65399134971544204,-0.18595993013751363,1.7058390095695888,0.31247878369805054,1.0397761367782095,0.30879835363855601,1.843196440465436,1.385641659838206,2.9418510397767283,3.0953213718543351,0.79559377861430369,1.5478887001437125,-0.3488607712147293,0.16426755780810576,2.1369584055682815,2.4079849249450431,0.94026919279893795,-0.30288553811624558,0.93928759388117378,0.28099368182496604,1.5836635503748289,2.1079247726557933,2.5797708042451566,0.24330649278177074,1.0698467518262822,3.1201936297212027,1.8807909933745088,0.80588257698833388,0.70527024199927768,0.90055083696118565,1.9418661811473374,2.2289393867157288,0.31374131040148057,-0.25213007258940956,0.63366884188194206,3.505586797091818,-0.18308624586490149,1.2034472288893232,2.1492470002833963,2.0363045494762728,1.3266903068110096,1.7416568772777392,0.46670292510463107,-0.2390749732905674,1.2294410808524172,1.6498962147633882,-0.17879004655521769,0.11370879149044542,0.8874103029882412,0.14578831943939666,0.77245218324231724,1.1229722954207053,-0.33462895510380175,0.0075410489676771371,3.0721585689960871,0.3351836533794606,3.3998590472046435,2.2615726183611997,0.8553398763529817,2.0188625278951462,0.3159091102200221,-0.27706177950351241,0.6836656082128858,1.3833549033961725,0.54485180030888025,0.52550275090645571,0.79682458944241641,3.5908993390415942,0.05156874342345958,0.11672643839517412,0.49273074916073045,-0.25356003854988146,0.55137739285770948,0.74164533128999621,1.2305549567637522,-0.084914372931967255,3.0992215415069526,1.2939791990544576,1.7246066613150788,0.96044699027615155,0.42147643961370218,0.61177082701551466,0.14634659586254159,2.1049272613032781,1.0861609738570039,2.620602114272776,-0.21940882506101264,0.78330623317255932,-0.079433005594920794,0.33773902274312623,0.97543535943730464,0.11666104301571201,1.3099984970703622,0.33985785041235395,-0.18413063149109113,-0.030585521801945192,0.34131116944803536,1.0578212634318067,0.049626726083158698,-0.18125476018766723,1.0874245141866188,0.29206979308439901,0.47557700609762144,0.47344608997502846,1.537774655840795,0.022064013878505068,1.0386834474126456,0.20078540925399926,0.69100203475674093,2.5292817473069822,0.73205996725322398,0.44163841868281412,1.0467340811982306,0.65512548292829298,1.067952058509549,0.43100845160952667,0.14974868344520151],"logFC":[2.1932946421991808,2.3272758577227544,1.8885751658250225,1.5079641318023498,2.2141511211902976,2.441204608826427,2.8126046656647472,1.949654162794612,1.9555945790729226,2.497758441724395,1.4492353522162986,1.6950255372731984,1.7258275475936951,1.8020086394237789,1.0469352422278013,1.4428824224310526,1.2554834192714319,1.7942327285842765,2.4869761469709499,2.1574570213265232,1.9614888423734269,1.4789989826957248,0.94108252231196865,2.2349770006132532,1.6101141594777224,1.6292611736955243,1.697069125554834,1.6348129962209323,1.0906062280632847,1.6933760709443391,1.313742464108747,2.0399875795032609,1.2295336757309587,1.0974545565260048,1.0095204622948113,2.2696386712936492,1.5999793219564107,0.96482498277406659,1.6795767702758035,0.89084097723627731,1.3035346589182968,1.1668214742817433,1.9811574869058111,1.1355242436617052,0.98725732625989238,0.69761765746253235,1.4133443404312429,1.2074264043200555,1.7343684576255223,1.5925339745379021,1.6021191308721687,1.7539436743093149,1.3995175733687479,1.2906650717382817,1.470125024032199,0.57976151711302293,1.4357793674079828,1.2904364212866022,1.5778629331582927,1.8499327804600358,0.74100682812101837,1.1768813888254934,1.4856264190642254,0.98216032669441933,1.5329754677128506,1.0165503073259634,1.7953455195535071,1.310020331996953,1.106985930768954,2.0134300057601271,1.5814231022439509,1.3217126749389303,1.3227529416100865,1.4677391370085318,1.2021491289046364,1.5901072387107276,1.172678803807147,0.96486487505633989,1.1705797230774928,1.5064400035938135,0.68017535576922217,0.62766337870905486,1.328010574722764,1.3718000760391802,1.7100480329116887,1.044995649049812,1.4696525070858912,1.2095181977188878,1.4337769712512485,1.0378105710638761,1.1764707504004805,0.72901741124940755,0.71095323029546464,1.2405043755766374,1.0632331288234524,1.7670156003217787,1.3898626846655773,0.92604055549552067,0.84406315677068733,1.1916687254693792,1.7067238567807475,1.3156271964122483,1.4139679813247972,1.0275554803837947,0.91600384375281008,0.78059666727671162,1.3528942515550966,1.1445289171065078,0.67369388659043616,0.98489429913109428,1.208963979398721,1.2252527872522003,1.1828130062624844,0.91566194761769804,0.8622839607617766,1.2943074605687921,1.6031495644416094,1.2117616885007294,0.58046140777246402,1.5909273396101546,1.0697955150908101,0.8475865221761848,0.86965753408671631,1.0360697273171653,0.92948858575880999,1.2270662265301611,1.6176670178555095,1.04880243850013,0.93462350068670652,1.5297005668033943,1.3492785421852633,1.1224567207178935,1.3963826122160243,1.1509826516667701,1.0525785600730324,1.5676643082912483,1.3637269952032667,0.63139155288493787,1.1730501920227674,0.57696659389476523,0.80630491302620733,1.1569598174968005,0.8714372784336023,1.2161857519285757,1.464018931963917,1.0813793986484759,0.96084668751429514,1.1446366932228986,1.1898233758195271,1.0500261750760711,0.53883502749639056,1.3330585432907738,1.2444925351362812,1.0775474936636575,1.4403737206816798,1.088196993552685,1.1183426492940751,0.96022496858828077,1.3420463832724046,0.59489221228198574,1.0022804558016709,0.86854419808184669,1.0248700228156478,1.1438872155929587,1.0449317754848793,1.2093054240628787,0.76316947415183312,1.0113786569486163,0.71379109550814035,1.3480835473466195,-1.1253576866979171,1.2336897705508447,1.1183323303750907,1.0312303705783497,1.190198901169786,0.92894146248242493,1.0936121477746592,1.313819298311149,1.2157752410221154,1.0848436139074669,0.89239525536392372,1.1946095537686641,-1.2589924979373246,0.92777222098515266,1.1148638750713151,1.0332643993587478,1.0600275288689482,0.83728160076120628,1.1659889869613826,0.8980107058535578,1.0792606234034126,1.0190150058694354,0.64044385998403808,1.0295566398546012,1.0148089869606551,0.92746654749028623,0.99954171021594573,0.87448452606850469,1.0180181635962808,1.0834748301671109],"FDR":[3.876948978149917e-11,1.093115199221392e-10,2.8841116984619091e-10,8.4325708843807125e-10,8.4325708843807125e-10,1.1809595713247864e-09,1.1809595713247864e-09,5.7437675143739617e-09,1.5818528351003574e-08,1.7017521342396204e-08,2.8253664895804547e-08,3.9585150408260889e-08,4.8370969503687874e-08,5.5663279159584303e-08,6.5612827694999825e-08,1.5345456007725617e-07,1.5345456007725617e-07,5.3688021966115551e-07,5.7192703686083546e-07,5.7192703686083546e-07,9.2381021833237632e-07,1.0046129165171557e-06,1.2047151010205652e-06,2.3726138842318925e-06,2.3726138842318925e-06,3.0305439827121583e-06,4.3429388703731902e-06,6.3335474895770965e-06,7.1975210094040267e-06,7.7312121136305821e-06,7.7312121136305821e-06,9.8718547839187253e-06,1.1603638471248065e-05,1.2064942913225906e-05,1.8144920376692039e-05,1.8144920376692039e-05,2.0647039877275954e-05,2.8553566704915754e-05,3.0897956631962846e-05,3.0897956631962846e-05,3.0897956631962846e-05,5.2081725733772101e-05,6.1776236610371459e-05,6.1916531531763463e-05,6.3435765213779391e-05,6.5247899068777483e-05,7.2720364035463336e-05,8.1863242777907179e-05,9.5347366060175817e-05,0.00010197449193482469,0.00013411605758245092,0.00013411605758245092,0.00013853641533285467,0.00014701181244540179,0.00014701181244540179,0.00015148395747235842,0.00017844475586667821,0.00017844475586667821,0.00017982947419018969,0.00017982947419018969,0.00023625103933115952,0.00024677677145333761,0.00027189853880078564,0.00027853558039022335,0.00030733351009800639,0.00030733351009800639,0.00032574762154835715,0.00035597693148759738,0.00045233886284333985,0.00045732072862219561,0.00045989282196790893,0.00046620286463466002,0.00054812978097010343,0.00059380113791322893,0.00059380113791322893,0.00059380113791322893,0.00059380113791322893,0.00059380113791322893,0.00059990768615503524,0.00060411234175063839,0.00060411234175063839,0.00063584540945047722,0.00063844711501278575,0.00072798147842115756,0.00081709773437586993,0.00083159243409152708,0.00088868007551784825,0.00091201729700881033,0.0010914799122978258,0.0011295884040005027,0.0011633776348801991,0.0011699111696523087,0.0012300718478905283,0.0012493913351534684,0.0013845120010564644,0.0014109489741894734,0.0014236540355576346,0.0015231909897087611,0.0015650361300308391,0.0015650361300308391,0.0017675111696935854,0.0018319300705149258,0.0018319300705149258,0.0019150823143293313,0.0019332028060420035,0.0019332028060420035,0.0021530680270087678,0.0021530680270087678,0.0021530680270087678,0.002173480157826617,0.0022162300708893044,0.002223793376334009,0.0026326976880768076,0.0026871464412058736,0.0027751761571576834,0.0027751761571576834,0.0031780863312426463,0.0032070469799980409,0.0032209009153997144,0.0032223804697354497,0.0036951900229099917,0.0042020450636621612,0.0042173749020268823,0.0042173749020268823,0.0042603421389991293,0.0046266080670651416,0.0049218383521452526,0.0049218383521452526,0.0049438435232438196,0.0052236977265042008,0.0055888668480737142,0.0057664247403200865,0.0057664247403200865,0.0066433467950057744,0.0066551786501101926,0.0069350951784818487,0.0071581060769012623,0.007230166942641736,0.0074076701463877161,0.0074076701463877161,0.0077479111893214098,0.007808177575744148,0.0079162907808166417,0.0095566191266924401,0.0097403284547624338,0.010088113323741066,0.010179060458525395,0.011157350473198973,0.011157350473198973,0.011902537722913404,0.011916310703214493,0.012388562767893237,0.012388562767893237,0.013829751159558401,0.015159285071713359,0.015159285071713359,0.015168949435074722,0.015618959778779746,0.015920957192535105,0.018109280014412567,0.018455392349815494,0.018455392349815494,0.018455392349815494,0.020291391835040008,0.020291391835040008,0.020586526017733447,0.023248184361831729,0.023248184361831729,0.023281905696563212,0.024680337411967214,0.024680337411967214,0.02536781283753765,0.025398370741856639,0.025655173472694293,0.026203004639460292,0.026617006038748042,0.02722181874177948,0.027392426496949424,0.028764589615869397,0.030524069525543499,0.032764175752184779,0.033720414333067224,0.034288244189418067,0.035668220680480478,0.036758573766220694,0.039653674861559275,0.039686730824160639,0.041888171698861901,0.042025852722269857,0.04270893975903578,0.04333536443578128,0.043731318106121637,0.043796018668964723,0.043796018668964723,0.045298724931315364,0.048811327988741653,0.049329182295280882,0.051665995674636117,0.056139744495638556,0.056810514054464578],"Macs2 Peak":[true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true],"Region":["Promoter (-1500/+500bp)","Intron","Upstream Promoter (<5kb)","Upstream Promoter (<5kb)","Intron","Intergenic (<10kb)","Intergenic (>10kb)","Upstream Promoter (<5kb)","Promoter (-1500/+500bp)","Intergenic (>10kb)","Intron","Intron","Upstream Promoter (<5kb)","Upstream Promoter (<5kb)","Intron","Intron","Intron","Intergenic (>10kb)","Intron","Promoter (-1500/+500bp)","Intergenic (>10kb)","Exon","Intergenic (<10kb)","Intron","Intergenic (<10kb)","Intron","Intergenic (>10kb)","Intron","Intron","Intron","Intergenic (>10kb)","Intergenic (>10kb)","Intergenic (<10kb)","Intron","Intron","Intergenic (<10kb)","Intron","Intron","Exon","Promoter (-1500/+500bp)","Intron","Intron","Intergenic (>10kb)","Intron","Intergenic (<10kb)","Intergenic (>10kb)","Intron","Intergenic (>10kb)","Intron","Intron","Intergenic (>10kb)","Intergenic (>10kb)","Intergenic (>10kb)","Intergenic (<10kb)","Intergenic (<10kb)","Intron","Intergenic (>10kb)","Intergenic (>10kb)","Intergenic (>10kb)","Intergenic (>10kb)","Intron","Upstream Promoter (<5kb)","Intergenic (>10kb)","Intron","Exon","Exon","Intron","Intron","Intergenic (>10kb)","Intron","Intron","Intron","Intron","Promoter (-1500/+500bp)","Upstream Promoter (<5kb)","Intron","Intron","Intron","Promoter (-1500/+500bp)","Intron","Intron","Intergenic (<10kb)","Upstream Promoter (<5kb)","Intron","Intron","Intergenic (>10kb)","Intron","Intergenic (>10kb)","Intron","Intergenic (>10kb)","Intron","Intergenic (>10kb)","Intergenic (>10kb)","Intron","Intron","Intron","Promoter (-1500/+500bp)","Intron","Intron","Intron","Intergenic (<10kb)","Intron","Intron","Intron","Intron","Intergenic (>10kb)","Upstream Promoter (<5kb)","Intron","Intron","Promoter (-1500/+500bp)","Intron","Upstream Promoter (<5kb)","Intergenic (<10kb)","Intron","Promoter (-1500/+500bp)","Promoter (-1500/+500bp)","Intron","Intron","Intron","Intron","Upstream Promoter (<5kb)","Intergenic (>10kb)","Intergenic (>10kb)","Intron","Intergenic (>10kb)","Intron","Exon","Intron","Intergenic (<10kb)","Intergenic (<10kb)","Intergenic (>10kb)","Intergenic (<10kb)","Intergenic (>10kb)","Intergenic (>10kb)","Exon","Intron","Intron","Intergenic (>10kb)","Intron","Intron","Promoter (-1500/+500bp)","Intron","Intron","Intergenic (<10kb)","Intron","Intron","Intergenic (>10kb)","Promoter (-1500/+500bp)","Intron","Intergenic (>10kb)","Intron","Promoter (-1500/+500bp)","Intron","Intron","Intergenic (>10kb)","Upstream Promoter (<5kb)","Intergenic (<10kb)","Intergenic (>10kb)","Intergenic (<10kb)","Upstream Promoter (<5kb)","Upstream Promoter (<5kb)","Intron","Intergenic (>10kb)","Intergenic (>10kb)","Intergenic (>10kb)","Intron","Intron","Intergenic (<10kb)","Intergenic (>10kb)","Promoter (-1500/+500bp)","Intron","Intron","Upstream Promoter (<5kb)","Intron","Intergenic (<10kb)","Exon","Intron","Promoter (-1500/+500bp)","Intergenic (>10kb)","Intron","Intron","Intron","Intron","Intergenic (>10kb)","Intron","Intron","Promoter (-1500/+500bp)","Intron","Intergenic (<10kb)","Upstream Promoter (<5kb)","Intron","Intron","Intergenic (>10kb)","Exon","Intergenic (>10kb)","Intron","Intron","Intergenic (>10kb)","Intergenic (<10kb)","Intergenic (>10kb)"],"Genes":["LINC00867","APBB2","PPIF","FAM214A","ZNRF1","AC092812.1","AC078864.2","ADAMTS8","SEC14L2; AC004832.3; RNF215","LINC00567","AL807740.1","RERG","RNF215","AC008537.1","AJ009632.2","IQCM","ZNF521","AC021035.1","ATAD2","ZBTB16","AC092952.1","LINC01856","AC143336.1","C2orf76","PPIF","AC091180.6","APIP","AL513164.1","TENM3-AS1","PRLR","PAICSP7","AC108515.1","RNU7-92P","KDM4B","PHACTR1","AQP3","LINC01911","BBS9","AC004947.1","PTPRN2","TUSC3","ACSM5P1","AC093159.1","ELP3","MRFAP1","CALM2P1","AC068633.1","AC011592.1","AC126121.3","LCLAT1","LSAMP","TAT-AS1","RN7SL658P","AL359999.1","FITM2","SGK3","SILC1","BDH2P1","snoU13","RPL37P4","AGR3","DENND5B","OLFM4","LINC00536","UGDH-AS1; AC021148.3","CTPS2","LINC01924","LPP","AC012391.1","CCDC57","RARB","SP2; SP2-AS1","GAA","TTC21A","CARTPT","AL138767.3; AL138767.1","ANKRD29","OPHN1","CRAMP1; AL031708.1","RYR2","RCL1","NHLRC2","C5","PDE10A","LINC01090","MYB","AL033523.1","AL591034.1","LINC02694; AC013652.1","APIP","MACO1","LINC01728","","HOMER2","GNG7","SH3TC2","CRAT; AL158151.2","LINC02457","AC008014.1","LINC01588","AQP3","LINC01828; LINC01829","SATB1-AS1","AC023590.1; SAMD12","PADI4","HMGB3P18","AC021087.5; PDCD6; AHRR","AC004593.2; CHN2","MAPT","VWA2","LINC01643","AL137786.2","AC092473.1","SOX2-OT","AL391421.1; DLG5","ASCC2","ILRUN","SYBU","GREB1L","MGAM","PDZD2","CDC14C","","AC055854.1; PEBP4","MIR720","AC036214.3; TPD52","KIAA0825","ABHD2","LINC02685","U3","","CST9","AL354984.3","MIR4686","C17orf100; SLC13A5","AUTS2","AC002451.1","SYTL4","TG","RAD51B","AC010280.1","AC008674.1","MREG","AC079031.2","VWA8","AC009262.1","","CHCHD5; AC012442.3; AC012442.2","CTCF","AC068295.1","AC011317.1","CDC14A; AL589990.1","FANCC","SCNN1B","","TIFA","TSPYL5","NDUFS5P6","RNU6-54P","SLC9A3R1","CCNJL","SDC4","PABPC5-AS1","AC108681.1","LINC02026","AL953883.1","DEC1; AL691420.1","AL136090.1","","AC108078.1; UGT2B24P","PGR-AS1","MARK4; PPP1R37","AC093714.1","PLAC1","SEC22C","AC007610.7","AC005821.1; MARCHF10","DECR1; NBN","AL356807.1","PKHD1","ACACA","WWC3","LINC02224","MTCO3P30","AC005821.1; MARCHF10","MACROD2","RCL1","SUB1","AC005324.6","BCAS4","LIMCH1","C5orf66","","AL117341.1; SGMS1","AC026462.4","PDLIM5","AFF3","H4C8","AC016583.2","AC005906.3"]},"columns":[{"id":"Range","name":"Range","type":"character","cell":["chr10: 120116131-120116580","chr4: 41079691-41080140","chr10: 81102391-81102870","chr15: 52948501-52948980","chr16: 75087601-75088080","chr1: 96354541-96354990","chr12: 52520341-52520640","chr11: 130299901-130300230","chr22: 30804721-30805110","chr13: 111490111-111490440","chrX: 20489491-20490000","chr12: 15426871-15427230","chr22: 30788911-30789330","chr19: 41360971-41361420","chr21: 16959061-16959690","chr4: 150780211-150780660","chr18: 22748521-22749000","chr10: 113150221-113150640","chr8: 124417831-124418040","chr11: 114050011-114050340","chr3: 190525501-190526010","chr2: 130703131-130703610","chr5: 71044111-71044830","chr2: 120095461-120095610","chr10: 81101911-81102240","chr17: 47468401-47468760","chr11: 34844761-34845090","chr6: 148579861-148580340","chr4: 182878051-182878710","chr5: 35205781-35206110","chrX: 93855181-93855810","chr4: 97953721-97954020","chr20: 46540801-46541310","chr19: 4991521-4992450","chr6: 12844471-12845010","chr9: 33437431-33437610","chr2: 147629551-147629940","chr7: 33872461-33873000","chr7: 26598151-26598480","chr7: 157413451-157414260","chr8: 15525301-15525750","chr16: 20599021-20599500","chr2: 62652151-62652300","chr8: 28005151-28005630","chr4: 6634801-6635430","chr17: 68267461-68268180","chr3: 117850051-117850410","chr3: 144174931-144175350","chr3: 59527171-59527440","chr2: 30827671-30828060","chr3: 116952781-116953110","chr16: 71627821-71628150","chrX: 16579861-16580160","chr11: 34771711-34772130","chr20: 42930901-42931260","chr8: 67626001-67626990","chr2: 6050131-6050550","chr6: 99658621-99659100","chr2: 75548911-75549240","chr21: 20939551-20939640","chr7: 16920121-16920660","chr12: 31612891-31613310","chr13: 53559721-53560050","chr8: 117010381-117010950","chr4: 39545731-39546060","chrX: 16607371-16607820","chr18: 61835821-61836060","chr3: 188000971-188001420","chr10: 124987321-124987770","chr17: 80071981-80072100","chr3: 25302121-25302360","chr17: 45983131-45983550","chr17: 78076771-78077040","chr3: 39172021-39172380","chr5: 71010751-71011140","chr10: 89407861-89407950","chr18: 21183991-21184350","chrX: 67583791-67584240","chr16: 1680841-1681230","chr1: 237301831-237302100","chr9: 4866151-4866840","chr10: 115681561-115682250","chr9: 123748081-123748380","chr6: 165980731-165981090","chr2: 188851081-188851260","chr6: 135467671-135468150","chr6: 3670141-3670440","chr6: 53333911-53334330","chr15: 39187441-39187680","chr11: 34847911-34848390","chr1: 25802611-25803090","chr20: 42509311-42509880","chr1: 163586491-163587150","chr15: 83588011-83588460","chr19: 2576161-2576580","chr5: 148398811-148398990","chr9: 131869651-131869890","chr12: 116931901-116932410","chr12: 46887421-46887990","chr14: 50437831-50438160","chr9: 33431461-33431610","chr2: 67392751-67393140","chr3: 18531721-18531990","chr8: 119395321-119395860","chr1: 17641171-17641710","chr6: 122138251-122138820","chr5: 301981-302850","chr7: 29495671-29496060","chr17: 43978321-43978980","chr10: 115998901-115999410","chr22: 34433371-34433700","chr14: 97127101-97127400","chrX: 40378291-40378680","chr3: 181019101-181019520","chr10: 79629661-79630260","chr22: 30212821-30213120","chr6: 34566721-34566840","chr8: 110605471-110605830","chr18: 18909961-18910650","chr7: 141701491-141701670","chr5: 32094181-32094690","chr7: 49007641-49008210","chr2: 193302751-193303350","chr8: 22592311-22592850","chr3: 164019811-164020230","chr8: 80990281-80990730","chr5: 93739351-93739440","chr15: 89672971-89673360","chr11: 45007321-45007830","chr8: 98361241-98361450","chr1: 163596421-163596660","chr20: 23576761-23577150","chr20: 56563051-56563290","chr11: 2214481-2214840","chr17: 6589531-6589950","chr7: 69278101-69278190","chr7: 95179501-95179740","chrX: 100000711-100001460","chr8: 133911241-133911540","chr14: 69036601-69037260","chr5: 67830031-67830660","chr5: 173193421-173193780","chr2: 216830971-216831420","chr12: 133042981-133043610","chr13: 42355501-42355590","chr7: 34275811-34276080","chr3: 104330071-104330550","chr2: 113346121-113346420","chr16: 67628611-67628970","chr3: 187634881-187635300","chr3: 150454861-150455610","chr1: 100951471-100951800","chr9: 98044591-98044860","chr16: 23373961-23374290","chr1: 106819171-106819290","chr4: 113204491-113204820","chr8: 98278651-98278950","chr8: 68722801-68723190","chr13: 71037841-71038110","chr17: 72754111-72754860","chr5: 159713881-159714300","chr20: 43968991-43969470","chrX: 90632731-90633120","chr3: 187498801-187499070","chr3: 193640131-193640400","chr9: 38993851-38994060","chr9: 118088701-118089150","chr20: 19110781-19111140","chr9: 121766881-121767570","chr4: 70274161-70274280","chr11: 101049151-101049630","chr19: 45622831-45623010","chr7: 115318081-115318410","chrX: 133704511-133704900","chr3: 42586081-42586530","chr16: 50163361-50163780","chr17: 60803191-60803520","chr8: 91013911-91014090","chr14: 84427621-84427770","chr6: 51877951-51878190","chr17: 35633461-35633880","chrX: 10020031-10020240","chr5: 44589961-44590110","chr9: 31545361-31545720","chr17: 60795691-60795810","chr20: 15421051-15421380","chr9: 4859011-4859430","chr5: 32563621-32564070","chr17: 15658141-15658410","chr20: 49459831-49460100","chr4: 41477491-41477790","chr5: 134544211-134544570","chr2: 227203441-227204040","chr10: 52233661-52234020","chr16: 52790791-52791030","chr4: 95481061-95481450","chr2: 100684201-100684500","chr6: 26296141-26296530","chr5: 53951971-53952240","chr12: 5186131-5186250"],"minWidth":120},{"id":"Width (bp)","name":"Width (bp)","type":"numeric","maxWidth":60,"style":[{"background":"#9FBCA4","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#9FBCA4","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#A4BEA6","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#A4BEA6","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#A4BEA6","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#9FBCA4","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#77AF98","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#7FB19A","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#8FB69F","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#7FB19A","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#A9C1A9","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#87B49D","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#97B9A1","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#9FBCA4","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#BFCDB3","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#9FBCA4","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#A4BEA6","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#97B9A1","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#5CA793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#7FB19A","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#A9C1A9","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#A4BEA6","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#CFD6BB","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#41A293","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#7FB19A","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#87B49D","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#7FB19A","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#A4BEA6","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#C4D0B6","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#7FB19A","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#BFCDB3","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#77AF98","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#A9C1A9","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#EDE4C5","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#AFC4AC","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#4FA593","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#8FB69F","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#AFC4AC","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#7FB19A","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#DFDFC2","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#9FBCA4","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#A4BEA6","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#41A293","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#A4BEA6","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#BFCDB3","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#CFD6BB","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#87B49D","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#97B9A1","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#6FAC96","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#8FB69F","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#7FB19A","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#7FB19A","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#77AF98","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#97B9A1","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#87B49D","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#E9DCBD","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#97B9A1","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#A4BEA6","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#7FB19A","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#279D93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#AFC4AC","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#97B9A1","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#7FB19A","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#B4C7AE","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#7FB19A","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#9FBCA4","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#67AA94","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#9FBCA4","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#9FBCA4","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#349F93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#67AA94","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#97B9A1","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#6FAC96","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#87B49D","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#8FB69F","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#279D93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#87B49D","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#9FBCA4","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#8FB69F","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#6FAC96","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#C9D3B9","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#C9D3B9","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#77AF98","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#87B49D","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#4FA593","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#A4BEA6","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#77AF98","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#97B9A1","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#67AA94","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#A4BEA6","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#A4BEA6","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#B4C7AE","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#C4D0B6","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#9FBCA4","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#97B9A1","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#4FA593","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#67AA94","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#A9C1A9","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#B4C7AE","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#7FB19A","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#41A293","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#8FB69F","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#6FAC96","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#AFC4AC","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#AFC4AC","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#B4C7AE","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#E9E5C6","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#8FB69F","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#C4D0B6","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#A9C1A9","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#7FB19A","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#77AF98","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#8FB69F","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#97B9A1","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#B9CAB1","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#77AF98","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#349F93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#87B49D","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#C9D3B9","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#4FA593","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#A9C1A9","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#B4C7AE","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#B9CAB1","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#AFC4AC","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#97B9A1","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#9FBCA4","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#279D93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#8FB69F","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#A9C1A9","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#5CA793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#67AA94","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#8FB69F","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#67AA94","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#87B49D","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#97B9A1","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#279D93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#67AA94","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#D4D9BD","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#77AF98","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#C4D0B6","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#BFCDB3","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#87B49D","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#9FBCA4","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#BFCDB3","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#279D93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#6FAC96","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#A4BEA6","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#77AF98","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#87B49D","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#97B9A1","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#D4D9BD","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#7FB19A","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#6FAC96","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#7FB19A","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#349F93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#7FB19A","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#77AF98","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#8FB69F","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#6FAC96","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#D4D9BD","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#97B9A1","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#A4BEA6","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#8FB69F","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#6FAC96","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#6FAC96","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#5CA793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#9FBCA4","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#87B49D","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#C9D3B9","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#349F93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#A4BEA6","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#4FA593","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#7FB19A","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#8FB69F","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#9FBCA4","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#97B9A1","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#7FB19A","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#4FA593","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#41A293","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#67AA94","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#97B9A1","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#5CA793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#41A293","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#87B49D","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#349F93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#7FB19A","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#97B9A1","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#9FBCA4","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#6FAC96","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#6FAC96","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#77AF98","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#87B49D","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#B9CAB1","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#87B49D","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#67AA94","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#8FB69F","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#77AF98","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#8FB69F","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#6FAC96","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#349F93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"}]},{"id":"AveExpr","name":"AveExpr","type":"numeric","cell":["1.39","1.22","1.76","2.35","0.96","0.44","0","1.22","1.06","0.11","2.15","1.46","1.61","1.28","3.13","1.93","2.31","0.93","-0.24","0.25","0.42","1.61","3.12","-0.07","1.05","0.92","0.76","0.99","2.45","0.63","1.67","-0.02","1.84","2.48","2.55","-0.38","0.74","2.59","0.58","2.83","1.54","1.84","0.03","1.85","2.4","3.44","0.93","1.67","0.12","0.49","0.46","0.07","0.75","1.08","0.84","3.98","0.81","1.32","0.25","-0.11","3.18","1.62","0.6","2.42","0.33","2.03","-0.05","1.04","1.59","-0.44","0.14","0.83","0.8","0.39","1.35","0.1","1.43","2.1","1.38","0.15","3.28","3.55","0.63","0.65","-0.19","1.71","0.31","1.04","0.31","1.84","1.39","2.94","3.1","0.8","1.55","-0.35","0.16","2.14","2.41","0.94","-0.3","0.94","0.28","1.58","2.11","2.58","0.24","1.07","3.12","1.88","0.81","0.71","0.9","1.94","2.23","0.31","-0.25","0.63","3.51","-0.18","1.2","2.15","2.04","1.33","1.74","0.47","-0.24","1.23","1.65","-0.18","0.11","0.89","0.15","0.77","1.12","-0.33","0.01","3.07","0.34","3.4","2.26","0.86","2.02","0.32","-0.28","0.68","1.38","0.54","0.53","0.8","3.59","0.05","0.12","0.49","-0.25","0.55","0.74","1.23","-0.08","3.1","1.29","1.72","0.96","0.42","0.61","0.15","2.1","1.09","2.62","-0.22","0.78","-0.08","0.34","0.98","0.12","1.31","0.34","-0.18","-0.03","0.34","1.06","0.05","-0.18","1.09","0.29","0.48","0.47","1.54","0.02","1.04","0.2","0.69","2.53","0.73","0.44","1.05","0.66","1.07","0.43","0.15"],"maxWidth":66,"style":[{"background":"#C2CFB5"},{"background":"#B7C9B0"},{"background":"#D9DCBF"},{"background":"#E9DCBD"},{"background":"#A7C0A8"},{"background":"#7AB099"},{"background":"#44A293"},{"background":"#B7C9B0"},{"background":"#ADC3AB"},{"background":"#55A693"},{"background":"#EDE6C6"},{"background":"#C7D2B8"},{"background":"#D0D7BB"},{"background":"#BBCBB2"},{"background":"#DBB69A"},{"background":"#E4E3C4"},{"background":"#EADFBF"},{"background":"#A5BFA7"},{"background":"#1F9B93"},{"background":"#68AA94"},{"background":"#78AF99"},{"background":"#D0D7BC"},{"background":"#DCB79B"},{"background":"#39A093"},{"background":"#ACC3AA"},{"background":"#A4BFA6"},{"background":"#98BAA2"},{"background":"#A9C1A9"},{"background":"#E7D8B9"},{"background":"#8CB69E"},{"background":"#D4D9BD"},{"background":"#42A293"},{"background":"#DEDFC2"},{"background":"#E6D7B8"},{"background":"#E4D3B4"},{"background":"#0A9793"},{"background":"#96B9A1"},{"background":"#E4D1B3"},{"background":"#87B49D"},{"background":"#DFC5A8"},{"background":"#CBD4BA"},{"background":"#DFDFC2"},{"background":"#49A393"},{"background":"#DFE0C2"},{"background":"#E8DABB"},{"background":"#D8A68D"},{"background":"#A5BFA7"},{"background":"#D4D9BD"},{"background":"#58A693"},{"background":"#7FB19A"},{"background":"#7CB09A"},{"background":"#4FA593"},{"background":"#97B9A1"},{"background":"#AEC4AB"},{"background":"#9FBCA4"},{"background":"#D5887C"},{"background":"#9DBBA3"},{"background":"#BECDB3"},{"background":"#68AA94"},{"background":"#349F93"},{"background":"#DBB498"},{"background":"#D0D7BC"},{"background":"#89B59D"},{"background":"#E7D9BA"},{"background":"#70AD96"},{"background":"#EAE6C7"},{"background":"#3DA193"},{"background":"#ACC3AA"},{"background":"#CFD6BB"},{"background":"#009593"},{"background":"#5AA793"},{"background":"#9FBCA4"},{"background":"#9CBBA3"},{"background":"#75AE98"},{"background":"#C0CEB4"},{"background":"#55A693"},{"background":"#C4D0B6"},{"background":"#EEE8C8"},{"background":"#C2CFB5"},{"background":"#5DA793"},{"background":"#DAAF93"},{"background":"#D7A08A"},{"background":"#8CB69E"},{"background":"#8EB69F"},{"background":"#289D93"},{"background":"#D6DABE"},{"background":"#6EAC96"},{"background":"#ACC3AA"},{"background":"#6EAC96"},{"background":"#DFDFC2"},{"background":"#C2CFB5"},{"background":"#DEC0A3"},{"background":"#DCB89C"},{"background":"#9CBBA3"},{"background":"#CCD5BA"},{"background":"#0E9893"},{"background":"#5EA893"},{"background":"#EEE7C7"},{"background":"#E8DABB"},{"background":"#A6BFA7"},{"background":"#169993"},{"background":"#A6BFA7"},{"background":"#6BAB95"},{"background":"#CED6BB"},{"background":"#EEE8C8"},{"background":"#E4D2B3"},{"background":"#67AA94"},{"background":"#AEC4AB"},{"background":"#DCB79B"},{"background":"#E1E1C3"},{"background":"#9DBBA3"},{"background":"#93B8A0"},{"background":"#A3BEA6"},{"background":"#E5E3C4"},{"background":"#ECE2C3"},{"background":"#6EAC96"},{"background":"#1D9B93"},{"background":"#8CB69E"},{"background":"#D8A38B"},{"background":"#289D93"},{"background":"#B6C9AF"},{"background":"#EDE6C6"},{"background":"#EBE6C7"},{"background":"#BECDB3"},{"background":"#D8DCBF"},{"background":"#7DB19A"},{"background":"#209B93"},{"background":"#B8C9B0"},{"background":"#D2D8BD"},{"background":"#299D93"},{"background":"#56A693"},{"background":"#A2BEA5"},{"background":"#5BA793"},{"background":"#99BAA2"},{"background":"#B1C6AD"},{"background":"#119893"},{"background":"#46A393"},{"background":"#DCB99D"},{"background":"#70AD96"},{"background":"#D8A98E"},{"background":"#EBE1C1"},{"background":"#A0BDA4"},{"background":"#EAE6C6"},{"background":"#6EAC96"},{"background":"#1A9A93"},{"background":"#91B7A0"},{"background":"#C2CFB5"},{"background":"#84B39C"},{"background":"#82B39B"},{"background":"#9CBBA3"},{"background":"#D79E88"},{"background":"#4DA493"},{"background":"#57A693"},{"background":"#7FB29B"},{"background":"#1D9B93"},{"background":"#85B39C"},{"background":"#97B9A1"},{"background":"#B8CAB0"},{"background":"#38A093"},{"background":"#DCB89C"},{"background":"#BCCCB2"},{"background":"#D7DBBF"},{"background":"#A7C0A8"},{"background":"#78AF99"},{"background":"#8AB59E"},{"background":"#5CA793"},{"background":"#EEE8C8"},{"background":"#AFC5AC"},{"background":"#E3D0B1"},{"background":"#239C93"},{"background":"#9ABAA2"},{"background":"#38A093"},{"background":"#70AD96"},{"background":"#A8C1A8"},{"background":"#57A693"},{"background":"#BDCCB3"},{"background":"#71AD96"},{"background":"#289D93"},{"background":"#40A293"},{"background":"#71AD97"},{"background":"#ADC4AB"},{"background":"#4DA493"},{"background":"#299D93"},{"background":"#AFC5AC"},{"background":"#6CAC95"},{"background":"#7DB19A"},{"background":"#7DB19A"},{"background":"#CBD4BA"},{"background":"#48A393"},{"background":"#ACC3AA"},{"background":"#63A993"},{"background":"#92B7A0"},{"background":"#E5D4B5"},{"background":"#96B9A1"},{"background":"#7AB099"},{"background":"#ACC3AA"},{"background":"#8EB69F"},{"background":"#AEC4AB"},{"background":"#79B099"},{"background":"#5CA793"}]},{"id":"logFC","name":"logFC","type":"numeric","cell":["2.19","2.33","1.89","1.51","2.21","2.44","2.81","1.95","1.96","2.5","1.45","1.7","1.73","1.8","1.05","1.44","1.26","1.79","2.49","2.16","1.96","1.48","0.94","2.23","1.61","1.63","1.7","1.63","1.09","1.69","1.31","2.04","1.23","1.1","1.01","2.27","1.6","0.96","1.68","0.89","1.3","1.17","1.98","1.14","0.99","0.7","1.41","1.21","1.73","1.59","1.6","1.75","1.4","1.29","1.47","0.58","1.44","1.29","1.58","1.85","0.74","1.18","1.49","0.98","1.53","1.02","1.8","1.31","1.11","2.01","1.58","1.32","1.32","1.47","1.2","1.59","1.17","0.96","1.17","1.51","0.68","0.63","1.33","1.37","1.71","1.04","1.47","1.21","1.43","1.04","1.18","0.73","0.71","1.24","1.06","1.77","1.39","0.93","0.84","1.19","1.71","1.32","1.41","1.03","0.92","0.78","1.35","1.14","0.67","0.98","1.21","1.23","1.18","0.92","0.86","1.29","1.6","1.21","0.58","1.59","1.07","0.85","0.87","1.04","0.93","1.23","1.62","1.05","0.93","1.53","1.35","1.12","1.4","1.15","1.05","1.57","1.36","0.63","1.17","0.58","0.81","1.16","0.87","1.22","1.46","1.08","0.96","1.14","1.19","1.05","0.54","1.33","1.24","1.08","1.44","1.09","1.12","0.96","1.34","0.59","1","0.87","1.02","1.14","1.04","1.21","0.76","1.01","0.71","1.35","-1.13","1.23","1.12","1.03","1.19","0.93","1.09","1.31","1.22","1.08","0.89","1.19","-1.26","0.93","1.11","1.03","1.06","0.84","1.17","0.9","1.08","1.02","0.64","1.03","1.01","0.93","1","0.87","1.02","1.08"],"maxWidth":66,"style":[{"background":"#D53851"},{"background":"#D32C46"},{"background":"#DB5369"},{"background":"#E27687"},{"background":"#D5364F"},{"background":"#D0213D"},{"background":"#CA0020"},{"background":"#DA4E64"},{"background":"#DA4D63"},{"background":"#CF1C38"},{"background":"#E37B8C"},{"background":"#DF6578"},{"background":"#DE6276"},{"background":"#DD5B70"},{"background":"#EBA0AB"},{"background":"#E37C8C"},{"background":"#E78D9B"},{"background":"#DD5C70"},{"background":"#D01D39"},{"background":"#D63B53"},{"background":"#DA4D63"},{"background":"#E37889"},{"background":"#EDA9B4"},{"background":"#D4344D"},{"background":"#E06D7F"},{"background":"#E06B7D"},{"background":"#DF6578"},{"background":"#E06A7D"},{"background":"#EA9CA8"},{"background":"#DF6578"},{"background":"#E68796"},{"background":"#D8465D"},{"background":"#E78F9D"},{"background":"#EA9BA7"},{"background":"#EBA3AE"},{"background":"#D4314B"},{"background":"#E06D80"},{"background":"#ECA7B2"},{"background":"#DF6679"},{"background":"#EEAEB8"},{"background":"#E68897"},{"background":"#E995A2"},{"background":"#D94B61"},{"background":"#E998A4"},{"background":"#ECA5B0"},{"background":"#F1BFC7"},{"background":"#E47E8E"},{"background":"#E8919F"},{"background":"#DE6175"},{"background":"#E06E80"},{"background":"#E06D7F"},{"background":"#DD5F73"},{"background":"#E48090"},{"background":"#E68998"},{"background":"#E3798A"},{"background":"#F4CAD1"},{"background":"#E37C8D"},{"background":"#E68A98"},{"background":"#E16F81"},{"background":"#DC576C"},{"background":"#F1BBC4"},{"background":"#E894A1"},{"background":"#E37889"},{"background":"#ECA5B1"},{"background":"#E27485"},{"background":"#EBA2AE"},{"background":"#DD5C70"},{"background":"#E68897"},{"background":"#EA9AA7"},{"background":"#D9485F"},{"background":"#E16F81"},{"background":"#E68796"},{"background":"#E68796"},{"background":"#E3798A"},{"background":"#E8929F"},{"background":"#E16E80"},{"background":"#E894A2"},{"background":"#ECA7B2"},{"background":"#E894A2"},{"background":"#E27687"},{"background":"#F2C1C9"},{"background":"#F3C6CD"},{"background":"#E58695"},{"background":"#E58292"},{"background":"#DE6377"},{"background":"#EBA0AC"},{"background":"#E3798A"},{"background":"#E8919F"},{"background":"#E37D8D"},{"background":"#EBA0AC"},{"background":"#E894A1"},{"background":"#F1BCC5"},{"background":"#F1BEC6"},{"background":"#E78E9C"},{"background":"#EA9EAA"},{"background":"#DD5E72"},{"background":"#E48090"},{"background":"#EDABB5"},{"background":"#EFB2BC"},{"background":"#E892A0"},{"background":"#DE6477"},{"background":"#E68796"},{"background":"#E47E8E"},{"background":"#EBA1AD"},{"background":"#EDABB6"},{"background":"#F0B8C1"},{"background":"#E58493"},{"background":"#E997A4"},{"background":"#F2C1C9"},{"background":"#ECA5B0"},{"background":"#E8919F"},{"background":"#E78F9D"},{"background":"#E893A1"},{"background":"#EDABB6"},{"background":"#EEB0BA"},{"background":"#E68998"},{"background":"#E06D7F"},{"background":"#E8919E"},{"background":"#F4CAD0"},{"background":"#E16E80"},{"background":"#EA9EAA"},{"background":"#EFB2BB"},{"background":"#EEB0BA"},{"background":"#EBA1AC"},{"background":"#EDAAB5"},{"background":"#E78F9D"},{"background":"#E06C7E"},{"background":"#EB9FAB"},{"background":"#EDAAB4"},{"background":"#E27485"},{"background":"#E58494"},{"background":"#E999A6"},{"background":"#E48090"},{"background":"#E996A3"},{"background":"#EB9FAB"},{"background":"#E17082"},{"background":"#E58392"},{"background":"#F3C5CC"},{"background":"#E894A1"},{"background":"#F4CAD1"},{"background":"#EFB5BF"},{"background":"#E996A3"},{"background":"#EEAFB9"},{"background":"#E8909E"},{"background":"#E37A8A"},{"background":"#EA9CA9"},{"background":"#ECA7B2"},{"background":"#E997A4"},{"background":"#E893A0"},{"background":"#EB9FAB"},{"background":"#F4CED4"},{"background":"#E58695"},{"background":"#E78E9C"},{"background":"#EA9DA9"},{"background":"#E37C8C"},{"background":"#EA9CA8"},{"background":"#E999A6"},{"background":"#ECA7B2"},{"background":"#E58594"},{"background":"#F3C9CF"},{"background":"#ECA4AF"},{"background":"#EEB0BA"},{"background":"#EBA2AD"},{"background":"#E997A4"},{"background":"#EBA0AC"},{"background":"#E8919F"},{"background":"#F0B9C2"},{"background":"#EBA3AE"},{"background":"#F1BEC6"},{"background":"#E58494"},{"background":"#9AC6DF"},{"background":"#E78F9D"},{"background":"#E999A6"},{"background":"#EBA1AD"},{"background":"#E893A0"},{"background":"#EDAAB5"},{"background":"#EA9BA8"},{"background":"#E68796"},{"background":"#E8909E"},{"background":"#EA9CA8"},{"background":"#EEAEB8"},{"background":"#E892A0"},{"background":"#8FBFDB"},{"background":"#EDAAB5"},{"background":"#E999A6"},{"background":"#EBA1AD"},{"background":"#EB9EAA"},{"background":"#EFB3BC"},{"background":"#E995A2"},{"background":"#EEADB7"},{"background":"#EA9DA9"},{"background":"#EBA2AE"},{"background":"#F2C4CC"},{"background":"#EBA1AD"},{"background":"#EBA2AE"},{"background":"#EDAAB5"},{"background":"#ECA4AF"},{"background":"#EEAFB9"},{"background":"#EBA2AE"},{"background":"#EA9CA9"}]},{"id":"FDR","name":"FDR","type":"numeric","cell":["3.88e-11","1.09e-10","2.88e-10","8.43e-10","8.43e-10","1.18e-09","1.18e-09","5.74e-09","1.58e-08","1.70e-08","2.83e-08","3.96e-08","4.84e-08","5.57e-08","6.56e-08","1.53e-07","1.53e-07","5.37e-07","5.72e-07","5.72e-07","9.24e-07","1.00e-06","1.20e-06","2.37e-06","2.37e-06","3.03e-06","4.34e-06","6.33e-06","7.20e-06","7.73e-06","7.73e-06","9.87e-06","1.16e-05","1.21e-05","1.81e-05","1.81e-05","2.06e-05","2.86e-05","3.09e-05","3.09e-05","3.09e-05","5.21e-05","6.18e-05","6.19e-05","6.34e-05","6.52e-05","7.27e-05","8.19e-05","9.53e-05","1.02e-04","1.34e-04","1.34e-04","1.39e-04","1.47e-04","1.47e-04","1.51e-04","1.78e-04","1.78e-04","1.80e-04","1.80e-04","2.36e-04","2.47e-04","2.72e-04","2.79e-04","3.07e-04","3.07e-04","3.26e-04","3.56e-04","4.52e-04","4.57e-04","4.60e-04","4.66e-04","5.48e-04","5.94e-04","5.94e-04","5.94e-04","5.94e-04","5.94e-04","6.00e-04","6.04e-04","6.04e-04","6.36e-04","6.38e-04","7.28e-04","8.17e-04","8.32e-04","8.89e-04","9.12e-04","1.09e-03","1.13e-03","1.16e-03","1.17e-03","1.23e-03","1.25e-03","1.38e-03","1.41e-03","1.42e-03","1.52e-03","1.57e-03","1.57e-03","1.77e-03","1.83e-03","1.83e-03","1.92e-03","1.93e-03","1.93e-03","2.15e-03","2.15e-03","2.15e-03","2.17e-03","2.22e-03","2.22e-03","2.63e-03","2.69e-03","2.78e-03","2.78e-03","3.18e-03","3.21e-03","3.22e-03","3.22e-03","3.70e-03","4.20e-03","4.22e-03","4.22e-03","4.26e-03","4.63e-03","4.92e-03","4.92e-03","4.94e-03","5.22e-03","5.59e-03","5.77e-03","5.77e-03","6.64e-03","6.66e-03","6.94e-03","7.16e-03","7.23e-03","7.41e-03","7.41e-03","7.75e-03","7.81e-03","7.92e-03","9.56e-03","9.74e-03","1.01e-02","1.02e-02","1.12e-02","1.12e-02","1.19e-02","1.19e-02","1.24e-02","1.24e-02","1.38e-02","1.52e-02","1.52e-02","1.52e-02","1.56e-02","1.59e-02","1.81e-02","1.85e-02","1.85e-02","1.85e-02","2.03e-02","2.03e-02","2.06e-02","2.32e-02","2.32e-02","2.33e-02","2.47e-02","2.47e-02","2.54e-02","2.54e-02","2.57e-02","2.62e-02","2.66e-02","2.72e-02","2.74e-02","2.88e-02","3.05e-02","3.28e-02","3.37e-02","3.43e-02","3.57e-02","3.68e-02","3.97e-02","3.97e-02","4.19e-02","4.20e-02","4.27e-02","4.33e-02","4.37e-02","4.38e-02","4.38e-02","4.53e-02","4.88e-02","4.93e-02","5.17e-02","5.61e-02","5.68e-02"],"maxWidth":66,"style":{"borderRight":"1px solid rgba(0, 0, 0, 0.1)"}},{"id":"Macs2 Peak","name":"Macs2 Peak","type":"logical","cell":["✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes"],"maxWidth":60,"style":[{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"}]},{"id":"Region","name":"Region","type":"factor","maxWidth":150},{"id":"Genes","name":"Genes","type":"character","cell":[{"name":"span","attribs":{"title":"LINC00867"},"children":["LINC00867"]},{"name":"span","attribs":{"title":"APBB2"},"children":["APBB2"]},{"name":"span","attribs":{"title":"PPIF"},"children":["PPIF"]},{"name":"span","attribs":{"title":"FAM214A"},"children":["FAM214A"]},{"name":"span","attribs":{"title":"ZNRF1"},"children":["ZNRF1"]},{"name":"span","attribs":{"title":"AC092812.1"},"children":["AC092812.1"]},{"name":"span","attribs":{"title":"AC078864.2"},"children":["AC078864.2"]},{"name":"span","attribs":{"title":"ADAMTS8"},"children":["ADAMTS8"]},{"name":"span","attribs":{"title":"SEC14L2; AC004832.3; RNF215"},"children":["SEC14L2; AC004832.3; RNF215"]},{"name":"span","attribs":{"title":"LINC00567"},"children":["LINC00567"]},{"name":"span","attribs":{"title":"AL807740.1"},"children":["AL807740.1"]},{"name":"span","attribs":{"title":"RERG"},"children":["RERG"]},{"name":"span","attribs":{"title":"RNF215"},"children":["RNF215"]},{"name":"span","attribs":{"title":"AC008537.1"},"children":["AC008537.1"]},{"name":"span","attribs":{"title":"AJ009632.2"},"children":["AJ009632.2"]},{"name":"span","attribs":{"title":"IQCM"},"children":["IQCM"]},{"name":"span","attribs":{"title":"ZNF521"},"children":["ZNF521"]},{"name":"span","attribs":{"title":"AC021035.1"},"children":["AC021035.1"]},{"name":"span","attribs":{"title":"ATAD2"},"children":["ATAD2"]},{"name":"span","attribs":{"title":"ZBTB16"},"children":["ZBTB16"]},{"name":"span","attribs":{"title":"AC092952.1"},"children":["AC092952.1"]},{"name":"span","attribs":{"title":"LINC01856"},"children":["LINC01856"]},{"name":"span","attribs":{"title":"AC143336.1"},"children":["AC143336.1"]},{"name":"span","attribs":{"title":"C2orf76"},"children":["C2orf76"]},{"name":"span","attribs":{"title":"PPIF"},"children":["PPIF"]},{"name":"span","attribs":{"title":"AC091180.6"},"children":["AC091180.6"]},{"name":"span","attribs":{"title":"APIP"},"children":["APIP"]},{"name":"span","attribs":{"title":"AL513164.1"},"children":["AL513164.1"]},{"name":"span","attribs":{"title":"TENM3-AS1"},"children":["TENM3-AS1"]},{"name":"span","attribs":{"title":"PRLR"},"children":["PRLR"]},{"name":"span","attribs":{"title":"PAICSP7"},"children":["PAICSP7"]},{"name":"span","attribs":{"title":"AC108515.1"},"children":["AC108515.1"]},{"name":"span","attribs":{"title":"RNU7-92P"},"children":["RNU7-92P"]},{"name":"span","attribs":{"title":"KDM4B"},"children":["KDM4B"]},{"name":"span","attribs":{"title":"PHACTR1"},"children":["PHACTR1"]},{"name":"span","attribs":{"title":"AQP3"},"children":["AQP3"]},{"name":"span","attribs":{"title":"LINC01911"},"children":["LINC01911"]},{"name":"span","attribs":{"title":"BBS9"},"children":["BBS9"]},{"name":"span","attribs":{"title":"AC004947.1"},"children":["AC004947.1"]},{"name":"span","attribs":{"title":"PTPRN2"},"children":["PTPRN2"]},{"name":"span","attribs":{"title":"TUSC3"},"children":["TUSC3"]},{"name":"span","attribs":{"title":"ACSM5P1"},"children":["ACSM5P1"]},{"name":"span","attribs":{"title":"AC093159.1"},"children":["AC093159.1"]},{"name":"span","attribs":{"title":"ELP3"},"children":["ELP3"]},{"name":"span","attribs":{"title":"MRFAP1"},"children":["MRFAP1"]},{"name":"span","attribs":{"title":"CALM2P1"},"children":["CALM2P1"]},{"name":"span","attribs":{"title":"AC068633.1"},"children":["AC068633.1"]},{"name":"span","attribs":{"title":"AC011592.1"},"children":["AC011592.1"]},{"name":"span","attribs":{"title":"AC126121.3"},"children":["AC126121.3"]},{"name":"span","attribs":{"title":"LCLAT1"},"children":["LCLAT1"]},{"name":"span","attribs":{"title":"LSAMP"},"children":["LSAMP"]},{"name":"span","attribs":{"title":"TAT-AS1"},"children":["TAT-AS1"]},{"name":"span","attribs":{"title":"RN7SL658P"},"children":["RN7SL658P"]},{"name":"span","attribs":{"title":"AL359999.1"},"children":["AL359999.1"]},{"name":"span","attribs":{"title":"FITM2"},"children":["FITM2"]},{"name":"span","attribs":{"title":"SGK3"},"children":["SGK3"]},{"name":"span","attribs":{"title":"SILC1"},"children":["SILC1"]},{"name":"span","attribs":{"title":"BDH2P1"},"children":["BDH2P1"]},{"name":"span","attribs":{"title":"snoU13"},"children":["snoU13"]},{"name":"span","attribs":{"title":"RPL37P4"},"children":["RPL37P4"]},{"name":"span","attribs":{"title":"AGR3"},"children":["AGR3"]},{"name":"span","attribs":{"title":"DENND5B"},"children":["DENND5B"]},{"name":"span","attribs":{"title":"OLFM4"},"children":["OLFM4"]},{"name":"span","attribs":{"title":"LINC00536"},"children":["LINC00536"]},{"name":"span","attribs":{"title":"UGDH-AS1; AC021148.3"},"children":["UGDH-AS1; AC021148.3"]},{"name":"span","attribs":{"title":"CTPS2"},"children":["CTPS2"]},{"name":"span","attribs":{"title":"LINC01924"},"children":["LINC01924"]},{"name":"span","attribs":{"title":"LPP"},"children":["LPP"]},{"name":"span","attribs":{"title":"AC012391.1"},"children":["AC012391.1"]},{"name":"span","attribs":{"title":"CCDC57"},"children":["CCDC57"]},{"name":"span","attribs":{"title":"RARB"},"children":["RARB"]},{"name":"span","attribs":{"title":"SP2; SP2-AS1"},"children":["SP2; SP2-AS1"]},{"name":"span","attribs":{"title":"GAA"},"children":["GAA"]},{"name":"span","attribs":{"title":"TTC21A"},"children":["TTC21A"]},{"name":"span","attribs":{"title":"CARTPT"},"children":["CARTPT"]},{"name":"span","attribs":{"title":"AL138767.3; AL138767.1"},"children":["AL138767.3; AL138767.1"]},{"name":"span","attribs":{"title":"ANKRD29"},"children":["ANKRD29"]},{"name":"span","attribs":{"title":"OPHN1"},"children":["OPHN1"]},{"name":"span","attribs":{"title":"CRAMP1; AL031708.1"},"children":["CRAMP1; AL031708.1"]},{"name":"span","attribs":{"title":"RYR2"},"children":["RYR2"]},{"name":"span","attribs":{"title":"RCL1"},"children":["RCL1"]},{"name":"span","attribs":{"title":"NHLRC2"},"children":["NHLRC2"]},{"name":"span","attribs":{"title":"C5"},"children":["C5"]},{"name":"span","attribs":{"title":"PDE10A"},"children":["PDE10A"]},{"name":"span","attribs":{"title":"LINC01090"},"children":["LINC01090"]},{"name":"span","attribs":{"title":"MYB"},"children":["MYB"]},{"name":"span","attribs":{"title":"AL033523.1"},"children":["AL033523.1"]},{"name":"span","attribs":{"title":"AL591034.1"},"children":["AL591034.1"]},{"name":"span","attribs":{"title":"LINC02694; AC013652.1"},"children":["LINC02694; AC013652.1"]},{"name":"span","attribs":{"title":"APIP"},"children":["APIP"]},{"name":"span","attribs":{"title":"MACO1"},"children":["MACO1"]},{"name":"span","attribs":{"title":"LINC01728"},"children":["LINC01728"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"HOMER2"},"children":["HOMER2"]},{"name":"span","attribs":{"title":"GNG7"},"children":["GNG7"]},{"name":"span","attribs":{"title":"SH3TC2"},"children":["SH3TC2"]},{"name":"span","attribs":{"title":"CRAT; AL158151.2"},"children":["CRAT; AL158151.2"]},{"name":"span","attribs":{"title":"LINC02457"},"children":["LINC02457"]},{"name":"span","attribs":{"title":"AC008014.1"},"children":["AC008014.1"]},{"name":"span","attribs":{"title":"LINC01588"},"children":["LINC01588"]},{"name":"span","attribs":{"title":"AQP3"},"children":["AQP3"]},{"name":"span","attribs":{"title":"LINC01828; LINC01829"},"children":["LINC01828; LINC01829"]},{"name":"span","attribs":{"title":"SATB1-AS1"},"children":["SATB1-AS1"]},{"name":"span","attribs":{"title":"AC023590.1; SAMD12"},"children":["AC023590.1; SAMD12"]},{"name":"span","attribs":{"title":"PADI4"},"children":["PADI4"]},{"name":"span","attribs":{"title":"HMGB3P18"},"children":["HMGB3P18"]},{"name":"span","attribs":{"title":"AC021087.5; PDCD6; AHRR"},"children":["AC021087.5; PDCD6; AHRR"]},{"name":"span","attribs":{"title":"AC004593.2; CHN2"},"children":["AC004593.2; CHN2"]},{"name":"span","attribs":{"title":"MAPT"},"children":["MAPT"]},{"name":"span","attribs":{"title":"VWA2"},"children":["VWA2"]},{"name":"span","attribs":{"title":"LINC01643"},"children":["LINC01643"]},{"name":"span","attribs":{"title":"AL137786.2"},"children":["AL137786.2"]},{"name":"span","attribs":{"title":"AC092473.1"},"children":["AC092473.1"]},{"name":"span","attribs":{"title":"SOX2-OT"},"children":["SOX2-OT"]},{"name":"span","attribs":{"title":"AL391421.1; DLG5"},"children":["AL391421.1; DLG5"]},{"name":"span","attribs":{"title":"ASCC2"},"children":["ASCC2"]},{"name":"span","attribs":{"title":"ILRUN"},"children":["ILRUN"]},{"name":"span","attribs":{"title":"SYBU"},"children":["SYBU"]},{"name":"span","attribs":{"title":"GREB1L"},"children":["GREB1L"]},{"name":"span","attribs":{"title":"MGAM"},"children":["MGAM"]},{"name":"span","attribs":{"title":"PDZD2"},"children":["PDZD2"]},{"name":"span","attribs":{"title":"CDC14C"},"children":["CDC14C"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"AC055854.1; PEBP4"},"children":["AC055854.1; PEBP4"]},{"name":"span","attribs":{"title":"MIR720"},"children":["MIR720"]},{"name":"span","attribs":{"title":"AC036214.3; TPD52"},"children":["AC036214.3; TPD52"]},{"name":"span","attribs":{"title":"KIAA0825"},"children":["KIAA0825"]},{"name":"span","attribs":{"title":"ABHD2"},"children":["ABHD2"]},{"name":"span","attribs":{"title":"LINC02685"},"children":["LINC02685"]},{"name":"span","attribs":{"title":"U3"},"children":["U3"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"CST9"},"children":["CST9"]},{"name":"span","attribs":{"title":"AL354984.3"},"children":["AL354984.3"]},{"name":"span","attribs":{"title":"MIR4686"},"children":["MIR4686"]},{"name":"span","attribs":{"title":"C17orf100; SLC13A5"},"children":["C17orf100; SLC13A5"]},{"name":"span","attribs":{"title":"AUTS2"},"children":["AUTS2"]},{"name":"span","attribs":{"title":"AC002451.1"},"children":["AC002451.1"]},{"name":"span","attribs":{"title":"SYTL4"},"children":["SYTL4"]},{"name":"span","attribs":{"title":"TG"},"children":["TG"]},{"name":"span","attribs":{"title":"RAD51B"},"children":["RAD51B"]},{"name":"span","attribs":{"title":"AC010280.1"},"children":["AC010280.1"]},{"name":"span","attribs":{"title":"AC008674.1"},"children":["AC008674.1"]},{"name":"span","attribs":{"title":"MREG"},"children":["MREG"]},{"name":"span","attribs":{"title":"AC079031.2"},"children":["AC079031.2"]},{"name":"span","attribs":{"title":"VWA8"},"children":["VWA8"]},{"name":"span","attribs":{"title":"AC009262.1"},"children":["AC009262.1"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"CHCHD5; AC012442.3; AC012442.2"},"children":["CHCHD5; AC012442.3; AC012442.2"]},{"name":"span","attribs":{"title":"CTCF"},"children":["CTCF"]},{"name":"span","attribs":{"title":"AC068295.1"},"children":["AC068295.1"]},{"name":"span","attribs":{"title":"AC011317.1"},"children":["AC011317.1"]},{"name":"span","attribs":{"title":"CDC14A; AL589990.1"},"children":["CDC14A; AL589990.1"]},{"name":"span","attribs":{"title":"FANCC"},"children":["FANCC"]},{"name":"span","attribs":{"title":"SCNN1B"},"children":["SCNN1B"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"TIFA"},"children":["TIFA"]},{"name":"span","attribs":{"title":"TSPYL5"},"children":["TSPYL5"]},{"name":"span","attribs":{"title":"NDUFS5P6"},"children":["NDUFS5P6"]},{"name":"span","attribs":{"title":"RNU6-54P"},"children":["RNU6-54P"]},{"name":"span","attribs":{"title":"SLC9A3R1"},"children":["SLC9A3R1"]},{"name":"span","attribs":{"title":"CCNJL"},"children":["CCNJL"]},{"name":"span","attribs":{"title":"SDC4"},"children":["SDC4"]},{"name":"span","attribs":{"title":"PABPC5-AS1"},"children":["PABPC5-AS1"]},{"name":"span","attribs":{"title":"AC108681.1"},"children":["AC108681.1"]},{"name":"span","attribs":{"title":"LINC02026"},"children":["LINC02026"]},{"name":"span","attribs":{"title":"AL953883.1"},"children":["AL953883.1"]},{"name":"span","attribs":{"title":"DEC1; AL691420.1"},"children":["DEC1; AL691420.1"]},{"name":"span","attribs":{"title":"AL136090.1"},"children":["AL136090.1"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"AC108078.1; UGT2B24P"},"children":["AC108078.1; UGT2B24P"]},{"name":"span","attribs":{"title":"PGR-AS1"},"children":["PGR-AS1"]},{"name":"span","attribs":{"title":"MARK4; PPP1R37"},"children":["MARK4; PPP1R37"]},{"name":"span","attribs":{"title":"AC093714.1"},"children":["AC093714.1"]},{"name":"span","attribs":{"title":"PLAC1"},"children":["PLAC1"]},{"name":"span","attribs":{"title":"SEC22C"},"children":["SEC22C"]},{"name":"span","attribs":{"title":"AC007610.7"},"children":["AC007610.7"]},{"name":"span","attribs":{"title":"AC005821.1; MARCHF10"},"children":["AC005821.1; MARCHF10"]},{"name":"span","attribs":{"title":"DECR1; NBN"},"children":["DECR1; NBN"]},{"name":"span","attribs":{"title":"AL356807.1"},"children":["AL356807.1"]},{"name":"span","attribs":{"title":"PKHD1"},"children":["PKHD1"]},{"name":"span","attribs":{"title":"ACACA"},"children":["ACACA"]},{"name":"span","attribs":{"title":"WWC3"},"children":["WWC3"]},{"name":"span","attribs":{"title":"LINC02224"},"children":["LINC02224"]},{"name":"span","attribs":{"title":"MTCO3P30"},"children":["MTCO3P30"]},{"name":"span","attribs":{"title":"AC005821.1; MARCHF10"},"children":["AC005821.1; MARCHF10"]},{"name":"span","attribs":{"title":"MACROD2"},"children":["MACROD2"]},{"name":"span","attribs":{"title":"RCL1"},"children":["RCL1"]},{"name":"span","attribs":{"title":"SUB1"},"children":["SUB1"]},{"name":"span","attribs":{"title":"AC005324.6"},"children":["AC005324.6"]},{"name":"span","attribs":{"title":"BCAS4"},"children":["BCAS4"]},{"name":"span","attribs":{"title":"LIMCH1"},"children":["LIMCH1"]},{"name":"span","attribs":{"title":"C5orf66"},"children":["C5orf66"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"AL117341.1; SGMS1"},"children":["AL117341.1; SGMS1"]},{"name":"span","attribs":{"title":"AC026462.4"},"children":["AC026462.4"]},{"name":"span","attribs":{"title":"PDLIM5"},"children":["PDLIM5"]},{"name":"span","attribs":{"title":"AFF3"},"children":["AFF3"]},{"name":"span","attribs":{"title":"H4C8"},"children":["H4C8"]},{"name":"span","attribs":{"title":"AC016583.2"},"children":["AC016583.2"]},{"name":"span","attribs":{"title":"AC005906.3"},"children":["AC005906.3"]}],"minWidth":132}],"filterable":true,"defaultPageSize":10,"showPageSizeOptions":true,"pageSizeOptions":[10,20,50,200],"borderless":true,"theme":{"style":{"fontSize":12}},"dataKey":"feecf6ab7c32dfb1d299eeab975d9964"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
</div>
<div id="summary-plots" class="section level2 tabset">
<h2 class="tabset">Summary Plots</h2>
<pre class="r"><code>profile_width &lt;- 5e3
n_bins &lt;- 100
bwfl &lt;- file.path(macs2_path, glue(&quot;{treat_levels}_merged_treat_pileup.bw&quot;)) %&gt;% 
  BigWigFileList() %&gt;% 
  setNames(treat_levels) 
sig_ranges &lt;- merged_results %&gt;% 
  filter(!!sym(fdr_column) &lt; fdr_alpha) %&gt;% 
  colToRanges(&quot;keyval_range&quot;) %&gt;% 
  splitAsList(f = .$direction) %&gt;% 
  .[vapply(., length, integer(1)) &gt; 0] 
fc_heat &lt;- names(sig_ranges) %&gt;% 
  lapply(
    function(x) {
      glue(
        &quot;
        *Heatmap and histogram for all regions considered to show evidence of 
        {ifelse(x == &#39;Up&#39;, &#39;increased&#39;, &#39;decreased&#39;)} {target} binding in 
        response to {treat_levels[[2]]} treatment. A total of 
        {comma(length(sig_ranges[[x]]))} regions were in this group.
        &quot;,
        ifelse(
          length(sig_ranges[[x]]) &gt; 2e4, 
          &quot; Due to the large number of regions, these were randomly down-sampled to 20,000 for viable plotting.&quot;,
          &quot;&quot;
        ),
        &quot;*&quot;
      )
    }
  ) %&gt;% 
  setNames(names(sig_ranges))
sig_profiles &lt;- lapply(sig_ranges, function(x) NULL)
for (i in names(sig_profiles)) {
  ## Restrict to 20,000. Plots work poorly above this number.
  ## Randomly sample
  set.seed(threads)
  temp_gr &lt;- granges(sig_ranges[[i]])
  n &lt;- length(temp_gr)
  if (n &gt; 2e4) temp_gr &lt;- temp_gr[sort(sample.int(n, 2e4))]
  sig_profiles[[i]] &lt;- getProfileData(
    bwfl, temp_gr, upstream = profile_width / 2, bins = n_bins,
    BPPARAM = bpparam()
  )
  rm(temp_gr)
}
profile_heatmaps &lt;- sig_profiles %&gt;% 
  parallel::mclapply(
    plotProfileHeatmap,
    profileCol = &quot;profile_data&quot;, 
    colour = &quot;name&quot;,
    xLab = &quot;Distance from Centre (bp)&quot;,
    fillLab = &quot;logCPM&quot;,
    mc.cores = length(sig_profiles)
  ) 
fill_range &lt;- profile_heatmaps %&gt;% 
  lapply(function(x) x$data[,&quot;score&quot;]) %&gt;% 
  unlist() %&gt;%
  range()
sidey_range &lt;- profile_heatmaps %&gt;% 
  lapply(function(x) x$layers[[3]]$data$y) %&gt;% 
  unlist() %&gt;% 
  range()
profile_heatmaps &lt;- profile_heatmaps %&gt;% 
  lapply(
    function(x) {
      suppressWarnings(
        x + 
          scale_fill_gradientn(colours = colours$heatmaps, limits = fill_range) +
          scale_xsidey_continuous(limits = sidey_range) +
          scale_colour_manual(values = treat_colours) +
          labs(
            x = &quot;Distance from Centre (bp)&quot;,
            fill = &quot;logCPM&quot;, 
            colour = &quot;Treat&quot;
          ) +
          theme(
            strip.text.y = element_text(angle = 0)
          )
      )
    }
  )</code></pre>
<div id="ma-plot" class="section level3">
<h3>MA Plot</h3>
<pre class="r"><code>if (!&quot;Ambiguous&quot; %in% levels(merged_results$status)) 
  direction_colours &lt;- direction_colours[names(direction_colours) != &quot;Ambiguous&quot;]
merged_results %&gt;% 
  as_tibble() %&gt;%
  ggplot(aes(AveExpr, logFC)) +
  geom_point(aes(colour = status), alpha = 0.5) +
  geom_smooth(se = FALSE, method = &quot;gam&quot;, formula = y ~ s(x, bs = &quot;cs&quot;)) +
  geom_hline(yintercept = 0) +
  geom_label_repel(
    aes(y = 1.1*logFC, label = annot, colour = status),
    data= . %&gt;%
      dplyr::filter(logFC == max(logFC) | logFC == min(logFC)) %&gt;%
      mutate(
        detected = gene_name %&gt;%
          lapply(paste, collapse = &quot;, &quot;) %&gt;%
          unlist() %&gt;%
          str_replace(&quot;^NA$&quot;, &quot;&quot;) %&gt;%
          str_wrap(width = 40),
        range = ifelse(detected == &quot;&quot;, range, paste0(range, &quot;\n&quot;)),
        annot = glue(&quot;{range}{detected}&quot;)
      ),
    fill = rgb(1, 1, 1, 0.2),
    size = 3,
    show.legend = FALSE
  ) +
  scale_colour_manual(values = direction_colours) +
  labs(
    x = &quot;Ave Signal (logCPM)&quot;,
    colour = &quot;Status&quot;
  ) </code></pre>
<div class="figure" style="text-align: center">
<img src="ER_E2_E2DHT_differential_binding_files/figure-html/plot-ma-1.png" alt="*MA plot showing the status of each window under consideration. The two most extreme regions are labelled by region and any associated genes, whilst the overall pattern of association between signal level (logCPM) and changed signal (logFC) is shown as the blue curve.*" width="960" />
<p class="caption">
<em>MA plot showing the status of each window under consideration. The
two most extreme regions are labelled by region and any associated
genes, whilst the overall pattern of association between signal level
(logCPM) and changed signal (logFC) is shown as the blue curve.</em>
</p>
</div>
</div>
<div id="volcano-plot" class="section level3">
<h3>Volcano Plot</h3>
<pre class="r"><code>merged_results %&gt;% 
  as_tibble() %&gt;% 
  ggplot(aes(logFC, -log10(P.Value))) +
  geom_point(aes(colour = status), alpha = 0.6) +
  geom_label_repel(
    aes(label = annot, colour = status),
    data= . %&gt;%
      arrange(sign(logFC)*log10(P.Value)) %&gt;% 
      dplyr::slice(1:2) %&gt;% 
      dplyr::filter(logFC &gt; 1) %&gt;%
      mutate(
        detected = gene_name %&gt;%
          lapply(paste, collapse = &quot;, &quot;) %&gt;%
          unlist() %&gt;%
          str_replace(&quot;^NA$&quot;, &quot;&quot;) %&gt;%
          str_wrap(width = 40),
        range = ifelse(detected == &quot;&quot;, range, paste0(range, &quot;\n&quot;)),
        annot = glue(&quot;{range}{detected}&quot;)
      ),
    fill = rgb(1, 1, 1, 0.2),
    size = 3,
    show.legend = FALSE
  ) +
  geom_label_repel(
    aes(label = annot, colour = status),
    data= . %&gt;%
      arrange(-sign(logFC)*log10(P.Value)) %&gt;% 
      dplyr::slice(1:2) %&gt;% 
      dplyr::filter(logFC &lt; -1) %&gt;% 
      mutate(
        detected = gene_name %&gt;%
          lapply(paste, collapse = &quot;, &quot;) %&gt;%
          unlist() %&gt;%
          str_replace(&quot;^NA$&quot;, &quot;&quot;) %&gt;%
          str_wrap(width = 40),
        range = ifelse(detected == &quot;&quot;, range, paste0(range, &quot;\n&quot;)),
        annot = glue(&quot;{range}{detected}&quot;)
      ),
    fill = rgb(1, 1, 1, 0.2),
    size = 3,
    show.legend = FALSE
  ) +
  scale_colour_manual(values = direction_colours) +
  labs(
    x = &quot;logFC&quot;,
    y = expression(paste(-log[10], &quot;p&quot;)),
    colour = &quot;Status&quot;
  )</code></pre>
<div class="figure" style="text-align: center">
<img src="ER_E2_E2DHT_differential_binding_files/figure-html/plot-volcano-1.png" alt="*Volcano plot showing regions with evidence of differential ER binding. The most significant regions are labelled along with any genes these regions are mapped to.*" width="960" />
<p class="caption">
<em>Volcano plot showing regions with evidence of differential ER
binding. The most significant regions are labelled along with any genes
these regions are mapped to.</em>
</p>
</div>
</div>
<div id="heatmap-increased-binding" class="section level3">
<h3>Heatmap: Increased Binding</h3>
<pre class="r"><code>profile_heatmaps$Up + guides(colour = &quot;none&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="ER_E2_E2DHT_differential_binding_files/figure-html/plot-up-heatmap-1.png" alt="*Heatmap and histogram for all regions considered to show evidence of 
increased ER binding in 
response to E2DHT treatment. A total of 
195 regions were in this group.
*" width="960" />
<p class="caption">
<em>Heatmap and histogram for all regions considered to show evidence of
increased ER binding in response to E2DHT treatment. A total of 195
regions were in this group. </em>
</p>
</div>
</div>
<div id="heatmap-decreased-binding" class="section level3">
<h3>Heatmap: Decreased Binding</h3>
<pre class="r"><code>profile_heatmaps$Down + guides(colour = &quot;none&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="ER_E2_E2DHT_differential_binding_files/figure-html/plot-down-heatmap-1.png" alt="*Heatmap and histogram for all regions considered to show evidence of 
decreased ER binding in 
response to E2DHT treatment. A total of 
2 regions were in this group.
*" width="960" />
<p class="caption">
<em>Heatmap and histogram for all regions considered to show evidence of
decreased ER binding in response to E2DHT treatment. A total of 2
regions were in this group. </em>
</p>
</div>
</div>
<div id="results-by-chromosome" class="section level3">
<h3>Results By Chromosome</h3>
<pre class="r"><code>merged_results %&gt;% 
  select(status) %&gt;% 
  as.data.frame() %&gt;% 
  mutate(
    merge_status = fct_collapse(
      status,
      Up = &quot;Up&quot;,
      Down = &quot;Down&quot;,
      `Ambiguous/Unchanged` = intersect(c(&quot;Ambiguous&quot;, &quot;Unchanged&quot;), levels(status))
    )
  ) %&gt;% 
  ggplot(aes(seqnames, fill = status)) +
  geom_bar() +
  facet_grid(merge_status~., scales = &quot;free_y&quot;) +
  scale_y_continuous(expand = expansion(c(0, 0.05)), labels = comma) +
  scale_fill_manual(values =  direction_colours) +
  labs(
    x = &quot;Chromosome&quot;, y = &quot;Number of Windows&quot;, 
    fill = &quot;Status&quot;, alpha = &quot;Macs2 Peak&quot;
  )</code></pre>
<div class="figure" style="text-align: center">
<img src="ER_E2_E2DHT_differential_binding_files/figure-html/plot-res-by-chromosome-1.png" alt="*Results for differential binding separated by chromosome*" width="960" />
<p class="caption">
<em>Results for differential binding separated by chromosome</em>
</p>
</div>
</div>
<div id="signal-by-region" class="section level3">
<h3>Signal By Region</h3>
<pre class="r"><code>merged_results %&gt;% 
  select(AveExpr, logFC, region) %&gt;% 
  as_tibble() %&gt;% 
  mutate(
    x = fct_relabel(
      region, str_replace_all, pattern = &quot; \\(&quot;, replacement = &quot;\n(&quot;
    ),
  ) %&gt;% 
  pivot_longer(cols = c(&quot;AveExpr&quot;, &quot;logFC&quot;)) %&gt;% 
  ggplot(aes(x, value, fill = region)) +
  geom_boxplot() +
  facet_grid(
    name~., scales = &quot;free&quot;, labeller = facet_labeller, switch = &quot;y&quot;
  ) +
  scale_fill_manual(values = region_colours) +
  labs(x = &quot;Region&quot;, y = c(), fill = &quot;Overlaps&quot;) </code></pre>
<div class="figure" style="text-align: center">
<img src="ER_E2_E2DHT_differential_binding_files/figure-html/boxplot-regions-1.png" alt="*Distributions of overall signal (logCPM) and changed signal (logFC) for each genomic region.*" width="960" />
<p class="caption">
<em>Distributions of overall signal (logCPM) and changed signal (logFC)
for each genomic region.</em>
</p>
</div>
</div>
<div id="differential-binding-by-region" class="section level3">
<h3>Differential Binding By Region</h3>
<pre class="r"><code>merged_results %&gt;% 
  subset(status %in% c(&quot;Up&quot;, &quot;Down&quot;)) %&gt;% 
  select(status, region) %&gt;% 
  as_tibble() %&gt;% 
  mutate(
    x = fct_relabel(
      region, str_replace_all, pattern = &quot; \\(&quot;, replacement = &quot;\n(&quot;
    )
  ) %&gt;% 
  group_by(x, region, status) %&gt;% 
  tally() %&gt;% 
  arrange(region, desc(status)) %&gt;% 
  mutate(y = cumsum(n)) %&gt;% 
  ggplot(
    aes(x, n, fill = status)
  ) +
  geom_col() +
  geom_label(
    aes(y = y - 0.5*n, label = comma(n, 1), colour = status),
    data = . %&gt;% ungroup() %&gt;% dplyr::filter(n &gt; 0.05*max(y)),
    fill = &quot;white&quot;, alpha = 0.8,
    show.legend = FALSE
  ) +
  geom_text(
    aes(y = y1, label = comma(y, 1)),
    data = . %&gt;% 
      dplyr::filter(y == max(y)) %&gt;% 
      ungroup() %&gt;% 
      mutate(y1 = y + 0.03*max(y))
  ) +
  scale_y_continuous(expand = expansion(c(0, 0.05)), labels = comma) +
  scale_fill_manual(values = direction_colours) +
  scale_colour_manual(values = direction_colours) +
  labs(x = &quot;Region&quot;, y = &quot;Merged Windows&quot;, fill = &quot;Status&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="ER_E2_E2DHT_differential_binding_files/figure-html/plot-db-regions-1.png" alt="*Merged windows considered as showing differential ER binding across all genomic regions.*" width="960" />
<p class="caption">
<em>Merged windows considered as showing differential ER binding across
all genomic regions.</em>
</p>
</div>
</div>
</div>
<div id="inspection-of-top-ranked-regions"
class="section level2 tabset">
<h2 class="tabset">Inspection of Top-Ranked Regions</h2>
<pre class="r"><code>## Define a GRL with the key ranges.
## Then we can step through it &amp; make all of the requisite plots
grl_to_plot &lt;- GRangesList(
  top_up_by_fdr = merged_results %&gt;% 
    filter(logFC &gt; 0) %&gt;% 
    filter(P.Value == min(P.Value)) %&gt;% 
    filter(!!sym(fdr_column) &lt; fdr_alpha),
  top_up_by_lfc = merged_results %&gt;% 
    filter(logFC == max(logFC), logFC &gt; 0, !!sym(fdr_column) &lt; fdr_alpha),
  top_up_by_expr = merged_results %&gt;% 
    filter(logFC &gt; 0, !!sym(fdr_column) &lt; fdr_alpha) %&gt;% 
    filter(AveExpr == max(AveExpr)),
  top_up_no_macs2 = merged_results %&gt;% 
    filter(logFC &gt; 0, !!sym(fdr_column) &lt; fdr_alpha, !overlaps_ref) %&gt;% 
    arrange(!!sym(fdr_column)) %&gt;% 
    arrange(P.Value),
  top_down_by_fdr = merged_results %&gt;% 
    filter(logFC &lt; 0) %&gt;% 
    filter(P.Value == min(P.Value)) %&gt;% 
    filter(!!sym(fdr_column) &lt; fdr_alpha),
  top_down_by_lfc = merged_results %&gt;% 
    filter(logFC == min(logFC), logFC &lt; 0, !!sym(fdr_column) &lt; fdr_alpha),
  top_down_by_expr = merged_results %&gt;% 
    filter(logFC &lt; 0, !!sym(fdr_column) &lt; fdr_alpha) %&gt;% 
    filter(AveExpr == max(AveExpr)),
  top_up_no_macs2 = merged_results %&gt;% 
    filter(logFC &lt; 0, !!sym(fdr_column) &lt; fdr_alpha, !overlaps_ref) %&gt;% 
    arrange(P.Value)
) %&gt;% 
  unlist() %&gt;% 
  .[!duplicated(.)] %&gt;%
  splitAsList(names(.)) %&gt;% 
  lapply(setNames, c()) %&gt;% 
  lapply(function(x) x[1]) %&gt;%
  GRangesList()
fh &lt;- max(4, 1 + 3 * ceiling(length(grl_to_plot) / 3)) %&gt;% 
  min(knitr::opts_chunk$get(&quot;fig.height&quot;))</code></pre>
<pre class="r"><code>## The coverage
bwfl &lt;- list2(
  &quot;{target}&quot; := file.path(
    macs2_path, glue(&quot;{treat_levels}_merged_treat_pileup.bw&quot;)
  ) %&gt;% 
    BigWigFileList() %&gt;% 
    setNames(treat_levels)
)
line_col &lt;- list2(&quot;{target}&quot; := treat_colours)
## Coverage annotations
annot &lt;- merged_results %&gt;% 
  splitAsList(.$status) %&gt;% 
  .[vapply(., length, integer(1)) &gt; 0] %&gt;% 
  endoapply(granges) %&gt;% 
  list() %&gt;% 
  setNames(target)
## Coverage y-limits
ind &lt;- filtered_counts %&gt;% 
  assay(&quot;counts&quot;) %&gt;% 
  apply(MARGIN = 2, which.max) %&gt;%
  unique()
max_ranges &lt;- rowRanges(filtered_counts[ind])
y_lim &lt;- bwfl[[target]] %&gt;%
  lapply(import.bw, which = max_ranges) %&gt;%
  lapply(function(x) c(0, max(x$score))) %&gt;% 
  unlist() %&gt;% 
  range() %&gt;% 
  list() %&gt;% 
  setNames(target)

## The features track
feat_gr &lt;- gene_regions %&gt;% 
  lapply(granges) %&gt;% 
  GRangesList()
feature_colours &lt;- colours$regions
if (has_features) {
  feat_gr &lt;- list(Regions = feat_gr)
  feat_gr$Features &lt;- splitAsList(external_features, external_features$feature)
  feature_colours &lt;- list(
    Regions = unlist(colours$regions),
    Features = unlist(colours$features)
  )
}

## The genes track
hfgc_genes &lt;- trans_models
gene_col &lt;- &quot;grey&quot;
if (has_rnaseq){
  if (!is.na(rna_lfc_col) &amp; !is.na(rna_fdr_col)) {
    hfgc_genes &lt;- trans_models %&gt;% 
      mutate(
        status = case_when(
          !gene %in% rnaseq$gene_id ~ &quot;Undetected&quot;,
          gene %in% dplyr::filter(
            rnaseq, !!sym(rna_lfc_col) &gt; 0, !!sym(rna_fdr_col) &lt; fdr_alpha
          )$gene_id ~ &quot;Up&quot;,
          gene %in% dplyr::filter(
            rnaseq, !!sym(rna_lfc_col) &lt; 0, !!sym(rna_fdr_col) &lt; fdr_alpha
          )$gene_id ~ &quot;Down&quot;,
          gene %in% dplyr::filter(
            rnaseq, !!sym(rna_fdr_col) &gt;= fdr_alpha
          )$gene_id ~ &quot;Unchanged&quot;,
        )
      ) %&gt;% 
      splitAsList(.$status) %&gt;% 
      lapply(select, -status) %&gt;% 
      GRangesList()
    gene_col &lt;- colours$direction %&gt;% 
      setNames(str_to_title(names(.)))
  }
}

## External Coverage (Optional)
if (!is.null(config$external$coverage)) {
  ext_cov_path &lt;- config$external$coverage %&gt;% 
    lapply(unlist) %&gt;% 
    lapply(function(x) setNames(here::here(x), names(x)))
  bwfl &lt;- c(
    bwfl[target],
    lapply(ext_cov_path, function(x) BigWigFileList(x) %&gt;% setNames(names(x)))
  )
  line_col &lt;- c(
    line_col[target],
    ext_cov_path %&gt;% 
      lapply(
        function(x) {
          missing &lt;- setdiff(names(x), names(colours$treat))
          cmn &lt;- intersect(names(x), names(colours$treat))
          col &lt;- setNames(character(length(names(x))), names(x))
          if (length(cmn) &gt; 0) col[cmn] &lt;- colours$treat[cmn]
          if (length(missing) &gt; 0) 
            col[missing] &lt;- hcl.colors(
              max(5, length(missing)), &quot;Zissou 1&quot;)[seq_along(missing)]
          col
        }
      )
  )
  
  y_ranges &lt;- grl_to_plot %&gt;% 
    unlist() %&gt;% 
    granges() %&gt;% 
    resize(w = 10 * width(.), fix = &#39;center&#39;)
  
  y_lim &lt;- c(
    y_lim[target],
    bwfl[names(bwfl) != target] %&gt;% 
      lapply(
        function(x) {
          GRangesList(lapply(x, import.bw, which = y_ranges)) %&gt;% 
            unlist() %&gt;% 
            filter(score == max(score)) %&gt;% 
            mcols() %&gt;% 
            .[[&quot;score&quot;]] %&gt;% 
            c(0) %&gt;% 
            range()
        }
      )
  )
  
}</code></pre>
<pre class="r"><code>makeCaption &lt;- function(.gr) {
  if (is.null(.gr)) return(NULL)
  dir &lt;- ifelse(.gr$direction == &quot;Down&quot;, &#39;decreased&#39;, &#39;increased&#39;)
  reg &lt;- case_when(
    str_detect(.gr$region, &quot;Inter&quot;) ~ paste(&quot;an&quot;, .gr$region, &quot;region&quot;),
    str_detect(.gr$region, &quot;Upstream&quot;) ~ paste(&quot;an&quot;, .gr$region),
    str_detect(.gr$region, &quot;(Ex|Intr)on&quot;) ~ paste(&quot;an&quot;, .gr$region),
    str_detect(.gr$region, &quot;^Prom&quot;) ~ paste(&quot;a&quot;, .gr$region)
  )
  feat &lt;- c()
  if (has_features) feat &lt;- case_when(
    str_detect(.gr$feature, &quot;^[AEIOU]&quot;) ~ paste(&quot;an&quot;, .gr$feature),
    !str_detect(.gr$feature, &quot;^[AEIOU]&quot;) ~ paste(&quot;a&quot;, .gr$feature)
  )
  gn &lt;- unlist(.gr$gene_name)
  fdr &lt;- mcols(.gr)[[fdr_column]]
  fdr &lt;- ifelse(
    fdr &lt; 0.001, sprintf(&#39;%.2e&#39;, fdr), sprintf(&#39;%.3f&#39;, fdr)
  )
  cp &lt;- c(
    glue(
      &quot;*The {width(.gr)}bp region showing {dir} {target} binding in response to &quot;, 
      &quot;{treat_levels[[2]]} treatment (FDR = {fdr}). &quot;,
      &quot;The range mostly overlapped with {reg}, with all &quot;,
      &quot;defined regions shown as a contiguous bar. &quot;,
      ifelse(
        has_features,
        glue(
          &quot;Using the features supplied in {basename(config$external$features)}, &quot;,
          &quot;this mostly overlapped {feat}, shown as a separate block &quot;,
          &quot;with the gene-centric regions. &quot;
        ),
        glue(&quot;&quot;)
      ),
    ),
    ifelse(
      .gr$overlaps_ref,
      paste(
        &quot;A consensus peak overlapping this region was identified by&quot;,
        &quot;`macs2 callpeak` when using merged samples.&quot;
      ),
      &quot;No consensus peak was identified using `macs2 callpeak`.&quot;
    ),
    ifelse(
      length(gn) &gt; 0,
      paste0(
        &quot;Using the above mapping strategy, this range is likely to regulate &quot;,
        collapseGenes(gn, format = &quot;&quot;)
      ),
      &quot;No genes were able to be assigned to this region.&quot;
    ),
    paste(
      &quot;For each sample, the y-axis limits represent the values from the window&quot;, 
      &quot;with the highest signal.*&quot;
    )
  )
  paste(cp, collapse = &quot; &quot;)
}</code></pre>
<div id="normalised-sliding-windows" class="section level3">
<h3>Normalised Sliding Windows</h3>
<pre class="r"><code>grl_to_plot %&gt;% 
  lapply(
    function(x){
      filtered_counts %&gt;% 
        subsetByOverlaps(x) %&gt;% 
        assay(&quot;qsmooth&quot;) %&gt;% 
        as_tibble(rownames = &quot;window&quot;) %&gt;% 
        pivot_longer(
          cols = all_of(colnames(filtered_counts)),
          names_to = &quot;sample&quot;,
          values_to = &quot;logCPM&quot;
        ) %&gt;% 
        mutate(range = as.character(x)) %&gt;% 
        list()
    }
  ) %&gt;% 
  as_tibble() %&gt;% 
  pivot_longer(everything()) %&gt;% 
  unnest(everything()) %&gt;% 
  mutate(
    direction = str_extract(name, &quot;down|up&quot;),
    by = str_extract(name, &quot;by_.+&quot;),
    across(
      all_of(c(&quot;name&quot;, &quot;direction&quot;, &quot;by&quot;)), str_sep_to_title
    ),
    name = str_remove_all(name, &quot;Top &quot;) %&gt;% 
      str_replace_all(&quot;Lfc&quot;, &quot;logFC&quot;) %&gt;% 
      str_replace_all(&quot;Fdr&quot;, &quot;FDR&quot;) %&gt;% 
      str_replace_all(&quot;Expr&quot;, &quot;Highest Signal&quot;) %&gt;% 
      str_replace_all(&quot;By (.+)&quot;, &quot;(\\1)&quot;) %&gt;% 
      str_replace_all(&quot;No Macs2&quot;, &quot;(No Macs2 Peak)&quot;),
    window = as.integer(window)
  ) %&gt;% 
  left_join(samples, by = &quot;sample&quot;) %&gt;% 
  ggplot(
    aes(window, logCPM, colour = treat, linetype = !!sym(rep_col))
  ) +
  geom_line(
    data = . %&gt;% 
      group_by(name) %&gt;% 
      dplyr::filter(max(window) &gt; 1)
  ) + 
  geom_point(
    data = . %&gt;% 
      group_by(name) %&gt;% 
      dplyr::filter(max(window) == 1),
    show.legend = FALSE
  ) + 
  facet_wrap(~name + range, scales = &quot;free_x&quot;) +
  scale_colour_manual(values = treat_colours) + 
  labs(
    x = &quot;Sliding Window\n(Unmerged)&quot;, y = &quot;SQ-Normalised logCPM&quot;, colour = &quot;Treat&quot;,
    linetype = str_sep_to_title(rep_col)
  )</code></pre>
<div class="figure" style="text-align: center">
<img src="ER_E2_E2DHT_differential_binding_files/figure-html/plot-qsmooth-top-ranges-1.png" alt="*Most highly ranked ranges for both gained and decreased ER binding in repsonse to E2DHT treatment. The smooth-quantile normalised values are shown across the initial set of sliding windows before merging. Ranges were chosen as the most extreme for FDR, Binding Strength (Signal) and logFC. Windows are shown free of the genomic context.*" width="960" />
<p class="caption">
<em>Most highly ranked ranges for both gained and decreased ER binding
in repsonse to E2DHT treatment. The smooth-quantile normalised values
are shown across the initial set of sliding windows before merging.
Ranges were chosen as the most extreme for FDR, Binding Strength
(Signal) and logFC. Windows are shown free of the genomic context.</em>
</p>
</div>
</div>
<div id="decreased-binding-lowest-fdr" class="section level3">
<h3>Decreased Binding (Lowest FDR)</h3>
<pre class="r"><code>plotHFGC(
  grl_to_plot$top_down_by_fdr,
  hic = hic,
  features = feat_gr,featcol = feature_colours, featsize = 1 + has_features,
  genes = hfgc_genes, genecol = gene_col,
  coverage = bwfl, linecol = line_col,
  annotation = annot, 
  annotcol = direction_colours,
  cytobands = bands_df,
  collapseTranscripts = FALSE,
  zoom = 10,
  max = 10 * width(grl_to_plot$top_down_by_fdr),
  ylim = y_lim,
  col.title = &quot;black&quot;, background.title = &quot;white&quot;, showAxis = FALSE,
  rotation.title = 90
)</code></pre>
<div class="figure" style="text-align: center">
<img src="ER_E2_E2DHT_differential_binding_files/figure-html/plot-top_down_by_fdr-1.png" alt="*The 480bp region showing decreased ER binding in response to E2DHT treatment (FDR = 0.025). The range mostly overlapped with an Intron, with all defined regions shown as a contiguous bar.  A consensus peak overlapping this region was identified by `macs2 callpeak` when using merged samples. Using the above mapping strategy, this range is likely to regulate PGR-AS1 For each sample, the y-axis limits represent the values from the window with the highest signal.*" width="960" />
<p class="caption">
<em>The 480bp region showing decreased ER binding in response to E2DHT
treatment (FDR = 0.025). The range mostly overlapped with an Intron,
with all defined regions shown as a contiguous bar. A consensus peak
overlapping this region was identified by <code>macs2 callpeak</code>
when using merged samples. Using the above mapping strategy, this range
is likely to regulate PGR-AS1 For each sample, the y-axis limits
represent the values from the window with the highest signal.</em>
</p>
</div>
</div>
<div id="decreased-binding-most-extreme" class="section level3">
<h3>Decreased Binding (Most Extreme)</h3>
<pre class="r"><code>plotHFGC(
  grl_to_plot$top_down_by_lfc,
  hic = hic,
  features = feat_gr,featcol = feature_colours, featsize = 1 + has_features,
  genes = hfgc_genes, genecol = gene_col,
  coverage = bwfl, linecol = line_col,
  annotation = annot, 
  annotcol = direction_colours,
  cytobands = bands_df,
  collapseTranscripts = FALSE,
  zoom = 10, 
  max = 10 * width(grl_to_plot$top_down_by_lfc),
  ylim = y_lim,
  col.title = &quot;black&quot;, background.title = &quot;white&quot;, showAxis = FALSE,
  rotation.title = 90
)</code></pre>
<div class="figure" style="text-align: center">
<img src="ER_E2_E2DHT_differential_binding_files/figure-html/plot-top_down_by_lfc-1.png" alt="*The 150bp region showing decreased ER binding in response to E2DHT treatment (FDR = 0.034). The range mostly overlapped with an Intron, with all defined regions shown as a contiguous bar.  A consensus peak overlapping this region was identified by `macs2 callpeak` when using merged samples. Using the above mapping strategy, this range is likely to regulate LINC02224 For each sample, the y-axis limits represent the values from the window with the highest signal.*" width="960" />
<p class="caption">
<em>The 150bp region showing decreased ER binding in response to E2DHT
treatment (FDR = 0.034). The range mostly overlapped with an Intron,
with all defined regions shown as a contiguous bar. A consensus peak
overlapping this region was identified by <code>macs2 callpeak</code>
when using merged samples. Using the above mapping strategy, this range
is likely to regulate LINC02224 For each sample, the y-axis limits
represent the values from the window with the highest signal.</em>
</p>
</div>
</div>
<div id="increased-binding-strongest-signal" class="section level3">
<h3>Increased Binding (Strongest Signal)</h3>
<pre class="r"><code>plotHFGC(
  grl_to_plot$top_up_by_expr,
  hic = hic,
  features = feat_gr,featcol = feature_colours, featsize = 1 + has_features,
  genes = hfgc_genes, genecol = gene_col,
  coverage = bwfl, linecol = line_col,
  annotation = annot, 
  annotcol = direction_colours,
  cytobands = bands_df,
  collapseTranscripts = FALSE,
  zoom = 10,
  max = 10 * width(grl_to_plot$top_up_by_expr),
  ylim = y_lim,
  col.title = &quot;black&quot;, background.title = &quot;white&quot;, showAxis = FALSE,
  rotation.title = 90
)</code></pre>
<div class="figure" style="text-align: center">
<img src="ER_E2_E2DHT_differential_binding_files/figure-html/plot-top-up-by-expr-1.png" alt="*The 990bp region showing increased ER binding in response to E2DHT treatment (FDR = 1.51e-04). The range mostly overlapped with an Intron, with all defined regions shown as a contiguous bar.  A consensus peak overlapping this region was identified by `macs2 callpeak` when using merged samples. Using the above mapping strategy, this range is likely to regulate SGK3 For each sample, the y-axis limits represent the values from the window with the highest signal.*" width="960" />
<p class="caption">
<em>The 990bp region showing increased ER binding in response to E2DHT
treatment (FDR = 1.51e-04). The range mostly overlapped with an Intron,
with all defined regions shown as a contiguous bar. A consensus peak
overlapping this region was identified by <code>macs2 callpeak</code>
when using merged samples. Using the above mapping strategy, this range
is likely to regulate SGK3 For each sample, the y-axis limits represent
the values from the window with the highest signal.</em>
</p>
</div>
</div>
<div id="increased-binding-lowest-fdr" class="section level3">
<h3>Increased Binding (Lowest FDR)</h3>
<pre class="r"><code>plotHFGC(
  grl_to_plot$top_up_by_fdr,
  hic = hic,
  features = feat_gr,featcol = feature_colours, featsize = 1 + has_features,
  genes = hfgc_genes, genecol = gene_col,
  coverage = bwfl, linecol = line_col,
  annotation = annot, 
  annotcol = direction_colours,
  cytobands = bands_df,
  collapseTranscripts = FALSE,
  zoom = 10,
  max = 10 * width(grl_to_plot$top_up_by_fdr),
  ylim = y_lim,
  col.title = &quot;black&quot;, background.title = &quot;white&quot;, showAxis = FALSE,
  rotation.title = 90
)</code></pre>
<div class="figure" style="text-align: center">
<img src="ER_E2_E2DHT_differential_binding_files/figure-html/plot-top_up_by_fdr-1.png" alt="*The 450bp region showing increased ER binding in response to E2DHT treatment (FDR = 3.88e-11). The range mostly overlapped with a Promoter (-1500/+500bp), with all defined regions shown as a contiguous bar.  A consensus peak overlapping this region was identified by `macs2 callpeak` when using merged samples. Using the above mapping strategy, this range is likely to regulate LINC00867 For each sample, the y-axis limits represent the values from the window with the highest signal.*" width="960" />
<p class="caption">
<em>The 450bp region showing increased ER binding in response to E2DHT
treatment (FDR = 3.88e-11). The range mostly overlapped with a Promoter
(-1500/+500bp), with all defined regions shown as a contiguous bar. A
consensus peak overlapping this region was identified by
<code>macs2 callpeak</code> when using merged samples. Using the above
mapping strategy, this range is likely to regulate LINC00867 For each
sample, the y-axis limits represent the values from the window with the
highest signal.</em>
</p>
</div>
</div>
<div id="increased-binding-most-extreme" class="section level3">
<h3>Increased Binding (Most Extreme)</h3>
<pre class="r"><code>plotHFGC(
  grl_to_plot$top_up_by_lfc,
  hic = hic,
  features = feat_gr,featcol = feature_colours, featsize = 1 + has_features,
  genes = hfgc_genes, genecol = gene_col,
  coverage = bwfl, linecol = line_col,
  annotation = annot, 
  annotcol = direction_colours,
  cytobands = bands_df,
  collapseTranscripts = FALSE,
  zoom = 10,
  max = 10 * width(grl_to_plot$top_up_by_lfc),
  ylim = y_lim,
  col.title = &quot;black&quot;, background.title = &quot;white&quot;, showAxis = FALSE,
  rotation.title = 90
)</code></pre>
<div class="figure" style="text-align: center">
<img src="ER_E2_E2DHT_differential_binding_files/figure-html/plot-top_up_by_lfc-1.png" alt="*The 300bp region showing increased ER binding in response to E2DHT treatment (FDR = 1.18e-09). The range mostly overlapped with an Intergenic (&gt;10kb) region, with all defined regions shown as a contiguous bar.  A consensus peak overlapping this region was identified by `macs2 callpeak` when using merged samples. Using the above mapping strategy, this range is likely to regulate AC078864.2 For each sample, the y-axis limits represent the values from the window with the highest signal.*" width="960" />
<p class="caption">
<em>The 300bp region showing increased ER binding in response to E2DHT
treatment (FDR = 1.18e-09). The range mostly overlapped with an
Intergenic (&gt;10kb) region, with all defined regions shown as a
contiguous bar. A consensus peak overlapping this region was identified
by <code>macs2 callpeak</code> when using merged samples. Using the
above mapping strategy, this range is likely to regulate AC078864.2 For
each sample, the y-axis limits represent the values from the window with
the highest signal.</em>
</p>
</div>
</div>
</div>
</div>
<div id="enrichment-testing" class="section level1">
<h1>Enrichment Testing</h1>
<pre class="r"><code>min_gs_size &lt;- params$enrichment$min_size
if (is.null(min_gs_size) | is.na(min_gs_size)) min_gs_size &lt;- 0
max_gs_size &lt;- params$enrichment$max_size
if (is.null(max_gs_size) | is.na(max_gs_size)) max_gs_size &lt;- Inf
min_sig &lt;- params$enrichment$min_sig
if (is.null(min_sig) | is.na(min_sig)) min_sig &lt;- 1
msigdb &lt;- msigdbr(species = &quot;Homo sapiens&quot;) %&gt;% 
  dplyr::filter(
    gs_cat %in% unlist(params$enrichment$msigdb$gs_cat) |
      gs_subcat %in% unlist(params$enrichment$msigdb$gs_subcat),
    str_detect(ensembl_gene, &quot;^E&quot;)
  ) %&gt;% 
  dplyr::rename(gene_id = ensembl_gene, gene_name = gene_symbol) %&gt;% # For easier integration 
  dplyr::select(-starts_with(&quot;human&quot;), -contains(&quot;entrez&quot;)) %&gt;% 
  dplyr::filter(gene_id %in% genes_gr$gene_id) %&gt;% 
  mutate(gs_url = str_extract(gs_url, &quot;^[^|]+&quot;)) %&gt;% 
  group_by(gs_name) %&gt;% 
  mutate(n = dplyr::n()) %&gt;% 
  ungroup() %&gt;% 
  dplyr::filter(n &gt;= min_gs_size, n &lt; max_gs_size) 
gs_by_gsid &lt;- msigdb %&gt;% 
  split(.$gs_name) %&gt;% 
  lapply(pull, &quot;gene_id&quot;) %&gt;% 
  lapply(unique)
gs_by_geneid &lt;- msigdb %&gt;% 
  split(.$gene_id) %&gt;% 
  lapply(pull, &quot;gs_name&quot;) %&gt;% 
  lapply(unique)
gs_url &lt;- msigdb %&gt;% 
  distinct(gs_name, gs_url) %&gt;%
  mutate(
    gs_url = ifelse(
      gs_url == &quot;&quot;, &quot;
      http://www.gsea-msigdb.org/gsea/msigdb/collections.jsp&quot;,
      gs_url
    )
  ) %&gt;% 
  with(setNames(gs_url, gs_name))

min_network_size &lt;- params$networks$min_size
if (is.null(min_network_size) | is.na(min_network_size))
  min_network_size &lt;- 2

max_network_size &lt;- params$networks$max_size
if (is.null(max_network_size) | is.na(max_network_size))
  max_network_size &lt;- Inf

max_network_dist &lt;- params$networks$max_distance
if (is.null(max_network_dist) | is.na(max_network_dist))
  max_network_dist &lt;- 1

net_layout &lt;- params$networks$layout
enrich_alpha &lt;- params$enrichment$alpha
adj_method &lt;- match.arg(params$enrichment$adj, p.adjust.methods)
adj_desc &lt;- case_when(
  p.adjust.methods %in% c(&quot;fdr&quot;, &quot;BH&quot;) ~ &quot;the Benjamini-Hochberg FDR&quot;,
  p.adjust.methods %in% c(&quot;BY&quot;) ~ &quot;the Benjamini-Yekutieli FDR&quot;,
  p.adjust.methods %in% c(&quot;bonferroni&quot;) ~ &quot;the Bonferroni&quot;,
  p.adjust.methods %in% c(&quot;holm&quot;) ~ &quot;Holm&#39;s&quot;,
  p.adjust.methods %in% c(&quot;hommel&quot;) ~ &quot;Hommel&#39;s&quot;,
  p.adjust.methods %in% c(&quot;hochberg&quot;) ~ &quot;Hochberg&#39;s&quot;,
  p.adjust.methods %in% c(&quot;none&quot;) ~ &quot;no&quot;
) %&gt;% 
  setNames(p.adjust.methods)
mapped_ids &lt;- merged_results %&gt;%
  as_tibble() %&gt;% 
  dplyr::select(status, gene_id) %&gt;%
  unnest(gene_id) %&gt;%
  group_by(gene_id) %&gt;%
  summarise(
    up = any(status == &quot;Up&quot;), 
    down = any(status == &quot;Down&quot;), 
    n_peaks = dplyr::n(),
    .groups = &quot;drop&quot;
  )</code></pre>
<p>As an initial exploration, the retained windows were compared to
pre-defined gene-sets. These were taken from the <code>MSigDB</code>
database and the gene-sets used for enrichment testing were restricted
to only include those with between 5 and 1000 genes.</p>
<pre class="r"><code>msigdb %&gt;% 
  distinct(gs_cat, gs_subcat, gs_name) %&gt;%
  group_by(gs_cat, gs_subcat) %&gt;% 
  summarise(`Gene Sets` = dplyr::n(),.groups = &quot;drop&quot;) %&gt;% 
  dplyr::rename(Category = gs_cat, `Sub-Category` = gs_subcat) %&gt;% 
  mutate(
    Description = case_when(
      Category == &quot;H&quot; ~ &quot;Hallmark Gene-Sets&quot;,
      Category == &quot;C1&quot; ~ &quot;Positional Gene Sets&quot;,
      Category == &quot;C2&quot; ~ str_replace_all(`Sub-Category`, &quot;CP:(.+)&quot;, &quot;Curated Gene-Sets: \\1&quot;),
      str_detect(`Sub-Category`, &quot;TFT&quot;) ~ str_replace_all(`Sub-Category`, &quot;TFT:(.+)&quot;, &quot;Transcription Factor Target Prediction Gene-Sets: \\1&quot;),
      str_detect(`Sub-Category`, &quot;MIR&quot;) ~ str_replace_all(`Sub-Category`, &quot;MIR:(.+)&quot;, &quot;microRNA Target Gene-Sets: \\1&quot;),
      str_detect(`Sub-Category`, &quot;CGN&quot;) ~ &quot;Cancer Gene Neighbourhoods&quot;,
      str_detect(`Sub-Category`, &quot;CM&quot;) ~ &quot;Cancer Modules&quot;,
      str_detect(`Sub-Category`, &quot;GO:BP&quot;) ~ &quot;Gene Ontology: Bological Process&quot;,
      str_detect(`Sub-Category`, &quot;GO:MF&quot;) ~ &quot;Gene Ontology: Molecular Function&quot;,
      str_detect(`Sub-Category`, &quot;GO:CC&quot;) ~ &quot;Gene Ontology: Cellular Component&quot;,
      str_detect(`Sub-Category`, &quot;HPO&quot;) ~ &quot;Human Phenotype Ontology&quot;,
      Category == &quot;C6&quot; ~ &quot;Oncogenic Signature Gene Sets&quot;,
      `Sub-Category` == &quot;IMMUNESIGDB&quot; ~ &quot;ImmuneSigDB Gene Sets&quot;,
      `Sub-Category` == &quot;VAX&quot; ~ &quot;Vaccine Response Gene Sets&quot;,
      Category == &quot;C8&quot; ~ &quot;Cell Type Signature Gene Sets&quot;
    ),
    Category = factor(Category, levels = c(&quot;H&quot;, paste0(&quot;C&quot;, 1:8)))
  ) %&gt;% 
  droplevels() %&gt;% 
  arrange(Category) %&gt;% 
  pander(
    justify = &quot;llrl&quot;, 
    caption = &quot;*Summary of Gene-Sets used for Enrichment Testing*&quot;
  )</code></pre>
<table style="width:99%;">
<caption><em>Summary of Gene-Sets used for Enrichment
Testing</em></caption>
<colgroup>
<col width="15%" />
<col width="25%" />
<col width="16%" />
<col width="41%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Category</th>
<th align="left">Sub-Category</th>
<th align="right">Gene Sets</th>
<th align="left">Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">H</td>
<td align="left"></td>
<td align="right">50</td>
<td align="left">Hallmark Gene-Sets</td>
</tr>
<tr class="even">
<td align="left">C2</td>
<td align="left">CP:KEGG</td>
<td align="right">186</td>
<td align="left">Curated Gene-Sets: KEGG</td>
</tr>
<tr class="odd">
<td align="left">C2</td>
<td align="left">CP:REACTOME</td>
<td align="right">1,611</td>
<td align="left">Curated Gene-Sets: REACTOME</td>
</tr>
<tr class="even">
<td align="left">C2</td>
<td align="left">CP:WIKIPATHWAYS</td>
<td align="right">664</td>
<td align="left">Curated Gene-Sets: WIKIPATHWAYS</td>
</tr>
<tr class="odd">
<td align="left">C3</td>
<td align="left">TFT:GTRD</td>
<td align="right">413</td>
<td align="left">Transcription Factor Target Prediction Gene-Sets:
GTRD</td>
</tr>
</tbody>
</table>
<p>Retained windows were tested for enrichment of these gene-sets
using:</p>
<ol style="list-style-type: decimal">
<li>Any window mapped to a gene from these gene-sets</li>
<li>Any window with evidence of differential binding mapped to a gene
from these gene-sets</li>
</ol>
<p>In the first case, mapped genes were tested for enrichment against
all annotated genes, or all detected genes if RNA-Seq data was provided.
In the second case, i.e. Differentially Bound Windows, the control set
of genes were those mapped to a window (i.e. those test set from step
1). All adjusted p-values below are calculated using the
Benjamini-Hochberg FDR adjustment. After adjustment, any enriched
gene-sets with fewer than 3 mapped genes were excluded as being
uninformative.</p>
<p>If 4 or more gene-sets were considered to be enriched, a network plot
will be produced for that specific analysis, with network sizes capped
at 80. The distances between gene-sets were calculated using the overlap
coefficient (OC). Gene-sets with a large overlap will thus be given
small distances and in the case of a complete overlap (<span
class="math inline">\(OC = 1\)</span>) the only most highly ranked
gene-set was retained. Gene-set (i.e. node) pairs with a distance &gt;
0.9 will not have edges drawn between them and edge width also
corresponds to the distance between nodes with closely related nodes
having thicker edges. All network plots were generated using the fr
layout algorithm.</p>
<div id="windows-with-detected-er" class="section level2 tabset">
<h2 class="tabset">Windows With Detected ER</h2>
<pre class="r"><code>pwf_mapped &lt;- genes_gr %&gt;% 
  mutate(w = width(.)) %&gt;% 
  as_tibble() %&gt;% 
  dplyr::select(gene_id, w) %&gt;% 
  arrange(desc(w)) %&gt;% 
  distinct(gene_id, .keep_all = TRUE) %&gt;% 
  mutate(mapped = gene_id %in% unlist(merged_results$gene_id)) %&gt;% 
  with(
    nullp(
      structure(mapped, names = gene_id),
      genome = config$genome$build,
      bias.data = log10(w),
      plot.fit = TRUE
    )
  )</code></pre>
<pre class="r"><code>goseq_mapped &lt;- tibble(gs_name = character(), adj_p = numeric())
if (sum(pwf_mapped$DEgenes) &gt; 0) {
  goseq_mapped &lt;- goseq(
    pwf_mapped, config$genome$build, gene2cat = gs_by_geneid
  ) %&gt;% 
    dplyr::rename(gs_name = category, pval = over_represented_pvalue) %&gt;% 
    dplyr::select(-starts_with(&quot;under&quot;)) %&gt;% 
    as_tibble() %&gt;% 
    arrange(pval) %&gt;% 
    dplyr::filter(numDEInCat &gt; 0) %&gt;% 
    mutate(adj_p = p.adjust(pval, adj_method)) %&gt;% 
    dplyr::filter(numDEInCat &gt;= min_sig)
}
any_goseq_mapped &lt;- sum(goseq_mapped$adj_p &lt; enrich_alpha) &gt; 0
tg_mapped &lt;- make_tbl_graph(
  goseq_mapped, 
  gs = gs_by_gsid %&gt;% 
    lapply(intersect, rownames(subset(pwf_mapped, DEgenes))) %&gt;% 
    .[vapply(., length, integer(1)) &gt; 0]
)
plot_network_mapped &lt;- length(tg_mapped) &gt;= min_network_size</code></pre>
<div id="results-table" class="section level3">
<h3>Results Table</h3>
<pre class="r"><code>cp &lt;- htmltools::tags$caption(
  htmltools::em(
    glue(
      &quot;
    All {sum(goseq_mapped$adj_p &lt; enrich_alpha)} gene sets considered as 
    significantly enriched (p
    &quot;
    ),
    htmltools::tags$sub(&quot;adj&quot;),
    htmltools::em(
      glue(
        &quot;
         &lt; {enrich_alpha}) amongst the merged windows with detectable {target} 
        binding. Genes mapped to a merged window were compared to those not 
        mapped to any merged windows. Gene width was used to capture any 
        gene-level sampling bias.
        &quot;
      )
    )
  )
)
tbl &lt;- goseq_mapped %&gt;%
  dplyr::filter(adj_p &lt; enrich_alpha) %&gt;% 
  left_join(
    dplyr::select(msigdb, gs_name, gene_name, gene_id, gs_url, gs_description),
    by = &quot;gs_name&quot;
  ) %&gt;% 
  dplyr::filter(gene_id %in% unlist(merged_results$gene_id)) %&gt;% 
  dplyr::select(-gene_id) %&gt;% 
  chop(gene_name) %&gt;% 
  mutate(
    gene_name = vapply(gene_name, paste, character(1), collapse = &quot;; &quot;),
    `%` = numDEInCat / numInCat
  ) %&gt;% 
  dplyr::select(
    `Gene Set` = gs_name, Description = gs_description,
    `%`, numDEInCat, numInCat, 
    p = pval, adj_p, 
    `Mapped Genes` = gene_name
  ) %&gt;% 
  reactable(
    filterable = TRUE,
    showPageSizeOptions = TRUE,
    columns = list(
      &quot;Gene Set&quot; = colDef(
        minWidth = 150,
        cell = function(value) htmltools::tags$a(
          href = gs_url[[value]], 
          target = &quot;_blank&quot;, 
          str_replace_all(value, &quot;_&quot;, &quot; &quot;)
        ),
        html = TRUE
      ),
      Description = colDef(
        minWidth = 150,
        cell = function(value) with_tooltip(value, width = 100)
      ),
      numDEInCat = colDef(name = &quot;Total Mapped&quot;, maxWidth = 80),
      numInCat = colDef(
        name = &quot;Gene Set Size&quot;,
        maxWidth = 80,
        cell = function(value) comma(value, 1)
      ),
      &quot;%&quot; = colDef(
        name = &quot;% Mapped&quot;,
        cell = function(value) percent(value, 0.1),
        maxWidth = 80
      ),
      p = colDef(
        cell = function(value) {
          ifelse(
            value &lt; 0.001, sprintf(&quot;%.2e&quot;, value), sprintf(&quot;%.3f&quot;, value)
          )
        },
        maxWidth = 80
      ),
      adj_p = colDef(
        name = &quot;p&lt;sub&gt;adj&lt;/sub&gt;&quot;, html = TRUE,
        cell = function(value) {
          ifelse(
            value &lt; 0.001, sprintf(&quot;%.2e&quot;, value), sprintf(&quot;%.3f&quot;, value)
          )
        },
        maxWidth = 80
      ),
      &quot;Mapped Genes&quot; = colDef(
        cell = function(value) with_tooltip(value, width = 100),
        minWidth = 150
      )
    )
  )
div(class = &quot;table&quot;,
  div(class = &quot;table-header&quot;,
      div(class = &quot;caption&quot;, cp),
      tbl
  )
)</code></pre>
<div class="table">
<div class="table-header">
<div class="caption">
<caption>
<em>
All 18 gene sets considered as 
significantly enriched (p
<sub>adj</sub>
<em> &lt; 0.05) amongst the merged windows with detectable ER 
binding. Genes mapped to a merged window were compared to those not 
mapped to any merged windows. Gene width was used to capture any 
gene-level sampling bias.</em>
</em>
</caption>
</div>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-052cf0695aad6862ea18" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-052cf0695aad6862ea18">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"Gene Set":["GREB1_TARGET_GENES","HALLMARK_ESTROGEN_RESPONSE_EARLY","ZNF197_TARGET_GENES","MCM2_TARGET_GENES","HALLMARK_ESTROGEN_RESPONSE_LATE","ZNF175_TARGET_GENES","ZNF584_TARGET_GENES","ZSCAN2_TARGET_GENES","HDAC4_TARGET_GENES","DLX6_TARGET_GENES","ZNF589_TARGET_GENES","ARID5B_TARGET_GENES","ZNF579_TARGET_GENES","HALLMARK_ANDROGEN_RESPONSE","SNAI1_TARGET_GENES","FOXJ2_TARGET_GENES","HALLMARK_PEROXISOME","KLF7_TARGET_GENES"],"Description":["Genes containing one or more binding sites for UniProt:Q4ZG55 (GREB1) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Genes defining early response to estrogen.","Genes containing one or more binding sites for UniProt:O14709 (ZNF197) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Genes containing one or more binding sites for UniProt:P49736 (MCM2) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Genes defining late response to estrogen.","Genes containing one or more binding sites for UniProt:Q9Y473 (ZNF175) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Genes containing one or more binding sites for UniProt:Q8IVC4 (ZNF584) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Genes containing one or more binding sites for UniProt:Q7Z7L9 (ZSCAN2) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Genes containing one or more binding sites for UniProt:P56524 (HDAC4) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Genes containing one or more binding sites for UniProt:P56179 (DLX6) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Genes containing one or more binding sites for UniProt:Q86UQ0 (ZNF589) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Genes containing one or more binding sites for UniProt:Q14865 (ARID5B) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Genes containing one or more binding sites for UniProt:Q8NAF0 (ZNF579) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Genes defining response to androgens.","Genes containing one or more binding sites for UniProt:O95863 (SNAI1) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Genes containing one or more binding sites for UniProt:Q9P0K8 (FOXJ2) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Genes encoding components of peroxisome.","Genes containing one or more binding sites for UniProt:O75840 (KLF7) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization."],"%":[0.40803382663847781,0.67000000000000004,0.36459909228441756,0.73770491803278693,0.48499999999999999,0.28691476590636256,0.38043478260869568,0.22662266226622663,0.24780058651026393,0.22818791946308725,0.24421965317919075,0.22186495176848875,0.22817764165390506,0.36633663366336633,0.22384937238493724,0.23539232053422371,0.32692307692307693,0.20000000000000001],"numDEInCat":[386,134,241,45,97,239,70,206,169,136,169,207,149,37,107,141,34,187],"numInCat":[946,200,661,61,200,833,184,909,682,596,692,933,653,101,478,599,104,935],"p":[2.0011382182938937e-65,2.4509332997766194e-50,1.6404901905524214e-34,4.036323643173527e-27,1.7621992270283114e-26,1.0354358710736653e-18,2.4381976271137566e-11,9.5955504707667317e-09,2.1771786544479104e-08,2.19116478613042e-08,7.4321879482157325e-08,6.6746725884658197e-06,1.1528226298442539e-05,4.3548971852099478e-05,4.9015361975325206e-05,5.1660485770907396e-05,6.0722923659032837e-05,0.00018386673278428174],"adj_p":[5.4991278238716196e-62,3.3675823538930751e-47,1.5026890145460181e-31,2.7729543428602129e-24,9.6850469517476002e-24,4.7422962895173874e-16,9.5716672561551471e-09,3.2960715867083725e-06,6.0213208322863941e-06,6.0213208322863941e-06,1.8566956801542575e-05,0.0015285000227586727,0.0024368896821630845,0.008548041046397812,0.0088726884311533458,0.0088726884311533458,0.0098156820126483676,0.028070321205067011],"Mapped Genes":["ABCA3; ABCC11; ABCC5-AS1; ABHD17C; ABHD2; ADAMTSL5; ADAT3; ADGRG1; AFAP1; AFF1; AGAP2; AHCYL2; AKAP1; AKAP1-DT; AKAP8L; AMZ1; ANKFN1; ANKRD2; ANXA9; ARID1B; ARNTL; ARRDC1; ARRDC4; ASCC2; ATP6V0A4; ATP8B1; ATXN7L1; BATF; BBOF1; BCAR3; BCAS3; BCL3; BCL9L; BDKRB1; BFSP2-AS1; BRWD1; C1QTNF6; C5AR2; C9orf152; CA12; CAMTA1; CASP7; CCDC159; CCDC192; CCDC40; CCDC88C; CCDC88C-DT; CCN5; CCNP; CCR3; CD22; CD276; CD59; CDH11; CDH26; CDS1; CEACAM16-AS1; CELSR1; CFAP45; CHD6; CHPT1; CIC; CLN8; CORO2A; CRACR2A; CRKL; CRNDE; CRPPA; CTSB; CYP1B1; CYP1B1-AS1; CYP4F23P; DAAM1; DDX47; DGAT2; DLC1; DLG5; DNAH11; DNAI1; DNAJC5B; DNPEP; DOCK4; DOK7; DOP1B; DRAIC; DSCAM-AS1; EHD1; EHF; ELAPOR1; EML6; ENSG00000197099; ENSG00000225676; ENSG00000229425; ENSG00000231170; ENSG00000235185; ENSG00000238005; ENSG00000251574; ENSG00000254153; ENSG00000268858; ENSG00000270571; ENSG00000271874; ENSG00000278899; ERBB2; ERCC1; ERGIC1; ESRP2; ETNK2; EVL; FAM107B; FAM126A; FAM174B; FAM178B; FAM234B; FAM86B3P; FAM86EP; FAM86GP; FBRS; FBXO38; FHDC1; FMN1; FMO9P; FNTAP2; FREM2; G3BP1; GALNT10; GATAD2A; GCLM; GCOM1; GLRX5; GPRC5A; GREB1; GRHL1; GRHL2; GSPT1; HK1; HNRNPL; HSH2D; HSPB6; ID1; IFRD1; IGF1R; IGSF3; IKBKG; INHA; INPP4B; IQCD; IQSEC1; IRF1-AS1; ISG20; ITGB2; JUND; KCTD11; KDM4B; KIAA0100; KIAA1217; KRT7; KRT8; KRTAP3-1; LAD1; LAMB1; LDLRAD4; LENG1; LFNG; LIMA1; LINC00475; LINC00511; LINC00536; LINC00824; LINC00885; LINC00963; LINC01016; LINC01275; LINC01344; LINC01588; LINC01656; LINC01975; LINC02102; LINC02130; LINC02408; LINC02518; LINC02620; LINC02652; LINC02707; LLGL2; LMCD1; LPP; LRG1; LRIG2; LY6E-DT; MAF1; MANEAL; MAP4K3-DT; MAPK10; MAPKAP1; MCCC2; MCF2L; MCF2L2; MCRIP2; MGRN1; MICOS10; MICOS10-NBL1; MIDEAS; MIR3168; MIR3188; MIR3622B; MIR4259; MIR4530; MIR5093; MIR550A1; MIR550A2; MIR99AHG; MLLT3; MSMO1; MTA1; MTMR12; MTMR4; MTUS1; MUC5B; MYZAP; NAALADL2; NAIF1; NCAM2; NCF4-AS1; NDUFAF8; NFATC2; NFKBIA; NINL; NIPSNAP2; NLRP7P1; NMT1; NOP14-AS1; NRP1; NT5DC3; NUDT16-DT; OSBPL10; OVOL2; OXR1; OXR1-AS1; PAK4; PAM; PARD3; PBX1; PCBP1-AS1; PCDH1; PDGFD; PDZK1; PGP; PIK3AP1; PIK3C2G; PITX1-AS1; PKD2L2; PKIB; PKP4-AS1; PLAC1; PNKD; PNPLA7; POU2F3; PPM1L; PRDX3P2; PREX1; PRKCD; PRKCH; PROSER3; PRR15L; PRSS22; PRSS23; PRSS27; PTPRH; PWWP2A; PXDN; PXMP4; RAB5B; RAD51B; RADIL; RAPGEFL1; RARA; RBBP8; RBL2; RBM20; RBM47; RFX1; RGMB; RGS3; RN7SL211P; RNU6-1013P; RNU6-1105P; RNU6-813P; RPL37P4; RPL5P7; RPL7P30; RPS29P8; RPS6KA2; RPS6KL1; S100A16; SCAMP4; SCGB1D2; SCIN; SCRIB; SDK2; SEC14L2; SEPTIN9; SERPINA1; SGMS2; SHARPIN; SIDT2; SIN3A; SLC19A3; SLC1A5; SLC25A25; SLC25A36; SLC25A42; SLC29A1; SLC38A1; SLC39A1; SLC45A4; SLC6A6; SLC7A2; SLC9A8; SMAD3; SNORD114-1; SPAG17; SPECC1L; SPRED2; SPTLC2; SPTSSB; SRI; STARD10; STAT3; SULT1A1; SYBU; SYT8; TAFAZZIN; TBC1D16; TBC1D8; TEPSIN; TFAP2C; TFF1; TH; THADA; THSD4; TIPARP; TIPARP-AS1; TJP1; TLE1; TMCO3; TMEM105; TMEM160; TMEM184B; TMEM210; TMEM229B; TMEM241; TMEM94; TMPRSS3; TMTC2; TNS1; TPD52L1; TPD52L2; TPRG1; TPRN; TRAF7; TRAPPC9; TSPAN9; TTC39A-AS1; TUBA1A; TUBA1C; UIMC1; UMODL1; UNKL; URB1; UTRN; VAV3; VEPH1; VIRMA; VLDLR-AS1; VMP1; VPS26C; VWA3B; WWC1; WWP1; ZFYVE28; ZNF282; ZNF444; ZNF552; ZNF669; ZNRF1; ZRANB2-AS2","ABAT; ABCA3; ABHD2; ABLIM1; ADCY9; ADD3; AFF1; AKAP1; AMFR; ANXA9; AQP3; AR; AREG; ARL3; ASB13; B4GALT1; BCL11B; BCL2; CA12; CALCR; CBFA2T3; CCN5; CELSR1; CELSR2; CHPT1; DEPTOR; DHRS2; DHRS3; DLC1; ELOVL5; ENDOD1; ESRP2; FAM102A; FARP1; FASN; FHL2; FLNB; FRK; GAB2; GFRA1; GREB1; HES1; HR; IGF1R; IGFBP4; IL17RB; IL6ST; INHBB; ITPK1; KAZN; KCNK15; KDM4B; KLF10; KLF4; KRT13; KRT18; KRT19; KRT8; LAD1; LRIG1; MAPT; MAST4; MED13L; MICB; MLPH; MPPED2; MREG; MSMB; MYB; MYOF; NADSYN1; NAV2; NBL1; NCOR2; NRIP1; OLFML3; OPN3; OVOL2; P2RY2; PAPSS2; PDLIM3; PDZK1; PGR; PPIF; PRSS23; RAB31; RAPGEFL1; RARA; RASGRP1; RBBP8; REEP1; RET; RETREG1; RHOD; RPS6KA2; RRP12; SCARB1; SEC14L2; SH3BP5; SIAH2; SLC1A1; SLC22A5; SLC24A3; SLC2A1; SLC39A6; SLC7A2; SLC7A5; SLC9A3R1; SNX24; SOX3; SULT2B1; SVIL; SYBU; SYNGR1; SYT12; TBC1D30; TFAP2C; TFF1; TGIF2; THSD4; TIAM1; TIPARP; TJP3; TMEM164; TMPRSS3; TOB1; TPBG; TPD52L1; TSKU; TTC39A; TUBB2B; WFS1; WWC1; ZNF185","AAMP; ACACA; ACAD8; ACP2; ACSL3; ACTG1; ADAT3; ADCY9; ADGRB3; AHCYL2; AHSA1; ALG1L13P; AMMECR1; AP2S1; ARAP1; ARNTL; ASAP3; AUP1; BANP; BCORL1; BOD1; BTBD9; BTRC; BUB1B; BZW2; C1orf159; CALCOCO1; CBFA2T3; CCDC146; CCDC163; CCDC88A; CCDC90B; CCDC90B-AS1; CCP110; CDC40; CDH23; CDK5; CDKL3; CEP290; CFAP99; CFL1; CHMP4B; CLEC16A; CNPY2; COX7A2L; COX7C; CPEB4; CPQ; CRADD; CRTC2; CYB561D2; DDX23; DENND5B; DFFA; DGCR8; DYNC1I2; ELOC; ENSG00000186019; ENSG00000229327; ENSG00000233834; ENSG00000235530; ENSG00000238005; ENSG00000278390; ENY2; EPC1; ERVK3-1; FAM178B; FAM185BP; FANCC; FASN; FASTKD1; FAXDC2; FBXL19; FBXO15; FBXO38; FBXO38-DT; FCSK; FYCO1; GAB2; GALM; GFER; GFUS; GNG4; GPKOW; GPR158; GRHL1; GSPT1; GTF2H4; GTF2I; H3-3B; HDAC8; HIF1A; HIKESHI; HTRA2; IFRD2; IFTAP; IL1R1; ING3; IRF1-AS1; IZUMO1; KBTBD6; KBTBD7; KCNAB1; KCNC3; KDM4C; KHK; LARP1; LARS1; LIG1; LINC00824; LINC01275; LYPLA1; LYSMD1; MADCAM1; MANEAL; MAPT; MARS1; MAX; MDH1; MITF; MLLT10; MMACHC; MRFAP1; MRPL24; MRPS22; MUS81; MYB; NOS3; NOSIP; NOXO1; NPEPPS; NPEPPSP1; NPRL2; NRDE2; NUDCD1; NUDT13; ODAD3; PAFAH2; PANK4; PCBP1-AS1; PGP; PHB; PHF12; PHF13; PHLPP1; PIK3AP1; PIPOX; PLAU; PLEKHA1; PNKD; PPP1R12C; PPP2R2C; PRPF31; PRSS27; PTDSS2; PTK2; PTPRN2; RABAC1; RABGAP1L; RANGAP1; RBL1; RBM6; RNASEH2C; RNF144A; RPP40; RPS26; RXRA; SAP30; SAP30-DT; SBK1; SCAF1; SCAMP4; SCAMP5; SCNM1; SEC61A1; SEL1L; SETMAR; SFI1; SGPL1; SIAH1; SIPA1L3; SLC4A2; SLMAP; SLX4IP; SMARCAL1; SMIM2-AS1; SMPD3; SNAP25-AS1; SNAPC3; SPINK4; SRCIN1; ST3GAL2; ST7; STARD10; STAT3; STAU1; STK25; SYNGR1; SYT3; TADA2A; TBX6; TCEA2; TFPT; THA1P; THADA; THYN1; TIMM21; TMCC1; TMCC1-DT; TMEM11; TMEM115; TMEM160; TMEM70; TMTC3; TPGS1; TPH2; TRABD; TULP2; UBE2V1P4; UBE3C; UCKL1; UNK; USF2; USP31; VCPIP1; VIPAS39; WASF1; WDPCP; WDR87; WWP1; XYLT2; ZFYVE1; ZFYVE28; ZNF221; ZNF225; ZNF32; ZNF331; ZNF341-AS1; ZNF37BP; ZNF444; ZSWIM9","AAMP; AP2S1; BOD1; C19orf44; C1orf159; CALCOCO1; CALR3; CHMP4B; CLPB; CRTC2; CYB561D2; DDX23; DESI2; DFFA; ENSG00000235530; FAM13A; FBXL19-AS1; FEM1A; GSK3A; GSPT1; GTF2H4; H3-3B; IFRD2; MADCAM1-AS1; NCLN; NDC1; NPRL2; ORMDL3; PANK4; PEMT; PHB; PHF23; PIPOX; PNKD; ST3GAL2; TCEA2; THA1P; TMEM11; TMEM115; TSSC4; TUBA1C; UCKL1; UNK; ZFYVE1; ZNF341-AS1","ABCA3; ABHD2; ACOX2; ADD3; AFF1; AMFR; ANXA9; AREG; ARL3; BATF; BCL2; CA12; CALCR; CCN5; CD9; CDC6; CDH1; CELSR2; CHPT1; CHST8; DHRS2; DLG5; ELOVL5; EMP2; FAM102A; FARP1; FLNB; FRK; GFUS; HMGCS2; HOMER2; HR; IDH2; IGFBP4; IGSF1; IL17RB; IL6ST; ISG20; ITPK1; JAK1; KIF20A; KLF4; KRT13; KRT19; LARGE1; LLGL2; LSR; MAPT; MICB; MYB; MYOF; NBL1; NCOR2; NRIP1; OPN3; OVOL2; PAPSS2; PDLIM3; PDZK1; PGR; PLAC1; PPIF; PRKAR2B; PRLR; PRSS23; PTGER3; RAB31; RABEP1; RAPGEFL1; RBBP8; RET; RPS6KA2; S100A9; SCARB1; SCUBE2; SERPINA1; SIAH2; SLC22A5; SLC24A3; SLC29A1; SLC7A5; SLC9A3R1; SNX10; SOX3; ST14; SULT2B1; TFAP2C; TFF1; TH; TIAM1; TJP3; TMPRSS3; TOB1; TPBG; TPD52L1; WFS1; ZFP36","AAMP; ABCA3; ACP2; ACSM3; ACTG1; ADD1; ADGRB3; AGPS; AHCYL2; ALG1L13P; ALKBH8; ALPK1; AMMECR1; ANKRD17; ANO4; AP2S1; ARAP1; ARHGAP24; ARHGAP25; ARHGDIA; ARNTL; ASAP3; ATP5F1D; B3GNT5; BANP; BCL7C; BIRC5; BOD1; BRWD1; C12orf43; C19orf44; C1orf159; C2CD3; CALCOCO1; CALR3; CAMTA1; CAND1; CAPN15; CCDC159; CCDC25; CCDC88A; CCT6B; CDC40; CDH23; CEP290; CFAP99; CFL1; CHMP4B; CLASP1; CLEC16A; CLPB; COX7C; CRTC2; DDX23; DFFA; DGCR8; DRAM2; DYNC1I2; EBAG9; EEF2K; ENSG00000186019; ENSG00000213963; ENSG00000229327; ENSG00000235530; ENSG00000238005; ENSG00000278390; EPC1; ERI1; ERP44; FASN; FAXDC2; FBXO15; FBXO38; FBXO38-DT; FBXW11; FEM1A; GAB2; GALM; GAPVD1; GFER; GFUS; GNG4; GPBP1L1; GRB10; GSPT1; GTF2H4; H4C8; HIKESHI; HMBOX1; IFRD2; IFT20; IFTAP; IGF1R; IL1R1; INTS9; INVS; ISG20; ITGAL; JOSD2; KBTBD6; KBTBD7; KCNAB1; KCTD20; KIAA0100; LARS1; LINC00235; LINC01258; LINC02863; LRP6; LUC7L; LYPLA1; LYSMD1; MAD2L1BP; MAG; MANEAL; MAP1LC3B2; MARS1; MAU2; MDH1; MGRN1; MIR3649; MLLT10; MTMR4; MUS81; MXD1; MYO18B; NAV2; NDC1; NFE2L2; NOSIP; NOXO1; NPEPPS; NPEPPSP1; PAFAH2; PAK1; PANK4; PCBP1-AS1; PEDS1-UBE2V1; PEMT; PGP; PHB; PHF12; PIPOX; PLAU; PNKD; PNPLA6; PPP1R12C; PPRC1; PRKCE; PRKCH; PRR5; PTDSS2; PTK2; PVT1; PXT1; RAB1A; RAB30; RAB30-DT; RABGAP1L; RANGAP1; REXO2; RFX1; RIC8B; RIPOR3; RNA5SP52; RNU6-1276P; RPIA; RPL19P16; RPRD1B; RSBN1; SAE1; SAP30; SAP30-DT; SCAF1; SCAMP5; SCNM1; SCOC; SCRIB; SEMA4B; SEPTIN4; SIAH1; SLC25A12; SLC25A6; SLC4A2; SLC7A5; SLMAP; SLX4IP; SMIM13; SNAP25-AS1; SNX10; SPINK4; ST3GAL2; STARD10; STAU1; STXBP5-AS1; SUGP1; SULT2B1; TANC1; TARBP2; TBX6; TCEA2; TEAD1; TEX14; TFEB; THADA; TIMM21; TMEM11; TMEM115; TMEM245; TMEM69; TMIGD3; TMTC3; TNFAIP1; TPGS1; TPM4; TSSC4; TTC1; TUFT1; TULP2; UBE2V1P4; UBE3C; UCKL1; USP10; USP31; VPS37B; VRK1; WASF1; WBP2; WDPCP; WWP1; XYLT2; ZCCHC8; ZFYVE1; ZFYVE28; ZNF224; ZNF225; ZNF337; ZNF341-AS1; ZNF444","ABRAXAS2; ACTG1; ADD1; ADGRB3; ADGRV1; ALG1L13P; AP2S1; ARNTL; B3GNT5; BOD1; C1orf159; CALCOCO1; CBFA2T3; CCDC88A; CD22; CFL1; CLPB; CRADD; CYB561D2; DYNC1I2; ENSG00000235530; FASN; FAXDC2; FEM1A; GFER; GNG4; GSPT1; GTPBP2; H3-3B; HMBOX1; IFRD2; IL1R1; INTS9; KBTBD7; KCNAB1; KCTD20; LTBP1; MIR5093; MTMR4; NOSIP; NOXO1; NPRL2; NUDT13; PAFAH2; PANK4; PCBP1-AS1; PLAU; PRR5; PTP4A2; PXT1; SCAMP5; SLC4A2; SLC7A5; SLX4IP; SMPD3; SNAP25-AS1; SPINK4; ST3GAL2; TBX6; TCEA2; TMEM11; TMEM115; TMEM140; TMEM268; TTC33; TULP2; UBE3C; UCKL1; UNK; WIPF1","ABHD2; ACKR2; ACTN4; ACYP2; ADK; AMBP; ANXA9; AOC4P; APIP; APOA1; APOC3; AXDND1; BABAM2; BATF; BICRA; BOLA1; C1QTNF6; CA12; CCN5; CDK5RAP1; CELSR1; CENPA; CFAP45; CHASERR; CHN2; CLASP1; CLHC1; CMSS1; CNPY2; CNPY2-AS1; COMMD1; COX7A2L; COX7C; CPVL; CTR9; DDX23; DDX47; DECR1; DGAT1; DGKZ; DHRS2; DLC1; DNAAF3; DNAJC5B; DNAL4; DNPEP; DOK7; DPH6; DSCAM-AS1; DTNA; DYNC2I2; EEPD1; EHD1; EIF2S3; ENSG00000238005; ERC1; ERCC1; ESPN; ETV4; EXT1; FAM13A; FAM178B; FAM214A; FGA; FIS1; FMN1; GPBP1L1; GREB1; H2AC11; H4C8; HDAC8; HMBOX1; HNMT; HSD17B2; HSPH1; HYAL3; IGF1R; INHA; INTS9; ISG20; JAG1; KANSL3; KLHDC2; LDLRAD4; LFNG; LINC00663; LINC00886; LINC00963; LINC01016; LINC01101; LINC01588; LINC02237; LINC02363; LINC02428; LINC02832; LONP2; LRMDA; LYSMD1; MACF1; MAPKAP1; MARK4; MIR4259; MNAT1; MRPL45; MSMO1; MUTYH; MYOF; NCAM2; NCLN; NDC1; NFE2L2; NHLRC2; NLRP7P1; NRDE2; NRP1; NUDT13; OBSL1; PCBP1-AS1; PDZK1; PES1; PGC; PPP1R37; PPP1R3D; PRKCI; R3HDM1; RAB31; RABGAP1L; RAD51B; RAD52; RAPGEFL1; RARA; RNF139-DT; RNU6-813P; RPL37A; RPL7P30; RPL7P41; RPRD1B; RPS20P25; RPS27A; RRBP1; SCNM1; SCRIB; SEC11C; SEC24D; SELENOP; SEMA3C; SERBP1; SERPINA1; SERPINA11; SLC25A19; SLC25A21; SLC35E3; SLC39A11; SLC45A4; SLCO2A1; SLX4IP; SMAD3; SMAD7; SMARCAL1; SMARCE1; SMIM2-AS1; SNRPF; SNX5; SPRED2; SRI; STARD10; STXBP5-AS1; SUGP2; TACR1; TANK-AS1; TAOK3; TBCK; TFF1; THADA; THSD4; TJP1; TLE3; TM4SF1-AS1; TMEM69; TNS1; TOE1; TOMM40; TRMT11; TTC30B; TTC33; UBE2I; UTRN; VAX2; VKORC1; VMP1; VPS50; WWC1; ZC4H2; ZFHX3; ZNF169; ZNF224; ZNF354B; ZNF37A; ZNF37BP; ZNF565; ZNF56P; ZNF609; ZRANB2-AS2; ZRANB3; ZSCAN5A; ZSCAN5A-AS1","ADD1; AFF3; AIFM1; AP2S1; AP3B1; AP3S2; ARNTL; AXIN2; BANP; BFAR; BOD1; BRINP2; CALCOCO1; CENPX; CHD6; CHD9; CHMP4B; CIC; CLPB; CMAHP; CMSS1; COA1; COMMD1; CRACDL; CTBP1; CTDP1; DGKZ; DHX8; DLGAP1-AS1; DPP9; DYM; EGR2; EHF; EMCN; ENSG00000227896; ERC1; ERCC1; ERP44; EVI2A; FASTKD1; FBXL19; FBXL19-AS1; FBXO15; FBXO38; FBXO38-DT; FEM1A; FHAD1; FOXK1; FOXK2; GALK2; GATB; GSK3A; H3-3B; HDAC9; HSD17B12; HSH2D; HSPB6; IFRD2; IFTAP; IGSF8; INSR; INVS; IRX5; KANSL3; KAT8; KCNH1; KCTD1; KIAA0825; KIF16B; KLF5; LINC00963; LINC01132; LINC01547; LINC02240; LINC02904; LRRC45; MADCAM1; MCRIP2; MDH1; METTL26; MIR100HG; MPP7; MTRF1; MXI1; MYCBP2; NCLN; NDC1; NDUFAF8; NEK10; NIPSNAP2; NMNAT1; NR1H2; NTHL1; NUDT16-DT; OAT; OTUD7B; PADI1; PAFAH1B2P2; PANK4; PARN; PCAT14; PCBP3; PEMT; PHF12; PIM1; PIP4K2A; PIPOX; PLAC1; PPP1R37; PRDM2; PROSER3; PSAP; PWWP2A; RAB5B; RABEP1; RAD51B; RAPGEF1; RBM47; RER1; RFX1; RPS26; RRAS; SAXO1; SCRIB; SEC22C; SEPTIN9; SGMS1; SHB; SLC1A5; SLX4IP; SMAD3; SMARCE1; SNAPC4; SP2; SP2-DT; SPRED2; STAU1; STK11; STK17A; STK25; TCEA2; TEPSIN; THADA; THNSL2; TIMM21; TJP3; TMEM245; TMEM94; TPM4; TRAPPC9; TSC2; UNK; VDR; VMP1; VPS13D; VPS50; VRK1; WAPL; WDPCP; XAB2; XPOT; ZC3HAV1; ZMIZ1; ZMIZ1-AS1; ZNF341-AS1; ZNF37BP; ZNF608; ZNF609; ZNF669","ABHD11; ADCY9; AHSA1; AP2B1; BANP; BCL3; BRCA1; CAND1; CBX3; CCDC9B; CDC42EP4; CENPP; CHASERR; CLASP1; CLHC1; CMBL; CMSS1; CNPY2-AS1; CNTD1; COX7A2L; CPEB4; CSNK2B; CTDP1; CYP1B1-AS1; DNAJA3; EARS2; EHD1; EHF; ELOB; ENSG00000238005; ESRP2; FADS2; FAM13A; FBXL19; FEM1A; FEN1; G3BP1; GMPR2; GOLPH3; GPANK1; GPRC5A; GSPT1; GTF2I; GTPBP2; H3-3B; HDAC1; HDGF; HIRIP3; HMG20A; HNRNPA2B1; HYAL3; ID1; IDH3A; ING3; INO80E; KCTD21-AS1; KIAA0319L; KRT8; LARS1; LINC00886; LINC02026; LINC02363; LINC02889; LRPPRC; MCCC2; MKLN1; MRPL46; MRPS11; NBR1; NDC1; NEDD8; NEDD8-MDP1; NOD1; NXPE2; P2RX4; PAN2; PCBP1-AS1; POLR1G; PPFIBP2; PPP1R13L; PRKCI; PRLR; PRPF31; RAB30; RAB30-DT; RBM47; RETREG1; RFX3-AS1; RIC8B; RNA5SP396; RPS27A; RRAS; S100P; SAE1; SCAF1; SCARB1; SEPTIN9; SH3RF2; SHLD3; SIK3; SLC25A6; SLC38A1; SLMAP; SMIM14; SNX8; TBX6; TDO2; TFPT; THADA; THRAP3; TMEM258; TMEM267; TMEM268; TMEM54; TRAF3IP2-AS1; TRAPPC13; TRIM23; TRIP13; TTC30B; TTC7A; UBR2; UNK; VASP; VIPAS39; VPS50; VTI1A; XPOT; ZBTB7C; ZMYM6; ZNF133; ZNF143; ZNF341-AS1; ZNF436; ZNF436-AS1; ZSCAN5A; ZSCAN5A-AS1","ABCA3; ACTG1; ADD1; ADGRB3; ALG1L13P; ANK3; ANPEP; AP2S1; ARNTL; ASAP3; BCL7C; BOD1; BRWD1; BUB3; CAMTA1; CASP2; CASZ1; CBFA2T3; CCDC88A; CD22; CDK5; CDKL3; CENPA; CFAP99; CFL1; CHMP4B; CLEC16A; CLPB; COX16; CRAMP1; CTSB; CXCL16; DGCR8; DPP9; DYNC1I2; EHD1; ENKUR; ENSG00000229327; ENSG00000235530; ENSG00000238005; ENY2; EPC1; EXPH5; FASN; FAXDC2; FBXO15; FEM1A; FGD6; FHL2; FOXK2; FRG1-DT; GAB2; GET4; GNG4; GPR158; GSK3B; GZF1; H3-3B; HIF1A; HMBOX1; HSP90B1; IFRD2; IFT140; IFT20; IGF1R; IL1R1; INTS9; IRF1-AS1; KCNAB1; KRT13; LIMA1; LINC01275; LINC02093; LINC02613; LINC02863; LYPLA1; LYSMD1; MAG; MDH1; MGRN1; MLLT10; MLXIP; MRFAP1; MRGBP; MTMR4; MUS81; MXD1; MYL6; MYL6B-AS1; NFX1; NOS3; NOSIP; NPEPPS; NPEPPSP1; NRXN3; NUCB2; NUDCD1; ODAD3; OLA1; ORMDL3; PAFAH2; PANK4; PCBP1-AS1; PGP; PHF12; PHF21A; PLAU; PPP1R12C; PPRC1; PPT1; PRKCE; PRKD1; PRR5; PTK2; PTPN22; PTPRN2; PXMP4; RAB30; RAB30-DT; RABGAP1L; RBL1; RERE; RIPOR3; RPS26; SAP30; SAP30-DT; SCAMP5; SCNM1; SCRIB; SFI1; SIAH1; SLC25A6; SLC35F6; SLC4A2; SLC7A5; SLMAP; SLX4IP; SMG1P3; SNAP25-AS1; SND1; SP2-AS1; SPINK4; ST3GAL2; ST7; STK25; STX18-AS1; SULT2B1; TANC1; TBX6; TEX14; TIMM21; TMEM11; TMEM268; TNFAIP1; TPH2; TTC39A; TTC41P; TULP2; UBE2V1P4; UBE3C; UBE3D; UNK; USP31; WDPCP; ZEB2; ZFYVE1; ZFYVE28; ZNF331; ZNF341-AS1","ACAP1; ACTN1; ACTR1A; ADAMTS17; ADAT3; ADPRHL1; AHCTF1; AHCYL2; ALG14; ANKRD26; AP2B1; AP2S1; APOLD1; ARHGAP15; ARRDC4; ARSG; ARV1; ASXL1; ATP5F1D; ATP6V1D; B4GALT1; BAG3; BBIP1; BCAR3-AS1; BCL9; BOD1; BUB3; BZW2; C11orf58; C19orf44; CACNA2D4; CALCOCO1; CALR3; CAMK1D; CAPN15; CASC15; CCDC12; CCDC148; CCDC171; CCDC88C; CDC6; CDK5RAP1; CDK6; CDKAL1; CENPJ; CFDP1; CFTR; CHASERR; CHCHD3; CHMP4B; CLUH; COA1; COX7A2L; CSNK2B; CTDP1; CTU1; DCAKD; DDX23; DFFA; DGLUCY; DIAPH3; DLC1; DLGAP1-AS1; DMWD; DPP9; EIF4G3; ELOC; ENSG00000224311; ENSG00000262050; ERP44; ERVK3-1; ESCO2; FAT1; FBXL18; FBXL19; FBXO15; FBXW11; FIS1; FNTB; FRG1-DT; FTO; GARNL3; GATAD2A; GATB; GPANK1; GPHN; GSPT1; HBS1L; HDGF; HHIP; HIKESHI; HSDL2; HTT; IFT20; IGF1R; INVS; IRX5; ITGB2; JADE1; JPX; KBTBD7; KEAP1; KIAA0319L; LCMT1; LDLRAD4; LEF1-AS1; LIN37; LINC00235; LINC02643; LTA4H; MAD2L1BP; MADCAM1; MAX; MDH1; MED13L; MEF2D; MFSD2A; MRGBP; MRPL24; MTPN; NFIA; NFKBIA; NICN1; NMT1; NOSIP; NREP; NUCB2; ORMDL3; PAK1; PARN; PDE10A; PEX26; PHB; PHF23; PKM; PLEKHG2; PPFIBP2; PPP1R12C; PRKCH; PRKCI; PRSS27; PTDSS2; RAB40C; RARA; RBL1; RNF215; RNU6-92P; RPGRIP1L; RPS26; RSBN1; RTEL1; RTEL1-TNFRSF6B; SCAF1; SCAMP4; SCRIB; SEC31A; SEPTIN9; SH3GL3; SIPA1L3; SLC25A12; SLC44A1; SMAD3; SMOC1; SMPD3; SMURF2; SOX6; SPIDR; SPPL3; ST3GAL2; STIM1; STMN3; TBC1D10A; TFAP4; TIMM21; TMEM115; TMEM70; TMEM94; TNFAIP1; TPRG1; TRAF3IP2-AS1; TRAPPC8; TSPAN5; TSPAN5-DT; TSSC4; TTC13; TTLL4; TUBD1; UBE2I; URB1; URB1-AS1; USP48; VCPIP1; VGLL4; VPS13B; VRK1; WDPCP; WDR87; WDTC1; XRCC5; XYLT1; ZCCHC2; ZCCHC8; ZNF341-AS1; ZNF37A; ZNF37BP; ZNF586; ZW10","ABCC13; ABR; ADGRV1; AIFM1; AKAP10; ANKRD2; APBB2; ARAP1; ARHGEF38; ASB13; ATXN7L1; BACE1; CAV2; CBX1; CCDC117; CCN5; CDHR3; CDK5RAP1; CLPB; CNTD1; COA3; COPS7B; CORO2A; CSNK1G3; CTPS2; CYP1B1; CYP1B1-AS1; CYREN; DEF6; DNAH11; DNPEP; DPP9; EBF4; EFNA5; ELOVL5; EML5; ERCC1; ESR2; EXOSC3; EYA3; FAM178B; FHIP1A; FNTB; FZD1; FZD4-DT; GALNT9; GINS3; GNA13; GPD1L; GSE1; HDGF; HIF1A; HOOK1; IL1RAP; IL4R; ILVBL; IPPK; ITGAL; ITGB2; KCNAB1; KIAA0100; KIAA0825; KLF4; KLHDC2; KLHL13; LAD1; LFNG; LINC01716; LINC02324; LMTK3; MARS1; MBNL1; MECOM; MED15; MICAL3; MIR9-1HG; MLYCD; MPO; MRPL45; MRPL45P2; MXI1; NDUFB9; NICN1; NOSIP; NRDE2; NT5DC3; PAPOLA; PCBP1-AS1; PDE11A; PDE6D; PEX14; PHF21A; PLEKHG2; PNPLA6; POU2F3; PPP1R37; PRKCI; PRSS8; PTPRF; RAB3D; REEP1; RGS3; RGS9; RNF220; RNF24; RPL39L; SEC1P; SERHL; SH3GL3; SLC14A2; SLC9A7; SMG7; SNAP25-AS1; SPECC1L; SPECC1L-ADORA2A; SPOPL; SPRY4; SRCIN1; SRI; STK3; SUN1; TATDN1; TBC1D31; TH; TJP3; TMEM205; TNIK; TPRN; TRAPPC8; TSC22D2; TUBA1C; UBE2I; UIMC1; UNKL; UPP2; USP48; VASP; VAT1L; VEPH1; VGLL4; VPS26C; WWP2; ZBED5-AS1; ZNF146; ZNF181; ZNF213; ZNF341; ZNF521; ZNF749","ABCC4; ABHD2; ACSL3; ACTN1; ANKH; ARID5B; B4GALT1; BMPR1B; CDK6; ELL2; ELOVL5; GPD1L; GSR; HERC3; HERC3; HOMER2; HPGD; INPP4B; KRT19; KRT8; MERTK; PDLIM5; PMEPA1; PTK2B; PTPN21; RRP12; SEC24D; SELENOP; SMS; SPDEF; STEAP4; STK39; TPD52; UBE2I; VAPA; XRCC5; ZMIZ1","ABR; ACP2; AGPS; AHSA1; ARV1; ASCC2; ATAD2; ATF2; BANP; BRD8; CADPS2; CAPN10; CCDC146; CFAP99; CNBD2; CYB5D1; DMPK; EHD1; ENSG00000186019; ENSG00000213963; EPB41L5; ESPN; ESRP2; FAM174B; FAM185BP; FBP1; FNTB; FOXA1; FOXP1; G3BP1; GANC; GRHL2; GRHL3; GSE1; GSK3A; H3-3B; HMBOX1; HNRNPL; IFRD2; IK; INTS9; IQCD; ISG20; JADE1; KIAA1671; KIF20A; LAMA3; LAMA5; LFNG; LLGL2; LNX1; LRIG2; LSR; MAL2; MBP; ME3; METTL15; MGRN1; MIR3621; MLLT3; NAA38; NDC1; NDUFA2; NEDD9; NFE2L2; NFKBIA; NOSIP; NPEPPS; NPEPPSP1; OAS1; ODC1-DT; PCDH12; PIM1; PRKCH; PRR15L; R3HDM1; RPL18A; SEC11C; SEMA4B; SEPTIN9; SHC2; SIN3A; SMAD3; SNX8; STARD10; STRBP; SULT2B1; SYT8; TMEM184A; TMEM87A; TOB1; TPCN1; TRAF4; TTC13; TTC6; TUFT1; USP48; VIPAS39; VPS26C; WRAP53; ZFYVE28; ZNF133; ZNF223; ZNF225; ZNF354B; ZNF749; ZRANB3","ADGRG1; ALDOA; ALG1L13P; AP2S1; ARHGAP25; ARID4B; ARMC2; ARNTL; ASB3; AZI2; C17orf100; C19orf44; CALR3; CC2D2A; CCDC191; CCDC40; CCDC91; CDH23; CFAP298-TCP10L; CFAP45; CFAP99; CFDP1; CHST11; CIC; CLMP; CLPB; CRIP2; CRTC2; CSF1; CTU1; DDX23; DHX8; DNAL4; DUS3L; DUSP16; DYNC1I2; DYRK1B; EHD1; ELAPOR1; EMC3; ENSG00000231170; ENSG00000236866; ENY2; ERCC1; FAM13A; FEM1A; GALK2; GEMIN6; GGPS1; GRK4; GSE1; H4C8; HBP1; HIPK1; HMBOX1; HSP90B1; IFRD2; IFT122; IFT20; IK; ING3; INTS9; JUND; KAZN; KCND2; KLF13; KLF4; KRT8; LAMTOR2; LCORL; LINC01275; LINC01876; LINC02246; LINC02532; LINC02863; LINS1; LMNA; MARCHF8; MBTPS1; MDH1; MIR3188; MIR4259; MIR4290HG; MRPL24; MYL3; NCOR2; NDUFA2; NFATC2; NFIA; NOD2; NOP14; NUDCD1; NUDT13; PANK4; PES1; PIM1; PPP1R12C; PSTPIP2; QTRT2; RABGAP1L; RARA; RERE; RFX1; RFX3-AS1; RPRD1B; RPS17P7; RRP12; RSRC1; SELENOP; SF3B5; SIK3; SLC44A1; SNX29; SPINK4; ST7; STXBP5-AS1; SUN1; SYNE2; TBC1D16; TCEA2; TCEANC; TENT5C; TFAP4; THADA; TIPARP; TMEM115; TNFAIP1; TOR2A; TTC41P; TUFT1; UBQLN4; VRK3; WBP2; WDPCP; YTHDC1; ZDHHC14; ZFP36L1; ZFYVE28; ZNF37BP; ZNF473; ZSWIM8","ABCC5; ABCC8; ABCD3; ACOX1; ACSL1; CADM1; CLN8; CRAT; CTBP1; DHRS3; DLG4; EHHADH; ELOVL5; ERCC1; ESR2; FIS1; GSTK1; HAO2; HSD17B4; IDH2; ISOC1; LONP2; MLYCD; NUDT19; PABPC1; PEX14; PRDX1; SEMA3C; SERPINA6; SIAH1; SLC25A17; SLC25A19; STS; SULT2B1","ACOX1; ACTG1; ADD2; ADD3; AHCYL2; ALDOA; ASAP3; ASXL1; ATAD2; ATP5MF; ATP5MF-PTCD1; BCAT2; BIRC5; BORCS5; BRWD1; C1orf159; C6orf141; CAP1; CASZ1; CCDC88A; CCNT2; CCNT2-AS1; CD9; CDC42; CDC42EP4; CECR2; CELSR1; CENPJ; CENPO; CERT1; CFAP298-TCP10L; CFL1; CIC; CMSS1; CNOT4; COPS7B; COX16; CPEB4; DISC1; DLEU1; DPP9; DST; DYNC2I2; DYRK1B; EIF2S3; ENSG00000204055; ENSG00000267575; ERP44; ETV4; EXOSC3; FAM168A; FGD6; GABARAP; GANC; GINS3; GLI3; GPRC5A; GSE1; GSPT1; GTF2H4; HBS1L; HNRNPL; HSPB6; HYAL3; IL12A; INVS; JPX; KANSL3; KAT6B; KATNAL1; KLC1; KRT8; LARGE1; LINC00662; LINC01006; LINC01144; LINC01275; LINC01547; LINC02026; LRP6; LRRC1; LRRFIP2; LSR; LTA4H; LYPD5; MACIR; MAN2C1; MARCHF8; MARS1; MBD2; MCF2L; METTL15; MIB1; MICAL3; MIPEP; MIR4530; MYL6; MYL6B-AS1; NAT9; NCAM1; NDUFAF6; NEIL1; NMNAT1; NR2F6; OTUD7B; PAFAH2; PDCD6; PDCD6IP; PDE6D; PEMT; PHF12; PHF20; PIDD1; PIM1; PIPOX; PMP22; PNPLA7; POLK; PPCDC; PPP2CB; PPT1; PRDX1; PRELID2; PRKAA1; PROSER3; PTCD1; PTP4A2; PTPN21; PTRHD1; PUF60; RABAC1; RAD52; RBPJ; RMI2; RPL37A; RPL38; RPLP2; RPS26; SAE1; SCAF1; SCAP; SEC31A; SGMS1; SLC25A6; SMG1P3; SMG7; SMIM14; SNAP29; SND1; SPRED2; SPRY4; SRCIN1; STAT3; STMN3; STX18-AS1; SVIL; SYNPO2; TJP3; TMEM87A; TOB1; TOB1-AS1; TPK1; TRABD; TRAF4; TTC1; TTC33; TUBA1A; TUBA1C; UBE2H; UBTD2; UCKL1; USP31; USP32; VGLL4; ZBTB40; ZCCHC2; ZFP36; ZFYVE1; ZKSCAN2; ZMIZ1-AS1; ZMYM1; ZMYM6; ZNF133; ZNF181; ZNF213-AS1; ZNF354A; ZNF609"]},"columns":[{"id":"Gene Set","name":"Gene Set","type":"character","cell":[{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["GREB1 TARGET GENES"]},{"name":"a","attribs":{"href":null,"target":"_blank"},"children":["HALLMARK ESTROGEN RESPONSE EARLY"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["ZNF197 TARGET GENES"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["MCM2 TARGET GENES"]},{"name":"a","attribs":{"href":null,"target":"_blank"},"children":["HALLMARK ESTROGEN RESPONSE LATE"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["ZNF175 TARGET GENES"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["ZNF584 TARGET GENES"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["ZSCAN2 TARGET GENES"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["HDAC4 TARGET GENES"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["DLX6 TARGET GENES"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["ZNF589 TARGET GENES"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["ARID5B TARGET GENES"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["ZNF579 TARGET GENES"]},{"name":"a","attribs":{"href":null,"target":"_blank"},"children":["HALLMARK ANDROGEN RESPONSE"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["SNAI1 TARGET GENES"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["FOXJ2 TARGET GENES"]},{"name":"a","attribs":{"href":null,"target":"_blank"},"children":["HALLMARK PEROXISOME"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["KLF7 TARGET GENES"]}],"html":true,"minWidth":150},{"id":"Description","name":"Description","type":"character","cell":[{"name":"span","attribs":{"title":"Genes containing one or more binding sites for UniProt:Q4ZG55 (GREB1) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization."},"children":["Genes containing one or more binding sites for UniProt:Q4ZG55 (GREB1) in their promoter regions (..."]},{"name":"span","attribs":{"title":"Genes defining early response to estrogen."},"children":["Genes defining early response to estrogen."]},{"name":"span","attribs":{"title":"Genes containing one or more binding sites for UniProt:O14709 (ZNF197) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization."},"children":["Genes containing one or more binding sites for UniProt:O14709 (ZNF197) in their promoter regions ..."]},{"name":"span","attribs":{"title":"Genes containing one or more binding sites for UniProt:P49736 (MCM2) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization."},"children":["Genes containing one or more binding sites for UniProt:P49736 (MCM2) in their promoter regions (T..."]},{"name":"span","attribs":{"title":"Genes defining late response to estrogen."},"children":["Genes defining late response to estrogen."]},{"name":"span","attribs":{"title":"Genes containing one or more binding sites for UniProt:Q9Y473 (ZNF175) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization."},"children":["Genes containing one or more binding sites for UniProt:Q9Y473 (ZNF175) in their promoter regions ..."]},{"name":"span","attribs":{"title":"Genes containing one or more binding sites for UniProt:Q8IVC4 (ZNF584) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization."},"children":["Genes containing one or more binding sites for UniProt:Q8IVC4 (ZNF584) in their promoter regions ..."]},{"name":"span","attribs":{"title":"Genes containing one or more binding sites for UniProt:Q7Z7L9 (ZSCAN2) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization."},"children":["Genes containing one or more binding sites for UniProt:Q7Z7L9 (ZSCAN2) in their promoter regions ..."]},{"name":"span","attribs":{"title":"Genes containing one or more binding sites for UniProt:P56524 (HDAC4) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization."},"children":["Genes containing one or more binding sites for UniProt:P56524 (HDAC4) in their promoter regions (..."]},{"name":"span","attribs":{"title":"Genes containing one or more binding sites for UniProt:P56179 (DLX6) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization."},"children":["Genes containing one or more binding sites for UniProt:P56179 (DLX6) in their promoter regions (T..."]},{"name":"span","attribs":{"title":"Genes containing one or more binding sites for UniProt:Q86UQ0 (ZNF589) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization."},"children":["Genes containing one or more binding sites for UniProt:Q86UQ0 (ZNF589) in their promoter regions ..."]},{"name":"span","attribs":{"title":"Genes containing one or more binding sites for UniProt:Q14865 (ARID5B) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization."},"children":["Genes containing one or more binding sites for UniProt:Q14865 (ARID5B) in their promoter regions ..."]},{"name":"span","attribs":{"title":"Genes containing one or more binding sites for UniProt:Q8NAF0 (ZNF579) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization."},"children":["Genes containing one or more binding sites for UniProt:Q8NAF0 (ZNF579) in their promoter regions ..."]},{"name":"span","attribs":{"title":"Genes defining response to androgens."},"children":["Genes defining response to androgens."]},{"name":"span","attribs":{"title":"Genes containing one or more binding sites for UniProt:O95863 (SNAI1) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization."},"children":["Genes containing one or more binding sites for UniProt:O95863 (SNAI1) in their promoter regions (..."]},{"name":"span","attribs":{"title":"Genes containing one or more binding sites for UniProt:Q9P0K8 (FOXJ2) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization."},"children":["Genes containing one or more binding sites for UniProt:Q9P0K8 (FOXJ2) in their promoter regions (..."]},{"name":"span","attribs":{"title":"Genes encoding components of peroxisome."},"children":["Genes encoding components of peroxisome."]},{"name":"span","attribs":{"title":"Genes containing one or more binding sites for UniProt:O75840 (KLF7) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization."},"children":["Genes containing one or more binding sites for UniProt:O75840 (KLF7) in their promoter regions (T..."]}],"minWidth":150},{"id":"%","name":"% Mapped","type":"numeric","cell":["40.8%","67.0%","36.5%","73.8%","48.5%","28.7%","38.0%","22.7%","24.8%","22.8%","24.4%","22.2%","22.8%","36.6%","22.4%","23.5%","32.7%","20.0%"],"maxWidth":80},{"id":"numDEInCat","name":"Total Mapped","type":"numeric","maxWidth":80},{"id":"numInCat","name":"Gene Set Size","type":"numeric","cell":["946","200","661","61","200","833","184","909","682","596","692","933","653","101","478","599","104","935"],"maxWidth":80},{"id":"p","name":"p","type":"numeric","cell":["2.00e-65","2.45e-50","1.64e-34","4.04e-27","1.76e-26","1.04e-18","2.44e-11","9.60e-09","2.18e-08","2.19e-08","7.43e-08","6.67e-06","1.15e-05","4.35e-05","4.90e-05","5.17e-05","6.07e-05","1.84e-04"],"maxWidth":80},{"id":"adj_p","name":"p<sub>adj<\/sub>","type":"numeric","cell":["5.50e-62","3.37e-47","1.50e-31","2.77e-24","9.69e-24","4.74e-16","9.57e-09","3.30e-06","6.02e-06","6.02e-06","1.86e-05","0.002","0.002","0.009","0.009","0.009","0.010","0.028"],"html":true,"maxWidth":80},{"id":"Mapped Genes","name":"Mapped Genes","type":"character","cell":[{"name":"span","attribs":{"title":"ABCA3; ABCC11; ABCC5-AS1; ABHD17C; ABHD2; ADAMTSL5; ADAT3; ADGRG1; AFAP1; AFF1; AGAP2; AHCYL2; AKAP1; AKAP1-DT; AKAP8L; AMZ1; ANKFN1; ANKRD2; ANXA9; ARID1B; ARNTL; ARRDC1; ARRDC4; ASCC2; ATP6V0A4; ATP8B1; ATXN7L1; BATF; BBOF1; BCAR3; BCAS3; BCL3; BCL9L; BDKRB1; BFSP2-AS1; BRWD1; C1QTNF6; C5AR2; C9orf152; CA12; CAMTA1; CASP7; CCDC159; CCDC192; CCDC40; CCDC88C; CCDC88C-DT; CCN5; CCNP; CCR3; CD22; CD276; CD59; CDH11; CDH26; CDS1; CEACAM16-AS1; CELSR1; CFAP45; CHD6; CHPT1; CIC; CLN8; CORO2A; CRACR2A; CRKL; CRNDE; CRPPA; CTSB; CYP1B1; CYP1B1-AS1; CYP4F23P; DAAM1; DDX47; DGAT2; DLC1; DLG5; DNAH11; DNAI1; DNAJC5B; DNPEP; DOCK4; DOK7; DOP1B; DRAIC; DSCAM-AS1; EHD1; EHF; ELAPOR1; EML6; ENSG00000197099; ENSG00000225676; ENSG00000229425; ENSG00000231170; ENSG00000235185; ENSG00000238005; ENSG00000251574; ENSG00000254153; ENSG00000268858; ENSG00000270571; ENSG00000271874; ENSG00000278899; ERBB2; ERCC1; ERGIC1; ESRP2; ETNK2; EVL; FAM107B; FAM126A; FAM174B; FAM178B; FAM234B; FAM86B3P; FAM86EP; FAM86GP; FBRS; FBXO38; FHDC1; FMN1; FMO9P; FNTAP2; FREM2; G3BP1; GALNT10; GATAD2A; GCLM; GCOM1; GLRX5; GPRC5A; GREB1; GRHL1; GRHL2; GSPT1; HK1; HNRNPL; HSH2D; HSPB6; ID1; IFRD1; IGF1R; IGSF3; IKBKG; INHA; INPP4B; IQCD; IQSEC1; IRF1-AS1; ISG20; ITGB2; JUND; KCTD11; KDM4B; KIAA0100; KIAA1217; KRT7; KRT8; KRTAP3-1; LAD1; LAMB1; LDLRAD4; LENG1; LFNG; LIMA1; LINC00475; LINC00511; LINC00536; LINC00824; LINC00885; LINC00963; LINC01016; LINC01275; LINC01344; LINC01588; LINC01656; LINC01975; LINC02102; LINC02130; LINC02408; LINC02518; LINC02620; LINC02652; LINC02707; LLGL2; LMCD1; LPP; LRG1; LRIG2; LY6E-DT; MAF1; MANEAL; MAP4K3-DT; MAPK10; MAPKAP1; MCCC2; MCF2L; MCF2L2; MCRIP2; MGRN1; MICOS10; MICOS10-NBL1; MIDEAS; MIR3168; MIR3188; MIR3622B; MIR4259; MIR4530; MIR5093; MIR550A1; MIR550A2; MIR99AHG; MLLT3; MSMO1; MTA1; MTMR12; MTMR4; MTUS1; MUC5B; MYZAP; NAALADL2; NAIF1; NCAM2; NCF4-AS1; NDUFAF8; NFATC2; NFKBIA; NINL; NIPSNAP2; NLRP7P1; NMT1; NOP14-AS1; NRP1; NT5DC3; NUDT16-DT; OSBPL10; OVOL2; OXR1; OXR1-AS1; PAK4; PAM; PARD3; PBX1; PCBP1-AS1; PCDH1; PDGFD; PDZK1; PGP; PIK3AP1; PIK3C2G; PITX1-AS1; PKD2L2; PKIB; PKP4-AS1; PLAC1; PNKD; PNPLA7; POU2F3; PPM1L; PRDX3P2; PREX1; PRKCD; PRKCH; PROSER3; PRR15L; PRSS22; PRSS23; PRSS27; PTPRH; PWWP2A; PXDN; PXMP4; RAB5B; RAD51B; RADIL; RAPGEFL1; RARA; RBBP8; RBL2; RBM20; RBM47; RFX1; RGMB; RGS3; RN7SL211P; RNU6-1013P; RNU6-1105P; RNU6-813P; RPL37P4; RPL5P7; RPL7P30; RPS29P8; RPS6KA2; RPS6KL1; S100A16; SCAMP4; SCGB1D2; SCIN; SCRIB; SDK2; SEC14L2; SEPTIN9; SERPINA1; SGMS2; SHARPIN; SIDT2; SIN3A; SLC19A3; SLC1A5; SLC25A25; SLC25A36; SLC25A42; SLC29A1; SLC38A1; SLC39A1; SLC45A4; SLC6A6; SLC7A2; SLC9A8; SMAD3; SNORD114-1; SPAG17; SPECC1L; SPRED2; SPTLC2; SPTSSB; SRI; STARD10; STAT3; SULT1A1; SYBU; SYT8; TAFAZZIN; TBC1D16; TBC1D8; TEPSIN; TFAP2C; TFF1; TH; THADA; THSD4; TIPARP; TIPARP-AS1; TJP1; TLE1; TMCO3; TMEM105; TMEM160; TMEM184B; TMEM210; TMEM229B; TMEM241; TMEM94; TMPRSS3; TMTC2; TNS1; TPD52L1; TPD52L2; TPRG1; TPRN; TRAF7; TRAPPC9; TSPAN9; TTC39A-AS1; TUBA1A; TUBA1C; UIMC1; UMODL1; UNKL; URB1; UTRN; VAV3; VEPH1; VIRMA; VLDLR-AS1; VMP1; VPS26C; VWA3B; WWC1; WWP1; ZFYVE28; ZNF282; ZNF444; ZNF552; ZNF669; ZNRF1; ZRANB2-AS2"},"children":["ABCA3; ABCC11; ABCC5-AS1; ABHD17C; ABHD2; ADAMTSL5; ADAT3; ADGRG1; AFAP1; AFF1; AGAP2; AHCYL2; AK..."]},{"name":"span","attribs":{"title":"ABAT; ABCA3; ABHD2; ABLIM1; ADCY9; ADD3; AFF1; AKAP1; AMFR; ANXA9; AQP3; AR; AREG; ARL3; ASB13; B4GALT1; BCL11B; BCL2; CA12; CALCR; CBFA2T3; CCN5; CELSR1; CELSR2; CHPT1; DEPTOR; DHRS2; DHRS3; DLC1; ELOVL5; ENDOD1; ESRP2; FAM102A; FARP1; FASN; FHL2; FLNB; FRK; GAB2; GFRA1; GREB1; HES1; HR; IGF1R; IGFBP4; IL17RB; IL6ST; INHBB; ITPK1; KAZN; KCNK15; KDM4B; KLF10; KLF4; KRT13; KRT18; KRT19; KRT8; LAD1; LRIG1; MAPT; MAST4; MED13L; MICB; MLPH; MPPED2; MREG; MSMB; MYB; MYOF; NADSYN1; NAV2; NBL1; NCOR2; NRIP1; OLFML3; OPN3; OVOL2; P2RY2; PAPSS2; PDLIM3; PDZK1; PGR; PPIF; PRSS23; RAB31; RAPGEFL1; RARA; RASGRP1; RBBP8; REEP1; RET; RETREG1; RHOD; RPS6KA2; RRP12; SCARB1; SEC14L2; SH3BP5; SIAH2; SLC1A1; SLC22A5; SLC24A3; SLC2A1; SLC39A6; SLC7A2; SLC7A5; SLC9A3R1; SNX24; SOX3; SULT2B1; SVIL; SYBU; SYNGR1; SYT12; TBC1D30; TFAP2C; TFF1; TGIF2; THSD4; TIAM1; TIPARP; TJP3; TMEM164; TMPRSS3; TOB1; TPBG; TPD52L1; TSKU; TTC39A; TUBB2B; WFS1; WWC1; ZNF185"},"children":["ABAT; ABCA3; ABHD2; ABLIM1; ADCY9; ADD3; AFF1; AKAP1; AMFR; ANXA9; AQP3; AR; AREG; ARL3; ASB13; B..."]},{"name":"span","attribs":{"title":"AAMP; ACACA; ACAD8; ACP2; ACSL3; ACTG1; ADAT3; ADCY9; ADGRB3; AHCYL2; AHSA1; ALG1L13P; AMMECR1; AP2S1; ARAP1; ARNTL; ASAP3; AUP1; BANP; BCORL1; BOD1; BTBD9; BTRC; BUB1B; BZW2; C1orf159; CALCOCO1; CBFA2T3; CCDC146; CCDC163; CCDC88A; CCDC90B; CCDC90B-AS1; CCP110; CDC40; CDH23; CDK5; CDKL3; CEP290; CFAP99; CFL1; CHMP4B; CLEC16A; CNPY2; COX7A2L; COX7C; CPEB4; CPQ; CRADD; CRTC2; CYB561D2; DDX23; DENND5B; DFFA; DGCR8; DYNC1I2; ELOC; ENSG00000186019; ENSG00000229327; ENSG00000233834; ENSG00000235530; ENSG00000238005; ENSG00000278390; ENY2; EPC1; ERVK3-1; FAM178B; FAM185BP; FANCC; FASN; FASTKD1; FAXDC2; FBXL19; FBXO15; FBXO38; FBXO38-DT; FCSK; FYCO1; GAB2; GALM; GFER; GFUS; GNG4; GPKOW; GPR158; GRHL1; GSPT1; GTF2H4; GTF2I; H3-3B; HDAC8; HIF1A; HIKESHI; HTRA2; IFRD2; IFTAP; IL1R1; ING3; IRF1-AS1; IZUMO1; KBTBD6; KBTBD7; KCNAB1; KCNC3; KDM4C; KHK; LARP1; LARS1; LIG1; LINC00824; LINC01275; LYPLA1; LYSMD1; MADCAM1; MANEAL; MAPT; MARS1; MAX; MDH1; MITF; MLLT10; MMACHC; MRFAP1; MRPL24; MRPS22; MUS81; MYB; NOS3; NOSIP; NOXO1; NPEPPS; NPEPPSP1; NPRL2; NRDE2; NUDCD1; NUDT13; ODAD3; PAFAH2; PANK4; PCBP1-AS1; PGP; PHB; PHF12; PHF13; PHLPP1; PIK3AP1; PIPOX; PLAU; PLEKHA1; PNKD; PPP1R12C; PPP2R2C; PRPF31; PRSS27; PTDSS2; PTK2; PTPRN2; RABAC1; RABGAP1L; RANGAP1; RBL1; RBM6; RNASEH2C; RNF144A; RPP40; RPS26; RXRA; SAP30; SAP30-DT; SBK1; SCAF1; SCAMP4; SCAMP5; SCNM1; SEC61A1; SEL1L; SETMAR; SFI1; SGPL1; SIAH1; SIPA1L3; SLC4A2; SLMAP; SLX4IP; SMARCAL1; SMIM2-AS1; SMPD3; SNAP25-AS1; SNAPC3; SPINK4; SRCIN1; ST3GAL2; ST7; STARD10; STAT3; STAU1; STK25; SYNGR1; SYT3; TADA2A; TBX6; TCEA2; TFPT; THA1P; THADA; THYN1; TIMM21; TMCC1; TMCC1-DT; TMEM11; TMEM115; TMEM160; TMEM70; TMTC3; TPGS1; TPH2; TRABD; TULP2; UBE2V1P4; UBE3C; UCKL1; UNK; USF2; USP31; VCPIP1; VIPAS39; WASF1; WDPCP; WDR87; WWP1; XYLT2; ZFYVE1; ZFYVE28; ZNF221; ZNF225; ZNF32; ZNF331; ZNF341-AS1; ZNF37BP; ZNF444; ZSWIM9"},"children":["AAMP; ACACA; ACAD8; ACP2; ACSL3; ACTG1; ADAT3; ADCY9; ADGRB3; AHCYL2; AHSA1; ALG1L13P; AMMECR1; A..."]},{"name":"span","attribs":{"title":"AAMP; AP2S1; BOD1; C19orf44; C1orf159; CALCOCO1; CALR3; CHMP4B; CLPB; CRTC2; CYB561D2; DDX23; DESI2; DFFA; ENSG00000235530; FAM13A; FBXL19-AS1; FEM1A; GSK3A; GSPT1; GTF2H4; H3-3B; IFRD2; MADCAM1-AS1; NCLN; NDC1; NPRL2; ORMDL3; PANK4; PEMT; PHB; PHF23; PIPOX; PNKD; ST3GAL2; TCEA2; THA1P; TMEM11; TMEM115; TSSC4; TUBA1C; UCKL1; UNK; ZFYVE1; ZNF341-AS1"},"children":["AAMP; AP2S1; BOD1; C19orf44; C1orf159; CALCOCO1; CALR3; CHMP4B; CLPB; CRTC2; CYB561D2; DDX23; DES..."]},{"name":"span","attribs":{"title":"ABCA3; ABHD2; ACOX2; ADD3; AFF1; AMFR; ANXA9; AREG; ARL3; BATF; BCL2; CA12; CALCR; CCN5; CD9; CDC6; CDH1; CELSR2; CHPT1; CHST8; DHRS2; DLG5; ELOVL5; EMP2; FAM102A; FARP1; FLNB; FRK; GFUS; HMGCS2; HOMER2; HR; IDH2; IGFBP4; IGSF1; IL17RB; IL6ST; ISG20; ITPK1; JAK1; KIF20A; KLF4; KRT13; KRT19; LARGE1; LLGL2; LSR; MAPT; MICB; MYB; MYOF; NBL1; NCOR2; NRIP1; OPN3; OVOL2; PAPSS2; PDLIM3; PDZK1; PGR; PLAC1; PPIF; PRKAR2B; PRLR; PRSS23; PTGER3; RAB31; RABEP1; RAPGEFL1; RBBP8; RET; RPS6KA2; S100A9; SCARB1; SCUBE2; SERPINA1; SIAH2; SLC22A5; SLC24A3; SLC29A1; SLC7A5; SLC9A3R1; SNX10; SOX3; ST14; SULT2B1; TFAP2C; TFF1; TH; TIAM1; TJP3; TMPRSS3; TOB1; TPBG; TPD52L1; WFS1; ZFP36"},"children":["ABCA3; ABHD2; ACOX2; ADD3; AFF1; AMFR; ANXA9; AREG; ARL3; BATF; BCL2; CA12; CALCR; CCN5; CD9; CDC..."]},{"name":"span","attribs":{"title":"AAMP; ABCA3; ACP2; ACSM3; ACTG1; ADD1; ADGRB3; AGPS; AHCYL2; ALG1L13P; ALKBH8; ALPK1; AMMECR1; ANKRD17; ANO4; AP2S1; ARAP1; ARHGAP24; ARHGAP25; ARHGDIA; ARNTL; ASAP3; ATP5F1D; B3GNT5; BANP; BCL7C; BIRC5; BOD1; BRWD1; C12orf43; C19orf44; C1orf159; C2CD3; CALCOCO1; CALR3; CAMTA1; CAND1; CAPN15; CCDC159; CCDC25; CCDC88A; CCT6B; CDC40; CDH23; CEP290; CFAP99; CFL1; CHMP4B; CLASP1; CLEC16A; CLPB; COX7C; CRTC2; DDX23; DFFA; DGCR8; DRAM2; DYNC1I2; EBAG9; EEF2K; ENSG00000186019; ENSG00000213963; ENSG00000229327; ENSG00000235530; ENSG00000238005; ENSG00000278390; EPC1; ERI1; ERP44; FASN; FAXDC2; FBXO15; FBXO38; FBXO38-DT; FBXW11; FEM1A; GAB2; GALM; GAPVD1; GFER; GFUS; GNG4; GPBP1L1; GRB10; GSPT1; GTF2H4; H4C8; HIKESHI; HMBOX1; IFRD2; IFT20; IFTAP; IGF1R; IL1R1; INTS9; INVS; ISG20; ITGAL; JOSD2; KBTBD6; KBTBD7; KCNAB1; KCTD20; KIAA0100; LARS1; LINC00235; LINC01258; LINC02863; LRP6; LUC7L; LYPLA1; LYSMD1; MAD2L1BP; MAG; MANEAL; MAP1LC3B2; MARS1; MAU2; MDH1; MGRN1; MIR3649; MLLT10; MTMR4; MUS81; MXD1; MYO18B; NAV2; NDC1; NFE2L2; NOSIP; NOXO1; NPEPPS; NPEPPSP1; PAFAH2; PAK1; PANK4; PCBP1-AS1; PEDS1-UBE2V1; PEMT; PGP; PHB; PHF12; PIPOX; PLAU; PNKD; PNPLA6; PPP1R12C; PPRC1; PRKCE; PRKCH; PRR5; PTDSS2; PTK2; PVT1; PXT1; RAB1A; RAB30; RAB30-DT; RABGAP1L; RANGAP1; REXO2; RFX1; RIC8B; RIPOR3; RNA5SP52; RNU6-1276P; RPIA; RPL19P16; RPRD1B; RSBN1; SAE1; SAP30; SAP30-DT; SCAF1; SCAMP5; SCNM1; SCOC; SCRIB; SEMA4B; SEPTIN4; SIAH1; SLC25A12; SLC25A6; SLC4A2; SLC7A5; SLMAP; SLX4IP; SMIM13; SNAP25-AS1; SNX10; SPINK4; ST3GAL2; STARD10; STAU1; STXBP5-AS1; SUGP1; SULT2B1; TANC1; TARBP2; TBX6; TCEA2; TEAD1; TEX14; TFEB; THADA; TIMM21; TMEM11; TMEM115; TMEM245; TMEM69; TMIGD3; TMTC3; TNFAIP1; TPGS1; TPM4; TSSC4; TTC1; TUFT1; TULP2; UBE2V1P4; UBE3C; UCKL1; USP10; USP31; VPS37B; VRK1; WASF1; WBP2; WDPCP; WWP1; XYLT2; ZCCHC8; ZFYVE1; ZFYVE28; ZNF224; ZNF225; ZNF337; ZNF341-AS1; ZNF444"},"children":["AAMP; ABCA3; ACP2; ACSM3; ACTG1; ADD1; ADGRB3; AGPS; AHCYL2; ALG1L13P; ALKBH8; ALPK1; AMMECR1; AN..."]},{"name":"span","attribs":{"title":"ABRAXAS2; ACTG1; ADD1; ADGRB3; ADGRV1; ALG1L13P; AP2S1; ARNTL; B3GNT5; BOD1; C1orf159; CALCOCO1; CBFA2T3; CCDC88A; CD22; CFL1; CLPB; CRADD; CYB561D2; DYNC1I2; ENSG00000235530; FASN; FAXDC2; FEM1A; GFER; GNG4; GSPT1; GTPBP2; H3-3B; HMBOX1; IFRD2; IL1R1; INTS9; KBTBD7; KCNAB1; KCTD20; LTBP1; MIR5093; MTMR4; NOSIP; NOXO1; NPRL2; NUDT13; PAFAH2; PANK4; PCBP1-AS1; PLAU; PRR5; PTP4A2; PXT1; SCAMP5; SLC4A2; SLC7A5; SLX4IP; SMPD3; SNAP25-AS1; SPINK4; ST3GAL2; TBX6; TCEA2; TMEM11; TMEM115; TMEM140; TMEM268; TTC33; TULP2; UBE3C; UCKL1; UNK; WIPF1"},"children":["ABRAXAS2; ACTG1; ADD1; ADGRB3; ADGRV1; ALG1L13P; AP2S1; ARNTL; B3GNT5; BOD1; C1orf159; CALCOCO1; ..."]},{"name":"span","attribs":{"title":"ABHD2; ACKR2; ACTN4; ACYP2; ADK; AMBP; ANXA9; AOC4P; APIP; APOA1; APOC3; AXDND1; BABAM2; BATF; BICRA; BOLA1; C1QTNF6; CA12; CCN5; CDK5RAP1; CELSR1; CENPA; CFAP45; CHASERR; CHN2; CLASP1; CLHC1; CMSS1; CNPY2; CNPY2-AS1; COMMD1; COX7A2L; COX7C; CPVL; CTR9; DDX23; DDX47; DECR1; DGAT1; DGKZ; DHRS2; DLC1; DNAAF3; DNAJC5B; DNAL4; DNPEP; DOK7; DPH6; DSCAM-AS1; DTNA; DYNC2I2; EEPD1; EHD1; EIF2S3; ENSG00000238005; ERC1; ERCC1; ESPN; ETV4; EXT1; FAM13A; FAM178B; FAM214A; FGA; FIS1; FMN1; GPBP1L1; GREB1; H2AC11; H4C8; HDAC8; HMBOX1; HNMT; HSD17B2; HSPH1; HYAL3; IGF1R; INHA; INTS9; ISG20; JAG1; KANSL3; KLHDC2; LDLRAD4; LFNG; LINC00663; LINC00886; LINC00963; LINC01016; LINC01101; LINC01588; LINC02237; LINC02363; LINC02428; LINC02832; LONP2; LRMDA; LYSMD1; MACF1; MAPKAP1; MARK4; MIR4259; MNAT1; MRPL45; MSMO1; MUTYH; MYOF; NCAM2; NCLN; NDC1; NFE2L2; NHLRC2; NLRP7P1; NRDE2; NRP1; NUDT13; OBSL1; PCBP1-AS1; PDZK1; PES1; PGC; PPP1R37; PPP1R3D; PRKCI; R3HDM1; RAB31; RABGAP1L; RAD51B; RAD52; RAPGEFL1; RARA; RNF139-DT; RNU6-813P; RPL37A; RPL7P30; RPL7P41; RPRD1B; RPS20P25; RPS27A; RRBP1; SCNM1; SCRIB; SEC11C; SEC24D; SELENOP; SEMA3C; SERBP1; SERPINA1; SERPINA11; SLC25A19; SLC25A21; SLC35E3; SLC39A11; SLC45A4; SLCO2A1; SLX4IP; SMAD3; SMAD7; SMARCAL1; SMARCE1; SMIM2-AS1; SNRPF; SNX5; SPRED2; SRI; STARD10; STXBP5-AS1; SUGP2; TACR1; TANK-AS1; TAOK3; TBCK; TFF1; THADA; THSD4; TJP1; TLE3; TM4SF1-AS1; TMEM69; TNS1; TOE1; TOMM40; TRMT11; TTC30B; TTC33; UBE2I; UTRN; VAX2; VKORC1; VMP1; VPS50; WWC1; ZC4H2; ZFHX3; ZNF169; ZNF224; ZNF354B; ZNF37A; ZNF37BP; ZNF565; ZNF56P; ZNF609; ZRANB2-AS2; ZRANB3; ZSCAN5A; ZSCAN5A-AS1"},"children":["ABHD2; ACKR2; ACTN4; ACYP2; ADK; AMBP; ANXA9; AOC4P; APIP; APOA1; APOC3; AXDND1; BABAM2; BATF; BI..."]},{"name":"span","attribs":{"title":"ADD1; AFF3; AIFM1; AP2S1; AP3B1; AP3S2; ARNTL; AXIN2; BANP; BFAR; BOD1; BRINP2; CALCOCO1; CENPX; CHD6; CHD9; CHMP4B; CIC; CLPB; CMAHP; CMSS1; COA1; COMMD1; CRACDL; CTBP1; CTDP1; DGKZ; DHX8; DLGAP1-AS1; DPP9; DYM; EGR2; EHF; EMCN; ENSG00000227896; ERC1; ERCC1; ERP44; EVI2A; FASTKD1; FBXL19; FBXL19-AS1; FBXO15; FBXO38; FBXO38-DT; FEM1A; FHAD1; FOXK1; FOXK2; GALK2; GATB; GSK3A; H3-3B; HDAC9; HSD17B12; HSH2D; HSPB6; IFRD2; IFTAP; IGSF8; INSR; INVS; IRX5; KANSL3; KAT8; KCNH1; KCTD1; KIAA0825; KIF16B; KLF5; LINC00963; LINC01132; LINC01547; LINC02240; LINC02904; LRRC45; MADCAM1; MCRIP2; MDH1; METTL26; MIR100HG; MPP7; MTRF1; MXI1; MYCBP2; NCLN; NDC1; NDUFAF8; NEK10; NIPSNAP2; NMNAT1; NR1H2; NTHL1; NUDT16-DT; OAT; OTUD7B; PADI1; PAFAH1B2P2; PANK4; PARN; PCAT14; PCBP3; PEMT; PHF12; PIM1; PIP4K2A; PIPOX; PLAC1; PPP1R37; PRDM2; PROSER3; PSAP; PWWP2A; RAB5B; RABEP1; RAD51B; RAPGEF1; RBM47; RER1; RFX1; RPS26; RRAS; SAXO1; SCRIB; SEC22C; SEPTIN9; SGMS1; SHB; SLC1A5; SLX4IP; SMAD3; SMARCE1; SNAPC4; SP2; SP2-DT; SPRED2; STAU1; STK11; STK17A; STK25; TCEA2; TEPSIN; THADA; THNSL2; TIMM21; TJP3; TMEM245; TMEM94; TPM4; TRAPPC9; TSC2; UNK; VDR; VMP1; VPS13D; VPS50; VRK1; WAPL; WDPCP; XAB2; XPOT; ZC3HAV1; ZMIZ1; ZMIZ1-AS1; ZNF341-AS1; ZNF37BP; ZNF608; ZNF609; ZNF669"},"children":["ADD1; AFF3; AIFM1; AP2S1; AP3B1; AP3S2; ARNTL; AXIN2; BANP; BFAR; BOD1; BRINP2; CALCOCO1; CENPX; ..."]},{"name":"span","attribs":{"title":"ABHD11; ADCY9; AHSA1; AP2B1; BANP; BCL3; BRCA1; CAND1; CBX3; CCDC9B; CDC42EP4; CENPP; CHASERR; CLASP1; CLHC1; CMBL; CMSS1; CNPY2-AS1; CNTD1; COX7A2L; CPEB4; CSNK2B; CTDP1; CYP1B1-AS1; DNAJA3; EARS2; EHD1; EHF; ELOB; ENSG00000238005; ESRP2; FADS2; FAM13A; FBXL19; FEM1A; FEN1; G3BP1; GMPR2; GOLPH3; GPANK1; GPRC5A; GSPT1; GTF2I; GTPBP2; H3-3B; HDAC1; HDGF; HIRIP3; HMG20A; HNRNPA2B1; HYAL3; ID1; IDH3A; ING3; INO80E; KCTD21-AS1; KIAA0319L; KRT8; LARS1; LINC00886; LINC02026; LINC02363; LINC02889; LRPPRC; MCCC2; MKLN1; MRPL46; MRPS11; NBR1; NDC1; NEDD8; NEDD8-MDP1; NOD1; NXPE2; P2RX4; PAN2; PCBP1-AS1; POLR1G; PPFIBP2; PPP1R13L; PRKCI; PRLR; PRPF31; RAB30; RAB30-DT; RBM47; RETREG1; RFX3-AS1; RIC8B; RNA5SP396; RPS27A; RRAS; S100P; SAE1; SCAF1; SCARB1; SEPTIN9; SH3RF2; SHLD3; SIK3; SLC25A6; SLC38A1; SLMAP; SMIM14; SNX8; TBX6; TDO2; TFPT; THADA; THRAP3; TMEM258; TMEM267; TMEM268; TMEM54; TRAF3IP2-AS1; TRAPPC13; TRIM23; TRIP13; TTC30B; TTC7A; UBR2; UNK; VASP; VIPAS39; VPS50; VTI1A; XPOT; ZBTB7C; ZMYM6; ZNF133; ZNF143; ZNF341-AS1; ZNF436; ZNF436-AS1; ZSCAN5A; ZSCAN5A-AS1"},"children":["ABHD11; ADCY9; AHSA1; AP2B1; BANP; BCL3; BRCA1; CAND1; CBX3; CCDC9B; CDC42EP4; CENPP; CHASERR; CL..."]},{"name":"span","attribs":{"title":"ABCA3; ACTG1; ADD1; ADGRB3; ALG1L13P; ANK3; ANPEP; AP2S1; ARNTL; ASAP3; BCL7C; BOD1; BRWD1; BUB3; CAMTA1; CASP2; CASZ1; CBFA2T3; CCDC88A; CD22; CDK5; CDKL3; CENPA; CFAP99; CFL1; CHMP4B; CLEC16A; CLPB; COX16; CRAMP1; CTSB; CXCL16; DGCR8; DPP9; DYNC1I2; EHD1; ENKUR; ENSG00000229327; ENSG00000235530; ENSG00000238005; ENY2; EPC1; EXPH5; FASN; FAXDC2; FBXO15; FEM1A; FGD6; FHL2; FOXK2; FRG1-DT; GAB2; GET4; GNG4; GPR158; GSK3B; GZF1; H3-3B; HIF1A; HMBOX1; HSP90B1; IFRD2; IFT140; IFT20; IGF1R; IL1R1; INTS9; IRF1-AS1; KCNAB1; KRT13; LIMA1; LINC01275; LINC02093; LINC02613; LINC02863; LYPLA1; LYSMD1; MAG; MDH1; MGRN1; MLLT10; MLXIP; MRFAP1; MRGBP; MTMR4; MUS81; MXD1; MYL6; MYL6B-AS1; NFX1; NOS3; NOSIP; NPEPPS; NPEPPSP1; NRXN3; NUCB2; NUDCD1; ODAD3; OLA1; ORMDL3; PAFAH2; PANK4; PCBP1-AS1; PGP; PHF12; PHF21A; PLAU; PPP1R12C; PPRC1; PPT1; PRKCE; PRKD1; PRR5; PTK2; PTPN22; PTPRN2; PXMP4; RAB30; RAB30-DT; RABGAP1L; RBL1; RERE; RIPOR3; RPS26; SAP30; SAP30-DT; SCAMP5; SCNM1; SCRIB; SFI1; SIAH1; SLC25A6; SLC35F6; SLC4A2; SLC7A5; SLMAP; SLX4IP; SMG1P3; SNAP25-AS1; SND1; SP2-AS1; SPINK4; ST3GAL2; ST7; STK25; STX18-AS1; SULT2B1; TANC1; TBX6; TEX14; TIMM21; TMEM11; TMEM268; TNFAIP1; TPH2; TTC39A; TTC41P; TULP2; UBE2V1P4; UBE3C; UBE3D; UNK; USP31; WDPCP; ZEB2; ZFYVE1; ZFYVE28; ZNF331; ZNF341-AS1"},"children":["ABCA3; ACTG1; ADD1; ADGRB3; ALG1L13P; ANK3; ANPEP; AP2S1; ARNTL; ASAP3; BCL7C; BOD1; BRWD1; BUB3;..."]},{"name":"span","attribs":{"title":"ACAP1; ACTN1; ACTR1A; ADAMTS17; ADAT3; ADPRHL1; AHCTF1; AHCYL2; ALG14; ANKRD26; AP2B1; AP2S1; APOLD1; ARHGAP15; ARRDC4; ARSG; ARV1; ASXL1; ATP5F1D; ATP6V1D; B4GALT1; BAG3; BBIP1; BCAR3-AS1; BCL9; BOD1; BUB3; BZW2; C11orf58; C19orf44; CACNA2D4; CALCOCO1; CALR3; CAMK1D; CAPN15; CASC15; CCDC12; CCDC148; CCDC171; CCDC88C; CDC6; CDK5RAP1; CDK6; CDKAL1; CENPJ; CFDP1; CFTR; CHASERR; CHCHD3; CHMP4B; CLUH; COA1; COX7A2L; CSNK2B; CTDP1; CTU1; DCAKD; DDX23; DFFA; DGLUCY; DIAPH3; DLC1; DLGAP1-AS1; DMWD; DPP9; EIF4G3; ELOC; ENSG00000224311; ENSG00000262050; ERP44; ERVK3-1; ESCO2; FAT1; FBXL18; FBXL19; FBXO15; FBXW11; FIS1; FNTB; FRG1-DT; FTO; GARNL3; GATAD2A; GATB; GPANK1; GPHN; GSPT1; HBS1L; HDGF; HHIP; HIKESHI; HSDL2; HTT; IFT20; IGF1R; INVS; IRX5; ITGB2; JADE1; JPX; KBTBD7; KEAP1; KIAA0319L; LCMT1; LDLRAD4; LEF1-AS1; LIN37; LINC00235; LINC02643; LTA4H; MAD2L1BP; MADCAM1; MAX; MDH1; MED13L; MEF2D; MFSD2A; MRGBP; MRPL24; MTPN; NFIA; NFKBIA; NICN1; NMT1; NOSIP; NREP; NUCB2; ORMDL3; PAK1; PARN; PDE10A; PEX26; PHB; PHF23; PKM; PLEKHG2; PPFIBP2; PPP1R12C; PRKCH; PRKCI; PRSS27; PTDSS2; RAB40C; RARA; RBL1; RNF215; RNU6-92P; RPGRIP1L; RPS26; RSBN1; RTEL1; RTEL1-TNFRSF6B; SCAF1; SCAMP4; SCRIB; SEC31A; SEPTIN9; SH3GL3; SIPA1L3; SLC25A12; SLC44A1; SMAD3; SMOC1; SMPD3; SMURF2; SOX6; SPIDR; SPPL3; ST3GAL2; STIM1; STMN3; TBC1D10A; TFAP4; TIMM21; TMEM115; TMEM70; TMEM94; TNFAIP1; TPRG1; TRAF3IP2-AS1; TRAPPC8; TSPAN5; TSPAN5-DT; TSSC4; TTC13; TTLL4; TUBD1; UBE2I; URB1; URB1-AS1; USP48; VCPIP1; VGLL4; VPS13B; VRK1; WDPCP; WDR87; WDTC1; XRCC5; XYLT1; ZCCHC2; ZCCHC8; ZNF341-AS1; ZNF37A; ZNF37BP; ZNF586; ZW10"},"children":["ACAP1; ACTN1; ACTR1A; ADAMTS17; ADAT3; ADPRHL1; AHCTF1; AHCYL2; ALG14; ANKRD26; AP2B1; AP2S1; APO..."]},{"name":"span","attribs":{"title":"ABCC13; ABR; ADGRV1; AIFM1; AKAP10; ANKRD2; APBB2; ARAP1; ARHGEF38; ASB13; ATXN7L1; BACE1; CAV2; CBX1; CCDC117; CCN5; CDHR3; CDK5RAP1; CLPB; CNTD1; COA3; COPS7B; CORO2A; CSNK1G3; CTPS2; CYP1B1; CYP1B1-AS1; CYREN; DEF6; DNAH11; DNPEP; DPP9; EBF4; EFNA5; ELOVL5; EML5; ERCC1; ESR2; EXOSC3; EYA3; FAM178B; FHIP1A; FNTB; FZD1; FZD4-DT; GALNT9; GINS3; GNA13; GPD1L; GSE1; HDGF; HIF1A; HOOK1; IL1RAP; IL4R; ILVBL; IPPK; ITGAL; ITGB2; KCNAB1; KIAA0100; KIAA0825; KLF4; KLHDC2; KLHL13; LAD1; LFNG; LINC01716; LINC02324; LMTK3; MARS1; MBNL1; MECOM; MED15; MICAL3; MIR9-1HG; MLYCD; MPO; MRPL45; MRPL45P2; MXI1; NDUFB9; NICN1; NOSIP; NRDE2; NT5DC3; PAPOLA; PCBP1-AS1; PDE11A; PDE6D; PEX14; PHF21A; PLEKHG2; PNPLA6; POU2F3; PPP1R37; PRKCI; PRSS8; PTPRF; RAB3D; REEP1; RGS3; RGS9; RNF220; RNF24; RPL39L; SEC1P; SERHL; SH3GL3; SLC14A2; SLC9A7; SMG7; SNAP25-AS1; SPECC1L; SPECC1L-ADORA2A; SPOPL; SPRY4; SRCIN1; SRI; STK3; SUN1; TATDN1; TBC1D31; TH; TJP3; TMEM205; TNIK; TPRN; TRAPPC8; TSC22D2; TUBA1C; UBE2I; UIMC1; UNKL; UPP2; USP48; VASP; VAT1L; VEPH1; VGLL4; VPS26C; WWP2; ZBED5-AS1; ZNF146; ZNF181; ZNF213; ZNF341; ZNF521; ZNF749"},"children":["ABCC13; ABR; ADGRV1; AIFM1; AKAP10; ANKRD2; APBB2; ARAP1; ARHGEF38; ASB13; ATXN7L1; BACE1; CAV2; ..."]},{"name":"span","attribs":{"title":"ABCC4; ABHD2; ACSL3; ACTN1; ANKH; ARID5B; B4GALT1; BMPR1B; CDK6; ELL2; ELOVL5; GPD1L; GSR; HERC3; HERC3; HOMER2; HPGD; INPP4B; KRT19; KRT8; MERTK; PDLIM5; PMEPA1; PTK2B; PTPN21; RRP12; SEC24D; SELENOP; SMS; SPDEF; STEAP4; STK39; TPD52; UBE2I; VAPA; XRCC5; ZMIZ1"},"children":["ABCC4; ABHD2; ACSL3; ACTN1; ANKH; ARID5B; B4GALT1; BMPR1B; CDK6; ELL2; ELOVL5; GPD1L; GSR; HERC3;..."]},{"name":"span","attribs":{"title":"ABR; ACP2; AGPS; AHSA1; ARV1; ASCC2; ATAD2; ATF2; BANP; BRD8; CADPS2; CAPN10; CCDC146; CFAP99; CNBD2; CYB5D1; DMPK; EHD1; ENSG00000186019; ENSG00000213963; EPB41L5; ESPN; ESRP2; FAM174B; FAM185BP; FBP1; FNTB; FOXA1; FOXP1; G3BP1; GANC; GRHL2; GRHL3; GSE1; GSK3A; H3-3B; HMBOX1; HNRNPL; IFRD2; IK; INTS9; IQCD; ISG20; JADE1; KIAA1671; KIF20A; LAMA3; LAMA5; LFNG; LLGL2; LNX1; LRIG2; LSR; MAL2; MBP; ME3; METTL15; MGRN1; MIR3621; MLLT3; NAA38; NDC1; NDUFA2; NEDD9; NFE2L2; NFKBIA; NOSIP; NPEPPS; NPEPPSP1; OAS1; ODC1-DT; PCDH12; PIM1; PRKCH; PRR15L; R3HDM1; RPL18A; SEC11C; SEMA4B; SEPTIN9; SHC2; SIN3A; SMAD3; SNX8; STARD10; STRBP; SULT2B1; SYT8; TMEM184A; TMEM87A; TOB1; TPCN1; TRAF4; TTC13; TTC6; TUFT1; USP48; VIPAS39; VPS26C; WRAP53; ZFYVE28; ZNF133; ZNF223; ZNF225; ZNF354B; ZNF749; ZRANB3"},"children":["ABR; ACP2; AGPS; AHSA1; ARV1; ASCC2; ATAD2; ATF2; BANP; BRD8; CADPS2; CAPN10; CCDC146; CFAP99; CN..."]},{"name":"span","attribs":{"title":"ADGRG1; ALDOA; ALG1L13P; AP2S1; ARHGAP25; ARID4B; ARMC2; ARNTL; ASB3; AZI2; C17orf100; C19orf44; CALR3; CC2D2A; CCDC191; CCDC40; CCDC91; CDH23; CFAP298-TCP10L; CFAP45; CFAP99; CFDP1; CHST11; CIC; CLMP; CLPB; CRIP2; CRTC2; CSF1; CTU1; DDX23; DHX8; DNAL4; DUS3L; DUSP16; DYNC1I2; DYRK1B; EHD1; ELAPOR1; EMC3; ENSG00000231170; ENSG00000236866; ENY2; ERCC1; FAM13A; FEM1A; GALK2; GEMIN6; GGPS1; GRK4; GSE1; H4C8; HBP1; HIPK1; HMBOX1; HSP90B1; IFRD2; IFT122; IFT20; IK; ING3; INTS9; JUND; KAZN; KCND2; KLF13; KLF4; KRT8; LAMTOR2; LCORL; LINC01275; LINC01876; LINC02246; LINC02532; LINC02863; LINS1; LMNA; MARCHF8; MBTPS1; MDH1; MIR3188; MIR4259; MIR4290HG; MRPL24; MYL3; NCOR2; NDUFA2; NFATC2; NFIA; NOD2; NOP14; NUDCD1; NUDT13; PANK4; PES1; PIM1; PPP1R12C; PSTPIP2; QTRT2; RABGAP1L; RARA; RERE; RFX1; RFX3-AS1; RPRD1B; RPS17P7; RRP12; RSRC1; SELENOP; SF3B5; SIK3; SLC44A1; SNX29; SPINK4; ST7; STXBP5-AS1; SUN1; SYNE2; TBC1D16; TCEA2; TCEANC; TENT5C; TFAP4; THADA; TIPARP; TMEM115; TNFAIP1; TOR2A; TTC41P; TUFT1; UBQLN4; VRK3; WBP2; WDPCP; YTHDC1; ZDHHC14; ZFP36L1; ZFYVE28; ZNF37BP; ZNF473; ZSWIM8"},"children":["ADGRG1; ALDOA; ALG1L13P; AP2S1; ARHGAP25; ARID4B; ARMC2; ARNTL; ASB3; AZI2; C17orf100; C19orf44; ..."]},{"name":"span","attribs":{"title":"ABCC5; ABCC8; ABCD3; ACOX1; ACSL1; CADM1; CLN8; CRAT; CTBP1; DHRS3; DLG4; EHHADH; ELOVL5; ERCC1; ESR2; FIS1; GSTK1; HAO2; HSD17B4; IDH2; ISOC1; LONP2; MLYCD; NUDT19; PABPC1; PEX14; PRDX1; SEMA3C; SERPINA6; SIAH1; SLC25A17; SLC25A19; STS; SULT2B1"},"children":["ABCC5; ABCC8; ABCD3; ACOX1; ACSL1; CADM1; CLN8; CRAT; CTBP1; DHRS3; DLG4; EHHADH; ELOVL5; ERCC1; ..."]},{"name":"span","attribs":{"title":"ACOX1; ACTG1; ADD2; ADD3; AHCYL2; ALDOA; ASAP3; ASXL1; ATAD2; ATP5MF; ATP5MF-PTCD1; BCAT2; BIRC5; BORCS5; BRWD1; C1orf159; C6orf141; CAP1; CASZ1; CCDC88A; CCNT2; CCNT2-AS1; CD9; CDC42; CDC42EP4; CECR2; CELSR1; CENPJ; CENPO; CERT1; CFAP298-TCP10L; CFL1; CIC; CMSS1; CNOT4; COPS7B; COX16; CPEB4; DISC1; DLEU1; DPP9; DST; DYNC2I2; DYRK1B; EIF2S3; ENSG00000204055; ENSG00000267575; ERP44; ETV4; EXOSC3; FAM168A; FGD6; GABARAP; GANC; GINS3; GLI3; GPRC5A; GSE1; GSPT1; GTF2H4; HBS1L; HNRNPL; HSPB6; HYAL3; IL12A; INVS; JPX; KANSL3; KAT6B; KATNAL1; KLC1; KRT8; LARGE1; LINC00662; LINC01006; LINC01144; LINC01275; LINC01547; LINC02026; LRP6; LRRC1; LRRFIP2; LSR; LTA4H; LYPD5; MACIR; MAN2C1; MARCHF8; MARS1; MBD2; MCF2L; METTL15; MIB1; MICAL3; MIPEP; MIR4530; MYL6; MYL6B-AS1; NAT9; NCAM1; NDUFAF6; NEIL1; NMNAT1; NR2F6; OTUD7B; PAFAH2; PDCD6; PDCD6IP; PDE6D; PEMT; PHF12; PHF20; PIDD1; PIM1; PIPOX; PMP22; PNPLA7; POLK; PPCDC; PPP2CB; PPT1; PRDX1; PRELID2; PRKAA1; PROSER3; PTCD1; PTP4A2; PTPN21; PTRHD1; PUF60; RABAC1; RAD52; RBPJ; RMI2; RPL37A; RPL38; RPLP2; RPS26; SAE1; SCAF1; SCAP; SEC31A; SGMS1; SLC25A6; SMG1P3; SMG7; SMIM14; SNAP29; SND1; SPRED2; SPRY4; SRCIN1; STAT3; STMN3; STX18-AS1; SVIL; SYNPO2; TJP3; TMEM87A; TOB1; TOB1-AS1; TPK1; TRABD; TRAF4; TTC1; TTC33; TUBA1A; TUBA1C; UBE2H; UBTD2; UCKL1; USP31; USP32; VGLL4; ZBTB40; ZCCHC2; ZFP36; ZFYVE1; ZKSCAN2; ZMIZ1-AS1; ZMYM1; ZMYM6; ZNF133; ZNF181; ZNF213-AS1; ZNF354A; ZNF609"},"children":["ACOX1; ACTG1; ADD2; ADD3; AHCYL2; ALDOA; ASAP3; ASXL1; ATAD2; ATP5MF; ATP5MF-PTCD1; BCAT2; BIRC5;..."]}],"minWidth":150}],"filterable":true,"showPageSizeOptions":true,"dataKey":"a3d80f1303696987c6d12b53b5a7e532"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
<div id="network-plot" class="section level3">
<h3>Network Plot</h3>
<pre class="r"><code>tg_mapped %&gt;% 
  ggraph(layout = net_layout, weights = oc^2) +
  geom_edge_link(aes(width = oc^2, alpha = oc^2)) +
  geom_node_point(
    aes(fill = -log10(pval), size = numDEInCat),
    shape = 21
  ) +
  geom_node_text(
    aes(label = label),
    colour = &quot;black&quot;, size = 3, 
    data = . %&gt;%
      mutate(
        label = str_replace_all(label, &quot;_&quot;, &quot; &quot;) %&gt;% str_trunc(60) %&gt;% str_wrap(width = 18)
      ),
    repel = TRUE, max.overlaps = max(10, round(length(tg_mapped) / 4, 0)),
    bg.color = &quot;white&quot;, bg.r = 0.1, 
  ) +
  scale_x_continuous(expand = expansion(c(0.1, 0.1))) +
  scale_y_continuous(expand = expansion(c(0.1, 0.1))) +
  scale_fill_viridis_c(option = &quot;inferno&quot;, begin = 0.25) +
  scale_size_continuous(range = c(1, 10)) +
  scale_edge_width(range = c(1, 6), limits = c(0, 1)) +
  scale_edge_alpha(range = c(0.1, 0.4), limits = c(0, 1))  +
  guides(edge_alpha= &quot;none&quot;) +
  labs(size = &quot;Mapped Genes&quot;, edge_width = expr(paste(OC^2))) +
  theme_void() </code></pre>
<div class="figure" style="text-align: center">
<img src="ER_E2_E2DHT_differential_binding_files/figure-html/plot-network-mapped-1.png" alt="*Network plot showing  gene-sets enriched amongst the overall set of sites with a binding site for ER.*" width="960" />
<p class="caption">
<em>Network plot showing gene-sets enriched amongst the overall set of
sites with a binding site for ER.</em>
</p>
</div>
</div>
</div>
<div id="windows-with-differentially-bound-er"
class="section level2 tabset">
<h2 class="tabset">Windows With Differentially Bound ER</h2>
<pre class="r"><code>pwf_diff &lt;- tibble(gene_id  = unique(genes_gr$gene_id)) %&gt;% 
  left_join(mapped_ids, by = &quot;gene_id&quot;) %&gt;% 
  dplyr::filter(!is.na(n_peaks)) %&gt;% 
  mutate(diff = up|down) %&gt;% 
  with(
    nullp(
      structure(diff, names = gene_id),
      genome = config$genome$build,
      bias.data = log1p(n_peaks),
      plot.fit = FALSE
    )
  )
goseq_diff &lt;- tibble(gs_name = character(), pval = numeric(), adj_p = numeric())
if (sum(pwf_diff$DEgenes) &gt; 0) {
  goseq_diff &lt;- goseq(
    pwf_diff, gene2cat = gs_by_geneid, method = &quot;Hypergeometric&quot;
  ) %&gt;% 
    dplyr::rename(gs_name = category, pval = over_represented_pvalue) %&gt;% 
    dplyr::select(-starts_with(&quot;under&quot;)) %&gt;% 
    as_tibble() %&gt;% 
    arrange(pval) %&gt;% 
    dplyr::filter(numDEInCat &gt; 0) %&gt;% 
    mutate(adj_p = p.adjust(pval, adj_method)) %&gt;% 
    dplyr::filter(numDEInCat &gt;= min_sig)
}
any_goseq_diff &lt;- sum(goseq_diff$adj_p &lt; enrich_alpha) &gt; 0
tg_diff &lt;- make_tbl_graph(
  goseq_diff, 
  gs = gs_by_gsid %&gt;% 
    lapply(intersect, rownames(subset(pwf_diff, DEgenes))) %&gt;%
    .[vapply(., length, integer(1)) &gt; 0]
)
plot_network_diff &lt;- length(tg_diff) &gt;= min_network_size</code></pre>
<p>No enrichment was found amongst genes mapped to sites showing any
differential binding</p>
</div>
<div id="windows-with-increased-er-binding"
class="section level2 tabset">
<h2 class="tabset">Windows With Increased ER Binding</h2>
<pre class="r"><code>pwf_up &lt;- tibble(gene_id  = unique(genes_gr$gene_id)) %&gt;% 
  left_join(mapped_ids, by = &quot;gene_id&quot;) %&gt;% 
  dplyr::filter(!is.na(n_peaks)) %&gt;% 
  mutate(diff = up) %&gt;% 
  with(
    nullp(
      structure(diff, names = gene_id),
      genome = config$genome$build,
      bias.data = log1p(n_peaks),
      plot.fit = FALSE
    )
  )
goseq_up &lt;- tibble(gs_name = character(), pval = numeric(), adj_p = numeric())
if (sum(pwf_up$DEgenes) &gt; 0) {
  goseq_up &lt;- goseq(
    pwf_up, gene2cat = gs_by_geneid, method = &quot;Hypergeometric&quot;
  ) %&gt;% 
    dplyr::rename(gs_name = category, pval = over_represented_pvalue) %&gt;% 
    dplyr::select(-starts_with(&quot;under&quot;)) %&gt;% 
    as_tibble() %&gt;% 
    arrange(pval) %&gt;% 
    dplyr::filter(numDEInCat &gt; 0) %&gt;% 
    mutate(adj_p = p.adjust(pval, adj_method)) %&gt;% 
    dplyr::filter(numDEInCat &gt;= min_sig)
}
any_goseq_up &lt;- sum(goseq_up$adj_p &lt; enrich_alpha) &gt; 0
tg_up &lt;- make_tbl_graph(
  goseq_up, 
  gs = gs_by_gsid %&gt;% 
    lapply(intersect, rownames(subset(pwf_up, DEgenes))) %&gt;%
    .[vapply(., length, integer(1)) &gt; 0]
)
plot_network_up &lt;- length(tg_up) &gt;= min_network_size</code></pre>
<p>No enrichment was found amongst genes mapped to sites showing
increased ER binding</p>
</div>
<div id="windows-with-decreased-er-binding"
class="section level2 tabset">
<h2 class="tabset">Windows With Decreased ER Binding</h2>
<pre class="r"><code>pwf_down &lt;- tibble(gene_id  = unique(genes_gr$gene_id)) %&gt;% 
  left_join(mapped_ids, by = &quot;gene_id&quot;) %&gt;% 
  dplyr::filter(!is.na(n_peaks)) %&gt;% 
  mutate(diff = down) %&gt;% 
  with(
    nullp(
      structure(diff, names = gene_id),
      genome = config$genome$build,
      bias.data = log1p(n_peaks),
      plot.fit = FALSE
    )
  )
goseq_down &lt;- tibble(gs_name = character(), pval = numeric(), adj_p = numeric())
if (sum(pwf_down$DEgenes) &gt; 0) {
  goseq_down &lt;- goseq(
    pwf_down, config$genome$build, gene2cat = gs_by_geneid, method = &quot;Hypergeometric&quot;
  ) %&gt;% 
    dplyr::rename(gs_name = category, pval = over_represented_pvalue) %&gt;% 
    dplyr::select(-starts_with(&quot;under&quot;)) %&gt;% 
    as_tibble() %&gt;% 
    arrange(pval) %&gt;% 
    dplyr::filter(numDEInCat &gt; 0) %&gt;% 
    mutate(adj_p = p.adjust(pval, adj_method)) %&gt;% 
    dplyr::filter(numDEInCat &gt;= min_sig)
}
any_goseq_down &lt;- sum(goseq_down$adj_p &lt; enrich_alpha) &gt; 0
tg_down &lt;- make_tbl_graph(
  goseq_down, 
  gs = gs_by_gsid %&gt;% 
    lapply(intersect, rownames(subset(pwf_down, DEgenes))) %&gt;%
    .[vapply(., length, integer(1)) &gt; 0]
)
plot_network_down &lt;- length(tg_down) &gt;= min_network_size</code></pre>
<p>No enrichment was found amongst genes mapped to sites showing
decreased ER binding</p>
</div>
</div>
<div id="data-export" class="section level1">
<h1>Data Export</h1>
<pre class="r"><code>merged_results %&gt;% 
  as_tibble() %&gt;%
  unnest(all_of(&quot;gene_id&quot;)) %&gt;%
  mutate(gene_name = id2gene[gene_id]) %&gt;% 
  dplyr::select(
    gene_id, gene_name, range, AveExpr, logFC, P.Value, FDR = !!sym(fdr_column), 
    status, any_of(c(&quot;region&quot;, &quot;feature&quot;)), macs2_peak = overlaps_ref,
  ) %&gt;% 
  mutate(
    distance_to_gene = distance(
      GRanges(range),
      setNames(genes_gr, genes_gr$gene_id)[gene_id]
    )
  ) %&gt;% 
  write_csv(
    gzfile(all_out$csv)
  )</code></pre>
<pre class="r"><code>write_rds(merged_results, all_out$results, compress = &quot;gz&quot;)
write_rds(filtered_counts, all_out$windows, compress = &quot;gz&quot;)
file.create(all_out$de_genes)
if (has_rnaseq) write_csv(de_genes_db_regions, all_out$de_genes)

file.create(all_out$up_regions)
if (sum(merged_results$status == &quot;Up&quot;) &gt; 0) {
  merged_results %&gt;% 
    filter(status == &quot;Up&quot;) %&gt;% 
    granges() %&gt;% 
    export.bed(all_out$up_regions)
}

file.create(all_out$down_regions)
if (sum(merged_results$status == &quot;Down&quot;) &gt; 0) {
  merged_results %&gt;% 
    filter(status == &quot;Down&quot;) %&gt;% 
    granges() %&gt;% 
    export.bed(all_out$up_regions)
}
## Enrichment results need a bit of tweaking to match the tables
list2(
  `All Differentially Bound` = goseq_diff,
  &quot;All Increased {target}&quot; := goseq_up,
  &quot;All Decreased {target}&quot; := goseq_down
) %&gt;% 
  bind_rows(.id = &#39;group&#39;) %&gt;% 
  dplyr::filter(adj_p &lt; enrich_alpha) %&gt;% 
  left_join(
    dplyr::select(msigdb, gs_name, gene_name, gene_id, gs_url, gs_description),
    by = &quot;gs_name&quot;
  ) %&gt;% 
  dplyr::filter(
    gene_id %in% unlist(subset(merged_results, status != &quot;Unchanged&quot;)$gene_id)
  ) %&gt;% 
  dplyr::select(-gene_id) %&gt;% 
  chop(gene_name) %&gt;% 
  mutate(
    gene_name = vapply(gene_name, paste, character(1), collapse = &quot;: &quot;),
    `%` = numDEInCat / numInCat,
  ) %&gt;% 
  dplyr::select(
    group,
    `Gene Set` = gs_name, Description = gs_description, URL = gs_url,
    `%`, numDEInCat, numInCat, p = pval, adj_p, 
    `Genes Mapped to DB Windows` = gene_name
  ) %&gt;% 
  write_csv(all_out$enrichment)
list2(
  `All Differentially Bound` = cmn_diff,
  &quot;All Increased {target}&quot; := cmn_up,
  &quot;All Decreased {target}&quot; := cmn_down
) %&gt;% 
  bind_rows(.id = &quot;group&quot;) %&gt;% 
  mutate(
    genes = vapply(leadingEdge, paste, character(1), collapse = &quot;; &quot;),
    leadingEdge = vapply(leadingEdge, paste, character(1), collapse = &quot;; &quot;),
  ) %&gt;% 
  write_csv(all_out$rna_enrichment)
## Save key memory on the HDD
rm(window_counts)
gc()
save.image(all_out$renv)</code></pre>
<p>During this workflow, the following files were exported:</p>
<ul>
<li><strong>results</strong>:
output/differential_binding/ER/ER_E2_E2DHT-differential_binding.rds</li>
<li><strong>csv</strong>:
output/differential_binding/ER/ER_E2_E2DHT-differential_binding.csv.gz</li>
<li><strong>windows</strong>:
output/differential_binding/ER/ER_E2_E2DHT-filtered_windows.rds</li>
<li><strong>up_regions</strong>:
output/differential_binding/ER/ER_E2_E2DHT-up.bed</li>
<li><strong>down_regions</strong>:
output/differential_binding/ER/ER_E2_E2DHT-down.bed</li>
<li><strong>de_genes</strong>:
output/differential_binding/ER/ER_E2_E2DHT-DE_genes.csv</li>
<li><strong>enrichment</strong>:
output/differential_binding/ER/ER_E2_E2DHT-enrichment.csv</li>
<li><strong>rna_enrichment</strong>:
output/differential_binding/ER/ER_E2_E2DHT-rnaseq_enrichment.csv</li>
<li><strong>renv</strong>:
output/envs/ER_E2_E2DHT-differential_binding.RData</li>
</ul>
<!-- end of list -->
<div id="references" class="section level2">
<h2>References</h2>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-HicksQuantro2015" class="csl-entry">
Hicks, S. C., and R. A. Irizarry. 2015. <span>“<span
class="nocase">quantro: a data-driven approach to guide the choice of an
appropriate normalization method</span>.”</span> <em>Genome Biol</em> 16
(June): 117.
</div>
<div id="ref-HicksSQN2017" class="csl-entry">
Hicks, Stephanie C, Kwame Okrah, Joseph N Paulson, John Quackenbush,
Rafael A Irizarry, and Héctor Corrada Bravo. 2017. <span>“<span
class="nocase">Smooth quantile normalization</span>.”</span>
<em>Biostatistics</em> 19 (2): 185–98. <a
href="https://doi.org/10.1093/biostatistics/kxx028">https://doi.org/10.1093/biostatistics/kxx028</a>.
</div>
<div id="ref-IgnatiadisIHW2016" class="csl-entry">
Ignatiadis, N., B. Klaus, J. B. Zaugg, and W. Huber. 2016. <span>“<span
class="nocase"><span>D</span>ata-driven hypothesis weighting increases
detection power in genome-scale multiple testing</span>.”</span> <em>Nat
Methods</em> 13 (7): 577–80.
</div>
<div id="ref-LawVoom2014" class="csl-entry">
Law, C. W., Y. Chen, W. Shi, and G. K. Smyth. 2014. <span>“<span
class="nocase">voom: <span>P</span>recision weights unlock linear model
analysis tools for <span>R</span><span>N</span><span>A</span>-seq read
counts</span>.”</span> <em>Genome Biol</em> 15 (2): R29.
</div>
<div id="ref-LunSmythF1000" class="csl-entry">
Lun, A. T., and G. K. Smyth. 2015. <span>“<span
class="nocase"><span>F</span>rom reads to regions: a
<span>B</span>ioconductor workflow to detect differential binding in
<span>C</span>h<span>I</span><span>P</span>-seq data</span>.”</span>
<em>F1000Res</em> 4: 1080.
</div>
<div id="ref-LunSmythCsaw2014" class="csl-entry">
Lun, Aaron T L, and Gordon K Smyth. 2014. <span>“<span>D</span>e Novo
Detection of Differentially Bound Regions for
<span>C</span>h<span>I</span><span>P</span>-Seq Data Using Peaks and
Windows: Controlling Error Rates Correctly.”</span> <em>Nucleic Acids
Res.</em> 42 (11): e95.
</div>
<div id="ref-McCarthyTreat2009" class="csl-entry">
McCarthy, Davis J., and Gordon K. Smyth. 2009. <span>“<span
class="nocase">Testing significance relative to a fold-change threshold
is a TREAT</span>.”</span> <em>Bioinformatics</em> 25 (6): 765–71. <a
href="https://doi.org/10.1093/bioinformatics/btp053">https://doi.org/10.1093/bioinformatics/btp053</a>.
</div>
<div id="ref-YoungGoseq2010" class="csl-entry">
Young, M. D., M. J. Wakefield, G. K. Smyth, and A. Oshlack. 2010.
<span>“<span class="nocase"><span>G</span>ene ontology analysis for
<span>R</span><span>N</span><span>A</span>-seq: accounting for selection
bias</span>.”</span> <em>Genome Biol</em> 11 (2): R14.
</div>
<div id="ref-Zhang18798982" class="csl-entry">
Zhang, Y., T. Liu, C. A. Meyer, J. Eeckhoute, D. S. Johnson, B. E.
Bernstein, C. Nusbaum, et al. 2008. <span>“<span
class="nocase"><span>M</span>odel-based analysis of
<span>C</span>h<span>I</span><span>P</span>-<span>S</span>eq
(<span>M</span><span>A</span><span>C</span><span>S</span>)</span>.”</span>
<em>Genome Biol</em> 9 (9): R137.
</div>
</div>
<br>
<button type="button" class="btn btn-default btn-sessioninfo" data-toggle="collapse" data-target="#sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span>
Session information </button>
</p>
<div id="sessioninfo" class="collapse">
<p><strong>R version 4.2.2 (2022-10-31)</strong></p>
<p><strong>Platform:</strong> x86_64-conda-linux-gnu (64-bit)</p>
<p><strong>locale:</strong> <em>LC_CTYPE=en_AU.UTF-8</em>,
<em>LC_NUMERIC=C</em>, <em>LC_TIME=en_AU.UTF-8</em>,
<em>LC_COLLATE=en_AU.UTF-8</em>, <em>LC_MONETARY=en_AU.UTF-8</em>,
<em>LC_MESSAGES=en_AU.UTF-8</em>, <em>LC_PAPER=en_AU.UTF-8</em>,
<em>LC_NAME=C</em>, <em>LC_ADDRESS=C</em>, <em>LC_TELEPHONE=C</em>,
<em>LC_MEASUREMENT=en_AU.UTF-8</em> and <em>LC_IDENTIFICATION=C</em></p>
<p><strong>attached base packages:</strong> <em>stats4</em>,
<em>stats</em>, <em>graphics</em>, <em>grDevices</em>, <em>utils</em>,
<em>datasets</em>, <em>methods</em> and <em>base</em></p>
<p><strong>other attached packages:</strong> <em>qsmooth(v.1.14.0)</em>,
<em>quantro(v.1.32.0)</em>, <em>metap(v.1.1)</em>,
<em>ggraph(v.2.1.0)</em>, <em>tidygraph(v.1.2.3)</em>,
<em>extraChIPs(v.1.2.4)</em>, <em>goseq(v.1.50.0)</em>,
<em>geneLenDataBase(v.1.34.0)</em>, <em>BiasedUrn(v.2.0.10)</em>,
<em>msigdbr(v.7.5.1)</em>, <em>htmltools(v.0.5.5)</em>,
<em>reactable(v.0.4.4)</em>, <em>GenomicInteractions(v.1.32.0)</em>,
<em>InteractionSet(v.1.26.0)</em>, <em>ggside(v.0.2.2)</em>,
<em>rlang(v.1.1.1)</em>, <em>ggrepel(v.0.9.3)</em>,
<em>IHW(v.1.26.0)</em>, <em>statmod(v.1.5.0)</em>,
<em>magrittr(v.2.0.3)</em>, <em>scales(v.1.2.1)</em>,
<em>edgeR(v.3.40.0)</em>, <em>limma(v.3.54.0)</em>,
<em>csaw(v.1.32.0)</em>, <em>SummarizedExperiment(v.1.28.0)</em>,
<em>Biobase(v.2.58.0)</em>, <em>MatrixGenerics(v.1.10.0)</em>,
<em>matrixStats(v.1.0.0)</em>, <em>ngsReports(v.2.0.0)</em>,
<em>patchwork(v.1.1.2)</em>, <em>cowplot(v.1.1.1)</em>,
<em>Rsamtools(v.2.14.0)</em>, <em>Biostrings(v.2.66.0)</em>,
<em>XVector(v.0.38.0)</em>, <em>rtracklayer(v.1.58.0)</em>,
<em>plyranges(v.1.18.0)</em>, <em>GenomicRanges(v.1.50.0)</em>,
<em>GenomeInfoDb(v.1.34.9)</em>, <em>IRanges(v.2.32.0)</em>,
<em>S4Vectors(v.0.36.0)</em>, <em>BiocGenerics(v.0.44.0)</em>,
<em>yaml(v.2.3.7)</em>, <em>BiocParallel(v.1.32.5)</em>,
<em>pander(v.0.6.5)</em>, <em>glue(v.1.6.2)</em>,
<em>lubridate(v.1.9.2)</em>, <em>forcats(v.1.0.0)</em>,
<em>stringr(v.1.5.0)</em>, <em>dplyr(v.1.1.2)</em>,
<em>purrr(v.1.0.1)</em>, <em>readr(v.2.1.4)</em>,
<em>tidyr(v.1.3.0)</em>, <em>tibble(v.3.2.1)</em>,
<em>ggplot2(v.3.4.2)</em> and <em>tidyverse(v.2.0.0)</em></p>
<p><strong>loaded via a namespace (and not attached):</strong>
<em>rappdirs(v.0.3.3)</em>, <em>bumphunter(v.1.40.0)</em>,
<em>minfi(v.1.44.0)</em>, <em>bit64(v.4.0.5)</em>,
<em>knitr(v.1.43)</em>, <em>DelayedArray(v.0.24.0)</em>,
<em>data.table(v.1.14.8)</em>, <em>rpart(v.4.1.19)</em>,
<em>GEOquery(v.2.66.0)</em>, <em>KEGGREST(v.1.38.0)</em>,
<em>RCurl(v.1.98-1.12)</em>, <em>AnnotationFilter(v.1.22.0)</em>,
<em>doParallel(v.1.0.17)</em>, <em>generics(v.0.1.3)</em>,
<em>preprocessCore(v.1.60.2)</em>, <em>GenomicFeatures(v.1.50.2)</em>,
<em>EnrichedHeatmap(v.1.27.2)</em>, <em>lambda.r(v.1.2.4)</em>,
<em>RSQLite(v.2.3.1)</em>, <em>bit(v.4.0.5)</em>,
<em>tzdb(v.0.4.0)</em>, <em>xml2(v.1.3.3)</em>,
<em>viridis(v.0.6.3)</em>, <em>xfun(v.0.39)</em>, <em>hms(v.1.1.3)</em>,
<em>jquerylib(v.0.1.4)</em>, <em>babelgene(v.22.9)</em>,
<em>evaluate(v.0.21)</em>, <em>scrime(v.1.3.5)</em>,
<em>fansi(v.1.0.4)</em>, <em>restfulr(v.0.0.15)</em>,
<em>progress(v.1.2.2)</em>, <em>dbplyr(v.2.3.2)</em>,
<em>igraph(v.1.4.2)</em>, <em>DBI(v.1.1.3)</em>,
<em>htmlwidgets(v.1.6.2)</em>, <em>futile.logger(v.1.4.3)</em>,
<em>reshape(v.0.8.9)</em>, <em>ellipsis(v.0.3.2)</em>,
<em>crosstalk(v.1.2.0)</em>, <em>backports(v.1.4.1)</em>,
<em>annotate(v.1.76.0)</em>, <em>biomaRt(v.2.54.0)</em>,
<em>deldir(v.1.0-9)</em>, <em>sparseMatrixStats(v.1.10.0)</em>,
<em>vctrs(v.0.6.3)</em>, <em>here(v.1.0.1)</em>,
<em>ensembldb(v.2.22.0)</em>, <em>cachem(v.1.0.8)</em>,
<em>withr(v.2.5.0)</em>, <em>ggforce(v.0.4.1)</em>,
<em>Gviz(v.1.42.0)</em>, <em>BSgenome(v.1.66.3)</em>,
<em>checkmate(v.2.2.0)</em>, <em>vroom(v.1.6.3)</em>,
<em>GenomicAlignments(v.1.34.0)</em>, <em>fdrtool(v.1.2.17)</em>,
<em>prettyunits(v.1.1.1)</em>, <em>mclust(v.6.0.0)</em>,
<em>cluster(v.2.1.4)</em>, <em>lazyeval(v.0.2.2)</em>,
<em>crayon(v.1.5.2)</em>, <em>genefilter(v.1.80.0)</em>,
<em>labeling(v.0.4.2)</em>, <em>pkgconfig(v.2.0.3)</em>,
<em>slam(v.0.1-50)</em>, <em>tweenr(v.2.0.2)</em>,
<em>nlme(v.3.1-162)</em>, <em>ProtGenerics(v.1.30.0)</em>,
<em>nnet(v.7.3-19)</em>, <em>lifecycle(v.1.0.3)</em>,
<em>filelock(v.1.0.2)</em>, <em>BiocFileCache(v.2.6.0)</em>,
<em>dichromat(v.2.0-0.1)</em>, <em>VennDiagram(v.1.7.3)</em>,
<em>rprojroot(v.2.0.3)</em>, <em>polyclip(v.1.10-4)</em>,
<em>reactR(v.0.4.4)</em>, <em>rngtools(v.1.5.2)</em>,
<em>base64(v.2.0.1)</em>, <em>Matrix(v.1.5-4.1)</em>,
<em>lpsymphony(v.1.26.0)</em>, <em>Rhdf5lib(v.1.20.0)</em>,
<em>zoo(v.1.8-12)</em>, <em>base64enc(v.0.1-3)</em>,
<em>GlobalOptions(v.0.1.2)</em>, <em>png(v.0.1-8)</em>,
<em>viridisLite(v.0.4.2)</em>, <em>rjson(v.0.2.21)</em>,
<em>bitops(v.1.0-7)</em>, <em>rhdf5filters(v.1.10.0)</em>,
<em>blob(v.1.2.4)</em>, <em>DelayedMatrixStats(v.1.20.0)</em>,
<em>doRNG(v.1.8.6)</em>, <em>shape(v.1.4.6)</em>,
<em>nor1mix(v.1.3-0)</em>, <em>jpeg(v.0.1-10)</em>,
<em>memoise(v.2.0.1)</em>, <em>plyr(v.1.8.8)</em>,
<em>zlibbioc(v.1.44.0)</em>, <em>compiler(v.4.2.2)</em>,
<em>BiocIO(v.1.8.0)</em>, <em>illuminaio(v.0.40.0)</em>,
<em>RColorBrewer(v.1.1-3)</em>, <em>clue(v.0.3-64)</em>,
<em>cli(v.3.6.1)</em>, <em>htmlTable(v.2.4.1)</em>,
<em>formatR(v.1.14)</em>, <em>Formula(v.1.2-5)</em>,
<em>MASS(v.7.3-60)</em>, <em>mgcv(v.1.8-42)</em>,
<em>tidyselect(v.1.2.0)</em>, <em>stringi(v.1.7.12)</em>,
<em>highr(v.0.10)</em>, <em>askpass(v.1.1)</em>,
<em>locfit(v.1.5-9.8)</em>, <em>latticeExtra(v.0.6-30)</em>,
<em>grid(v.4.2.2)</em>, <em>sass(v.0.4.6)</em>,
<em>VariantAnnotation(v.1.44.0)</em>, <em>tools(v.4.2.2)</em>,
<em>timechange(v.0.2.0)</em>, <em>parallel(v.4.2.2)</em>,
<em>circlize(v.0.4.15)</em>, <em>rstudioapi(v.0.14)</em>,
<em>foreach(v.1.5.2)</em>, <em>foreign(v.0.8-84)</em>,
<em>metapod(v.1.6.0)</em>, <em>gridExtra(v.2.3)</em>,
<em>farver(v.2.1.1)</em>, <em>ComplexUpset(v.1.3.3)</em>,
<em>digest(v.0.6.31)</em>, <em>quadprog(v.1.5-8)</em>,
<em>Rcpp(v.1.0.10)</em>, <em>siggenes(v.1.72.0)</em>,
<em>broom(v.1.0.5)</em>, <em>httr(v.1.4.6)</em>,
<em>ggdendro(v.0.1.23)</em>, <em>AnnotationDbi(v.1.60.0)</em>,
<em>biovizBase(v.1.46.0)</em>, <em>ComplexHeatmap(v.2.14.0)</em>,
<em>Rdpack(v.2.4)</em>, <em>colorspace(v.2.1-0)</em>,
<em>XML(v.3.99-0.14)</em>, <em>splines(v.4.2.2)</em>,
<em>graphlayouts(v.1.0.0)</em>, <em>multtest(v.2.54.0)</em>,
<em>plotly(v.4.10.2)</em>, <em>xtable(v.1.8-4)</em>,
<em>jsonlite(v.1.8.5)</em>, <em>futile.options(v.1.0.1)</em>,
<em>R6(v.2.5.1)</em>, <em>Hmisc(v.5.1-0)</em>, <em>pillar(v.1.9.0)</em>,
<em>fastmap(v.1.1.1)</em>, <em>DT(v.0.28)</em>,
<em>beanplot(v.1.3.1)</em>, <em>codetools(v.0.2-19)</em>,
<em>utf8(v.1.2.3)</em>, <em>sva(v.3.46.0)</em>,
<em>lattice(v.0.21-8)</em>, <em>bslib(v.0.5.0)</em>,
<em>curl(v.5.0.1)</em>, <em>openssl(v.2.0.6)</em>,
<em>GO.db(v.3.16.0)</em>, <em>interp(v.1.1-4)</em>,
<em>survival(v.3.5-5)</em>, <em>rmarkdown(v.2.22)</em>,
<em>munsell(v.0.5.0)</em>, <em>GetoptLong(v.1.0.5)</em>,
<em>rhdf5(v.2.42.0)</em>, <em>GenomeInfoDbData(v.1.2.9)</em>,
<em>iterators(v.1.0.14)</em>, <em>HDF5Array(v.1.26.0)</em>,
<em>gtable(v.0.3.3)</em> and <em>rbibutils(v.2.2.13)</em></p>
</div>
</div>
</div>

<div class="footer">
  Workflow developed by:<br>
  Dr Stephen Pederson<br>
  Dame Roma Mitchell Cancer Research Laboratories<br>
  Adelaide Medical School, University of Adelaide, SA, Australia
</div>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
