<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>H3K27ac Differential Binding: E2DHT Vs. E2</title>

<script src="site_libs/header-attrs-2.22/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/sandstone.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/core-js-2.5.3/shim.min.js"></script>
<script src="site_libs/react-17.0.0/react.min.js"></script>
<script src="site_libs/react-17.0.0/react-dom.min.js"></script>
<script src="site_libs/reactwidget-1.0.0/react-tools.js"></script>
<script src="site_libs/htmlwidgets-1.6.2/htmlwidgets.js"></script>
<link href="site_libs/reactable-0.4.4/reactable.css" rel="stylesheet" />
<script src="site_libs/reactable-binding-0.4.4/reactable.js"></script>
<link href="site_libs/font-awesome-6.4.0/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.0/css/v4-shims.min.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>





<style type="text/css">
/* for pandoc --citeproc since 2.11 */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="gravi.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">GRAVI</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="annotation_description.html">Annotations</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    MACS2 Peak Calling
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="AR_macs2_summary.html">AR</a>
    </li>
    <li>
      <a href="ER_macs2_summary.html">ER</a>
    </li>
    <li>
      <a href="H3K27ac_macs2_summary.html">H3K27ac</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Differential Binding
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">AR</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="AR_E2_E2DHT_differential_binding.html">E2DHT Vs. E2</a>
        </li>
      </ul>
    </li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">ER</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="ER_E2_E2DHT_differential_binding.html">E2DHT Vs. E2</a>
        </li>
      </ul>
    </li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">H3K27ac</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="H3K27ac_E2_E2DHT_differential_binding.html">E2DHT Vs. E2</a>
        </li>
      </ul>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Pairwise Comparisons
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">AR-ER</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="AR_E2_E2DHT_ER_E2_E2DHT_pairwise_comparison.html">E2DHT Vs. E2 / E2DHT Vs. E2</a>
        </li>
      </ul>
    </li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">AR-H3K27ac</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="AR_E2_E2DHT_H3K27ac_E2_E2DHT_pairwise_comparison.html">E2DHT Vs. E2 / E2DHT Vs. E2</a>
        </li>
      </ul>
    </li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">ER-H3K27ac</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="ER_E2_E2DHT_H3K27ac_E2_E2DHT_pairwise_comparison.html">E2DHT Vs. E2 / E2DHT Vs. E2</a>
        </li>
      </ul>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/steveped/GRAVI">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">H3K27ac Differential Binding: E2DHT Vs.
E2</h1>
<h4 class="date">01 July, 2023</h4>

</div>


<pre class="r"><code>target &lt;- &quot;H3K27ac&quot;
treat_levels &lt;- c(&quot;E2&quot;, &quot;E2DHT&quot;)
threads &lt;- 1</code></pre>
<pre class="r"><code>library(tidyverse)
library(glue)
library(pander)
library(BiocParallel)
library(yaml)
library(plyranges)
library(rtracklayer)
library(GenomeInfoDb)
library(Rsamtools)
library(cowplot)
library(ngsReports)
library(csaw)
library(edgeR)
library(scales)
library(magrittr)
library(statmod)
library(IHW)
library(ggrepel)
library(rlang)
library(ggside)
library(InteractionSet)
library(GenomicInteractions)
library(reactable)
library(htmltools)
library(msigdbr)
library(goseq)
library(extraChIPs)
library(tidygraph)
library(ggraph)
library(metap)</code></pre>
<pre class="r"><code>panderOptions(&quot;big.mark&quot;, &quot;,&quot;)
panderOptions(&quot;missing&quot;, &quot;&quot;)
panderOptions(&quot;table.split.table&quot;, Inf)
theme_set(
  theme_bw() +
  theme(
    plot.title = element_text(hjust = 0.5)
  )
)
register(MulticoreParam(workers = threads))
source(here::here(&quot;workflow&quot;, &quot;scripts&quot;, &quot;custom_functions.R&quot;))</code></pre>
<pre class="r"><code>params &lt;- read_yaml(here::here(&quot;config&quot;, &quot;params.yml&quot;))
config &lt;- read_yaml(here::here(&quot;config&quot;, &quot;config.yml&quot;))
macs2_path &lt;- here::here(&quot;output&quot;, &quot;macs2&quot;, target)
samples &lt;- file.path(macs2_path, glue(&quot;{target}_qc_samples.tsv&quot;)) %&gt;% 
  read_tsv() %&gt;% 
  dplyr::filter(qc == &quot;pass&quot;, treat %in% treat_levels) %&gt;% 
  mutate(treat = factor(treat, levels = treat_levels)) %&gt;% 
  droplevels()
stopifnot(nrow(samples) &gt; 0)
rep_col &lt;- setdiff(
  colnames(samples), c(&quot;sample&quot;, &quot;treat&quot;, &quot;target&quot;, &quot;input&quot;, &quot;label&quot;, &quot;qc&quot;)
)
samples[[rep_col]] &lt;- as.factor(samples[[rep_col]])
fdr_alpha &lt;- config$comparisons$fdr
db_method &lt;- match.arg(config$comparisons$method, c(&quot;sq-lt&quot;, &quot;ls-ql&quot;))
ihw_opts &lt;- c(&quot;regions&quot;, &quot;features&quot;, &quot;targets&quot;)
ihw_method &lt;- c(
  intersect(str_to_lower(config$comparisons$ihw), ihw_opts), &quot;none&quot;
)
ihw_method &lt;- ihw_method[[1]]</code></pre>
<pre class="r"><code>bam_path &lt;- here::here(config$paths$bam)
stopifnot(dir.exists(bam_path))
annotation_path &lt;- here::here(&quot;output&quot;, &quot;annotations&quot;)
stopifnot(dir.exists(annotation_path))
out_path &lt;- here::here(&quot;output&quot;, &quot;differential_binding&quot;, target)
if (!dir.exists(out_path)) dir.create(out_path, recursive = TRUE)</code></pre>
<pre class="r"><code>sq &lt;- file.path(annotation_path, &quot;seqinfo.rds&quot;) %&gt;% 
  read_rds()
blacklist &lt;- import.bed(here::here(config$external$blacklist), seqinfo = sq)
cb &lt;- config$genome$build %&gt;% 
  str_to_lower() %&gt;% 
  paste0(&quot;.cytobands&quot;) 
data(list = cb)
bands_df &lt;- get(cb)
stopifnot(
  colnames(bands_df) == c(&quot;chrom&quot;, &quot;chromStart&quot;, &quot;chromEnd&quot;, &quot;name&quot;, &quot;gieStain&quot;)
)

gtf_gene &lt;- read_rds(file.path(annotation_path, &quot;gtf_gene.rds&quot;))
id2gene &lt;- structure(gtf_gene$gene_name, names = gtf_gene$gene_id)
trans_models &lt;- file.path(annotation_path, &quot;trans_models.rds&quot;) %&gt;% 
  read_rds() 

external_features &lt;- GRanges(seqinfo = sq)
if (!is.null(config$external$features)) {
  external_features &lt;- suppressWarnings(
    import.gff(here::here(config$external$features), genome = sq)
  )
  keep_cols &lt;- !vapply(
    mcols(external_features), function(x) all(is.na(x)), logical(1)
  )
  mcols(external_features) &lt;- mcols(external_features)[keep_cols]
}
has_features &lt;- length(external_features) &gt; 0
gene_regions &lt;- file.path(annotation_path, &quot;gene_regions.rds&quot;) %&gt;% 
  read_rds()
regions &lt;- vapply(gene_regions, function(x) unique(x$region), character(1))

rna_path &lt;- here::here(config$external$rnaseq)
rnaseq &lt;- tibble(gene_id = character())
if (length(rna_path) &gt; 0) {
  stopifnot(file.exists(rna_path))
  if (str_detect(rna_path, &quot;tsv$&quot;)) rnaseq &lt;- read_tsv(rna_path)
  if (str_detect(rna_path, &quot;csv$&quot;)) rnaseq &lt;- read_csv(rna_path)
  if (!&quot;gene_id&quot; %in% colnames(rnaseq)) stop(&quot;Supplied RNA-Seq data must contain the column &#39;gene_id&#39;&quot;)
  gtf_gene &lt;- subset(gtf_gene, gene_id %in% rnaseq$gene_id)
  rna_lfc_col &lt;- colnames(rnaseq)[str_detect(str_to_lower(colnames(rnaseq)), &quot;logfc&quot;)][1]
  rna_fdr_col &lt;- colnames(rnaseq)[str_detect(str_to_lower(colnames(rnaseq)), &quot;fdr|adjp&quot;)][1]
}
has_rnaseq &lt;- as.logical(nrow(rnaseq))
tx_col &lt;- intersect(c(&quot;tx_id&quot;, &quot;transcript_id&quot;), colnames(rnaseq))
rna_gr_col &lt;- ifelse(length(tx_col) &gt; 0, &quot;transcript_id&quot;, &quot;gene_id&quot;)
rna_col &lt;- c(tx_col, &quot;gene_id&quot;)[[1]]</code></pre>
<pre class="r"><code>colours &lt;- read_rds(here::here(&quot;output&quot;, &quot;annotations&quot;, &quot;colours.rds&quot;)) %&gt;% 
  lapply(unlist)
region_colours &lt;- setNames(colours$regions, regions[names(colours$regions)])
if (has_features)
  feature_colours &lt;- setNames(
    colours$features, str_sep_to_title(names(colours$features))
  )
direction_colours &lt;- colours$direction %&gt;% 
  setNames(str_to_title(names(.))) %&gt;% 
  .[names(.) %in% c(&quot;Up&quot;, &quot;Down&quot;, &quot;Unchanged&quot;, &quot;Ambiguous&quot;)]
treat_colours &lt;- colours$treat[treat_levels]
fig_dev &lt;- knitr::opts_chunk$get(&quot;dev&quot;)
fig_type &lt;- fig_dev[[1]]
if (is.null(fig_type)) stop(&quot;Couldn&#39;t detect figure type&quot;)</code></pre>
<pre class="r"><code>hic &lt;- GInteractions()
hic_path &lt;- here::here(config$external$hic)
if (length(hic_path) &gt; 0)
  if (file.exists(hic_path)) {
    has_hic &lt;- TRUE
    hic &lt;- makeGenomicInteractionsFromFile(hic_path, type = &quot;bedpe&quot;)
    reg_combs &lt;- expand.grid(regions, regions) %&gt;% 
      as.matrix() %&gt;% 
      apply(
        MARGIN = 1, 
        function(x) {
          x &lt;- sort(factor(x, levels = regions))
          paste(as.character(x), collapse = &quot; - &quot;)
        }
      ) %&gt;% 
      unique()
    hic$regions &lt;- anchors(hic) %&gt;% 
      vapply(
        bestOverlap,
        y = GRangesList(lapply(gene_regions, granges)),
        character(length(hic))
      ) %&gt;% 
      apply(MARGIN = 2, function(x) regions[x]) %&gt;% 
      apply(
        MARGIN = 1, 
        function(x) {
          x &lt;- sort(factor(x, levels = regions))
          paste(as.character(x), collapse = &quot; - &quot;)
        }
      ) %&gt;% 
      factor(levels = reg_combs) %&gt;%
      fct_relabel(
        str_replace_all,
        pattern = &quot;Promoter \\([0-9kbp/\\+-]+\\)&quot;, replacement = &quot;Promoter&quot;
      )
    if (has_features) {
      feat_combs &lt;- expand.grid(names(feature_colours), names(feature_colours)) %&gt;% 
        as.matrix() %&gt;% 
        apply(
          MARGIN = 1, 
          function(x) {
            x &lt;- sort(factor(x, levels = names(feature_colours)))
            paste(as.character(x), collapse = &quot; - &quot;)
          }
        ) %&gt;% 
        unique()
      hic$features &lt;- vapply(
        anchors(hic),
        function(x) bestOverlap(
          x, external_features, var = &quot;feature&quot;, missing = &quot;no_feature&quot;
        ),
        character(length(hic))
      )  %&gt;% 
        apply(
          MARGIN = 1, 
          function(x) {
            x &lt;- sort(factor(x, levels = names(feature_colours)))
            paste(as.character(x), collapse = &quot; - &quot;)
          }
        ) %&gt;% 
        factor(levels = feat_combs) %&gt;%
        fct_relabel(str_sep_to_title, pattern = &quot;_&quot;)
    }
  }
stopifnot(is(hic, &quot;GInteractions&quot;))
seqlevels(hic) &lt;- seqlevels(sq)
seqinfo(hic) &lt;- sq
hic &lt;- hic[!overlapsAny(hic, blacklist)]
has_hic &lt;- as.logical(length(hic))</code></pre>
<pre class="r"><code>facet_labeller &lt;- as_labeller(
  c(
    c(
      `TRUE` = &quot;Union Peak&quot;,
      `FALSE` = &quot;No Union Peak&quot;
    ),
    structure(treat_levels, names = treat_levels),
    c(
      Up = glue(&quot;Increased {target} Binding&quot;),
      Down = glue(&quot;Decreased {target} Binding&quot;)
    ),
    c(
      AveExpr = &quot;logCPM&quot;, logFC = &quot;logFC&quot;
    )
  )
)</code></pre>
<pre class="r"><code>all_out &lt;- list(
  results = file.path(
    out_path, 
    glue(&quot;{target}_{treat_levels[[1]]}_{treat_levels[[2]]}-differential_binding.rds&quot;)
  ),
  csv = file.path(
    out_path, 
    glue(&quot;{target}_{treat_levels[[1]]}_{treat_levels[[2]]}-differential_binding.csv.gz&quot;)
  ),
  windows  = file.path(
    out_path,
    glue(&quot;{target}_{treat_levels[[1]]}_{treat_levels[[2]]}-filtered_windows.rds&quot;)
  ),
  up_regions = file.path(
    out_path, glue(&quot;{target}_{treat_levels[[1]]}_{treat_levels[[2]]}-up.bed&quot;)
  ),
  down_regions = file.path(
    out_path, glue(&quot;{target}_{treat_levels[[1]]}_{treat_levels[[2]]}-down.bed&quot;)
  ),
  de_genes = file.path(
    out_path, glue(&quot;{target}_{treat_levels[1]}_{treat_levels[2]}-DE_genes.csv&quot;)
  ),
  enrichment = file.path(
    out_path, glue(&quot;{target}_{treat_levels[1]}_{treat_levels[2]}-enrichment.csv&quot;)
  ),
  rna_enrichment = file.path(
    out_path, 
    glue(&quot;{target}_{treat_levels[1]}_{treat_levels[2]}-rnaseq_enrichment.csv&quot;)
  ),
  renv = here::here(
    &quot;output/envs&quot;,
    glue(
      &quot;{target}_{treat_levels[[1]]}_{treat_levels[[2]]}-differential_binding.RData&quot;
    )
  )
)
## Initialise outputs from RNA-Seq
cmn_diff &lt;- cmn_up &lt;- cmn_down &lt;- tibble(leadingEdge = list())</code></pre>
<div id="outline" class="section level1">
<h1>Outline</h1>
<div id="differential-binding" class="section level2">
<h2>Differential Binding</h2>
<p>This step of the workflow uses a sliding window approach to
differential binding, as a logical extension to the methods suggested
and implemented in the <code>csaw</code> package <span
class="citation">(<a href="#ref-LunSmythF1000" role="doc-biblioref">A.
T. Lun and Smyth 2015</a>)</span>.</p>
<ol style="list-style-type: decimal">
<li>Union Peaks derived from <code>macs2 callpeak</code> <span
class="citation">(<a href="#ref-Zhang18798982"
role="doc-biblioref">Zhang et al. 2008</a>)</span> results are used to
determine inclusion/exclusion values for sliding windows</li>
<li>Smooth Quantile Normalisation <span class="citation">(<a
href="#ref-HicksSQN2017" role="doc-biblioref">Stephanie C. Hicks et al.
2017</a>)</span> is used on the set of logCPM values obtained from
retained sliding windows. This was chosen given that many transcription
factors can show vastly different cytoplasmic/nuclear distributions
across treatments</li>
<li>The <em>limma-trend</em> method <span class="citation">(<a
href="#ref-LawVoom2014" role="doc-biblioref">Law et al. 2014</a>)</span>
is used alongside <code>treat</code> <span class="citation">(<a
href="#ref-McCarthyTreat2009" role="doc-biblioref">McCarthy and Smyth
2009</a>)</span> for detection of differential binding. This should
correctly control the FDR, and allows the specification as a percentage
of a suitable threshold for the estimated change in binding, below which
we are not interested in any changed signal</li>
<li>Independent Hypothesis Weighting (IHW) <span class="citation">(<a
href="#ref-IgnatiadisIHW2016" role="doc-biblioref">Ignatiadis et al.
2016</a>)</span> is additionally used to improve the power of the
results. Under this strategy, p-values are partitioned based on the
presence/absence of any other ChIP targets under consideration in any
condition, which is considered here to be a statistically independent
variable.</li>
</ol>
<p>The workflow also depends heavily on the function implemented in the
Bioconductor package <a
href="https://bioconductor.org/packages/release/bioc/html/extraChIPs.html">extraChIPs</a></p>
</div>
<div id="enrichment-analysis" class="section level2">
<h2>Enrichment Analysis</h2>
<p>Beyond the simple analysis of differential binding, peaks are mapped
to genes and enrichment testing is performed on the following:</p>
<ol style="list-style-type: decimal">
<li>Genes mapped to any window with detected H3K27ac are compared to all
genes not mapped to any window</li>
<li>Genes mapped to <em>all differentially bound windows</em> are
compared to genes mapped to windows which are not differentially
bound</li>
<li>Genes mapped to windows with <em>increasing</em> H3K27ac binding are
compared to genes mapped to windows which are not differentially
bound</li>
<li>Genes mapped to windows with <em>decreasing</em> H3K27ac binding are
compared to genes mapped to windows which are not differentially
bound</li>
</ol>
<p>Enrichment testing is performed using <code>goseq</code> <span
class="citation">(<a href="#ref-YoungGoseq2010"
role="doc-biblioref">Young et al. 2010</a>)</span> with <em>no term
accounting for sampling bias</em>, except when comparing genes mapped to
any window. For this case <strong>only</strong>, gene width is used to
capture any sampling bias and Wallenius’ Non-Central Hypergeometric
Distribution is used. As RNA-seq data was provided, the genes considered
for enrichment analysis are the 12,990 genes considered as detected in
the RNA-Seq data.</p>
</div>
<div id="incorporation-with-rna-seq" class="section level2">
<h2>Incorporation with RNA-Seq</h2>
<p>Any association between differentially expressed genes and
differentially bound sites will be assessed using Gene Set Enrichment
Analysis <span class="citation">(<a href="#ref-SubramanianGsea2005"
role="doc-biblioref">Subramanian et al. 2005</a>)</span>, as implemented
in the fgsea package <span class="citation">(<a
href="#ref-KorotkevichFgsea2019" role="doc-biblioref">Korotkevich,
Sukhov, and Sergushichev 2019</a>)</span>. The sets of genes associated
with changed binding will be subset by regions and any provided external
features, and these novel gene-sets will be used to test for enrichment
within the RNA-Seq results. ChIP-seq derived gene-sets will be tested
for differential expression using genes ranked directionally and by
significance alone.</p>
</div>
</div>
<div id="data-preparation" class="section level1">
<h1>Data Preparation</h1>
<pre class="r"><code>union_peaks &lt;- file.path(macs2_path, glue(&quot;{target}_union_peaks.bed&quot;)) %&gt;%
    import.bed(seqinfo = sq)</code></pre>
<pre class="r"><code>bfl &lt;- bam_path %&gt;%
  file.path(glue(&quot;{unique(c(samples$sample, samples$input))}.bam&quot;)) %&gt;% 
  BamFileList() %&gt;%
  setNames(c(samples$sample, unique(samples$input)))</code></pre>
<pre class="r"><code>ys &lt;- 1000
bfl_summary &lt;- bfl %&gt;%
    bplapply(
        function(x) {
            yieldSize(x) &lt;- ys
            sbp &lt;- ScanBamParam(what = c(&quot;qwidth&quot;, &quot;mapq&quot;))
            open(x)
            vals &lt;- scanBam(x, param = sbp)[[1]]
            close(x)
            list(as_tibble(vals))
        }
    ) %&gt;% 
    as_tibble() %&gt;%
    pivot_longer(everything(), names_to = &quot;sample&quot;) %&gt;%
    unnest(value) %&gt;% 
    pivot_longer(
        cols = c(&quot;qwidth&quot;, &quot;mapq&quot;),
        names_to = &quot;stat&quot;,
        values_to = &quot;value&quot;
    ) %&gt;%
    left_join(samples, by = &quot;sample&quot;) %&gt;%
    mutate(
        across(
            all_of(c(&quot;target&quot;, &quot;label&quot;, &quot;treat&quot;)),
      \(x) str_replace_na(x, replacement = &quot;Input&quot;)
        ),
        treat = factor(treat, levels = c(&quot;Input&quot;, treat_levels)),
        label = fct_inorder(label)
    ) %&gt;% 
    split(f = .$stat)
anyDups &lt;- bplapply(
    bfl,
    function(x) {
        sbp &lt;- ScanBamParam(
            flag = scanBamFlag(isDuplicate = TRUE),
            which = GRanges(sq)[which.min(seqlengths(sq))],
            what = &quot;qname&quot;
            )
        length(scanBam(x, param = sbp)[[1]]$qname)  &gt; 0
    }
) %&gt;%
  unlist()
anyPE &lt;- bplapply(
    bfl,
    function(x){
        yieldSize(x) &lt;- ys
        open(x)
        flag &lt;- scanBam(x, param=ScanBamParam(what=&quot;flag&quot;))[[1]]$flag
        close(x)
        any(bamFlagTest(flag, &quot;isPaired&quot;))
    }
) %&gt;% 
  unlist()</code></pre>
<p>Taking the first 1,000 alignments, a brief inspection of the Bam
Files revealed:</p>
<ul>
<li>Aligned read lengths ranged between 50 and 76. The median length was
76.</li>
<li>MAPQ scores of aligned reads ranged between 2 and 42. The median
score was 30.</li>
<li>All bam files have been de-duplicated</li>
<li>All alignments were single-end</li>
</ul>
<div id="sliding-window-counts" class="section level2">
<h2>Sliding Window Counts</h2>
<pre class="r"><code>greylist &lt;- file.path(
  annotation_path, glue(&quot;{unique(samples$input)}_greylist.bed&quot;)
  ) %&gt;%
  lapply(import.bed, seqinfo = sq) %&gt;%
  GRangesList() %&gt;%
  unlist() %&gt;%
  reduce()</code></pre>
<p>Prior to this workflow, high-signal regions were detected in any
input samples associated with H3K27ac libraries and grey-lists formed.
For these samples, this constituted 3,985 regions with total width of
82,913kb. Regions assigned to the greylist were added to the blacklisted
regions and excluded from all analyses.</p>
<pre class="r"><code>macs2_merged_logs &lt;- file.path(
  macs2_path, glue(&quot;{target}_{treat_levels}_merged_callpeak.log&quot;)
    ) %&gt;%
    importNgsLogs()
fl &lt;- max(macs2_merged_logs$fragment_length)
rp &lt;- readParam(
  pe = ifelse(any(anyPE), &quot;both&quot;, &quot;none&quot;),
  dedup = any(anyDups),
  restrict = seqnames(sq),
  discard = c(granges(blacklist), greylist)
)
win_step &lt;- 10*(1 + fl %/% 60)
win_size &lt;- 3*win_step
window_counts &lt;- windowCounts(
  bam.files = bfl,
  spacing = win_step,
  width = win_size,
  ext = fl,
  filter = length(bfl) - 1,
  param = rp,
  BPPARAM = bpparam()
)
colData(window_counts) &lt;- colData(window_counts) %&gt;% 
  as_tibble(rownames = &quot;sample&quot;) %&gt;% 
  dplyr::select(all_of(c(&quot;sample&quot;, &quot;bam.files&quot;, &quot;totals&quot;, &quot;ext&quot;, &quot;rlen&quot;))) %&gt;% 
  left_join(samples, by = &quot;sample&quot;) %&gt;% 
  mutate(
    treat = fct_explicit_na(treat, &quot;Input&quot;), 
    target = str_replace_na(target, &quot;Input&quot;)
  ) %&gt;% 
  as.data.frame() %&gt;% 
  column_to_rownames(&quot;sample&quot;) %&gt;% 
  DataFrame()
window_counts &lt;- sortSeqlevels(window_counts)
seqinfo(window_counts) &lt;- sq</code></pre>
<p>Using the macs2-estimated fragment length of 230nt, a set of genomic
sliding windows were defined using a window size of 120bp, sliding in
increments of 40bp. With the exclusion of black-listed and grey-listed
regions, all alignments within each window were counted for each
H3K27ac-associated sample, and all relevant input samples. Any windows
with fewer than 6 alignments (i.e.1 read/sample) across all samples were
discarded, leaving a total of 34,610,621 sliding windows, covering 52%
of the reference genome.</p>
<p>These windows were then filtered using the <code>dualFilter()</code>
function from <code>extraChIPs</code>, discarding windows with low
counts. Under this approach, two thresholds are determined above which
windows are retained, and with values chosen to return 60% of sliding
windows which overlap a macs2-derived union peak. These values are set
to filter based on 1) signal relative to input over an extended range
and, 2) overall signal level. Both filtering strategies rely on the
infrastructure provided by <code>csaw</code> <span class="citation">(<a
href="#ref-LunSmythCsaw2014" role="doc-biblioref">A. T. L. Lun and Smyth
2014</a>)</span>.</p>
<pre class="r"><code>filtered_counts &lt;- dualFilter(
  x = window_counts[, samples$sample],
  bg = window_counts[, samples$input],
  ref = union_peaks,
  keep.totals = TRUE,
  q = config$comparisons$filter_q
)
colData(filtered_counts) &lt;- droplevels(colData(filtered_counts))
n_max &lt;- min(5e4, nrow(filtered_counts))
sample_labeller &lt;- as_labeller(
  setNames(colData(filtered_counts)$label, colnames(filtered_counts))
)</code></pre>
<pre class="r"><code>list(
  `Genomic Windows` = window_counts,
  `Retained Windows` = filtered_counts,
  `Union Peaks` = union_peaks
) %&gt;% 
  lapply(granges) %&gt;% 
  lapply(
    function(x) {
      tibble(
        N = comma(length(x)),
        `Total Width (kb)` = comma(sum(width(reduce(x))) / 1e3),
        `Median Width (bp)` = median(width(x))
      )
    }
  ) %&gt;% 
  lapply(list) %&gt;% 
  as_tibble() %&gt;% 
  pivot_longer(everything(), names_to = &quot;Dataset&quot;) %&gt;% 
  unnest(everything()) %&gt;% 
  left_join(
    tibble(
      Dataset = c(&quot;Retained Windows&quot;, &quot;Union Peaks&quot;),
      `Unique (kb)` = c(
        round(sum(width(setdiff(granges(filtered_counts), union_peaks))) / 1e3, 1),
        round(sum(width(setdiff(union_peaks, granges(filtered_counts)))) / 1e3, 1)
      ),
      `% Unique` = percent(
        1e3*`Unique (kb)` / c(
          sum(width(reduce(granges(filtered_counts)))),
          sum(width(union_peaks))
        ),
        0.1
      )
    )
  ) %&gt;% 
  pander(
    justify = &quot;lrrrrr&quot;,
    caption = paste(
      &quot;A dual filtering strategy was used based on retaining&quot;, 
      percent(config$comparisons$filter_q, 0.1), 
      &quot;of genomic windows overlapping the union peaks identified by &quot;, 
      &quot;`macs2 callpeak` on merged samples. This approach combined both &quot;, 
      &quot;1) expression percentiles, and 2) signal strength in relation to the input sample.&quot;,
      &quot;The complete set of sliding windows covered the majority of the genome,&quot;,
      &quot;whilst those retained after filtering were focussed on strong binding&quot;,
      &quot;signal. Union peaks were as identified by `macs2 callpeak` in a&quot;, 
      &quot;previous step of the workflow.&quot;,
      &quot;Importantly, union peaks are non-overlapping, whilst the other two&quot;,
      &quot;datasets are derived from overlapping sliding windows.&quot;,
      &quot;This strategy of filtering the set of initial sliding windows retained&quot;,
      percent(
        1 - sum(width(setdiff(union_peaks, rowRanges(filtered_counts)))) / sum(width(union_peaks))
      ), 
      &quot;of the genomic regions covered by union peaks, with&quot;, 
      percent(
        sum(width(setdiff(rowRanges(filtered_counts), union_peaks))) / sum(width(reduce(granges(filtered_counts)))))
      , 
      &quot;of the retained windows being outside genomic regions covered by union peaks.&quot;,
      ifelse(
        n_max == 5e4, 
        &quot;For subsequent density and RLE plots, a subsample of 50,000 genomic regions will be used for speed.&quot;, 
        &quot;&quot;
      )
    )
  )</code></pre>
<table>
<caption>A dual filtering strategy was used based on retaining 60.0% of
genomic windows overlapping the union peaks identified by
<code>macs2 callpeak</code> on merged samples. This approach combined
both 1) expression percentiles, and 2) signal strength in relation to
the input sample. The complete set of sliding windows covered the
majority of the genome, whilst those retained after filtering were
focussed on strong binding signal. Union peaks were as identified by
<code>macs2 callpeak</code> in a previous step of the workflow.
Importantly, union peaks are non-overlapping, whilst the other two
datasets are derived from overlapping sliding windows. This strategy of
filtering the set of initial sliding windows retained 65% of the genomic
regions covered by union peaks, with 0% of the retained windows being
outside genomic regions covered by union peaks. For subsequent density
and RLE plots, a subsample of 50,000 genomic regions will be used for
speed.</caption>
<colgroup>
<col width="19%" />
<col width="13%" />
<col width="19%" />
<col width="20%" />
<col width="14%" />
<col width="11%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Dataset</th>
<th align="right">N</th>
<th align="right">Total Width (kb)</th>
<th align="right">Median Width (bp)</th>
<th align="right">Unique (kb)</th>
<th align="right">% Unique</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Genomic Windows</td>
<td align="right">34,610,621</td>
<td align="right">1,605,126</td>
<td align="right">120</td>
<td align="right"></td>
<td align="right"></td>
</tr>
<tr class="even">
<td align="left">Retained Windows</td>
<td align="right">1,150,761</td>
<td align="right">49,018</td>
<td align="right">120</td>
<td align="right">175.8</td>
<td align="right">0.4%</td>
</tr>
<tr class="odd">
<td align="left">Union Peaks</td>
<td align="right">49,998</td>
<td align="right">75,059</td>
<td align="right">957</td>
<td align="right">26,217</td>
<td align="right">34.9%</td>
</tr>
</tbody>
</table>
</div>
<div id="normalisation" class="section level2">
<h2>Normalisation</h2>
<pre class="r"><code>library(quantro)
library(qsmooth)</code></pre>
<p>The <code>quantro</code> test was first applied <span
class="citation">(<a href="#ref-HicksQuantro2015"
role="doc-biblioref">S. C. Hicks and Irizarry 2015</a>)</span> to
determine if treatment-specific binding distributions were found in the
data. Whilst this may not always be the case, the robustness of
smooth-quantile normalisation (SQN) <span class="citation">(<a
href="#ref-HicksSQN2017" role="doc-biblioref">Stephanie C. Hicks et al.
2017</a>)</span> will be applicable if data is drawn from different or
near-identical distributions, and this method was applied grouping data
by treatment.</p>
<pre class="r"><code>qtest &lt;- quantro(
  assay(filtered_counts, &quot;logCPM&quot;),
  groupFactor = filtered_counts$treat
)
pander(
  anova(qtest), 
  caption = paste(
    &quot;*Results from qtest suggesting that the two treatment groups are drawn from&quot;,
    ifelse(
      qtest@anova$`Pr(&gt;F)`[[1]] &lt; 0.05,
      &quot;different distributions.*&quot;,
      &quot;the same distribution.*&quot;
    )
  )
)</code></pre>
<table style="width:92%;">
<caption><em>Results from qtest suggesting that the two treatment groups
are drawn from the same distribution.</em></caption>
<colgroup>
<col width="25%" />
<col width="6%" />
<col width="15%" />
<col width="16%" />
<col width="13%" />
<col width="13%" />
</colgroup>
<thead>
<tr class="header">
<th align="center"> </th>
<th align="center">Df</th>
<th align="center">Sum Sq</th>
<th align="center">Mean Sq</th>
<th align="center">F value</th>
<th align="center">Pr(&gt;F)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center"><strong>groupFactor</strong></td>
<td align="center">1</td>
<td align="center">0.001491</td>
<td align="center">0.001491</td>
<td align="center">5.334</td>
<td align="center">0.08207</td>
</tr>
<tr class="even">
<td align="center"><strong>Residuals</strong></td>
<td align="center">4</td>
<td align="center">0.001118</td>
<td align="center">0.0002795</td>
<td align="center"></td>
<td align="center"></td>
</tr>
</tbody>
</table>
<pre class="r"><code>qs &lt;-qsmooth(
  assay(filtered_counts, &quot;logCPM&quot;), group_factor = filtered_counts$treat
)
assay(filtered_counts, &quot;qsmooth&quot;) &lt;- qsmoothData(qs)</code></pre>
</div>
<div id="data-inspection" class="section level2 tabset">
<h2 class="tabset">Data Inspection</h2>
<div id="qsmooth-weights" class="section level3">
<h3>QSmooth Weights</h3>
<pre class="r"><code>qsmoothPlotWeights(qs, xLab = &quot;Quantiles&quot;, yLab = &quot;Weights&quot;, mainLab = &quot;QSmooth Weights&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-qs-weights-1.png" alt="*Quantile-specific weights used by the Smooth-Quantile normalisation. Low weights indicate signal quantiles which appear to be more specific within a group, whilst higher weights indicate similarity between groups.*" width="960" />
<p class="caption">
<em>Quantile-specific weights used by the Smooth-Quantile normalisation.
Low weights indicate signal quantiles which appear to be more specific
within a group, whilst higher weights indicate similarity between
groups.</em>
</p>
</div>
</div>
<div id="density-plot-all-sliding-windows" class="section level3">
<h3>Density Plot: All Sliding Windows</h3>
<pre class="r"><code>plotAssayDensities(
  window_counts, trans = &quot;log1p&quot;, n_max = n_max, colour = &quot;treat&quot;,
  linetype = &quot;target&quot;
) +
  scale_colour_manual(values = colours$treat) + 
  labs(
    x = &quot;log(counts + 1)&quot;, y = &quot;Density&quot;, colour = &quot;Treat&quot;, linetype = &quot;Target&quot;
  )</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-density-windows-1.png" alt="*Density plot for __all windows prior to the selection of windows__ more likely to contain true signal. Retained windows will be those at the upper end, whilst discarded windows will be at the lower end.*" width="768" />
<p class="caption">
<em>Density plot for <strong>all windows prior to the selection of
windows</strong> more likely to contain true signal. Retained windows
will be those at the upper end, whilst discarded windows will be at the
lower end.</em>
</p>
</div>
</div>
<div id="logcpm-densities" class="section level3">
<h3>logCPM Densities</h3>
<pre class="r"><code>a &lt;- filtered_counts %&gt;% 
  plotAssayDensities(&quot;logCPM&quot;, colour = &quot;treat&quot;, n_max = n_max) +
  scale_colour_manual(values = treat_colours) +
  labs(colour = &quot;Treat&quot;)
b &lt;- filtered_counts %&gt;% 
  plotAssayDensities(&quot;qsmooth&quot;, colour = &quot;treat&quot;, n_max = n_max) +
  scale_colour_manual(values = treat_colours) +
  labs(
    x = &quot;Normalised logCPM&quot;, colour = &quot;Treat&quot;
  )
plot_grid(
  a + theme(legend.position = &quot;none&quot;), 
  b + theme(legend.position = &quot;none&quot;),
  get_legend(b),
  nrow = 1,
  rel_widths = c(5, 5, 1),
  labels = c(&quot;A&quot;, &quot;B&quot;)
)</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-logcpm-densities-1.png" alt="*logCPM distributions for all retained windows A) prior to smooth quantile normalisation, and 2) after SQ normalisation.*" width="960" />
<p class="caption">
<em>logCPM distributions for all retained windows A) prior to smooth
quantile normalisation, and 2) after SQ normalisation.</em>
</p>
</div>
</div>
<div id="rle-pre-normalisation" class="section level3">
<h3>RLE (Pre-Normalisation)</h3>
<pre class="r"><code>filtered_counts %&gt;% 
  plotAssayRle(
  &quot;logCPM&quot;, fill = &quot;treat&quot;, rle_group = &quot;treat&quot;, n_max = n_max
) +
  geom_hline(yintercept = 0) + 
  facet_grid(.~treat, scales = &quot;free_x&quot;, space = &quot;free_x&quot;) +
  scale_x_discrete(labels = sample_labeller) +
  scale_fill_manual(values = treat_colours) +
  labs(x = &quot;Sample&quot;, fill = &quot;Treat&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-rle-1.png" alt="*RLE plot showing logCPM values. RLE values were calculated within each treatment group to account for the potentially different binding dynamics of H3K27ac.*" width="960" />
<p class="caption">
<em>RLE plot showing logCPM values. RLE values were calculated within
each treatment group to account for the potentially different binding
dynamics of H3K27ac.</em>
</p>
</div>
</div>
<div id="rle-post-normalisation" class="section level3">
<h3>RLE (Post-Normalisation)</h3>
<pre class="r"><code>filtered_counts %&gt;% 
  plotAssayRle(
  &quot;qsmooth&quot;, fill = &quot;treat&quot;, rle_group = &quot;treat&quot;, n_max = n_max
) +
  geom_hline(yintercept = 0) + 
  facet_grid(.~treat, scales = &quot;free_x&quot;, space = &quot;free_x&quot;) +
  scale_x_discrete(labels = sample_labeller) +
  scale_fill_manual(values = treat_colours) +
  labs(x = &quot;Sample&quot;, fill = &quot;Treat&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-rle-normalised-1.png" alt="*RLE plot showing normalised logCPM. RLE values were calculated within each treatment group to account for the potentially different binding dynamics of H3K27ac.*" width="960" />
<p class="caption">
<em>RLE plot showing normalised logCPM. RLE values were calculated
within each treatment group to account for the potentially different
binding dynamics of H3K27ac.</em>
</p>
</div>
</div>
<div id="pca" class="section level3">
<h3>PCA</h3>
<pre class="r"><code>a &lt;- plotAssayPCA(filtered_counts, &quot;logCPM&quot;, colour = &quot;treat&quot;, label = &quot;label&quot;) +
  scale_colour_manual(values = treat_colours) +
  ggtitle(&quot;Pre-Normalisation&quot;)
b &lt;- plotAssayPCA(filtered_counts, &quot;qsmooth&quot;, colour = &quot;treat&quot;, label = &quot;label&quot;) +
  scale_colour_manual(values = treat_colours) +
  ggtitle(&quot;Post-Normalisation&quot;)
plot_grid(
  a + theme(legend.position = &quot;none&quot;), b + theme(legend.position = &quot;none&quot;), 
  get_legend(b + labs(colour = &quot;Treat&quot;)), rel_widths = c(1, 1, 0.2), 
  nrow = 1, labels = c(&quot;A&quot;, &quot;B&quot;)
  )</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-pca-1.png" alt="*PCA plots for logCPM values A) before and B) after Smooth Quantile normalisation*" width="960" />
<p class="caption">
<em>PCA plots for logCPM values A) before and B) after Smooth Quantile
normalisation</em>
</p>
</div>
</div>
</div>
</div>
<div id="differential-binding-analysis" class="section level1">
<h1>Differential Binding Analysis</h1>
<div id="primary-analysis" class="section level2 tabset">
<h2 class="tabset">Primary Analysis</h2>
<pre class="r"><code>X &lt;- model.matrix(~treat, data = colData(filtered_counts)) %&gt;%
  set_colnames(str_remove(colnames(.), &quot;treat&quot;))
colData(filtered_counts)$design &lt;- X
paired_cors &lt;- block &lt;- txt &lt;- NULL
if (config$comparisons$paired) {
  block &lt;- colData(filtered_counts)[[rep_col]]
  set.seed(1e6)
  ind &lt;- sample.int(nrow(filtered_counts), n_max, replace = FALSE)
  paired_cors &lt;- duplicateCorrelation(
    object = assay(filtered_counts, &quot;qsmooth&quot;)[ind, ],
    design = X,
    block = block
  )$consensus.correlation
  txt &lt;- glue(
    &quot;Data were nested within {{rep_col}} as a potential source of correlation. &quot;,
    &quot;The estimated correlation within replicate samples was $\\hat{\\rho} = {{round(paired_cors, 3)}}$&quot;,
    .open = &quot;{{&quot;, .close = &quot;}}&quot;
  )
}
fc &lt;- ifelse(is.null(config$comparisons$fc), 1, config$comparisons$fc)
fit &lt;- assay(filtered_counts, &quot;qsmooth&quot;) %&gt;%
  lmFit(design = X, block = block, correlation = paired_cors) %&gt;% 
  treat(fc = fc, trend = TRUE, robust = FALSE)
fit_mu0 &lt;- assay(filtered_counts, &quot;qsmooth&quot;) %&gt;%
  lmFit(design = X, block = block, correlation = paired_cors) %&gt;% 
  treat(fc = 1, trend = TRUE, robust = FALSE)
res_cols &lt;- c(&quot;logFC&quot;, &quot;AveExpr&quot;, &quot;t&quot;, &quot;P.Value&quot;, &quot;fdr&quot;)
rowData(filtered_counts) &lt;- rowData(filtered_counts) %&gt;% 
  .[!colnames(.) %in% c(res_cols, &quot;p_mu0&quot;)] %&gt;% 
  cbind(
    fit %&gt;% 
      topTable(sort.by = &quot;none&quot;, number = Inf) %&gt;% 
      as.list %&gt;% 
      setNames(res_cols) %&gt;% 
      c(
        list(
          p_mu0 = topTable(fit_mu0, sort.by = &quot;none&quot;, number = Inf)$P.Value
        )
      )
  ) </code></pre>
<p>After SQ-normalisation of logCPM values, the <code>limma-trend</code>
method <span class="citation">(<a href="#ref-LawVoom2014"
role="doc-biblioref">Law et al. 2014</a>)</span> was applied to all
retained windows. A simple linear model was fitted taking E2 as the
baseline and estimating the effects of E2DHT on H3K27ac binding within
each sliding window, incorporating a trended prior-variance.</p>
<p>After model fitting, hypothesis testing was performed testing:</p>
<p><span class="math display">\[
H_0: -\lambda &lt; \mu &lt; \lambda
\]</span> against <span class="math display">\[
H_A: |\mu| &gt; \lambda
\]</span> where <span class="math inline">\(\mu\)</span> represents the
true mean change in H3K27ac binding. The value <span
class="math inline">\(\lambda =\)</span> 0.26 was chosen as this
corresponds to a 20% change in detected signal on the log<sub>2</sub>
scale. This is known as the <code>treat</code> method <span
class="citation">(<a href="#ref-McCarthyTreat2009"
role="doc-biblioref">McCarthy and Smyth 2009</a>)</span>, and p-values
were obtained for each initial window, before merging adjoining
windows.</p>
<p>For subsequent classification when combining across multiple
differential binding results, an additional p-value testing <span
class="math inline">\(\mu = 0\)</span> was added, however this is not
used for detection of sites showing evidence of altered H3K27ac
binding.</p>
<pre class="r"><code>merged_results &lt;- mergeByCol(
  filtered_counts, col = &quot;AveExpr&quot;, merge_within = 2 * win_step,
  inc_cols = &quot;p_mu0&quot;
) %&gt;% 
  mutate(
    overlaps_ref = overlapsAny(., union_peaks),
    direction = case_when(
      n_up == 0 &amp; n_down == 0 ~ &quot;Unchanged&quot;,
      logFC &gt; 0 &amp; n_up &gt; n_down ~ &quot;Up&quot;,
      logFC &lt; 0 &amp; n_up &lt; n_down ~ &quot;Down&quot;,
      TRUE ~ &quot;Ambiguous&quot;
    ) %&gt;% 
      factor(levels = c(&quot;Up&quot;, &quot;Down&quot;, &quot;Ambiguous&quot;, &quot;Unchanged&quot;)) %&gt;% 
      droplevels()
  ) </code></pre>
<p>After selection of the 1,150,761 sliding windows (120bp), these were
merged into 33,976 genomic regions with size ranging from 120 to
33920bp, with the median width being 1040bp. Representative estimates of
differential binding (i.e. logFC) were taken from the sliding window
with the <em>highest average signal</em> across all samples, within each
set of windows to be merged. Similarly, p-values from the above tests
were also selected from the same sliding window as representative of the
merged region <span class="citation">(<a href="#ref-LunSmythF1000"
role="doc-biblioref">A. T. Lun and Smyth 2015</a>)</span>. The number of
windows showing increased or decreased binding within each merged region
were also included, by counting those within each set of windows with
p-values lower than the selected window. FDR-adjustment was performed
with 177 regions (0.5%) showing significance, based on an FDR-adjusted
p-value alone.</p>
</div>
<div id="independent-hypothesis-weighting"
class="section level2 tabset">
<h2 class="tabset">Independent Hypothesis Weighting</h2>
<pre class="r"><code>fdr_column &lt;- str_subset(names(mcols(merged_results)), &quot;fdr|FDR&quot;)
## Add status (direction) for the ihw = &quot;none&quot; possibility
merged_results &lt;- merged_results %&gt;% 
  mutate(
    status = case_when(
      !!sym(fdr_column) &lt; fdr_alpha ~ as.character(direction),
      !!sym(fdr_column) &gt;= fdr_alpha ~ &quot;Unchanged&quot;
    ) %&gt;% 
      factor(levels = c(levels(direction), &quot;Unchanged&quot;)) %&gt;% 
      droplevels()
  )</code></pre>
<pre class="r"><code>other_targets &lt;- here::here(config$samples$file) %&gt;% 
  read_tsv() %&gt;% 
  pull(&quot;target&quot;) %&gt;% 
  unique() %&gt;% 
  setdiff(target)
ihw_all &lt;- other_targets %&gt;% 
  sapply(
    function(x) {
      here::here(dirname(macs2_path), x, glue(&quot;{x}_union_peaks.bed&quot;)) %&gt;%
        import.bed(seqinfo = sq)
    },
    simplify = FALSE
  ) %&gt;% 
  lapply(granges) %&gt;% 
  lapply(
    function(x) overlapsAny(merged_results, x)
  ) %&gt;% 
  as_tibble() %&gt;% 
  mutate(
    range = as.character(merged_results), id = seq_along(range)
  ) %&gt;% 
  pivot_longer(
    cols = all_of(other_targets),
    names_to = &quot;targets&quot;, 
    values_to = &quot;overlap&quot;
  ) %&gt;% 
  group_by(id, range) %&gt;% 
  summarise(
    targets = ifelse(
      any(overlap), 
      paste(sort(targets[overlap]), collapse = &quot;+ &quot;),
      &quot;No Other Targets&quot;
    ),
    .groups = &quot;drop&quot;
  ) %&gt;% 
  arrange(id) %&gt;% 
  mutate(targets = fct_infreq(targets)) %&gt;% 
  .[[&quot;targets&quot;]]
n_levels &lt;- sum(table(ihw_all) &gt; 1e3) - 1
merged_results$ihw_covariate &lt;- fct_lump_n(ihw_all, n = n_levels)
ihw_proceed &lt;- length(levels(merged_results$ihw_covariate)) &gt; 1
if (ihw_proceed)  fdr_column &lt;- &quot;fdr_ihw&quot;
if (ihw_proceed) {
  ihw &lt;- ihw(
    pvalues = merged_results$P.Value,
    covariates = merged_results$ihw_covariate,
    alpha &lt;- fdr_alpha,
    covariate_type = &quot;nominal&quot;
  )
  merged_results &lt;- mutate(merged_results, fdr_ihw = adj_pvalues(ihw))
} 
merged_results &lt;- merged_results %&gt;% 
  mutate(
    status = case_when(
      !!sym(fdr_column) &lt; fdr_alpha ~ as.character(direction),
      !!sym(fdr_column) &gt;= fdr_alpha ~ &quot;Unchanged&quot;
    ) %&gt;% 
      factor(levels = c(levels(direction), &quot;Unchanged&quot;)) %&gt;% 
      droplevels()
  )</code></pre>
<p>Independent Hypothesis Weighting (IHW) <span class="citation">(<a
href="#ref-IgnatiadisIHW2016" role="doc-biblioref">Ignatiadis et al.
2016</a>)</span> was then used to partition the raw p-values for H3K27ac
differential binding by the detection of the other ChIP targets under
investigation in this workflow (AR and ER). The presence of AR and ER
was defined simply using the union peaks detected under any treatment by
<code>macs2 callpeak</code>, as determined <a
href="H3K27ac_macs2_summary.html">previously</a>. This allows
recalculation of the FDR using <em>weighted p-values</em> instead of
<em>raw p-values</em>. In order for IHW to be a viable strategy,
partitions should be greater than 1000. The provided union peaks were
used as initial partitions in combination, merging the smallest groups
below this size until all partitions were of a suitable size.</p>
<p>Windows were classified as overlapping a peak from a secondary ChIP
target if <em>any</em> section of the window overlapped the secondary
peak.</p>
<div id="summary-table" class="section level3">
<h3>Summary Table</h3>
<pre class="r"><code>ihw %&gt;%
  as.data.frame() %&gt;% 
  mutate(
    fdr = p.adjust(pvalue, &quot;fdr&quot;),
    status = case_when(
      fdr &lt; fdr_alpha &amp; adj_pvalue &gt; fdr_alpha ~ &quot;Lost&quot;,
      fdr &lt; fdr_alpha &amp; adj_pvalue &lt; fdr_alpha ~ &quot;Retained&quot;,
      fdr &gt; fdr_alpha &amp; adj_pvalue &lt; fdr_alpha ~ &quot;Gained&quot;,
      fdr &gt; fdr_alpha &amp; adj_pvalue &gt; fdr_alpha ~ &quot;Never Significant&quot;
    ) %&gt;% 
      factor(
        levels = c(&quot;Never Significant&quot;, &quot;Retained&quot;, &quot;Gained&quot;, &quot;Lost&quot;)
      )
  ) %&gt;% 
  as_tibble() %&gt;% 
  group_by(covariate, status) %&gt;% 
  tally() %&gt;% 
  ungroup() %&gt;% 
  complete(status = status, covariate = covariate, fill = list(n = 0)) %&gt;% 
  pivot_wider(
    names_from = status, values_from = n, values_fill = 0
  ) %&gt;% 
  mutate(
    `Nett Change` = Gained - Lost
  ) %&gt;% 
  bind_rows(
    summarise_if(., is.numeric, sum)
  ) %&gt;% 
  mutate(
    covariate = str_replace_na(covariate, &quot;Total&quot;),
    Significant = Retained + Gained
  ) %&gt;% 
  dplyr::select(
    Covariate = covariate, ends_with(&quot;Significant&quot;), Retained, everything()
  ) %&gt;% 
  pander(
    justify = &quot;lrrrrrr&quot;,
    caption = paste(
      &quot;*Summary of changes introduced by IHW for windows considered as being&quot;, 
      paste0(&quot;differentially bound by &quot;, target, &quot;.&quot;),
      &quot;This corresponds to a __nett change of&quot;, 
        dplyr::slice(., nrow(.)) %&gt;% 
        mutate(p = `Nett Change` / (Significant - `Nett Change`)) %&gt;% 
        .[[&quot;p&quot;]] %&gt;% 
        percent(0.1),
      &quot;from the initial list__.*&quot;
    ),
    emphasize.strong.rows = nrow(.)
  )</code></pre>
<table>
<caption><em>Summary of changes introduced by IHW for windows considered
as being differentially bound by H3K27ac. This corresponds to a
<strong>nett change of 37.3% from the initial
list</strong>.</em></caption>
<colgroup>
<col width="19%" />
<col width="20%" />
<col width="14%" />
<col width="11%" />
<col width="9%" />
<col width="9%" />
<col width="14%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Covariate</th>
<th align="right">Never Significant</th>
<th align="right">Significant</th>
<th align="right">Retained</th>
<th align="right">Gained</th>
<th align="right">Lost</th>
<th align="right">Nett Change</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">No Other Targets</td>
<td align="right">26,903</td>
<td align="right">20</td>
<td align="right">20</td>
<td align="right">0</td>
<td align="right">14</td>
<td align="right">-14</td>
</tr>
<tr class="even">
<td align="left">AR</td>
<td align="right">2,893</td>
<td align="right">151</td>
<td align="right">87</td>
<td align="right">64</td>
<td align="right">0</td>
<td align="right">64</td>
</tr>
<tr class="odd">
<td align="left">ER</td>
<td align="right">2,135</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">1</td>
<td align="right">-1</td>
</tr>
<tr class="even">
<td align="left">Other</td>
<td align="right">1,787</td>
<td align="right">72</td>
<td align="right">55</td>
<td align="right">17</td>
<td align="right">0</td>
<td align="right">17</td>
</tr>
<tr class="odd">
<td align="left"><strong>Total</strong></td>
<td align="right"><strong>33,718</strong></td>
<td align="right"><strong>243</strong></td>
<td align="right"><strong>162</strong></td>
<td align="right"><strong>81</strong></td>
<td align="right"><strong>15</strong></td>
<td align="right"><strong>66</strong></td>
</tr>
</tbody>
</table>
</div>
<div id="initial-group-sizes" class="section level3">
<h3>Initial Group Sizes</h3>
<pre class="r"><code>tibble(ihw = ihw_all) %&gt;% 
  ggplot(
    aes(ihw, fill = ihw)
  ) +
  geom_bar() +
  geom_hline(yintercept = 1e3, colour = &quot;blue&quot;) +
  geom_text(
    aes(y = n + 0.025*max(n), label = comma(n, 1)),
    data = . %&gt;%
      group_by(ihw) %&gt;%
      summarise(n = dplyr::n(), .groups = &quot;drop&quot;) 
  ) +
  scale_y_continuous(expand = expansion(c(0, 0.05)), labels = comma) +
  labs(x = &quot;Peaks Overlapping Other ChIP Targets&quot;, y = &quot;Total&quot;) +
  theme(legend.position = &quot;none&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/ihw-partitions-raw-1.png" alt="*Breakdown of all windows which overlapped peaks from additional ChIP targets. Any partitions with fewer than 1000 windows (indicated as the blue horizontal line) were combined into the next smallest partition consecutively, until all partitions contained &gt; 1000 windows.*" width="960" />
<p class="caption">
<em>Breakdown of all windows which overlapped peaks from additional ChIP
targets. Any partitions with fewer than 1000 windows (indicated as the
blue horizontal line) were combined into the next smallest partition
consecutively, until all partitions contained &gt; 1000 windows.</em>
</p>
</div>
</div>
<div id="p-value-distributions" class="section level3">
<h3>P-Value Distributions</h3>
<pre class="r"><code>y &lt;- ggplot_build(
  tibble(
    P.Value = merged_results$P.Value,
    ihw_covariate = merged_results$ihw_covariate
    ) %&gt;% 
    ggplot(aes(P.Value, stat(density))) + 
    geom_histogram(bins = 100) +
    facet_wrap(~ihw_covariate) 
) %&gt;% .[[&quot;data&quot;]] %&gt;% 
  .[[1]] %&gt;% 
  .[[&quot;y&quot;]] %&gt;% 
  max()
merged_results %&gt;% 
  select(P.Value, ihw_covariate) %&gt;% 
  mcols() %&gt;% 
  as_tibble() %&gt;% 
  ggplot(aes(P.Value, stat(density))) +
  geom_histogram(fill = &quot;grey&quot;, colour = &quot;black&quot;, bins = 100) +
  geom_text(
    aes(0.8, y, label = lab),
    data = . %&gt;% 
      group_by(ihw_covariate) %&gt;% 
      summarise(n = dplyr::n(), .groups = &quot;drop&quot;) %&gt;% 
      mutate(
        lab = glue(&quot;N = {comma(n, 1)}&quot;),
        y = y
      )
  ) +
  facet_wrap(~ihw_covariate) +
  scale_y_continuous(expand = expansion(c(0, 0.05))) +
  labs(x = &quot;P Value&quot;, y = &quot;Density&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-ihw-pvals-1.png" alt="*P-Value distributions within all final data partitions. The size of each partition is given within each panel.*" width="960" />
<p class="caption">
<em>P-Value distributions within all final data partitions. The size of
each partition is given within each panel.</em>
</p>
</div>
</div>
<div id="ihw-weights" class="section level3">
<h3>IHW Weights</h3>
<pre class="r"><code>plot(ihw) +
  geom_hline(yintercept = 1) +
  facet_wrap(~group) +
  scale_y_continuous(expand = expansion(c(0, 0.05))) +
  labs(x = &quot;Fold&quot;, y = &quot;Weight&quot;, fill = &quot;Fold&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-ihw-weights-1.png" alt="*Weights applied to p-values within each partition. 'Folds' represent random sub-partitions within each larger partition generated as part of the IHW process.*" width="960" />
<p class="caption">
<em>Weights applied to p-values within each partition. ‘Folds’ represent
random sub-partitions within each larger partition generated as part of
the IHW process.</em>
</p>
</div>
</div>
<div id="fdr-comparison" class="section level3">
<h3>FDR Comparison</h3>
<pre class="r"><code>ihw %&gt;%
  as.data.frame() %&gt;% 
  ggplot(
    aes(-log10(pvalue), -log10(weighted_pvalue), colour = fold)
  ) +
  geom_point(size = 0.4) +
  geom_hline(
    aes(yintercept = -log10(weighted_pvalue)),
    data = . %&gt;% 
      dplyr::select(-group) %&gt;% 
      dplyr::filter(adj_pvalue &lt; fdr_alpha) %&gt;% 
      dplyr::filter(weighted_pvalue == max(weighted_pvalue)),
    linetype = 2, colour = &quot;blue&quot;
  ) +
  geom_vline(
    aes(xintercept = -log10(pvalue)),
    data =. %&gt;% 
      dplyr::filter(p.adjust(pvalue, &quot;fdr&quot;) &lt; fdr_alpha) %&gt;% 
      dplyr::filter(pvalue == max(pvalue)) %&gt;% 
      dplyr::select(-group),
    linetype = 2, colour = &quot;blue&quot;
  ) +
  facet_wrap(~group, scales = &quot;free&quot;) +
  coord_cartesian(xlim = c(0, 5), ylim = c(0, 5)) +
  labs(
    x = expression(-log[10](p)),
    y = expression(-log[10](p[IHW])),
    colour = &quot;Fold&quot;
  )</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-fdr-comparison-1.png" alt="*Comparison of raw and weighted p-values for each partition. Blue dashed lines indicate FDR = 0.05 for each set of p-values. Those in the lower-right quadrant would no longer be considered significant after IHW, whilst those in the upper-left quadrant would only be considered as significant after the IHW process. Those in the upper-right quadrant would be considered as significant regardless of the methodology.*" width="960" />
<p class="caption">
<em>Comparison of raw and weighted p-values for each partition. Blue
dashed lines indicate FDR = 0.05 for each set of p-values. Those in the
lower-right quadrant would no longer be considered significant after
IHW, whilst those in the upper-left quadrant would only be considered as
significant after the IHW process. Those in the upper-right quadrant
would be considered as significant regardless of the methodology.</em>
</p>
</div>
</div>
</div>
<div id="mapping-windows" class="section level2">
<h2>Mapping Windows</h2>
<pre class="r"><code>merged_results &lt;- merged_results %&gt;% 
  mutate(
    region = bestOverlap(
      ., 
      gene_regions %&gt;% 
        lapply(select, region) %&gt;% 
        GRangesList() %&gt;% 
        unlist(),
      var = &quot;region&quot;
    ) %&gt;% 
      factor(levels = regions)
  )
feat_prom &lt;- GRanges()
feat_enh &lt;- GRanges()
if (has_features) {
  merged_results &lt;- merged_results %&gt;% 
    mutate(
      feature = bestOverlap(
        ., external_features, &quot;feature&quot;, missing = &quot;no_feature&quot;
      ) %&gt;% 
        factor(levels = names(colours$features)) %&gt;% 
        fct_relabel(str_sep_to_title)
    )
  if (&quot;feature&quot; %in% colnames(mcols(external_features))) {
    feat_prom &lt;- external_features %&gt;%
      subset(str_detect(feature, &quot;[Pp]rom&quot;)) %&gt;%
      granges()
    feat_enh &lt;- external_features %&gt;%
      subset(str_detect(feature, &quot;[Ee]nhanc&quot;)) %&gt;%
      granges()
  }
}
merged_results &lt;- merged_results %&gt;%
  mapByFeature(
    genes = gtf_gene, 
    prom = reduce(c(feat_prom, granges(gene_regions$promoter))),
    enh = feat_enh,
    gi = hic, 
    gr2gene = params$mapping$gr2gene,
    prom2gene = params$mapping$prom2gene,
    enh2gene = params$mapping$enh2gene,
    gi2gene = params$mapping$gi2gene
  )</code></pre>
<p>In addition to statistical analysis, merged windows were first mapped
to the gene-centric region with the largest overlap. Windows were then
mapped to the external features provided in the same manner, followed by
mapping to all annotated genes.</p>
<p>During mapping to genes, promoters were defined the union of all
regions defined based on transcript-level annotation, and any external
features which were defined as promoters. Enhancers were any regions
defined in enhancer_atlas_2.0_zr75.gtf.gz as enhancers.</p>
<p>These features were used to map merged windows to features using the
process defined in the function
<code>extraChIPs::mapByFeature()</code>:</p>
<ol style="list-style-type: decimal">
<li>Ranges <em>overlapping a promoter</em> are assigned to genes
directly overlapping that specific promoter</li>
<li>Ranges <em>overlapping an enhancer</em> are assigned to all genes
within 100 kb of the enhancer</li>
<li>Ranges <em>overlapping a long-range interaction</em> are assigned to
all genes directly overlapping either end of the interaction</li>
<li>Ranges with <em>no gene assignment from the previous steps</em> are
assigned to all overlapping genes, or the nearest gene within 100kb</li>
</ol>
<p>Notably, genes are only passed to step 4 if no gene assignment has
been made in steps 1, 2 or 3. For visualisation purposes, only genes
which were considered as detected in any provided RNA-Seq data will be
shown as the mapping targets.</p>
</div>
</div>
<div id="results" class="section level1">
<h1>Results</h1>
<div id="result-tables" class="section level2 tabset">
<h2 class="tabset">Result Tables</h2>
<pre class="r"><code>up_col &lt;- function(x) {
  if (is.na(x) | is.nan(x)) return(&quot;#ffffff&quot;)
  rgb(
    colorRamp(c(&quot;#ffffff&quot;, colours$direction[[&quot;up&quot;]]))(x), maxColorValue = 255
  )
}
down_col &lt;- function(x) {
  if (is.na(x) | is.nan(x)) return(&quot;#ffffff&quot;)
  rgb(
    colorRamp(c(&quot;#ffffff&quot;, colours$direction[[&quot;down&quot;]]))(x), maxColorValue = 255
  )
}
unch_col &lt;- function(x) {
  if (is.na(x) | is.nan(x)) return(&quot;#ffffff&quot;)
  rgb(
    colorRamp(c(&quot;#ffffff&quot;, colours$direction[[&quot;unchanged&quot;]]))(x), 
    maxColorValue = 255
  )
}
lfc_col &lt;- function(x){
  if (is.na(x) | is.nan(x)) return(&quot;#ffffff&quot;)
  rgb(
    colorRamp(c(colours$direction[[&quot;down&quot;]], &quot;#ffffff&quot;, colours$direction[[&quot;up&quot;]]))(x), 
    maxColorValue = 255
  )
}
expr_col &lt;- function(x){
  if (is.na(x) | is.nan(x)) return(&quot;#ffffff&quot;)
  rgb(colorRamp(hcl.colors(9, &quot;TealRose&quot;))(x), maxColorValue = 255)
}
bar_style &lt;- function(width = 1, fill = &quot;#e6e6e6&quot;, height = &quot;75%&quot;, align = c(&quot;left&quot;, &quot;right&quot;), color = NULL, fontSize = c()) {
  align &lt;- match.arg(align)
  if (align == &quot;left&quot;) {
    position &lt;- paste0(width * 100, &quot;%&quot;)
    image &lt;- sprintf(&quot;linear-gradient(90deg, %1$s %2$s, transparent %2$s)&quot;, fill, position)
  } else {
    position &lt;- paste0(100 - width * 100, &quot;%&quot;)
    image &lt;- sprintf(&quot;linear-gradient(90deg, transparent %1$s, %2$s %1$s)&quot;, position, fill)
  }
  styles &lt;- list(
    backgroundImage = image,
    backgroundSize = paste(&quot;100%&quot;, height),
    backgroundRepeat = &quot;no-repeat&quot;,
    backgroundPosition = &quot;center&quot;,
    color = color
  )
  if (!is.null(fontSize)) styles$fontSize = fontSize
  styles
}
with_tooltip &lt;- function(value, width = 30) {
  tags$span(title = value, str_trunc(value, width))
}
n_unch &lt;- sum(merged_results$status == &quot;Unchanged&quot;)
n_windows &lt;- length(merged_results)
tbl_cols &lt;- list(
  Region = colDef(
    footer = htmltools::tags$b(&quot;Total&quot;), minWidth = 200
  ),
  Feature = colDef(
    footer = htmltools::tags$b(&quot;Total&quot;), minWidth = 200
  ),
  Up = colDef(
    cell = function(value) comma(value, 1),
    style = function(value) {
      normalized &lt;- value / (n_windows - n_unch)
      color &lt;- up_col(normalized)
      list(background = color)
    },
    footer = function(values) htmltools::tags$b(comma(sum(values)))
  ),
  Down = colDef(
    cell = function(value) comma(value, 1),
    style = function(value) {
      normalized &lt;- value / (n_windows - n_unch)
      color &lt;- down_col(normalized)
      list(background = color)
    },
    footer = function(values) htmltools::tags$b(comma(sum(values)))
  ),
  Ambiguous = colDef(
    cell = function(value) comma(value, 1),
    style = function(value) {
      normalized &lt;- value / (n_windows - n_unch)
      color &lt;- unch_col(normalized)
      list(background = color)
    },
    footer = function(values) htmltools::tags$b(comma(sum(values)))
  ),
  Unchanged = colDef(
    cell = function(value) comma(value, 1),
    style = function(value) {
      normalized &lt;- value / n_windows
      color &lt;- unch_col(normalized)
      list(background = color)
    },
    footer = function(values) htmltools::tags$b(comma(sum(values)))
  ),
  `% Changed` = colDef(
    cell = function(value) percent(value, 0.1),
    style = function(value) bar_style(width = value, fill = &quot;#B3B3B3&quot;, align = &quot;right&quot;),
    align = &quot;right&quot;,
    footer = htmltools::tags$b(
      percent(sum(1 - n_unch/ n_windows), 0.1)
    )
  ),
  Total = colDef(
    cell = function(value) comma(value, 1),
    style = function(value) {
      bar_style(width = value / n_windows, fill = &quot;#B3B3B3&quot;, align = &quot;right&quot;)
    },
    align = &quot;right&quot;,
    footer = function(values) htmltools::tags$b(comma(sum(values)))
  ),
  `% Of All Windows` = colDef(
    style = function(value) bar_style(width = value, fill = &quot;#B3B3B3&quot;, align = &quot;right&quot;),
    format = colFormat(digits = 2, percent = TRUE)
  )
)</code></pre>
<div id="overall-results" class="section level3">
<h3>Overall Results</h3>
<pre class="r"><code>tbl &lt;- merged_results %&gt;% 
  mutate(w = width) %&gt;% 
  as_tibble() %&gt;% 
  group_by(status) %&gt;% 
  summarise(
    n = dplyr::n(), 
    width = median(w),
    AveExpr = median(AveExpr),
    .groups = &quot;drop&quot;
  ) %&gt;% 
  mutate(`%` = n / sum(n)) %&gt;% 
  dplyr::select(status, n, `%`, everything()) %&gt;% 
  reactable(
    searchable = FALSE, filterable = FALSE,
    columns = list(
      status = colDef(
        name = &quot;Status&quot;, maxWidth = 150,
        footer = htmltools::tags$b(&quot;Overall&quot;)
      ),
      n = colDef(
        name = &quot;Nbr of Windows&quot;, maxWidth = 150, format = colFormat(separators = TRUE),
        footer = htmltools::tags$b(comma(length(merged_results)))
      ),
      width = colDef(
        name = &quot;Median Width (bp)&quot;, format = colFormat(digits = 1),
        footer = htmltools::tags$b(round(median(width(merged_results)), 1)),
        maxWidth = 200
      ),
      `%` = colDef(
        name = &quot;% Total Windows&quot;, format = colFormat(percent = TRUE, digits = 1),
        style = function(value) bar_style(width = value, align = &quot;right&quot;),
        maxWidth = 150
      ),
      AveExpr = colDef(
        name = &quot;Median Signal (logCPM)&quot;,
        format = colFormat(digits = 3),
        maxWidth = 200,
        footer = htmltools::tags$b(
          round(median(merged_results$AveExpr), 3)
        )
      )
    )
  )
cp &lt;- htmltools::em(
  glue(
    &quot;Overall results for changed {target} binding in the &quot;,
    glue_collapse(rev(treat_levels), last = &quot; Vs. &quot;),
    &quot; comparison.&quot;
  )
)
div(class = &quot;table&quot;,
  div(class = &quot;table-header&quot;,
      div(class = &quot;caption&quot;, cp),
      tbl
  )
)</code></pre>
<div class="table">
<div class="table-header">
<div class="caption">
<em>Overall results for changed H3K27ac binding in the E2DHT Vs. E2 comparison.</em>
</div>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-f5c7bde175ea0352f38f" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-f5c7bde175ea0352f38f">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"status":["Up","Down","Unchanged"],"n":[241,2,33733],"%":[0.00709324228867436,5.88650812338121e-05,0.992847892630092],"width":[600,980,1040],"AveExpr":[0.917714423326314,0.759999066666796,1.59172102838499]},"columns":[{"id":"status","name":"Status","type":"factor","footer":{"name":"b","attribs":{},"children":["Overall"]},"maxWidth":150},{"id":"n","name":"Nbr of Windows","type":"numeric","format":{"cell":{"separators":true},"aggregated":{"separators":true}},"footer":{"name":"b","attribs":{},"children":["33,976"]},"maxWidth":150},{"id":"%","name":"% Total Windows","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":150,"style":[{"backgroundImage":"linear-gradient(90deg, transparent 99.2906757711326%, #e6e6e6 99.2906757711326%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 99.9941134918766%, #e6e6e6 99.9941134918766%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 0.715210736990812%, #e6e6e6 0.715210736990812%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null}]},{"id":"width","name":"Median Width (bp)","type":"numeric","format":{"cell":{"digits":1},"aggregated":{"digits":1}},"footer":{"name":"b","attribs":{},"children":["1040"]},"maxWidth":200},{"id":"AveExpr","name":"Median Signal (logCPM)","type":"numeric","format":{"cell":{"digits":3},"aggregated":{"digits":3}},"footer":{"name":"b","attribs":{},"children":["1.582"]},"maxWidth":200}],"dataKey":"1fd6ed80b1d82f4717a9b89373283cef"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
<div id="summary-by-region" class="section level3">
<h3>Summary By Region</h3>
<pre class="r"><code>df &lt;- merged_results %&gt;% 
  select(
    status, all_of(fdr_column), region
  ) %&gt;% 
  as_tibble() %&gt;% 
  group_by(region, status) %&gt;% 
  summarise(n = dplyr::n(), .groups = &quot;drop&quot;) %&gt;% 
  complete(region, status, fill = list(n = 0)) %&gt;% 
  group_by(region) %&gt;% 
  mutate(Total = sum(n)) %&gt;% 
  ungroup() %&gt;% 
  pivot_wider(names_from = &quot;status&quot;, values_from = &quot;n&quot;) %&gt;% 
  mutate(
    `% Of All Windows` = Total / length(merged_results),
    `% Changed` = 1 - Unchanged / Total
  ) %&gt;% 
  arrange(region) %&gt;% 
  dplyr::select(
    Region = region,
    any_of(names(direction_colours)),
    `% Changed`, Total, `% Of All Windows`
  )
cp &lt;- htmltools::em(
  glue(
    &quot;Overall results for changed {target} binding in the &quot;,
    glue_collapse(rev(treat_levels), last = &quot; Vs. &quot;),
    &quot; comparison, broken down by which genomic region contains the largest &quot;, 
    &quot;overlap with each merged window.&quot;
  )
)
tbl &lt;- df %&gt;%
  reactable(
    columns = tbl_cols[colnames(df)],
    defaultColDef = colDef(
      footer = function(values) htmltools::tags$b(comma(sum(values)))
    ),
    fullWidth = TRUE
  )
div(class = &quot;table&quot;,
  div(class = &quot;table-header&quot;,
      div(class = &quot;caption&quot;, cp),
      tbl
  )
)</code></pre>
<div class="table">
<div class="table-header">
<div class="caption">
<em>Overall results for changed H3K27ac binding in the E2DHT Vs. E2 comparison, broken down by which genomic region contains the largest overlap with each merged window.</em>
</div>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-fa261e95ba1a11c94ed5" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-fa261e95ba1a11c94ed5">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"Region":["Promoter (-1500/+500)","Upstream Promoter (-5kb)","Exon","Intron","Intergenic (<10kb)","Intergenic (>10kb)"],"Up":[22,27,4,133,26,29],"Down":[0,0,0,1,0,1],"Unchanged":[13409,3119,624,10283,2258,4040],"% Changed":[0.00163800163800165,0.00858232676414494,0.00636942675159236,0.0128635883651723,0.01138353765324,0.00737100737100738],"Total":[13431,3146,628,10417,2284,4070],"% Of All Windows":[0.395308453025665,0.0925947727807864,0.018483635507417,0.30659877560631,0.0672239227690134,0.119790440310808]},"columns":[{"id":"Region","name":"Region","type":"factor","footer":{"name":"b","attribs":{},"children":["Total"]},"minWidth":200},{"id":"Up","name":"Up","type":"numeric","footer":{"name":"b","attribs":{},"children":["241"]},"cell":["22","27","4","133","26","29"],"style":[{"background":"#FAE7EA"},{"background":"#F9E2E6"},{"background":"#FEFAFB"},{"background":"#E17384"},{"background":"#F9E3E7"},{"background":"#F8E0E4"}]},{"id":"Down","name":"Down","type":"numeric","footer":{"name":"b","attribs":{},"children":["2"]},"cell":["0","0","0","1","0","1"],"style":[{"background":"#FFFFFF"},{"background":"#FFFFFF"},{"background":"#FFFFFF"},{"background":"#FDFEFE"},{"background":"#FFFFFF"},{"background":"#FDFEFE"}]},{"id":"Unchanged","name":"Unchanged","type":"numeric","footer":{"name":"b","attribs":{},"children":["33,733"]},"cell":["13,409","3,119","624","10,283","2,258","4,040"],"style":[{"background":"#CCCCCC"},{"background":"#F3F3F3"},{"background":"#FCFCFC"},{"background":"#D8D8D8"},{"background":"#F6F6F6"},{"background":"#EFEFEF"}]},{"id":"% Changed","name":"% Changed","type":"numeric","footer":{"name":"b","attribs":{},"children":["0.7%"]},"cell":["0.2%","0.9%","0.6%","1.3%","1.1%","0.7%"],"align":"right","style":[{"backgroundImage":"linear-gradient(90deg, transparent 99.8361998361998%, #B3B3B3 99.8361998361998%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 99.1417673235855%, #B3B3B3 99.1417673235855%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 99.3630573248408%, #B3B3B3 99.3630573248408%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 98.7136411634828%, #B3B3B3 98.7136411634828%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 98.861646234676%, #B3B3B3 98.861646234676%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 99.2628992628993%, #B3B3B3 99.2628992628993%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null}]},{"id":"Total","name":"Total","type":"numeric","footer":{"name":"b","attribs":{},"children":["33,976"]},"cell":["13,431","3,146","628","10,417","2,284","4,070"],"align":"right","style":[{"backgroundImage":"linear-gradient(90deg, transparent 60.4691546974335%, #B3B3B3 60.4691546974335%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 90.7405227219214%, #B3B3B3 90.7405227219214%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 98.1516364492583%, #B3B3B3 98.1516364492583%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 69.340122439369%, #B3B3B3 69.340122439369%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 93.2776077230987%, #B3B3B3 93.2776077230987%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 88.0209559689192%, #B3B3B3 88.0209559689192%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null}]},{"id":"% Of All Windows","name":"% Of All Windows","type":"numeric","footer":{"name":"b","attribs":{},"children":["1"]},"format":{"cell":{"digits":2,"percent":true},"aggregated":{"digits":2,"percent":true}},"style":[{"backgroundImage":"linear-gradient(90deg, transparent 60.4691546974335%, #B3B3B3 60.4691546974335%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 90.7405227219214%, #B3B3B3 90.7405227219214%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 98.1516364492583%, #B3B3B3 98.1516364492583%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 69.340122439369%, #B3B3B3 69.340122439369%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 93.2776077230987%, #B3B3B3 93.2776077230987%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 88.0209559689192%, #B3B3B3 88.0209559689192%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null}]}],"dataKey":"cf4ffb5162e01aa71ab36abb2fb133b7"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
<div id="summary-by-feature" class="section level3">
<h3>Summary By Feature</h3>
<pre class="r"><code>df &lt;- merged_results %&gt;% 
  select(status, all_of(fdr_column), feature) %&gt;% 
  as_tibble() %&gt;% 
  group_by(feature, status) %&gt;% 
  tally() %&gt;% 
  ungroup() %&gt;% 
  complete(feature, status, fill = list(n = 0)) %&gt;% 
  group_by(feature) %&gt;% 
  mutate(Total = sum(n)) %&gt;% 
  ungroup() %&gt;% 
  pivot_wider(names_from = &quot;status&quot;, values_from = &quot;n&quot;) %&gt;% 
  mutate(
    `% Of All Windows` = Total / length(merged_results),
    `% Changed` = 1 - Unchanged / Total
  ) %&gt;% 
  arrange(feature) %&gt;% 
  dplyr::select(
    Feature = feature,
    any_of(names(direction_colours)),
    `% Changed`, Total, `% Of All Windows`
  )
cp &lt;- htmltools::em(
  glue(
    &quot;Overall results for changed {target} binding in the &quot;,
    glue_collapse(rev(treat_levels), last = &quot; Vs. &quot;),
    &quot; comparison, broken down by which external feature contains the largest &quot;, 
    &quot;overlap with each merged window.&quot;
  )
)
tbl &lt;- df %&gt;%
  reactable(
    columns = tbl_cols[colnames(df)],
    defaultColDef = colDef(
      footer = function(values) htmltools::tags$b(comma(sum(values)))
    )
  )
div(class = &quot;table&quot;,
  div(class = &quot;table-header&quot;,
      div(class = &quot;caption&quot;, cp),
      tbl
  )
)</code></pre>
<div class="table">
<div class="table-header">
<div class="caption">
<em>Overall results for changed H3K27ac binding in the E2DHT Vs. E2 comparison, broken down by which external feature contains the largest overlap with each merged window.</em>
</div>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-ace7d63121c39b06110f" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-ace7d63121c39b06110f">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"Feature":["Enhancer","No Feature"],"Up":[128,113],"Down":[2,0],"Unchanged":[17500,16233],"% Changed":[0.00737379466817922,0.00691300624005875],"Total":[17630,16346],"% Of All Windows":[0.518895691076054,0.481104308923946]},"columns":[{"id":"Feature","name":"Feature","type":"factor","footer":{"name":"b","attribs":{},"children":["Total"]},"minWidth":200},{"id":"Up","name":"Up","type":"numeric","footer":{"name":"b","attribs":{},"children":["241"]},"cell":["128","113"],"style":[{"background":"#E37889"},{"background":"#E68897"}]},{"id":"Down","name":"Down","type":"numeric","footer":{"name":"b","attribs":{},"children":["2"]},"cell":["2","0"],"style":[{"background":"#FCFDFE"},{"background":"#FFFFFF"}]},{"id":"Unchanged","name":"Unchanged","type":"numeric","footer":{"name":"b","attribs":{},"children":["33,733"]},"cell":["17,500","16,233"],"style":[{"background":"#BDBDBD"},{"background":"#C1C1C1"}]},{"id":"% Changed","name":"% Changed","type":"numeric","footer":{"name":"b","attribs":{},"children":["0.7%"]},"cell":["0.7%","0.7%"],"align":"right","style":[{"backgroundImage":"linear-gradient(90deg, transparent 99.2626205331821%, #B3B3B3 99.2626205331821%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 99.3086993759941%, #B3B3B3 99.3086993759941%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null}]},{"id":"Total","name":"Total","type":"numeric","footer":{"name":"b","attribs":{},"children":["33,976"]},"cell":["17,630","16,346"],"align":"right","style":[{"backgroundImage":"linear-gradient(90deg, transparent 48.1104308923946%, #B3B3B3 48.1104308923946%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 51.8895691076054%, #B3B3B3 51.8895691076054%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null}]},{"id":"% Of All Windows","name":"% Of All Windows","type":"numeric","footer":{"name":"b","attribs":{},"children":["1"]},"format":{"cell":{"digits":2,"percent":true},"aggregated":{"digits":2,"percent":true}},"style":[{"backgroundImage":"linear-gradient(90deg, transparent 48.1104308923946%, #B3B3B3 48.1104308923946%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null},{"backgroundImage":"linear-gradient(90deg, transparent 51.8895691076054%, #B3B3B3 51.8895691076054%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null}]}],"dataKey":"4890dcf21122d863a503f7d829dc21bc"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
<div id="most-highly-ranked" class="section level3">
<h3>Most Highly Ranked</h3>
<pre class="r"><code>show_n &lt;- min(200, length(merged_results))
scaling_vals &lt;- list(
  logFC = c(-1, 1)*max(abs(merged_results$logFC)),
  AveExpr = range(merged_results$AveExpr),
  Width = max(width(merged_results))
)
cp &lt;- htmltools::em(
  glue(
    &quot;The {show_n} most highly-ranked windows by FDR, with &quot;,
    sum(merged_results$P.Value_fdr &lt; fdr_alpha), 
    &quot; showing evidence of changed {target} binding. &quot;,
    &quot;Regions were assigned based on which genomic region showed the largest &quot;,
    &quot;overlap with the final merged window. &quot;,
    ifelse(
      has_features, 
      glue(&quot;Features are as provided in the file {basename(config$external$features)}. &quot;),
      glue(&quot;&quot;)
    ),
    &quot;For windows mapped to large numbers of genes, hovering a mouse over the &quot;, 
    &quot;cell will reveal the full set of genes. &quot;,
    ifelse(
      has_rnaseq,
      &quot;Only genes considered as detected in the RNA-Seq data are shown. &quot;,
      &quot;&quot;
    ),
    &quot;The Macs2 Peak column is filterable using T/F values&quot;
  )
)
fs &lt;- 12
tbl &lt;- merged_results %&gt;%
  mutate(w = width) %&gt;% 
  arrange(!!sym(fdr_column)) %&gt;% 
  plyranges::slice(seq_len(show_n)) %&gt;% 
  select(
    w, AveExpr, logFC, FDR = !!sym(fdr_column), 
    overlaps_ref, Region = region, any_of(&quot;feature&quot;), Genes = gene_name
  ) %&gt;% 
  as_tibble() %&gt;% 
  dplyr::rename(
    Range = range, `Width (bp)` = w, `Macs2 Peak` = overlaps_ref
  ) %&gt;% 
  rename_all(str_replace_all, &quot;feature&quot;, &quot;Feature&quot;) %&gt;% 
  mutate(Genes = vapply(Genes, paste, character(1), collapse = &quot;; &quot;)) %&gt;% 
  reactable(
    filterable = TRUE,
    showPageSizeOptions = TRUE,
    pageSizeOptions = c(10, 20, 50, show_n), defaultPageSize = 10,
    borderless  = TRUE,
    columns = list(
      Range = colDef(
        minWidth = 10 * fs,
        cell = function(value) {
          str_replace_all(value, &quot;:&quot;, &quot;: &quot;)
        }
      ),
      `Width (bp)` = colDef(
        style = function(value) {
          x &lt;- value / scaling_vals$Width
          colour &lt;- expr_col(x)
          list(
            background = colour, 
            borderRight = &quot;1px solid rgba(0, 0, 0, 0.1)&quot;
          )
        },
        maxWidth = 5 * fs
      ),
      AveExpr = colDef(
        cell = function(value) round(value, 2),
        style = function(value) {
          x &lt;- (value - min(scaling_vals$AveExpr)) / diff(scaling_vals$AveExpr)
          colour = expr_col(x)
          list(background = colour)
        },
        maxWidth = 5.5 * fs
      ),
      logFC = colDef(
        cell = function(value) round(value, 2),
        style = function(value) {
          x &lt;- (value - min(scaling_vals$logFC)) / diff(scaling_vals$logFC)
          colour &lt;- lfc_col(x)
          list(background = colour)
        },
        maxWidth = 5.5 * fs
      ),
      FDR = colDef(
        cell = function(value) sprintf(&quot;%.2e&quot;, value),
        style = list(borderRight = &quot;1px solid rgba(0, 0, 0, 0.1)&quot;),
        maxWidth = 5.5 * fs
      ),
      `Macs2 Peak` = colDef(
        cell = function(value) ifelse(value, &quot;\u2714 Yes&quot;, &quot;\u2716 No&quot;),
        style = function(value) {
          color &lt;- ifelse(value, &quot;#008000&quot;, &quot;#e00000&quot;)
          list(color = color, borderRight = &quot;1px solid rgba(0, 0, 0, 0.1)&quot;)
        },
        maxWidth = 5 * fs
      ),
      Region = colDef(maxWidth = 150),
      Genes = colDef(
        cell = function(value) with_tooltip(value),
        minWidth = 11 * fs
      )
    ),
    theme = reactableTheme(style = list(fontSize = fs))
  )
div(class = &quot;table&quot;,
  div(class = &quot;table-header&quot;,
      div(class = &quot;caption&quot;, cp),
      tbl
  )
)</code></pre>
<div class="table">
<div class="table-header">
<div class="caption">
<em>The 200 most highly-ranked windows by FDR, with 177 showing evidence of changed H3K27ac binding. Regions were assigned based on which genomic region showed the largest overlap with the final merged window. Features are as provided in the file enhancer_atlas_2.0_zr75.gtf.gz. For windows mapped to large numbers of genes, hovering a mouse over the cell will reveal the full set of genes. Only genes considered as detected in the RNA-Seq data are shown. The Macs2 Peak column is filterable using T/F values</em>
</div>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-3aebcbe31c8b449fd9cd" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-3aebcbe31c8b449fd9cd">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"Range":["chr17:78041721-78043240","chr17:71856921-71857720","chr1:146580841-146582200","chr17:35630121-35634400","chr19:3429841-3431000","chr11:34692801-34693720","chr1:184601281-184603080","chr18:60167801-60168320","chr17:56008201-56009880","chr12:64983521-64984000","chr10:79257401-79258680","chr8:133910801-133911240","chr5:148980801-148981200","chr1:179840601-179841960","chr4:24805681-24807640","chr14:97126721-97128760","chr8:50996801-50998000","chr17:35678601-35679080","chr5:36285601-36286000","chr19:4989881-4993440","chr4:70261521-70261960","chr4:8353881-8354600","chr12:102009521-102009920","chr9:33430521-33432480","chr19:41360601-41362760","chr19:46246201-46247280","chr17:71856241-71856720","chr6:6748681-6749240","chr18:55952481-55953040","chr1:163586921-163587360","chr22:24364321-24364480","chr18:12589121-12589560","chr1:109370441-109371080","chr18:20066161-20066560","chr1:36705801-36706080","chr9:33436761-33437720","chr1:19752521-19753720","chr16:16135561-16135680","chr5:148980281-148980520","chr12:46640041-46640560","chr15:86080681-86081360","chr5:71009841-71010920","chr12:15427161-15427560","chr17:35675241-35675360","chr9:98045761-98046360","chrX:44751281-44751960","chr10:43689241-43690120","chr17:35672201-35672480","chr10:102180001-102180280","chr10:74008161-74008720","chr17:47470801-47473080","chr17:71212001-71212880","chr4:41079321-41079640","chr6:3669761-3670240","chr18:60168481-60168920","chr5:36286321-36286680","chr5:172437721-172438720","chr17:47466041-47470360","chr4:41080161-41080920","chr16:30626521-30627200","chr6:35694801-35695960","chr22:30804321-30804760","chr13:77881041-77881600","chr18:56216881-56217040","chr8:110592641-110593400","chr20:36474201-36475720","chr6:34565441-34567480","chr5:71048921-71049440","chr2:62651721-62651880","chr9:4866601-4867240","chr6:35696041-35696400","chr10:81101681-81102040","chr12:52519161-52520880","chr15:45329681-45330040","chr16:34751281-34751600","chr1:160126441-160127120","chr4:70073801-70073960","chr1:47108201-47110120","chr1:57254081-57254720","chr4:8353121-8353520","chr16:1681001-1681880","chr10:76706361-76709080","chr16:1680121-1680920","chr1:118283001-118283320","chr5:172438841-172439440","chr5:32093721-32094160","chr8:68723241-68723600","chr4:6958121-6960000","chr1:184599401-184599680","chr3:42585721-42587320","chr20:32887401-32887960","chr3:58135441-58136000","chr12:117074121-117074840","chr5:32094601-32095200","chr13:77880361-77880760","chr5:148993321-148993720","chr15:38508601-38508960","chr2:67742641-67743360","chr1:167880761-167881080","chrX:24109121-24109800","chr1:160127361-160127760","chr17:62300401-62300560","chr7:113656641-113656880","chr10:43690481-43690840","chr20:36618321-36618720","chr2:220711481-220712160","chr7:95178921-95179600","chr12:6375561-6376440","chr3:181019521-181020000","chr3:39172241-39173080","chr13:108395201-108395960","chr1:242309921-242310680","chr4:106831281-106831720","chr21:16958721-16959200","chr9:110803441-110803640","chr7:148082041-148082560","chr6:34571041-34571440","chr1:242310801-242311560","chr18:8703721-8706880","chr17:29381601-29381840","chr11:34771481-34771880","chr7:134537041-134537600","chr15:76155401-76156280","chr5:148398281-148398640","chr4:184182001-184182400","chr6:148579321-148580120","chr6:33683841-33685120","chr13:42354521-42356000","chr10:33529921-33530440","chr18:56220401-56220560","chr6:53333121-53333960","chr1:60234961-60235280","chr16:67492761-67497800","chr16:34757841-34758360","chr12:6129681-6130200","chr17:56009961-56010080","chr10:101098961-101099600","chr7:134537721-134538280","chr9:131869041-131869600","chr14:24305321-24306320","chr14:21242961-21243400","chr7:36824521-36825400","chr6:6749481-6750720","chr13:45428001-45428840","chr15:45977001-45977640","chr20:49459121-49460000","chr13:86298321-86299200","chr5:71011001-71012000","chr12:102010281-102010560","chr22:30788401-30789040","chr9:111471241-111473840","chr5:35206081-35206520","chrX:40376681-40378880","chr1:19785841-19786320","chr21:30014361-30015040","chr17:69906641-69907160","chr7:105856761-105857880","chr1:146585001-146587360","chr11:83121521-83121960","chr1:163595881-163596480","chr5:302441-303400","chr3:177803081-177803680","chr9:4865721-4866240","chr9:123391201-123391880","chr4:151644361-151645080","chr13:77665601-77665800","chrX:13082881-13083800","chr8:11143801-11144280","chr11:113211161-113213760","chr11:48063041-48063560","chr9:123294081-123295880","chr2:65134281-65135160","chr4:70152081-70153240","chr18:9717961-9718360","chr1:10575441-10578000","chr8:67442481-67446000","chr10:117903121-117903520","chr3:72347281-72347480","chr5:149375441-149377160","chr1:19784841-19785480","chr3:45588161-45589880","chr8:80989321-80989560","chr17:71211201-71211840","chrX:24108241-24108960","chr1:162993481-162993840","chr11:83122161-83122600","chr7:98622081-98622240","chr19:49085721-49086400","chr20:61837881-61839880","chr2:191100481-191101000","chr10:101098401-101098840","chr3:77741601-77742280","chr12:12727481-12730200","chr18:55953241-55953400","chr20:49460121-49460600","chr4:68577201-68579200","chr10:76546281-76546680","chr15:83586961-83589360","chr10:76780521-76782800","chr1:110266241-110271680"],"Width (bp)":[1520,800,1360,4280,1160,920,1800,520,1680,480,1280,440,400,1360,1960,2040,1200,480,400,3560,440,720,400,1960,2160,1080,480,560,560,440,160,440,640,400,280,960,1200,120,240,520,680,1080,400,120,600,680,880,280,280,560,2280,880,320,480,440,360,1000,4320,760,680,1160,440,560,160,760,1520,2040,520,160,640,360,360,1720,360,320,680,160,1920,640,400,880,2720,800,320,600,440,360,1880,280,1600,560,560,720,600,400,400,360,720,320,680,400,160,240,360,400,680,680,880,480,840,760,760,440,480,200,520,400,760,3160,240,400,560,880,360,400,800,1280,1480,520,160,840,320,5040,520,520,120,640,560,560,1000,440,880,1240,840,640,880,880,1000,280,640,2600,440,2200,480,680,520,1120,2360,440,600,960,600,520,680,720,200,920,480,2600,520,1800,880,1160,400,2560,3520,400,200,1720,640,1720,240,640,720,360,440,160,680,2000,520,440,680,2720,160,480,2000,400,2400,2280,5440],"AveExpr":[0.823135174130934,1.44248318471411,1.17093701507858,1.82057252464914,1.61733919189812,1.1544715278787,2.51434549301481,0.81292276196005,1.90417891768876,0.571324524932743,1.31096777488741,0.778202549223532,0.502332852754016,0.843566336628882,2.30897541736488,1.92467395094588,1.28927150369856,0.422831032300052,0.606095042162411,1.90520962787039,0.569084412944317,1.1199727989944,0.650647595168367,1.45153346765225,1.874677825182,1.83310426168671,0.594512491402729,0.954594413513217,0.905340544706101,0.998474076802713,0.174166745819192,0.703629625467286,0.968952687860535,0.651588696086806,0.375976394262909,0.700892501206581,1.71101484808271,0.0465499301689779,0.374180175121845,0.658455665160191,0.674078589941087,2.14401848918671,0.42756756423455,0.0548117507852947,0.590093293140949,1.02240722510088,0.854682212405129,0.204343274010728,0.268100204799448,0.764442555665521,2.99854497296139,0.792319458000859,0.459135868826809,0.740886959466433,0.622822277343426,0.455728468676655,1.52819795473849,3.90228235955822,0.437177093202054,1.00749235402435,1.42555669042997,0.473100455887273,0.667453093583136,0.289369748761341,1.66898335721257,2.04914335781177,1.61124945474779,0.742599990432829,0.27932264796948,1.13129277171903,0.582034993436207,0.54876269927677,1.32534671649558,0.475128430813545,0.541231182623955,1.54796462291426,0.1838529220019,2.01385442264215,1.04322617801838,0.608146877456,1.36138765757461,1.55061499887487,1.28856509650746,0.584658363153806,1.22163099578932,0.664960803924057,0.628487223091819,1.58857759827964,0.334126186685148,1.43903058523889,0.72929765799818,0.841114164961245,0.92484060892815,1.04515982382699,0.722107178055049,0.625458963782821,0.548135853333627,1.5435349721503,0.478449944538064,1.23442045071152,0.577016575038144,0.26193522313901,0.411469333569187,0.554524768562601,0.656977319100648,0.897853224278131,1.34363706136795,0.940232399263435,0.704955804878327,1.23196903414009,1.05998183458138,1.19705158469768,0.433754833722145,0.680359483727799,0.38755132951114,1.13504834375427,0.511731411268027,1.66625724149294,3.24856496367446,0.35127856476209,0.636297082171202,1.15034602382142,0.868255881049371,0.539302001382851,0.517989810855932,1.63299974811303,0.917714423326314,1.23984526109637,0.724846238163832,0.36356674212542,0.889872740293576,0.501144157891233,2.9091984775359,1.13361179432573,0.896350047339165,0.248522220135686,1.61101255031104,0.955511624353102,1.0464270444066,2.08868320767969,0.724149426867626,1.32755903204115,1.2565820948974,1.07321179715517,0.790020706052951,1.57986888031506,1.24694693841024,1.80504482919831,0.578150956804093,0.823579108706249,2.47759348633995,0.675256115946692,2.11549467260317,0.915975929925236,1.45006229961664,0.724242307504709,1.4985871870446,2.20998538211679,0.686250014113044,1.29640787779495,0.795656555231136,0.82760242225212,0.46506835315539,0.898458954008414,0.938022927631903,0.309723648834611,1.80624326792475,0.836260910340728,2.09873646117241,1.00548622112433,1.60434998830009,2.14138554902369,2.32200753567939,0.477835008456009,1.29441765669018,2.12849075312198,0.80115842656914,0.384375838623072,1.47448928425916,0.976382689468172,1.75823002500586,0.326601668652171,1.1571239239671,1.44062520673913,0.474379307664294,0.874920427176112,0.289493911227668,1.07495866451101,1.26206590806004,0.797976573089147,0.728099468099842,1.18635367618861,2.19126093256669,0.34845978375078,0.84990209357016,1.44288932695677,0.669484002014837,2.6059805154263,1.97858954716395,2.27125056158861],"logFC":[2.75729326505891,2.13718536559281,2.14555014485682,1.64828176891789,1.69324685097006,1.93516864679129,1.26865752549512,2.05809469881306,1.45000770881694,2.09199885062349,1.61142437834713,1.87511896720541,2.04276815271224,1.86821797893718,1.18665561390145,1.32384615499571,1.54765177530724,1.95668498139714,1.8235971809009,1.25796927065605,1.8147535150362,1.52256386757495,1.74425817017536,1.39823281200676,1.21027774619659,1.2289518980116,1.7020852729384,1.50541084396863,1.5273570890103,1.51663306832412,2.15630905741589,1.58395867746142,1.45896404829868,1.57657098010898,1.74543072215221,1.60226003520684,1.20290095073066,2.25005131316281,1.69771015100169,1.57056646354248,1.52615885769248,1.09800067106466,1.66498271937694,2.18929928904854,1.55767924970704,1.36033514627922,1.41395464133231,1.98574605825611,1.71581379112586,1.44533933625285,0.802006236334251,1.46712862216361,1.57997239158129,1.46409182331984,1.48198301488245,1.55500414855242,1.13579538916921,0.597295829874359,1.52377767443966,1.27572496936763,1.14537306549312,1.53190859973304,1.47635921597279,1.56761366920234,1.07688442840706,0.96297016230663,1.08728173350024,1.34767946997121,1.82631903026331,1.2192504661929,1.36162510777024,1.41533333732303,1.12608559575753,1.43387523100057,1.4002703040174,1.06963752951369,1.90173246056934,0.953489120383333,1.20441433603751,1.369737311328,1.16231194661696,1.02785309582856,1.14010409142336,1.34513407387831,1.10183154904376,1.56229508591474,1.32327972132764,1.03015659401469,1.71949389386798,1.12151277955721,1.27665409263716,1.2415054663932,1.201724884767,1.18339335991182,1.26423918104631,1.29812999180203,1.33160935841154,1.0128336277178,1.34339817289465,1.08882510746748,1.28540851723912,1.70910647762495,1.33271440684364,1.27689765893977,1.23690015214619,1.16339776932895,1.10426886208829,1.12052492022596,1.21110995247776,1.09633499038367,1.1280441041415,1.0392670720821,1.60315945795304,1.22427618406508,1.35896923756371,1.08114896327545,1.25370674977906,0.914029325983439,0.613170415127525,1.31226984881106,1.26125807378763,1.05145534233106,1.11136139150478,1.20420335538512,1.22425014751139,0.936547832997297,1.09842128989123,1.00859479661808,1.1352933329328,1.26737620776935,1.12242764080228,1.4971109875444,0.676535786523487,1.03036739407385,1.07176757198885,1.60536722604791,0.895114890177461,1.05616676292035,1.03187896177667,0.803771063806937,1.12680811705493,0.953083681607996,0.964764665941108,1.02797866007613,1.20048254710176,0.91825638961042,1.004569301444,0.900780149666252,1.13481226705767,1.13814350924045,0.715500915090672,1.13060325891257,0.80856213704093,1.01733827071252,0.926188179859507,1.08243258614989,0.891235826015923,0.755899581176508,1.11515021615604,0.980479624777832,1.0693465354121,1.04450806418082,1.21027814442949,1.02164424155564,1.00777384463937,1.54626442446481,0.836124338046209,1.04139016882497,0.797447357593344,1.0081356153294,0.851314823856472,0.776157038563727,0.715035850502635,1.13495378502265,0.918863317989688,0.746647577932247,1.07555498138983,1.14696173067956,0.862240235870201,1.22863825117617,0.794666229653028,1.526734575761,0.924658231316843,0.896910915030256,-1.42552156335236,1.02764931184133,1.49765214717938,0.933066499297104,0.89243324089729,1.02955466729311,1.02760910725976,0.922525047392347,0.709541239334626,1.13704556653601,1.019358300542,0.884629782968442,1.03878356970807,0.672726776333942,0.754077690829082,0.726803777452022],"FDR":[4.85755388129347e-16,2.02654817500827e-14,2.50102042578393e-12,4.12795957522259e-11,2.01642967519281e-10,2.25161902535312e-10,5.70379922815615e-10,1.31948536442818e-09,1.53067186025821e-09,5.81808575838974e-09,2.05465491455419e-08,3.98893391755404e-08,4.19432891885972e-08,4.19698734624799e-08,6.35089884237397e-08,1.1739593097372e-07,1.85395680565459e-07,5.35344323703351e-07,6.98247627437307e-07,8.05009311171575e-07,8.64785569988651e-07,1.08014458456214e-06,1.9609997680847e-06,2.11189490548428e-06,3.51068137078113e-06,3.93747865056371e-06,3.96157815703993e-06,6.27083206583414e-06,7.08086172905856e-06,7.55295629779229e-06,8.69713411465234e-06,1.02274152345122e-05,1.12323859779982e-05,1.51774129263078e-05,1.91315184620398e-05,1.91315184620398e-05,2.38869495520327e-05,2.38869495520327e-05,2.5757516834627e-05,2.5757516834627e-05,3.05034186792599e-05,3.08393846852528e-05,3.08393846852528e-05,3.08393846852528e-05,3.18415802135768e-05,4.38857473300691e-05,5.99396705466733e-05,7.28512602520537e-05,7.62689234067973e-05,7.71247433559577e-05,7.71247433559577e-05,7.71247433559577e-05,8.062792970994e-05,8.062792970994e-05,8.062792970994e-05,0.000112202275139175,0.000123521713680966,0.000153627201703339,0.000167065727649047,0.000204911333118627,0.000214859478744076,0.000214859478744076,0.000244053592942696,0.000265455710510812,0.000280962029208953,0.000280962029208953,0.000335873070265358,0.000385485217417995,0.000487098073304221,0.000487098073304221,0.000495328944987452,0.00051013912823401,0.00051013912823401,0.00051013912823401,0.00051013912823401,0.00052962141740747,0.00052962141740747,0.000565318083443912,0.00068263109039356,0.000691812365872114,0.000729289345182808,0.000778737476331179,0.000815904682031278,0.00081733548854326,0.000832190597589416,0.000833545258752377,0.000833545258752377,0.000842263504192747,0.000847317085179684,0.000847317085179684,0.000847317085179684,0.000955897814222664,0.000966169156016758,0.000978487281587181,0.00102368633442086,0.00109677366245181,0.00113192039037051,0.0012283510215545,0.00124049965725349,0.00159675815801113,0.00190815695291947,0.00194499901200677,0.00198599843963446,0.00210146352085732,0.0021061244745888,0.00233267133008141,0.00233267133008141,0.00233267133008141,0.00233691846824934,0.00248498868884494,0.00250153035097324,0.00254640124146152,0.00261108373834047,0.0030151975123844,0.00312037657241545,0.00360889610423946,0.00363801337905701,0.00418070049622485,0.00418070049622485,0.00429284642229736,0.00443021920402166,0.00451111465397806,0.00455181513947276,0.00462759434312061,0.00470238589765484,0.00482621326512358,0.00538388183659583,0.00538388183659583,0.00552205441596043,0.00566731137630516,0.00578446288442745,0.00621012373063191,0.00621012373063191,0.00665196764114169,0.0067368015163302,0.0067368015163302,0.00684946891313792,0.00689573441540009,0.00689573441540009,0.00689573441540009,0.00740283814827074,0.00770594049609901,0.00794099479611224,0.00794099479611224,0.00794099479611224,0.00796000481975374,0.00817049525990943,0.00845844458070839,0.00845844458070839,0.00893558860375,0.00906474671076143,0.00924435233437624,0.00924435233437624,0.00956982896309875,0.00957927699110084,0.00973128057095716,0.00977504911126438,0.00985122343951636,0.0103606387998427,0.0107987227630292,0.0110758885072289,0.0112984154085833,0.0113197150019352,0.0114137324911485,0.0116305146437795,0.011805121966305,0.0119187939560491,0.0128781013289445,0.0131221511088425,0.0134904879247849,0.0135651055785705,0.0141909438112799,0.0142468361541419,0.0145964518222547,0.0150602274914983,0.0155712368003616,0.0158274237419385,0.0167495352536565,0.0169246669956255,0.0172326328989345,0.0177198087677641,0.0177198087677641,0.0177198087677641,0.0177198087677641,0.0185620345727638,0.0185620345727638,0.0190870373343181,0.0190870373343181,0.0194123805478604,0.0194632222473196,0.0196868609236293,0.0197031586811841,0.0197031586811841,0.0197031586811841,0.0197031586811841,0.0198262067995444,0.0201223181407776,0.0203027343342572,0.0207224763147494,0.0208580181076501],"Macs2 Peak":[true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true],"Region":["Intron","Intergenic (>10kb)","Intron","Intron","Upstream Promoter (-5kb)","Intergenic (>10kb)","Intron","Intergenic (>10kb)","Intron","Intron","Intron","Intron","Intron","Intron","Intergenic (<10kb)","Upstream Promoter (-5kb)","Intron","Intron","Intron","Intron","Intron","Intergenic (<10kb)","Promoter (-1500/+500)","Intergenic (<10kb)","Intron","Intron","Intergenic (>10kb)","Intergenic (<10kb)","Intron","Intergenic (>10kb)","Upstream Promoter (-5kb)","Intron","Intron","Intergenic (>10kb)","Intron","Intergenic (<10kb)","Intron","Intron","Intron","Intron","Intron","Upstream Promoter (-5kb)","Intron","Intron","Intron","Intron","Intergenic (<10kb)","Intron","Intron","Intergenic (>10kb)","Intron","Intron","Intron","Intron","Intergenic (>10kb)","Intron","Intron","Intron","Intron","Promoter (-1500/+500)","Intron","Promoter (-1500/+500)","Intron","Intron","Promoter (-1500/+500)","Promoter (-1500/+500)","Intron","Intergenic (<10kb)","Intergenic (>10kb)","Intron","Promoter (-1500/+500)","Intergenic (<10kb)","Intergenic (>10kb)","Intron","Intron","Intron","Upstream Promoter (-5kb)","Intron","Intron","Intergenic (<10kb)","Promoter (-1500/+500)","Intron","Upstream Promoter (-5kb)","Intergenic (>10kb)","Intron","Upstream Promoter (-5kb)","Intergenic (>10kb)","Intron","Intron","Intergenic (<10kb)","Intron","Promoter (-1500/+500)","Intergenic (>10kb)","Upstream Promoter (-5kb)","Intron","Intron","Intron","Intergenic (>10kb)","Intron","Intergenic (>10kb)","Intron","Intron","Intron","Exon","Intron","Intergenic (>10kb)","Intron","Intergenic (<10kb)","Intron","Promoter (-1500/+500)","Intron","Promoter (-1500/+500)","Upstream Promoter (-5kb)","Intron","Intergenic (<10kb)","Upstream Promoter (-5kb)","Intron","Promoter (-1500/+500)","Promoter (-1500/+500)","Upstream Promoter (-5kb)","Intergenic (<10kb)","Intron","Intron","Intron","Upstream Promoter (-5kb)","Intron","Upstream Promoter (-5kb)","Intron","Intron","Upstream Promoter (-5kb)","Intergenic (>10kb)","Intron","Upstream Promoter (-5kb)","Intergenic (<10kb)","Intron","Intron","Intron","Intron","Promoter (-1500/+500)","Intergenic (<10kb)","Intergenic (<10kb)","Intergenic (<10kb)","Intergenic (<10kb)","Intergenic (>10kb)","Intron","Upstream Promoter (-5kb)","Intron","Upstream Promoter (-5kb)","Promoter (-1500/+500)","Upstream Promoter (-5kb)","Intergenic (<10kb)","Intron","Intergenic (<10kb)","Intron","Intron","Intergenic (>10kb)","Intergenic (<10kb)","Intron","Intron","Intergenic (>10kb)","Promoter (-1500/+500)","Intron","Intron","Intron","Intron","Upstream Promoter (-5kb)","Intergenic (>10kb)","Promoter (-1500/+500)","Promoter (-1500/+500)","Upstream Promoter (-5kb)","Promoter (-1500/+500)","Promoter (-1500/+500)","Intron","Intron","Intron","Intergenic (>10kb)","Intron","Intergenic (<10kb)","Upstream Promoter (-5kb)","Intron","Exon","Intron","Intron","Intergenic (>10kb)","Intergenic (>10kb)","Intron","Intron","Intron","Intron","Intron","Intron","Intergenic (>10kb)","Intergenic (<10kb)","Intron","Upstream Promoter (-5kb)","Intron","Intergenic (>10kb)","Intron","Exon","Intron"],"Feature":["No Feature","No Feature","Enhancer","Enhancer","No Feature","No Feature","Enhancer","No Feature","Enhancer","No Feature","Enhancer","No Feature","Enhancer","Enhancer","Enhancer","No Feature","No Feature","No Feature","No Feature","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","No Feature","Enhancer","No Feature","Enhancer","No Feature","Enhancer","No Feature","No Feature","No Feature","No Feature","No Feature","No Feature","Enhancer","No Feature","Enhancer","No Feature","No Feature","No Feature","Enhancer","No Feature","No Feature","No Feature","No Feature","No Feature","Enhancer","No Feature","No Feature","No Feature","Enhancer","Enhancer","No Feature","No Feature","No Feature","Enhancer","Enhancer","No Feature","Enhancer","No Feature","No Feature","Enhancer","No Feature","No Feature","Enhancer","Enhancer","Enhancer","No Feature","Enhancer","No Feature","Enhancer","No Feature","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","No Feature","No Feature","No Feature","Enhancer","Enhancer","Enhancer","No Feature","Enhancer","Enhancer","No Feature","No Feature","No Feature","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","No Feature","Enhancer","No Feature","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","No Feature","No Feature","No Feature","Enhancer","Enhancer","Enhancer","Enhancer","No Feature","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","No Feature","Enhancer","No Feature","No Feature","Enhancer","No Feature","Enhancer","Enhancer","Enhancer","No Feature","No Feature","Enhancer","No Feature","No Feature","No Feature","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","No Feature","No Feature","Enhancer","No Feature","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","No Feature","Enhancer","Enhancer","Enhancer","Enhancer","No Feature","No Feature","No Feature","No Feature","Enhancer","No Feature","No Feature","No Feature","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","No Feature","No Feature","Enhancer","Enhancer","No Feature","No Feature","Enhancer","No Feature","No Feature","No Feature","No Feature","Enhancer","Enhancer","No Feature","No Feature","No Feature","No Feature","No Feature","Enhancer","Enhancer","No Feature","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer"],"Genes":["CCDC40","","PRKAB2; PDIA3P1; FMO5","ACACA","NFIC","EHF","C1orf21; EDEM3","ZCCHC2","MRPS23; CUEDC1; VEZF1; AC015813.1; SRSF1; AC015813.5","RASSF3","KCNMA1","PHF20L1","CSNK1A1; AC021078.1; ARHGEF37","TOR1AIP2; TOR1AIP1; CEP350","CCDC149","","","ACACA","RANBP3L","UHRF1; KDM4B","","AC104825.1; ACOX3; TRMT44","CHPT1","NFX1; AQP3; NOL6; ANKRD18B","EGLN2","GIPR; QPCTL; EML2; SNRPD2; FBXO46; AC007191.1; SIX5; DMPK; DMWD; SYMPK","","","NEDD4L","","DDT","SPIRE1","STXBP3","","THRAP3","AQP3","SLC66A1; CAPZB; MICOS10","ABCC1","CSNK1A1; AC021078.1; ARHGEF37","SLC38A1","AKAP13","MCCC2","RERG","ACACA","FANCC","KDM6A","RASGEF1A","ACACA","SCD; OLMALINC","ANAPC16","PHB","FAM104A","APBB2","AL033523.1; PXDC1","ZCCHC2","RANBP3L","ATP6V0E1","ZNF652; PHB; EIF4EP2","APBB2","ZNF689","FKBP5","RNF215; SEC14L2","MYCBP2","","SYBU","CTNNBL1","PACSIN1; SPDEF; ILRUN; AL451165.2","MCCC2","TMEM17","RCL1","FKBP5","PPIF","NR4A1; ATG101; KRT80","SORD","","IGSF8; PEA15; DCAF8","","MKNK1; MOB3C; ATPAF1; EFCAB14","PRKAA2","AC104825.1; ACOX3; TRMT44","CRAMP1; IFT140; JPT2; AL031009.1; MAPK8IP3","KAT6B","CRAMP1; IFT140; JPT2; AL031009.1; MAPK8IP3","","ATP6V0E1","PDZD2","","KIAA0232; TBC1D14; CCDC96; TADA2B","C1orf21; EDEM3","VIPR1; VIPR1-AS1; SEC22C; SS18L2; NKTR; AC006059.1","AHCY","FLNB; FLNB-AS1; ABHD6","C12orf49","PDZD2","MYCBP2","ARHGEF37","SPRED1","","MPC2; DCAF6","KLHL15; EIF2S3; ZFX","IGSF8; PEA15; DCAF8","TEX2","FOXP2","RASGEF1A","VSTM2L; TTI1; RPRD1B","","","CD9; PLEKHG6; TNFRSF1A; SCNN1A","","TTC21A; WDR48; GORASP1; CSRNP1; AC092053.4","FAM155A","","NPNT","","","CNTNAP2","PACSIN1; SPDEF; ILRUN; AL451165.2","","MTCL1; RAB12","NF1","EHF","","UBE2Q2; FBXO22","","WWC2","SASH1","UQCC2","VWA8","NRP1","","GCLC","FGGY","ATP6V0D1; ZDHHC1; HSD11B2; AC009061.2; RIPOR1; CTCF","","ANO2","CUEDC1","CNNM1","","CRAT","","ANG","ELMO1","","NUFIP1","BLOC1S6; SQOR","PARD6B; BCAS4; ADNP; ADNP-AS1; DPM1","SLITRK6","MCCC2","CHPT1","RNF215","","PRLR","ATP6AP2","CAPZB; MICOS10","","","NAMPT","PRKAB2; PDIA3P1; FMO5","DLG2","","PDCD6","","RCL1","MEGF9","LRBA","MYCBP2","TMSB4X","MTMR9","TTC12","PTPRJ","CDK5RAP2; MEGF9","SLC1A4","","RAB31","PEX14","MYBL1; VCPIP1","GFRA1","RYBP","TIGD6","CAPZB; MICOS10","LARS2","TPD52","FAM104A","EIF2S3","","DLG2","TRRAP","SULT2B1","YTHDF1","HIBCH","CNNM1","","DUSP16; CREBL2","NEDD4L","PARD6B; BCAS4; ADNP; ADNP-AS1; DPM1","UBA6; UBA6-AS1","ADK; KAT6B","WHAMM; HOMER2; RAMAC; C15orf40; BTBD1","KAT6B; SAMD8","AMPD2; GSTM4; GSTM3"]},"columns":[{"id":"Range","name":"Range","type":"character","cell":["chr17: 78041721-78043240","chr17: 71856921-71857720","chr1: 146580841-146582200","chr17: 35630121-35634400","chr19: 3429841-3431000","chr11: 34692801-34693720","chr1: 184601281-184603080","chr18: 60167801-60168320","chr17: 56008201-56009880","chr12: 64983521-64984000","chr10: 79257401-79258680","chr8: 133910801-133911240","chr5: 148980801-148981200","chr1: 179840601-179841960","chr4: 24805681-24807640","chr14: 97126721-97128760","chr8: 50996801-50998000","chr17: 35678601-35679080","chr5: 36285601-36286000","chr19: 4989881-4993440","chr4: 70261521-70261960","chr4: 8353881-8354600","chr12: 102009521-102009920","chr9: 33430521-33432480","chr19: 41360601-41362760","chr19: 46246201-46247280","chr17: 71856241-71856720","chr6: 6748681-6749240","chr18: 55952481-55953040","chr1: 163586921-163587360","chr22: 24364321-24364480","chr18: 12589121-12589560","chr1: 109370441-109371080","chr18: 20066161-20066560","chr1: 36705801-36706080","chr9: 33436761-33437720","chr1: 19752521-19753720","chr16: 16135561-16135680","chr5: 148980281-148980520","chr12: 46640041-46640560","chr15: 86080681-86081360","chr5: 71009841-71010920","chr12: 15427161-15427560","chr17: 35675241-35675360","chr9: 98045761-98046360","chrX: 44751281-44751960","chr10: 43689241-43690120","chr17: 35672201-35672480","chr10: 102180001-102180280","chr10: 74008161-74008720","chr17: 47470801-47473080","chr17: 71212001-71212880","chr4: 41079321-41079640","chr6: 3669761-3670240","chr18: 60168481-60168920","chr5: 36286321-36286680","chr5: 172437721-172438720","chr17: 47466041-47470360","chr4: 41080161-41080920","chr16: 30626521-30627200","chr6: 35694801-35695960","chr22: 30804321-30804760","chr13: 77881041-77881600","chr18: 56216881-56217040","chr8: 110592641-110593400","chr20: 36474201-36475720","chr6: 34565441-34567480","chr5: 71048921-71049440","chr2: 62651721-62651880","chr9: 4866601-4867240","chr6: 35696041-35696400","chr10: 81101681-81102040","chr12: 52519161-52520880","chr15: 45329681-45330040","chr16: 34751281-34751600","chr1: 160126441-160127120","chr4: 70073801-70073960","chr1: 47108201-47110120","chr1: 57254081-57254720","chr4: 8353121-8353520","chr16: 1681001-1681880","chr10: 76706361-76709080","chr16: 1680121-1680920","chr1: 118283001-118283320","chr5: 172438841-172439440","chr5: 32093721-32094160","chr8: 68723241-68723600","chr4: 6958121-6960000","chr1: 184599401-184599680","chr3: 42585721-42587320","chr20: 32887401-32887960","chr3: 58135441-58136000","chr12: 117074121-117074840","chr5: 32094601-32095200","chr13: 77880361-77880760","chr5: 148993321-148993720","chr15: 38508601-38508960","chr2: 67742641-67743360","chr1: 167880761-167881080","chrX: 24109121-24109800","chr1: 160127361-160127760","chr17: 62300401-62300560","chr7: 113656641-113656880","chr10: 43690481-43690840","chr20: 36618321-36618720","chr2: 220711481-220712160","chr7: 95178921-95179600","chr12: 6375561-6376440","chr3: 181019521-181020000","chr3: 39172241-39173080","chr13: 108395201-108395960","chr1: 242309921-242310680","chr4: 106831281-106831720","chr21: 16958721-16959200","chr9: 110803441-110803640","chr7: 148082041-148082560","chr6: 34571041-34571440","chr1: 242310801-242311560","chr18: 8703721-8706880","chr17: 29381601-29381840","chr11: 34771481-34771880","chr7: 134537041-134537600","chr15: 76155401-76156280","chr5: 148398281-148398640","chr4: 184182001-184182400","chr6: 148579321-148580120","chr6: 33683841-33685120","chr13: 42354521-42356000","chr10: 33529921-33530440","chr18: 56220401-56220560","chr6: 53333121-53333960","chr1: 60234961-60235280","chr16: 67492761-67497800","chr16: 34757841-34758360","chr12: 6129681-6130200","chr17: 56009961-56010080","chr10: 101098961-101099600","chr7: 134537721-134538280","chr9: 131869041-131869600","chr14: 24305321-24306320","chr14: 21242961-21243400","chr7: 36824521-36825400","chr6: 6749481-6750720","chr13: 45428001-45428840","chr15: 45977001-45977640","chr20: 49459121-49460000","chr13: 86298321-86299200","chr5: 71011001-71012000","chr12: 102010281-102010560","chr22: 30788401-30789040","chr9: 111471241-111473840","chr5: 35206081-35206520","chrX: 40376681-40378880","chr1: 19785841-19786320","chr21: 30014361-30015040","chr17: 69906641-69907160","chr7: 105856761-105857880","chr1: 146585001-146587360","chr11: 83121521-83121960","chr1: 163595881-163596480","chr5: 302441-303400","chr3: 177803081-177803680","chr9: 4865721-4866240","chr9: 123391201-123391880","chr4: 151644361-151645080","chr13: 77665601-77665800","chrX: 13082881-13083800","chr8: 11143801-11144280","chr11: 113211161-113213760","chr11: 48063041-48063560","chr9: 123294081-123295880","chr2: 65134281-65135160","chr4: 70152081-70153240","chr18: 9717961-9718360","chr1: 10575441-10578000","chr8: 67442481-67446000","chr10: 117903121-117903520","chr3: 72347281-72347480","chr5: 149375441-149377160","chr1: 19784841-19785480","chr3: 45588161-45589880","chr8: 80989321-80989560","chr17: 71211201-71211840","chrX: 24108241-24108960","chr1: 162993481-162993840","chr11: 83122161-83122600","chr7: 98622081-98622240","chr19: 49085721-49086400","chr20: 61837881-61839880","chr2: 191100481-191101000","chr10: 101098401-101098840","chr3: 77741601-77742280","chr12: 12727481-12730200","chr18: 55953241-55953400","chr20: 49460121-49460600","chr4: 68577201-68579200","chr10: 76546281-76546680","chr15: 83586961-83589360","chr10: 76780521-76782800","chr1: 110266241-110271680"],"minWidth":120},{"id":"Width (bp)","name":"Width (bp)","type":"numeric","maxWidth":60,"style":[{"background":"#239C93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#129893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#1F9B93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#63A993","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#1B9A93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#159993","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#2A9D93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0C9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#279C93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0B9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#1D9B93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0A9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#099693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#1F9B93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#2D9E93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#2F9E93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#1C9A93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0B9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#099693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#53A593","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0A9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#109893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#099693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#2D9E93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#329F93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#199A93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0B9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0D9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0D9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0A9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#039593","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0A9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0E9893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#099693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#069693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#169993","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#1C9A93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#029593","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#059693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0C9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0F9893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#199A93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#099693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#029593","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0E9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0F9893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#149993","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#069693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#069693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0D9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#359F93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#149993","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#079693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0B9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0A9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#089693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#179993","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#64A993","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#119893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0F9893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#1B9A93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0A9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0D9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#039593","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#119893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#239C93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#2F9E93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0C9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#039593","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0E9893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#089693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#089693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#289D93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#089693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#079693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0F9893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#039593","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#2C9E93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0E9893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#099693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#149993","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#3FA193","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#129893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#079693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0E9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0A9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#089693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#2B9D93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#069693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#259C93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0D9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0D9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#109893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0E9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#099693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#099693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#089693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#109893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#079693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0F9893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#099693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#039593","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#059693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#089693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#099693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0F9893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0F9893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#149993","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0B9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#139893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#119893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#119893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0A9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0B9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#049593","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0C9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#099693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#119893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#49A393","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#059693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#099693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0D9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#149993","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#089693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#099693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#129893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#1D9B93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#229B93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0C9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#039593","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#139893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#079693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#6EAC96","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0C9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0C9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#029593","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0E9893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0D9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0D9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#179993","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0A9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#149993","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#1C9A93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#139893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0E9893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#149993","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#149993","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#179993","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#069693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0E9893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#3CA193","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0A9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#339F93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0B9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0F9893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0C9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#1A9A93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#37A093","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0A9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0E9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#169993","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0E9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0C9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0F9893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#109893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#049593","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#159993","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0B9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#3CA193","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0C9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#2A9D93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#149993","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#1B9A93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#099693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#3BA193","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#52A593","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#099693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#049593","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#289D93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0E9893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#289D93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#059693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0E9893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#109893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#089693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0A9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#039593","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0F9893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#2E9E93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0C9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0A9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0F9893","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#3FA193","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#039593","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#0B9793","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#2E9E93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#099693","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#38A093","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#359F93","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"background":"#73AE97","borderRight":"1px solid rgba(0, 0, 0, 0.1)"}]},{"id":"AveExpr","name":"AveExpr","type":"numeric","cell":["0.82","1.44","1.17","1.82","1.62","1.15","2.51","0.81","1.9","0.57","1.31","0.78","0.5","0.84","2.31","1.92","1.29","0.42","0.61","1.91","0.57","1.12","0.65","1.45","1.87","1.83","0.59","0.95","0.91","1","0.17","0.7","0.97","0.65","0.38","0.7","1.71","0.05","0.37","0.66","0.67","2.14","0.43","0.05","0.59","1.02","0.85","0.2","0.27","0.76","3","0.79","0.46","0.74","0.62","0.46","1.53","3.9","0.44","1.01","1.43","0.47","0.67","0.29","1.67","2.05","1.61","0.74","0.28","1.13","0.58","0.55","1.33","0.48","0.54","1.55","0.18","2.01","1.04","0.61","1.36","1.55","1.29","0.58","1.22","0.66","0.63","1.59","0.33","1.44","0.73","0.84","0.92","1.05","0.72","0.63","0.55","1.54","0.48","1.23","0.58","0.26","0.41","0.55","0.66","0.9","1.34","0.94","0.7","1.23","1.06","1.2","0.43","0.68","0.39","1.14","0.51","1.67","3.25","0.35","0.64","1.15","0.87","0.54","0.52","1.63","0.92","1.24","0.72","0.36","0.89","0.5","2.91","1.13","0.9","0.25","1.61","0.96","1.05","2.09","0.72","1.33","1.26","1.07","0.79","1.58","1.25","1.81","0.58","0.82","2.48","0.68","2.12","0.92","1.45","0.72","1.5","2.21","0.69","1.3","0.8","0.83","0.47","0.9","0.94","0.31","1.81","0.84","2.1","1.01","1.6","2.14","2.32","0.48","1.29","2.13","0.8","0.38","1.47","0.98","1.76","0.33","1.16","1.44","0.47","0.87","0.29","1.07","1.26","0.8","0.73","1.19","2.19","0.35","0.85","1.44","0.67","2.61","1.98","2.27"],"maxWidth":66,"style":[{"background":"#72AE97"},{"background":"#A9C1A9"},{"background":"#94B8A1"},{"background":"#C2CFB5"},{"background":"#B5C8AF"},{"background":"#93B8A0"},{"background":"#EEE8C8"},{"background":"#71AD97"},{"background":"#C7D2B8"},{"background":"#54A693"},{"background":"#A1BDA5"},{"background":"#6EAC96"},{"background":"#49A393"},{"background":"#74AE98"},{"background":"#E2E1C3"},{"background":"#C9D3B8"},{"background":"#9FBCA4"},{"background":"#3CA193"},{"background":"#5AA793"},{"background":"#C8D2B8"},{"background":"#54A693"},{"background":"#8FB79F"},{"background":"#61A893"},{"background":"#AAC2A9"},{"background":"#C6D1B7"},{"background":"#C3CFB6"},{"background":"#58A693"},{"background":"#7FB29B"},{"background":"#7AB099"},{"background":"#83B39C"},{"background":"#149993"},{"background":"#67AA94"},{"background":"#81B29B"},{"background":"#61A893"},{"background":"#359F93"},{"background":"#66AA94"},{"background":"#BBCBB2"},{"background":"#009593"},{"background":"#349F93"},{"background":"#62A893"},{"background":"#64A993"},{"background":"#D7DBBF"},{"background":"#3DA193"},{"background":"#019593"},{"background":"#57A693"},{"background":"#86B49D"},{"background":"#75AE98"},{"background":"#199A93"},{"background":"#239C93"},{"background":"#6DAC95"},{"background":"#E3D0B2"},{"background":"#6FAD96"},{"background":"#42A293"},{"background":"#6AAB95"},{"background":"#5CA793"},{"background":"#41A293"},{"background":"#AFC5AC"},{"background":"#D7A18A"},{"background":"#3EA193"},{"background":"#84B39C"},{"background":"#A8C1A8"},{"background":"#44A293"},{"background":"#63A993"},{"background":"#279C93"},{"background":"#B8CAB0"},{"background":"#D1D7BC"},{"background":"#B4C8AE"},{"background":"#6AAB95"},{"background":"#259C93"},{"background":"#90B7A0"},{"background":"#56A693"},{"background":"#50A593"},{"background":"#A2BDA5"},{"background":"#45A293"},{"background":"#4FA593"},{"background":"#B0C5AC"},{"background":"#169993"},{"background":"#CFD6BB"},{"background":"#88B49D"},{"background":"#5AA793"},{"background":"#A4BFA6"},{"background":"#B0C5AC"},{"background":"#9FBCA4"},{"background":"#56A693"},{"background":"#99BAA2"},{"background":"#63A993"},{"background":"#5DA793"},{"background":"#B3C7AE"},{"background":"#2E9E93"},{"background":"#A9C1A9"},{"background":"#69AB94"},{"background":"#74AE97"},{"background":"#7CB19A"},{"background":"#88B49D"},{"background":"#68AA94"},{"background":"#5DA793"},{"background":"#50A593"},{"background":"#B0C5AC"},{"background":"#45A393"},{"background":"#9BBAA2"},{"background":"#55A693"},{"background":"#229C93"},{"background":"#3AA093"},{"background":"#51A593"},{"background":"#62A893"},{"background":"#7AB099"},{"background":"#A3BEA6"},{"background":"#7EB19A"},{"background":"#67AA94"},{"background":"#9ABAA2"},{"background":"#89B59E"},{"background":"#97B9A1"},{"background":"#3EA193"},{"background":"#64A993"},{"background":"#36A093"},{"background":"#91B7A0"},{"background":"#4AA493"},{"background":"#B8CAB0"},{"background":"#DFC4A6"},{"background":"#319E93"},{"background":"#5FA893"},{"background":"#92B8A0"},{"background":"#77AF98"},{"background":"#4FA593"},{"background":"#4BA493"},{"background":"#B6C8AF"},{"background":"#7CB09A"},{"background":"#9BBAA3"},{"background":"#69AA94"},{"background":"#339F93"},{"background":"#79B099"},{"background":"#49A393"},{"background":"#E5D5B6"},{"background":"#91B7A0"},{"background":"#7AB099"},{"background":"#209B93"},{"background":"#B4C8AE"},{"background":"#7FB29B"},{"background":"#88B49D"},{"background":"#D3D9BD"},{"background":"#69AA94"},{"background":"#A2BDA5"},{"background":"#9DBBA3"},{"background":"#8BB59E"},{"background":"#6FAC96"},{"background":"#B2C6AD"},{"background":"#9CBBA3"},{"background":"#C1CEB5"},{"background":"#55A693"},{"background":"#72AE97"},{"background":"#EDE8C8"},{"background":"#64A993"},{"background":"#D5DABE"},{"background":"#7BB09A"},{"background":"#AAC2A9"},{"background":"#69AA94"},{"background":"#ADC4AB"},{"background":"#DBDEC0"},{"background":"#65A993"},{"background":"#A0BCA4"},{"background":"#70AD96"},{"background":"#73AE97"},{"background":"#43A293"},{"background":"#7AB099"},{"background":"#7EB19A"},{"background":"#2A9D93"},{"background":"#C1CFB5"},{"background":"#74AE97"},{"background":"#D4D9BD"},{"background":"#84B39C"},{"background":"#B4C7AE"},{"background":"#D7DBBE"},{"background":"#E3E2C3"},{"background":"#45A393"},{"background":"#A0BCA4"},{"background":"#D6DABE"},{"background":"#70AD96"},{"background":"#369F93"},{"background":"#ABC3AA"},{"background":"#81B29B"},{"background":"#BECDB3"},{"background":"#2D9E93"},{"background":"#93B8A0"},{"background":"#A9C1A9"},{"background":"#44A293"},{"background":"#77AF98"},{"background":"#279C93"},{"background":"#8BB59E"},{"background":"#9DBBA3"},{"background":"#70AD96"},{"background":"#69AB94"},{"background":"#96B9A1"},{"background":"#DADDC0"},{"background":"#309E93"},{"background":"#75AE98"},{"background":"#A9C2A9"},{"background":"#63A993"},{"background":"#ECE4C4"},{"background":"#CCD5BA"},{"background":"#DFE0C2"}]},{"id":"logFC","name":"logFC","type":"numeric","cell":["2.76","2.14","2.15","1.65","1.69","1.94","1.27","2.06","1.45","2.09","1.61","1.88","2.04","1.87","1.19","1.32","1.55","1.96","1.82","1.26","1.81","1.52","1.74","1.4","1.21","1.23","1.7","1.51","1.53","1.52","2.16","1.58","1.46","1.58","1.75","1.6","1.2","2.25","1.7","1.57","1.53","1.1","1.66","2.19","1.56","1.36","1.41","1.99","1.72","1.45","0.8","1.47","1.58","1.46","1.48","1.56","1.14","0.6","1.52","1.28","1.15","1.53","1.48","1.57","1.08","0.96","1.09","1.35","1.83","1.22","1.36","1.42","1.13","1.43","1.4","1.07","1.9","0.95","1.2","1.37","1.16","1.03","1.14","1.35","1.1","1.56","1.32","1.03","1.72","1.12","1.28","1.24","1.2","1.18","1.26","1.3","1.33","1.01","1.34","1.09","1.29","1.71","1.33","1.28","1.24","1.16","1.1","1.12","1.21","1.1","1.13","1.04","1.6","1.22","1.36","1.08","1.25","0.91","0.61","1.31","1.26","1.05","1.11","1.2","1.22","0.94","1.1","1.01","1.14","1.27","1.12","1.5","0.68","1.03","1.07","1.61","0.9","1.06","1.03","0.8","1.13","0.95","0.96","1.03","1.2","0.92","1","0.9","1.13","1.14","0.72","1.13","0.81","1.02","0.93","1.08","0.89","0.76","1.12","0.98","1.07","1.04","1.21","1.02","1.01","1.55","0.84","1.04","0.8","1.01","0.85","0.78","0.72","1.13","0.92","0.75","1.08","1.15","0.86","1.23","0.79","1.53","0.92","0.9","-1.43","1.03","1.5","0.93","0.89","1.03","1.03","0.92","0.71","1.14","1.02","0.88","1.04","0.67","0.75","0.73"],"maxWidth":66,"style":[{"background":"#CA0020"},{"background":"#D53952"},{"background":"#D53851"},{"background":"#DF6679"},{"background":"#DE6276"},{"background":"#D94C62"},{"background":"#E68998"},{"background":"#D74058"},{"background":"#E37889"},{"background":"#D63D55"},{"background":"#E0697C"},{"background":"#DA5167"},{"background":"#D74259"},{"background":"#DB5267"},{"background":"#E8919F"},{"background":"#E58493"},{"background":"#E16F81"},{"background":"#D94A60"},{"background":"#DB566B"},{"background":"#E68A99"},{"background":"#DC576C"},{"background":"#E17283"},{"background":"#DD5D71"},{"background":"#E47D8D"},{"background":"#E78F9D"},{"background":"#E78D9B"},{"background":"#DE6175"},{"background":"#E27385"},{"background":"#E17183"},{"background":"#E17284"},{"background":"#D53750"},{"background":"#E06C7E"},{"background":"#E27889"},{"background":"#E06D7F"},{"background":"#DD5D71"},{"background":"#E06A7D"},{"background":"#E78F9D"},{"background":"#D32E49"},{"background":"#DE6175"},{"background":"#E06D7F"},{"background":"#E17183"},{"background":"#E999A6"},{"background":"#DE6578"},{"background":"#D4344D"},{"background":"#E16E81"},{"background":"#E48190"},{"background":"#E37C8C"},{"background":"#D8475E"},{"background":"#DE6074"},{"background":"#E3798A"},{"background":"#EFB4BE"},{"background":"#E27788"},{"background":"#E06C7F"},{"background":"#E27788"},{"background":"#E27587"},{"background":"#E16F81"},{"background":"#E995A3"},{"background":"#F3C7CE"},{"background":"#E17283"},{"background":"#E68997"},{"background":"#E895A2"},{"background":"#E17183"},{"background":"#E27687"},{"background":"#E06E80"},{"background":"#EA9BA7"},{"background":"#ECA5B1"},{"background":"#EA9AA7"},{"background":"#E58292"},{"background":"#DB566B"},{"background":"#E78E9C"},{"background":"#E48190"},{"background":"#E37C8C"},{"background":"#E996A3"},{"background":"#E37A8B"},{"background":"#E47D8D"},{"background":"#EA9CA8"},{"background":"#DA4F65"},{"background":"#ECA6B1"},{"background":"#E78F9D"},{"background":"#E48090"},{"background":"#E893A0"},{"background":"#EB9FAB"},{"background":"#E995A2"},{"background":"#E58292"},{"background":"#E999A5"},{"background":"#E06E80"},{"background":"#E58493"},{"background":"#EB9FAB"},{"background":"#DD5F73"},{"background":"#E997A4"},{"background":"#E68897"},{"background":"#E78C9A"},{"background":"#E78F9D"},{"background":"#E8919F"},{"background":"#E68A98"},{"background":"#E68696"},{"background":"#E58393"},{"background":"#EBA1AD"},{"background":"#E58292"},{"background":"#EA9AA6"},{"background":"#E68897"},{"background":"#DE6074"},{"background":"#E58393"},{"background":"#E68897"},{"background":"#E78C9A"},{"background":"#E893A0"},{"background":"#E998A5"},{"background":"#E997A4"},{"background":"#E78E9D"},{"background":"#E999A6"},{"background":"#E996A3"},{"background":"#EB9EAA"},{"background":"#E06A7D"},{"background":"#E78D9B"},{"background":"#E48191"},{"background":"#EA9BA7"},{"background":"#E68B99"},{"background":"#EDAAB5"},{"background":"#F3C6CD"},{"background":"#E58594"},{"background":"#E68A98"},{"background":"#EA9DA9"},{"background":"#E998A5"},{"background":"#E78F9D"},{"background":"#E78D9B"},{"background":"#ECA8B3"},{"background":"#E999A6"},{"background":"#EBA1AD"},{"background":"#E996A3"},{"background":"#E68998"},{"background":"#E997A4"},{"background":"#E27485"},{"background":"#F1C0C8"},{"background":"#EB9FAB"},{"background":"#EA9BA8"},{"background":"#E06A7D"},{"background":"#EDACB6"},{"background":"#EA9DA9"},{"background":"#EB9FAB"},{"background":"#EFB4BD"},{"background":"#E996A3"},{"background":"#ECA6B1"},{"background":"#ECA5B0"},{"background":"#EB9FAB"},{"background":"#E78F9D"},{"background":"#EDAAB4"},{"background":"#EBA2AD"},{"background":"#EDABB6"},{"background":"#E996A3"},{"background":"#E995A2"},{"background":"#F1BCC5"},{"background":"#E996A3"},{"background":"#EFB4BD"},{"background":"#EBA0AC"},{"background":"#EDA9B4"},{"background":"#EA9AA7"},{"background":"#EDACB6"},{"background":"#F0B9C1"},{"background":"#E997A4"},{"background":"#ECA4AF"},{"background":"#EA9CA8"},{"background":"#EA9EAA"},{"background":"#E78F9D"},{"background":"#EBA0AC"},{"background":"#EBA1AD"},{"background":"#E16F81"},{"background":"#EEB1BB"},{"background":"#EA9EAA"},{"background":"#EFB5BE"},{"background":"#EBA1AD"},{"background":"#EEB0BA"},{"background":"#F0B7C0"},{"background":"#F1BCC5"},{"background":"#E996A3"},{"background":"#EDAAB4"},{"background":"#F0B9C2"},{"background":"#EA9BA8"},{"background":"#E894A2"},{"background":"#EEAFB9"},{"background":"#E78D9B"},{"background":"#EFB5BE"},{"background":"#E17183"},{"background":"#EDA9B4"},{"background":"#EDACB6"},{"background":"#7DB5D6"},{"background":"#EB9FAB"},{"background":"#E27485"},{"background":"#EDA8B3"},{"background":"#EDACB6"},{"background":"#EB9FAB"},{"background":"#EB9FAB"},{"background":"#EDA9B4"},{"background":"#F1BDC5"},{"background":"#E995A3"},{"background":"#EBA0AC"},{"background":"#EDADB7"},{"background":"#EB9EAA"},{"background":"#F2C0C8"},{"background":"#F0B9C2"},{"background":"#F1BBC4"}]},{"id":"FDR","name":"FDR","type":"numeric","cell":["4.86e-16","2.03e-14","2.50e-12","4.13e-11","2.02e-10","2.25e-10","5.70e-10","1.32e-09","1.53e-09","5.82e-09","2.05e-08","3.99e-08","4.19e-08","4.20e-08","6.35e-08","1.17e-07","1.85e-07","5.35e-07","6.98e-07","8.05e-07","8.65e-07","1.08e-06","1.96e-06","2.11e-06","3.51e-06","3.94e-06","3.96e-06","6.27e-06","7.08e-06","7.55e-06","8.70e-06","1.02e-05","1.12e-05","1.52e-05","1.91e-05","1.91e-05","2.39e-05","2.39e-05","2.58e-05","2.58e-05","3.05e-05","3.08e-05","3.08e-05","3.08e-05","3.18e-05","4.39e-05","5.99e-05","7.29e-05","7.63e-05","7.71e-05","7.71e-05","7.71e-05","8.06e-05","8.06e-05","8.06e-05","1.12e-04","1.24e-04","1.54e-04","1.67e-04","2.05e-04","2.15e-04","2.15e-04","2.44e-04","2.65e-04","2.81e-04","2.81e-04","3.36e-04","3.85e-04","4.87e-04","4.87e-04","4.95e-04","5.10e-04","5.10e-04","5.10e-04","5.10e-04","5.30e-04","5.30e-04","5.65e-04","6.83e-04","6.92e-04","7.29e-04","7.79e-04","8.16e-04","8.17e-04","8.32e-04","8.34e-04","8.34e-04","8.42e-04","8.47e-04","8.47e-04","8.47e-04","9.56e-04","9.66e-04","9.78e-04","1.02e-03","1.10e-03","1.13e-03","1.23e-03","1.24e-03","1.60e-03","1.91e-03","1.94e-03","1.99e-03","2.10e-03","2.11e-03","2.33e-03","2.33e-03","2.33e-03","2.34e-03","2.48e-03","2.50e-03","2.55e-03","2.61e-03","3.02e-03","3.12e-03","3.61e-03","3.64e-03","4.18e-03","4.18e-03","4.29e-03","4.43e-03","4.51e-03","4.55e-03","4.63e-03","4.70e-03","4.83e-03","5.38e-03","5.38e-03","5.52e-03","5.67e-03","5.78e-03","6.21e-03","6.21e-03","6.65e-03","6.74e-03","6.74e-03","6.85e-03","6.90e-03","6.90e-03","6.90e-03","7.40e-03","7.71e-03","7.94e-03","7.94e-03","7.94e-03","7.96e-03","8.17e-03","8.46e-03","8.46e-03","8.94e-03","9.06e-03","9.24e-03","9.24e-03","9.57e-03","9.58e-03","9.73e-03","9.78e-03","9.85e-03","1.04e-02","1.08e-02","1.11e-02","1.13e-02","1.13e-02","1.14e-02","1.16e-02","1.18e-02","1.19e-02","1.29e-02","1.31e-02","1.35e-02","1.36e-02","1.42e-02","1.42e-02","1.46e-02","1.51e-02","1.56e-02","1.58e-02","1.67e-02","1.69e-02","1.72e-02","1.77e-02","1.77e-02","1.77e-02","1.77e-02","1.86e-02","1.86e-02","1.91e-02","1.91e-02","1.94e-02","1.95e-02","1.97e-02","1.97e-02","1.97e-02","1.97e-02","1.97e-02","1.98e-02","2.01e-02","2.03e-02","2.07e-02","2.09e-02"],"maxWidth":66,"style":{"borderRight":"1px solid rgba(0, 0, 0, 0.1)"}},{"id":"Macs2 Peak","name":"Macs2 Peak","type":"logical","cell":["✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes"],"maxWidth":60,"style":[{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"},{"color":"#008000","borderRight":"1px solid rgba(0, 0, 0, 0.1)"}]},{"id":"Region","name":"Region","type":"factor","maxWidth":150},{"id":"Feature","name":"Feature","type":"factor"},{"id":"Genes","name":"Genes","type":"character","cell":[{"name":"span","attribs":{"title":"CCDC40"},"children":["CCDC40"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"PRKAB2; PDIA3P1; FMO5"},"children":["PRKAB2; PDIA3P1; FMO5"]},{"name":"span","attribs":{"title":"ACACA"},"children":["ACACA"]},{"name":"span","attribs":{"title":"NFIC"},"children":["NFIC"]},{"name":"span","attribs":{"title":"EHF"},"children":["EHF"]},{"name":"span","attribs":{"title":"C1orf21; EDEM3"},"children":["C1orf21; EDEM3"]},{"name":"span","attribs":{"title":"ZCCHC2"},"children":["ZCCHC2"]},{"name":"span","attribs":{"title":"MRPS23; CUEDC1; VEZF1; AC015813.1; SRSF1; AC015813.5"},"children":["MRPS23; CUEDC1; VEZF1; AC01..."]},{"name":"span","attribs":{"title":"RASSF3"},"children":["RASSF3"]},{"name":"span","attribs":{"title":"KCNMA1"},"children":["KCNMA1"]},{"name":"span","attribs":{"title":"PHF20L1"},"children":["PHF20L1"]},{"name":"span","attribs":{"title":"CSNK1A1; AC021078.1; ARHGEF37"},"children":["CSNK1A1; AC021078.1; ARHGEF37"]},{"name":"span","attribs":{"title":"TOR1AIP2; TOR1AIP1; CEP350"},"children":["TOR1AIP2; TOR1AIP1; CEP350"]},{"name":"span","attribs":{"title":"CCDC149"},"children":["CCDC149"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"ACACA"},"children":["ACACA"]},{"name":"span","attribs":{"title":"RANBP3L"},"children":["RANBP3L"]},{"name":"span","attribs":{"title":"UHRF1; KDM4B"},"children":["UHRF1; KDM4B"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"AC104825.1; ACOX3; TRMT44"},"children":["AC104825.1; ACOX3; TRMT44"]},{"name":"span","attribs":{"title":"CHPT1"},"children":["CHPT1"]},{"name":"span","attribs":{"title":"NFX1; AQP3; NOL6; ANKRD18B"},"children":["NFX1; AQP3; NOL6; ANKRD18B"]},{"name":"span","attribs":{"title":"EGLN2"},"children":["EGLN2"]},{"name":"span","attribs":{"title":"GIPR; QPCTL; EML2; SNRPD2; FBXO46; AC007191.1; SIX5; DMPK; DMWD; SYMPK"},"children":["GIPR; QPCTL; EML2; SNRPD2; ..."]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"NEDD4L"},"children":["NEDD4L"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"DDT"},"children":["DDT"]},{"name":"span","attribs":{"title":"SPIRE1"},"children":["SPIRE1"]},{"name":"span","attribs":{"title":"STXBP3"},"children":["STXBP3"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"THRAP3"},"children":["THRAP3"]},{"name":"span","attribs":{"title":"AQP3"},"children":["AQP3"]},{"name":"span","attribs":{"title":"SLC66A1; CAPZB; MICOS10"},"children":["SLC66A1; CAPZB; MICOS10"]},{"name":"span","attribs":{"title":"ABCC1"},"children":["ABCC1"]},{"name":"span","attribs":{"title":"CSNK1A1; AC021078.1; ARHGEF37"},"children":["CSNK1A1; AC021078.1; ARHGEF37"]},{"name":"span","attribs":{"title":"SLC38A1"},"children":["SLC38A1"]},{"name":"span","attribs":{"title":"AKAP13"},"children":["AKAP13"]},{"name":"span","attribs":{"title":"MCCC2"},"children":["MCCC2"]},{"name":"span","attribs":{"title":"RERG"},"children":["RERG"]},{"name":"span","attribs":{"title":"ACACA"},"children":["ACACA"]},{"name":"span","attribs":{"title":"FANCC"},"children":["FANCC"]},{"name":"span","attribs":{"title":"KDM6A"},"children":["KDM6A"]},{"name":"span","attribs":{"title":"RASGEF1A"},"children":["RASGEF1A"]},{"name":"span","attribs":{"title":"ACACA"},"children":["ACACA"]},{"name":"span","attribs":{"title":"SCD; OLMALINC"},"children":["SCD; OLMALINC"]},{"name":"span","attribs":{"title":"ANAPC16"},"children":["ANAPC16"]},{"name":"span","attribs":{"title":"PHB"},"children":["PHB"]},{"name":"span","attribs":{"title":"FAM104A"},"children":["FAM104A"]},{"name":"span","attribs":{"title":"APBB2"},"children":["APBB2"]},{"name":"span","attribs":{"title":"AL033523.1; PXDC1"},"children":["AL033523.1; PXDC1"]},{"name":"span","attribs":{"title":"ZCCHC2"},"children":["ZCCHC2"]},{"name":"span","attribs":{"title":"RANBP3L"},"children":["RANBP3L"]},{"name":"span","attribs":{"title":"ATP6V0E1"},"children":["ATP6V0E1"]},{"name":"span","attribs":{"title":"ZNF652; PHB; EIF4EP2"},"children":["ZNF652; PHB; EIF4EP2"]},{"name":"span","attribs":{"title":"APBB2"},"children":["APBB2"]},{"name":"span","attribs":{"title":"ZNF689"},"children":["ZNF689"]},{"name":"span","attribs":{"title":"FKBP5"},"children":["FKBP5"]},{"name":"span","attribs":{"title":"RNF215; SEC14L2"},"children":["RNF215; SEC14L2"]},{"name":"span","attribs":{"title":"MYCBP2"},"children":["MYCBP2"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"SYBU"},"children":["SYBU"]},{"name":"span","attribs":{"title":"CTNNBL1"},"children":["CTNNBL1"]},{"name":"span","attribs":{"title":"PACSIN1; SPDEF; ILRUN; AL451165.2"},"children":["PACSIN1; SPDEF; ILRUN; AL45..."]},{"name":"span","attribs":{"title":"MCCC2"},"children":["MCCC2"]},{"name":"span","attribs":{"title":"TMEM17"},"children":["TMEM17"]},{"name":"span","attribs":{"title":"RCL1"},"children":["RCL1"]},{"name":"span","attribs":{"title":"FKBP5"},"children":["FKBP5"]},{"name":"span","attribs":{"title":"PPIF"},"children":["PPIF"]},{"name":"span","attribs":{"title":"NR4A1; ATG101; KRT80"},"children":["NR4A1; ATG101; KRT80"]},{"name":"span","attribs":{"title":"SORD"},"children":["SORD"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"IGSF8; PEA15; DCAF8"},"children":["IGSF8; PEA15; DCAF8"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"MKNK1; MOB3C; ATPAF1; EFCAB14"},"children":["MKNK1; MOB3C; ATPAF1; EFCAB14"]},{"name":"span","attribs":{"title":"PRKAA2"},"children":["PRKAA2"]},{"name":"span","attribs":{"title":"AC104825.1; ACOX3; TRMT44"},"children":["AC104825.1; ACOX3; TRMT44"]},{"name":"span","attribs":{"title":"CRAMP1; IFT140; JPT2; AL031009.1; MAPK8IP3"},"children":["CRAMP1; IFT140; JPT2; AL031..."]},{"name":"span","attribs":{"title":"KAT6B"},"children":["KAT6B"]},{"name":"span","attribs":{"title":"CRAMP1; IFT140; JPT2; AL031009.1; MAPK8IP3"},"children":["CRAMP1; IFT140; JPT2; AL031..."]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"ATP6V0E1"},"children":["ATP6V0E1"]},{"name":"span","attribs":{"title":"PDZD2"},"children":["PDZD2"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"KIAA0232; TBC1D14; CCDC96; TADA2B"},"children":["KIAA0232; TBC1D14; CCDC96; ..."]},{"name":"span","attribs":{"title":"C1orf21; EDEM3"},"children":["C1orf21; EDEM3"]},{"name":"span","attribs":{"title":"VIPR1; VIPR1-AS1; SEC22C; SS18L2; NKTR; AC006059.1"},"children":["VIPR1; VIPR1-AS1; SEC22C; S..."]},{"name":"span","attribs":{"title":"AHCY"},"children":["AHCY"]},{"name":"span","attribs":{"title":"FLNB; FLNB-AS1; ABHD6"},"children":["FLNB; FLNB-AS1; ABHD6"]},{"name":"span","attribs":{"title":"C12orf49"},"children":["C12orf49"]},{"name":"span","attribs":{"title":"PDZD2"},"children":["PDZD2"]},{"name":"span","attribs":{"title":"MYCBP2"},"children":["MYCBP2"]},{"name":"span","attribs":{"title":"ARHGEF37"},"children":["ARHGEF37"]},{"name":"span","attribs":{"title":"SPRED1"},"children":["SPRED1"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"MPC2; DCAF6"},"children":["MPC2; DCAF6"]},{"name":"span","attribs":{"title":"KLHL15; EIF2S3; ZFX"},"children":["KLHL15; EIF2S3; ZFX"]},{"name":"span","attribs":{"title":"IGSF8; PEA15; DCAF8"},"children":["IGSF8; PEA15; DCAF8"]},{"name":"span","attribs":{"title":"TEX2"},"children":["TEX2"]},{"name":"span","attribs":{"title":"FOXP2"},"children":["FOXP2"]},{"name":"span","attribs":{"title":"RASGEF1A"},"children":["RASGEF1A"]},{"name":"span","attribs":{"title":"VSTM2L; TTI1; RPRD1B"},"children":["VSTM2L; TTI1; RPRD1B"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"CD9; PLEKHG6; TNFRSF1A; SCNN1A"},"children":["CD9; PLEKHG6; TNFRSF1A; SCNN1A"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"TTC21A; WDR48; GORASP1; CSRNP1; AC092053.4"},"children":["TTC21A; WDR48; GORASP1; CSR..."]},{"name":"span","attribs":{"title":"FAM155A"},"children":["FAM155A"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"NPNT"},"children":["NPNT"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"CNTNAP2"},"children":["CNTNAP2"]},{"name":"span","attribs":{"title":"PACSIN1; SPDEF; ILRUN; AL451165.2"},"children":["PACSIN1; SPDEF; ILRUN; AL45..."]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"MTCL1; RAB12"},"children":["MTCL1; RAB12"]},{"name":"span","attribs":{"title":"NF1"},"children":["NF1"]},{"name":"span","attribs":{"title":"EHF"},"children":["EHF"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"UBE2Q2; FBXO22"},"children":["UBE2Q2; FBXO22"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"WWC2"},"children":["WWC2"]},{"name":"span","attribs":{"title":"SASH1"},"children":["SASH1"]},{"name":"span","attribs":{"title":"UQCC2"},"children":["UQCC2"]},{"name":"span","attribs":{"title":"VWA8"},"children":["VWA8"]},{"name":"span","attribs":{"title":"NRP1"},"children":["NRP1"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"GCLC"},"children":["GCLC"]},{"name":"span","attribs":{"title":"FGGY"},"children":["FGGY"]},{"name":"span","attribs":{"title":"ATP6V0D1; ZDHHC1; HSD11B2; AC009061.2; RIPOR1; CTCF"},"children":["ATP6V0D1; ZDHHC1; HSD11B2; ..."]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"ANO2"},"children":["ANO2"]},{"name":"span","attribs":{"title":"CUEDC1"},"children":["CUEDC1"]},{"name":"span","attribs":{"title":"CNNM1"},"children":["CNNM1"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"CRAT"},"children":["CRAT"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"ANG"},"children":["ANG"]},{"name":"span","attribs":{"title":"ELMO1"},"children":["ELMO1"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"NUFIP1"},"children":["NUFIP1"]},{"name":"span","attribs":{"title":"BLOC1S6; SQOR"},"children":["BLOC1S6; SQOR"]},{"name":"span","attribs":{"title":"PARD6B; BCAS4; ADNP; ADNP-AS1; DPM1"},"children":["PARD6B; BCAS4; ADNP; ADNP-A..."]},{"name":"span","attribs":{"title":"SLITRK6"},"children":["SLITRK6"]},{"name":"span","attribs":{"title":"MCCC2"},"children":["MCCC2"]},{"name":"span","attribs":{"title":"CHPT1"},"children":["CHPT1"]},{"name":"span","attribs":{"title":"RNF215"},"children":["RNF215"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"PRLR"},"children":["PRLR"]},{"name":"span","attribs":{"title":"ATP6AP2"},"children":["ATP6AP2"]},{"name":"span","attribs":{"title":"CAPZB; MICOS10"},"children":["CAPZB; MICOS10"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"NAMPT"},"children":["NAMPT"]},{"name":"span","attribs":{"title":"PRKAB2; PDIA3P1; FMO5"},"children":["PRKAB2; PDIA3P1; FMO5"]},{"name":"span","attribs":{"title":"DLG2"},"children":["DLG2"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"PDCD6"},"children":["PDCD6"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"RCL1"},"children":["RCL1"]},{"name":"span","attribs":{"title":"MEGF9"},"children":["MEGF9"]},{"name":"span","attribs":{"title":"LRBA"},"children":["LRBA"]},{"name":"span","attribs":{"title":"MYCBP2"},"children":["MYCBP2"]},{"name":"span","attribs":{"title":"TMSB4X"},"children":["TMSB4X"]},{"name":"span","attribs":{"title":"MTMR9"},"children":["MTMR9"]},{"name":"span","attribs":{"title":"TTC12"},"children":["TTC12"]},{"name":"span","attribs":{"title":"PTPRJ"},"children":["PTPRJ"]},{"name":"span","attribs":{"title":"CDK5RAP2; MEGF9"},"children":["CDK5RAP2; MEGF9"]},{"name":"span","attribs":{"title":"SLC1A4"},"children":["SLC1A4"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"RAB31"},"children":["RAB31"]},{"name":"span","attribs":{"title":"PEX14"},"children":["PEX14"]},{"name":"span","attribs":{"title":"MYBL1; VCPIP1"},"children":["MYBL1; VCPIP1"]},{"name":"span","attribs":{"title":"GFRA1"},"children":["GFRA1"]},{"name":"span","attribs":{"title":"RYBP"},"children":["RYBP"]},{"name":"span","attribs":{"title":"TIGD6"},"children":["TIGD6"]},{"name":"span","attribs":{"title":"CAPZB; MICOS10"},"children":["CAPZB; MICOS10"]},{"name":"span","attribs":{"title":"LARS2"},"children":["LARS2"]},{"name":"span","attribs":{"title":"TPD52"},"children":["TPD52"]},{"name":"span","attribs":{"title":"FAM104A"},"children":["FAM104A"]},{"name":"span","attribs":{"title":"EIF2S3"},"children":["EIF2S3"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"DLG2"},"children":["DLG2"]},{"name":"span","attribs":{"title":"TRRAP"},"children":["TRRAP"]},{"name":"span","attribs":{"title":"SULT2B1"},"children":["SULT2B1"]},{"name":"span","attribs":{"title":"YTHDF1"},"children":["YTHDF1"]},{"name":"span","attribs":{"title":"HIBCH"},"children":["HIBCH"]},{"name":"span","attribs":{"title":"CNNM1"},"children":["CNNM1"]},{"name":"span","attribs":{"title":""},"children":[""]},{"name":"span","attribs":{"title":"DUSP16; CREBL2"},"children":["DUSP16; CREBL2"]},{"name":"span","attribs":{"title":"NEDD4L"},"children":["NEDD4L"]},{"name":"span","attribs":{"title":"PARD6B; BCAS4; ADNP; ADNP-AS1; DPM1"},"children":["PARD6B; BCAS4; ADNP; ADNP-A..."]},{"name":"span","attribs":{"title":"UBA6; UBA6-AS1"},"children":["UBA6; UBA6-AS1"]},{"name":"span","attribs":{"title":"ADK; KAT6B"},"children":["ADK; KAT6B"]},{"name":"span","attribs":{"title":"WHAMM; HOMER2; RAMAC; C15orf40; BTBD1"},"children":["WHAMM; HOMER2; RAMAC; C15or..."]},{"name":"span","attribs":{"title":"KAT6B; SAMD8"},"children":["KAT6B; SAMD8"]},{"name":"span","attribs":{"title":"AMPD2; GSTM4; GSTM3"},"children":["AMPD2; GSTM4; GSTM3"]}],"minWidth":132}],"filterable":true,"defaultPageSize":10,"showPageSizeOptions":true,"pageSizeOptions":[10,20,50,200],"borderless":true,"theme":{"style":{"fontSize":12}},"dataKey":"3f9c3e29b97c3418e72a098991697d54"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
</div>
<div id="summary-plots" class="section level2 tabset">
<h2 class="tabset">Summary Plots</h2>
<pre class="r"><code>profile_width &lt;- 5e3
n_bins &lt;- 100
bwfl &lt;- file.path(
    macs2_path, glue(&quot;{target}_{treat_levels}_merged_treat_pileup.bw&quot;)
  ) %&gt;% 
  BigWigFileList() %&gt;% 
  setNames(treat_levels) 
sig_ranges &lt;- merged_results %&gt;% 
  filter(!!sym(fdr_column) &lt; fdr_alpha) %&gt;% 
  colToRanges(&quot;keyval_range&quot;) %&gt;% 
  splitAsList(f = .$direction) %&gt;% 
  .[vapply(., length, integer(1)) &gt; 0] 
fc_heat &lt;- names(sig_ranges) %&gt;% 
  lapply(
    function(x) {
      glue(
        &quot;
        *Heatmap and histogram for all regions considered to show evidence of 
        {ifelse(x == &#39;Up&#39;, &#39;increased&#39;, &#39;decreased&#39;)} {target} binding in 
        response to {treat_levels[[2]]} treatment. A total of 
        {comma(length(sig_ranges[[x]]))} regions were in this group.
        &quot;,
        ifelse(
          length(sig_ranges[[x]]) &gt; 2e4, 
          &quot; Due to the large number of regions, these were randomly down-sampled to 20,000 for viable plotting.&quot;,
          &quot;&quot;
        ),
        &quot;*&quot;
      )
    }
  ) %&gt;% 
  setNames(names(sig_ranges))
sig_profiles &lt;- lapply(sig_ranges, function(x) NULL)
for (i in names(sig_profiles)) {
  ## Restrict to 20,000. Plots work poorly above this number.
  ## Randomly sample
  set.seed(threads)
  temp_gr &lt;- granges(sig_ranges[[i]])
  n &lt;- length(temp_gr)
  if (n &gt; 2e4) temp_gr &lt;- temp_gr[sort(sample.int(n, 2e4))]
  sig_profiles[[i]] &lt;- getProfileData(
    bwfl, temp_gr, upstream = profile_width / 2, bins = n_bins,
    BPPARAM = bpparam()
  )
  rm(temp_gr)
}
profile_heatmaps &lt;- sig_profiles %&gt;% 
  parallel::mclapply(
    plotProfileHeatmap,
    profileCol = &quot;profile_data&quot;, 
    colour = &quot;name&quot;,
    xLab = &quot;Distance from Centre (bp)&quot;,
    fillLab = &quot;logCPM&quot;,
    mc.cores = length(sig_profiles)
  ) 
fill_range &lt;- profile_heatmaps %&gt;% 
  lapply(function(x) x$data[,&quot;score&quot;]) %&gt;% 
  unlist() %&gt;%
  range()
sidey_range &lt;- profile_heatmaps %&gt;% 
  lapply(function(x) x$layers[[3]]$data$y) %&gt;% 
  unlist() %&gt;% 
  range()
profile_heatmaps &lt;- profile_heatmaps %&gt;% 
  lapply(
    function(x) {
      suppressWarnings(
        x + 
          scale_fill_gradientn(colours = colours$heatmaps, limits = fill_range) +
          scale_xsidey_continuous(limits = sidey_range) +
          scale_colour_manual(values = treat_colours) +
          labs(
            x = &quot;Distance from Centre (bp)&quot;,
            fill = &quot;logCPM&quot;, 
            colour = &quot;Treat&quot;
          ) +
          theme(
            strip.text.y = element_text(angle = 0)
          )
      )
    }
  )</code></pre>
<div id="ma-plot" class="section level3">
<h3>MA Plot</h3>
<pre class="r"><code>if (!&quot;Ambiguous&quot; %in% levels(merged_results$status)) 
  direction_colours &lt;- direction_colours[names(direction_colours) != &quot;Ambiguous&quot;]
merged_results %&gt;% 
  as_tibble() %&gt;%
  ggplot(aes(AveExpr, logFC)) +
  geom_point(aes(colour = status), alpha = 0.5) +
  geom_smooth(se = FALSE, method = &quot;gam&quot;, formula = y ~ s(x, bs = &quot;cs&quot;)) +
  geom_hline(yintercept = 0) +
  geom_label_repel(
    aes(y = 1.1*logFC, label = annot, colour = status),
    data= . %&gt;%
      dplyr::filter(logFC == max(logFC) | logFC == min(logFC)) %&gt;%
      mutate(
        detected = gene_name %&gt;%
          lapply(paste, collapse = &quot;, &quot;) %&gt;%
          unlist() %&gt;%
          str_replace(&quot;^NA$&quot;, &quot;&quot;) %&gt;%
          str_wrap(width = 40),
        range = ifelse(detected == &quot;&quot;, range, paste0(range, &quot;\n&quot;)),
        annot = glue(&quot;{range}{detected}&quot;)
      ),
    fill = rgb(1, 1, 1, 0.2),
    size = 3,
    show.legend = FALSE
  ) +
  scale_colour_manual(values = direction_colours) +
  labs(
    x = &quot;Ave Signal (logCPM)&quot;,
    colour = &quot;Status&quot;
  ) </code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-ma-1.png" alt="*MA plot showing the status of each window under consideration. The two most extreme regions are labelled by region and any associated genes, whilst the overall pattern of association between signal level (logCPM) and changed signal (logFC) is shown as the blue curve.*" width="960" />
<p class="caption">
<em>MA plot showing the status of each window under consideration. The
two most extreme regions are labelled by region and any associated
genes, whilst the overall pattern of association between signal level
(logCPM) and changed signal (logFC) is shown as the blue curve.</em>
</p>
</div>
</div>
<div id="volcano-plot" class="section level3">
<h3>Volcano Plot</h3>
<pre class="r"><code>merged_results %&gt;% 
  as_tibble() %&gt;% 
  ggplot(aes(logFC, -log10(P.Value))) +
  geom_point(aes(colour = status), alpha = 0.6) +
  geom_label_repel(
    aes(label = annot, colour = status),
    data= . %&gt;%
      arrange(sign(logFC)*log10(P.Value)) %&gt;% 
      dplyr::slice(1:2) %&gt;% 
      dplyr::filter(logFC &gt; 1) %&gt;%
      mutate(
        detected = gene_name %&gt;%
          lapply(paste, collapse = &quot;, &quot;) %&gt;%
          unlist() %&gt;%
          str_replace(&quot;^NA$&quot;, &quot;&quot;) %&gt;%
          str_wrap(width = 40),
        range = ifelse(detected == &quot;&quot;, range, paste0(range, &quot;\n&quot;)),
        annot = glue(&quot;{range}{detected}&quot;)
      ),
    fill = rgb(1, 1, 1, 0.2),
    size = 3,
    show.legend = FALSE
  ) +
  geom_label_repel(
    aes(label = annot, colour = status),
    data= . %&gt;%
      arrange(-sign(logFC)*log10(P.Value)) %&gt;% 
      dplyr::slice(1:2) %&gt;% 
      dplyr::filter(logFC &lt; -1) %&gt;% 
      mutate(
        detected = gene_name %&gt;%
          lapply(paste, collapse = &quot;, &quot;) %&gt;%
          unlist() %&gt;%
          str_replace(&quot;^NA$&quot;, &quot;&quot;) %&gt;%
          str_wrap(width = 40),
        range = ifelse(detected == &quot;&quot;, range, paste0(range, &quot;\n&quot;)),
        annot = glue(&quot;{range}{detected}&quot;)
      ),
    fill = rgb(1, 1, 1, 0.2),
    size = 3,
    show.legend = FALSE
  ) +
  scale_colour_manual(values = direction_colours) +
  labs(
    x = &quot;logFC&quot;,
    y = expression(paste(-log[10], &quot;p&quot;)),
    colour = &quot;Status&quot;
  )</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-volcano-1.png" alt="*Volcano plot showing regions with evidence of differential H3K27ac binding. The most significant regions are labelled along with any genes these regions are mapped to.*" width="960" />
<p class="caption">
<em>Volcano plot showing regions with evidence of differential H3K27ac
binding. The most significant regions are labelled along with any genes
these regions are mapped to.</em>
</p>
</div>
</div>
<div id="heatmap-increased-binding" class="section level3">
<h3>Heatmap: Increased Binding</h3>
<pre class="r"><code>profile_heatmaps$Up + guides(colour = &quot;none&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-up-heatmap-1.png" alt="*Heatmap and histogram for all regions considered to show evidence of 
increased H3K27ac binding in 
response to E2DHT treatment. A total of 
241 regions were in this group.
*" width="960" />
<p class="caption">
<em>Heatmap and histogram for all regions considered to show evidence of
increased H3K27ac binding in response to E2DHT treatment. A total of 241
regions were in this group. </em>
</p>
</div>
</div>
<div id="heatmap-decreased-binding" class="section level3">
<h3>Heatmap: Decreased Binding</h3>
<pre class="r"><code>profile_heatmaps$Down + guides(colour = &quot;none&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-down-heatmap-1.png" alt="*Heatmap and histogram for all regions considered to show evidence of 
decreased H3K27ac binding in 
response to E2DHT treatment. A total of 
2 regions were in this group.
*" width="960" />
<p class="caption">
<em>Heatmap and histogram for all regions considered to show evidence of
decreased H3K27ac binding in response to E2DHT treatment. A total of 2
regions were in this group. </em>
</p>
</div>
</div>
<div id="results-by-chromosome" class="section level3">
<h3>Results By Chromosome</h3>
<pre class="r"><code>merged_results %&gt;% 
  select(status) %&gt;% 
  as.data.frame() %&gt;% 
  mutate(
    merge_status = fct_collapse(
      status,
      Up = &quot;Up&quot;,
      Down = &quot;Down&quot;,
      `Ambiguous/Unchanged` = intersect(c(&quot;Ambiguous&quot;, &quot;Unchanged&quot;), levels(status))
    )
  ) %&gt;% 
  ggplot(aes(seqnames, fill = status)) +
  geom_bar() +
  facet_grid(merge_status~., scales = &quot;free_y&quot;) +
  scale_y_continuous(expand = expansion(c(0, 0.05)), labels = comma) +
  scale_fill_manual(values =  direction_colours) +
  labs(
    x = &quot;Chromosome&quot;, y = &quot;Number of Windows&quot;, 
    fill = &quot;Status&quot;, alpha = &quot;Macs2 Peak&quot;
  )</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-res-by-chromosome-1.png" alt="*Results for differential binding separated by chromosome*" width="960" />
<p class="caption">
<em>Results for differential binding separated by chromosome</em>
</p>
</div>
</div>
<div id="signal-by-region" class="section level3">
<h3>Signal By Region</h3>
<pre class="r"><code>merged_results %&gt;% 
  select(AveExpr, logFC, region) %&gt;% 
  as_tibble() %&gt;% 
  mutate(
    x = fct_relabel(
      region, str_replace_all, pattern = &quot; \\(&quot;, replacement = &quot;\n(&quot;
    ),
  ) %&gt;% 
  pivot_longer(cols = c(&quot;AveExpr&quot;, &quot;logFC&quot;)) %&gt;% 
  ggplot(aes(x, value, fill = region)) +
  geom_boxplot() +
  facet_grid(
    name~., scales = &quot;free&quot;, labeller = facet_labeller, switch = &quot;y&quot;
  ) +
  scale_fill_manual(values = region_colours) +
  labs(x = &quot;Region&quot;, y = c(), fill = &quot;Overlaps&quot;) </code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/boxplot-regions-1.png" alt="*Distributions of overall signal (logCPM) and changed signal (logFC) for each genomic region.*" width="960" />
<p class="caption">
<em>Distributions of overall signal (logCPM) and changed signal (logFC)
for each genomic region.</em>
</p>
</div>
</div>
<div id="differential-binding-by-region" class="section level3">
<h3>Differential Binding By Region</h3>
<pre class="r"><code>merged_results %&gt;% 
  subset(status %in% c(&quot;Up&quot;, &quot;Down&quot;)) %&gt;% 
  select(status, region) %&gt;% 
  as_tibble() %&gt;% 
  mutate(
    x = fct_relabel(
      region, str_replace_all, pattern = &quot; \\(&quot;, replacement = &quot;\n(&quot;
    )
  ) %&gt;% 
  group_by(x, region, status) %&gt;% 
  tally() %&gt;% 
  arrange(region, desc(status)) %&gt;% 
  mutate(y = cumsum(n)) %&gt;% 
  ggplot(
    aes(x, n, fill = status)
  ) +
  geom_col() +
  geom_label(
    aes(y = y - 0.5*n, label = comma(n, 1), colour = status),
    data = . %&gt;% ungroup() %&gt;% dplyr::filter(n &gt; 0.05*max(y)),
    fill = &quot;white&quot;, alpha = 0.8,
    show.legend = FALSE
  ) +
  geom_text(
    aes(y = y1, label = comma(y, 1)),
    data = . %&gt;% 
      dplyr::filter(y == max(y)) %&gt;% 
      ungroup() %&gt;% 
      mutate(y1 = y + 0.03*max(y))
  ) +
  scale_y_continuous(expand = expansion(c(0, 0.05)), labels = comma) +
  scale_fill_manual(values = direction_colours) +
  scale_colour_manual(values = direction_colours) +
  labs(x = &quot;Region&quot;, y = &quot;Merged Windows&quot;, fill = &quot;Status&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-db-regions-1.png" alt="*Merged windows considered as showing differential H3K27ac binding across all genomic regions.*" width="960" />
<p class="caption">
<em>Merged windows considered as showing differential H3K27ac binding
across all genomic regions.</em>
</p>
</div>
</div>
<div id="signal-by-feature" class="section level3">
<h3>Signal By Feature</h3>
<pre class="r"><code>merged_results %&gt;% 
  select(AveExpr, logFC, feature) %&gt;% 
  as_tibble() %&gt;% 
  pivot_longer(cols = c(&quot;AveExpr&quot;, &quot;logFC&quot;)) %&gt;% 
  ggplot(aes(feature, value, fill = feature)) +
  geom_boxplot() +
  facet_grid(
    name~., scales = &quot;free&quot;, labeller = facet_labeller, switch = &quot;y&quot;
  ) +
  scale_fill_manual(values = feature_colours) +
  labs(x = &quot;Region&quot;, y = c(), fill = &quot;Overlaps&quot;) </code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/boxplot-features-1.png" alt="*Distributions of overall signal (logCPM) and changed signal (logFC) for each external features.*" width="960" />
<p class="caption">
<em>Distributions of overall signal (logCPM) and changed signal (logFC)
for each external features.</em>
</p>
</div>
</div>
<div id="differential-binding-by-feature" class="section level3">
<h3>Differential Binding By Feature</h3>
<pre class="r"><code>merged_results %&gt;% 
  subset(status %in% c(&quot;Up&quot;, &quot;Down&quot;)) %&gt;% 
  select(status, feature) %&gt;% 
  as_tibble() %&gt;% 
  group_by(feature, status) %&gt;% 
  tally() %&gt;% 
  arrange(feature, desc(status)) %&gt;% 
  mutate(y = cumsum(n)) %&gt;% 
  ungroup() %&gt;% 
  ggplot(
    aes(feature, n, fill = status)
  ) +
  geom_col() +
  geom_label(
    aes(y = y - 0.5*n, label = comma(n, 1), colour = status),
    data = . %&gt;%  dplyr::filter(n &gt; 0.05*max(y)),
    fill = &quot;white&quot;, alpha = 0.8,
    show.legend = FALSE
  ) +
  geom_text(
    aes(y = y1, label = comma(y, 1)),
    data = . %&gt;% 
      group_by(feature) %&gt;% 
      dplyr::filter(y == max(y)) %&gt;% 
      ungroup() %&gt;% 
      mutate(y1 = y + 0.03*max(y))
  ) +
  scale_y_continuous(expand = expansion(c(0, 0.05)), labels = comma) +
  scale_fill_manual(values = direction_colours) +
  scale_colour_manual(values = direction_colours) +
  labs(x = &quot;Feature&quot;, y = &quot;Merged Windows&quot;, fill = &quot;Status&quot;)</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-db-features-1.png" alt="*Merged windows considered as showing differential H3K27ac binding across all external features.*" width="960" />
<p class="caption">
<em>Merged windows considered as showing differential H3K27ac binding
across all external features.</em>
</p>
</div>
</div>
<div id="differential-binding-by-region-and-feature"
class="section level3">
<h3>Differential Binding By Region And Feature</h3>
<pre class="r"><code>df &lt;- merged_results %&gt;% 
  as_tibble() %&gt;% 
  dplyr::filter(status %in% c(&quot;Up&quot;, &quot;Down&quot;)) %&gt;% 
  droplevels() %&gt;% 
  group_by(region, feature, status) %&gt;% 
  summarise(n = dplyr::n(), .groups = &quot;drop_last&quot;) %&gt;% 
  mutate(total = sum(n)) %&gt;% 
  ungroup() %&gt;% 
  mutate(
    region = fct_relabel(region, str_wrap, width = 20),
    x0 = as.integer(region),
    y0 = as.integer(feature),
    r = sqrt(total / sum(total)),
    r = 0.5* r / max(r)
  ) 
df %&gt;% 
  ggplot() +
  ggforce::stat_pie(
    aes(x0 = x0, y0 = y0, r0 = 0, r = r, fill = status, amount = n)
  ) +
  coord_equal() +
  geom_label(
    aes(x0, y0, label = comma(total)),
    data = . %&gt;% 
      distinct(x0, y0, total) %&gt;% 
      dplyr::filter(total &gt; 0.01 * sum(total)),
    alpha = 0.6,
    inherit.aes = FALSE,
    show.legend = FALSE
  ) +
  geom_xsidecol(aes(x = x0, y = n, fill = status)) +
  geom_ysidecol(aes(x = n, y = y0, fill = status)) +
  scale_x_continuous(
    breaks = seq_along(levels(df$region)), labels = levels(df$region)
  ) +
  scale_y_continuous(
    breaks = seq_along(levels(df$feature)), labels = levels(df$feature)
  ) +
  scale_fill_manual(values = direction_colours) +
  scale_xsidey_continuous(name = c(), expand = expansion(c(0, 0.1))) +
  scale_ysidex_continuous(name = c(), expand = expansion(c(0, 0.1))) +
  labs(
    x = &quot;Region&quot;, y = &quot;Feature&quot;, fill = &quot;Status&quot;
  ) +
  theme(
    legend.position = &quot;right&quot;,
    ggside.panel.scale = 0.2
  )</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/scatterpie-region-feature-1.png" alt="*Comparison of changes in H3K27ac binding separated by gene-centric region and external features provided in enhancer_atlas_2.0_zr75.gtf.gz*" width="960" />
<p class="caption">
<em>Comparison of changes in H3K27ac binding separated by gene-centric
region and external features provided in
enhancer_atlas_2.0_zr75.gtf.gz</em>
</p>
</div>
</div>
</div>
<div id="inspection-of-top-ranked-regions"
class="section level2 tabset">
<h2 class="tabset">Inspection of Top-Ranked Regions</h2>
<pre class="r"><code>## Define a GRL with the key ranges.
## Then we can step through it &amp; make all of the requisite plots
grl_to_plot &lt;- GRangesList(
  top_up_by_fdr = merged_results %&gt;% 
    filter(logFC &gt; 0) %&gt;% 
    filter(P.Value == min(P.Value)) %&gt;% 
    filter(!!sym(fdr_column) &lt; fdr_alpha),
  top_up_by_lfc = merged_results %&gt;% 
    filter(logFC == max(logFC), logFC &gt; 0, !!sym(fdr_column) &lt; fdr_alpha),
  top_up_by_expr = merged_results %&gt;% 
    filter(logFC &gt; 0, !!sym(fdr_column) &lt; fdr_alpha) %&gt;% 
    filter(AveExpr == max(AveExpr)),
  top_up_no_macs2 = merged_results %&gt;% 
    filter(logFC &gt; 0, !!sym(fdr_column) &lt; fdr_alpha, !overlaps_ref) %&gt;% 
    arrange(!!sym(fdr_column)) %&gt;% 
    arrange(P.Value),
  top_down_by_fdr = merged_results %&gt;% 
    filter(logFC &lt; 0) %&gt;% 
    filter(P.Value == min(P.Value)) %&gt;% 
    filter(!!sym(fdr_column) &lt; fdr_alpha),
  top_down_by_lfc = merged_results %&gt;% 
    filter(logFC == min(logFC), logFC &lt; 0, !!sym(fdr_column) &lt; fdr_alpha),
  top_down_by_expr = merged_results %&gt;% 
    filter(logFC &lt; 0, !!sym(fdr_column) &lt; fdr_alpha) %&gt;% 
    filter(AveExpr == max(AveExpr)),
  top_up_no_macs2 = merged_results %&gt;% 
    filter(logFC &lt; 0, !!sym(fdr_column) &lt; fdr_alpha, !overlaps_ref) %&gt;% 
    arrange(P.Value)
) %&gt;% 
  unlist() %&gt;% 
  .[!duplicated(.)] %&gt;%
  splitAsList(names(.)) %&gt;% 
  lapply(setNames, c()) %&gt;% 
  lapply(function(x) x[1]) %&gt;%
  GRangesList()
fh &lt;- max(4, 1 + 3 * ceiling(length(grl_to_plot) / 3)) %&gt;% 
  min(knitr::opts_chunk$get(&quot;fig.height&quot;))</code></pre>
<pre class="r"><code>## The coverage
bwfl &lt;- list2(
  &quot;{target}&quot; := file.path(
    macs2_path, glue(&quot;{target}_{treat_levels}_merged_treat_pileup.bw&quot;)
  ) %&gt;% 
    BigWigFileList() %&gt;% 
    setNames(treat_levels)
)
line_col &lt;- list2(&quot;{target}&quot; := treat_colours)
## Coverage annotations
annot &lt;- merged_results %&gt;% 
  splitAsList(.$status) %&gt;% 
  .[vapply(., length, integer(1)) &gt; 0] %&gt;% 
  endoapply(granges) %&gt;% 
  list() %&gt;% 
  setNames(target)
## Coverage y-limits
ind &lt;- filtered_counts %&gt;% 
  assay(&quot;counts&quot;) %&gt;% 
  apply(MARGIN = 2, which.max) %&gt;%
  unique()
max_ranges &lt;- rowRanges(filtered_counts[ind])
y_lim &lt;- bwfl[[target]] %&gt;%
  lapply(import.bw, which = max_ranges) %&gt;%
  lapply(function(x) c(0, max(x$score))) %&gt;% 
  unlist() %&gt;% 
  range() %&gt;% 
  list() %&gt;% 
  setNames(target)

## The features track
feat_gr &lt;- gene_regions %&gt;% 
  lapply(granges) %&gt;% 
  GRangesList()
feature_colours &lt;- colours$regions
if (has_features) {
  feat_gr &lt;- list(Regions = feat_gr)
  feat_gr$Features &lt;- splitAsList(external_features, external_features$feature)
  feature_colours &lt;- list(
    Regions = unlist(colours$regions),
    Features = unlist(colours$features)
  )
}

## The genes track
hfgc_genes &lt;- trans_models
gene_col &lt;- &quot;grey&quot;
if (has_rnaseq){
  if (!is.na(rna_lfc_col) &amp; !is.na(rna_fdr_col)) {
    hfgc_genes &lt;- trans_models %&gt;% 
      mutate(
        status = case_when(
          !gene %in% rnaseq$gene_id ~ &quot;Undetected&quot;,
          gene %in% dplyr::filter(
            rnaseq, !!sym(rna_lfc_col) &gt; 0, !!sym(rna_fdr_col) &lt; fdr_alpha
          )$gene_id ~ &quot;Up&quot;,
          gene %in% dplyr::filter(
            rnaseq, !!sym(rna_lfc_col) &lt; 0, !!sym(rna_fdr_col) &lt; fdr_alpha
          )$gene_id ~ &quot;Down&quot;,
          gene %in% dplyr::filter(
            rnaseq, !!sym(rna_fdr_col) &gt;= fdr_alpha
          )$gene_id ~ &quot;Unchanged&quot;,
        )
      ) %&gt;% 
      splitAsList(.$status) %&gt;% 
      lapply(select, -status) %&gt;% 
      GRangesList()
    gene_col &lt;- colours$direction %&gt;% 
      setNames(str_to_title(names(.)))
  }
}

## External Coverage (Optional)
if (!is.null(config$external$coverage)) {
  ext_cov_path &lt;- config$external$coverage %&gt;% 
    lapply(unlist) %&gt;% 
    lapply(function(x) setNames(here::here(x), names(x)))
  bwfl &lt;- c(
    bwfl[target],
    lapply(ext_cov_path, function(x) BigWigFileList(x) %&gt;% setNames(names(x)))
  )
  line_col &lt;- c(
    line_col[target],
    ext_cov_path %&gt;% 
      lapply(
        function(x) {
          missing &lt;- setdiff(names(x), names(colours$treat))
          cmn &lt;- intersect(names(x), names(colours$treat))
          col &lt;- setNames(character(length(names(x))), names(x))
          if (length(cmn) &gt; 0) col[cmn] &lt;- colours$treat[cmn]
          if (length(missing) &gt; 0) 
            col[missing] &lt;- hcl.colors(
              max(5, length(missing)), &quot;Zissou 1&quot;)[seq_along(missing)]
          col
        }
      )
  )
  
  y_ranges &lt;- grl_to_plot %&gt;% 
    unlist() %&gt;% 
    granges() %&gt;% 
    resize(w = 10 * width(.), fix = &#39;center&#39;)
  
  y_lim &lt;- c(
    y_lim[target],
    bwfl[names(bwfl) != target] %&gt;% 
      lapply(
        function(x) {
          GRangesList(lapply(x, import.bw, which = y_ranges)) %&gt;% 
            unlist() %&gt;% 
            filter(score == max(score)) %&gt;% 
            mcols() %&gt;% 
            .[[&quot;score&quot;]] %&gt;% 
            c(0) %&gt;% 
            range()
        }
      )
  )
  
}</code></pre>
<pre class="r"><code>makeCaption &lt;- function(.gr) {
  if (is.null(.gr)) return(NULL)
  dir &lt;- ifelse(.gr$direction == &quot;Down&quot;, &#39;decreased&#39;, &#39;increased&#39;)
  reg &lt;- case_when(
    str_detect(.gr$region, &quot;Inter&quot;) ~ paste(&quot;an&quot;, .gr$region, &quot;region&quot;),
    str_detect(.gr$region, &quot;Upstream&quot;) ~ paste(&quot;an&quot;, .gr$region),
    str_detect(.gr$region, &quot;(Ex|Intr)on&quot;) ~ paste(&quot;an&quot;, .gr$region),
    str_detect(.gr$region, &quot;^Prom&quot;) ~ paste(&quot;a&quot;, .gr$region)
  )
  feat &lt;- c()
  if (has_features) feat &lt;- case_when(
    str_detect(.gr$feature, &quot;^[AEIOU]&quot;) ~ paste(&quot;an&quot;, .gr$feature),
    !str_detect(.gr$feature, &quot;^[AEIOU]&quot;) ~ paste(&quot;a&quot;, .gr$feature)
  )
  gn &lt;- unlist(.gr$gene_name)
  fdr &lt;- mcols(.gr)[[fdr_column]]
  fdr &lt;- ifelse(
    fdr &lt; 0.001, sprintf(&#39;%.2e&#39;, fdr), sprintf(&#39;%.3f&#39;, fdr)
  )
  cp &lt;- c(
    glue(
      &quot;*The {width(.gr)}bp region showing {dir} {target} binding in response to &quot;, 
      &quot;{treat_levels[[2]]} treatment (FDR = {fdr}). &quot;,
      &quot;The range mostly overlapped with {reg}, with all &quot;,
      &quot;defined regions shown as a contiguous bar. &quot;,
      ifelse(
        has_features,
        glue(
          &quot;Using the features supplied in {basename(config$external$features)}, &quot;,
          &quot;this mostly overlapped {feat}, shown as a separate block &quot;,
          &quot;with the gene-centric regions. &quot;
        ),
        glue(&quot;&quot;)
      ),
    ),
    ifelse(
      .gr$overlaps_ref,
      paste(
        &quot;A union peak overlapping this region was identified by&quot;,
        &quot;`macs2 callpeak` when using merged samples.&quot;
      ),
      &quot;No union peak was identified using `macs2 callpeak`.&quot;
    ),
    ifelse(
      length(gn) &gt; 0,
      paste0(
        &quot;Using the above mapping strategy, this range is likely to regulate &quot;,
        collapseGenes(gn, format = &quot;&quot;)
      ),
      &quot;No genes were able to be assigned to this region.&quot;
    ),
    paste(
      &quot;For each sample, the y-axis limits represent the values from the window&quot;, 
      &quot;with the highest signal.*&quot;
    )
  )
  paste(cp, collapse = &quot; &quot;)
}</code></pre>
<div id="normalised-sliding-windows" class="section level3">
<h3>Normalised Sliding Windows</h3>
<pre class="r"><code>grl_to_plot %&gt;% 
  lapply(
    function(x){
      filtered_counts %&gt;% 
        subsetByOverlaps(x) %&gt;% 
        assay(&quot;qsmooth&quot;) %&gt;% 
        as_tibble(rownames = &quot;window&quot;) %&gt;% 
        pivot_longer(
          cols = all_of(colnames(filtered_counts)),
          names_to = &quot;sample&quot;,
          values_to = &quot;logCPM&quot;
        ) %&gt;% 
        mutate(range = as.character(x)) %&gt;% 
        list()
    }
  ) %&gt;% 
  as_tibble() %&gt;% 
  pivot_longer(everything()) %&gt;% 
  unnest(everything()) %&gt;% 
  mutate(
    direction = str_extract(name, &quot;down|up&quot;),
    by = str_extract(name, &quot;by_.+&quot;),
    across(
      all_of(c(&quot;name&quot;, &quot;direction&quot;, &quot;by&quot;)), str_sep_to_title
    ),
    name = str_remove_all(name, &quot;Top &quot;) %&gt;% 
      str_replace_all(&quot;Lfc&quot;, &quot;logFC&quot;) %&gt;% 
      str_replace_all(&quot;Fdr&quot;, &quot;FDR&quot;) %&gt;% 
      str_replace_all(&quot;Expr&quot;, &quot;Highest Signal&quot;) %&gt;% 
      str_replace_all(&quot;By (.+)&quot;, &quot;(\\1)&quot;) %&gt;% 
      str_replace_all(&quot;No Macs2&quot;, &quot;(No Macs2 Peak)&quot;),
    window = as.integer(window)
  ) %&gt;% 
  left_join(samples, by = &quot;sample&quot;) %&gt;% 
  ggplot(
    aes(window, logCPM, colour = treat, linetype = !!sym(rep_col))
  ) +
  geom_line(
    data = . %&gt;% 
      group_by(name) %&gt;% 
      dplyr::filter(max(window) &gt; 1)
  ) + 
  geom_point(
    data = . %&gt;% 
      group_by(name) %&gt;% 
      dplyr::filter(max(window) == 1),
    show.legend = FALSE
  ) + 
  facet_wrap(~name + range, scales = &quot;free_x&quot;) +
  scale_colour_manual(values = treat_colours) + 
  labs(
    x = &quot;Sliding Window\n(Unmerged)&quot;, y = &quot;SQ-Normalised logCPM&quot;, colour = &quot;Treat&quot;,
    linetype = str_sep_to_title(rep_col)
  )</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-qsmooth-top-ranges-1.png" alt="*Most highly ranked ranges for both gained and decreased H3K27ac binding in repsonse to E2DHT treatment. The smooth-quantile normalised values are shown across the initial set of sliding windows before merging. Ranges were chosen as the most extreme for FDR, Binding Strength (Signal) and logFC. Windows are shown free of the genomic context.*" width="960" />
<p class="caption">
<em>Most highly ranked ranges for both gained and decreased H3K27ac
binding in repsonse to E2DHT treatment. The smooth-quantile normalised
values are shown across the initial set of sliding windows before
merging. Ranges were chosen as the most extreme for FDR, Binding
Strength (Signal) and logFC. Windows are shown free of the genomic
context.</em>
</p>
</div>
</div>
<div id="decreased-binding-strongest-signal" class="section level3">
<h3>Decreased Binding (Strongest Signal)</h3>
<pre class="r"><code>plotHFGC(
  grl_to_plot$top_down_by_expr,
  hic = hic,
  features = feat_gr,featcol = feature_colours, featsize = 1 + has_features,
  genes = hfgc_genes, genecol = gene_col,
  coverage = bwfl, linecol = line_col,
  annotation = annot, 
  annotcol = direction_colours,
  cytobands = bands_df,
  collapseTranscripts = FALSE,
  zoom = 10,
  max = 10 * width(grl_to_plot$top_down_by_expr),
  ylim = y_lim,
  col.title = &quot;black&quot;, background.title = &quot;white&quot;, showAxis = FALSE,
  rotation.title = 90
)</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-top-down-by-expr-1.png" alt="*The 1600bp region showing decreased H3K27ac binding in response to E2DHT treatment (FDR = 0.026). The range mostly overlapped with an Intron, with all defined regions shown as a contiguous bar. Using the features supplied in enhancer_atlas_2.0_zr75.gtf.gz, this mostly overlapped an Enhancer, shown as a separate block with the gene-centric regions.  A union peak overlapping this region was identified by `macs2 callpeak` when using merged samples. Using the above mapping strategy, this range is likely to regulate WWOX For each sample, the y-axis limits represent the values from the window with the highest signal.*" width="960" />
<p class="caption">
<em>The 1600bp region showing decreased H3K27ac binding in response to
E2DHT treatment (FDR = 0.026). The range mostly overlapped with an
Intron, with all defined regions shown as a contiguous bar. Using the
features supplied in enhancer_atlas_2.0_zr75.gtf.gz, this mostly
overlapped an Enhancer, shown as a separate block with the gene-centric
regions. A union peak overlapping this region was identified by
<code>macs2 callpeak</code> when using merged samples. Using the above
mapping strategy, this range is likely to regulate WWOX For each sample,
the y-axis limits represent the values from the window with the highest
signal.</em>
</p>
</div>
</div>
<div id="decreased-binding-lowest-fdr" class="section level3">
<h3>Decreased Binding (Lowest FDR)</h3>
<pre class="r"><code>plotHFGC(
  grl_to_plot$top_down_by_fdr,
  hic = hic,
  features = feat_gr,featcol = feature_colours, featsize = 1 + has_features,
  genes = hfgc_genes, genecol = gene_col,
  coverage = bwfl, linecol = line_col,
  annotation = annot, 
  annotcol = direction_colours,
  cytobands = bands_df,
  collapseTranscripts = FALSE,
  zoom = 10,
  max = 10 * width(grl_to_plot$top_down_by_fdr),
  ylim = y_lim,
  col.title = &quot;black&quot;, background.title = &quot;white&quot;, showAxis = FALSE,
  rotation.title = 90
)</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-top_down_by_fdr-1.png" alt="*The 360bp region showing decreased H3K27ac binding in response to E2DHT treatment (FDR = 0.019). The range mostly overlapped with an Intergenic (&gt;10kb) region, with all defined regions shown as a contiguous bar. Using the features supplied in enhancer_atlas_2.0_zr75.gtf.gz, this mostly overlapped an Enhancer, shown as a separate block with the gene-centric regions.  A union peak overlapping this region was identified by `macs2 callpeak` when using merged samples. No genes were able to be assigned to this region. For each sample, the y-axis limits represent the values from the window with the highest signal.*" width="960" />
<p class="caption">
<em>The 360bp region showing decreased H3K27ac binding in response to
E2DHT treatment (FDR = 0.019). The range mostly overlapped with an
Intergenic (&gt;10kb) region, with all defined regions shown as a
contiguous bar. Using the features supplied in
enhancer_atlas_2.0_zr75.gtf.gz, this mostly overlapped an Enhancer,
shown as a separate block with the gene-centric regions. A union peak
overlapping this region was identified by <code>macs2 callpeak</code>
when using merged samples. No genes were able to be assigned to this
region. For each sample, the y-axis limits represent the values from the
window with the highest signal.</em>
</p>
</div>
</div>
<div id="increased-binding-strongest-signal" class="section level3">
<h3>Increased Binding (Strongest Signal)</h3>
<pre class="r"><code>plotHFGC(
  grl_to_plot$top_up_by_expr,
  hic = hic,
  features = feat_gr,featcol = feature_colours, featsize = 1 + has_features,
  genes = hfgc_genes, genecol = gene_col,
  coverage = bwfl, linecol = line_col,
  annotation = annot, 
  annotcol = direction_colours,
  cytobands = bands_df,
  collapseTranscripts = FALSE,
  zoom = 10,
  max = 10 * width(grl_to_plot$top_up_by_expr),
  ylim = y_lim,
  col.title = &quot;black&quot;, background.title = &quot;white&quot;, showAxis = FALSE,
  rotation.title = 90
)</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-top-up-by-expr-1.png" alt="*The 4320bp region showing increased H3K27ac binding in response to E2DHT treatment (FDR = 1.54e-04). The range mostly overlapped with an Intron, with all defined regions shown as a contiguous bar. Using the features supplied in enhancer_atlas_2.0_zr75.gtf.gz, this mostly overlapped an Enhancer, shown as a separate block with the gene-centric regions.  A union peak overlapping this region was identified by `macs2 callpeak` when using merged samples. Using the above mapping strategy, this range is likely to regulate EIF4EP2, PHB and ZNF652 For each sample, the y-axis limits represent the values from the window with the highest signal.*" width="960" />
<p class="caption">
<em>The 4320bp region showing increased H3K27ac binding in response to
E2DHT treatment (FDR = 1.54e-04). The range mostly overlapped with an
Intron, with all defined regions shown as a contiguous bar. Using the
features supplied in enhancer_atlas_2.0_zr75.gtf.gz, this mostly
overlapped an Enhancer, shown as a separate block with the gene-centric
regions. A union peak overlapping this region was identified by
<code>macs2 callpeak</code> when using merged samples. Using the above
mapping strategy, this range is likely to regulate EIF4EP2, PHB and
ZNF652 For each sample, the y-axis limits represent the values from the
window with the highest signal.</em>
</p>
</div>
</div>
<div id="increased-binding-lowest-fdr" class="section level3">
<h3>Increased Binding (Lowest FDR)</h3>
<pre class="r"><code>plotHFGC(
  grl_to_plot$top_up_by_fdr,
  hic = hic,
  features = feat_gr,featcol = feature_colours, featsize = 1 + has_features,
  genes = hfgc_genes, genecol = gene_col,
  coverage = bwfl, linecol = line_col,
  annotation = annot, 
  annotcol = direction_colours,
  cytobands = bands_df,
  collapseTranscripts = FALSE,
  zoom = 10,
  max = 10 * width(grl_to_plot$top_up_by_fdr),
  ylim = y_lim,
  col.title = &quot;black&quot;, background.title = &quot;white&quot;, showAxis = FALSE,
  rotation.title = 90
)</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-top_up_by_fdr-1.png" alt="*The 1520bp region showing increased H3K27ac binding in response to E2DHT treatment (FDR = 4.86e-16). The range mostly overlapped with an Intron, with all defined regions shown as a contiguous bar. Using the features supplied in enhancer_atlas_2.0_zr75.gtf.gz, this mostly overlapped a No Feature, shown as a separate block with the gene-centric regions.  A union peak overlapping this region was identified by `macs2 callpeak` when using merged samples. Using the above mapping strategy, this range is likely to regulate CCDC40 For each sample, the y-axis limits represent the values from the window with the highest signal.*" width="960" />
<p class="caption">
<em>The 1520bp region showing increased H3K27ac binding in response to
E2DHT treatment (FDR = 4.86e-16). The range mostly overlapped with an
Intron, with all defined regions shown as a contiguous bar. Using the
features supplied in enhancer_atlas_2.0_zr75.gtf.gz, this mostly
overlapped a No Feature, shown as a separate block with the gene-centric
regions. A union peak overlapping this region was identified by
<code>macs2 callpeak</code> when using merged samples. Using the above
mapping strategy, this range is likely to regulate CCDC40 For each
sample, the y-axis limits represent the values from the window with the
highest signal.</em>
</p>
</div>
</div>
</div>
</div>
<div id="enrichment-testing" class="section level1">
<h1>Enrichment Testing</h1>
<pre class="r"><code>min_gs_size &lt;- params$enrichment$min_size
if (is.null(min_gs_size) | is.na(min_gs_size)) min_gs_size &lt;- 0
max_gs_size &lt;- params$enrichment$max_size
if (is.null(max_gs_size) | is.na(max_gs_size)) max_gs_size &lt;- Inf
min_sig &lt;- params$enrichment$min_sig
if (is.null(min_sig) | is.na(min_sig)) min_sig &lt;- 1
msigdb &lt;- msigdbr(species = &quot;Homo sapiens&quot;) %&gt;% 
  dplyr::filter(
    gs_cat %in% unlist(params$enrichment$msigdb$gs_cat) |
      gs_subcat %in% unlist(params$enrichment$msigdb$gs_subcat),
    str_detect(ensembl_gene, &quot;^E&quot;)
  ) %&gt;% 
  dplyr::rename(gene_id = ensembl_gene, gene_name = gene_symbol) %&gt;% # For easier integration 
  dplyr::select(-starts_with(&quot;human&quot;), -contains(&quot;entrez&quot;)) %&gt;% 
  dplyr::filter(gene_id %in% gtf_gene$gene_id) %&gt;% 
  mutate(gs_url = str_extract(gs_url, &quot;^[^|]+&quot;)) %&gt;% 
  group_by(gs_name) %&gt;% 
  mutate(n = dplyr::n()) %&gt;% 
  ungroup() %&gt;% 
  dplyr::filter(n &gt;= min_gs_size, n &lt; max_gs_size) 
gs_by_gsid &lt;- msigdb %&gt;% 
  split(.$gs_name) %&gt;% 
  lapply(pull, &quot;gene_id&quot;) %&gt;% 
  lapply(unique)
gs_by_geneid &lt;- msigdb %&gt;% 
  split(.$gene_id) %&gt;% 
  lapply(pull, &quot;gs_name&quot;) %&gt;% 
  lapply(unique)
gs_url &lt;- msigdb %&gt;% 
  distinct(gs_name, gs_url) %&gt;%
  mutate(
    gs_url = ifelse(
      gs_url == &quot;&quot;, &quot;
      http://www.gsea-msigdb.org/gsea/msigdb/collections.jsp&quot;,
      gs_url
    )
  ) %&gt;% 
  with(setNames(gs_url, gs_name))

min_network_size &lt;- params$networks$min_size
if (is.null(min_network_size) | is.na(min_network_size))
  min_network_size &lt;- 2

max_network_size &lt;- params$networks$max_size
if (is.null(max_network_size) | is.na(max_network_size))
  max_network_size &lt;- Inf

max_network_dist &lt;- params$networks$max_distance
if (is.null(max_network_dist) | is.na(max_network_dist))
  max_network_dist &lt;- 1

net_layout &lt;- params$networks$layout
enrich_alpha &lt;- params$enrichment$alpha
adj_method &lt;- match.arg(params$enrichment$adj, p.adjust.methods)
adj_desc &lt;- case_when(
  p.adjust.methods %in% c(&quot;fdr&quot;, &quot;BH&quot;) ~ &quot;the Benjamini-Hochberg FDR&quot;,
  p.adjust.methods %in% c(&quot;BY&quot;) ~ &quot;the Benjamini-Yekutieli FDR&quot;,
  p.adjust.methods %in% c(&quot;bonferroni&quot;) ~ &quot;the Bonferroni&quot;,
  p.adjust.methods %in% c(&quot;holm&quot;) ~ &quot;Holm&#39;s&quot;,
  p.adjust.methods %in% c(&quot;hommel&quot;) ~ &quot;Hommel&#39;s&quot;,
  p.adjust.methods %in% c(&quot;hochberg&quot;) ~ &quot;Hochberg&#39;s&quot;,
  p.adjust.methods %in% c(&quot;none&quot;) ~ &quot;no&quot;
) %&gt;% 
  setNames(p.adjust.methods)
mapped_ids &lt;- merged_results %&gt;%
  as_tibble() %&gt;% 
  dplyr::select(status, gene_id) %&gt;%
  unnest(gene_id) %&gt;%
  group_by(gene_id) %&gt;%
  summarise(
    up = any(status == &quot;Up&quot;), 
    down = any(status == &quot;Down&quot;), 
    n_peaks = dplyr::n(),
    .groups = &quot;drop&quot;
  )</code></pre>
<p>As an initial exploration, the retained windows were compared to
pre-defined gene-sets. These were taken from the <code>MSigDB</code>
database and the gene-sets used for enrichment testing were restricted
to only include those with between 5 and 1000 genes.</p>
<pre class="r"><code>msigdb %&gt;% 
  distinct(gs_cat, gs_subcat, gs_name) %&gt;%
  group_by(gs_cat, gs_subcat) %&gt;% 
  summarise(`Gene Sets` = dplyr::n(),.groups = &quot;drop&quot;) %&gt;% 
  dplyr::rename(Category = gs_cat, `Sub-Category` = gs_subcat) %&gt;% 
  mutate(
    Description = case_when(
      Category == &quot;H&quot; ~ &quot;Hallmark Gene-Sets&quot;,
      Category == &quot;C1&quot; ~ &quot;Positional Gene Sets&quot;,
      Category == &quot;C2&quot; ~ str_replace_all(`Sub-Category`, &quot;CP:(.+)&quot;, &quot;Curated Gene-Sets: \\1&quot;),
      str_detect(`Sub-Category`, &quot;TFT&quot;) ~ str_replace_all(`Sub-Category`, &quot;TFT:(.+)&quot;, &quot;Transcription Factor Target Prediction Gene-Sets: \\1&quot;),
      str_detect(`Sub-Category`, &quot;MIR&quot;) ~ str_replace_all(`Sub-Category`, &quot;MIR:(.+)&quot;, &quot;microRNA Target Gene-Sets: \\1&quot;),
      str_detect(`Sub-Category`, &quot;CGN&quot;) ~ &quot;Cancer Gene Neighbourhoods&quot;,
      str_detect(`Sub-Category`, &quot;CM&quot;) ~ &quot;Cancer Modules&quot;,
      str_detect(`Sub-Category`, &quot;GO:BP&quot;) ~ &quot;Gene Ontology: Bological Process&quot;,
      str_detect(`Sub-Category`, &quot;GO:MF&quot;) ~ &quot;Gene Ontology: Molecular Function&quot;,
      str_detect(`Sub-Category`, &quot;GO:CC&quot;) ~ &quot;Gene Ontology: Cellular Component&quot;,
      str_detect(`Sub-Category`, &quot;HPO&quot;) ~ &quot;Human Phenotype Ontology&quot;,
      Category == &quot;C6&quot; ~ &quot;Oncogenic Signature Gene Sets&quot;,
      `Sub-Category` == &quot;IMMUNESIGDB&quot; ~ &quot;ImmuneSigDB Gene Sets&quot;,
      `Sub-Category` == &quot;VAX&quot; ~ &quot;Vaccine Response Gene Sets&quot;,
      Category == &quot;C8&quot; ~ &quot;Cell Type Signature Gene Sets&quot;
    ),
    Category = factor(Category, levels = c(&quot;H&quot;, paste0(&quot;C&quot;, 1:8)))
  ) %&gt;% 
  droplevels() %&gt;% 
  arrange(Category) %&gt;% 
  pander(
    justify = &quot;llrl&quot;, 
    caption = &quot;*Summary of Gene-Sets used for Enrichment Testing*&quot;
  )</code></pre>
<table style="width:99%;">
<caption><em>Summary of Gene-Sets used for Enrichment
Testing</em></caption>
<colgroup>
<col width="15%" />
<col width="25%" />
<col width="16%" />
<col width="41%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Category</th>
<th align="left">Sub-Category</th>
<th align="right">Gene Sets</th>
<th align="left">Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">H</td>
<td align="left"></td>
<td align="right">50</td>
<td align="left">Hallmark Gene-Sets</td>
</tr>
<tr class="even">
<td align="left">C2</td>
<td align="left">CP:KEGG</td>
<td align="right">183</td>
<td align="left">Curated Gene-Sets: KEGG</td>
</tr>
<tr class="odd">
<td align="left">C2</td>
<td align="left">CP:REACTOME</td>
<td align="right">1,388</td>
<td align="left">Curated Gene-Sets: REACTOME</td>
</tr>
<tr class="even">
<td align="left">C2</td>
<td align="left">CP:WIKIPATHWAYS</td>
<td align="right">605</td>
<td align="left">Curated Gene-Sets: WIKIPATHWAYS</td>
</tr>
<tr class="odd">
<td align="left">C3</td>
<td align="left">TFT:GTRD</td>
<td align="right">425</td>
<td align="left">Transcription Factor Target Prediction Gene-Sets:
GTRD</td>
</tr>
</tbody>
</table>
<p>Retained windows were tested for enrichment of these gene-sets
using:</p>
<ol style="list-style-type: decimal">
<li>Any window mapped to a gene from these gene-sets</li>
<li>Any window with evidence of differential binding mapped to a gene
from these gene-sets</li>
</ol>
<p>In the first case, mapped genes were tested for enrichment against
all annotated genes, or all detected genes if RNA-Seq data was provided.
In the second case, i.e. Differentially Bound Windows, the control set
of genes were those mapped to a window (i.e. those test set from step
1). All adjusted p-values below are calculated using the
Benjamini-Hochberg FDR adjustment. After adjustment, any enriched
gene-sets with fewer than 3 mapped genes were excluded as being
uninformative.</p>
<p>If 4 or more gene-sets were considered to be enriched, a network plot
will be produced for that specific analysis, with network sizes capped
at 80. The distances between gene-sets were calculated using the overlap
coefficient (OC). Gene-sets with a large overlap will thus be given
small distances and in the case of a complete overlap (<span
class="math inline">\(OC = 1\)</span>) the only most highly ranked
gene-set was retained. Gene-set (i.e. node) pairs with a distance &gt;
0.9 will not have edges drawn between them and edge width also
corresponds to the distance between nodes with closely related nodes
having thicker edges. All network plots were generated using the fr
layout algorithm.</p>
<div id="windows-with-detected-h3k27ac" class="section level2 tabset">
<h2 class="tabset">Windows With Detected H3K27ac</h2>
<pre class="r"><code>pwf_mapped &lt;- gtf_gene %&gt;% 
  mutate(w = width(.)) %&gt;% 
  as_tibble() %&gt;% 
  dplyr::select(gene_id, w) %&gt;% 
  arrange(desc(w)) %&gt;% 
  distinct(gene_id, .keep_all = TRUE) %&gt;% 
  mutate(mapped = gene_id %in% unlist(merged_results$gene_id)) %&gt;% 
  with(
    nullp(
      structure(mapped, names = gene_id),
      genome = config$genome$build,
      bias.data = log10(w),
      plot.fit = TRUE
    )
  )</code></pre>
<pre class="r"><code>goseq_mapped &lt;- tibble(gs_name = character(), adj_p = numeric())
if (sum(pwf_mapped$DEgenes) &gt; 0) {
  goseq_mapped &lt;- goseq(
    pwf_mapped, config$genome$build, gene2cat = gs_by_geneid
  ) %&gt;% 
    dplyr::rename(gs_name = category, pval = over_represented_pvalue) %&gt;% 
    dplyr::select(-starts_with(&quot;under&quot;)) %&gt;% 
    as_tibble() %&gt;% 
    arrange(pval) %&gt;% 
    dplyr::filter(numDEInCat &gt; 0) %&gt;% 
    mutate(adj_p = p.adjust(pval, adj_method)) %&gt;% 
    dplyr::filter(numDEInCat &gt;= min_sig)
}
any_goseq_mapped &lt;- sum(goseq_mapped$adj_p &lt; enrich_alpha) &gt; 0
tg_mapped &lt;- make_tbl_graph(
  goseq_mapped, 
  gs = gs_by_gsid %&gt;% 
    lapply(intersect, rownames(subset(pwf_mapped, DEgenes))) %&gt;% 
    .[vapply(., length, integer(1)) &gt; 0]
)
plot_network_mapped &lt;- length(tg_mapped) &gt;= min_network_size</code></pre>
<div id="results-table" class="section level3">
<h3>Results Table</h3>
<pre class="r"><code>cp &lt;- htmltools::tags$caption(
  htmltools::em(
    glue(
      &quot;
    All {sum(goseq_mapped$adj_p &lt; enrich_alpha)} gene sets considered as 
    significantly enriched (p
    &quot;
    ),
    htmltools::tags$sub(&quot;adj&quot;),
    htmltools::em(
      glue(
        &quot;
         &lt; {enrich_alpha}) amongst the merged windows with detectable {target} 
        binding. Genes mapped to a merged window were compared to those not 
        mapped to any merged windows. Gene width was used to capture any 
        gene-level sampling bias.
        &quot;
      )
    )
  )
)
tbl &lt;- goseq_mapped %&gt;%
  dplyr::filter(adj_p &lt; enrich_alpha) %&gt;% 
  left_join(
    dplyr::select(msigdb, gs_name, gene_name, gene_id, gs_url, gs_description),
    by = &quot;gs_name&quot;
  ) %&gt;% 
  dplyr::filter(gene_id %in% unlist(merged_results$gene_id)) %&gt;% 
  dplyr::select(-gene_id) %&gt;% 
  chop(gene_name) %&gt;% 
  mutate(
    gene_name = vapply(gene_name, paste, character(1), collapse = &quot;; &quot;),
    `%` = numDEInCat / numInCat
  ) %&gt;% 
  dplyr::select(
    `Gene Set` = gs_name, Description = gs_description,
    `%`, numDEInCat, numInCat, 
    p = pval, adj_p, 
    `Mapped Genes` = gene_name
  ) %&gt;% 
  reactable(
    filterable = TRUE,
    showPageSizeOptions = TRUE,
    columns = list(
      &quot;Gene Set&quot; = colDef(
        minWidth = 150,
        cell = function(value) htmltools::tags$a(
          href = gs_url[[value]], 
          target = &quot;_blank&quot;, 
          str_replace_all(value, &quot;_&quot;, &quot; &quot;)
        ),
        html = TRUE
      ),
      Description = colDef(
        minWidth = 150,
        cell = function(value) with_tooltip(value, width = 100)
      ),
      numDEInCat = colDef(name = &quot;Total Mapped&quot;, maxWidth = 80),
      numInCat = colDef(
        name = &quot;Gene Set Size&quot;,
        maxWidth = 80,
        cell = function(value) comma(value, 1)
      ),
      &quot;%&quot; = colDef(
        name = &quot;% Mapped&quot;,
        cell = function(value) percent(value, 0.1),
        maxWidth = 80
      ),
      p = colDef(
        cell = function(value) {
          ifelse(
            value &lt; 0.001, sprintf(&quot;%.2e&quot;, value), sprintf(&quot;%.3f&quot;, value)
          )
        },
        maxWidth = 80
      ),
      adj_p = colDef(
        name = &quot;p&lt;sub&gt;adj&lt;/sub&gt;&quot;, html = TRUE,
        cell = function(value) {
          ifelse(
            value &lt; 0.001, sprintf(&quot;%.2e&quot;, value), sprintf(&quot;%.3f&quot;, value)
          )
        },
        maxWidth = 80
      ),
      &quot;Mapped Genes&quot; = colDef(
        cell = function(value) with_tooltip(value, width = 100),
        minWidth = 150
      )
    )
  )
div(class = &quot;table&quot;,
  div(class = &quot;table-header&quot;,
      div(class = &quot;caption&quot;, cp),
      tbl
  )
)</code></pre>
<div class="table">
<div class="table-header">
<div class="caption">
<caption>
<em>
All 1 gene sets considered as 
significantly enriched (p
<sub>adj</sub>
<em> &lt; 0.05) amongst the merged windows with detectable H3K27ac 
binding. Genes mapped to a merged window were compared to those not 
mapped to any merged windows. Gene width was used to capture any 
gene-level sampling bias.</em>
</em>
</caption>
</div>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-2954c62e3c5ff9e6b229" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-2954c62e3c5ff9e6b229">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"Gene Set":["REACTOME_INFECTIOUS_DISEASE"],"Description":["Infectious disease"],"%":[0.990895295902883],"numDEInCat":[653],"numInCat":[659],"p":[7.07296250558441e-06],"adj_p":[0.0187504236023043],"Mapped Genes":["AAAS; ABI1; ABI2; ABL1; ACTB; ACTG1; ACTR2; ACTR3; ADAM17; ADCY1; ADCY2; ADCY3; ADCY6; ADCY7; ADCY9; ADM2; ADORA2B; AHCYL1; ANTXR2; AP1B1; AP1G1; AP1M1; AP1M2; AP1S1; AP1S2; AP1S3; AP2A1; AP2A2; AP2B1; AP2M1; AP2S1; APP; ARF1; ARID4A; ARID4B; ARPC1A; ARPC1B; ARPC2; ARPC3; ARPC4; ARPC5; ATP1A1; ATP1B1; ATP1B3; ATP6V1H; B2M; BAIAP2; BANF1; BECN1; BRD4; BRK1; BRMS1; BTRC; CALM1; CALR; CANX; CBL; CBLL1; CBX1; CCNH; CCNK; CCNT1; CCNT2; CD9; CDC42; CDH1; CDK7; CDK9; CEBPD; CGA; CHD3; CHD4; CHMP1A; CHMP2A; CHMP2B; CHMP3; CHMP4B; CHMP4C; CHMP5; CHMP6; CHMP7; CLTA; CLTC; COMT; CORO1A; CPSF4; CRBN; CREB1; CRK; CSNK1A1; CTDP1; CTNNB1; CTNND1; CTSL; CUL3; CUL5; CXCR4; CYBA; CYFIP1; CYFIP2; DAD1; DAXX; DDOST; DDX5; DNAJC3; DOCK1; DOCK2; DUSP16; DVL1; DVL2; DVL3; DYNC1H1; DYNC1I2; DYNC1LI1; DYNC1LI2; DYNLL1; DYNLL2; EDEM2; EED; EEF2; EGFR; EIF2AK2; ELK1; ELL; ELMO1; ELMO2; ELOA; ELOB; ELOC; ENO1; ENTPD1; ENTPD5; EPS15; ERCC2; ERCC3; EZH2; FAU; FEN1; FKBP1A; FKBP4; FNTA; FNTB; FURIN; FUT8; FXYD3; FZD7; GALNT1; GANAB; GATAD2A; GATAD2B; GIPR; GNAI2; GNAI3; GNAS; GNAZ; GNB1; GNB2; GNB3; GNB4; GNB5; GNG10; GNG12; GNG4; GNG5; GPR27; GPR39; GPS2; GRB2; GRSF1; GSDMD; GSK3A; GSK3B; GTF2A1; GTF2A2; GTF2B; GTF2E1; GTF2E2; GTF2F1; GTF2F2; GTF2H1; GTF2H3; GTF2H5; H2AC11; H2AC12; H2AC13; H2AC14; H2AC16; H2AC17; H2AC20; H2AC21; H2AC4; H2AC6; H2AC7; H2AC8; H2BC10; H2BC11; H2BC12; H2BC13; H2BC14; H2BC15; H2BC17; H2BC18; H2BC21; H2BC3; H2BC4; H2BC5; H3C1; H3C10; H3C12; H3C13; H3C14; H3C15; H3C2; H3C3; H3C6; H4C1; H4C11; H4C12; H4C14; H4C15; H4C3; H4C5; H4C8; H4C9; HDAC1; HDAC2; HDAC3; HGS; HLA-A; HMG20B; HMGA1; HMOX1; HNRNPK; HSP90AA1; HSP90AB1; HSPA1A; IFNAR1; IFNGR1; IFNGR2; IL18; IL1R1; IL6R; IMPDH1; IMPDH2; IPO5; ISG15; ITGB1; ITPR1; ITPR2; ITPR3; JAK1; JAK2; JUN; KDM1A; KEAP1; KPNA1; KPNA2; KPNA3; KPNA4; KPNA5; KPNB1; LIG1; LIG4; MAGT1; MAN1B1; MAN2A1; MAP1LC3B; MAP2K1; MAP2K2; MAP2K3; MAP2K4; MAP2K6; MAP2K7; MAPK1; MAPK14; MAPK3; MAPK8; MBD3; MET; MGAT1; MGAT2; MGAT4A; MGAT4B; MGAT5; MNAT1; MOGS; MTA1; MTA2; MTA3; MVB12A; MVB12B; MYH9; MYO10; MYO1C; MYO5A; MYO9B; NCBP1; NCBP2; NCK1; NCKAP1; NCKIPSD; NCOR1; NCOR2; NDC1; NEDD4L; NELFA; NELFB; NELFCD; NELFE; NFE2L2; NFKB1; NFKB2; NMT1; NMT2; NOXA1; NPM1; NR3C1; NT5E; NUP107; NUP133; NUP153; NUP155; NUP160; NUP188; NUP205; NUP210; NUP214; NUP35; NUP37; NUP42; NUP43; NUP50; NUP54; NUP58; NUP62; NUP85; NUP88; NUP93; NUP98; P2RX4; PABPN1; PACS1; PAK2; PARP1; PARP10; PARP14; PARP16; PARP4; PARP6; PARP8; PARP9; PDCD6IP; PGK1; PHF21A; PIK3C3; PIK3R4; PLCG1; PLCG2; PLK2; PML; POLR2A; POLR2B; POLR2C; POLR2D; POLR2E; POLR2F; POLR2G; POLR2H; POLR2I; POLR2J; POLR2K; POLR2L; POM121; POM121C; PPIA; PRKACA; PRKACB; PRKAR1A; PRKAR1B; PRKAR2A; PRKAR2B; PRKCSH; PRKX; PRMT1; PSIP1; PSMA1; PSMA2; PSMA3; PSMA4; PSMA5; PSMA6; PSMA7; PSMB1; PSMB10; PSMB2; PSMB3; PSMB4; PSMB5; PSMB6; PSMB7; PSMB8; PSMC1; PSMC2; PSMC3; PSMC4; PSMC5; PSMC6; PSMD1; PSMD10; PSMD11; PSMD12; PSMD13; PSMD14; PSMD2; PSMD3; PSMD5; PSMD6; PSMD7; PSMD8; PSMD9; PSME1; PSME2; PSME3; PSME4; PSMF1; PTGER4; PTGES3; PTK2; PYCARD; RAB5A; RAB7A; RAC1; RAE1; RAMP1; RAN; RANBP1; RANBP2; RANGAP1; RB1; RBBP4; RBBP7; RBX1; RCC1; RCOR1; RELA; REST; RHBDF2; RIPK1; RNF213; RNGTT; RNMT; ROCK1; ROCK2; RPL10; RPL10A; RPL11; RPL12; RPL13; RPL13A; RPL14; RPL15; RPL17; RPL18; RPL18A; RPL19; RPL21; RPL22; RPL22L1; RPL23; RPL23A; RPL24; RPL26; RPL26L1; RPL27; RPL27A; RPL28; RPL29; RPL3; RPL30; RPL31; RPL32; RPL34; RPL35; RPL35A; RPL36; RPL36A; RPL36AL; RPL37; RPL37A; RPL38; RPL39; RPL4; RPL41; RPL5; RPL6; RPL7; RPL7A; RPL8; RPL9; RPLP0; RPLP1; RPLP2; RPN1; RPN2; RPS10; RPS11; RPS12; RPS13; RPS14; RPS15; RPS15A; RPS16; RPS17; RPS18; RPS19; RPS2; RPS20; RPS21; RPS23; RPS24; RPS25; RPS26; RPS27; RPS27A; RPS27L; RPS28; RPS29; RPS3; RPS3A; RPS4X; RPS5; RPS6; RPS7; RPS8; RPS9; RPSA; SAP18; SAP30; SAP30L; SEC13; SEH1L; SEM1; SFPQ; SH3GL1; SH3GL2; SH3KBP1; SIGMAR1; SKP1; SLC25A4; SLC25A5; SLC25A6; SNF8; SRC; SRPK1; SRPK2; SSRP1; ST3GAL1; ST3GAL2; ST3GAL4; ST6GAL1; ST6GALNAC2; ST6GALNAC4; STAM; STAM2; STAT2; STT3A; STX1A; SUDS3; SUGT1; SUMO1; SUPT16H; SUPT4H1; SUPT5H; SUZ12; SYK; SYT1; TAF1; TAF10; TAF11; TAF12; TAF13; TAF15; TAF2; TAF3; TAF4; TAF4B; TAF5; TAF6; TAF7; TAF9; TAF9B; TBK1; TBL1X; TBL1XR1; TBP; TCEA1; TRIM27; TRIM28; TSG101; TUBA1A; TUBA1B; TUBA1C; TUBA4A; TUBB; TUBB2A; TUBB2B; TUBB3; TUBB4B; TUBB6; TUSC3; TXN; TXNIP; TXNRD1; TYK2; UBA52; UBAP1; UBB; UBC; UBE2I; UVRAG; VAMP1; VAMP2; VAV1; VAV2; VAV3; VCP; VEGFA; VHL; VIPR1; VIPR2; VPS25; VPS28; VPS33B; VPS36; VPS37A; VPS37B; VPS37C; VPS37D; VPS4A; VPS4B; VTA1; WASF1; WASF2; WASF3; WASL; WIPF2; WNT5A; XPO1; XRCC4; XRCC5; XRCC6; YES1; ZCRB1"]},"columns":[{"id":"Gene Set","name":"Gene Set","type":"character","cell":[{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-5663205","target":"_blank"},"children":["REACTOME INFECTIOUS DISEASE"]}],"html":true,"minWidth":150},{"id":"Description","name":"Description","type":"character","cell":[{"name":"span","attribs":{"title":"Infectious disease"},"children":["Infectious disease"]}],"minWidth":150},{"id":"%","name":"% Mapped","type":"numeric","cell":["99.1%"],"maxWidth":80},{"id":"numDEInCat","name":"Total Mapped","type":"numeric","maxWidth":80},{"id":"numInCat","name":"Gene Set Size","type":"numeric","cell":["659"],"maxWidth":80},{"id":"p","name":"p","type":"numeric","cell":["7.07e-06"],"maxWidth":80},{"id":"adj_p","name":"p<sub>adj<\/sub>","type":"numeric","cell":["0.019"],"html":true,"maxWidth":80},{"id":"Mapped Genes","name":"Mapped Genes","type":"character","cell":[{"name":"span","attribs":{"title":"AAAS; ABI1; ABI2; ABL1; ACTB; ACTG1; ACTR2; ACTR3; ADAM17; ADCY1; ADCY2; ADCY3; ADCY6; ADCY7; ADCY9; ADM2; ADORA2B; AHCYL1; ANTXR2; AP1B1; AP1G1; AP1M1; AP1M2; AP1S1; AP1S2; AP1S3; AP2A1; AP2A2; AP2B1; AP2M1; AP2S1; APP; ARF1; ARID4A; ARID4B; ARPC1A; ARPC1B; ARPC2; ARPC3; ARPC4; ARPC5; ATP1A1; ATP1B1; ATP1B3; ATP6V1H; B2M; BAIAP2; BANF1; BECN1; BRD4; BRK1; BRMS1; BTRC; CALM1; CALR; CANX; CBL; CBLL1; CBX1; CCNH; CCNK; CCNT1; CCNT2; CD9; CDC42; CDH1; CDK7; CDK9; CEBPD; CGA; CHD3; CHD4; CHMP1A; CHMP2A; CHMP2B; CHMP3; CHMP4B; CHMP4C; CHMP5; CHMP6; CHMP7; CLTA; CLTC; COMT; CORO1A; CPSF4; CRBN; CREB1; CRK; CSNK1A1; CTDP1; CTNNB1; CTNND1; CTSL; CUL3; CUL5; CXCR4; CYBA; CYFIP1; CYFIP2; DAD1; DAXX; DDOST; DDX5; DNAJC3; DOCK1; DOCK2; DUSP16; DVL1; DVL2; DVL3; DYNC1H1; DYNC1I2; DYNC1LI1; DYNC1LI2; DYNLL1; DYNLL2; EDEM2; EED; EEF2; EGFR; EIF2AK2; ELK1; ELL; ELMO1; ELMO2; ELOA; ELOB; ELOC; ENO1; ENTPD1; ENTPD5; EPS15; ERCC2; ERCC3; EZH2; FAU; FEN1; FKBP1A; FKBP4; FNTA; FNTB; FURIN; FUT8; FXYD3; FZD7; GALNT1; GANAB; GATAD2A; GATAD2B; GIPR; GNAI2; GNAI3; GNAS; GNAZ; GNB1; GNB2; GNB3; GNB4; GNB5; GNG10; GNG12; GNG4; GNG5; GPR27; GPR39; GPS2; GRB2; GRSF1; GSDMD; GSK3A; GSK3B; GTF2A1; GTF2A2; GTF2B; GTF2E1; GTF2E2; GTF2F1; GTF2F2; GTF2H1; GTF2H3; GTF2H5; H2AC11; H2AC12; H2AC13; H2AC14; H2AC16; H2AC17; H2AC20; H2AC21; H2AC4; H2AC6; H2AC7; H2AC8; H2BC10; H2BC11; H2BC12; H2BC13; H2BC14; H2BC15; H2BC17; H2BC18; H2BC21; H2BC3; H2BC4; H2BC5; H3C1; H3C10; H3C12; H3C13; H3C14; H3C15; H3C2; H3C3; H3C6; H4C1; H4C11; H4C12; H4C14; H4C15; H4C3; H4C5; H4C8; H4C9; HDAC1; HDAC2; HDAC3; HGS; HLA-A; HMG20B; HMGA1; HMOX1; HNRNPK; HSP90AA1; HSP90AB1; HSPA1A; IFNAR1; IFNGR1; IFNGR2; IL18; IL1R1; IL6R; IMPDH1; IMPDH2; IPO5; ISG15; ITGB1; ITPR1; ITPR2; ITPR3; JAK1; JAK2; JUN; KDM1A; KEAP1; KPNA1; KPNA2; KPNA3; KPNA4; KPNA5; KPNB1; LIG1; LIG4; MAGT1; MAN1B1; MAN2A1; MAP1LC3B; MAP2K1; MAP2K2; MAP2K3; MAP2K4; MAP2K6; MAP2K7; MAPK1; MAPK14; MAPK3; MAPK8; MBD3; MET; MGAT1; MGAT2; MGAT4A; MGAT4B; MGAT5; MNAT1; MOGS; MTA1; MTA2; MTA3; MVB12A; MVB12B; MYH9; MYO10; MYO1C; MYO5A; MYO9B; NCBP1; NCBP2; NCK1; NCKAP1; NCKIPSD; NCOR1; NCOR2; NDC1; NEDD4L; NELFA; NELFB; NELFCD; NELFE; NFE2L2; NFKB1; NFKB2; NMT1; NMT2; NOXA1; NPM1; NR3C1; NT5E; NUP107; NUP133; NUP153; NUP155; NUP160; NUP188; NUP205; NUP210; NUP214; NUP35; NUP37; NUP42; NUP43; NUP50; NUP54; NUP58; NUP62; NUP85; NUP88; NUP93; NUP98; P2RX4; PABPN1; PACS1; PAK2; PARP1; PARP10; PARP14; PARP16; PARP4; PARP6; PARP8; PARP9; PDCD6IP; PGK1; PHF21A; PIK3C3; PIK3R4; PLCG1; PLCG2; PLK2; PML; POLR2A; POLR2B; POLR2C; POLR2D; POLR2E; POLR2F; POLR2G; POLR2H; POLR2I; POLR2J; POLR2K; POLR2L; POM121; POM121C; PPIA; PRKACA; PRKACB; PRKAR1A; PRKAR1B; PRKAR2A; PRKAR2B; PRKCSH; PRKX; PRMT1; PSIP1; PSMA1; PSMA2; PSMA3; PSMA4; PSMA5; PSMA6; PSMA7; PSMB1; PSMB10; PSMB2; PSMB3; PSMB4; PSMB5; PSMB6; PSMB7; PSMB8; PSMC1; PSMC2; PSMC3; PSMC4; PSMC5; PSMC6; PSMD1; PSMD10; PSMD11; PSMD12; PSMD13; PSMD14; PSMD2; PSMD3; PSMD5; PSMD6; PSMD7; PSMD8; PSMD9; PSME1; PSME2; PSME3; PSME4; PSMF1; PTGER4; PTGES3; PTK2; PYCARD; RAB5A; RAB7A; RAC1; RAE1; RAMP1; RAN; RANBP1; RANBP2; RANGAP1; RB1; RBBP4; RBBP7; RBX1; RCC1; RCOR1; RELA; REST; RHBDF2; RIPK1; RNF213; RNGTT; RNMT; ROCK1; ROCK2; RPL10; RPL10A; RPL11; RPL12; RPL13; RPL13A; RPL14; RPL15; RPL17; RPL18; RPL18A; RPL19; RPL21; RPL22; RPL22L1; RPL23; RPL23A; RPL24; RPL26; RPL26L1; RPL27; RPL27A; RPL28; RPL29; RPL3; RPL30; RPL31; RPL32; RPL34; RPL35; RPL35A; RPL36; RPL36A; RPL36AL; RPL37; RPL37A; RPL38; RPL39; RPL4; RPL41; RPL5; RPL6; RPL7; RPL7A; RPL8; RPL9; RPLP0; RPLP1; RPLP2; RPN1; RPN2; RPS10; RPS11; RPS12; RPS13; RPS14; RPS15; RPS15A; RPS16; RPS17; RPS18; RPS19; RPS2; RPS20; RPS21; RPS23; RPS24; RPS25; RPS26; RPS27; RPS27A; RPS27L; RPS28; RPS29; RPS3; RPS3A; RPS4X; RPS5; RPS6; RPS7; RPS8; RPS9; RPSA; SAP18; SAP30; SAP30L; SEC13; SEH1L; SEM1; SFPQ; SH3GL1; SH3GL2; SH3KBP1; SIGMAR1; SKP1; SLC25A4; SLC25A5; SLC25A6; SNF8; SRC; SRPK1; SRPK2; SSRP1; ST3GAL1; ST3GAL2; ST3GAL4; ST6GAL1; ST6GALNAC2; ST6GALNAC4; STAM; STAM2; STAT2; STT3A; STX1A; SUDS3; SUGT1; SUMO1; SUPT16H; SUPT4H1; SUPT5H; SUZ12; SYK; SYT1; TAF1; TAF10; TAF11; TAF12; TAF13; TAF15; TAF2; TAF3; TAF4; TAF4B; TAF5; TAF6; TAF7; TAF9; TAF9B; TBK1; TBL1X; TBL1XR1; TBP; TCEA1; TRIM27; TRIM28; TSG101; TUBA1A; TUBA1B; TUBA1C; TUBA4A; TUBB; TUBB2A; TUBB2B; TUBB3; TUBB4B; TUBB6; TUSC3; TXN; TXNIP; TXNRD1; TYK2; UBA52; UBAP1; UBB; UBC; UBE2I; UVRAG; VAMP1; VAMP2; VAV1; VAV2; VAV3; VCP; VEGFA; VHL; VIPR1; VIPR2; VPS25; VPS28; VPS33B; VPS36; VPS37A; VPS37B; VPS37C; VPS37D; VPS4A; VPS4B; VTA1; WASF1; WASF2; WASF3; WASL; WIPF2; WNT5A; XPO1; XRCC4; XRCC5; XRCC6; YES1; ZCRB1"},"children":["AAAS; ABI1; ABI2; ABL1; ACTB; ACTG1; ACTR2; ACTR3; ADAM17; ADCY1; ADCY2; ADCY3; ADCY6; ADCY7; ADC..."]}],"minWidth":150}],"filterable":true,"showPageSizeOptions":true,"dataKey":"24b28037c6dc6a30ea085e3cdf527c2f"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
</div>
<div id="windows-with-differentially-bound-h3k27ac"
class="section level2 tabset">
<h2 class="tabset">Windows With Differentially Bound H3K27ac</h2>
<pre class="r"><code>pwf_diff &lt;- tibble(gene_id  = unique(gtf_gene$gene_id)) %&gt;% 
  left_join(mapped_ids, by = &quot;gene_id&quot;) %&gt;% 
  dplyr::filter(!is.na(n_peaks)) %&gt;% 
  mutate(diff = up|down) %&gt;% 
  with(
    nullp(
      structure(diff, names = gene_id),
      genome = config$genome$build,
      bias.data = log1p(n_peaks),
      plot.fit = FALSE
    )
  )
goseq_diff &lt;- tibble(gs_name = character(), pval = numeric(), adj_p = numeric())
if (sum(pwf_diff$DEgenes) &gt; 0) {
  goseq_diff &lt;- goseq(
    pwf_diff, gene2cat = gs_by_geneid, method = &quot;Hypergeometric&quot;
  ) %&gt;% 
    dplyr::rename(gs_name = category, pval = over_represented_pvalue) %&gt;% 
    dplyr::select(-starts_with(&quot;under&quot;)) %&gt;% 
    as_tibble() %&gt;% 
    arrange(pval) %&gt;% 
    dplyr::filter(numDEInCat &gt; 0) %&gt;% 
    mutate(adj_p = p.adjust(pval, adj_method)) %&gt;% 
    dplyr::filter(numDEInCat &gt;= min_sig)
}
any_goseq_diff &lt;- sum(goseq_diff$adj_p &lt; enrich_alpha) &gt; 0
tg_diff &lt;- make_tbl_graph(
  goseq_diff, 
  gs = gs_by_gsid %&gt;% 
    lapply(intersect, rownames(subset(pwf_diff, DEgenes))) %&gt;%
    .[vapply(., length, integer(1)) &gt; 0]
)
plot_network_diff &lt;- length(tg_diff) &gt;= min_network_size</code></pre>
<div id="results-table-1" class="section level3">
<h3>Results Table</h3>
<pre class="r"><code>cp &lt;- htmltools::tags$caption(
  htmltools::em(
    glue(
      &quot;
    All {sum(goseq_diff$adj_p &lt; enrich_alpha)} gene sets considered as 
    significantly enriched (p
    &quot;
    ),
    htmltools::tags$sub(&quot;adj&quot;),
    htmltools::em(
      glue(
        &quot;
         &lt; {enrich_alpha}) amongst the windows showing evidence of changed 
        {target} binding. Genes mapped to a differentially-bound window were 
        compared to those mapped to any merged window. A standard (unbiased)
        hypergeomtric test was used to test for enrichment.
        &quot;
      )
    )
  )
)
tbl &lt;- goseq_diff %&gt;% 
  dplyr::filter(adj_p &lt; enrich_alpha) %&gt;%
  left_join(
    dplyr::select(msigdb, gs_name, gene_name, gene_id, gs_url, gs_description),
    by = &quot;gs_name&quot;
  ) %&gt;% 
  dplyr::filter(
    gene_id %in% unlist(subset(merged_results, status != &quot;Unchanged&quot;)$gene_id)
  ) %&gt;% 
  dplyr::select(-gene_id) %&gt;% 
  chop(gene_name) %&gt;% 
  mutate(
    gene_name = vapply(gene_name, paste, character(1), collapse = &quot;: &quot;),
    `%` = numDEInCat / numInCat
  ) %&gt;% 
  dplyr::select(
    `Gene Set` = gs_name, Description = gs_description,
    `%`, numDEInCat, numInCat, p = pval, adj_p, 
    `Genes Mapped to DB Windows` = gene_name
  ) %&gt;% 
  reactable(
    filterable = TRUE,
    showPageSizeOptions = TRUE,
    columns = list(
      &quot;Gene Set&quot; = colDef(
        minWidth = 150,
        cell = function(value) htmltools::tags$a(
          href = gs_url[[value]], 
          target = &quot;_blank&quot;, 
          str_replace_all(value, &quot;_&quot;, &quot; &quot;)
        ),
        html = TRUE
      ),
      Description = colDef(
        minWidth = 150,
        cell = function(value) with_tooltip(value, width = 100)
      ),
      &quot;%&quot; = colDef(
        name = &quot;% DB&quot;,
        maxWidth = 80,
        cell = function(value) percent(value, 0.1)
      ),
      numDEInCat = colDef(name = &quot;Total Mapped To DB&quot;, maxWidth = 80),
      numInCat = colDef(
        name = &quot;Total Mapped&quot;,
        maxWidth = 80,
        cell = function(value) comma(value, 1)
      ),
      p = colDef(
        cell = function(value) {
          ifelse(
            value &lt; 0.001, sprintf(&quot;%.2e&quot;, value), sprintf(&quot;%.3f&quot;, value)
          )
        },
        maxWidth = 70
      ),
      adj_p = colDef(
        name = &quot;p&lt;sub&gt;adj&lt;/sub&gt;&quot;, html = TRUE,
        cell = function(value) {
          ifelse(
            value &lt; 0.001, sprintf(&quot;%.2e&quot;, value), sprintf(&quot;%.3f&quot;, value)
          )
        },
        maxWidth = 70
      ),
      &quot;Genes Mapped to DB Windows&quot; = colDef(
        cell = function(value) with_tooltip(value, width = 100),
        minWidth = 200
      )
    )
  )
div(class = &quot;table&quot;,
  div(class = &quot;table-header&quot;,
      div(class = &quot;caption&quot;, cp),
      tbl
  )
)</code></pre>
<div class="table">
<div class="table-header">
<div class="caption">
<caption>
<em>
All 3 gene sets considered as 
significantly enriched (p
<sub>adj</sub>
<em> &lt; 0.05) amongst the windows showing evidence of changed 
H3K27ac binding. Genes mapped to a differentially-bound window were 
compared to those mapped to any merged window. A standard (unbiased)
hypergeomtric test was used to test for enrichment.</em>
</em>
</caption>
</div>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-f409cb0f9c82a85b2ee4" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-f409cb0f9c82a85b2ee4">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"Gene Set":["HALLMARK_ESTROGEN_RESPONSE_EARLY","HES2_TARGET_GENES","HALLMARK_ESTROGEN_RESPONSE_LATE"],"Description":["Genes defining early response to estrogen.","Genes containing one or more binding sites for UniProt:Q9Y543 (HES2) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Genes defining late response to estrogen."],"%":[0.0862068965517241,0.0421892816419612,0.0797546012269939],"numDEInCat":[15,37,13],"numInCat":[174,877,163],"p":[2.14863163870354e-06,1.17935266763681e-05,2.39677278225732e-05],"adj_p":[0.00235275164438038,0.00645695585531152,0.00874822065523922],"Genes Mapped to DB Windows":["AQP3: CHPT1: FKBP5: FLNB: GFRA1: KDM4B: MYBL1: PPIF: RAB31: SCNN1A: SEC14L2: SLC1A4: SULT2B1: SYBU: WWC1","AQP3: ATF6: C1orf21: CDK5RAP2: CECR2: CUEDC1: CYB5A: DLEU2: EHF: EIF2S3: EML2: FLNB: GCLC: IQGAP2: KIAA0232: LRBA: MCCC2: MRPS23: MTMR9: NEDD4L: NFIC: NFX1: NR4A1: NRP1: PDCD6: PTPRM: RAB31: RCL1: SCNN1A: SGK3: SLC38A1: SQOR: SYBU: TEX2: TMSB4X: TNFRSF1A: VWA8","CD9: CHPT1: EMP2: FKBP5: FLNB: HOMER2: PPIF: PRLR: RAB31: SCNN1A: SLC1A4: SORD: SULT2B1"]},"columns":[{"id":"Gene Set","name":"Gene Set","type":"character","cell":[{"name":"a","attribs":{"href":null,"target":"_blank"},"children":["HALLMARK ESTROGEN RESPONSE EARLY"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["HES2 TARGET GENES"]},{"name":"a","attribs":{"href":null,"target":"_blank"},"children":["HALLMARK ESTROGEN RESPONSE LATE"]}],"html":true,"minWidth":150},{"id":"Description","name":"Description","type":"character","cell":[{"name":"span","attribs":{"title":"Genes defining early response to estrogen."},"children":["Genes defining early response to estrogen."]},{"name":"span","attribs":{"title":"Genes containing one or more binding sites for UniProt:Q9Y543 (HES2) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization."},"children":["Genes containing one or more binding sites for UniProt:Q9Y543 (HES2) in their promoter regions (T..."]},{"name":"span","attribs":{"title":"Genes defining late response to estrogen."},"children":["Genes defining late response to estrogen."]}],"minWidth":150},{"id":"%","name":"% DB","type":"numeric","cell":["8.6%","4.2%","8.0%"],"maxWidth":80},{"id":"numDEInCat","name":"Total Mapped To DB","type":"numeric","maxWidth":80},{"id":"numInCat","name":"Total Mapped","type":"numeric","cell":["174","877","163"],"maxWidth":80},{"id":"p","name":"p","type":"numeric","cell":["2.15e-06","1.18e-05","2.40e-05"],"maxWidth":70},{"id":"adj_p","name":"p<sub>adj<\/sub>","type":"numeric","cell":["0.002","0.006","0.009"],"html":true,"maxWidth":70},{"id":"Genes Mapped to DB Windows","name":"Genes Mapped to DB Windows","type":"character","cell":[{"name":"span","attribs":{"title":"AQP3: CHPT1: FKBP5: FLNB: GFRA1: KDM4B: MYBL1: PPIF: RAB31: SCNN1A: SEC14L2: SLC1A4: SULT2B1: SYBU: WWC1"},"children":["AQP3: CHPT1: FKBP5: FLNB: GFRA1: KDM4B: MYBL1: PPIF: RAB31: SCNN1A: SEC14L2: SLC1A4: SULT2B1: SYB..."]},{"name":"span","attribs":{"title":"AQP3: ATF6: C1orf21: CDK5RAP2: CECR2: CUEDC1: CYB5A: DLEU2: EHF: EIF2S3: EML2: FLNB: GCLC: IQGAP2: KIAA0232: LRBA: MCCC2: MRPS23: MTMR9: NEDD4L: NFIC: NFX1: NR4A1: NRP1: PDCD6: PTPRM: RAB31: RCL1: SCNN1A: SGK3: SLC38A1: SQOR: SYBU: TEX2: TMSB4X: TNFRSF1A: VWA8"},"children":["AQP3: ATF6: C1orf21: CDK5RAP2: CECR2: CUEDC1: CYB5A: DLEU2: EHF: EIF2S3: EML2: FLNB: GCLC: IQGAP2..."]},{"name":"span","attribs":{"title":"CD9: CHPT1: EMP2: FKBP5: FLNB: HOMER2: PPIF: PRLR: RAB31: SCNN1A: SLC1A4: SORD: SULT2B1"},"children":["CD9: CHPT1: EMP2: FKBP5: FLNB: HOMER2: PPIF: PRLR: RAB31: SCNN1A: SLC1A4: SORD: SULT2B1"]}],"minWidth":200}],"filterable":true,"showPageSizeOptions":true,"dataKey":"fa3c460777a64056d516538d66358e66"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
</div>
<div id="windows-with-increased-h3k27ac-binding"
class="section level2 tabset">
<h2 class="tabset">Windows With Increased H3K27ac Binding</h2>
<pre class="r"><code>pwf_up &lt;- tibble(gene_id  = unique(gtf_gene$gene_id)) %&gt;% 
  left_join(mapped_ids, by = &quot;gene_id&quot;) %&gt;% 
  dplyr::filter(!is.na(n_peaks)) %&gt;% 
  mutate(diff = up) %&gt;% 
  with(
    nullp(
      structure(diff, names = gene_id),
      genome = config$genome$build,
      bias.data = log1p(n_peaks),
      plot.fit = FALSE
    )
  )
goseq_up &lt;- tibble(gs_name = character(), pval = numeric(), adj_p = numeric())
if (sum(pwf_up$DEgenes) &gt; 0) {
  goseq_up &lt;- goseq(
    pwf_up, gene2cat = gs_by_geneid, method = &quot;Hypergeometric&quot;
  ) %&gt;% 
    dplyr::rename(gs_name = category, pval = over_represented_pvalue) %&gt;% 
    dplyr::select(-starts_with(&quot;under&quot;)) %&gt;% 
    as_tibble() %&gt;% 
    arrange(pval) %&gt;% 
    dplyr::filter(numDEInCat &gt; 0) %&gt;% 
    mutate(adj_p = p.adjust(pval, adj_method)) %&gt;% 
    dplyr::filter(numDEInCat &gt;= min_sig)
}
any_goseq_up &lt;- sum(goseq_up$adj_p &lt; enrich_alpha) &gt; 0
tg_up &lt;- make_tbl_graph(
  goseq_up, 
  gs = gs_by_gsid %&gt;% 
    lapply(intersect, rownames(subset(pwf_up, DEgenes))) %&gt;%
    .[vapply(., length, integer(1)) &gt; 0]
)
plot_network_up &lt;- length(tg_up) &gt;= min_network_size</code></pre>
<div id="results-table-2" class="section level3">
<h3>Results Table</h3>
<pre class="r"><code>cp &lt;- htmltools::tags$caption(
  htmltools::em(
    glue(
      &quot;
    All {sum(goseq_up$adj_p &lt; enrich_alpha)} gene sets considered as 
    significantly enriched (p
    &quot;
    ),
    htmltools::tags$sub(&quot;adj&quot;),
    htmltools::em(
      glue(
        &quot;
         &lt; {enrich_alpha}) amongst the windows showing evidence of increased 
        {target} binding. Genes mapped to an increasing window were 
        compared to those mapped to any merged window. A standard (unbiased)
        hypergeomtric test was used to test for enrichment.
        &quot;
      )
    )
  )
)
tbl &lt;- goseq_up %&gt;% 
  dplyr::filter(adj_p &lt; enrich_alpha) %&gt;%
  left_join(
    dplyr::select(msigdb, gs_name, gene_name, gene_id, gs_url, gs_description),
    by = &quot;gs_name&quot;
  ) %&gt;% 
  dplyr::filter(
    gene_id %in% unlist(subset(merged_results, status == &quot;Up&quot;)$gene_id)
  ) %&gt;% 
  dplyr::select(-gene_id) %&gt;% 
  chop(gene_name) %&gt;% 
  mutate(
    gene_name = vapply(gene_name, paste, character(1), collapse = &quot;: &quot;),
    `%` = numDEInCat / numInCat
  ) %&gt;% 
  dplyr::select(
    `Gene Set` = gs_name, Description = gs_description,
    `%`, numDEInCat, numInCat, p = pval, adj_p, 
    `Genes Mapped to Gained Windows` = gene_name
  ) %&gt;% 
  reactable(
    filterable = TRUE,
    showPageSizeOptions = TRUE,
    columns = list(
      &quot;Gene Set&quot; = colDef(
        minWidth = 150,
        cell = function(value) htmltools::tags$a(
          href = gs_url[[value]], 
          target = &quot;_blank&quot;, 
          str_replace_all(value, &quot;_&quot;, &quot; &quot;)
        ),
        html = TRUE
      ),
      Description = colDef(
        minWidth = 150,
        cell = function(value) with_tooltip(value, width = 100)
      ),
      &quot;%&quot; = colDef(
        name = &quot;% Gained&quot;,
        cell = function(value) percent(value, 0.1),
        maxWidth = 80
      ),
      numDEInCat = colDef(name = &quot;Mapped to Gained Windows&quot;, maxWidth = 80),
      numInCat = colDef(
        name = &quot;Total Mapped&quot;,
        maxWidth = 80,
        cell = function(value) comma(value, 1)
      ),
      p = colDef(
        cell = function(value) {
          ifelse(
            value &lt; 0.001, sprintf(&quot;%.2e&quot;, value), sprintf(&quot;%.3f&quot;, value)
          )
        },
        maxWidth = 70
      ),
      adj_p = colDef(
        name = &quot;p&lt;sub&gt;adj&lt;/sub&gt;&quot;, html = TRUE,
        cell = function(value) {
          ifelse(
            value &lt; 0.001, sprintf(&quot;%.2e&quot;, value), sprintf(&quot;%.3f&quot;, value)
          )
        },
        maxWidth = 70
      ),
      &quot;Genes Mapped to Gained Windows&quot; = colDef(
        cell = function(value) with_tooltip(value, width = 100),
        minWidth = 200
      )
    )
  )
div(class = &quot;table&quot;,
  div(class = &quot;table-header&quot;,
      div(class = &quot;caption&quot;, cp),
      tbl
  )
)</code></pre>
<div class="table">
<div class="table-header">
<div class="caption">
<caption>
<em>
All 3 gene sets considered as 
significantly enriched (p
<sub>adj</sub>
<em> &lt; 0.05) amongst the windows showing evidence of increased 
H3K27ac binding. Genes mapped to an increasing window were 
compared to those mapped to any merged window. A standard (unbiased)
hypergeomtric test was used to test for enrichment.</em>
</em>
</caption>
</div>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-eb110bfd8aba667049ab" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-eb110bfd8aba667049ab">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"Gene Set":["HALLMARK_ESTROGEN_RESPONSE_EARLY","HES2_TARGET_GENES","HALLMARK_ESTROGEN_RESPONSE_LATE"],"Description":["Genes defining early response to estrogen.","Genes containing one or more binding sites for UniProt:Q9Y543 (HES2) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization.","Genes defining late response to estrogen."],"%":[0.0862068965517241,0.0421892816419612,0.0797546012269939],"numDEInCat":[15,37,13],"numInCat":[174,877,163],"p":[2.0370661683289e-06,1.06796607565638e-05,2.29102407177082e-05],"adj_p":[0.0022204021234785,0.00582041511232729,0.00832405412743398],"Genes Mapped to Gained Windows":["AQP3: CHPT1: FKBP5: FLNB: GFRA1: KDM4B: MYBL1: PPIF: RAB31: SCNN1A: SEC14L2: SLC1A4: SULT2B1: SYBU: WWC1","AQP3: ATF6: C1orf21: CDK5RAP2: CECR2: CUEDC1: CYB5A: DLEU2: EHF: EIF2S3: EML2: FLNB: GCLC: IQGAP2: KIAA0232: LRBA: MCCC2: MRPS23: MTMR9: NEDD4L: NFIC: NFX1: NR4A1: NRP1: PDCD6: PTPRM: RAB31: RCL1: SCNN1A: SGK3: SLC38A1: SQOR: SYBU: TEX2: TMSB4X: TNFRSF1A: VWA8","CD9: CHPT1: EMP2: FKBP5: FLNB: HOMER2: PPIF: PRLR: RAB31: SCNN1A: SLC1A4: SORD: SULT2B1"]},"columns":[{"id":"Gene Set","name":"Gene Set","type":"character","cell":[{"name":"a","attribs":{"href":null,"target":"_blank"},"children":["HALLMARK ESTROGEN RESPONSE EARLY"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["HES2 TARGET GENES"]},{"name":"a","attribs":{"href":null,"target":"_blank"},"children":["HALLMARK ESTROGEN RESPONSE LATE"]}],"html":true,"minWidth":150},{"id":"Description","name":"Description","type":"character","cell":[{"name":"span","attribs":{"title":"Genes defining early response to estrogen."},"children":["Genes defining early response to estrogen."]},{"name":"span","attribs":{"title":"Genes containing one or more binding sites for UniProt:Q9Y543 (HES2) in their promoter regions (TSS -1000,+100 bp) as identified by GTRD version 20.06 ChIP-seq harmonization."},"children":["Genes containing one or more binding sites for UniProt:Q9Y543 (HES2) in their promoter regions (T..."]},{"name":"span","attribs":{"title":"Genes defining late response to estrogen."},"children":["Genes defining late response to estrogen."]}],"minWidth":150},{"id":"%","name":"% Gained","type":"numeric","cell":["8.6%","4.2%","8.0%"],"maxWidth":80},{"id":"numDEInCat","name":"Mapped to Gained Windows","type":"numeric","maxWidth":80},{"id":"numInCat","name":"Total Mapped","type":"numeric","cell":["174","877","163"],"maxWidth":80},{"id":"p","name":"p","type":"numeric","cell":["2.04e-06","1.07e-05","2.29e-05"],"maxWidth":70},{"id":"adj_p","name":"p<sub>adj<\/sub>","type":"numeric","cell":["0.002","0.006","0.008"],"html":true,"maxWidth":70},{"id":"Genes Mapped to Gained Windows","name":"Genes Mapped to Gained Windows","type":"character","cell":[{"name":"span","attribs":{"title":"AQP3: CHPT1: FKBP5: FLNB: GFRA1: KDM4B: MYBL1: PPIF: RAB31: SCNN1A: SEC14L2: SLC1A4: SULT2B1: SYBU: WWC1"},"children":["AQP3: CHPT1: FKBP5: FLNB: GFRA1: KDM4B: MYBL1: PPIF: RAB31: SCNN1A: SEC14L2: SLC1A4: SULT2B1: SYB..."]},{"name":"span","attribs":{"title":"AQP3: ATF6: C1orf21: CDK5RAP2: CECR2: CUEDC1: CYB5A: DLEU2: EHF: EIF2S3: EML2: FLNB: GCLC: IQGAP2: KIAA0232: LRBA: MCCC2: MRPS23: MTMR9: NEDD4L: NFIC: NFX1: NR4A1: NRP1: PDCD6: PTPRM: RAB31: RCL1: SCNN1A: SGK3: SLC38A1: SQOR: SYBU: TEX2: TMSB4X: TNFRSF1A: VWA8"},"children":["AQP3: ATF6: C1orf21: CDK5RAP2: CECR2: CUEDC1: CYB5A: DLEU2: EHF: EIF2S3: EML2: FLNB: GCLC: IQGAP2..."]},{"name":"span","attribs":{"title":"CD9: CHPT1: EMP2: FKBP5: FLNB: HOMER2: PPIF: PRLR: RAB31: SCNN1A: SLC1A4: SORD: SULT2B1"},"children":["CD9: CHPT1: EMP2: FKBP5: FLNB: HOMER2: PPIF: PRLR: RAB31: SCNN1A: SLC1A4: SORD: SULT2B1"]}],"minWidth":200}],"filterable":true,"showPageSizeOptions":true,"dataKey":"5ae8f4f44993b8b458783f3a0f2b8565"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
</div>
<div id="windows-with-decreased-h3k27ac-binding"
class="section level2 tabset">
<h2 class="tabset">Windows With Decreased H3K27ac Binding</h2>
<pre class="r"><code>pwf_down &lt;- tibble(gene_id  = unique(gtf_gene$gene_id)) %&gt;% 
  left_join(mapped_ids, by = &quot;gene_id&quot;) %&gt;% 
  dplyr::filter(!is.na(n_peaks)) %&gt;% 
  mutate(diff = down) %&gt;% 
  with(
    nullp(
      structure(diff, names = gene_id),
      genome = config$genome$build,
      bias.data = log1p(n_peaks),
      plot.fit = FALSE
    )
  )
goseq_down &lt;- tibble(gs_name = character(), pval = numeric(), adj_p = numeric())
if (sum(pwf_down$DEgenes) &gt; 0) {
  goseq_down &lt;- goseq(
    pwf_down, config$genome$build, gene2cat = gs_by_geneid, method = &quot;Hypergeometric&quot;
  ) %&gt;% 
    dplyr::rename(gs_name = category, pval = over_represented_pvalue) %&gt;% 
    dplyr::select(-starts_with(&quot;under&quot;)) %&gt;% 
    as_tibble() %&gt;% 
    arrange(pval) %&gt;% 
    dplyr::filter(numDEInCat &gt; 0) %&gt;% 
    mutate(adj_p = p.adjust(pval, adj_method)) %&gt;% 
    dplyr::filter(numDEInCat &gt;= min_sig)
}
any_goseq_down &lt;- sum(goseq_down$adj_p &lt; enrich_alpha) &gt; 0
tg_down &lt;- make_tbl_graph(
  goseq_down, 
  gs = gs_by_gsid %&gt;% 
    lapply(intersect, rownames(subset(pwf_down, DEgenes))) %&gt;%
    .[vapply(., length, integer(1)) &gt; 0]
)
plot_network_down &lt;- length(tg_down) &gt;= min_network_size</code></pre>
<p>No enrichment was found amongst genes mapped to sites showing
decreased H3K27ac binding</p>
</div>
</div>
<div id="comparison-with-rna-seq" class="section level1">
<h1>Comparison With RNA Seq</h1>
<pre class="r"><code>library(fgsea)
library(metap)
rna_status &lt;- rnaseq %&gt;% 
  mutate(
    de_status = case_when(
      !!sym(rna_fdr_col) &lt; fdr_alpha &amp; !!sym(rna_lfc_col) &lt; 0 ~ &quot;Down&quot;,
      !!sym(rna_fdr_col) &lt; fdr_alpha &amp; !!sym(rna_lfc_col) &gt; 0 ~ &quot;Up&quot;,
      !!sym(rna_fdr_col) &gt;= fdr_alpha ~ &quot;Unchanged&quot;
    )
  ) %&gt;% 
  dplyr::select(gene_id, gene_name, de_status) %&gt;% 
  bind_rows(
    gtf_gene %&gt;% 
      subset(!gene_id %in% rnaseq$gene_id) %&gt;% 
      select(gene_id, gene_name) %&gt;% 
      mcols() %&gt;% 
      as_tibble() %&gt;% 
      mutate(de_status = &quot;Undetected&quot;)
  ) %&gt;% 
  mutate(
    de_status = factor(
      de_status, levels = str_to_title(names(colours$direction))
    )
  ) %&gt;% 
  arrange(gene_id)
rna_p_col &lt;- colnames(rnaseq)[
  str_detect(str_to_lower(colnames(rnaseq)), &quot;pval|p\\.val&quot;)
][1]
stopifnot(length(rna_p_col) == 1)
diff_windows &lt;- merged_results %&gt;% 
  splitAsList(.$status) %&gt;% 
  lapply(select, gene_id) %&gt;% 
  lapply(mcols) %&gt;% 
  lapply(as_tibble) %&gt;% 
  lapply(pull, &quot;gene_id&quot;) %&gt;% 
  lapply(unlist) %&gt;% 
  lapply(unique)</code></pre>
<div id="distribution-of-detected-genes" class="section level2 tabset">
<h2 class="tabset">Distribution Of Detected Genes</h2>
<div id="detected-genes-by-h3k27ac-binding-region"
class="section level3">
<h3>Detected Genes By H3K27ac Binding Region</h3>
<p>The distribution of merged windows, where H3K27ac was considered as
present, was firstly compared to genes within the RNA-Seq dataset. Genes
were simply considered as detected if present in the RNA-Seq data, or
undetected if not present. Gene-centric regions were as defined
previously.</p>
<pre class="r"><code>w &lt;- 0.8
min_p &lt;- 0.025
merged_results %&gt;%
  mutate(
    any_detected = ifelse(
      vapply(gene_name, length, integer(1)) &gt; 1,
      &quot;Mapped to Detected Genes&quot;,
      &quot;Mapped to Undetected Genes&quot;
    )
  ) %&gt;% 
  plotPie(fill = &quot;region&quot;, x = &quot;any_detected&quot;, label_size = 4, width = w) +
  geom_text(
    aes(xlab, ylab, label =lab),
    data = . %&gt;% 
      mutate(
        xlab = x + w*(r + 0.1)*sin(label_radians),
        ylab = 1 + w*(r + 0.1)*cos(label_radians),
        lab = glue(&quot;{region}\n{percent(p, 0.1)}&quot;) %&gt;% 
          str_wrap(10)
      ) %&gt;% 
      dplyr::filter(p * N &gt; min_p * max(N)),
    size = 3
  ) +
  geom_segment(
    aes(x_inner, y_inner, xend = x_outer, yend = y_outer),
     data = . %&gt;% 
      mutate(
        x_inner = x + w*r*sin(label_radians),
        y_inner = 1 + w*r*cos(label_radians),
        x_outer = x + w*(r + 0.01)*sin(label_radians),
        y_outer = 1 + w*(r + 0.01)*cos(label_radians),
      ) %&gt;% 
      dplyr::filter(p * N &gt; min_p * max(N))
  ) +
  scale_fill_manual(
    values = colours$regions %&gt;% 
      setNames(regions[names(.)]) 
  ) +
  labs(x = &quot;&quot;, fill = &quot;Region&quot;) +
  theme(panel.grid = element_blank())</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-region-detected-1.png" alt="*Distribution of H3K27ac-bound windows by region, according to whether the window is mapped to a detected gene in the RNA-Seq dataset.*" width="960" />
<p class="caption">
<em>Distribution of H3K27ac-bound windows by region, according to
whether the window is mapped to a detected gene in the RNA-Seq
dataset.</em>
</p>
</div>
</div>
<div id="detected-genes-by-feature" class="section level3">
<h3>Detected Genes By Feature</h3>
<pre class="r"><code>merged_results %&gt;% 
  mutate(
    any_detected = ifelse(
      vapply(gene_name, length, integer(1)) &gt; 1,
      &quot;Mapped to Detected Genes&quot;,
      &quot;Mapped to Undetected Genes&quot;
    )
  ) %&gt;% 
  plotPie(fill = &quot;feature&quot;, x = &quot;any_detected&quot;, label_size = 4, width = w) +
  geom_text(
    aes(xlab, ylab, label =lab),
    data = . %&gt;% 
      mutate(
        xlab = x + w*(r + 0.1)*sin(label_radians),
        ylab = 1 + w*(r + 0.1)*cos(label_radians),
        lab = glue(&quot;{feature}\n{percent(p, 0.1)}&quot;) %&gt;% 
          str_wrap(10)
      ) %&gt;% 
      dplyr::filter(p * N &gt; min_p * max(N)),
    size = 3
  ) +
  geom_segment(
    aes(x_inner, y_inner, xend = x_outer, yend = y_outer),
     data = . %&gt;% 
      mutate(
        x_inner = x + w*r*sin(label_radians),
        y_inner = 1 + w*r*cos(label_radians),
        x_outer = x + w*(r + 0.01)*sin(label_radians),
        y_outer = 1 + w*(r + 0.01)*cos(label_radians),
      ) %&gt;% 
      dplyr::filter(p * N &gt; min_p * max(N))
  ) +
  scale_fill_manual(
    values = colours$features %&gt;% 
      setNames(str_sep_to_title(names(.)))
  ) +
  labs(x = &quot;&quot;, fill = &quot;External Feature&quot;) +
  theme(panel.grid = element_blank())</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-feature-detected-1.png" alt="*Distribution of H3K27ac-bound windows by externally-defined features from enhancer_atlas_2.0_zr75.gtf.gz, according to whether the window is mapped to a detected gene in the RNA-Seq dataset.*" width="960" />
<p class="caption">
<em>Distribution of H3K27ac-bound windows by externally-defined features
from enhancer_atlas_2.0_zr75.gtf.gz, according to whether the window is
mapped to a detected gene in the RNA-Seq dataset.</em>
</p>
</div>
</div>
</div>
<div id="relationship-with-differentially-expressed-genes"
class="section level2 tabset">
<h2 class="tabset">Relationship With Differentially Expressed Genes</h2>
<pre class="r"><code>rna_dir_ranks &lt;- rnaseq %&gt;% 
  mutate(
    ranking_stat = -sign(!!sym(rna_lfc_col))*log10(!!sym(rna_p_col)) %&gt;% 
      setNames(gene_id)
  ) %&gt;% 
  arrange(ranking_stat) %&gt;% 
  dplyr::filter(!!sym(rna_p_col) &lt; 1) %&gt;% 
  pull(&quot;ranking_stat&quot;) 
rna_sig_ranks &lt;- rna_dir_ranks %&gt;% 
  abs() %&gt;% 
  sort()
dir_gs &lt;- merged_results %&gt;% 
  select(status, starts_with(&quot;gene&quot;)) %&gt;% 
  as_tibble() %&gt;% 
  mutate(group = glue(&quot;{target} {status}&quot;)) %&gt;% 
  split(.$group)%&gt;% 
  lapply(pull, &quot;gene_id&quot;) %&gt;% 
  lapply(unlist) %&gt;% 
  lapply(unique) %&gt;% 
  .[vapply(., length, integer(1)) &gt;= min_gs_size]
reg_dir_gs &lt;- merged_results %&gt;% 
  select(
    overlaps_ref, status, any_of(c(&quot;region&quot;, &quot;feature&quot;)), starts_with(&quot;gene&quot;)
  ) %&gt;% 
  as_tibble() %&gt;% 
  mutate(group = glue(&quot;{region} / {target} {status}&quot;)) %&gt;% 
  split(.$group)%&gt;% 
  lapply(pull, &quot;gene_id&quot;) %&gt;% 
  lapply(unlist) %&gt;% 
  lapply(unique) %&gt;% 
  .[vapply(., length, integer(1)) &gt;= min_gs_size]
if (has_features) {
  feat_dir_gs &lt;- merged_results %&gt;% 
  select(
    overlaps_ref, status, any_of(c(&quot;feature&quot;)), starts_with(&quot;gene&quot;)
  ) %&gt;% 
  as_tibble() %&gt;% 
  mutate(group = glue(&quot;{feature} / {target} {status}&quot;)) %&gt;% 
  split(.$group)%&gt;% 
  lapply(pull, &quot;gene_id&quot;) %&gt;% 
  lapply(unlist) %&gt;% 
  lapply(unique) %&gt;% 
  .[vapply(., length, integer(1)) &gt;= min_gs_size]
}</code></pre>
<div id="de-genes-associated-with-changed-h3k27ac-binding"
class="section level3">
<h3>DE Genes Associated with Changed H3K27ac Binding</h3>
<pre class="r"><code>de_genes_db_regions &lt;- tibble()
any_diff_sites &lt;- sum(mcols(merged_results)[[fdr_column]] &lt; fdr_alpha) &gt; 0
any_de &lt;- any(rnaseq$DE)
para &lt;- &quot;&quot;
if (any_de &amp; any_diff_sites) {
  ft_data &lt;- rnaseq %&gt;%
    dplyr::filter(gene_id %in% unlist(diff_windows)) %&gt;% 
    mutate(
      window = case_when(
        gene_id %in% unlist(diff_windows[c(&quot;Up&quot;, &quot;Down&quot;)]) ~ &quot;Diff&quot;,
        TRUE ~ &quot;Unchanged&quot;
      ) %&gt;%
        fct_infreq()
    ) %&gt;% 
    group_by(DE, window) %&gt;% 
    tally() %&gt;% 
    pivot_wider(
      names_from = &quot;DE&quot;, values_from = &quot;n&quot;, 
      names_prefix = &quot;DE_&quot;, values_fill = 0
    ) %&gt;% 
    as.data.frame() %&gt;% 
    column_to_rownames(&quot;window&quot;) %&gt;% 
    as.matrix()
  ft &lt;- fisher.test(ft_data)
  para &lt;- glue(
    &quot;
    Using the {comma(length(unique(unlist(diff_windows))))} genes mapped to a 
    {target} binding site, genes were tested for any association between sites 
    showing evidence of diferential binding and differential expression.
    {comma(length(unique(unlist(diff_windows[c(&#39;Up&#39;, &#39;Down&#39;)]))))} genes were 
    mapped to at least one site showing differential {target} binding with 
    {comma(ft_data[&#39;Diff&#39;, &#39;DE_TRUE&#39;])} 
    ({percent(ft_data[&#39;Diff&#39;, &#39;DE_TRUE&#39;] / sum(ft_data[&#39;Diff&#39;,]))}) of these
    being considered differentially expressed. This compares to 
    {percent(ft_data[&#39;Unchanged&#39;, &#39;DE_TRUE&#39;] / sum(ft_data[&#39;Unchanged&#39;, ]), 0.1)}
    of genes where no differential {target} binding was detected. Fisher&#39;s 
    Exact test for any association gave this a p-value of 
    {ifelse(ft$p.value &lt; 0.01, sprintf(&#39;%.2e&#39;, ft$p.value), round(ft$p.value, 3))}.
    &quot;
  )
}</code></pre>
<p>Using the 12,351 genes mapped to a H3K27ac binding site, genes were
tested for any association between sites showing evidence of diferential
binding and differential expression. 249 genes were mapped to at least
one site showing differential H3K27ac binding with 26 (10%) of these
being considered differentially expressed. This compares to 0.7% of
genes where no differential H3K27ac binding was detected. Fisher’s Exact
test for any association gave this a p-value of 7.61e-21.</p>
<pre class="r"><code>cmn_genes &lt;- intersect(
  rnaseq %&gt;% dplyr::filter(!!sym(rna_fdr_col) &lt; fdr_alpha) %&gt;% pull(&quot;gene_id&quot;),
  merged_results %&gt;% 
    filter(!!sym(fdr_column) &lt; fdr_alpha) %&gt;% 
    as_tibble() %&gt;% 
    pull(&quot;gene_id&quot;) %&gt;% 
    unlist()
)
de_genes_db_regions &lt;- merged_results %&gt;% 
  filter(!!sym(fdr_column) &lt; fdr_alpha) %&gt;% 
  filter(vapply(gene_id, function(x) any(x %in% cmn_genes), logical(1))) %&gt;% 
  as_tibble() %&gt;% 
  dplyr::select(
    range, ChIP_logCPM = AveExpr, ChIP_logFC = logFC, ChIP_FDR := !!sym(fdr_column), overlaps_ref,
    status, any_of(c(&quot;region&quot;, &quot;feature&quot;)), gene_id
  ) %&gt;% 
  unnest(gene_id) %&gt;% 
  dplyr::filter(gene_id %in% cmn_genes) %&gt;% 
  left_join(
    rnaseq %&gt;% 
      dplyr::filter(gene_id %in% cmn_genes) %&gt;% 
      dplyr::select(
        all_of(c(&quot;gene_id&quot;, &quot;gene_name&quot;)), 
        RNA_logFC := !!sym(rna_lfc_col), RNA_FDR := !!sym(rna_fdr_col)
      ),
    by = &quot;gene_id&quot;
  ) %&gt;% 
  arrange(RNA_FDR) %&gt;% 
  # dplyr::slice(1:20) %&gt;%
  mutate(
    dist2Gene = mapply(
      function(x, y) {
        distance(GRanges(x), subset(gtf_gene, gene_id == y)) 
      },
      x = range,
      y = gene_id
    )
  ) %&gt;% 
  mutate(
    region = case_when(
      dist2Gene &gt; params$gene_regions$upstream &amp; str_detect(region, &quot;Promoter|Exon|Intron&quot;) ~ paste(
        str_extract(region, &quot;.*(Promoter|Exon|Intron)&quot;), &quot;(Other Gene)&quot;),
      TRUE ~ as.character(region)
    ),
    region = factor(
      region, 
      levels = c(
        levels(merged_results$region), 
        paste(
          str_extract(levels(merged_results$region), &quot;.*(Promoter|Exon|Intron)&quot;), 
          &quot;(Other Gene)&quot;
        )
      ) %&gt;% 
        unique()
    )
  ) %&gt;% 
  dplyr::select(
    gene_name, gene_id, starts_with(&quot;RNA&quot;), range, dist2Gene, 
    starts_with(&quot;ChIP&quot;), everything()
  ) %&gt;% 
  droplevels()</code></pre>
<pre class="r"><code>range_js &lt;- JS(
  &quot;function(values) {
    var min_val = Math.round(100 * Math.min(...values)) / 100
    var max_val = Math.round(100 * Math.max(...values)) / 100
    if (min_val == max_val) {
      var ret_val = min_val.toString()
    } else {
      var ret_val = &#39;[&#39; + min_val.toString() + &#39;, &#39; + max_val.toString() + &#39;]&#39;
    }
    return ret_val
  }&quot;
)
max_signal &lt;- max(merged_results$AveExpr)
feat_def &lt;- NULL
if (has_features) feat_def = list(
  feature = colDef (
    name = &quot;Feature&quot;,
    aggregate = &quot;unique&quot;
  )
)
cp &lt;-  htmltools::em(
  glue(
    &quot;
    Differentially Expressed (DE) genes were checked for {target} binding
    regions which showed evidence of changed binding. 
    {comma(length(unique(de_genes_db_regions$range)), 1)} unique regions were 
    mapped to {comma(length(unique(de_genes_db_regions$gene_id)), 1)} DE genes. 
    Data are presented in a gene-centric manner and ranges may map to multiple
    genes. The full set of ranges considered to be showing evidence of altered 
    {target} binding are shown for each gene. The summarised range containing 
    all mapped ranges is given in the collapsed row, along with the range of 
    distances to gene and the range of signal levels. In the collapsed row, the 
    value for logFC represents the average across all mapped regions, with the 
    maximum FDR also shown. If {target} binding occurs within another gene this 
    is indicated in the &#39;Region&#39; column.
    All sites and genes were exported as {basename(all_out$de_genes)}.
    &quot;
  )
)
fs &lt;- 12
tbl &lt;- de_genes_db_regions %&gt;% 
  reactable(
    groupBy = &quot;gene_name&quot;,
    filterable = TRUE,
    borderless = TRUE,
    showPageSizeOptions = TRUE,
    columns = c(
      list(
        gene_name = colDef(
          name = &quot;Gene&quot;, maxWidth = fs*11,
          style = list(
            borderRight = &quot;1px solid rgba(0, 0, 0, 0.1)&quot;,
            fontSize = fs
          )
        ),
        gene_id = colDef(
          name = &quot;ID&quot;, aggregate = &quot;unique&quot;, cell = function(value) &quot;&quot;,
          show = FALSE
        ),
        RNA_logFC = colDef(
          name = &quot;logFC&quot;,
          aggregate = &quot;mean&quot;, 
          style = JS(
            glue(
            &quot;function(rowInfo) {
              var value = rowInfo.row[&#39;RNA_logFC&#39;]
              if (value &lt; 0) {
                var color = &#39;{{direction_colours[[&#39;Down&#39;]]}}&#39;
              } else if (value &gt; 0) {
                var color = &#39;{{direction_colours[[&#39;Up&#39;]]}}&#39;
              } else {
                var color = &#39;#000000&#39;
              }
              return { color: color, fontSize: {{fs}} }
            }&quot;,
            .open = &quot;{{&quot;, .close = &quot;}}&quot;
            )
          ),
          format = colFormat(digits= 2),
          maxWidth = fs*6,
          cell = function(value) &quot;&quot;
        ),
        RNA_FDR = colDef(
          name = &quot;FDR&quot;,
          aggregate = &quot;min&quot;,
          format = colFormat(digits = 4),
          maxWidth = fs*6,
          cell = function(value) &quot;&quot;,
          style = list(
            borderRight = &quot;1px solid rgba(0, 0, 0, 0.1)&quot;, 
            fontSize = fs
          )
        ),
        range = colDef(
          name = &quot;Range&quot;,
          minWidth = 160,
          aggregate = JS(
            &quot;function(values){
              var chrom = [];
              var rng = [];
              var start = [];
              var end = [];
              for (i = 0; i &lt; values.length; i++) {
                chrom[i] = values[i].split(&#39;:&#39;)[0];
                rng[i] = values[i].split(&#39;:&#39;)[1];
                start[i] = rng[i].split(&#39;-&#39;)[0];
                end[i] = rng[i].split(&#39;-&#39;)[1];
              }
              
              min_start = Math.min(...start);
              max_end = Math.max(...end);
              var ret_val = [...new Set(chrom)] + &#39;:&#39; + min_start.toString() + &#39;-&#39; + max_end.toString();
              
              return ret_val
              
            }&quot;
          )
        ),
        dist2Gene = colDef(
          name = &quot;Distance to Gene&quot;,
          aggregate = range_js,
          format = colFormat(digits = 0, separators = TRUE),
          maxWidth = fs*8,
          style = list(
            borderRight = &quot;1px solid rgba(0, 0, 0, 0.1)&quot;,
            fontSize = fs
          ),
        ),
        ChIP_logCPM = colDef(
          name = &quot;logCPM&quot;,
          maxWidth = fs*6,
          aggregate = range_js,
          format = colFormat(digits = 2),
          style = function(value) bar_style(
            width = value / max_signal, fontSize = fs
          )
        ),
        ChIP_logFC = colDef(
          name = &quot;logFC&quot;,
          aggregate = &quot;mean&quot;,
          maxWidth = fs*5,
          format = colFormat(digits = 2),
          cell = function(value) sprintf(&quot;%.2f&quot;, value),
          style =  JS(
            glue(
            &quot;function(rowInfo) {
              var value = rowInfo.row[&#39;ChIP_logFC&#39;]
              if (value &lt; 0) {
                var color = &#39;{{direction_colours[[&#39;Down&#39;]]}}&#39;
              } else if (value &gt; 0) {
                var color = &#39;{{direction_colours[[&#39;Up&#39;]]}}&#39;
              } else {
                var color = &#39;#000000&#39;
              }
              return { color: color, fontSize: {{fs}} }
            }&quot;,
            .open = &quot;{{&quot;, .close = &quot;}}&quot;
            )
          )
        ),
        ChIP_FDR = colDef(
          name = &quot;FDR&quot;,
          aggregate = &quot;max&quot;,
          maxWidth = 10 + fs*5,
          format = colFormat(prefix = &quot;&lt; &quot;, digits = 4),
          cell = function(value) ifelse(
            value &lt; 0.001,
            sprintf(&quot;%.2e&quot;, value),
            sprintf(&quot;%.3f&quot;, value)
          )
        ),
        overlaps_ref = colDef(
          name = &quot;Macs Peak&quot;,
          show = FALSE,
          aggregate = &quot;frequency&quot;,
          cell = function(value) ifelse(value, &quot;\u2714 Yes&quot;, &quot;\u2716 No&quot;),
          style = function(value) {
            color &lt;- ifelse(value, &quot;#008000&quot;, &quot;#e00000&quot;)
            list(color = color, fontSize = fs)
          },
          maxWidth = fs*5
        ),
        status = colDef(
          name = &quot;Status&quot;,
          aggregate = &quot;frequency&quot;,
          maxWidth = fs*6,
          cell = function(value) {
            html_symbol &lt;- &quot;&quot;
            if (str_detect(value, &quot;Up&quot;)) html_symbol &lt;- &quot;\u21E7&quot;
            if (str_detect(value, &quot;Down&quot;)) html_symbol &lt;- &quot;\u21E9&quot;
            paste(html_symbol, value)
          },
          style = JS(
            glue(
            &quot;function(rowInfo) {
              var value = rowInfo.row[&#39;ChIP_logFC&#39;]
              if (value &lt; 0) {
                var color = &#39;{{direction_colours[[&#39;Down&#39;]]}}&#39;
              } else if (value &gt; 0) {
                var color = &#39;{{direction_colours[[&#39;Up&#39;]]}}&#39;
              } else {
                var color = &#39;#000000&#39;
              }
              return { color: color, fontSize: {{fs}} }
            }&quot;,
            .open = &quot;{{&quot;, .close = &quot;}}&quot;
            )
          )
        ),
        region = colDef(
          name = &quot;Binding Region&quot;,
          aggregate = &quot;unique&quot;
        )
      ),
      feat_def
    ),
    columnGroups = list(
      colGroup(name = &quot;RNA-Seq&quot;, columns = c(&quot;RNA_logFC&quot;, &quot;RNA_FDR&quot;)),
      colGroup(
        name = paste(target, &quot;ChIP-Seq&quot;), 
        columns = c(&quot;ChIP_logCPM&quot;, &quot;ChIP_logFC&quot;, &quot;ChIP_FDR&quot;, &quot;status&quot;)
      )
    ),
    theme = reactableTheme(style = list(fontSize = fs))
  )
div(class = &quot;table&quot;,
  div(class = &quot;table-header&quot;,
      div(class = &quot;caption&quot;, cp),
      tbl
  )
)</code></pre>
<div class="table">
<div class="table-header">
<div class="caption">
<em>Differentially Expressed (DE) genes were checked for H3K27ac binding
regions which showed evidence of changed binding. 
38 unique regions were 
mapped to 26 DE genes. 
Data are presented in a gene-centric manner and ranges may map to multiple
genes. The full set of ranges considered to be showing evidence of altered 
H3K27ac binding are shown for each gene. The summarised range containing 
all mapped ranges is given in the collapsed row, along with the range of 
distances to gene and the range of signal levels. In the collapsed row, the 
value for logFC represents the average across all mapped regions, with the 
maximum FDR also shown. If H3K27ac binding occurs within another gene this 
is indicated in the 'Region' column.
All sites and genes were exported as H3K27ac_E2_E2DHT-DE_genes.csv.</em>
</div>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-c016348dce47bb18ec56" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-c016348dce47bb18ec56">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"gene_name":["AQP3","AQP3","FKBP5","FKBP5","FKBP5","SEC14L2","RANBP3L","RANBP3L","HSD11B2","ZNF652","ZNF689","EHF","EHF","ARHGEF37","ARHGEF37","ARHGEF37","NRP1","AL033523.1","TTC12","PTPRJ","KDM4B","SYBU","NEDD4L","NEDD4L","NEDD4L","CSRNP1","WWC1","SASH1","CNTNAP2","CNTNAP2","AKAP13","CD9","CD9","SPRED1","VSTM2L","IQGAP2","PRKAB2","PRKAB2"],"gene_id":["ENSG00000165272","ENSG00000165272","ENSG00000096060","ENSG00000096060","ENSG00000096060","ENSG00000100003","ENSG00000164188","ENSG00000164188","ENSG00000176387","ENSG00000198740","ENSG00000156853","ENSG00000135373","ENSG00000135373","ENSG00000183111","ENSG00000183111","ENSG00000183111","ENSG00000099250","ENSG00000228793","ENSG00000149292","ENSG00000149177","ENSG00000127663","ENSG00000147642","ENSG00000049759","ENSG00000049759","ENSG00000049759","ENSG00000144655","ENSG00000113645","ENSG00000111961","ENSG00000174469","ENSG00000174469","ENSG00000170776","ENSG00000010278","ENSG00000010278","ENSG00000166068","ENSG00000132821","ENSG00000145703","ENSG00000131791","ENSG00000131791"],"RNA_logFC":[2.36942177842281,2.36942177842281,1.99899166960751,1.99899166960751,1.99899166960751,2.65364804947665,4.40570018537827,4.40570018537827,1.13830350246976,0.768462734396911,1.21208048500515,0.591050336429137,0.591050336429137,0.959466645703423,0.959466645703423,0.959466645703423,0.621252048359839,2.21378646833637,0.753322494150793,0.601234639862719,0.738557156216351,0.84070988458889,0.537173436697143,0.537173436697143,0.537173436697143,0.724290454212561,0.563530986380658,0.836022499007808,0.536655769928441,0.536655769928441,0.502026789166318,0.457093842888302,0.457093842888302,0.463356342035222,1.19659099143407,0.687472232519211,0.477686269893968,0.477686269893968],"RNA_FDR":[1.16928925721457e-11,1.16928925721457e-11,6.50784235592507e-10,6.50784235592507e-10,6.50784235592507e-10,8.86897562716368e-10,3.14499432423427e-07,3.14499432423427e-07,3.14499432423427e-07,2.63491905437816e-06,3.30744958812175e-06,1.13353850709501e-05,1.13353850709501e-05,1.81130969725679e-05,1.81130969725679e-05,1.81130969725679e-05,3.64361628603203e-05,6.5239055808791e-05,8.79863174841873e-05,0.000101952565775092,0.000101952565775092,0.00019400293929134,0.000322652428942889,0.000322652428942889,0.000322652428942889,0.00033264520126481,0.000563673949153319,0.000676942857145875,0.000888165218966668,0.000888165218966668,0.00556889493652094,0.0161653001414337,0.0161653001414337,0.0169213404627554,0.0176140096451741,0.0183443417418862,0.0256164175205572,0.0256164175205572],"range":["chr9:33430521-33432480","chr9:33436761-33437720","chr6:35694801-35695960","chr6:35696041-35696400","chr6:35699321-35700240","chr22:30804321-30804760","chr5:36285601-36286000","chr5:36286321-36286680","chr16:67492761-67497800","chr17:47466041-47470360","chr16:30626521-30627200","chr11:34692801-34693720","chr11:34771481-34771880","chr5:148980281-148980520","chr5:148980801-148981200","chr5:148993321-148993720","chr10:33529921-33530440","chr6:3669761-3670240","chr11:113211161-113213760","chr11:48063041-48063560","chr19:4989881-4993440","chr8:110592641-110593400","chr18:55901561-55903040","chr18:55952481-55953040","chr18:55953241-55953400","chr3:39172241-39173080","chr5:167710561-167710880","chr6:148579321-148580120","chr7:148082041-148082560","chr7:148082721-148083720","chr15:86080681-86081360","chr12:6375561-6376440","chr12:6376601-6377160","chr15:38508601-38508960","chr20:36618321-36618720","chr5:75709041-75709440","chr1:146580841-146582200","chr1:146585001-146587360"],"dist2Gene":[8673,3433,0,0,2960,0,0,0,21304,26205,0,10196,88876,0,0,0,0,0,0,0,0,0,0,0,0,10265,7775,13319,0,0,0,28123,29163,36076,44568,0,44484,39324],"ChIP_logCPM":[1.45153346765225,0.700892501206581,1.42555669042997,0.582034993436207,0.92297147074282,0.473100455887273,0.606095042162411,0.455728468676655,2.9091984775359,3.90228235955822,1.00749235402435,1.1544715278787,0.636297082171202,0.374180175121845,0.502332852754016,0.625458963782821,0.724846238163832,0.740886959466433,2.09873646117241,1.00548622112433,1.90520962787039,1.66898335721257,1.00421200626649,0.905340544706101,0.34845978375078,1.23196903414009,0.362933142999854,1.63299974811303,1.13504834375427,1.6618463686182,0.674078589941087,0.940232399263435,0.764341293092677,0.548135853333627,0.656977319100648,0.502936056417527,1.17093701507858,2.20998538211679],"ChIP_logFC":[1.39823281200676,1.60226003520684,1.14537306549312,1.36162510777024,0.886051545853066,1.53190859973304,1.8235971809009,1.55500414855242,0.676535786523487,0.597295829874359,1.27572496936763,1.93516864679129,1.26125807378763,1.69771015100169,2.04276815271224,1.29812999180203,1.1352933329328,1.46409182331984,0.797447357593344,1.0081356153294,1.25796927065605,1.07688442840706,0.900754736101035,1.5273570890103,1.13704556653601,1.09633499038367,1.39562411629575,0.936547832997297,1.08114896327545,0.750141995620045,1.52615885769248,1.12052492022596,0.998638164688341,1.33160935841154,1.23690015214619,1.33542681036038,2.14555014485682,0.755899581176508],"ChIP_FDR":[2.11189490548428e-06,1.91315184620398e-05,0.000214859478744076,0.000495328944987452,0.047582318360659,0.000214859478744076,6.98247627437307e-07,0.000112202275139175,0.00621012373063191,0.000153627201703339,0.000204911333118627,2.25161902535312e-10,0.00443021920402166,2.5757516834627e-05,4.19432891885972e-08,0.00109677366245181,0.00552205441596043,8.062792970994e-05,0.0131221511088425,0.0134904879247849,8.05009311171575e-07,0.000280962029208953,0.0262179210628113,7.08086172905856e-06,0.0197031586811841,0.00248498868884494,0.0441490375517978,0.00482621326512358,0.00360889610423946,0.0380457302644569,3.05034186792599e-05,0.00233267133008141,0.0208580181076501,0.00113192039037051,0.0021061244745888,0.0348281570657621,2.50102042578393e-12,0.00985122343951636],"overlaps_ref":[true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true],"status":["Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up"],"region":["Intergenic (<10kb)","Intergenic (<10kb)","Intron","Promoter (-1500/+500)","Upstream Promoter (-5kb)","Promoter (-1500/+500)","Intron","Intron","Upstream Promoter (Other Gene)","Intron (Other Gene)","Promoter (-1500/+500)","Intergenic (>10kb)","Intergenic (<10kb)","Intron","Intron","Intron","Intron","Intron","Promoter (-1500/+500)","Upstream Promoter (-5kb)","Intron","Promoter (-1500/+500)","Intron","Intron","Intron","Promoter (Other Gene)","Intergenic (<10kb)","Intron (Other Gene)","Upstream Promoter (-5kb)","Upstream Promoter (-5kb)","Intron","Intergenic (<10kb)","Intergenic (<10kb)","Intron (Other Gene)","Intron (Other Gene)","Intron","Intron (Other Gene)","Intron (Other Gene)"],"feature":["Enhancer","No Feature","Enhancer","Enhancer","No Feature","No Feature","No Feature","No Feature","Enhancer","Enhancer","No Feature","No Feature","Enhancer","Enhancer","Enhancer","No Feature","No Feature","Enhancer","Enhancer","Enhancer","Enhancer","No Feature","No Feature","No Feature","No Feature","Enhancer","No Feature","Enhancer","Enhancer","Enhancer","No Feature","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer","Enhancer"]},"columns":[{"id":"gene_name","name":"Gene","type":"character","maxWidth":132,"style":{"borderRight":"1px solid rgba(0, 0, 0, 0.1)","fontSize":12}},{"id":"gene_id","name":"ID","type":"character","aggregate":"unique","show":false,"cell":["","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","",""]},{"id":"RNA_logFC","name":"logFC","type":"numeric","aggregate":"mean","format":{"cell":{"digits":2},"aggregated":{"digits":2}},"cell":["","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","",""],"maxWidth":72,"style":"function(rowInfo) {\n  var value = rowInfo.row['RNA_logFC']\n  if (value < 0) {\n    var color = '#0571B0'\n  } else if (value > 0) {\n    var color = '#CA0020'\n  } else {\n    var color = '#000000'\n  }\n  return { color: color, fontSize: 12 }\n}"},{"id":"RNA_FDR","name":"FDR","type":"numeric","aggregate":"min","format":{"cell":{"digits":4},"aggregated":{"digits":4}},"cell":["","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","",""],"maxWidth":72,"style":{"borderRight":"1px solid rgba(0, 0, 0, 0.1)","fontSize":12}},{"id":"range","name":"Range","type":"character","aggregate":"function(values){\n              var chrom = [];\n              var rng = [];\n              var start = [];\n              var end = [];\n              for (i = 0; i < values.length; i++) {\n                chrom[i] = values[i].split(':')[0];\n                rng[i] = values[i].split(':')[1];\n                start[i] = rng[i].split('-')[0];\n                end[i] = rng[i].split('-')[1];\n              }\n              \n              min_start = Math.min(...start);\n              max_end = Math.max(...end);\n              var ret_val = [...new Set(chrom)] + ':' + min_start.toString() + '-' + max_end.toString();\n              \n              return ret_val\n              \n            }","minWidth":160},{"id":"dist2Gene","name":"Distance to Gene","type":"numeric","aggregate":"function(values) {\n    var min_val = Math.round(100 * Math.min(...values)) / 100\n    var max_val = Math.round(100 * Math.max(...values)) / 100\n    if (min_val == max_val) {\n      var ret_val = min_val.toString()\n    } else {\n      var ret_val = '[' + min_val.toString() + ', ' + max_val.toString() + ']'\n    }\n    return ret_val\n  }","format":{"cell":{"digits":0,"separators":true},"aggregated":{"digits":0,"separators":true}},"maxWidth":96,"style":{"borderRight":"1px solid rgba(0, 0, 0, 0.1)","fontSize":12}},{"id":"ChIP_logCPM","name":"logCPM","type":"numeric","aggregate":"function(values) {\n    var min_val = Math.round(100 * Math.min(...values)) / 100\n    var max_val = Math.round(100 * Math.max(...values)) / 100\n    if (min_val == max_val) {\n      var ret_val = min_val.toString()\n    } else {\n      var ret_val = '[' + min_val.toString() + ', ' + max_val.toString() + ']'\n    }\n    return ret_val\n  }","format":{"cell":{"digits":2},"aggregated":{"digits":2}},"maxWidth":72,"style":[{"backgroundImage":"linear-gradient(90deg, #e6e6e6 29.2591677457748%, transparent 29.2591677457748%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 14.1281835531692%, transparent 14.1281835531692%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 28.735542972954%, transparent 28.735542972954%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 11.7323230131273%, transparent 11.7323230131273%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 18.6047223084067%, transparent 18.6047223084067%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 9.53648393777485%, transparent 9.53648393777485%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 12.2173114872754%, transparent 12.2173114872754%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 9.18630951934064%, transparent 9.18630951934064%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 58.6419315550834%, transparent 58.6419315550834%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 78.6599390879815%, transparent 78.6599390879815%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 20.3084451346915%, transparent 20.3084451346915%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 23.2711658702288%, transparent 23.2711658702288%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 12.8261066508558%, transparent 12.8261066508558%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 7.54250642855753%, transparent 7.54250642855753%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 10.1257335986308%, transparent 10.1257335986308%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 12.6076381614678%, transparent 12.6076381614678%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 14.6110290564864%, transparent 14.6110290564864%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 14.9343685907206%, transparent 14.9343685907206%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 42.3051094171046%, transparent 42.3051094171046%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 20.268006674022%, transparent 20.268006674022%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 38.4041080243836%, transparent 38.4041080243836%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 33.6423961981209%, transparent 33.6423961981209%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 20.242321791723%, transparent 20.242321791723%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 18.2493283516583%, transparent 18.2493283516583%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 7.02404973266737%, transparent 7.02404973266737%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 24.8333155458052%, transparent 24.8333155458052%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 7.31579529386237%, transparent 7.31579529386237%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 32.9170595261081%, transparent 32.9170595261081%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 22.8796446169345%, transparent 22.8796446169345%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 33.4985329313508%, transparent 33.4985329313508%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 13.5876843189997%, transparent 13.5876843189997%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 18.9526580703354%, transparent 18.9526580703354%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 15.4071474120354%, transparent 15.4071474120354%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 11.0490038552831%, transparent 11.0490038552831%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 13.242966843766%, transparent 13.242966843766%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 10.1378926273882%, transparent 10.1378926273882%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 23.6030675884692%, transparent 23.6030675884692%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12},{"backgroundImage":"linear-gradient(90deg, #e6e6e6 44.547600487402%, transparent 44.547600487402%)","backgroundSize":"100% 75%","backgroundRepeat":"no-repeat","backgroundPosition":"center","color":null,"fontSize":12}]},{"id":"ChIP_logFC","name":"logFC","type":"numeric","aggregate":"mean","format":{"cell":{"digits":2},"aggregated":{"digits":2}},"cell":["1.40","1.60","1.15","1.36","0.89","1.53","1.82","1.56","0.68","0.60","1.28","1.94","1.26","1.70","2.04","1.30","1.14","1.46","0.80","1.01","1.26","1.08","0.90","1.53","1.14","1.10","1.40","0.94","1.08","0.75","1.53","1.12","1.00","1.33","1.24","1.34","2.15","0.76"],"maxWidth":60,"style":"function(rowInfo) {\n  var value = rowInfo.row['ChIP_logFC']\n  if (value < 0) {\n    var color = '#0571B0'\n  } else if (value > 0) {\n    var color = '#CA0020'\n  } else {\n    var color = '#000000'\n  }\n  return { color: color, fontSize: 12 }\n}"},{"id":"ChIP_FDR","name":"FDR","type":"numeric","aggregate":"max","format":{"cell":{"prefix":"< ","digits":4},"aggregated":{"prefix":"< ","digits":4}},"cell":["2.11e-06","1.91e-05","2.15e-04","4.95e-04","0.048","2.15e-04","6.98e-07","1.12e-04","0.006","1.54e-04","2.05e-04","2.25e-10","0.004","2.58e-05","4.19e-08","0.001","0.006","8.06e-05","0.013","0.013","8.05e-07","2.81e-04","0.026","7.08e-06","0.020","0.002","0.044","0.005","0.004","0.038","3.05e-05","0.002","0.021","0.001","0.002","0.035","2.50e-12","0.010"],"maxWidth":70},{"id":"overlaps_ref","name":"Macs Peak","type":"logical","aggregate":"frequency","show":false,"cell":["✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes","✔ Yes"],"maxWidth":60,"style":[{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12},{"color":"#008000","fontSize":12}]},{"id":"status","name":"Status","type":"factor","aggregate":"frequency","cell":["⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up"],"maxWidth":72,"style":"function(rowInfo) {\n  var value = rowInfo.row['ChIP_logFC']\n  if (value < 0) {\n    var color = '#0571B0'\n  } else if (value > 0) {\n    var color = '#CA0020'\n  } else {\n    var color = '#000000'\n  }\n  return { color: color, fontSize: 12 }\n}"},{"id":"region","name":"Binding Region","type":"factor","aggregate":"unique"},{"id":"feature","name":"Feature","type":"factor","aggregate":"unique"}],"columnGroups":[{"name":"RNA-Seq","columns":["RNA_logFC","RNA_FDR"]},{"name":"H3K27ac ChIP-Seq","columns":["ChIP_logCPM","ChIP_logFC","ChIP_FDR","status"]}],"groupBy":["gene_name"],"filterable":true,"showPageSizeOptions":true,"borderless":true,"theme":{"style":{"fontSize":12}},"dataKey":"9fbbe0ebd1076cddb3ac961905ca64f9"},"children":[]},"class":"reactR_markup"},"evals":["tag.attribs.columns.2.style","tag.attribs.columns.4.aggregate","tag.attribs.columns.5.aggregate","tag.attribs.columns.6.aggregate","tag.attribs.columns.7.style","tag.attribs.columns.10.style"],"jsHooks":[]}</script>
</div>
</div>
</div>
<div id="volcano-plots-by-h3k27ac-binding" class="section level3">
<h3>Volcano Plots By H3K27ac Binding</h3>
<pre class="r"><code>diff_tbl &lt;- diff_windows %&gt;% 
  lapply(function(x) tibble(gene_id = x)) %&gt;% 
  bind_rows(.id = &quot;status&quot;)
rnaseq %&gt;% 
  left_join(diff_tbl) %&gt;% 
  mutate(
    status = str_replace_na(status, &quot;Undetected&quot;),
    status = glue(&quot;{target} {status}&quot;) %&gt;%
      factor(levels = paste(target, c(&quot;Up&quot;, &quot;Down&quot;, &quot;Ambiguous&quot;, &quot;Unchanged&quot;, &quot;Undetected&quot;)))
  ) %&gt;% 
  left_join(rna_status) %&gt;% 
  droplevels() %&gt;% 
  ggplot(
    aes(!!sym(rna_lfc_col), -log10(!!sym(rna_p_col)), colour = de_status)
  ) +
  geom_point() +
  geom_text_repel(
    aes(label = gene_name),
    data = . %&gt;% 
      dplyr::arrange(!!sym(rna_p_col)) %&gt;% 
      dplyr::filter(!!sym(rna_lfc_col) &gt; 0, DE) %&gt;% 
      split(.$status) %&gt;% 
      lapply(dplyr::slice, 1:2) %&gt;% 
      bind_rows(),
    show.legend = FALSE
  ) +
  geom_text_repel(
    aes(label = gene_name),
    data = . %&gt;% 
      dplyr::arrange(!!sym(rna_p_col)) %&gt;% 
      dplyr::filter(!!sym(rna_lfc_col) &lt; 0, DE) %&gt;% 
      split(.$status) %&gt;% 
      lapply(dplyr::slice, 1:2) %&gt;% 
      bind_rows(),
    show.legend = FALSE
  ) +
  geom_text(
    aes(label = lab),
    data = . %&gt;% 
      group_by(status) %&gt;% 
      summarise(n = dplyr::n()) %&gt;% 
      mutate(lab = glue(&quot;n = {comma(n)}&quot;)),
    x = min(rnaseq[[rna_lfc_col]]) + 0.9 * diff(range(rnaseq[[rna_lfc_col]])),
    y = 1,
    colour = &quot;black&quot;,
    inherit.aes = FALSE,
    show.legend = FALSE
  ) +
  facet_wrap(~status, ncol = 2) +
  scale_colour_manual(
    values = direction_colours
  ) +
  labs(colour = &quot;DE Status&quot;) </code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-volcano-by-target-1.png" alt="*Volcano plot showing gene expression patterns separated by H3K27ac status. The two most highly ranked genes for up and down regulation are labelled in each panel. Given that some genes may be mapped to multiple H3K27ac binding events which include different binding patterns, genes may appear in multiple panels.*" width="960" />
<p class="caption">
<em>Volcano plot showing gene expression patterns separated by H3K27ac
status. The two most highly ranked genes for up and down regulation are
labelled in each panel. Given that some genes may be mapped to multiple
H3K27ac binding events which include different binding patterns, genes
may appear in multiple panels.</em>
</p>
</div>
</div>
<div id="p-values-by-h3k27ac-status" class="section level3">
<h3>P-Values By H3K27ac Status</h3>
<pre class="r"><code>p &lt;- rnaseq %&gt;% 
  mutate(
    mapped = case_when(
      gene_id %in% unlist(diff_windows[names(diff_windows) %in% c(&quot;Up&quot;, &quot;Down&quot;)]) ~ paste(target, &quot;Differentially Bound&quot;),
      gene_id %in% unlist(diff_windows[names(diff_windows) %in% c(&quot;Unchanged&quot;)]) ~ paste(target, &quot;Unchanged&quot;),
      TRUE ~ paste(&quot;No&quot;, target, &quot;Detected&quot;)
    )
  ) %&gt;% 
  ggplot(
    aes(!!sym(rna_p_col), stat(density))
  ) +
  geom_histogram(
    colour = &quot;black&quot;, fill = &quot;grey&quot;, bins = 100
  ) +
  facet_wrap(~mapped, nrow = 1) +
  scale_y_continuous(expand = expansion(c(0, 0.05))) +
  labs(y = &quot;Density&quot;, x = &quot;P (Differential Expression)&quot;)
y &lt;- 0.95 * max(layer_scales(p)$y$range$range)
p  +
  geom_text(
    aes(0.8, y, label = lab),
    data = . %&gt;% 
      group_by(mapped) %&gt;% 
      tally() %&gt;% 
      mutate(lab = glue(&quot;N = {comma(n, 1)}&quot;))
  ) </code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-p-target-1.png" alt="*P-Values for differential expression partitioned by H3K27ac ChIP peak status. Genes were considered as being mapped to a differentially bound peak if one or more peaks was considered as differentially bound. No IHW procedure was undertaken using this data.*" width="960" />
<p class="caption">
<em>P-Values for differential expression partitioned by H3K27ac ChIP
peak status. Genes were considered as being mapped to a differentially
bound peak if one or more peaks was considered as differentially bound.
No IHW procedure was undertaken using this data.</em>
</p>
</div>
</div>
<div id="chip-logfc-vs-rna-logfc" class="section level3">
<h3>ChIP logFC Vs RNA logFC</h3>
<pre class="r"><code>de_genes_db_regions %&gt;% 
  dplyr::filter(ChIP_FDR &lt; fdr_alpha) %&gt;% 
  group_by(gene_name) %&gt;%
  dplyr::filter(ChIP_FDR == min(ChIP_FDR)) %&gt;%
  mutate(DE = ifelse(RNA_logFC &gt; 0, &quot;Up&quot;, &quot;Down&quot;)) %&gt;% 
  ungroup() %&gt;% 
  ggplot(
    aes(ChIP_logFC, RNA_logFC, colour = DE)
  ) +
  geom_point() +
  geom_label_repel(
    aes(label = lab),
    data = . %&gt;% 
      mutate(grp = paste(DE, status)) %&gt;% 
      # arrange(desc(abs(ChIP_logFC))) %&gt;% 
      arrange(ChIP_FDR, RNA_FDR) %&gt;% 
      split(.$grp) %&gt;% 
      lapply(dplyr::slice, 1) %&gt;% 
      bind_rows() %&gt;% 
      mutate(
        range = GRanges(range) %&gt;% 
          resize(width = 1, fix = &quot;center&quot;) %&gt;% 
          as.character(),
        lab = paste(gene_name, &quot;\n&quot;, range)
      ),
    show.legend = FALSE,
    alpha = 0.7
  ) +
  geom_text(
    aes(x, y, label = lab),
    data = . %&gt;% 
      group_by(DE, status) %&gt;% 
      summarise(
        n = dplyr::n(),
        lab = glue(&quot;n = {comma(n)}&quot;),
        x = max(abs(ChIP_logFC)) * sign(min(ChIP_logFC)) * 1.05,
        y = max(abs(RNA_logFC)) * sign(min(RNA_logFC)) * 1.05,
        .groups = &quot;drop&quot;
      ) %&gt;% 
      group_by(DE) %&gt;% 
      mutate(y = max(abs(y)) * sign(y)) %&gt;% 
      ungroup() %&gt;% 
      group_by(status) %&gt;% 
      mutate(x = max(abs(x)) * sign(x)),
    colour = &quot;black&quot;
  ) +
  geom_hline(yintercept = 0, colour = &quot;grey20&quot;) +
  geom_vline(xintercept = 0, colour = &quot;grey20&quot;) +
  scale_colour_manual(values = direction_colours[c(&quot;Up&quot;, &quot;Down&quot;)]) +
  labs(
    x = glue(&quot;logFC ({target} ChIP-Seq)&quot;),
    y = &quot;logFC (RNA-Seq)&quot;,
    colour = &quot;Differential\nExpression&quot;
  )</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-chip-rna-logfc-1.png" alt="*logFC values for differentially bound ChIP-Seq peaks mapped to DE genes. The most highly ranked peak for each quandrant is labelled at showing both the gene and centre of the binding region. Whilst binding sites may be mapped to multiple genes, only the most highly ranked binding site is shown mapped to the mosthighly ranked DE genes. Points are coloured by differential expression status.*" width="960" />
<p class="caption">
<em>logFC values for differentially bound ChIP-Seq peaks mapped to DE
genes. The most highly ranked peak for each quandrant is labelled at
showing both the gene and centre of the binding region. Whilst binding
sites may be mapped to multiple genes, only the most highly ranked
binding site is shown mapped to the mosthighly ranked DE genes. Points
are coloured by differential expression status.</em>
</p>
</div>
</div>
</div>
<div id="binding-pattern-gsea" class="section level2">
<h2>Binding Pattern GSEA</h2>
<div id="gsea-by-direction-of-regulation" class="section level3 tabset">
<h3 class="tabset">GSEA By Direction of Regulation</h3>
<p>GSEA analysis was first performed by taking the genes mapped to
various sets of peaks, and checking for enrichment amongst the RNA-Seq
results, ranking genes from up- to down-regulated, by significance of
the logFC estimates.</p>
<div id="all-windows" class="section level4">
<h4>All Windows</h4>
<pre class="r"><code>gsea_dir &lt;- fgsea(dir_gs, rna_dir_ranks)
gsea_dir_sig &lt;- gsea_dir %&gt;% 
  arrange(pval) %&gt;% 
  mutate(padj = p.adjust(pval, adj_method)) %&gt;% 
  dplyr::filter(padj &lt; enrich_alpha, size &gt;= min_sig) %&gt;% 
  as_tibble() 
p &lt;- gsea_dir_sig %&gt;% 
  dplyr::slice(1:9) %&gt;% 
  pull(&quot;pathway&quot;) %&gt;% 
  lapply(
    function(x) {
      plotEnrichment(dir_gs[[x]], rna_dir_ranks) +
        ggtitle(x) +
        theme(
          plot.title = element_text(hjust = 0.5, size = 10)
        )
    }
  )</code></pre>
<pre class="r"><code>cp &lt;-  htmltools::em(
  glue(
    &quot;Merged windows were mapped to genes and their position amongst the &quot;,
    &quot;RNA-Seq results was assessed. Windows were classified based on  &quot;,
    &quot;the direction of {target} binding with &quot;,
    &quot;{treat_levels[[2]]} treatment. {nrow(gsea_dir_sig)} sets of windows were &quot;,
    &quot;associated with changes in gene expression, using the sign of &quot;, 
    &quot;fold-change and ranking statistic to initially rank the &quot;, 
    &quot;{comma(nrow(rnaseq), 1)} genes considered as detected.&quot;
  )
)
tbl &lt;- gsea_dir_sig %&gt;%
  mutate(
    `Edge Size` = vapply(leadingEdge, length, integer(1)),
    leadingEdge = lapply(leadingEdge, function(x) id2gene[x]) %&gt;% 
      vapply(paste, character(1), collapse = &quot;; &quot;),
    Direction = ifelse(NES &gt; 0, &quot;\u21E7 Up-regulated&quot;, &quot;\u21E9 Down-regulated&quot;),
    &quot;{target}&quot; := str_extract(pathway, &quot;(Up|Down|Unchanged)$&quot;),
    pathway = str_remove_all(pathway, &quot; / [^/]+$&quot;)
  ) %&gt;% 
  dplyr::select(
    !!sym(target), Windows = size, Direction, 
    p = pval, padj, `Edge Size`, `Leading Edge` = leadingEdge
  ) %&gt;% 
  reactable(
    filterable = TRUE,
    columns = list2(
      &quot;{target}&quot; := colDef(
        maxWidth = 90,
        cell = function(value) {
          html_symbol &lt;- &quot;&quot;
          if (str_detect(value, &quot;Up&quot;)) html_symbol &lt;- &quot;\u21E7&quot;
          if (str_detect(value, &quot;Down&quot;)) html_symbol &lt;- &quot;\u21E9&quot;
          paste(html_symbol, value)
        },
        style = function(value) {
          colour &lt;- case_when(
            str_detect(value, &quot;Up&quot;) ~ colours$direction[[&quot;up&quot;]],
            str_detect(value, &quot;Down&quot;) ~ colours$direction[[&quot;down&quot;]],
            TRUE ~ colours$direction[[&quot;unchanged&quot;]]
          )
          list(color = colour)
        }
      ),
      Windows = colDef(maxWidth = 80, format = colFormat(separators = TRUE)),
      Direction = colDef(
        name = &quot;Gene Direction&quot;,
        maxWidth = 150,
        style = function(value) {
          colour &lt;- ifelse(
            str_detect(value, &quot;Up&quot;), 
            colours$direction[[&quot;up&quot;]], 
            colours$direction[[&quot;down&quot;]]
          )
          list(color = colour)
        },
      ),
      p = colDef(
        cell = function(value) sprintf(&quot;%.2e&quot;, value), maxWidth = 80
      ),
      padj = colDef(
        name = glue(&quot;p&lt;sub&gt;{adj_method}&lt;/sub&gt;&quot;), html = TRUE,
        cell = function(value) {
          ifelse(
            value &lt; 0.001, sprintf(&quot;%.2e&quot;, value), sprintf(&quot;%.3f&quot;, value)
          )
        }
      ),
      &quot;Edge Size&quot; = colDef(maxWidth = 80),
     &quot;Leading Edge&quot; = colDef(
       minWidth = 150,
       cell = function(value) with_tooltip(value, width = 50)
     ) 
    )
  )
div(class = &quot;table&quot;,
  div(class = &quot;table-header&quot;,
      div(class = &quot;caption&quot;, cp),
      tbl
  )
)</code></pre>
<div class="table">
<div class="table-header">
<div class="caption">
<em>Merged windows were mapped to genes and their position amongst the RNA-Seq results was assessed. Windows were classified based on  the direction of H3K27ac binding with E2DHT treatment. 1 sets of windows were associated with changes in gene expression, using the sign of fold-change and ranking statistic to initially rank the 13,043 genes considered as detected.</em>
</div>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-8cf57203f00c68a86aea" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-8cf57203f00c68a86aea">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"H3K27ac":["Up"],"Windows":[248],"Direction":["⇧ Up-regulated"],"p":[1.06624823766999e-14],"padj":[2.13249647533998e-14],"Edge Size":[56],"Leading Edge":["AQP3; FKBP5; SEC14L2; RANBP3L; HSD11B2; ZNF652; ZNF689; EHF; ARHGEF37; NRP1; AL033523.1; TTC12; KDM4B; PTPRJ; SYBU; NEDD4L; CSRNP1; WWC1; SASH1; CNTNAP2; AKAP13; CD9; SPRED1; VSTM2L; IQGAP2; PRKAB2; MCCC2; AC092053.4; FGFR2; GSTM3; PEA15; TPD52; SGK3; SORD; MYCBP2; TTC21A; TMEM17; CCDC40; CNNM1; MTMR9; CRAT; NAMPT; RASSF3; ACACA; PEX10; PXDC1; PRKAA2; VIPR1; CHPT1; VIPR1-AS1; DUSP16; ZFX; NOL6; CCDC96; MAFB; TOR1AIP2"]},"columns":[{"id":"H3K27ac","name":"H3K27ac","type":"character","cell":["⇧ Up"],"maxWidth":90,"style":[{"color":"#CA0020"}]},{"id":"Windows","name":"Windows","type":"numeric","format":{"cell":{"separators":true},"aggregated":{"separators":true}},"maxWidth":80},{"id":"Direction","name":"Gene Direction","type":"character","maxWidth":150,"style":[{"color":"#CA0020"}]},{"id":"p","name":"p","type":"numeric","cell":["1.07e-14"],"maxWidth":80},{"id":"padj","name":"p<sub>fdr<\/sub>","type":"numeric","cell":["2.13e-14"],"html":true},{"id":"Edge Size","name":"Edge Size","type":"numeric","maxWidth":80},{"id":"Leading Edge","name":"Leading Edge","type":"character","cell":[{"name":"span","attribs":{"title":"AQP3; FKBP5; SEC14L2; RANBP3L; HSD11B2; ZNF652; ZNF689; EHF; ARHGEF37; NRP1; AL033523.1; TTC12; KDM4B; PTPRJ; SYBU; NEDD4L; CSRNP1; WWC1; SASH1; CNTNAP2; AKAP13; CD9; SPRED1; VSTM2L; IQGAP2; PRKAB2; MCCC2; AC092053.4; FGFR2; GSTM3; PEA15; TPD52; SGK3; SORD; MYCBP2; TTC21A; TMEM17; CCDC40; CNNM1; MTMR9; CRAT; NAMPT; RASSF3; ACACA; PEX10; PXDC1; PRKAA2; VIPR1; CHPT1; VIPR1-AS1; DUSP16; ZFX; NOL6; CCDC96; MAFB; TOR1AIP2"},"children":["AQP3; FKBP5; SEC14L2; RANBP3L; HSD11B2; ZNF652;..."]}],"minWidth":150}],"filterable":true,"dataKey":"db0383acc3ca04ef4c587fb27b888ff2"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<pre class="r"><code>plot_grid(plotlist = p, nrow = ceiling(length(p) / 3))</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/barcode-gsea-dir-sig-1.png" alt="*Barcode plots for the top 1 sets of windows associated with __directional__ changes in gene expression.*" width="960" />
<p class="caption">
<em>Barcode plots for the top 1 sets of windows associated with
<strong>directional</strong> changes in gene expression.</em>
</p>
</div>
</div>
<div id="windows-by-region" class="section level4">
<h4>Windows By Region</h4>
<pre class="r"><code>gsea_reg_dir &lt;- fgsea(reg_dir_gs, rna_dir_ranks)
gsea_reg_dir_sig &lt;- gsea_reg_dir %&gt;% 
  arrange(pval) %&gt;% 
  mutate(padj = p.adjust(pval, adj_method)) %&gt;% 
  dplyr::filter(padj &lt; enrich_alpha, size &gt;= min_sig) %&gt;% 
  as_tibble() 
p &lt;- gsea_reg_dir_sig %&gt;% 
  dplyr::slice(1:9) %&gt;% 
  pull(&quot;pathway&quot;) %&gt;% 
  lapply(
    function(x) {
      plotEnrichment(reg_dir_gs[[x]], rna_dir_ranks) +
        ggtitle(x) +
        theme(
          plot.title = element_text(hjust = 0.5, size = 10)
        )
    }
  )</code></pre>
<pre class="r"><code>cp &lt;-  htmltools::em(
  glue(
    &quot;Merged windows were mapped to genes and their position amongst the &quot;,
    &quot;RNA-Seq results was assessed. Windows were classified based on which &quot;,
    &quot;region was the best overlap, and the direction of {target} binding with &quot;,
    &quot;{treat_levels[[2]]} treatment. {nrow(gsea_reg_dir_sig)} sets of windows were &quot;,
    &quot;associated with changes in gene expression, using the sign of &quot;, 
    &quot;fold-change and ranking statistic to initially rank the &quot;, 
    &quot;{comma(nrow(rnaseq), 1)} genes considered as detected.&quot;
  )
)
tbl &lt;- gsea_reg_dir_sig %&gt;%
  mutate(
    `Edge Size` = vapply(leadingEdge, length, integer(1)),
    leadingEdge = lapply(leadingEdge, function(x) id2gene[x]) %&gt;% 
      vapply(paste, character(1), collapse = &quot;; &quot;),
    Direction = ifelse(NES &gt; 0, &quot;\u21E7 Up-regulated&quot;, &quot;\u21E9 Down-regulated&quot;),
    &quot;{target}&quot; := str_extract(pathway, &quot;(Up|Down|Unchanged)$&quot;),
    pathway = str_remove_all(pathway, &quot; / [^/]+$&quot;)
  ) %&gt;% 
  dplyr::select(
    pathway, !!sym(target), Windows = size, Direction, 
    p = pval, padj, `Edge Size`, `Leading Edge` = leadingEdge
  ) %&gt;% 
  reactable(
    filterable = TRUE,
    columns = list2(
      pathway = colDef(
        minWidth = 100, name = &quot;Region&quot;
      ),
      &quot;{target}&quot; := colDef(
        maxWidth = 90,
        cell = function(value) {
          html_symbol &lt;- &quot;&quot;
          if (str_detect(value, &quot;Up&quot;)) html_symbol &lt;- &quot;\u21E7&quot;
          if (str_detect(value, &quot;Down&quot;)) html_symbol &lt;- &quot;\u21E9&quot;
          paste(html_symbol, value)
        },
        style = function(value) {
          colour &lt;- case_when(
            str_detect(value, &quot;Up&quot;) ~ colours$direction[[&quot;up&quot;]],
            str_detect(value, &quot;Down&quot;) ~ colours$direction[[&quot;down&quot;]],
            TRUE ~ colours$direction[[&quot;unchanged&quot;]]
          )
          list(color = colour)
        }
      ),
      Windows = colDef(maxWidth = 80, format = colFormat(separators = TRUE)),
      Direction = colDef(
        name = &quot;Gene Direction&quot;,
        maxWidth = 150,
        style = function(value) {
          colour &lt;- ifelse(
            str_detect(value, &quot;Up&quot;), 
            colours$direction[[&quot;up&quot;]], 
            colours$direction[[&quot;down&quot;]]
          )
          list(color = colour)
        },
      ),
      p = colDef(
        cell = function(value) sprintf(&quot;%.2e&quot;, value), maxWidth = 80
      ),
      padj = colDef(
        name = glue(&quot;p&lt;sub&gt;{adj_method}&lt;/sub&gt;&quot;), html = TRUE,
        cell = function(value) {
          ifelse(
            value &lt; 0.001, sprintf(&quot;%.2e&quot;, value), sprintf(&quot;%.3f&quot;, value)
          )
        }
      ),
      &quot;Edge Size&quot; = colDef(maxWidth = 80),
     &quot;Leading Edge&quot; = colDef(
       minWidth = 150,
       cell = function(value) with_tooltip(value, width = 50)
     ) 
    )
  )
div(class = &quot;table&quot;,
  div(class = &quot;table-header&quot;,
      div(class = &quot;caption&quot;, cp),
      tbl
  )
)</code></pre>
<div class="table">
<div class="table-header">
<div class="caption">
<em>Merged windows were mapped to genes and their position amongst the RNA-Seq results was assessed. Windows were classified based on which region was the best overlap, and the direction of H3K27ac binding with E2DHT treatment. 6 sets of windows were associated with changes in gene expression, using the sign of fold-change and ranking statistic to initially rank the 13,043 genes considered as detected.</em>
</div>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-cf16d280d56c4ff51c44" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-cf16d280d56c4ff51c44">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"pathway":["Intron","Promoter (-1500/+500)","Upstream Promoter (-5kb)","Intergenic (<10kb)","Intergenic (>10kb)","Upstream Promoter (-5kb)"],"H3K27ac":["Up","Up","Up","Up","Up","Unchanged"],"Windows":[142,30,32,31,24,3387],"Direction":["⇧ Up-regulated","⇧ Up-regulated","⇧ Up-regulated","⇧ Up-regulated","⇧ Up-regulated","⇧ Up-regulated"],"p":[1.63212677853851e-07,0.000109481509480251,0.000190338178375652,0.00166311443388312,0.00664614088048166,0.0160711027559931],"padj":[1.79533945639236e-06,0.000602148302141383,0.000697906654044056,0.00457356469317857,0.0146215099370597,0.0294636883859874],"Edge Size":[41,12,7,13,7,319],"Leading Edge":["FKBP5; RANBP3L; ZNF652; ARHGEF37; NRP1; AL033523.1; KDM4B; NEDD4L; SASH1; AKAP13; SPRED1; VSTM2L; IQGAP2; PRKAB2; FGFR2; GSTM3; PEA15; TPD52; SGK3; SORD; MYCBP2; CCDC40; CNNM1; RASSF3; ACACA; PXDC1; PRKAA2; CCDC96; TOR1AIP2; MOB3C; AL451165.2; MPC2; FMO5; KCNMA1; OLMALINC; HOMER2; IGSF8; CAPZB; SIX5; CUEDC1; MED21","FKBP5; SEC14L2; ZNF689; TTC12; SYBU; CSRNP1; AC092053.4; TTC21A; MTMR9; CRAT; PEX10; CHPT1","FKBP5; HSD11B2; PTPRJ; CNTNAP2; MCCC2; MYCBP2; RNF215","AQP3; EHF; WWC1; CD9; MCCC2; NAMPT; VIPR1; VIPR1-AS1; DUSP16; NOL6; TUT7; AC104825.1; RASGEF1A","EHF; TMEM17; ZFX; MAFB; SPRYD7; ATG101; NR4A1","AQP3; UGDH; FKBP5; SEC14L2; HSD11B2; AC021148.3; PISD; ZNF652; ZNF689; ANKRD22; ARHGEF26; EHF; KRT4; PARM1; ARHGEF37; THBS1; NRP1; GREB1; TTC12; KDM4B; PTPRJ; ADD3; SYBU; NEDD4L; CSRNP1; GPRC5A; WWC1; CNTNAP2; TBC1D16; OSGIN1; SLC26A2; SERHL; PER1; AKAP13; SLC22A23; KRT15; CD9; TRGC1; VSTM2L; IQGAP2; PRKAB2; PMEPA1; NDRG1; TMEM164; MCCC2; GRHL2; HEY2; LNX2; FAM214A; AC092053.4; CYP2B7P; ECI2; AL135905.2; ASB13; MTSS1; AL606500.1; KLF9; TPD52; EMP1; TBC1D8; EPN2; BCL2L1; STARD13-AS; STARD13; MYCBP2; IFITM10; AL445309.1; CRYBG2; LRP2BP; YPEL3; FSBP; TTC21A; ZBTB7C; PAQR5; TRGC2; MTMR9; CRAT; NAMPT; GASK1B; ZNF703; CHN1; CBX4; ARHGEF26-AS1; TESK2; TOE1; RASSF3; NYNRIN; BCL6; FAH; HERC3; KCNB1; GDPD3; APCDD1; SLC2A1; ADIRF-AS1; PXYLP1; FZD5; NSRP1; SLC22A5; RN7SL674P; HEG1; SPRY4; MYO1F; RNF227; SEPHS2; CASD1; GET4; ATP7B; MRFAP1; LBP; RFLNA; PARTICL; AL158211.5; SORD2P; PRR15L; PRAG1; CORO1A; DBI; DOK7; PIM1; GRK5; RPH3AL; BAG3; TSC22D3; FAM198B-AS1; EGLN3; TMEM63C; SPRED2; AC105339.2; MRPL38; ARRB1; KRT81; ID3; VCPKMT; PSMD6-AS2; HGD; NEDD4; ZFX; DRC3; LOXL3; PSCA; NOL6; ID1; SKAP1; GADD45B; ENPP1; AIFM2; IFNLR1; CNOT2-DT; AL135905.1; BTN2A3P; RNU4-1; KL; CCDC96; ENTPD1-AS1; MFSD2A; VAMP5; RPS10P3; CARMIL1; MAGIX; AC020904.2; GRAMD1B; HAP1; LMNTD2-AS1; RBM26-AS1; RNY3; RABGEF1; TMEM62; LRRC26; TLK2P1; SMIM14; TGFB1I1; IZUMO4; HERC6; IGFBP3; AC234917.3; AMH; UBXN10; RAB11B-AS1; ZFP36L2; LIF; C15orf48; SERTAD1; CFAP45; HES1; MGAT2; AC007405.4; PRDM4; RASD2; ATP1A1; TMEM160; BPIFA4P; PXMP4; EFNA4; HMGCS2; HEBP1; SNHG22; AHR; BCCIP; BCAR3; LINC01089; MIR3936HG; AC090772.5; PAM16; TMEM254-AS1; IQCD; ADPRM; HOOK1; TSKU; EFNA3; AC141930.1; AC092910.3; RNF215; MPC2; SPRYD7; RILP; FMO5; CRELD1; NNAT; DNMT3B; ABTB1; DNMBP; AL139385.1; ATG101; FEM1A; SLC24A3; CCDC124; EML5; HDAC10; AC084082.1; RASGEF1A; PRELID2; SDHAF2; SLC27A5; RFLNB; MAPK15; WDR60; ZNF688; AC104109.2; CYP4Z2P; C3orf18; PLEKHG5; EEPD1; PSG4; KCNMA1; VPS37D; DHCR24; LIN7B; DIPK1B; OLMALINC; GPKOW; AC097523.1; SNORD3C; ZNF764; SNHG19; AC087289.5; LARGE1; ETHE1; KLHL8; ZC3H10; ZSCAN31; CAP2; C1QTNF6; RILPL2; HINT2; PPP1R37; TSC22D1; TAGLN; IGSF8; ZFPM1; CUEDC1; CHST3; LTBP4; TMEM107; OSBPL6; MBD3; UQCRHL; DGLUCY; UCKL1-AS1; RPL13P5; ARID5B; AL031666.1; LINC02482; RDM1; CASP9; ACYP2; ENTPD8; AC018521.5; AC108134.2; ABCC12; MAFF; SIX1; CALML5; SESTD1; NR4A1; DHRS2; EFNA5; ANKRD18B; TMEM186; ZC3H12A; NDFIP2; MRC2; TPO; ANAPC11; CST6; KIF13B; ZNF786; TOB1-AS1; AC005914.1; BCL2L12; SH3TC1; AC139149.1; AL731571.1; TMC5; PYDC1; TTPA; DCST1-AS1; TPRN; AL445363.3"]},"columns":[{"id":"pathway","name":"Region","type":"character","minWidth":100},{"id":"H3K27ac","name":"H3K27ac","type":"character","cell":["⇧ Up","⇧ Up","⇧ Up","⇧ Up","⇧ Up"," Unchanged"],"maxWidth":90,"style":[{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#7F7F7F"}]},{"id":"Windows","name":"Windows","type":"numeric","format":{"cell":{"separators":true},"aggregated":{"separators":true}},"maxWidth":80},{"id":"Direction","name":"Gene Direction","type":"character","maxWidth":150,"style":[{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"}]},{"id":"p","name":"p","type":"numeric","cell":["1.63e-07","1.09e-04","1.90e-04","1.66e-03","6.65e-03","1.61e-02"],"maxWidth":80},{"id":"padj","name":"p<sub>fdr<\/sub>","type":"numeric","cell":["1.80e-06","6.02e-04","6.98e-04","0.005","0.015","0.029"],"html":true},{"id":"Edge Size","name":"Edge Size","type":"numeric","maxWidth":80},{"id":"Leading Edge","name":"Leading Edge","type":"character","cell":[{"name":"span","attribs":{"title":"FKBP5; RANBP3L; ZNF652; ARHGEF37; NRP1; AL033523.1; KDM4B; NEDD4L; SASH1; AKAP13; SPRED1; VSTM2L; IQGAP2; PRKAB2; FGFR2; GSTM3; PEA15; TPD52; SGK3; SORD; MYCBP2; CCDC40; CNNM1; RASSF3; ACACA; PXDC1; PRKAA2; CCDC96; TOR1AIP2; MOB3C; AL451165.2; MPC2; FMO5; KCNMA1; OLMALINC; HOMER2; IGSF8; CAPZB; SIX5; CUEDC1; MED21"},"children":["FKBP5; RANBP3L; ZNF652; ARHGEF37; NRP1; AL03352..."]},{"name":"span","attribs":{"title":"FKBP5; SEC14L2; ZNF689; TTC12; SYBU; CSRNP1; AC092053.4; TTC21A; MTMR9; CRAT; PEX10; CHPT1"},"children":["FKBP5; SEC14L2; ZNF689; TTC12; SYBU; CSRNP1; AC..."]},{"name":"span","attribs":{"title":"FKBP5; HSD11B2; PTPRJ; CNTNAP2; MCCC2; MYCBP2; RNF215"},"children":["FKBP5; HSD11B2; PTPRJ; CNTNAP2; MCCC2; MYCBP2; ..."]},{"name":"span","attribs":{"title":"AQP3; EHF; WWC1; CD9; MCCC2; NAMPT; VIPR1; VIPR1-AS1; DUSP16; NOL6; TUT7; AC104825.1; RASGEF1A"},"children":["AQP3; EHF; WWC1; CD9; MCCC2; NAMPT; VIPR1; VIPR..."]},{"name":"span","attribs":{"title":"EHF; TMEM17; ZFX; MAFB; SPRYD7; ATG101; NR4A1"},"children":["EHF; TMEM17; ZFX; MAFB; SPRYD7; ATG101; NR4A1"]},{"name":"span","attribs":{"title":"AQP3; UGDH; FKBP5; SEC14L2; HSD11B2; AC021148.3; PISD; ZNF652; ZNF689; ANKRD22; ARHGEF26; EHF; KRT4; PARM1; ARHGEF37; THBS1; NRP1; GREB1; TTC12; KDM4B; PTPRJ; ADD3; SYBU; NEDD4L; CSRNP1; GPRC5A; WWC1; CNTNAP2; TBC1D16; OSGIN1; SLC26A2; SERHL; PER1; AKAP13; SLC22A23; KRT15; CD9; TRGC1; VSTM2L; IQGAP2; PRKAB2; PMEPA1; NDRG1; TMEM164; MCCC2; GRHL2; HEY2; LNX2; FAM214A; AC092053.4; CYP2B7P; ECI2; AL135905.2; ASB13; MTSS1; AL606500.1; KLF9; TPD52; EMP1; TBC1D8; EPN2; BCL2L1; STARD13-AS; STARD13; MYCBP2; IFITM10; AL445309.1; CRYBG2; LRP2BP; YPEL3; FSBP; TTC21A; ZBTB7C; PAQR5; TRGC2; MTMR9; CRAT; NAMPT; GASK1B; ZNF703; CHN1; CBX4; ARHGEF26-AS1; TESK2; TOE1; RASSF3; NYNRIN; BCL6; FAH; HERC3; KCNB1; GDPD3; APCDD1; SLC2A1; ADIRF-AS1; PXYLP1; FZD5; NSRP1; SLC22A5; RN7SL674P; HEG1; SPRY4; MYO1F; RNF227; SEPHS2; CASD1; GET4; ATP7B; MRFAP1; LBP; RFLNA; PARTICL; AL158211.5; SORD2P; PRR15L; PRAG1; CORO1A; DBI; DOK7; PIM1; GRK5; RPH3AL; BAG3; TSC22D3; FAM198B-AS1; EGLN3; TMEM63C; SPRED2; AC105339.2; MRPL38; ARRB1; KRT81; ID3; VCPKMT; PSMD6-AS2; HGD; NEDD4; ZFX; DRC3; LOXL3; PSCA; NOL6; ID1; SKAP1; GADD45B; ENPP1; AIFM2; IFNLR1; CNOT2-DT; AL135905.1; BTN2A3P; RNU4-1; KL; CCDC96; ENTPD1-AS1; MFSD2A; VAMP5; RPS10P3; CARMIL1; MAGIX; AC020904.2; GRAMD1B; HAP1; LMNTD2-AS1; RBM26-AS1; RNY3; RABGEF1; TMEM62; LRRC26; TLK2P1; SMIM14; TGFB1I1; IZUMO4; HERC6; IGFBP3; AC234917.3; AMH; UBXN10; RAB11B-AS1; ZFP36L2; LIF; C15orf48; SERTAD1; CFAP45; HES1; MGAT2; AC007405.4; PRDM4; RASD2; ATP1A1; TMEM160; BPIFA4P; PXMP4; EFNA4; HMGCS2; HEBP1; SNHG22; AHR; BCCIP; BCAR3; LINC01089; MIR3936HG; AC090772.5; PAM16; TMEM254-AS1; IQCD; ADPRM; HOOK1; TSKU; EFNA3; AC141930.1; AC092910.3; RNF215; MPC2; SPRYD7; RILP; FMO5; CRELD1; NNAT; DNMT3B; ABTB1; DNMBP; AL139385.1; ATG101; FEM1A; SLC24A3; CCDC124; EML5; HDAC10; AC084082.1; RASGEF1A; PRELID2; SDHAF2; SLC27A5; RFLNB; MAPK15; WDR60; ZNF688; AC104109.2; CYP4Z2P; C3orf18; PLEKHG5; EEPD1; PSG4; KCNMA1; VPS37D; DHCR24; LIN7B; DIPK1B; OLMALINC; GPKOW; AC097523.1; SNORD3C; ZNF764; SNHG19; AC087289.5; LARGE1; ETHE1; KLHL8; ZC3H10; ZSCAN31; CAP2; C1QTNF6; RILPL2; HINT2; PPP1R37; TSC22D1; TAGLN; IGSF8; ZFPM1; CUEDC1; CHST3; LTBP4; TMEM107; OSBPL6; MBD3; UQCRHL; DGLUCY; UCKL1-AS1; RPL13P5; ARID5B; AL031666.1; LINC02482; RDM1; CASP9; ACYP2; ENTPD8; AC018521.5; AC108134.2; ABCC12; MAFF; SIX1; CALML5; SESTD1; NR4A1; DHRS2; EFNA5; ANKRD18B; TMEM186; ZC3H12A; NDFIP2; MRC2; TPO; ANAPC11; CST6; KIF13B; ZNF786; TOB1-AS1; AC005914.1; BCL2L12; SH3TC1; AC139149.1; AL731571.1; TMC5; PYDC1; TTPA; DCST1-AS1; TPRN; AL445363.3"},"children":["AQP3; UGDH; FKBP5; SEC14L2; HSD11B2; AC021148.3..."]}],"minWidth":150}],"filterable":true,"dataKey":"628ea9f1ef6b2993864f80e7cb300258"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<pre class="r"><code>plot_grid(plotlist = p, nrow = ceiling(length(p) / 3))</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/barcode-gsea-reg-dir-sig-1.png" alt="*Barcode plots for the top 6 sets of windows associated with __directional__ changes in gene expression.*" width="960" />
<p class="caption">
<em>Barcode plots for the top 6 sets of windows associated with
<strong>directional</strong> changes in gene expression.</em>
</p>
</div>
</div>
<div id="windows-by-feature" class="section level4">
<h4>Windows By Feature</h4>
<pre class="r"><code>gsea_feat_dir &lt;- fgsea(feat_dir_gs, rna_dir_ranks)
gsea_feat_dir_sig &lt;- gsea_feat_dir %&gt;% 
  arrange(pval) %&gt;% 
  dplyr::filter(padj &lt; enrich_alpha, size &gt;= min_sig) %&gt;% 
  as_tibble() 
p &lt;- gsea_feat_dir_sig %&gt;% 
  dplyr::slice(1:9) %&gt;% 
  pull(&quot;pathway&quot;) %&gt;% 
  lapply(
    function(x) {
      plotEnrichment(feat_dir_gs[[x]], rna_dir_ranks) +
        ggtitle(x) +
        theme(
          plot.title = element_text(hjust = 0.5, size = 10)
        )
    }
  )</code></pre>
<pre class="r"><code>cp &lt;-  htmltools::em(
  glue(
    &quot;Merged windows were mapped to genes and their position amongst the &quot;,
    &quot;RNA-Seq results was assessed. Windows were classified based on which &quot;,
    &quot;feature was the best overlap, and the direction of {target} binding with &quot;,
    &quot;{treat_levels[[2]]} treatment. {nrow(gsea_feat_dir_sig)} sets of windows were &quot;,
    &quot;associated with changes in gene expression, using the sign of &quot;, 
    &quot;fold-change and ranking statistic to initially rank the &quot;, 
    &quot;{comma(nrow(rnaseq), 1)} genes considered as detected.&quot;
  )
)
tbl &lt;- gsea_feat_dir_sig %&gt;%
  mutate(
    `Edge Size` = vapply(leadingEdge, length, integer(1)),
    leadingEdge = lapply(leadingEdge, function(x) id2gene[x]) %&gt;% 
      vapply(paste, character(1), collapse = &quot;; &quot;),
    Direction = ifelse(NES &gt; 0, &quot;\u21E7 Up-regulated&quot;, &quot;\u21E9 Down-regulated&quot;),
    &quot;{target}&quot; := str_extract(pathway, &quot;(Up|Down|Unchanged)$&quot;),
    pathway = str_remove_all(pathway, &quot; / [^/]+$&quot;)
  ) %&gt;% 
  dplyr::select(
    pathway, !!sym(target), Windows = size, Direction, 
    p = pval, FDR = padj, `Edge Size`, `Leading Edge` = leadingEdge
  ) %&gt;% 
  reactable(
    filterable = TRUE,
    columns = list2(
      pathway = colDef(
        minWidth = 100, name = &quot;Region&quot;
      ),
      &quot;{target}&quot; := colDef(
        maxWidth = 120,
        cell = function(value) {
          html_symbol &lt;- &quot;&quot;
          if (str_detect(value, &quot;Up&quot;)) html_symbol &lt;- &quot;\u21E7&quot;
          if (str_detect(value, &quot;Down&quot;)) html_symbol &lt;- &quot;\u21E9&quot;
          paste(html_symbol, value)
        },
        style = function(value) {
          colour &lt;- case_when(
            str_detect(value, &quot;Up&quot;) ~ colours$direction[[&quot;up&quot;]],
            str_detect(value, &quot;Down&quot;) ~ colours$direction[[&quot;down&quot;]],
            TRUE ~ colours$direction[[&quot;unchanged&quot;]]
          )
          list(color = colour)
        }
      ),
      Windows = colDef(maxWidth = 80, format = colFormat(separators = TRUE)),
      Direction = colDef(
        name = &quot;Gene Direction&quot;,
        maxWidth = 120,
        style = function(value) {
          colour &lt;- ifelse(
            str_detect(value, &quot;Up&quot;), 
            colours$direction[[&quot;up&quot;]], 
            colours$direction[[&quot;down&quot;]]
          )
          list(color = colour)
        },
      ),
      p = colDef(
        cell = function(value) sprintf(&quot;%.2e&quot;, value), maxWidth = 80
      ),
      FDR = colDef(
        cell = function(value) sprintf(&quot;%.2e&quot;, value), maxWidth = 80
      ),
      &quot;Edge Size&quot; = colDef(maxWidth = 80),
     &quot;Leading Edge&quot; = colDef(
       minWidth = 150,
       cell = function(value) with_tooltip(value, width = 50)
     ) 
    )
  )
div(class = &quot;table&quot;,
  div(class = &quot;table-header&quot;,
      div(class = &quot;caption&quot;, cp),
      tbl
  )
)</code></pre>
<div class="table">
<div class="table-header">
<div class="caption">
<em>Merged windows were mapped to genes and their position amongst the RNA-Seq results was assessed. Windows were classified based on which feature was the best overlap, and the direction of H3K27ac binding with E2DHT treatment. 2 sets of windows were associated with changes in gene expression, using the sign of fold-change and ranking statistic to initially rank the 13,043 genes considered as detected.</em>
</div>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-d37e8a81a591ed3e9b6c" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-d37e8a81a591ed3e9b6c">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"pathway":["No Feature","Enhancer"],"H3K27ac":["Up","Up"],"Windows":[80,178],"Direction":["⇧ Up-regulated","⇧ Up-regulated"],"p":[5.11704982401799e-10,2.77541146853367e-09],"FDR":[2.0468199296072e-09,5.55082293706735e-09],"Edge Size":[25,44],"Leading Edge":["AQP3; FKBP5; SEC14L2; RANBP3L; ZNF689; EHF; ARHGEF37; NRP1; SYBU; NEDD4L; WWC1; AKAP13; MCCC2; FGFR2; TPD52; SGK3; SORD; MYCBP2; CCDC40; CNNM1; MTMR9; CRAT; RASSF3; ACACA; PEX10","AQP3; FKBP5; HSD11B2; ZNF652; EHF; ARHGEF37; AL033523.1; TTC12; KDM4B; PTPRJ; CSRNP1; SASH1; CNTNAP2; CD9; SPRED1; VSTM2L; IQGAP2; PRKAB2; MCCC2; AC092053.4; GSTM3; PEA15; TTC21A; TMEM17; NAMPT; ACACA; PXDC1; PRKAA2; VIPR1; CHPT1; VIPR1-AS1; DUSP16; ZFX; NOL6; CCDC96; MAFB; TOR1AIP2; MOB3C; AL451165.2; MPC2; SPRYD7; FMO5; AC104825.1; ATG101"]},"columns":[{"id":"pathway","name":"Region","type":"character","minWidth":100},{"id":"H3K27ac","name":"H3K27ac","type":"character","cell":["⇧ Up","⇧ Up"],"maxWidth":120,"style":[{"color":"#CA0020"},{"color":"#CA0020"}]},{"id":"Windows","name":"Windows","type":"numeric","format":{"cell":{"separators":true},"aggregated":{"separators":true}},"maxWidth":80},{"id":"Direction","name":"Gene Direction","type":"character","maxWidth":120,"style":[{"color":"#CA0020"},{"color":"#CA0020"}]},{"id":"p","name":"p","type":"numeric","cell":["5.12e-10","2.78e-09"],"maxWidth":80},{"id":"FDR","name":"FDR","type":"numeric","cell":["2.05e-09","5.55e-09"],"maxWidth":80},{"id":"Edge Size","name":"Edge Size","type":"numeric","maxWidth":80},{"id":"Leading Edge","name":"Leading Edge","type":"character","cell":[{"name":"span","attribs":{"title":"AQP3; FKBP5; SEC14L2; RANBP3L; ZNF689; EHF; ARHGEF37; NRP1; SYBU; NEDD4L; WWC1; AKAP13; MCCC2; FGFR2; TPD52; SGK3; SORD; MYCBP2; CCDC40; CNNM1; MTMR9; CRAT; RASSF3; ACACA; PEX10"},"children":["AQP3; FKBP5; SEC14L2; RANBP3L; ZNF689; EHF; ARH..."]},{"name":"span","attribs":{"title":"AQP3; FKBP5; HSD11B2; ZNF652; EHF; ARHGEF37; AL033523.1; TTC12; KDM4B; PTPRJ; CSRNP1; SASH1; CNTNAP2; CD9; SPRED1; VSTM2L; IQGAP2; PRKAB2; MCCC2; AC092053.4; GSTM3; PEA15; TTC21A; TMEM17; NAMPT; ACACA; PXDC1; PRKAA2; VIPR1; CHPT1; VIPR1-AS1; DUSP16; ZFX; NOL6; CCDC96; MAFB; TOR1AIP2; MOB3C; AL451165.2; MPC2; SPRYD7; FMO5; AC104825.1; ATG101"},"children":["AQP3; FKBP5; HSD11B2; ZNF652; EHF; ARHGEF37; AL..."]}],"minWidth":150}],"filterable":true,"dataKey":"86d4865e4c39f51a0b953b3da2b03c5c"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<pre class="r"><code>plot_grid(plotlist = p, nrow = ceiling(length(p) / 3))</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/barcode-gsea-feat-dir-sig-1.png" alt="*Barcode plots for the top 2 sets of windows associated with __directional__ changes in gene expression.*" width="960" />
<p class="caption">
<em>Barcode plots for the top 2 sets of windows associated with
<strong>directional</strong> changes in gene expression.</em>
</p>
</div>
</div>
</div>
<div id="gsea-by-significance-only" class="section level3 tabset">
<h3 class="tabset">GSEA By Significance Only</h3>
<p>GSEA analysis was then performed by taking the genes mapped to
various sets of peaks, and checking for enrichment amongst the RNA-Seq
results, ranking genes only from least to most significant, which
effectively treats up- and down-regulation as indistinguishable.</p>
<div id="all-windows-1" class="section level4">
<h4>All Windows</h4>
<pre class="r"><code>gsea_nondir &lt;- fgsea(dir_gs, rna_sig_ranks, scoreType = &quot;pos&quot;)
gsea_nondir_sig &lt;- gsea_nondir %&gt;% 
  arrange(pval) %&gt;% 
  dplyr::filter(padj &lt; enrich_alpha, size &gt;= min_sig) %&gt;% 
  as_tibble() 
p &lt;- gsea_nondir_sig %&gt;% 
  dplyr::slice(1:9) %&gt;% 
  pull(&quot;pathway&quot;) %&gt;% 
  lapply(
    function(x) {
      plotEnrichment(dir_gs[[x]], rna_sig_ranks) +
        ggtitle(x) +
        theme(plot.title = element_text(hjust = 0.5, size = 10))
    }
  )</code></pre>
<pre class="r"><code>cp &lt;-  htmltools::em(
  glue(
    &quot;Merged windows were mapped to genes and their position amongst the &quot;,
    &quot;RNA-Seq results was assessed. Windows were classified based on  &quot;,
    &quot;the direction of {target} binding with &quot;,
    &quot;{treat_levels[[2]]} treatment. {nrow(gsea_nondir_sig)} sets of windows were &quot;,
    &quot;associated with changes in gene expression, using only the p-value to &quot;, 
    &quot;rank the {comma(nrow(rnaseq), 1)} genes considered as detected.&quot;
  )
)
tbl &lt;- gsea_nondir_sig %&gt;%
  mutate(
    `Edge Size` = vapply(leadingEdge, length, integer(1)),
    leadingEdge = lapply(leadingEdge, function(x) id2gene[x]) %&gt;% 
      vapply(paste, character(1), collapse = &quot;; &quot;),
    &quot;{target}&quot; := str_extract(pathway, &quot;(Up|Down|Unchanged)$&quot;),
    pathway = str_remove_all(pathway, &quot; / [^/]+$&quot;)
  ) %&gt;% 
  dplyr::select(
    !!sym(target), Windows = size, p = pval, FDR = padj, 
    `Edge Size`, `Leading Edge` = leadingEdge
  ) %&gt;% 
  reactable(
    filterable = TRUE,
    columns = list2(
      &quot;{target}&quot; := colDef(
        maxWidth = 150,
        cell = function(value) {
          html_symbol &lt;- &quot;&quot;
          if (str_detect(value, &quot;Up&quot;)) html_symbol &lt;- &quot;\u21E7&quot;
          if (str_detect(value, &quot;Down&quot;)) html_symbol &lt;- &quot;\u21E9&quot;
          paste(html_symbol, value)
        },
        style = function(value) {
           colour &lt;- case_when(
            str_detect(value, &quot;Up&quot;) ~ colours$direction[[&quot;up&quot;]],
            str_detect(value, &quot;Down&quot;) ~ colours$direction[[&quot;down&quot;]],
            TRUE ~ colours$direction[[&quot;unchanged&quot;]]
          )
          list(color = colour)
        }
      ),
      Windows = colDef(maxWidth = 80, format = colFormat(separators = TRUE)),
      p = colDef(
        cell = function(value) sprintf(&quot;%.2e&quot;, value), maxWidth = 80
      ),
      FDR = colDef(
        cell = function(value) sprintf(&quot;%.2e&quot;, value), maxWidth = 80
      ),
      &quot;Edge Size&quot; = colDef(maxWidth = 80),
     &quot;Leading Edge&quot; = colDef(
       minWidth = 150,
       cell = function(value) with_tooltip(value, width = 50)
     ) 
    )
  )
div(class = &quot;table&quot;,
  div(class = &quot;table-header&quot;,
      div(class = &quot;caption&quot;, cp),
      tbl
  )
)</code></pre>
<div class="table">
<div class="table-header">
<div class="caption">
<em>Merged windows were mapped to genes and their position amongst the RNA-Seq results was assessed. Windows were classified based on  the direction of H3K27ac binding with E2DHT treatment. 1 sets of windows were associated with changes in gene expression, using only the p-value to rank the 13,043 genes considered as detected.</em>
</div>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-eb1713a0b9fb05c51af3" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-eb1713a0b9fb05c51af3">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"H3K27ac":["Up"],"Windows":[248],"p":[4.9883421132714e-12],"FDR":[9.9766842265428e-12],"Edge Size":[55],"Leading Edge":["AQP3; FKBP5; SEC14L2; RANBP3L; HSD11B2; ZNF652; ZNF689; EHF; ARHGEF37; NRP1; AL033523.1; TTC12; KDM4B; PTPRJ; SYBU; NEDD4L; CSRNP1; WWC1; SASH1; CNTNAP2; AKAP13; CD9; SPRED1; VSTM2L; IQGAP2; PRKAB2; MCCC2; AC092053.4; FGFR2; GSTM3; PEA15; TPD52; SGK3; SORD; MYCBP2; TTC21A; TMEM17; CCDC40; CNNM1; MTMR9; CRAT; NAMPT; RASSF3; ACACA; GIPR; PEX10; PXDC1; DLG2; PRKAA2; VIPR1; CHPT1; VIPR1-AS1; DUSP16; ZFX; NOL6"]},"columns":[{"id":"H3K27ac","name":"H3K27ac","type":"character","cell":["⇧ Up"],"maxWidth":150,"style":[{"color":"#CA0020"}]},{"id":"Windows","name":"Windows","type":"numeric","format":{"cell":{"separators":true},"aggregated":{"separators":true}},"maxWidth":80},{"id":"p","name":"p","type":"numeric","cell":["4.99e-12"],"maxWidth":80},{"id":"FDR","name":"FDR","type":"numeric","cell":["9.98e-12"],"maxWidth":80},{"id":"Edge Size","name":"Edge Size","type":"numeric","maxWidth":80},{"id":"Leading Edge","name":"Leading Edge","type":"character","cell":[{"name":"span","attribs":{"title":"AQP3; FKBP5; SEC14L2; RANBP3L; HSD11B2; ZNF652; ZNF689; EHF; ARHGEF37; NRP1; AL033523.1; TTC12; KDM4B; PTPRJ; SYBU; NEDD4L; CSRNP1; WWC1; SASH1; CNTNAP2; AKAP13; CD9; SPRED1; VSTM2L; IQGAP2; PRKAB2; MCCC2; AC092053.4; FGFR2; GSTM3; PEA15; TPD52; SGK3; SORD; MYCBP2; TTC21A; TMEM17; CCDC40; CNNM1; MTMR9; CRAT; NAMPT; RASSF3; ACACA; GIPR; PEX10; PXDC1; DLG2; PRKAA2; VIPR1; CHPT1; VIPR1-AS1; DUSP16; ZFX; NOL6"},"children":["AQP3; FKBP5; SEC14L2; RANBP3L; HSD11B2; ZNF652;..."]}],"minWidth":150}],"filterable":true,"dataKey":"f063909741cc792c53cf3f1b634c95a6"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<pre class="r"><code>plot_grid(plotlist = p, nrow = ceiling(length(p) / 3))</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/barcode-gsea-nondir-sig-1.png" alt="*Barcode plots for the top 1 sets of windows associated with __non-directional__ changes in gene expression.*" width="960" />
<p class="caption">
<em>Barcode plots for the top 1 sets of windows associated with
<strong>non-directional</strong> changes in gene expression.</em>
</p>
</div>
</div>
<div id="windows-by-region-1" class="section level4">
<h4>Windows By Region</h4>
<pre class="r"><code>gsea_reg_nondir &lt;- fgsea(reg_dir_gs, rna_sig_ranks, scoreType = &quot;pos&quot;)
gsea_reg_nondir_sig &lt;- gsea_reg_nondir %&gt;% 
  arrange(pval) %&gt;% 
  dplyr::filter(padj &lt; enrich_alpha, size &gt;= min_sig) %&gt;% 
  as_tibble() 
p &lt;- gsea_reg_nondir_sig %&gt;% 
  dplyr::slice(1:9) %&gt;% 
  pull(&quot;pathway&quot;) %&gt;% 
  lapply(
    function(x) {
      plotEnrichment(reg_dir_gs[[x]], rna_sig_ranks) +
        ggtitle(x) +
        theme(plot.title = element_text(hjust = 0.5, size = 10))
    }
  )</code></pre>
<pre class="r"><code>cp &lt;-  htmltools::em(
  glue(
    &quot;Merged windows were mapped to genes and their position amongst the &quot;,
    &quot;RNA-Seq results was assessed. Windows were classified based on which &quot;,
    &quot;region was the best overlap, and the direction of {target} binding with &quot;,
    &quot;{treat_levels[[2]]} treatment. {nrow(gsea_reg_nondir_sig)} sets of windows were &quot;,
    &quot;associated with changes in gene expression, using only the p-value to &quot;, 
    &quot;rank the {comma(nrow(rnaseq), 1)} genes considered as detected.&quot;
  )
)
tbl &lt;- gsea_reg_nondir_sig %&gt;%
  mutate(
    `Edge Size` = vapply(leadingEdge, length, integer(1)),
    leadingEdge = lapply(leadingEdge, function(x) id2gene[x]) %&gt;% 
      vapply(paste, character(1), collapse = &quot;; &quot;),
    &quot;{target}&quot; := str_extract(pathway, &quot;(Up|Down|Unchanged)$&quot;),
    pathway = str_remove_all(pathway, &quot; / [^/]+$&quot;)
  ) %&gt;% 
  dplyr::select(
    pathway, !!sym(target), Windows = size, p = pval, FDR = padj, 
    `Edge Size`, `Leading Edge` = leadingEdge
  ) %&gt;% 
  reactable(
    filterable = TRUE,
    columns = list2(
      pathway = colDef(
        minWidth = 100, name = &quot;Region&quot;
      ),
      &quot;{target}&quot; := colDef(
        maxWidth = 90,
        cell = function(value) {
          html_symbol &lt;- &quot;&quot;
          if (str_detect(value, &quot;Up&quot;)) html_symbol &lt;- &quot;\u21E7&quot;
          if (str_detect(value, &quot;Down&quot;)) html_symbol &lt;- &quot;\u21E9&quot;
          paste(html_symbol, value)
        },
        style = function(value) {
           colour &lt;- case_when(
            str_detect(value, &quot;Up&quot;) ~ colours$direction[[&quot;up&quot;]],
            str_detect(value, &quot;Down&quot;) ~ colours$direction[[&quot;down&quot;]],
            TRUE ~ colours$direction[[&quot;unchanged&quot;]]
          )
          list(color = colour)
        }
      ),
      Windows = colDef(maxWidth = 80, format = colFormat(separators = TRUE)),
      p = colDef(
        cell = function(value) sprintf(&quot;%.2e&quot;, value), maxWidth = 80
      ),
      FDR = colDef(
        cell = function(value) sprintf(&quot;%.2e&quot;, value), maxWidth = 80
      ),
      &quot;Edge Size&quot; = colDef(maxWidth = 80),
     &quot;Leading Edge&quot; = colDef(
       minWidth = 150,
       cell = function(value) with_tooltip(value, width = 50)
     ) 
    )
  )
div(class = &quot;table&quot;,
  div(class = &quot;table-header&quot;,
      div(class = &quot;caption&quot;, cp),
      tbl
  )
)</code></pre>
<div class="table">
<div class="table-header">
<div class="caption">
<em>Merged windows were mapped to genes and their position amongst the RNA-Seq results was assessed. Windows were classified based on which region was the best overlap, and the direction of H3K27ac binding with E2DHT treatment. 8 sets of windows were associated with changes in gene expression, using only the p-value to rank the 13,043 genes considered as detected.</em>
</div>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-c66001ec1bdc3c13d6e2" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-c66001ec1bdc3c13d6e2">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"pathway":["Intron","Intron","Intergenic (>10kb)","Intergenic (<10kb)","Upstream Promoter (-5kb)","Promoter (-1500/+500)","Upstream Promoter (-5kb)","Intergenic (<10kb)"],"H3K27ac":["Up","Unchanged","Unchanged","Unchanged","Unchanged","Up","Up","Up"],"Windows":[142,5614,1668,2081,3387,30,32,31],"p":[3.6219709032596e-07,1.08665887360595e-06,1.71981950968647e-06,3.21586659270006e-06,8.12800847413481e-06,0.000554114181380259,0.000709812531060426,0.0096400727810369],"FDR":[3.98416799358555e-06,5.97662380483272e-06,6.30600486885039e-06,8.84363312992517e-06,1.78816186430966e-05,0.00101587599919714,0.00111541969166638,0.0132551000739257],"Edge Size":[28,998,321,423,599,12,6,10],"Leading Edge":["FKBP5; RANBP3L; ZNF652; ARHGEF37; NRP1; AL033523.1; KDM4B; NEDD4L; SASH1; AKAP13; SPRED1; VSTM2L; IQGAP2; PRKAB2; FGFR2; GSTM3; PEA15; TPD52; SGK3; SORD; MYCBP2; CCDC40; CNNM1; RASSF3; ACACA; GIPR; PXDC1; DLG2","UGDH; FKBP5; SEC14L2; RANBP3L; HSD11B2; AC021148.3; PISD; ZNF652; OTULINL; ARHGEF26; EHF; AMIGO2; ARHGEF37; GLUL; THBS1; NRP1; EPGN; WWC3; CROT; GREB1; AL033523.1; KDM4B; PTPRJ; ADD3; SYBU; AR; MYLIP; ARHGEF28; NEDD4L; GPRC5A; FAM49A; WWC1; SASH1; PRR15; CNTNAP2; IL1R1; TBC1D16; SCUBE2; SLC26A2; AL035661.1; SOWAHC; PER1; DOCK2; SLC22A23; TBC1D2; CRYBG1; KRT15; LEF1; GRIN2B; BAMBI; TMTC2; DAB2; CD9; PDE10A; PLCL1; SPRED1; PPP3CA; TRGC1; CEACAM7; MCC; VSTM2L; IQGAP2; SOX9; FOXI1; STON2; CABLES1; JUN; SOSTDC1; PRKAB2; NADK2; PMEPA1; SRGAP3; SLC37A2; TMEM164; IL20RA; BCL2; CEACAM5; RRP7A; PPFIBP2; SLC16A6; MCCC2; GRHL2; AC007255.1; TRIM9; FHDC1; BLNK; TGFB3; CYP2B7P; FAM167A; FGFR2; ESR1; TMCC3; ASB13; SSTR2; PARP9; SLC25A18; ADGRV1; MTSS1; ADH1C; PREX1; ATP13A4; POU2F3; MKLN1-AS; GSTM3; KLF9; PGR; PEA15; GRM4; PCED1B; RNF152; NXPE3; TPD52; INSYN2A; SGK3; EMP1; TBC1D8; EPN2; BCL2L1; CDRT1; AC005920.1; SORD; ELL2; STARD13-AS; STARD13; MYCBP2; KLF5; OXGR1; AL445309.1; FAM107B; GNA12; LFNG; AC005912.1; LRP2BP; FRMD3; FSBP; WWTR1; CACNB4; DOCK4; ZBTB7C; ANK3; F2R; MUC19; AL157895.2; RARRES1; CCDC40; CNNM1; ALCAM; PAQR5; LRCH1; SEMA6D; TRGC2; GUCY1A2; MYH11; CLEC3A; MYRFL; ID2-AS1; SNX25; VWFP1; PRKCE; PDE9A; CRAT; GPAT3; NR3C2; SOX5; PACSIN2; GPSM2; SPSB1; PXK; LINC02303; CHN1; FBXO32; TNFAIP8; SLC9A1; KMO; GATA3; AC004918.5; BOC; FAM161A; GPR137C; IKZF2; IGFBP4; ARHGEF26-AS1; TESK2; CEACAM6; SYT16; SLC6A9; Z83843.1; NPIPB3; ZNF552; SAP30L-AS1; TOE1; LINC01341; EPHB3; ITGA5; GCNT1; SGSM1; RNF144B; LRIG1; RASSF3; SOCS3; RBBP8NL; NYNRIN; DNM3; BCL6; UAP1; HERC3; AP000331.1; AC022007.1; KCNB1; GARNL3; SCIN; ACACA; AC008268.1; CREB3L1; SLC38A2; GLDC; DSCAM-AS1; ADCY2; RAB11FIP1P1; TIPARP; PGM5; LYPD6; LIPH; AC022400.8; PXDC1; SH3PXD2A; PHLDA1; AL603750.1; STX18-AS1; CMAHP; DOCK8; AL441992.2; SPAG5-AS1; DLG2; SLC2A1; CD44; ADIRF-AS1; PDE4B; PXYLP1; TGFBR3; DLC1; LPAR3; LRATD2; AL158206.1; CENPS; SEMA3F-AS1; CA13; FZD5; GGT6; PIK3C2G; SAMD12-AS1; RGL1; AC105285.1; H4C8; SLC22A5; RN7SL674P; HEG1; GPR161; SPRY4; MYO1F; AC005083.1; VAV3; HMGCS1; CRYM; DLGAP1-AS1; NKD1; LINC01814; AC040162.1; ARHGAP8; TFPI2; ZNF232; FHL2; TTC32; C20orf144; SLC29A3; ARL10; CASD1; GET4; ATP7B; AL353622.1; MAP2; CICP14; NRCAM; PLEKHH1; DTX4; CACNA1D; AL157392.3; PLCH2; DHRS3; DDR2; RFLNA; SLC44A5; WNT9A; TAMM41; FSTL1; PRR15L; PRAG1; GLCCI1; SMKR1; CD160; AP000844.2; HECW2; TRERF1; KRT23; DBI; KLF3-AS1; LRRC46; RAD51B; KIAA0825; VMP1; ARHGAP40; AC005562.1; AL390726.5; PDIA5; ACACB; PIM1; LEKR1; CGNL1; MAGI2; SRCIN1; MOK; BAG3; SHANK2; ZMIZ1; LINC01881; THSD1; OSR2; TSC22D3; PRKAA2; SMIM5; AL137782.1; EGLN3; CMTM8; TMEM63C; VIPR1; RAB30; LIMD2; SNORA37; SPRED2; FOXO6; C14orf132; SYTL2; LMO4; MEGF11; GSN-AS1; ADAM19; RNF150; PDE11A; LYRM9; SMAD6; CHPT1; SCYL3; ZNRF1; AC083843.3; ULK1; RAD17P1; HNRNPCP2; DEPTOR; NRARP; IRX6; ARRB1; ZNF385A; VIPR1-AS1; ID3; PHLDB1; ANXA3; BDNF-AS; SUSD2; DUSP16; VCPKMT; SLC35F1; AL357568.1; ZBTB10; RBM20; TJP2; EXOC3-AS1; TCF7L2; SLC6A17; AC026979.4; ST6GAL1; AC138028.4; AL133230.2; AKAP7; AZGP1P1; PSMD6-AS2; STARD5; AC113404.3; LINC01176; HGD; ZMYND10; WNT2B; NEDD4; ICOSLG; PGAM4; RHBDL3; ERVMER34-1; CSF1; FAM83B; BDKRB2; CBARP; CCDC82; FAM117A; MAST3; CD163L1; LRRC37A3; RNF43; AL133243.2; CEP112; NCCRP1; REEP1; ID1; DTX2; COL5A2; INPP5J; AC116533.1; GADD45B; ENPP1; BCYRN1; AIFM2; PIFO; MNS1; PTAFR; FRAS1; IFNLR1; FAM216A; CNOT2-DT; PPARGC1B; MYRIP; CNKSR3; NT5E; FGD6; BTN2A3P; IFI6; MFF-DT; CTSO; GUCY1B1; KL; CCDC96; ANKRD61; AP000873.2; Z99129.4; AP005131.6; SAMD10; LIFR; PLCH1; PLEKHA2; SLC16A2; URB1-AS1; GRHL3; UNC5B; PDPR; CARMIL1; SEMA3C; MAGIX; WNT5A; GRAMD1B; DRAIC; SLC20A1; ACSM1; NFATC2; MLLT3; IRS2; ANKRD44; AL391001.1; RBM26-AS1; AC007996.1; PRUNE2; NTN1; TMEM62; LRRC26; EPM2A; CCDC34; RPS7P1; TDO2; SMIM14; ABCC3; GAB2; FP565260.3; TOR1AIP2; SNTB1; HERC6; FAM222B; GRIK3; GBP2; DPF1; AC007637.1; CEMIP2; APBA1; TMOD1; MYB; AC108519.1; RHOBTB1; MOCS1; AC093904.4; GNB1L; G3BP2; CXCR6; LINC00511; DBP; ADAMTSL4; IL17RE; TXNDC16; PINK1-AS; AC009061.2; AC232271.1; ARHGEF3; RAB11B-AS1; MAGI3; TUBB3; ZNF501; ZFP36L2; LIF; LINC00265; NFKBIZ; CBR3-AS1; ZNF410; GADD45G; RMDN2; SUPT3H; CATSPERB; BBOX1; MEIS1; AC005831.1; HCAR1; OPN3; HES1; TSFM; LURAP1L; TSPAN12; TSPAN5; AC007405.4; EDN2; ADAMTS13; EML6; PRDM4; PLAAT1; TP53INP2; ATP1A1; ENTPD3-AS1; AADAT; KCNE4; SPOCD1; FANCE; BPIFA4P; PXMP4; FRMD4A; RAPGEF4; RNF166; TCF7; SLC39A14; AC027097.2; EFNA4; VASN; SLC16A4; HMGCS2; PLEKHG4B; PLEKHG1; RPS6KA2; HEBP1; LINC01116; SP140L; SLC16A10; LINC01522; PPIAP29; LDHD; FAM221A; AHR; CCDC167; SPIN3; EVL; BCCIP; GPRIN3; AC010184.1; BCAR3; LINC01089; AF111167.2; GALNT10; DNM1; BCL2L11; LINS1; MIR3936HG; NR1D1; ARMCX1; PCDHA12; MCOLN2; IL17RD; LINC02615; MMEL1; DAPK1; EPN3; GMDS-DT; GATA3-AS1; LAMTOR5-AS1; TRIM45; PAM16; PHKA1; SIAH2; IQCD; ARHGAP27P1-BPTFP1-KPNA2P3; FOXN3-AS1; CD302; BMF; SIDT1; TMEM42; NRM; AC013394.1; AC090517.5; AC093642.2; AP4S1; HOOK1; ZNF117; AC018809.2; CACNA2D2; LRRC15; AC020978.9; RCBTB2; AC051619.7; EFNA3; CDKN2B-AS1; AXIN2; AL451123.1; KSR1; AC141930.1; STBD1; GRB7; CLIP2; STAMBPL1; KALRN; PTOV1-AS2; PPARD; AC138028.2; AL390719.1; FAM66C; GLIS2; CSTA; RPS2P55; C9orf106; LINC02809; LYSMD4; AL451165.2; TBC1D30; ZSCAN12; MB21D2; LCMT1-AS2; SHROOM2; RNF215; USP49; MPC2; MS4A7; ASB16-AS1; BHLHE41; FKRP; STXBP6; BRD7P2; CCDC7; SMCO4; WDFY3-AS2; ZBED3; RILP; LINC-PINT; CCDC134; RSKR; CMPK2; RGS2; AL121906.2; RAP1GAP; FOXN3; NFIL3; POLR2J4; PCDHAC1; MAP2K6; FMO5; SRRM2; OAS1; GPR157; AL158163.1; AC093752.1; TATDN2P2; CDC42EP1; CRELD1; NCOR2; PIK3IP1; SELENON; AC104667.2; LRRTM2; FAM151B; GAR1; USP35; AC104825.1; DICER1-AS1; AC074117.1; AL591848.4; TAF13; ATP6V1B2; CASC2; C15orf62; EEFSEC; HLF; ATP6V1B1; MAN1C1; PARD3B; DNMT3B; ANKRD62P1; PPM1M; FAM47E; ATP13A5; MIR34AHG; FAM189A2; DNMBP; H3P47; FAM124A; SEMA7A; AC008895.1; CCN1; THAP7-AS1; AL157838.1; ANKRD34A; AL139385.1; ZNF814; MTMR11; ZCCHC17; CCDC157; HTR7P1; ZNF431; AC135050.6; SUFU; SLC24A3; SH3RF2; AL157935.2; ANP32B; AC080013.1; MAP1S; CLIC6; MPP7; EML5; NRSN2-AS1; EOGT; AC084082.1; C15orf41; PALM2AKAP2; APTR; SMPDL3A; AC087521.2; AC097376.3; PRELID2; AL009174.1; SLC27A5; LASP1; SCAMP5; ATXN1; GSEC; LINC00624; WDR60; ACTN2; FTX; FLNB-AS1; FAM81A; PLAAT4; FGFR1OP; AC104109.2; PITX1; CYP4Z2P; TNS1; RTN4RL1; AL049834.1; PLEKHG5; SLC25A32; C3orf62; DAPK2; AC009126.1; NSMCE1-DT; KRT8P45; CHST15; NMB; CLSTN2; PGD; PSG4; BICRA; KCNMA1; SRP9P1; FBXL16; CRY2; ATL1; MPND; HOXB7; CCDC71L; VPS37D; CRYBB2P1; ZNF594; CAMKMT; HDAC9; AL158212.3; XG; ALPK3; PLD6; TRIM68; AK1; CLN3; DIPK1B; OLMALINC; GPKOW; RABEP1; KCNQ1OT1; ZSWIM3; PRSS53; RFX2; LRRC49; HOMER2; APIP; CPS1; OCEL1; LARGE1; PDLIM1; SRCAP; JOSD2; ABLIM3; ETHE1; KLHL8; POLE4; UBALD2; MFSD13A; PLA2R1; MAPT; CPQ; ASAP1; CENPQ; METAP1D; RAD51-AS1; CSGALNACT1; AC139887.2; ELK3; RPS17; TMEM143; TMEM8B; MTFP1; NOL4L; AC010536.1; KCNS3; HINT2; KLF3; CYSRT1; SUSD3; ACSS1; ENOX1; PPP1R37; RIPOR3; TSC22D1; TMEM178B; LINC01521; METTL15P1; MR1; ELAVL2; CAMKK1; KCNJ13; GGACT; TIMP2; AC092279.2; KLC3; IGSF8; CPAMD8; ZFPM1; HS6ST1; DLEU1; CAPZB; DAB2IP; ERMAP; ROBO1; LRRC27; CUEDC1; MLXIP; OTUD7A; MAOB; PLAAT3; RBIS; DOCK3; MBTPS1; TUBD1; DGLUCY; MED21; AL391684.1; UCKL1-AS1; SERPINB5; UNC5C; HNF4G; ARID5B; BCAS3; SMOC1; AL031666.1; FBXO27; CEP126; LZTS3; TTYH2; AC005632.6; MATN3; WNT3; AL355488.1; SCUBE3; ULBP2; B3GLCT; ALKBH7; NDST2; MGAM; MTCL1; FAM161B; SIN3B; MAP3K5; ARHGAP44; TEX9; ACYP2; AC018521.5; CAMP; PROM1; SERGEF; STRADB; AC011476.3; ADAMTS17; ANGPTL4; AC018797.3; CRISPLD2; AUH; IGF2BP2; PIK3CB; DNAH11; RGS9BP; RTN4IP1; DBNDD2; ABCC12; A4GALT; AC004943.2; MAFF; EPS8; CTU2; NOD2; GIHCG; PPP3CB-AS1; SIX1; SESTD1; NR4A1; TRIM62; EFNA5; NEK5; FUT4; PAX9; GRTP1; TMEM186; AC015712.2; ASH1L-AS1; ZNF428; MYC; ZC3H12A; FBXL3; SPAG6; GSDMB; CTSS; RERG; JMJD7-PLA2G4B; RAB19; NDFIP2; MRC2; NHSL1; TPO; ANAPC11; EML3; CTNND2; CST6; CRTC3; KIF13B; MED4; KCNC3; ZBED3-AS1; TOB1-AS1; KIAA1522; CDON; TLE1; DHRS4; KTN1-AS1; AC008014.1; CCDC74A; NCALD; WDR66; TTLL1; IQCG; SH3TC1; AC139149.1; KIAA1671; AL121839.2; AL731571.1; TMC5; MAML2; LRRC8C; PYDC1; HS6ST1P1; H2BC10; DACH1; ARSJ; H3P36; DCST1-AS1; TPRN","EHF; KRT4; AMIGO2; SERHL2; WWC3; AL033523.1; PTPRJ; PWWP3B; AR; ARHGEF28; NEDD4L; GPCPD1; SASH1; TBC1D16; SOWAHC; SERHL; DOCK2; AKAP13; SLC22A23; ARL4C; TMTC2; CD9; MCC; IQGAP2; FOXI1; CABLES1; PMEPA1; SLC7A11; MPDZ; NDRG1; TMEM164; BCL2; NFKBIA; RRP7A; MCCC2; RRP7BP; FHDC1; CYP2B7P; AL135905.2; ESR1; TMCC3; DKK1; MTSS1; PREX1; TACR3; CDC42EP3; CXCR4; ENC1; PDGFD; KLF9; GRM4; PCED1B; RNF152; TBC1D8; BCL2L1; COLCA1; STARD13-AS; STARD13; OXGR1; GNA12; YPEL3; FRMD3; GUCY1A1; ZBTB7C; ANK3; PID1; TBX3; SEMA6D; GUCY1A2; MYH11; CLEC3A; GPAT3; NAMPT; SLC16A14; SOX5; SPSB1; PXK; LINC02303; TNFAIP8; FAM161A; LINC01341; RNF144B; LRIG1; KCTD15; DNM3; BCL6; GDPD3; AC008268.1; TIPARP; AL603750.1; SLC2A1; HSPA2; PDE4B; RORC; TAPT1; FZD5; NPIPB13; H4C8; SPRY4; MALT1; FHL2; CASD1; MNX1; GET4; AL353622.1; EIF2AK3-DT; TMEM154; CACNA1D; SYDE1; PARTICL; KCNK5; AL158211.5; FSTL1; CDK18; SMKR1; CORO1A; KCTD6; KRT23; DBI; RAD51B; RBM34; KRT10; LEKR1; GRK5; SHANK2; TSC22D3; SNORA37; SPRED2; FOXO6; C14orf132; SYTL2; ITGAM; PDE11A; SMAD6; MUC16; AC083843.3; ULK1; RAD17P1; CNOT4; ARRB1; SUSD2; DUSP16; RBM20; ST6GAL1; SOX11; COLCA2; CSF1; FAM83B; BDKRB2; FAM117A; LRRC37A3; ID1; SKAP1; INPP5J; GADD45B; PIPSL; PLLP; MYRIP; KL; AC132192.2; MFSD2A; VAMP5; GRHL3; SEMA3C; SILC1; WNT5A; HAP1; MLLT3; RBM26-AS1; MAFB; SNORA23; NOVA1; GRIK3; ZNF519; RHOBTB1; AC093904.4; RRS1; LINC00511; TRIM35; SLC52A3; UBXN10; COL12A1; CYTH1; ZFP36L2; ELF3-AS1; LINC00265; C15orf48; CATSPERB; BBOX1; SPRY1; HES1; LURAP1L; MGAT2; EDN2; PRDM4; RASD2; ATP1A1; MIR1915HG; AADAT; KCNE4; NKILA; TCF7; AC027097.2; PTCHD1; VASN; HMGCS2; FAM225A; LINC01522; TFCP2L1; AHR; EVL; GPRIN3; MRPL12; AF111167.2; LINC02615; TTYH1; AC106782.1; FAM89A; PAM16; AREG; TMEM254-AS1; NABP1; NUP50-DT; TAPT1-AS1; AP4S1; AC007114.1; MTUS2; FGD1; AC092910.3; KALRN; STC2; SLC12A5; AL359715.1; GLIS2; C9orf106; BHLHE41; SMCO4; SEMA6B; SIK1B; FOXN3; AL158163.1; SIK1; AL591848.4; EEFSEC; HLF; FAM241B; TNFRSF11A; GABPB1-AS1; CCN1; AL157838.1; AL139385.1; ANG; ZCCHC4; ATG101; ZCCHC17; AL157935.2; MPP7; EPHA2; RASGEF1A; ATXN1; FAM53A; B3GNTL1; NFIX; DAPK2; APOL3; CHST15; RHOB; ACOX2; PPP1R26-AS1; KIF12; ZNF594; DHCR24; NTRK3; AC009133.1; TNS2; AL109627.1; RABEP1; GLCE; LRRC49; THBD; APIP; SRCAP; AC139530.1; AL691482.4; ASAP1; CAP2; CCDC18-AS1; RIPOR3; TSC22D1; METTL15P1; MR1; LCA5; ARHGEF19; CSPG4P12; CUEDC1; SLC25A25-AS1; FOXD1; SERPINB5; ARID5B; PYROXD2; TMLHE; ARMCX5-GPRASP2; CEP126; TTYH2; AL360270.2; SCUBE3; LINC02482; RPS26P6; PROM1; AC108134.2; ADAMTS17; PPARG; IGF2BP2; PIK3CB; A4GALT; BCL9L; GIHCG; SIX1; NR4A1; TRIM62; DHRS2","AQP3; FKBP5; ZNF689; ANKRD22; OLR1; EHF; KRT4; AMIGO2; ARHGEF37; GLUL; WWC3; CROT; AR; CSRNP1; GPRC5A; TBC1D16; AL035661.1; AC068580.3; PER1; DOCK2; KRT15; LEF1; BAMBI; TENT5C; TRGC1; IQGAP2; TNNI2; FOXI1; NFKBIA; MCCC2; GRHL2; PLEKHD1; BIRC3; KLF13; TRIM2; BMP4; BLNK; PARP14; AC092053.4; CYP2B7P; TMCC3; ASB13; PARP9; PREX1; ENC1; GSTM3; PGR; BMP7; GRM4; PCED1B; RNF152; NXPE3; TPD52; EMP1; BCL2L1; COLCA1; RAB6C; AC005920.1; SORD; ELL2; AL162377.1; IFITM10; TTC21A; ZBTB7C; ANK3; MUC19; TMEM17; TP53TG1; SERPINB2; TRGC2; CLEC3A; ATP10A; GNMT; VWFP1; PRKCE; HECTD3; NAMPT; TEX19; SPSB1; ZNF703; LINC02303; CHN1; CBX4; MUC5B; FBXO32; TRMT9B; GATA3; IKZF2; IGFBP4; TESK2; TOE1; EPHB3; KCTD15; SOCS3; NYNRIN; IGSF9; BCL6; AC008268.1; CREB3L1; SLC38A2; GLDC; GIPR; PHLDA1; AL603750.1; SLC2A1; CD44; LPAR3; TAPT1; AC024896.1; SAMD12-AS1; H4C8; VAV3; CRYM; SEPHS2; FHL2; C20orf144; CASD1; ATP7B; AC068580.4; CACNA1D; MRFAP1; PLCH2; DHRS3; SYDE1; SLC44A5; PARTICL; AL158211.5; QSOX2; TSPAN33; PRAG1; HECW2; TRERF1; SYT8; KRT23; KLF3-AS1; VMP1; SGPP1; BAG3; S100A8; VIPR1; SPRED2; FOXO6; C14orf132; GSN-AS1; AC083843.3; ULK1; RAD17P1; IQCN; GLI4; NUDT11; VIPR1-AS1; ID3; SUSD2; ZBTB10; RBM20; SLC6A17; AC026979.4; KLRG2; AL133230.2; AZGP1P1; HGD; C7orf31; COLCA2; CSF1; BDKRB2; LRRC37A3; POP1; NOL6; ID1; DTX2; SKAP1; MGAT5B; GADD45B; LINC00641; IFNLR1; IFI6; POTEKP; AC132192.2; FRAT1; AL135999.1; MFSD2A; VAMP5; CARMIL1; THTPA; AC020904.2; SILC1; RHPN1-AS1; SLC20A1; AC007191.1; FZD7; PAK6; SETP14; IZUMO4; HERC6; GRIK3; CEMIP2; AC234917.3; MYB; SESN3; RRS1; AP000350.6; CLHC1; DBP; SLC52A3; ALPK1; ELF3-AS1; LIF; LINC00265; NFKBIZ; HSPBAP1; C15orf48; CFAP45; HES1; UROD; TSFM; TSPAN5; EDN2; MCOLN3; RASD2; BX470102.2; MIR1915HG; AADAT; SPOCD1; FANCE; PXMP4; NKILA; KLF8; PTCHD1; HMGCS2; AHR; EVL; MRPL12; AF111167.2; RNU4-2; PCDHA12; LINC02615; MMEL1; GATA3-AS1; SERPINE1; RPSAP47; AREG; TMEM254-AS1; BMF; FAM89B; SALL2; TAPT1-AS1; HOOK1; FOXA3; LRRC15; AC007114.1; TSKU; MSX1; KSR1; GRB7; CLIP2; AC092910.3; STC1; PPARD; STC2; MTHFS; AL359715.1; TMEM216; AP000350.7; C9orf106; AL451165.2; KCNK1; BHLHE41; FKRP; AL121906.2; DDT; MAP2K6; GPR157; AL158163.1; CDC42EP1; NCOR2; C15orf62; TNFRSF11A; ANKRD62P1; DNMBP; SEMA7A; AL157838.1; ANG; ZCCHC4; ATG101; ZCCHC17; FEM1A; SUFU; MAP1S; EPHA2; MAD2L1BP; SEMA4G; ITGAV; PALM2AKAP2; SDHAF2; LASP1; WDR60; ZNF688; FAM81A; PITX1; Z99572.1; POU2F1; CHST15; RHOB; BICRA; HOXB7; AP1AR; FKBP9P1; ZNF594; XG; CLN3; RABEP1; SYNE3; AC005332.3; ZNF764; GLCE; LRRC49; KIAA1211L; SRCAP; ABLIM3; ETHE1; KLHL8; AC139530.1; ZNF319; SRRM3; EMSLR; AL691482.4; ASAP1; CENPQ; RNF125; ELK3; TMEM143; ZDHHC18; ZBTB9; KLF3; SUSD3; ACSS1; FLT4; RIPOR3; TSC22D1; LINC01504; RFX1; TIMP2; HS6ST1; DAB2IP; AC009078.3; SIX5; AC131971.1; CUEDC1; MLXIP; CHST3; FOXD1; TUBD1; MED21; ARID5B; PARP8; CEP126; WNT3; B3GLCT; MYPOP; LINC02482; RPS26P6; PROM1; AC108134.2; LGALS1; ZNF114; RGS9BP; RTN4IP1; A4GALT; BCL9L; MAFF; GIHCG; SIX1; CALML5; SESTD1; TRIM62; DHRS2; ANKRD18B; MYC; ZC3H12A; GSDMB; RAB19; NHSL1; AL360270.1; KCNG1; KIF13B; AL009179.1; FBXO4; ZNF786; TOB1-AS1; EIF4A3; KIAA1522; TNFRSF14; SLC15A1; AC005914.1; CCDC74A; NCALD; SH3TC1; HS6ST1P1; H2BC10; ZSCAN21; PSD4; EIF2AK4; NCEH1; HSPA12A; SCARA3; MINCR; AL390719.2; BTN2A2; ZNF19; AC007114.2; FAM241A; CACNA1H; SECTM1; MRM1; ITPRIPL2; C9orf152; LSM10; DLGAP3; ABTB2; NKAIN1; VAV1; CCNB3; RPL14P1","AQP3; UGDH; FKBP5; SEC14L2; HSD11B2; AC021148.3; PISD; ZNF652; ZNF689; ANKRD22; ARHGEF26; EHF; KRT4; PARM1; ARHGEF37; THBS1; NRP1; GREB1; TTC12; KDM4B; PTPRJ; ADD3; SYBU; NEDD4L; CSRNP1; GPRC5A; WWC1; CNTNAP2; IL1R1; TBC1D16; OSGIN1; SLC26A2; AL035661.1; SERHL; PER1; AKAP13; SLC22A23; KRT15; CD9; TRGC1; CEACAM7; VSTM2L; IQGAP2; SOX9; CABLES1; PRKAB2; PMEPA1; NDRG1; SRGAP3; TMEM164; BCL2; CEACAM5; PPFIBP2; MCCC2; GRHL2; HEY2; LNX2; TRIM9; BLNK; FAM214A; AC092053.4; CYP2B7P; ECI2; AL135905.2; ESR1; VAMP1; ASB13; MTSS1; PREX1; AL606500.1; POU2F3; GCNT2; ENC1; KLF9; BMP7; PCED1B; RNF152; TPD52; EMP1; TBC1D8; EPN2; BCL2L1; COLCA1; AC005920.1; STARD13-AS; ADGRG1; STARD13; MYCBP2; IFITM10; AL445309.1; CRYBG2; LRP2BP; YPEL3; FSBP; TTC21A; CACNB4; ZBTB7C; ANK3; MUC19; SERPINB2; PAQR5; TRGC2; MTMR9; MYH11; CLEC3A; MYRFL; CRAT; NAMPT; GASK1B; PXK; ZNF703; CHN1; CBX4; MUC5B; DENND2B; TRMT9B; IKZF2; ARHGEF26-AS1; TESK2; CEACAM6; TOE1; RNF144B; RASSF3; SOCS3; BX842570.1; RBBP8NL; NYNRIN; IGSF9; BCL6; TRIM66; FAH; HERC3; AC022007.1; KCNB1; GARNL3; SCIN; GDPD3; AC008268.1; CREB3L1; LIPH; AC022400.8; CISH; CMAHP; APCDD1; DLG2; SLC2A1; ADIRF-AS1; OAZ3; PXYLP1; BNIPL; RORC; FZD5; NSRP1; SLC22A5; RN7SL674P; HEG1; SPRY4; MYO1F; RNF227; VAV3; ARHGAP8; SEPHS2; NR1H3; SLC29A3; ARL10; CASD1; GET4; PCDH8; ATP7B; NRCAM; PLEKHH1; DTX4; CACNA1D; AL157392.3; MRFAP1; LBP; RFLNA; PARTICL; AL158211.5; SORD2P; PRR15L; PRAG1; CORO1A; KCTD6; CD160; AC027702.2; FERMT1; TRERF1; DBI; KLF3-AS1; LRRC46; RAD51B; ACACB; DOK7; PIM1; GRK5; RPH3AL; BAG3; SHANK2; ZMIZ1; PPP1R3D; PROSER3; ARID5A; HOXC4; OSR2; TSC22D3; SLC25A35; SMIM5; FAM198B-AS1; AL137782.1; EGLN3; CMTM8; TMEM63C; RAB30; SNORA37; SPRED2; GSDME; ITGAM; GSN-AS1; PDE11A; AC118344.4; AC105339.2; ULK1; CNOT4; SLC4A8; IQCN; MRPL38; IQCH-AS1; IRX6; ARRB1; KRT81; ZNF385A; ID3; BDNF-AS; VCPKMT; NPRL3; RBM20; TCF7L2; AC026979.4; ST6GAL1; PSMD6-AS2; LINC01176; HGD; NEDD4; ICOSLG; COLCA2; ZFX; AL353593.1; DRC3; SYT15; LOXL3; MAST3; LRRC37A3; PSCA; NOL6; MMP25; ID1; SKAP1; INPP5J; GADD45B; ENPP1; BCYRN1; AIFM2; PIFO; IFNLR1; CNOT2-DT; AL135905.1; FGD6; BTN2A3P; RNU4-1; CTSO; GUCY1B1; KL; CCDC96; ANKRD61; ENTPD1-AS1; AP005131.6; MFSD2A; SAMD10; VAMP5; RPS10P3; GRHL3; PDPR; CARMIL1; MAGIX; AC020904.2; GRAMD1B; DRAIC; HAP1; LMNTD2-AS1; NFATC2; RBM26-AS1; RNY3; RABGEF1; TMEM62; LRRC26; TLK2P1; TDO2; SMIM14; ABCC3; RAVER1; TGFB1I1; FP565260.3; CHRM1; IZUMO4; HERC6; GRIK3; GBP2; DPF1; AC007637.1; IGFBP3; AC234917.3; MYB; AMH; AC093904.4; LINC00511; ADAMTSL4; IL17RE; UBXN10; AC232271.1; ARHGEF3; RAB11B-AS1; ZFP36L2; LIF; LINC00265; C15orf48; SERTAD1; CFAP45; CATSPERB; HCAR1; HES1; DNASE1L2; HUS1; MGAT2; AC007405.4; BACE1-AS; PRDM4; RASD2; ATP1A1; TMEM160; BPIFA4P; PXMP4; FRMD4A; AC027097.2; KLF8; EFNA4; HMGCS2; HEBP1; SNHG22; LINC01522; AHR; EVL; BCCIP; AC010184.1; BCAR3; LINC01089; MIR3936HG; TMEM44; AC090772.5; NAV1; EPN3; TRIM45; AC106782.1; PAM16; SAMD15; TMEM254-AS1; GPR162; IQCD; SIDT1; AL928654.1; ADPRM; NRM; FAM89B; BAHCC1; AC013394.1; HOOK1; ZNF117; FOXA3; AC018809.2; LRRC15; AC020978.9; TSKU; EFNA3; CDKN2B-AS1; FCGRT; KSR1; AC141930.1; AC092910.3; KALRN; AC121338.2; IRF5; FO681492.1; CSTA; ZSCAN12; SHROOM2; RNF215; MPC2; ZFPL1; MS4A7; CSRNP2; FKRP; AC005041.1; SPRYD7; KDM6B; RILP; RAP1GAP; FMO5; SRRM2; AL158163.1; CDC42EP1; CRELD1; NCOR2; SELENON; GXYLT2; NNAT; DICER1-AS1; C15orf62; EEFSEC; UBAP1L; MAN1C1; DNMT3B; ABTB1; DNMBP; MLKL; AC008895.1; THAP7-AS1; AL157838.1; AL139385.1; ANG; ATG101; MTMR11; MLLT11; FEM1A; CCDC157; HTR7P1; AC133555.3; SLC24A3; SEPTIN1; CCDC124; MPP7; EML5; HDAC10; AC084082.1; RASGEF1A; PALM2AKAP2; AC097376.3; PRELID2; SDHAF2; SLC27A5; RFLNB; SCAMP5; MAPK15; GSEC; WDR60; ZNF688; ID2; FTX; PRR12; TRIM17; PLAAT4; B3GNTL1; AC104109.2; CYP4Z2P; C3orf18; NFIX; AL049834.1; PLEKHG5; AC009126.1; EEPD1; PSG4; BICRA; KCNMA1; ACOX2; POM121C; BAALC-AS1; VPS37D; KIF12; CRYBB2P1; TMC8; DHCR24; HDAC9; AL158212.3; ALPK3; LIN7B; CLN3; TNS2; DIPK1B; OLMALINC; GPKOW; AC097523.1; SNORD3C; SYNE3; ZNF764; SNHG19; RFX2; LRRC49; NHEJ1; AC087289.5; FFAR2; LARGE1; SRCAP; ETHE1; KLHL8; UBALD2; ZC3H10; ZSCAN31; HOXC10; CAP2; C1QTNF6; TMEM8B; LIN37; RILPL2; HINT2; KLF3; CYSRT1; PPP1R37; TSC22D1; TAGLN; MR1; TIMP2; IGSF8; ZFPM1; HS6ST1; DLEU1; AC131971.1; TTC13; CUEDC1; MLXIP; CHST3; LTBP4; TMEM107; PLAAT3; OSBPL6; MBD3; DOCK3; UQCRHL; USP3-AS1; DGLUCY; UCKL1-AS1; RPL13P5; UNC5C; ARID5B; AL031666.1; CEP126; NDST2; MGAM; LINC02482; RDM1; CASP9; RBL2; ACYP2; ENTPD8; AC018521.5; PROM1; STRADB; AC108134.2; LGALS1; PPARG; CRISPLD2; PDP2; RGS9BP; ABCC12; BCL9L; MAFF; SIX1; CALML5; SESTD1; NR4A1; DHRS2; EFNA5; ANKRD18B; TMEM186; MYC; ZC3H12A; GSDMB; NDFIP2; MRC2; TPO; ANAPC11; CST6; CRTC3; KIF13B; KCNC3; ZNF786; TOB1-AS1; KIAA1522; TLE1; PLXNB3; KTN1-AS1; AC005914.1; NCALD; WDR66; BCL2L12; IQCG; SH3TC1; AC139149.1; GAD1; AL731571.1; TMC5","FKBP5; SEC14L2; ZNF689; TTC12; SYBU; CSRNP1; AC092053.4; TTC21A; MTMR9; CRAT; PEX10; CHPT1","FKBP5; HSD11B2; PTPRJ; CNTNAP2; MCCC2; MYCBP2","AQP3; EHF; WWC1; CD9; MCCC2; NAMPT; VIPR1; VIPR1-AS1; DUSP16; NOL6"]},"columns":[{"id":"pathway","name":"Region","type":"character","minWidth":100},{"id":"H3K27ac","name":"H3K27ac","type":"character","cell":["⇧ Up"," Unchanged"," Unchanged"," Unchanged"," Unchanged","⇧ Up","⇧ Up","⇧ Up"],"maxWidth":90,"style":[{"color":"#CA0020"},{"color":"#7F7F7F"},{"color":"#7F7F7F"},{"color":"#7F7F7F"},{"color":"#7F7F7F"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"}]},{"id":"Windows","name":"Windows","type":"numeric","format":{"cell":{"separators":true},"aggregated":{"separators":true}},"maxWidth":80},{"id":"p","name":"p","type":"numeric","cell":["3.62e-07","1.09e-06","1.72e-06","3.22e-06","8.13e-06","5.54e-04","7.10e-04","9.64e-03"],"maxWidth":80},{"id":"FDR","name":"FDR","type":"numeric","cell":["3.98e-06","5.98e-06","6.31e-06","8.84e-06","1.79e-05","1.02e-03","1.12e-03","1.33e-02"],"maxWidth":80},{"id":"Edge Size","name":"Edge Size","type":"numeric","maxWidth":80},{"id":"Leading Edge","name":"Leading Edge","type":"character","cell":[{"name":"span","attribs":{"title":"FKBP5; RANBP3L; ZNF652; ARHGEF37; NRP1; AL033523.1; KDM4B; NEDD4L; SASH1; AKAP13; SPRED1; VSTM2L; IQGAP2; PRKAB2; FGFR2; GSTM3; PEA15; TPD52; SGK3; SORD; MYCBP2; CCDC40; CNNM1; RASSF3; ACACA; GIPR; PXDC1; DLG2"},"children":["FKBP5; RANBP3L; ZNF652; ARHGEF37; NRP1; AL03352..."]},{"name":"span","attribs":{"title":"UGDH; FKBP5; SEC14L2; RANBP3L; HSD11B2; AC021148.3; PISD; ZNF652; OTULINL; ARHGEF26; EHF; AMIGO2; ARHGEF37; GLUL; THBS1; NRP1; EPGN; WWC3; CROT; GREB1; AL033523.1; KDM4B; PTPRJ; ADD3; SYBU; AR; MYLIP; ARHGEF28; NEDD4L; GPRC5A; FAM49A; WWC1; SASH1; PRR15; CNTNAP2; IL1R1; TBC1D16; SCUBE2; SLC26A2; AL035661.1; SOWAHC; PER1; DOCK2; SLC22A23; TBC1D2; CRYBG1; KRT15; LEF1; GRIN2B; BAMBI; TMTC2; DAB2; CD9; PDE10A; PLCL1; SPRED1; PPP3CA; TRGC1; CEACAM7; MCC; VSTM2L; IQGAP2; SOX9; FOXI1; STON2; CABLES1; JUN; SOSTDC1; PRKAB2; NADK2; PMEPA1; SRGAP3; SLC37A2; TMEM164; IL20RA; BCL2; CEACAM5; RRP7A; PPFIBP2; SLC16A6; MCCC2; GRHL2; AC007255.1; TRIM9; FHDC1; BLNK; TGFB3; CYP2B7P; FAM167A; FGFR2; ESR1; TMCC3; ASB13; SSTR2; PARP9; SLC25A18; ADGRV1; MTSS1; ADH1C; PREX1; ATP13A4; POU2F3; MKLN1-AS; GSTM3; KLF9; PGR; PEA15; GRM4; PCED1B; RNF152; NXPE3; TPD52; INSYN2A; SGK3; EMP1; TBC1D8; EPN2; BCL2L1; CDRT1; AC005920.1; SORD; ELL2; STARD13-AS; STARD13; MYCBP2; KLF5; OXGR1; AL445309.1; FAM107B; GNA12; LFNG; AC005912.1; LRP2BP; FRMD3; FSBP; WWTR1; CACNB4; DOCK4; ZBTB7C; ANK3; F2R; MUC19; AL157895.2; RARRES1; CCDC40; CNNM1; ALCAM; PAQR5; LRCH1; SEMA6D; TRGC2; GUCY1A2; MYH11; CLEC3A; MYRFL; ID2-AS1; SNX25; VWFP1; PRKCE; PDE9A; CRAT; GPAT3; NR3C2; SOX5; PACSIN2; GPSM2; SPSB1; PXK; LINC02303; CHN1; FBXO32; TNFAIP8; SLC9A1; KMO; GATA3; AC004918.5; BOC; FAM161A; GPR137C; IKZF2; IGFBP4; ARHGEF26-AS1; TESK2; CEACAM6; SYT16; SLC6A9; Z83843.1; NPIPB3; ZNF552; SAP30L-AS1; TOE1; LINC01341; EPHB3; ITGA5; GCNT1; SGSM1; RNF144B; LRIG1; RASSF3; SOCS3; RBBP8NL; NYNRIN; DNM3; BCL6; UAP1; HERC3; AP000331.1; AC022007.1; KCNB1; GARNL3; SCIN; ACACA; AC008268.1; CREB3L1; SLC38A2; GLDC; DSCAM-AS1; ADCY2; RAB11FIP1P1; TIPARP; PGM5; LYPD6; LIPH; AC022400.8; PXDC1; SH3PXD2A; PHLDA1; AL603750.1; STX18-AS1; CMAHP; DOCK8; AL441992.2; SPAG5-AS1; DLG2; SLC2A1; CD44; ADIRF-AS1; PDE4B; PXYLP1; TGFBR3; DLC1; LPAR3; LRATD2; AL158206.1; CENPS; SEMA3F-AS1; CA13; FZD5; GGT6; PIK3C2G; SAMD12-AS1; RGL1; AC105285.1; H4C8; SLC22A5; RN7SL674P; HEG1; GPR161; SPRY4; MYO1F; AC005083.1; VAV3; HMGCS1; CRYM; DLGAP1-AS1; NKD1; LINC01814; AC040162.1; ARHGAP8; TFPI2; ZNF232; FHL2; TTC32; C20orf144; SLC29A3; ARL10; CASD1; GET4; ATP7B; AL353622.1; MAP2; CICP14; NRCAM; PLEKHH1; DTX4; CACNA1D; AL157392.3; PLCH2; DHRS3; DDR2; RFLNA; SLC44A5; WNT9A; TAMM41; FSTL1; PRR15L; PRAG1; GLCCI1; SMKR1; CD160; AP000844.2; HECW2; TRERF1; KRT23; DBI; KLF3-AS1; LRRC46; RAD51B; KIAA0825; VMP1; ARHGAP40; AC005562.1; AL390726.5; PDIA5; ACACB; PIM1; LEKR1; CGNL1; MAGI2; SRCIN1; MOK; BAG3; SHANK2; ZMIZ1; LINC01881; THSD1; OSR2; TSC22D3; PRKAA2; SMIM5; AL137782.1; EGLN3; CMTM8; TMEM63C; VIPR1; RAB30; LIMD2; SNORA37; SPRED2; FOXO6; C14orf132; SYTL2; LMO4; MEGF11; GSN-AS1; ADAM19; RNF150; PDE11A; LYRM9; SMAD6; CHPT1; SCYL3; ZNRF1; AC083843.3; ULK1; RAD17P1; HNRNPCP2; DEPTOR; NRARP; IRX6; ARRB1; ZNF385A; VIPR1-AS1; ID3; PHLDB1; ANXA3; BDNF-AS; SUSD2; DUSP16; VCPKMT; SLC35F1; AL357568.1; ZBTB10; RBM20; TJP2; EXOC3-AS1; TCF7L2; SLC6A17; AC026979.4; ST6GAL1; AC138028.4; AL133230.2; AKAP7; AZGP1P1; PSMD6-AS2; STARD5; AC113404.3; LINC01176; HGD; ZMYND10; WNT2B; NEDD4; ICOSLG; PGAM4; RHBDL3; ERVMER34-1; CSF1; FAM83B; BDKRB2; CBARP; CCDC82; FAM117A; MAST3; CD163L1; LRRC37A3; RNF43; AL133243.2; CEP112; NCCRP1; REEP1; ID1; DTX2; COL5A2; INPP5J; AC116533.1; GADD45B; ENPP1; BCYRN1; AIFM2; PIFO; MNS1; PTAFR; FRAS1; IFNLR1; FAM216A; CNOT2-DT; PPARGC1B; MYRIP; CNKSR3; NT5E; FGD6; BTN2A3P; IFI6; MFF-DT; CTSO; GUCY1B1; KL; CCDC96; ANKRD61; AP000873.2; Z99129.4; AP005131.6; SAMD10; LIFR; PLCH1; PLEKHA2; SLC16A2; URB1-AS1; GRHL3; UNC5B; PDPR; CARMIL1; SEMA3C; MAGIX; WNT5A; GRAMD1B; DRAIC; SLC20A1; ACSM1; NFATC2; MLLT3; IRS2; ANKRD44; AL391001.1; RBM26-AS1; AC007996.1; PRUNE2; NTN1; TMEM62; LRRC26; EPM2A; CCDC34; RPS7P1; TDO2; SMIM14; ABCC3; GAB2; FP565260.3; TOR1AIP2; SNTB1; HERC6; FAM222B; GRIK3; GBP2; DPF1; AC007637.1; CEMIP2; APBA1; TMOD1; MYB; AC108519.1; RHOBTB1; MOCS1; AC093904.4; GNB1L; G3BP2; CXCR6; LINC00511; DBP; ADAMTSL4; IL17RE; TXNDC16; PINK1-AS; AC009061.2; AC232271.1; ARHGEF3; RAB11B-AS1; MAGI3; TUBB3; ZNF501; ZFP36L2; LIF; LINC00265; NFKBIZ; CBR3-AS1; ZNF410; GADD45G; RMDN2; SUPT3H; CATSPERB; BBOX1; MEIS1; AC005831.1; HCAR1; OPN3; HES1; TSFM; LURAP1L; TSPAN12; TSPAN5; AC007405.4; EDN2; ADAMTS13; EML6; PRDM4; PLAAT1; TP53INP2; ATP1A1; ENTPD3-AS1; AADAT; KCNE4; SPOCD1; FANCE; BPIFA4P; PXMP4; FRMD4A; RAPGEF4; RNF166; TCF7; SLC39A14; AC027097.2; EFNA4; VASN; SLC16A4; HMGCS2; PLEKHG4B; PLEKHG1; RPS6KA2; HEBP1; LINC01116; SP140L; SLC16A10; LINC01522; PPIAP29; LDHD; FAM221A; AHR; CCDC167; SPIN3; EVL; BCCIP; GPRIN3; AC010184.1; BCAR3; LINC01089; AF111167.2; GALNT10; DNM1; BCL2L11; LINS1; MIR3936HG; NR1D1; ARMCX1; PCDHA12; MCOLN2; IL17RD; LINC02615; MMEL1; DAPK1; EPN3; GMDS-DT; GATA3-AS1; LAMTOR5-AS1; TRIM45; PAM16; PHKA1; SIAH2; IQCD; ARHGAP27P1-BPTFP1-KPNA2P3; FOXN3-AS1; CD302; BMF; SIDT1; TMEM42; NRM; AC013394.1; AC090517.5; AC093642.2; AP4S1; HOOK1; ZNF117; AC018809.2; CACNA2D2; LRRC15; AC020978.9; RCBTB2; AC051619.7; EFNA3; CDKN2B-AS1; AXIN2; AL451123.1; KSR1; AC141930.1; STBD1; GRB7; CLIP2; STAMBPL1; KALRN; PTOV1-AS2; PPARD; AC138028.2; AL390719.1; FAM66C; GLIS2; CSTA; RPS2P55; C9orf106; LINC02809; LYSMD4; AL451165.2; TBC1D30; ZSCAN12; MB21D2; LCMT1-AS2; SHROOM2; RNF215; USP49; MPC2; MS4A7; ASB16-AS1; BHLHE41; FKRP; STXBP6; BRD7P2; CCDC7; SMCO4; WDFY3-AS2; ZBED3; RILP; LINC-PINT; CCDC134; RSKR; CMPK2; RGS2; AL121906.2; RAP1GAP; FOXN3; NFIL3; POLR2J4; PCDHAC1; MAP2K6; FMO5; SRRM2; OAS1; GPR157; AL158163.1; AC093752.1; TATDN2P2; CDC42EP1; CRELD1; NCOR2; PIK3IP1; SELENON; AC104667.2; LRRTM2; FAM151B; GAR1; USP35; AC104825.1; DICER1-AS1; AC074117.1; AL591848.4; TAF13; ATP6V1B2; CASC2; C15orf62; EEFSEC; HLF; ATP6V1B1; MAN1C1; PARD3B; DNMT3B; ANKRD62P1; PPM1M; FAM47E; ATP13A5; MIR34AHG; FAM189A2; DNMBP; H3P47; FAM124A; SEMA7A; AC008895.1; CCN1; THAP7-AS1; AL157838.1; ANKRD34A; AL139385.1; ZNF814; MTMR11; ZCCHC17; CCDC157; HTR7P1; ZNF431; AC135050.6; SUFU; SLC24A3; SH3RF2; AL157935.2; ANP32B; AC080013.1; MAP1S; CLIC6; MPP7; EML5; NRSN2-AS1; EOGT; AC084082.1; C15orf41; PALM2AKAP2; APTR; SMPDL3A; AC087521.2; AC097376.3; PRELID2; AL009174.1; SLC27A5; LASP1; SCAMP5; ATXN1; GSEC; LINC00624; WDR60; ACTN2; FTX; FLNB-AS1; FAM81A; PLAAT4; FGFR1OP; AC104109.2; PITX1; CYP4Z2P; TNS1; RTN4RL1; AL049834.1; PLEKHG5; SLC25A32; C3orf62; DAPK2; AC009126.1; NSMCE1-DT; KRT8P45; CHST15; NMB; CLSTN2; PGD; PSG4; BICRA; KCNMA1; SRP9P1; FBXL16; CRY2; ATL1; MPND; HOXB7; CCDC71L; VPS37D; CRYBB2P1; ZNF594; CAMKMT; HDAC9; AL158212.3; XG; ALPK3; PLD6; TRIM68; AK1; CLN3; DIPK1B; OLMALINC; GPKOW; RABEP1; KCNQ1OT1; ZSWIM3; PRSS53; RFX2; LRRC49; HOMER2; APIP; CPS1; OCEL1; LARGE1; PDLIM1; SRCAP; JOSD2; ABLIM3; ETHE1; KLHL8; POLE4; UBALD2; MFSD13A; PLA2R1; MAPT; CPQ; ASAP1; CENPQ; METAP1D; RAD51-AS1; CSGALNACT1; AC139887.2; ELK3; RPS17; TMEM143; TMEM8B; MTFP1; NOL4L; AC010536.1; KCNS3; HINT2; KLF3; CYSRT1; SUSD3; ACSS1; ENOX1; PPP1R37; RIPOR3; TSC22D1; TMEM178B; LINC01521; METTL15P1; MR1; ELAVL2; CAMKK1; KCNJ13; GGACT; TIMP2; AC092279.2; KLC3; IGSF8; CPAMD8; ZFPM1; HS6ST1; DLEU1; CAPZB; DAB2IP; ERMAP; ROBO1; LRRC27; CUEDC1; MLXIP; OTUD7A; MAOB; PLAAT3; RBIS; DOCK3; MBTPS1; TUBD1; DGLUCY; MED21; AL391684.1; UCKL1-AS1; SERPINB5; UNC5C; HNF4G; ARID5B; BCAS3; SMOC1; AL031666.1; FBXO27; CEP126; LZTS3; TTYH2; AC005632.6; MATN3; WNT3; AL355488.1; SCUBE3; ULBP2; B3GLCT; ALKBH7; NDST2; MGAM; MTCL1; FAM161B; SIN3B; MAP3K5; ARHGAP44; TEX9; ACYP2; AC018521.5; CAMP; PROM1; SERGEF; STRADB; AC011476.3; ADAMTS17; ANGPTL4; AC018797.3; CRISPLD2; AUH; IGF2BP2; PIK3CB; DNAH11; RGS9BP; RTN4IP1; DBNDD2; ABCC12; A4GALT; AC004943.2; MAFF; EPS8; CTU2; NOD2; GIHCG; PPP3CB-AS1; SIX1; SESTD1; NR4A1; TRIM62; EFNA5; NEK5; FUT4; PAX9; GRTP1; TMEM186; AC015712.2; ASH1L-AS1; ZNF428; MYC; ZC3H12A; FBXL3; SPAG6; GSDMB; CTSS; RERG; JMJD7-PLA2G4B; RAB19; NDFIP2; MRC2; NHSL1; TPO; ANAPC11; EML3; CTNND2; CST6; CRTC3; KIF13B; MED4; KCNC3; ZBED3-AS1; TOB1-AS1; KIAA1522; CDON; TLE1; DHRS4; KTN1-AS1; AC008014.1; CCDC74A; NCALD; WDR66; TTLL1; IQCG; SH3TC1; AC139149.1; KIAA1671; AL121839.2; AL731571.1; TMC5; MAML2; LRRC8C; PYDC1; HS6ST1P1; H2BC10; DACH1; ARSJ; H3P36; DCST1-AS1; TPRN"},"children":["UGDH; FKBP5; SEC14L2; RANBP3L; HSD11B2; AC02114..."]},{"name":"span","attribs":{"title":"EHF; KRT4; AMIGO2; SERHL2; WWC3; AL033523.1; PTPRJ; PWWP3B; AR; ARHGEF28; NEDD4L; GPCPD1; SASH1; TBC1D16; SOWAHC; SERHL; DOCK2; AKAP13; SLC22A23; ARL4C; TMTC2; CD9; MCC; IQGAP2; FOXI1; CABLES1; PMEPA1; SLC7A11; MPDZ; NDRG1; TMEM164; BCL2; NFKBIA; RRP7A; MCCC2; RRP7BP; FHDC1; CYP2B7P; AL135905.2; ESR1; TMCC3; DKK1; MTSS1; PREX1; TACR3; CDC42EP3; CXCR4; ENC1; PDGFD; KLF9; GRM4; PCED1B; RNF152; TBC1D8; BCL2L1; COLCA1; STARD13-AS; STARD13; OXGR1; GNA12; YPEL3; FRMD3; GUCY1A1; ZBTB7C; ANK3; PID1; TBX3; SEMA6D; GUCY1A2; MYH11; CLEC3A; GPAT3; NAMPT; SLC16A14; SOX5; SPSB1; PXK; LINC02303; TNFAIP8; FAM161A; LINC01341; RNF144B; LRIG1; KCTD15; DNM3; BCL6; GDPD3; AC008268.1; TIPARP; AL603750.1; SLC2A1; HSPA2; PDE4B; RORC; TAPT1; FZD5; NPIPB13; H4C8; SPRY4; MALT1; FHL2; CASD1; MNX1; GET4; AL353622.1; EIF2AK3-DT; TMEM154; CACNA1D; SYDE1; PARTICL; KCNK5; AL158211.5; FSTL1; CDK18; SMKR1; CORO1A; KCTD6; KRT23; DBI; RAD51B; RBM34; KRT10; LEKR1; GRK5; SHANK2; TSC22D3; SNORA37; SPRED2; FOXO6; C14orf132; SYTL2; ITGAM; PDE11A; SMAD6; MUC16; AC083843.3; ULK1; RAD17P1; CNOT4; ARRB1; SUSD2; DUSP16; RBM20; ST6GAL1; SOX11; COLCA2; CSF1; FAM83B; BDKRB2; FAM117A; LRRC37A3; ID1; SKAP1; INPP5J; GADD45B; PIPSL; PLLP; MYRIP; KL; AC132192.2; MFSD2A; VAMP5; GRHL3; SEMA3C; SILC1; WNT5A; HAP1; MLLT3; RBM26-AS1; MAFB; SNORA23; NOVA1; GRIK3; ZNF519; RHOBTB1; AC093904.4; RRS1; LINC00511; TRIM35; SLC52A3; UBXN10; COL12A1; CYTH1; ZFP36L2; ELF3-AS1; LINC00265; C15orf48; CATSPERB; BBOX1; SPRY1; HES1; LURAP1L; MGAT2; EDN2; PRDM4; RASD2; ATP1A1; MIR1915HG; AADAT; KCNE4; NKILA; TCF7; AC027097.2; PTCHD1; VASN; HMGCS2; FAM225A; LINC01522; TFCP2L1; AHR; EVL; GPRIN3; MRPL12; AF111167.2; LINC02615; TTYH1; AC106782.1; FAM89A; PAM16; AREG; TMEM254-AS1; NABP1; NUP50-DT; TAPT1-AS1; AP4S1; AC007114.1; MTUS2; FGD1; AC092910.3; KALRN; STC2; SLC12A5; AL359715.1; GLIS2; C9orf106; BHLHE41; SMCO4; SEMA6B; SIK1B; FOXN3; AL158163.1; SIK1; AL591848.4; EEFSEC; HLF; FAM241B; TNFRSF11A; GABPB1-AS1; CCN1; AL157838.1; AL139385.1; ANG; ZCCHC4; ATG101; ZCCHC17; AL157935.2; MPP7; EPHA2; RASGEF1A; ATXN1; FAM53A; B3GNTL1; NFIX; DAPK2; APOL3; CHST15; RHOB; ACOX2; PPP1R26-AS1; KIF12; ZNF594; DHCR24; NTRK3; AC009133.1; TNS2; AL109627.1; RABEP1; GLCE; LRRC49; THBD; APIP; SRCAP; AC139530.1; AL691482.4; ASAP1; CAP2; CCDC18-AS1; RIPOR3; TSC22D1; METTL15P1; MR1; LCA5; ARHGEF19; CSPG4P12; CUEDC1; SLC25A25-AS1; FOXD1; SERPINB5; ARID5B; PYROXD2; TMLHE; ARMCX5-GPRASP2; CEP126; TTYH2; AL360270.2; SCUBE3; LINC02482; RPS26P6; PROM1; AC108134.2; ADAMTS17; PPARG; IGF2BP2; PIK3CB; A4GALT; BCL9L; GIHCG; SIX1; NR4A1; TRIM62; DHRS2"},"children":["EHF; KRT4; AMIGO2; SERHL2; WWC3; AL033523.1; PT..."]},{"name":"span","attribs":{"title":"AQP3; FKBP5; ZNF689; ANKRD22; OLR1; EHF; KRT4; AMIGO2; ARHGEF37; GLUL; WWC3; CROT; AR; CSRNP1; GPRC5A; TBC1D16; AL035661.1; AC068580.3; PER1; DOCK2; KRT15; LEF1; BAMBI; TENT5C; TRGC1; IQGAP2; TNNI2; FOXI1; NFKBIA; MCCC2; GRHL2; PLEKHD1; BIRC3; KLF13; TRIM2; BMP4; BLNK; PARP14; AC092053.4; CYP2B7P; TMCC3; ASB13; PARP9; PREX1; ENC1; GSTM3; PGR; BMP7; GRM4; PCED1B; RNF152; NXPE3; TPD52; EMP1; BCL2L1; COLCA1; RAB6C; AC005920.1; SORD; ELL2; AL162377.1; IFITM10; TTC21A; ZBTB7C; ANK3; MUC19; TMEM17; TP53TG1; SERPINB2; TRGC2; CLEC3A; ATP10A; GNMT; VWFP1; PRKCE; HECTD3; NAMPT; TEX19; SPSB1; ZNF703; LINC02303; CHN1; CBX4; MUC5B; FBXO32; TRMT9B; GATA3; IKZF2; IGFBP4; TESK2; TOE1; EPHB3; KCTD15; SOCS3; NYNRIN; IGSF9; BCL6; AC008268.1; CREB3L1; SLC38A2; GLDC; GIPR; PHLDA1; AL603750.1; SLC2A1; CD44; LPAR3; TAPT1; AC024896.1; SAMD12-AS1; H4C8; VAV3; CRYM; SEPHS2; FHL2; C20orf144; CASD1; ATP7B; AC068580.4; CACNA1D; MRFAP1; PLCH2; DHRS3; SYDE1; SLC44A5; PARTICL; AL158211.5; QSOX2; TSPAN33; PRAG1; HECW2; TRERF1; SYT8; KRT23; KLF3-AS1; VMP1; SGPP1; BAG3; S100A8; VIPR1; SPRED2; FOXO6; C14orf132; GSN-AS1; AC083843.3; ULK1; RAD17P1; IQCN; GLI4; NUDT11; VIPR1-AS1; ID3; SUSD2; ZBTB10; RBM20; SLC6A17; AC026979.4; KLRG2; AL133230.2; AZGP1P1; HGD; C7orf31; COLCA2; CSF1; BDKRB2; LRRC37A3; POP1; NOL6; ID1; DTX2; SKAP1; MGAT5B; GADD45B; LINC00641; IFNLR1; IFI6; POTEKP; AC132192.2; FRAT1; AL135999.1; MFSD2A; VAMP5; CARMIL1; THTPA; AC020904.2; SILC1; RHPN1-AS1; SLC20A1; AC007191.1; FZD7; PAK6; SETP14; IZUMO4; HERC6; GRIK3; CEMIP2; AC234917.3; MYB; SESN3; RRS1; AP000350.6; CLHC1; DBP; SLC52A3; ALPK1; ELF3-AS1; LIF; LINC00265; NFKBIZ; HSPBAP1; C15orf48; CFAP45; HES1; UROD; TSFM; TSPAN5; EDN2; MCOLN3; RASD2; BX470102.2; MIR1915HG; AADAT; SPOCD1; FANCE; PXMP4; NKILA; KLF8; PTCHD1; HMGCS2; AHR; EVL; MRPL12; AF111167.2; RNU4-2; PCDHA12; LINC02615; MMEL1; GATA3-AS1; SERPINE1; RPSAP47; AREG; TMEM254-AS1; BMF; FAM89B; SALL2; TAPT1-AS1; HOOK1; FOXA3; LRRC15; AC007114.1; TSKU; MSX1; KSR1; GRB7; CLIP2; AC092910.3; STC1; PPARD; STC2; MTHFS; AL359715.1; TMEM216; AP000350.7; C9orf106; AL451165.2; KCNK1; BHLHE41; FKRP; AL121906.2; DDT; MAP2K6; GPR157; AL158163.1; CDC42EP1; NCOR2; C15orf62; TNFRSF11A; ANKRD62P1; DNMBP; SEMA7A; AL157838.1; ANG; ZCCHC4; ATG101; ZCCHC17; FEM1A; SUFU; MAP1S; EPHA2; MAD2L1BP; SEMA4G; ITGAV; PALM2AKAP2; SDHAF2; LASP1; WDR60; ZNF688; FAM81A; PITX1; Z99572.1; POU2F1; CHST15; RHOB; BICRA; HOXB7; AP1AR; FKBP9P1; ZNF594; XG; CLN3; RABEP1; SYNE3; AC005332.3; ZNF764; GLCE; LRRC49; KIAA1211L; SRCAP; ABLIM3; ETHE1; KLHL8; AC139530.1; ZNF319; SRRM3; EMSLR; AL691482.4; ASAP1; CENPQ; RNF125; ELK3; TMEM143; ZDHHC18; ZBTB9; KLF3; SUSD3; ACSS1; FLT4; RIPOR3; TSC22D1; LINC01504; RFX1; TIMP2; HS6ST1; DAB2IP; AC009078.3; SIX5; AC131971.1; CUEDC1; MLXIP; CHST3; FOXD1; TUBD1; MED21; ARID5B; PARP8; CEP126; WNT3; B3GLCT; MYPOP; LINC02482; RPS26P6; PROM1; AC108134.2; LGALS1; ZNF114; RGS9BP; RTN4IP1; A4GALT; BCL9L; MAFF; GIHCG; SIX1; CALML5; SESTD1; TRIM62; DHRS2; ANKRD18B; MYC; ZC3H12A; GSDMB; RAB19; NHSL1; AL360270.1; KCNG1; KIF13B; AL009179.1; FBXO4; ZNF786; TOB1-AS1; EIF4A3; KIAA1522; TNFRSF14; SLC15A1; AC005914.1; CCDC74A; NCALD; SH3TC1; HS6ST1P1; H2BC10; ZSCAN21; PSD4; EIF2AK4; NCEH1; HSPA12A; SCARA3; MINCR; AL390719.2; BTN2A2; ZNF19; AC007114.2; FAM241A; CACNA1H; SECTM1; MRM1; ITPRIPL2; C9orf152; LSM10; DLGAP3; ABTB2; NKAIN1; VAV1; CCNB3; RPL14P1"},"children":["AQP3; FKBP5; ZNF689; ANKRD22; OLR1; EHF; KRT4; ..."]},{"name":"span","attribs":{"title":"AQP3; UGDH; FKBP5; SEC14L2; HSD11B2; AC021148.3; PISD; ZNF652; ZNF689; ANKRD22; ARHGEF26; EHF; KRT4; PARM1; ARHGEF37; THBS1; NRP1; GREB1; TTC12; KDM4B; PTPRJ; ADD3; SYBU; NEDD4L; CSRNP1; GPRC5A; WWC1; CNTNAP2; IL1R1; TBC1D16; OSGIN1; SLC26A2; AL035661.1; SERHL; PER1; AKAP13; SLC22A23; KRT15; CD9; TRGC1; CEACAM7; VSTM2L; IQGAP2; SOX9; CABLES1; PRKAB2; PMEPA1; NDRG1; SRGAP3; TMEM164; BCL2; CEACAM5; PPFIBP2; MCCC2; GRHL2; HEY2; LNX2; TRIM9; BLNK; FAM214A; AC092053.4; CYP2B7P; ECI2; AL135905.2; ESR1; VAMP1; ASB13; MTSS1; PREX1; AL606500.1; POU2F3; GCNT2; ENC1; KLF9; BMP7; PCED1B; RNF152; TPD52; EMP1; TBC1D8; EPN2; BCL2L1; COLCA1; AC005920.1; STARD13-AS; ADGRG1; STARD13; MYCBP2; IFITM10; AL445309.1; CRYBG2; LRP2BP; YPEL3; FSBP; TTC21A; CACNB4; ZBTB7C; ANK3; MUC19; SERPINB2; PAQR5; TRGC2; MTMR9; MYH11; CLEC3A; MYRFL; CRAT; NAMPT; GASK1B; PXK; ZNF703; CHN1; CBX4; MUC5B; DENND2B; TRMT9B; IKZF2; ARHGEF26-AS1; TESK2; CEACAM6; TOE1; RNF144B; RASSF3; SOCS3; BX842570.1; RBBP8NL; NYNRIN; IGSF9; BCL6; TRIM66; FAH; HERC3; AC022007.1; KCNB1; GARNL3; SCIN; GDPD3; AC008268.1; CREB3L1; LIPH; AC022400.8; CISH; CMAHP; APCDD1; DLG2; SLC2A1; ADIRF-AS1; OAZ3; PXYLP1; BNIPL; RORC; FZD5; NSRP1; SLC22A5; RN7SL674P; HEG1; SPRY4; MYO1F; RNF227; VAV3; ARHGAP8; SEPHS2; NR1H3; SLC29A3; ARL10; CASD1; GET4; PCDH8; ATP7B; NRCAM; PLEKHH1; DTX4; CACNA1D; AL157392.3; MRFAP1; LBP; RFLNA; PARTICL; AL158211.5; SORD2P; PRR15L; PRAG1; CORO1A; KCTD6; CD160; AC027702.2; FERMT1; TRERF1; DBI; KLF3-AS1; LRRC46; RAD51B; ACACB; DOK7; PIM1; GRK5; RPH3AL; BAG3; SHANK2; ZMIZ1; PPP1R3D; PROSER3; ARID5A; HOXC4; OSR2; TSC22D3; SLC25A35; SMIM5; FAM198B-AS1; AL137782.1; EGLN3; CMTM8; TMEM63C; RAB30; SNORA37; SPRED2; GSDME; ITGAM; GSN-AS1; PDE11A; AC118344.4; AC105339.2; ULK1; CNOT4; SLC4A8; IQCN; MRPL38; IQCH-AS1; IRX6; ARRB1; KRT81; ZNF385A; ID3; BDNF-AS; VCPKMT; NPRL3; RBM20; TCF7L2; AC026979.4; ST6GAL1; PSMD6-AS2; LINC01176; HGD; NEDD4; ICOSLG; COLCA2; ZFX; AL353593.1; DRC3; SYT15; LOXL3; MAST3; LRRC37A3; PSCA; NOL6; MMP25; ID1; SKAP1; INPP5J; GADD45B; ENPP1; BCYRN1; AIFM2; PIFO; IFNLR1; CNOT2-DT; AL135905.1; FGD6; BTN2A3P; RNU4-1; CTSO; GUCY1B1; KL; CCDC96; ANKRD61; ENTPD1-AS1; AP005131.6; MFSD2A; SAMD10; VAMP5; RPS10P3; GRHL3; PDPR; CARMIL1; MAGIX; AC020904.2; GRAMD1B; DRAIC; HAP1; LMNTD2-AS1; NFATC2; RBM26-AS1; RNY3; RABGEF1; TMEM62; LRRC26; TLK2P1; TDO2; SMIM14; ABCC3; RAVER1; TGFB1I1; FP565260.3; CHRM1; IZUMO4; HERC6; GRIK3; GBP2; DPF1; AC007637.1; IGFBP3; AC234917.3; MYB; AMH; AC093904.4; LINC00511; ADAMTSL4; IL17RE; UBXN10; AC232271.1; ARHGEF3; RAB11B-AS1; ZFP36L2; LIF; LINC00265; C15orf48; SERTAD1; CFAP45; CATSPERB; HCAR1; HES1; DNASE1L2; HUS1; MGAT2; AC007405.4; BACE1-AS; PRDM4; RASD2; ATP1A1; TMEM160; BPIFA4P; PXMP4; FRMD4A; AC027097.2; KLF8; EFNA4; HMGCS2; HEBP1; SNHG22; LINC01522; AHR; EVL; BCCIP; AC010184.1; BCAR3; LINC01089; MIR3936HG; TMEM44; AC090772.5; NAV1; EPN3; TRIM45; AC106782.1; PAM16; SAMD15; TMEM254-AS1; GPR162; IQCD; SIDT1; AL928654.1; ADPRM; NRM; FAM89B; BAHCC1; AC013394.1; HOOK1; ZNF117; FOXA3; AC018809.2; LRRC15; AC020978.9; TSKU; EFNA3; CDKN2B-AS1; FCGRT; KSR1; AC141930.1; AC092910.3; KALRN; AC121338.2; IRF5; FO681492.1; CSTA; ZSCAN12; SHROOM2; RNF215; MPC2; ZFPL1; MS4A7; CSRNP2; FKRP; AC005041.1; SPRYD7; KDM6B; RILP; RAP1GAP; FMO5; SRRM2; AL158163.1; CDC42EP1; CRELD1; NCOR2; SELENON; GXYLT2; NNAT; DICER1-AS1; C15orf62; EEFSEC; UBAP1L; MAN1C1; DNMT3B; ABTB1; DNMBP; MLKL; AC008895.1; THAP7-AS1; AL157838.1; AL139385.1; ANG; ATG101; MTMR11; MLLT11; FEM1A; CCDC157; HTR7P1; AC133555.3; SLC24A3; SEPTIN1; CCDC124; MPP7; EML5; HDAC10; AC084082.1; RASGEF1A; PALM2AKAP2; AC097376.3; PRELID2; SDHAF2; SLC27A5; RFLNB; SCAMP5; MAPK15; GSEC; WDR60; ZNF688; ID2; FTX; PRR12; TRIM17; PLAAT4; B3GNTL1; AC104109.2; CYP4Z2P; C3orf18; NFIX; AL049834.1; PLEKHG5; AC009126.1; EEPD1; PSG4; BICRA; KCNMA1; ACOX2; POM121C; BAALC-AS1; VPS37D; KIF12; CRYBB2P1; TMC8; DHCR24; HDAC9; AL158212.3; ALPK3; LIN7B; CLN3; TNS2; DIPK1B; OLMALINC; GPKOW; AC097523.1; SNORD3C; SYNE3; ZNF764; SNHG19; RFX2; LRRC49; NHEJ1; AC087289.5; FFAR2; LARGE1; SRCAP; ETHE1; KLHL8; UBALD2; ZC3H10; ZSCAN31; HOXC10; CAP2; C1QTNF6; TMEM8B; LIN37; RILPL2; HINT2; KLF3; CYSRT1; PPP1R37; TSC22D1; TAGLN; MR1; TIMP2; IGSF8; ZFPM1; HS6ST1; DLEU1; AC131971.1; TTC13; CUEDC1; MLXIP; CHST3; LTBP4; TMEM107; PLAAT3; OSBPL6; MBD3; DOCK3; UQCRHL; USP3-AS1; DGLUCY; UCKL1-AS1; RPL13P5; UNC5C; ARID5B; AL031666.1; CEP126; NDST2; MGAM; LINC02482; RDM1; CASP9; RBL2; ACYP2; ENTPD8; AC018521.5; PROM1; STRADB; AC108134.2; LGALS1; PPARG; CRISPLD2; PDP2; RGS9BP; ABCC12; BCL9L; MAFF; SIX1; CALML5; SESTD1; NR4A1; DHRS2; EFNA5; ANKRD18B; TMEM186; MYC; ZC3H12A; GSDMB; NDFIP2; MRC2; TPO; ANAPC11; CST6; CRTC3; KIF13B; KCNC3; ZNF786; TOB1-AS1; KIAA1522; TLE1; PLXNB3; KTN1-AS1; AC005914.1; NCALD; WDR66; BCL2L12; IQCG; SH3TC1; AC139149.1; GAD1; AL731571.1; TMC5"},"children":["AQP3; UGDH; FKBP5; SEC14L2; HSD11B2; AC021148.3..."]},{"name":"span","attribs":{"title":"FKBP5; SEC14L2; ZNF689; TTC12; SYBU; CSRNP1; AC092053.4; TTC21A; MTMR9; CRAT; PEX10; CHPT1"},"children":["FKBP5; SEC14L2; ZNF689; TTC12; SYBU; CSRNP1; AC..."]},{"name":"span","attribs":{"title":"FKBP5; HSD11B2; PTPRJ; CNTNAP2; MCCC2; MYCBP2"},"children":["FKBP5; HSD11B2; PTPRJ; CNTNAP2; MCCC2; MYCBP2"]},{"name":"span","attribs":{"title":"AQP3; EHF; WWC1; CD9; MCCC2; NAMPT; VIPR1; VIPR1-AS1; DUSP16; NOL6"},"children":["AQP3; EHF; WWC1; CD9; MCCC2; NAMPT; VIPR1; VIPR..."]}],"minWidth":150}],"filterable":true,"dataKey":"3d6d1f679e5418174ae3f73b26821c40"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<pre class="r"><code>plot_grid(plotlist = p, nrow = ceiling(length(p) / 3))</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/barcode-gsea-reg-nondir-sig-1.png" alt="*Barcode plots for the top 8 sets of windows associated with __non-directional__ changes in gene expression.*" width="960" />
<p class="caption">
<em>Barcode plots for the top 8 sets of windows associated with
<strong>non-directional</strong> changes in gene expression.</em>
</p>
</div>
</div>
<div id="windows-by-feature-1" class="section level4">
<h4>Windows By Feature</h4>
<pre class="r"><code>gsea_feat_nondir &lt;- fgsea(feat_dir_gs, rna_sig_ranks, scoreType = &quot;pos&quot;)
gsea_feat_nondir_sig &lt;- gsea_feat_nondir %&gt;% 
  arrange(pval) %&gt;% 
  dplyr::filter(padj &lt; enrich_alpha, size &gt;= min_sig) %&gt;% 
  as_tibble() 
p &lt;- gsea_feat_nondir_sig %&gt;% 
  dplyr::slice(1:9) %&gt;% 
  pull(&quot;pathway&quot;) %&gt;% 
  lapply(
    function(x) {
      plotEnrichment(feat_dir_gs[[x]], rna_sig_ranks) +
        ggtitle(x) +
        theme(
          plot.title = element_text(hjust = 0.5, size = 10)
        )
    }
  )</code></pre>
<pre class="r"><code>cp &lt;-  htmltools::em(
  glue(
    &quot;Merged windows were mapped to genes and their position amongst the &quot;,
    &quot;RNA-Seq results was assessed. Windows were classified based on which &quot;,
    &quot;feature was the best overlap, and the direction of {target} binding with &quot;,
    &quot;{treat_levels[[2]]} treatment. {nrow(gsea_feat_nondir_sig)} sets of windows were &quot;,
    &quot;associated with overall changes in gene expression, using only the &quot;, 
    &quot;test statistic to rank the &quot;, 
    &quot;{comma(nrow(rnaseq), 1)} genes considered as detected.&quot;
  )
)
tbl &lt;- gsea_feat_nondir_sig %&gt;%
  mutate(
    `Edge Size` = vapply(leadingEdge, length, integer(1)),
    leadingEdge = lapply(leadingEdge, function(x) id2gene[x]) %&gt;% 
      vapply(paste, character(1), collapse = &quot;; &quot;),
    Direction = ifelse(NES &gt; 0, &quot;\u21E7 Up-regulated&quot;, &quot;\u21E9 Down-regulated&quot;),
    &quot;{target}&quot; := str_extract(pathway, &quot;(Up|Down|Unchanged)$&quot;),
    pathway = str_remove_all(pathway, &quot; / [^/]+$&quot;)
  ) %&gt;% 
  dplyr::select(
    pathway, !!sym(target), Windows = size, Direction, 
    p = pval, FDR = padj, `Edge Size`, `Leading Edge` = leadingEdge
  ) %&gt;% 
  reactable(
    filterable = TRUE,
    columns = list2(
      pathway = colDef(
        minWidth = 100, name = &quot;Region&quot;
      ),
      &quot;{target}&quot; := colDef(
        maxWidth = 90,
        cell = function(value) {
          html_symbol &lt;- &quot;&quot;
          if (str_detect(value, &quot;Up&quot;)) html_symbol &lt;- &quot;\u21E7&quot;
          if (str_detect(value, &quot;Down&quot;)) html_symbol &lt;- &quot;\u21E9&quot;
          paste(html_symbol, value)
        },
        style = function(value) {
          colour &lt;- case_when(
            str_detect(value, &quot;Up&quot;) ~ colours$direction[[&quot;up&quot;]],
            str_detect(value, &quot;Down&quot;) ~ colours$direction[[&quot;down&quot;]],
            TRUE ~ colours$direction[[&quot;unchanged&quot;]]
          )
          list(color = colour)
        }
      ),
      Windows = colDef(maxWidth = 80, format = colFormat(separators = TRUE)),
      Direction = colDef(
        name = &quot;Gene Direction&quot;,
        maxWidth = 120,
        style = function(value) {
          colour &lt;- ifelse(
            str_detect(value, &quot;Up&quot;), 
            colours$direction[[&quot;up&quot;]], 
            colours$direction[[&quot;down&quot;]]
          )
          list(color = colour)
        },
      ),
      p = colDef(
        cell = function(value) sprintf(&quot;%.2e&quot;, value), maxWidth = 80
      ),
      FDR = colDef(
        cell = function(value) sprintf(&quot;%.2e&quot;, value), maxWidth = 80
      ),
      &quot;Edge Size&quot; = colDef(maxWidth = 80),
     &quot;Leading Edge&quot; = colDef(
       minWidth = 150,
       cell = function(value) with_tooltip(value, width = 50)
     ) 
    )
  )
div(class = &quot;table&quot;,
  div(class = &quot;table-header&quot;,
      div(class = &quot;caption&quot;, cp),
      tbl
  )
)</code></pre>
<div class="table">
<div class="table-header">
<div class="caption">
<em>Merged windows were mapped to genes and their position amongst the RNA-Seq results was assessed. Windows were classified based on which feature was the best overlap, and the direction of H3K27ac binding with E2DHT treatment. 2 sets of windows were associated with overall changes in gene expression, using only the test statistic to rank the 13,043 genes considered as detected.</em>
</div>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-1e448617003b12019db2" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-1e448617003b12019db2">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"pathway":["No Feature","Enhancer"],"H3K27ac":["Up","Up"],"Windows":[80,178],"Direction":["⇧ Up-regulated","⇧ Up-regulated"],"p":[1.57057754368517e-08,2.73237580595273e-07],"FDR":[6.28231017474068e-08,5.46475161190545e-07],"Edge Size":[25,40],"Leading Edge":["AQP3; FKBP5; SEC14L2; RANBP3L; ZNF689; EHF; ARHGEF37; NRP1; SYBU; NEDD4L; WWC1; AKAP13; MCCC2; FGFR2; TPD52; SGK3; SORD; MYCBP2; CCDC40; CNNM1; MTMR9; CRAT; RASSF3; ACACA; PEX10","AQP3; FKBP5; HSD11B2; ZNF652; EHF; ARHGEF37; AL033523.1; TTC12; KDM4B; PTPRJ; CSRNP1; SASH1; CNTNAP2; CD9; SPRED1; VSTM2L; IQGAP2; PRKAB2; MCCC2; AC092053.4; GSTM3; PEA15; TTC21A; TMEM17; NAMPT; ACACA; GIPR; PXDC1; DLG2; PRKAA2; VIPR1; CHPT1; VIPR1-AS1; DUSP16; ZFX; NOL6; CCDC96; AC007191.1; MAFB; TOR1AIP2"]},"columns":[{"id":"pathway","name":"Region","type":"character","minWidth":100},{"id":"H3K27ac","name":"H3K27ac","type":"character","cell":["⇧ Up","⇧ Up"],"maxWidth":90,"style":[{"color":"#CA0020"},{"color":"#CA0020"}]},{"id":"Windows","name":"Windows","type":"numeric","format":{"cell":{"separators":true},"aggregated":{"separators":true}},"maxWidth":80},{"id":"Direction","name":"Gene Direction","type":"character","maxWidth":120,"style":[{"color":"#CA0020"},{"color":"#CA0020"}]},{"id":"p","name":"p","type":"numeric","cell":["1.57e-08","2.73e-07"],"maxWidth":80},{"id":"FDR","name":"FDR","type":"numeric","cell":["6.28e-08","5.46e-07"],"maxWidth":80},{"id":"Edge Size","name":"Edge Size","type":"numeric","maxWidth":80},{"id":"Leading Edge","name":"Leading Edge","type":"character","cell":[{"name":"span","attribs":{"title":"AQP3; FKBP5; SEC14L2; RANBP3L; ZNF689; EHF; ARHGEF37; NRP1; SYBU; NEDD4L; WWC1; AKAP13; MCCC2; FGFR2; TPD52; SGK3; SORD; MYCBP2; CCDC40; CNNM1; MTMR9; CRAT; RASSF3; ACACA; PEX10"},"children":["AQP3; FKBP5; SEC14L2; RANBP3L; ZNF689; EHF; ARH..."]},{"name":"span","attribs":{"title":"AQP3; FKBP5; HSD11B2; ZNF652; EHF; ARHGEF37; AL033523.1; TTC12; KDM4B; PTPRJ; CSRNP1; SASH1; CNTNAP2; CD9; SPRED1; VSTM2L; IQGAP2; PRKAB2; MCCC2; AC092053.4; GSTM3; PEA15; TTC21A; TMEM17; NAMPT; ACACA; GIPR; PXDC1; DLG2; PRKAA2; VIPR1; CHPT1; VIPR1-AS1; DUSP16; ZFX; NOL6; CCDC96; AC007191.1; MAFB; TOR1AIP2"},"children":["AQP3; FKBP5; HSD11B2; ZNF652; EHF; ARHGEF37; AL..."]}],"minWidth":150}],"filterable":true,"dataKey":"5fd21aaabe8e0a832f3ac393418d81a7"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<pre class="r"><code>plot_grid(plotlist = p, nrow = ceiling(length(p) / 3))</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/barcode-gsea-feat-nondir-sig-1.png" alt="*Barcode plots for the top 2 sets of windows associated with __nondirectional__ changes in gene expression.*" width="960" />
<p class="caption">
<em>Barcode plots for the top 2 sets of windows associated with
<strong>nondirectional</strong> changes in gene expression.</em>
</p>
</div>
<!-- Could be done regardless of changes in expression -->
</div>
</div>
</div>
<div id="integration-of-rna-and-chip-enrichment-analyses"
class="section level2">
<h2>Integration of RNA and ChIP Enrichment Analyses</h2>
<pre class="r"><code>rnaseq_gsea &lt;- fgsea(
  pathways = gs_by_gsid,
  stats = rna_dir_ranks, 
  minSize = min_gs_size, 
  maxSize = max_gs_size
) %&gt;% 
  dplyr::filter(!is.na(pval))
status_cols &lt;- setNames(
  hcl.colors(4, &quot;Viridis&quot;, rev = TRUE),
  c(&quot;Both&quot;, &quot;ChIP Only&quot;, &quot;RNA Only&quot;, &quot;Neither&quot;)
)</code></pre>
<p>The enrichment testing results from the H3K27ac differential binding
analysis were then compared to GSEA results obtained from the RNA-Seq
data set. Given the orthogonal nature of the two dataset, p-values from
each analysis were combined using Wilkinson’s maximum p-value method and
the resulting p-values adjusted as previously. Distances between nodes
(gene-sets) for all network plots were determined by using ChIP target
genes within the leading edge only.</p>
<div id="differentially-bound-windows" class="section level3 tabset">
<h3 class="tabset">Differentially Bound Windows</h3>
<pre class="r"><code>cmn_diff &lt;- rnaseq_gsea %&gt;%
  dplyr::select(-log2err, -ES, -size) %&gt;% 
  left_join(
    goseq_diff, by = c(&quot;pathway&quot; = &quot;gs_name&quot;), suffix = c(&quot;.rna&quot;, &quot;.chip&quot;)
  ) %&gt;%
  dplyr::filter(!is.na(pval.rna), !is.na(pval.chip)) %&gt;% 
  nest(p = starts_with(&quot;pval&quot;)) %&gt;% 
  mutate(
    pval = vapply(p, function(x) metap::maximump(unlist(x))$p, numeric(1))
  ) %&gt;%
  unnest(p) %&gt;% 
  dplyr::rename(padj_chip = adj_p, padj_rna = padj) %&gt;% 
  mutate(adj_p = p.adjust(pval, params$enrichment$adj)) %&gt;% 
  arrange(pval) %&gt;% 
  dplyr::filter(adj_p &lt; enrich_alpha) %&gt;% 
  mutate(
    leadingEdge = lapply(
      leadingEdge, intersect, dplyr::filter(mapped_ids, up | down)$gene_id
    ),
    numDEInCat = vapply(leadingEdge, length, integer(1)),
    Status = case_when(
      padj_rna &lt; enrich_alpha &amp; padj_chip &lt; enrich_alpha ~ &quot;Both&quot;,
      padj_rna &lt; enrich_alpha &amp; !padj_chip &lt; enrich_alpha ~ &quot;RNA Only&quot;,
      !padj_rna &lt; enrich_alpha &amp; padj_chip &lt; enrich_alpha ~ &quot;ChIP Only&quot;,
      !padj_rna &lt; enrich_alpha &amp; !padj_chip &lt; enrich_alpha ~ &quot;Neither&quot;
    ) %&gt;% 
      factor(levels = names(status_cols))
  ) %&gt;% 
  dplyr::filter(numDEInCat &gt;= min_sig) %&gt;% 
  dplyr::select(
    gs_name = pathway, adj_p, Status, NES, starts_with(&quot;padj&quot;),
    starts_with(&quot;num&quot;), leadingEdge
  ) 
tg_diff &lt;- cmn_diff %&gt;% 
  make_tbl_graph(setNames(.$leadingEdge, .$gs_name))
plot_net &lt;- length(tg_diff) &gt;= min_network_size
txt &lt;- ifelse(
  nrow(cmn_diff) == 0,
  glue(&quot;No gene sets were jointly enriched between all {target} differentially bound windows and the RNA-Seq data&quot;),
  &quot;#### Results Table&quot;
)</code></pre>
<div id="results-table-3" class="section level4">
<h4>Results Table</h4>
<pre class="r"><code>cp &lt;- htmltools::tags$em(
  glue(
    &quot;
    Genes mapped to all windows showing evidence of differential binding were 
    tested for enrichment using goseq, and results were compared to GSEA results
    for the RNA-Seq dataset. Wilkinson&#39;s method was used to integrate the two
    p-values with a combined FDR-adjusted p-value &lt; {enrich_alpha} being taken
    as evidence of potential regulatory targets from differentially bound 
    {target} binding sites. Of the {nrow(cmn_diff)} gene-sets of interest, 
    {sum(cmn_diff$Status == &#39;Both&#39;)} were individually enriched in both datasets,
    as indicated in the &#39;Status&#39; column
    &quot;
  )
)
tbl &lt;- cmn_diff %&gt;%
  mutate(
    leadingEdge = vapply(
      leadingEdge,
      function(x) paste(id2gene[x], collapse = &quot;; &quot;),
      character(1)
    )
  ) %&gt;% 
  reactable(
    filterable = TRUE, showPageSizeOptions = TRUE,
    columns = list(
      gs_name = colDef(
        name = &quot;Gene Set&quot;,
        cell = function(value) htmltools::tags$a(
          href = gs_url[[value]], 
          target = &quot;_blank&quot;, 
          str_replace_all(value, &quot;_&quot;, &quot; &quot;)
        ),
        minWidth = 120
      ),
      adj_p = colDef(
        name = glue(&quot;p&lt;sub&gt;{params$enrichment$adj}&lt;/sub&gt;&quot;), html = TRUE,
        cell = function(value) ifelse(
          value &lt; 0.001,
          sprintf(&quot;%.2e&quot;, value),
          sprintf(&quot;%.3f&quot;, value)
        ),
        maxWidth = 80
      ),
      Status = colDef(name = &quot;Prior Status&quot;, maxWidth = 100),
      NES = colDef(
        format = colFormat(digits = 2),
        style = function(value) {
          col &lt;- ifelse(value &gt; 0, colours$direction[&quot;up&quot;], colours$direction[&quot;down&quot;])
          list(color = col)
        },
        maxWidth = 80
      ),
      padj_rna = colDef(
        name =  glue(&quot;p&lt;sub&gt;{params$enrichment$adj}&lt;/sub&gt; (RNA)&quot;), html = TRUE,
        cell = function(value) ifelse(
          value &lt; 0.001,
          sprintf(&quot;%.2e&quot;, value),
          sprintf(&quot;%.3f&quot;, value)
        ),
        maxWidth = 80
      ),
      padj_chip = colDef(
        name =  glue(&quot;p&lt;sub&gt;{params$enrichment$adj}&lt;/sub&gt; (ChIP)&quot;), html = TRUE,
        cell = function(value) ifelse(
          value &lt; 0.001,
          sprintf(&quot;%.2e&quot;, value),
          sprintf(&quot;%.3f&quot;, value)
        ),
        maxWidth = 80
      ),
      numDEInCat = colDef(name = &quot;ChIP Targets in Leading Edge&quot;, maxWidth = 100),
      numInCat = colDef(name = &quot;Mapped Genes&quot;, maxWidth = 100),
      leadingEdge = colDef(
        name = &quot;Leading Edge ChIP Targets&quot;,
        cell = function(value) with_tooltip(value, width = 50),
        minWidth = 120
      )
    )
  )
div(
  class = &quot;table&quot;, 
  div(
    class = &quot;table-header&quot;,
    div(class = &quot;caption&quot;, cp)
  ),
  tbl
)</code></pre>
<div class="table">
<div class="table-header">
<div class="caption">
<em>Genes mapped to all windows showing evidence of differential binding were 
tested for enrichment using goseq, and results were compared to GSEA results
for the RNA-Seq dataset. Wilkinson's method was used to integrate the two
p-values with a combined FDR-adjusted p-value &lt; 0.05 being taken
as evidence of potential regulatory targets from differentially bound 
H3K27ac binding sites. Of the 14 gene-sets of interest, 
0 were individually enriched in both datasets,
as indicated in the 'Status' column</em>
</div>
</div>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-76c226e54bcac84146d6" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-76c226e54bcac84146d6">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"gs_name":["HES2_TARGET_GENES","HALLMARK_ANDROGEN_RESPONSE","HALLMARK_ESTROGEN_RESPONSE_EARLY","ZBTB18_TARGET_GENES","REACTOME_FATTY_ACID_METABOLISM","HALLMARK_ESTROGEN_RESPONSE_LATE","ZNF510_TARGET_GENES","WP_STEROL_REGULATORY_ELEMENTBINDING_PROTEINS_SREBP_SIGNALING","HALLMARK_MTORC1_SIGNALING","REACTOME_METABOLISM_OF_WATER_SOLUBLE_VITAMINS_AND_COFACTORS","REACTOME_BIOLOGICAL_OXIDATIONS","WP_VEGFAVEGFR2_SIGNALING_PATHWAY","REACTOME_TRANSPORT_OF_SMALL_MOLECULES","REACTOME_MAPK_FAMILY_SIGNALING_CASCADES"],"adj_p":[1.81519578832483e-05,0.000604443288387096,0.00253105906166235,0.002768818232134,0.00331186552457219,0.00684508000089431,0.0162178292711163,0.0296720516756859,0.0358492851179593,0.0383216935562989,0.0394658991777948,0.0403377903448983,0.0423348947936844,0.0442717604129584],"Status":["ChIP Only","Neither","ChIP Only","Neither","Neither","ChIP Only","Neither","Neither","Neither","Neither","Neither","Neither","Neither","Neither"],"NES":[1.35810716465367,1.64068667797514,1.46634642436493,1.32663860909647,1.60138825064716,1.38844809578348,1.4751600837208,1.40966132559505,1.47104535363106,1.38352487133649,1.4675910376099,1.26698316743663,1.46886664874965,1.2888069072452],"padj_rna":[0.0725206809518914,0.0630045895037077,0.215137991939811,0.216790927210254,0.0725206809518914,0.31295139090779,0.341435035259147,0.444020932311352,0.172739201145105,0.488252172990311,0.290679472081219,0.47341927562922,0.0725206809518914,0.515553582155894],"padj_chip":[0.00645695585531152,0.244454088275486,0.00235275164438038,0.0702290561348864,0.382661071306773,0.00874822065523922,0.542443993083426,0.382661071306773,0.65281262415547,0.65281262415547,0.658160684395007,0.67503838689422,0.681147309077167,0.65281262415547],"numDEInCat":[9,4,7,6,4,4,3,3,4,3,3,3,4,6],"numInCat":[877,87,174,615,110,163,30,63,190,91,95,343,425,224],"leadingEdge":["AQP3; EHF; NRP1; SYBU; NEDD4L; IQGAP2; MCCC2; SGK3; MTMR9","FKBP5; IQGAP2; TPD52; SORD","AQP3; FKBP5; SEC14L2; KDM4B; SYBU; WWC1; CHPT1","KDM4B; CD9; IQGAP2; MCCC2; SGK3; TTC21A","PRKAB2; CRAT; ACACA; PRKAA2","FKBP5; CD9; SORD; CHPT1","RANBP3L; NOL6; MED21","PRKAB2; ACACA; PRKAA2","CD9; SORD; NAMPT; ACACA","MCCC2; NAMPT; ACACA","GSTM3; SULT2B1; GSTM4","PTPRJ; ACACA; PRKAA2","AQP3; NEDD4L; SGK3; SLC66A1","SPRED1; FGFR2; PEA15; DUSP16; RASGEF1A; GFRA1"]},"columns":[{"id":"gs_name","name":"Gene Set","type":"character","cell":[{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["HES2 TARGET GENES"]},{"name":"a","attribs":{"href":null,"target":"_blank"},"children":["HALLMARK ANDROGEN RESPONSE"]},{"name":"a","attribs":{"href":null,"target":"_blank"},"children":["HALLMARK ESTROGEN RESPONSE EARLY"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["ZBTB18 TARGET GENES"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-8978868","target":"_blank"},"children":["REACTOME FATTY ACID METABOLISM"]},{"name":"a","attribs":{"href":null,"target":"_blank"},"children":["HALLMARK ESTROGEN RESPONSE LATE"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["ZNF510 TARGET GENES"]},{"name":"a","attribs":{"href":"http://www.wikipathways.org/instance/WP1982_r116442","target":"_blank"},"children":["WP STEROL REGULATORY ELEMENTBINDING PROTEINS SREBP SIGNALING"]},{"name":"a","attribs":{"href":null,"target":"_blank"},"children":["HALLMARK MTORC1 SIGNALING"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-196849","target":"_blank"},"children":["REACTOME METABOLISM OF WATER SOLUBLE VITAMINS AND COFACTORS"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-211859","target":"_blank"},"children":["REACTOME BIOLOGICAL OXIDATIONS"]},{"name":"a","attribs":{"href":"http://www.wikipathways.org/instance/WP3888_r116782","target":"_blank"},"children":["WP VEGFAVEGFR2 SIGNALING PATHWAY"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-382551","target":"_blank"},"children":["REACTOME TRANSPORT OF SMALL MOLECULES"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-5683057","target":"_blank"},"children":["REACTOME MAPK FAMILY SIGNALING CASCADES"]}],"minWidth":120},{"id":"adj_p","name":"p<sub>fdr<\/sub>","type":"numeric","cell":["1.82e-05","6.04e-04","0.003","0.003","0.003","0.007","0.016","0.030","0.036","0.038","0.039","0.040","0.042","0.044"],"html":true,"maxWidth":80},{"id":"Status","name":"Prior Status","type":"factor","maxWidth":100},{"id":"NES","name":"NES","type":"numeric","format":{"cell":{"digits":2},"aggregated":{"digits":2}},"maxWidth":80,"style":[{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"}]},{"id":"padj_rna","name":"p<sub>fdr<\/sub> (RNA)","type":"numeric","cell":["0.073","0.063","0.215","0.217","0.073","0.313","0.341","0.444","0.173","0.488","0.291","0.473","0.073","0.516"],"html":true,"maxWidth":80},{"id":"padj_chip","name":"p<sub>fdr<\/sub> (ChIP)","type":"numeric","cell":["0.006","0.244","0.002","0.070","0.383","0.009","0.542","0.383","0.653","0.653","0.658","0.675","0.681","0.653"],"html":true,"maxWidth":80},{"id":"numDEInCat","name":"ChIP Targets in Leading Edge","type":"numeric","maxWidth":100},{"id":"numInCat","name":"Mapped Genes","type":"numeric","maxWidth":100},{"id":"leadingEdge","name":"Leading Edge ChIP Targets","type":"character","cell":[{"name":"span","attribs":{"title":"AQP3; EHF; NRP1; SYBU; NEDD4L; IQGAP2; MCCC2; SGK3; MTMR9"},"children":["AQP3; EHF; NRP1; SYBU; NEDD4L; IQGAP2; MCCC2; S..."]},{"name":"span","attribs":{"title":"FKBP5; IQGAP2; TPD52; SORD"},"children":["FKBP5; IQGAP2; TPD52; SORD"]},{"name":"span","attribs":{"title":"AQP3; FKBP5; SEC14L2; KDM4B; SYBU; WWC1; CHPT1"},"children":["AQP3; FKBP5; SEC14L2; KDM4B; SYBU; WWC1; CHPT1"]},{"name":"span","attribs":{"title":"KDM4B; CD9; IQGAP2; MCCC2; SGK3; TTC21A"},"children":["KDM4B; CD9; IQGAP2; MCCC2; SGK3; TTC21A"]},{"name":"span","attribs":{"title":"PRKAB2; CRAT; ACACA; PRKAA2"},"children":["PRKAB2; CRAT; ACACA; PRKAA2"]},{"name":"span","attribs":{"title":"FKBP5; CD9; SORD; CHPT1"},"children":["FKBP5; CD9; SORD; CHPT1"]},{"name":"span","attribs":{"title":"RANBP3L; NOL6; MED21"},"children":["RANBP3L; NOL6; MED21"]},{"name":"span","attribs":{"title":"PRKAB2; ACACA; PRKAA2"},"children":["PRKAB2; ACACA; PRKAA2"]},{"name":"span","attribs":{"title":"CD9; SORD; NAMPT; ACACA"},"children":["CD9; SORD; NAMPT; ACACA"]},{"name":"span","attribs":{"title":"MCCC2; NAMPT; ACACA"},"children":["MCCC2; NAMPT; ACACA"]},{"name":"span","attribs":{"title":"GSTM3; SULT2B1; GSTM4"},"children":["GSTM3; SULT2B1; GSTM4"]},{"name":"span","attribs":{"title":"PTPRJ; ACACA; PRKAA2"},"children":["PTPRJ; ACACA; PRKAA2"]},{"name":"span","attribs":{"title":"AQP3; NEDD4L; SGK3; SLC66A1"},"children":["AQP3; NEDD4L; SGK3; SLC66A1"]},{"name":"span","attribs":{"title":"SPRED1; FGFR2; PEA15; DUSP16; RASGEF1A; GFRA1"},"children":["SPRED1; FGFR2; PEA15; DUSP16; RASGEF1A; GFRA1"]}],"minWidth":120}],"filterable":true,"showPageSizeOptions":true,"dataKey":"fc88d3ffd2a243e91ca0036d138e151b"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div id="network-plot" class="section level4">
<h4>Network Plot</h4>
<pre class="r"><code>tg_diff %&gt;% 
  activate(nodes) %&gt;% 
  mutate(direction = ifelse(NES &gt; 0, &quot;Up&quot;, &quot;Down&quot;))%&gt;% 
  ggraph(layout = net_layout, weights = oc^2) +
  geom_edge_link(aes(width = oc^2, alpha = oc^2)) +
  geom_node_point(
    aes(fill = Status, size = numDEInCat),
    shape = 21
  ) +
  geom_node_text(
    aes(label = label, colour = direction),
    size = 3, 
    data = . %&gt;%
      mutate(
        label = str_replace_all(label, &quot;_&quot;, &quot; &quot;) %&gt;% str_trunc(80) %&gt;% str_wrap(width = 18)
      ),
    repel = TRUE, max.overlaps = max(10, round(length(tg_diff) / 4, 0)),
    bg.color = &quot;white&quot;, bg.r = 0.1, 
  ) +
  scale_x_continuous(expand = expansion(c(0.1, 0.1))) +
  scale_y_continuous(expand = expansion(c(0.1, 0.1))) +
  scale_fill_manual(values = status_cols) +
  scale_size_continuous(range = c(1, 10)) +
  scale_edge_width(range = c(1, 6), limits = c(0, 1)) +
  scale_edge_alpha(range = c(0.1, 0.4), limits = c(0, 1))  +
  scale_colour_manual(
    values = setNames(direction_colours, str_to_title(names(direction_colours)))
  ) +
  guides(edge_alpha= &quot;none&quot;) +
  labs(
    size = glue(&quot;{target} Targets In Leading Edge&quot;) %&gt;% str_wrap(15),
    colour = &quot;GSEA\nDirection&quot;,
    fill = &quot;Prior\nSignificance&quot;,
    edge_width = expr(paste(OC^2))
  ) +
  theme_void() +
  theme(legend.text = element_text(hjust = 0.5))</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/network-cmn-alldiff-1.png" alt="*Network plot for pathways considered significant when assessing all differentially bound H3K27ac sites along with GSEA on the provided RNA-Seq data. Distances are derived from all detected genes within the pathway. Label colours indicate the direction of enrichment in GSEA results for genes in the pathway.*" width="960" />
<p class="caption">
<em>Network plot for pathways considered significant when assessing all
differentially bound H3K27ac sites along with GSEA on the provided
RNA-Seq data. Distances are derived from all detected genes within the
pathway. Label colours indicate the direction of enrichment in GSEA
results for genes in the pathway.</em>
</p>
</div>
</div>
</div>
<div id="windows-with-increased-h3k27ac" class="section level3 tabset">
<h3 class="tabset">Windows With Increased H3K27ac</h3>
<pre class="r"><code>cmn_up &lt;- rnaseq_gsea %&gt;%
  dplyr::select(-log2err, -ES, -size) %&gt;% 
  left_join(
    goseq_up, by = c(&quot;pathway&quot; = &quot;gs_name&quot;), suffix = c(&quot;.rna&quot;, &quot;.chip&quot;)
  ) %&gt;%
  dplyr::filter(!is.na(pval.rna), !is.na(pval.chip)) %&gt;% 
  nest(p = starts_with(&quot;pval&quot;)) %&gt;% 
  mutate(
    pval = vapply(p, function(x) metap::maximump(unlist(x))$p, numeric(1))
    ) %&gt;%
  unnest(p) %&gt;% 
  dplyr::rename(padj_chip = adj_p, padj_rna = padj) %&gt;% 
  mutate(adj_p = p.adjust(pval, params$enrichment$adj)) %&gt;% 
  arrange(pval) %&gt;% 
  dplyr::filter(adj_p &lt; enrich_alpha) %&gt;% 
  mutate(
    leadingEdge = lapply(
      leadingEdge, intersect, dplyr::filter(mapped_ids, up)$gene_id
    ),
    numDEInCat = vapply(leadingEdge, length, integer(1)),
    Status = case_when(
      padj_rna &lt; enrich_alpha &amp; padj_chip &lt; enrich_alpha ~ &quot;Both&quot;,
      padj_rna &lt; enrich_alpha &amp; !padj_chip &lt; enrich_alpha ~ &quot;RNA Only&quot;,
      !padj_rna &lt; enrich_alpha &amp; padj_chip &lt; enrich_alpha ~ &quot;ChIP Only&quot;,
      !padj_rna &lt; enrich_alpha &amp; !padj_chip &lt; enrich_alpha ~ &quot;Neither&quot;
    ) %&gt;% 
      factor(levels = names(status_cols))
  ) %&gt;% 
  dplyr::filter(numDEInCat &gt;= min_sig) %&gt;% 
  dplyr::select(
    gs_name = pathway, adj_p, Status, NES, starts_with(&quot;padj&quot;),
    starts_with(&quot;num&quot;), leadingEdge
  ) 
tg_up &lt;- cmn_up %&gt;% 
  make_tbl_graph(
    setNames(.$leadingEdge, .$gs_name)
  )
plot_net &lt;- length(tg_up) &gt;= min_network_size
txt &lt;- ifelse(
  nrow(cmn_up) == 0,
  glue(&quot;No gene sets were jointly enriched between all increasing {target} bound windows and the RNA-Seq data&quot;),
  &quot;#### Results Table&quot;
)</code></pre>
<div id="results-table-4" class="section level4">
<h4>Results Table</h4>
<pre class="r"><code>cp &lt;- htmltools::tags$em(
  glue(
    &quot;
    Genes mapped to all windows showing evidence of increased {target} binding were 
    tested for enrichment using goseq, and results were compared to GSEA results
    for the RNA-Seq dataset. Wilkinson&#39;s method was used to integrate the two
    p-values with a combined FDR-adjusted p-value &lt; {enrich_alpha} being taken
    as evidence of potential regulatory targets from increasingly bound 
    {target} binding sites. Of the {nrow(cmn_up)} gene-sets of interest, 
    {sum(cmn_up$Status == &#39;Both&#39;)} were individually enriched in both datasets,
    as indicated in the &#39;Status&#39; column
    &quot;
  )
)
tbl &lt;- cmn_up %&gt;%
  mutate(
    leadingEdge = vapply(
      leadingEdge,
      function(x) paste(id2gene[x], collapse = &quot;; &quot;),
      character(1)
    )
  ) %&gt;% 
  reactable(
    filterable = TRUE, showPageSizeOptions = TRUE,
    columns = list(
      gs_name = colDef(
        name = &quot;Gene Set&quot;,
        cell = function(value) htmltools::tags$a(
          href = gs_url[[value]], 
          target = &quot;_blank&quot;, 
          str_replace_all(value, &quot;_&quot;, &quot; &quot;)
        ),
        minWidth = 120
      ),
      adj_p = colDef(
        name = glue(&quot;p&lt;sub&gt;{params$enrichment$adj}&lt;/sub&gt;&quot;), html = TRUE,
        cell = function(value) ifelse(
          value &lt; 0.001,
          sprintf(&quot;%.2e&quot;, value),
          sprintf(&quot;%.3f&quot;, value)
        ),
        maxWidth = 80
      ),
      Status = colDef(name = &quot;Prior Status&quot;, maxWidth = 100),
      NES = colDef(
        format = colFormat(digits = 2),
        style = function(value) {
          col &lt;- ifelse(value &gt; 0, colours$direction[&quot;up&quot;], colours$direction[&quot;down&quot;])
          list(color = col)
        },
        maxWidth = 80
      ),
      padj_rna = colDef(
        name =  glue(&quot;p&lt;sub&gt;{params$enrichment$adj}&lt;/sub&gt; (RNA)&quot;), html = TRUE,
        cell = function(value) ifelse(
          value &lt; 0.001,
          sprintf(&quot;%.2e&quot;, value),
          sprintf(&quot;%.3f&quot;, value)
        ),
        maxWidth = 80
      ),
      padj_chip = colDef(
        name =  glue(&quot;p&lt;sub&gt;{params$enrichment$adj}&lt;/sub&gt; (ChIP)&quot;), html = TRUE,
        cell = function(value) ifelse(
          value &lt; 0.001,
          sprintf(&quot;%.2e&quot;, value),
          sprintf(&quot;%.3f&quot;, value)
        ),
        maxWidth = 80
      ),
      numDEInCat = colDef(name = &quot;ChIP Targets in Leading Edge&quot;, maxWidth = 100),
      numInCat = colDef(name = &quot;Mapped Genes&quot;, maxWidth = 100),
      leadingEdge = colDef(
        name = &quot;Leading Edge ChIP Targets&quot;,
        cell = function(value) with_tooltip(value, width = 50),
        minWidth = 120
      )
    )
  )
div(
  class = &quot;table&quot;, 
  div(
    class = &quot;table-header&quot;,
    div(class = &quot;caption&quot;, cp)
  ),
  tbl
)</code></pre>
<div class="table">
<div class="table-header">
<div class="caption">
<em>Genes mapped to all windows showing evidence of increased H3K27ac binding were 
tested for enrichment using goseq, and results were compared to GSEA results
for the RNA-Seq dataset. Wilkinson's method was used to integrate the two
p-values with a combined FDR-adjusted p-value &lt; 0.05 being taken
as evidence of potential regulatory targets from increasingly bound 
H3K27ac binding sites. Of the 14 gene-sets of interest, 
0 were individually enriched in both datasets,
as indicated in the 'Status' column</em>
</div>
</div>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-4b6f377ce36bbda748c4" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-4b6f377ce36bbda748c4">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"gs_name":["HES2_TARGET_GENES","HALLMARK_ANDROGEN_RESPONSE","HALLMARK_ESTROGEN_RESPONSE_EARLY","ZBTB18_TARGET_GENES","REACTOME_FATTY_ACID_METABOLISM","HALLMARK_ESTROGEN_RESPONSE_LATE","ZNF510_TARGET_GENES","WP_STEROL_REGULATORY_ELEMENTBINDING_PROTEINS_SREBP_SIGNALING","HALLMARK_MTORC1_SIGNALING","REACTOME_METABOLISM_OF_WATER_SOLUBLE_VITAMINS_AND_COFACTORS","REACTOME_BIOLOGICAL_OXIDATIONS","WP_VEGFAVEGFR2_SIGNALING_PATHWAY","REACTOME_TRANSPORT_OF_SMALL_MOLECULES","REACTOME_MAPK_FAMILY_SIGNALING_CASCADES"],"adj_p":[1.81040635352714e-05,0.000574136454430522,0.00252438080556297,0.002761512643131,0.00315624726914998,0.00682701910379433,0.0158171063371377,0.0295937613018714,0.0342615693386697,0.0381082175705559,0.0381082175705559,0.0381082175705559,0.0398022823493333,0.0441549483802065],"Status":["ChIP Only","Neither","ChIP Only","Neither","Neither","ChIP Only","Neither","Neither","Neither","Neither","Neither","Neither","Neither","Neither"],"NES":[1.35810716465367,1.64068667797514,1.46634642436493,1.32663860909647,1.60138825064716,1.38844809578348,1.4751600837208,1.40966132559505,1.47104535363106,1.38352487133649,1.4675910376099,1.26698316743663,1.46886664874965,1.2888069072452],"padj_rna":[0.0725206809518914,0.0630045895037077,0.215137991939811,0.216790927210254,0.0725206809518914,0.31295139090779,0.341435035259147,0.444020932311352,0.172739201145105,0.488252172990311,0.290679472081219,0.47341927562922,0.0725206809518914,0.515553582155894],"padj_chip":[0.00582041511232729,0.237472415814747,0.0022204021234785,0.0653035536882752,0.376511856182506,0.00832405412743398,0.531481046285402,0.376511856182506,0.642997197626098,0.641389432862096,0.645163533365343,0.6539850490281,0.672885929468437,0.641389432862096],"numDEInCat":[9,4,7,6,4,4,3,3,4,3,3,3,4,6],"numInCat":[877,87,174,615,110,163,30,63,190,91,95,343,425,224],"leadingEdge":["AQP3; EHF; NRP1; SYBU; NEDD4L; IQGAP2; MCCC2; SGK3; MTMR9","FKBP5; IQGAP2; TPD52; SORD","AQP3; FKBP5; SEC14L2; KDM4B; SYBU; WWC1; CHPT1","KDM4B; CD9; IQGAP2; MCCC2; SGK3; TTC21A","PRKAB2; CRAT; ACACA; PRKAA2","FKBP5; CD9; SORD; CHPT1","RANBP3L; NOL6; MED21","PRKAB2; ACACA; PRKAA2","CD9; SORD; NAMPT; ACACA","MCCC2; NAMPT; ACACA","GSTM3; SULT2B1; GSTM4","PTPRJ; ACACA; PRKAA2","AQP3; NEDD4L; SGK3; SLC66A1","SPRED1; FGFR2; PEA15; DUSP16; RASGEF1A; GFRA1"]},"columns":[{"id":"gs_name","name":"Gene Set","type":"character","cell":[{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["HES2 TARGET GENES"]},{"name":"a","attribs":{"href":null,"target":"_blank"},"children":["HALLMARK ANDROGEN RESPONSE"]},{"name":"a","attribs":{"href":null,"target":"_blank"},"children":["HALLMARK ESTROGEN RESPONSE EARLY"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["ZBTB18 TARGET GENES"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-8978868","target":"_blank"},"children":["REACTOME FATTY ACID METABOLISM"]},{"name":"a","attribs":{"href":null,"target":"_blank"},"children":["HALLMARK ESTROGEN RESPONSE LATE"]},{"name":"a","attribs":{"href":"http://gtrd.biouml.org/","target":"_blank"},"children":["ZNF510 TARGET GENES"]},{"name":"a","attribs":{"href":"http://www.wikipathways.org/instance/WP1982_r116442","target":"_blank"},"children":["WP STEROL REGULATORY ELEMENTBINDING PROTEINS SREBP SIGNALING"]},{"name":"a","attribs":{"href":null,"target":"_blank"},"children":["HALLMARK MTORC1 SIGNALING"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-196849","target":"_blank"},"children":["REACTOME METABOLISM OF WATER SOLUBLE VITAMINS AND COFACTORS"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-211859","target":"_blank"},"children":["REACTOME BIOLOGICAL OXIDATIONS"]},{"name":"a","attribs":{"href":"http://www.wikipathways.org/instance/WP3888_r116782","target":"_blank"},"children":["WP VEGFAVEGFR2 SIGNALING PATHWAY"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-382551","target":"_blank"},"children":["REACTOME TRANSPORT OF SMALL MOLECULES"]},{"name":"a","attribs":{"href":"https://www.reactome.org/content/detail/R-HSA-5683057","target":"_blank"},"children":["REACTOME MAPK FAMILY SIGNALING CASCADES"]}],"minWidth":120},{"id":"adj_p","name":"p<sub>fdr<\/sub>","type":"numeric","cell":["1.81e-05","5.74e-04","0.003","0.003","0.003","0.007","0.016","0.030","0.034","0.038","0.038","0.038","0.040","0.044"],"html":true,"maxWidth":80},{"id":"Status","name":"Prior Status","type":"factor","maxWidth":100},{"id":"NES","name":"NES","type":"numeric","format":{"cell":{"digits":2},"aggregated":{"digits":2}},"maxWidth":80,"style":[{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"},{"color":"#CA0020"}]},{"id":"padj_rna","name":"p<sub>fdr<\/sub> (RNA)","type":"numeric","cell":["0.073","0.063","0.215","0.217","0.073","0.313","0.341","0.444","0.173","0.488","0.291","0.473","0.073","0.516"],"html":true,"maxWidth":80},{"id":"padj_chip","name":"p<sub>fdr<\/sub> (ChIP)","type":"numeric","cell":["0.006","0.237","0.002","0.065","0.377","0.008","0.531","0.377","0.643","0.641","0.645","0.654","0.673","0.641"],"html":true,"maxWidth":80},{"id":"numDEInCat","name":"ChIP Targets in Leading Edge","type":"numeric","maxWidth":100},{"id":"numInCat","name":"Mapped Genes","type":"numeric","maxWidth":100},{"id":"leadingEdge","name":"Leading Edge ChIP Targets","type":"character","cell":[{"name":"span","attribs":{"title":"AQP3; EHF; NRP1; SYBU; NEDD4L; IQGAP2; MCCC2; SGK3; MTMR9"},"children":["AQP3; EHF; NRP1; SYBU; NEDD4L; IQGAP2; MCCC2; S..."]},{"name":"span","attribs":{"title":"FKBP5; IQGAP2; TPD52; SORD"},"children":["FKBP5; IQGAP2; TPD52; SORD"]},{"name":"span","attribs":{"title":"AQP3; FKBP5; SEC14L2; KDM4B; SYBU; WWC1; CHPT1"},"children":["AQP3; FKBP5; SEC14L2; KDM4B; SYBU; WWC1; CHPT1"]},{"name":"span","attribs":{"title":"KDM4B; CD9; IQGAP2; MCCC2; SGK3; TTC21A"},"children":["KDM4B; CD9; IQGAP2; MCCC2; SGK3; TTC21A"]},{"name":"span","attribs":{"title":"PRKAB2; CRAT; ACACA; PRKAA2"},"children":["PRKAB2; CRAT; ACACA; PRKAA2"]},{"name":"span","attribs":{"title":"FKBP5; CD9; SORD; CHPT1"},"children":["FKBP5; CD9; SORD; CHPT1"]},{"name":"span","attribs":{"title":"RANBP3L; NOL6; MED21"},"children":["RANBP3L; NOL6; MED21"]},{"name":"span","attribs":{"title":"PRKAB2; ACACA; PRKAA2"},"children":["PRKAB2; ACACA; PRKAA2"]},{"name":"span","attribs":{"title":"CD9; SORD; NAMPT; ACACA"},"children":["CD9; SORD; NAMPT; ACACA"]},{"name":"span","attribs":{"title":"MCCC2; NAMPT; ACACA"},"children":["MCCC2; NAMPT; ACACA"]},{"name":"span","attribs":{"title":"GSTM3; SULT2B1; GSTM4"},"children":["GSTM3; SULT2B1; GSTM4"]},{"name":"span","attribs":{"title":"PTPRJ; ACACA; PRKAA2"},"children":["PTPRJ; ACACA; PRKAA2"]},{"name":"span","attribs":{"title":"AQP3; NEDD4L; SGK3; SLC66A1"},"children":["AQP3; NEDD4L; SGK3; SLC66A1"]},{"name":"span","attribs":{"title":"SPRED1; FGFR2; PEA15; DUSP16; RASGEF1A; GFRA1"},"children":["SPRED1; FGFR2; PEA15; DUSP16; RASGEF1A; GFRA1"]}],"minWidth":120}],"filterable":true,"showPageSizeOptions":true,"dataKey":"f95c63bed5553ca17fe24e837e9ad9d7"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div id="network-plot-1" class="section level4">
<h4>Network Plot</h4>
<pre class="r"><code>tg_up %&gt;% 
  activate(nodes) %&gt;% 
  mutate(direction = ifelse(NES &gt; 0, &quot;Up&quot;, &quot;Down&quot;)) %&gt;% 
  ggraph(layout = net_layout, weights = oc^2) +
  geom_edge_link(aes(width = oc^2, alpha = oc^2)) +
  geom_node_point(
    aes(fill = Status, size = numDEInCat),
    shape = 21
  ) +
  geom_node_text(
    aes(label = label, colour = direction),
    size = 3, 
    data = . %&gt;%
      mutate(
        label = str_replace_all(label, &quot;_&quot;, &quot; &quot;) %&gt;% str_trunc(80) %&gt;% str_wrap(width = 18)
      ),
    repel = TRUE, max.overlaps = max(10, round(length(tg_up) / 4, 0)),
    bg.color = &quot;white&quot;, bg.r = 0.1, 
  ) +
  scale_x_continuous(expand = expansion(c(0.1, 0.1))) +
  scale_y_continuous(expand = expansion(c(0.1, 0.1))) +
  scale_fill_manual(values = status_cols) +
  scale_size_continuous(range = c(1, 10)) +
  scale_edge_width(range = c(1, 6), limits = c(0, 1)) +
  scale_edge_alpha(range = c(0.1, 0.4), limits = c(0, 1))  +
  scale_colour_manual(
    values = setNames(direction_colours, str_to_title(names(direction_colours)))
  ) +
  guides(edge_alpha= &quot;none&quot;) +
  labs(
    size = glue(&quot;{target} Targets In Leading Edge&quot;) %&gt;% str_wrap(15),
    colour = &quot;GSEA\nDirection&quot;,
    fill = &quot;Prior\nSignificance&quot;,
    edge_width = expr(paste(OC^2))
  ) +
  theme_void() +
  theme(legend.text = element_text(hjust = 0.5))</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/network-cmn-allup-1.png" alt="*Network plot for pathways considered significant when assessing all increasingly bound H3K27ac sites along with GSEA on the provided RNA-Seq data. Distances are derived from all detected genes within the pathway. Label colours indicate the direction of enrichment in GSEA results for genes in the pathway.*" width="960" />
<p class="caption">
<em>Network plot for pathways considered significant when assessing all
increasingly bound H3K27ac sites along with GSEA on the provided RNA-Seq
data. Distances are derived from all detected genes within the pathway.
Label colours indicate the direction of enrichment in GSEA results for
genes in the pathway.</em>
</p>
</div>
</div>
</div>
<div id="windows-with-decreased-h3k27ac" class="section level3 tabset">
<h3 class="tabset">Windows With Decreased H3K27ac</h3>
<pre class="r"><code>cmn_down &lt;- rnaseq_gsea %&gt;%
  dplyr::select(-log2err, -ES, -size) %&gt;% 
  left_join(
    goseq_down, by = c(&quot;pathway&quot; = &quot;gs_name&quot;), suffix = c(&quot;.rna&quot;, &quot;.chip&quot;)
  ) %&gt;%
  dplyr::filter(!is.na(pval.rna), !is.na(pval.chip)) %&gt;% 
  nest(p = starts_with(&quot;pval&quot;)) %&gt;% 
  mutate(
    pval = vapply(p, function(x) metap::maximump(unlist(x))$p, numeric(1))
    ) %&gt;%
  dplyr::rename(padj_chip = adj_p, padj_rna = padj) %&gt;% 
  mutate(adj_p = p.adjust(pval, params$enrichment$adj)) %&gt;% 
  arrange(pval) %&gt;% 
  dplyr::filter(adj_p &lt; enrich_alpha) %&gt;% 
  mutate(
    leadingEdge = lapply(
      leadingEdge, intersect, dplyr::filter(mapped_ids, down)$gene_id
    ),
    numDEInCat = vapply(leadingEdge, length, integer(1)),
    Status = case_when(
      padj_rna &lt; enrich_alpha &amp; padj_chip &lt; enrich_alpha ~ &quot;Both&quot;,
      padj_rna &lt; enrich_alpha &amp; !padj_chip &lt; enrich_alpha ~ &quot;RNA Only&quot;,
      !padj_rna &lt; enrich_alpha &amp; padj_chip &lt; enrich_alpha ~ &quot;ChIP Only&quot;,
      !padj_rna &lt; enrich_alpha &amp; !padj_chip &lt; enrich_alpha ~ &quot;Neither&quot;
    ) %&gt;% 
      factor(levels = names(status_cols))
  ) %&gt;% 
  dplyr::filter(numDEInCat &gt;= min_sig) %&gt;% 
  dplyr::select(
    gs_name = pathway, adj_p, Status, NES, starts_with(&quot;padj&quot;),
    starts_with(&quot;num&quot;), leadingEdge
  ) 
tg_down &lt;- cmn_down %&gt;% 
  make_tbl_graph(
    setNames(.$leadingEdge, .$gs_name)
  )
plot_net &lt;- length(tg_down) &gt;= min_network_size
txt &lt;- ifelse(
  nrow(cmn_down) == 0,
  glue(&quot;No gene sets were jointly enriched between all decreasing {target} bound windows and the RNA-Seq data&quot;),
  &quot;#### Results Table&quot;
)</code></pre>
<p>No gene sets were jointly enriched between all decreasing H3K27ac
bound windows and the RNA-Seq data</p>
<pre class="r"><code>tg_down %&gt;% 
  activate(nodes) %&gt;% 
  mutate(direction = ifelse(NES &gt; 0, &quot;Up&quot;, &quot;Down&quot;)) %&gt;% 
  ggraph(layout = net_layout, weights = oc^2) +
  geom_edge_link(aes(width = oc^2, alpha = oc^2)) +
  geom_node_point(
    aes(fill = Status, size = numDEInCat),
    shape = 21
  ) +
  geom_node_text(
    aes(label = label, colour = direction),
    size = 3, 
    data = . %&gt;%
      mutate(
        label = str_replace_all(label, &quot;_&quot;, &quot; &quot;) %&gt;% str_trunc(80) %&gt;% str_wrap(width = 18)
      ),
    repel = TRUE, max.overlaps = max(10, round(length(tg_down) / 4, 0)),
    bg.color = &quot;white&quot;, bg.r = 0.1, 
  ) +
  scale_x_continuous(expand = expansion(c(0.1, 0.1))) +
  scale_y_continuous(expand = expansion(c(0.1, 0.1))) +
  scale_fill_manual(values = status_cols) +
  scale_size_continuous(range = c(1, 10)) +
  scale_edge_width(range = c(1, 6), limits = c(0, 1)) +
  scale_edge_alpha(range = c(0.1, 0.4), limits = c(0, 1))  +
  scale_colour_manual(
    values = setNames(direction_colours, str_to_title(names(direction_colours)))
  ) +
  guides(edge_alpha= &quot;none&quot;) +
  labs(
    size = glue(&quot;{target} Targets In Leading Edge&quot;) %&gt;% str_wrap(15),
    colour = &quot;GSEA\nDirection&quot;,
    fill = &quot;Prior\nSignificance&quot;,
    edge_width = expr(paste(OC^2))
  ) +
  theme_void() +
  theme(legend.text = element_text(hjust = 0.5))</code></pre>
</div>
</div>
<div id="highly-ranked-genes" class="section level2 tabset">
<h2 class="tabset">Highly Ranked Genes</h2>
<pre class="r"><code>knitr::opts_chunk$set(dev = fig_type)
all_ids &lt;- unlist(merged_results$gene_id)
db_ids &lt;- unlist(filter(merged_results, !!sym(fdr_column) &lt; fdr_alpha)$gene_id)
top_diff_win &lt;- rnaseq %&gt;% 
  mutate(
    n_windows = vapply(
      gene_id,
      function(x) sum(str_detect(all_ids, x)),
      integer(1)
    ),
    n_db_windows = vapply(
      gene_id,
      function(x) sum(str_detect(db_ids, x)),
      integer(1)
    )
  ) %&gt;% 
  arrange(!!sym(rna_p_col)) %&gt;% 
  dplyr::filter(n_windows &gt; 1) %&gt;% 
  dplyr::slice(1:5)
grl_to_plot &lt;- top_diff_win$gene_id %&gt;% 
  sapply(
    function(x) {
      subset(
        merged_results, 
        vapply(gene_id, function(id) x %in% id, logical(1))
      ) %&gt;% 
        granges() %&gt;% 
        ## Restrict this to only sites within 5Mb of a gene
        subsetByOverlaps(
          resize(subset(gtf_gene, gene_id == x), width = 1e7, fix = &#39;center&#39;)
        ) %&gt;% 
        c(
          granges(subset(gtf_gene, gene_id == x))
        ) %&gt;% 
        unstrand() %&gt;% 
        sort()
    },
    simplify = FALSE
  ) %&gt;% 
  lapply(range) %&gt;% 
  GRangesList()
hfgc_genes &lt;- hfgc_genes[names(hfgc_genes) != &quot;Undetected&quot;]
if (!is.null(config$external$coverage)) {
  y_ranges &lt;- granges(unlist(grl_to_plot))
  y_lim &lt;- c(
    y_lim[target],
    bwfl[names(bwfl) != target] %&gt;% 
      lapply(
        function(x) {
          GRangesList(lapply(x, import.bw, which = y_ranges)) %&gt;% 
            unlist() %&gt;% 
            filter(score == max(score)) %&gt;% 
            mcols() %&gt;% 
            .[[&quot;score&quot;]] %&gt;% 
            c(0) %&gt;% 
            range()
        }
      )
  )
}
i &lt;- 1</code></pre>
<p>Using the ranked list of genes from the RNA-Seq data, the 5 genes
most highly ranked for differential expression, with more than one
H3K27ac-bound window mapped to it were selected for visualisation.</p>
<div id="aqp3" class="section level3">
<h3>AQP3</h3>
<pre class="r"><code>ct &lt;- trans_models %&gt;% 
  subsetByOverlaps(grl_to_plot[[i]]) %&gt;% 
  as_tibble() %&gt;% 
  pull(&quot;transcript&quot;) %&gt;% 
  unique() %&gt;% 
  length() %&gt;% 
  is_greater_than(10) %&gt;% 
  ifelse(&quot;meta&quot;, FALSE)
plotHFGC(
  grl_to_plot[[i]],
  hic = hic %&gt;% 
     subsetByOverlaps(
       subset(gtf_gene, gene_id == names(grl_to_plot)[[i]])
     ) %&gt;% 
    subsetByOverlaps(
      subsetByOverlaps(merged_results, grl_to_plot[[i]])
    ),
  features = feat_gr, featcol = feature_colours, featstack = &quot;dense&quot;, 
  genes = hfgc_genes, genecol = gene_col, genesize = 0.7,
  coverage = bwfl,
  linecol = line_col,
  annotation = annot,
  annotcol = colours$direction %&gt;%
    setNames(str_to_title(names(.))),
  cytobands = bands_df,
  collapseTranscripts = ct, # Set this as a function of n_transcripts
  zoom = 1.1,
  fontsize = 10,
  max = width(grl_to_plot[[i]]),
  ylim = y_lim,
  highlight = c(),
  col.title = &quot;black&quot;, background.title = &quot;white&quot;, showAxis = FALSE,
  rotation.title = 90
)</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-gene1-1.png" alt="*Region showing all merged H3K27ac-bound windows mapped to UGDH. Windows considered to be unchanged for H3K27ac are annotated in grey, with other colours indicating H3K27ac gain or loss. Mapped windows are only shown if within 5Mb of UGDH. Gene-centric regions and external features are shown in the top panel. The estimated logFC for UGDH is 1.52 with an FDR of 2.34e-10. Undetected and unchanged genes are shown on separate tracks.*" width="960" />
<p class="caption">
<em>Region showing all merged H3K27ac-bound windows mapped to UGDH.
Windows considered to be unchanged for H3K27ac are annotated in grey,
with other colours indicating H3K27ac gain or loss. Mapped windows are
only shown if within 5Mb of UGDH. Gene-centric regions and external
features are shown in the top panel. The estimated logFC for UGDH is
1.52 with an FDR of 2.34e-10. Undetected and unchanged genes are shown
on separate tracks.</em>
</p>
</div>
<pre class="r"><code>i &lt;- i + 1</code></pre>
</div>
<div id="ugdh" class="section level3">
<h3>UGDH</h3>
<pre class="r"><code>ct &lt;- trans_models %&gt;% 
  subsetByOverlaps(grl_to_plot[[i]]) %&gt;% 
  as_tibble() %&gt;% 
  pull(&quot;transcript&quot;) %&gt;% 
  unique() %&gt;% 
  length() %&gt;% 
  is_greater_than(10) %&gt;% 
  ifelse(&quot;meta&quot;, FALSE)
plotHFGC(
  grl_to_plot[[i]],
  hic = hic %&gt;% 
     subsetByOverlaps(
       subset(gtf_gene, gene_id == names(grl_to_plot)[[i]])
     ) %&gt;% 
    subsetByOverlaps(
      subsetByOverlaps(merged_results, grl_to_plot[[i]])
    ),
  features = feat_gr, featcol = feature_colours, featstack = &quot;dense&quot;, 
  genes = hfgc_genes, genecol = gene_col, genesize = 0.7,
  coverage = bwfl,
  linecol = line_col,
  annotation = annot,
  annotcol = colours$direction %&gt;%
    setNames(str_to_title(names(.))),
  cytobands = bands_df,
  collapseTranscripts = ct, # Set this as a function of n_transcripts
  zoom = 1.1,
  max = width(grl_to_plot[[i]]),
  fontsize = 10,
  ylim = y_lim,
  highlight = c(),
  col.title = &quot;black&quot;, background.title = &quot;white&quot;, showAxis = FALSE,
  rotation.title = 90
)</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-gene2-1.png" alt="*Region showing all merged H3K27ac-bound windows mapped to FKBP5. Windows considered to be unchanged for H3K27ac are annotated in grey, with other colours indicating H3K27ac gain or loss. Mapped windows are only shown if within 5Mb of FKBP5. Gene-centric regions and external features are shown in the top panel. The estimated logFC for FKBP5 is 2.00 with an FDR of 6.51e-10. Undetected and unchanged genes are shown on separate tracks.*" width="960" />
<p class="caption">
<em>Region showing all merged H3K27ac-bound windows mapped to FKBP5.
Windows considered to be unchanged for H3K27ac are annotated in grey,
with other colours indicating H3K27ac gain or loss. Mapped windows are
only shown if within 5Mb of FKBP5. Gene-centric regions and external
features are shown in the top panel. The estimated logFC for FKBP5 is
2.00 with an FDR of 6.51e-10. Undetected and unchanged genes are shown
on separate tracks.</em>
</p>
</div>
<pre class="r"><code>i &lt;- i + 1</code></pre>
</div>
<div id="fkbp5" class="section level3">
<h3>FKBP5</h3>
<pre class="r"><code>ct &lt;- trans_models %&gt;% 
  subsetByOverlaps(grl_to_plot[[i]]) %&gt;% 
  as_tibble() %&gt;% 
  pull(&quot;transcript&quot;) %&gt;% 
  unique() %&gt;% 
  length() %&gt;% 
  is_greater_than(10) %&gt;% 
  ifelse(&quot;meta&quot;, FALSE)
plotHFGC(
  grl_to_plot[[i]],
  hic = hic %&gt;% 
     subsetByOverlaps(
       subset(gtf_gene, gene_id == names(grl_to_plot)[[i]])
     ) %&gt;% 
    subsetByOverlaps(
      subsetByOverlaps(merged_results, grl_to_plot[[i]])
    ),
  features = feat_gr, featcol = feature_colours, featstack = &quot;dense&quot;, 
  genes = hfgc_genes, genecol = gene_col, genesize = 0.7,
  coverage = bwfl,
  linecol = line_col,
  annotation = annot,
  annotcol = colours$direction %&gt;%
    setNames(str_to_title(names(.))),
  cytobands = bands_df,
  collapseTranscripts = ct, # Set this as a function of n_transcripts
  zoom = 1.1,
  max = width(grl_to_plot[[i]]),
  fontsize = 10,
  ylim = y_lim,
  highlight = c(),
  col.title = &quot;black&quot;, background.title = &quot;white&quot;, showAxis = FALSE,
  rotation.title = 90
)</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-gene3-1.png" alt="*Region showing all merged H3K27ac-bound windows mapped to SEC14L2. Windows considered to be unchanged for H3K27ac are annotated in grey, with other colours indicating H3K27ac gain or loss. Mapped windows are only shown if within 5Mb of SEC14L2. Gene-centric regions and external features are shown in the top panel. The estimated logFC for SEC14L2 is 2.65 with an FDR of 8.87e-10. Undetected and unchanged genes are shown on separate tracks.*" width="960" />
<p class="caption">
<em>Region showing all merged H3K27ac-bound windows mapped to SEC14L2.
Windows considered to be unchanged for H3K27ac are annotated in grey,
with other colours indicating H3K27ac gain or loss. Mapped windows are
only shown if within 5Mb of SEC14L2. Gene-centric regions and external
features are shown in the top panel. The estimated logFC for SEC14L2 is
2.65 with an FDR of 8.87e-10. Undetected and unchanged genes are shown
on separate tracks.</em>
</p>
</div>
<pre class="r"><code>i &lt;- i + 1</code></pre>
</div>
<div id="sec14l2" class="section level3">
<h3>SEC14L2</h3>
<pre class="r"><code>ct &lt;- trans_models %&gt;% 
  subsetByOverlaps(grl_to_plot[[i]]) %&gt;% 
  as_tibble() %&gt;% 
  pull(&quot;transcript&quot;) %&gt;% 
  unique() %&gt;% 
  length() %&gt;% 
  is_greater_than(10) %&gt;% 
  ifelse(&quot;meta&quot;, FALSE)
plotHFGC(
  grl_to_plot[[i]],
  hic = hic %&gt;% 
     subsetByOverlaps(
       subset(gtf_gene, gene_id == names(grl_to_plot)[[i]])
     ) %&gt;% 
    subsetByOverlaps(
      subsetByOverlaps(merged_results, grl_to_plot[[i]])
    ),
  features = feat_gr, featcol = feature_colours, featstack = &quot;dense&quot;, 
  genes = hfgc_genes, genecol = gene_col, genesize = 0.7,
  coverage = bwfl,
  linecol = line_col,
  annotation = annot,
  annotcol = colours$direction %&gt;%
    setNames(str_to_title(names(.))),
  cytobands = bands_df,
  collapseTranscripts = ct, # Set this as a function of n_transcripts
  zoom = 1.1,
  max = width(grl_to_plot[[i]]),
  fontsize = 10,
  ylim = y_lim,
  highlight = c(),
  col.title = &quot;black&quot;, background.title = &quot;white&quot;, showAxis = FALSE,
  rotation.title = 90
)</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-gene4-1.png" alt="*Region showing all merged H3K27ac-bound windows mapped to EAF2. Windows considered to be unchanged for H3K27ac are annotated in grey, with other colours indicating H3K27ac gain or loss. Mapped windows are only shown if within 5Mb of EAF2. Gene-centric regions and external features are shown in the top panel. The estimated logFC for EAF2 is 1.87 with an FDR of 1.19e-07. Undetected and unchanged genes are shown on separate tracks.*" width="960" />
<p class="caption">
<em>Region showing all merged H3K27ac-bound windows mapped to EAF2.
Windows considered to be unchanged for H3K27ac are annotated in grey,
with other colours indicating H3K27ac gain or loss. Mapped windows are
only shown if within 5Mb of EAF2. Gene-centric regions and external
features are shown in the top panel. The estimated logFC for EAF2 is
1.87 with an FDR of 1.19e-07. Undetected and unchanged genes are shown
on separate tracks.</em>
</p>
</div>
<pre class="r"><code>i &lt;- i + 1</code></pre>
</div>
<div id="eaf2" class="section level3">
<h3>EAF2</h3>
<pre class="r"><code>ct &lt;- trans_models %&gt;% 
  subsetByOverlaps(grl_to_plot[[i]]) %&gt;% 
  as_tibble() %&gt;% 
  pull(&quot;transcript&quot;) %&gt;% 
  unique() %&gt;% 
  length() %&gt;% 
  is_greater_than(10) %&gt;% 
  ifelse(&quot;meta&quot;, FALSE)
plotHFGC(
  grl_to_plot[[i]],
  hic = hic %&gt;% 
     subsetByOverlaps(
       subset(gtf_gene, gene_id == names(grl_to_plot)[[i]])
     ) %&gt;% 
    subsetByOverlaps(
      subsetByOverlaps(merged_results, grl_to_plot[[i]])
    ),
  features = feat_gr, featcol = feature_colours, featstack = &quot;dense&quot;, 
  genes = hfgc_genes, genecol = gene_col, genesize = 0.7,
  coverage = bwfl,
  linecol = line_col,
  annotation = annot,
  annotcol = colours$direction %&gt;%
    setNames(str_to_title(names(.))),
  cytobands = bands_df,
  collapseTranscripts = ct, # Set this as a function of n_transcripts
  zoom = 1.1,
  max = width(grl_to_plot[[i]]),
  fontsize = 10,
  ylim = y_lim,
  highlight = c(),
  col.title = &quot;black&quot;, background.title = &quot;white&quot;, showAxis = FALSE,
  rotation.title = 90
)</code></pre>
<div class="figure" style="text-align: center">
<img src="H3K27ac_E2_E2DHT_differential_binding_files/figure-html/plot-gene5-1.png" alt="*Region showing all merged H3K27ac-bound windows mapped to EAF2. Windows considered to be unchanged for H3K27ac are annotated in grey, with other colours indicating H3K27ac gain or loss. Mapped windows are only shown if within 5Mb of EAF2. Gene-centric regions and external features are shown in the top panel. The estimated logFC for EAF2 is 1.87 with an FDR of 1.19e-07. Undetected and unchanged genes are shown on separate tracks.*" width="960" />
<p class="caption">
<em>Region showing all merged H3K27ac-bound windows mapped to EAF2.
Windows considered to be unchanged for H3K27ac are annotated in grey,
with other colours indicating H3K27ac gain or loss. Mapped windows are
only shown if within 5Mb of EAF2. Gene-centric regions and external
features are shown in the top panel. The estimated logFC for EAF2 is
1.87 with an FDR of 1.19e-07. Undetected and unchanged genes are shown
on separate tracks.</em>
</p>
</div>
</div>
</div>
</div>
<div id="data-export" class="section level1">
<h1>Data Export</h1>
<pre class="r"><code>merged_results %&gt;% 
  as_tibble() %&gt;%
  unnest(all_of(&quot;gene_id&quot;)) %&gt;%
  mutate(gene_name = id2gene[gene_id]) %&gt;% 
  dplyr::select(
    gene_id, gene_name, range, AveExpr, logFC, P.Value, FDR = !!sym(fdr_column), 
    status, any_of(c(&quot;region&quot;, &quot;feature&quot;)), macs2_peak = overlaps_ref,
  ) %&gt;% 
  mutate(
    distance_to_gene = distance(
      GRanges(range),
      setNames(gtf_gene, gtf_gene$gene_id)[gene_id]
    )
  ) %&gt;% 
  write_csv(
    gzfile(all_out$csv)
  )</code></pre>
<pre class="r"><code>write_rds(merged_results, all_out$results, compress = &quot;gz&quot;)
write_rds(filtered_counts, all_out$windows, compress = &quot;gz&quot;)
file.create(all_out$de_genes)
if (has_rnaseq) write_csv(de_genes_db_regions, all_out$de_genes)

file.create(all_out$up_regions)
if (sum(merged_results$status == &quot;Up&quot;) &gt; 0) {
  merged_results %&gt;% 
    filter(status == &quot;Up&quot;) %&gt;% 
    granges() %&gt;% 
    export.bed(all_out$up_regions)
}

file.create(all_out$down_regions)
if (sum(merged_results$status == &quot;Down&quot;) &gt; 0) {
  merged_results %&gt;% 
    filter(status == &quot;Down&quot;) %&gt;% 
    granges() %&gt;% 
    export.bed(all_out$up_regions)
}
## Enrichment results need a bit of tweaking to match the tables
list2(
  `All Differentially Bound` = goseq_diff,
  &quot;All Increased {target}&quot; := goseq_up,
  &quot;All Decreased {target}&quot; := goseq_down
) %&gt;% 
  bind_rows(.id = &#39;group&#39;) %&gt;% 
  dplyr::filter(adj_p &lt; enrich_alpha) %&gt;% 
  left_join(
    dplyr::select(msigdb, gs_name, gene_name, gene_id, gs_url, gs_description),
    by = &quot;gs_name&quot;
  ) %&gt;% 
  dplyr::filter(
    gene_id %in% unlist(subset(merged_results, status != &quot;Unchanged&quot;)$gene_id)
  ) %&gt;% 
  dplyr::select(-gene_id) %&gt;% 
  chop(gene_name) %&gt;% 
  mutate(
    gene_name = vapply(gene_name, paste, character(1), collapse = &quot;: &quot;),
    `%` = numDEInCat / numInCat,
  ) %&gt;% 
  dplyr::select(
    group,
    `Gene Set` = gs_name, Description = gs_description, URL = gs_url,
    `%`, numDEInCat, numInCat, p = pval, adj_p, 
    `Genes Mapped to DB Windows` = gene_name
  ) %&gt;% 
  write_csv(all_out$enrichment)
list2(
  `All Differentially Bound` = cmn_diff,
  &quot;All Increased {target}&quot; := cmn_up,
  &quot;All Decreased {target}&quot; := cmn_down
) %&gt;% 
  bind_rows(.id = &quot;group&quot;) %&gt;% 
  mutate(
    genes = vapply(leadingEdge, paste, character(1), collapse = &quot;; &quot;),
    leadingEdge = vapply(leadingEdge, paste, character(1), collapse = &quot;; &quot;),
  ) %&gt;% 
  write_csv(all_out$rna_enrichment)
## Save key memory on the HDD
rm(window_counts)
gc()
save.image(all_out$renv)</code></pre>
<p>During this workflow, the following files were exported:</p>
<ul>
<li><strong>results</strong>:
output/differential_binding/H3K27ac/H3K27ac_E2_E2DHT-differential_binding.rds</li>
<li><strong>csv</strong>:
output/differential_binding/H3K27ac/H3K27ac_E2_E2DHT-differential_binding.csv.gz</li>
<li><strong>windows</strong>:
output/differential_binding/H3K27ac/H3K27ac_E2_E2DHT-filtered_windows.rds</li>
<li><strong>up_regions</strong>:
output/differential_binding/H3K27ac/H3K27ac_E2_E2DHT-up.bed</li>
<li><strong>down_regions</strong>:
output/differential_binding/H3K27ac/H3K27ac_E2_E2DHT-down.bed</li>
<li><strong>de_genes</strong>:
output/differential_binding/H3K27ac/H3K27ac_E2_E2DHT-DE_genes.csv</li>
<li><strong>enrichment</strong>:
output/differential_binding/H3K27ac/H3K27ac_E2_E2DHT-enrichment.csv</li>
<li><strong>rna_enrichment</strong>:
output/differential_binding/H3K27ac/H3K27ac_E2_E2DHT-rnaseq_enrichment.csv</li>
<li><strong>renv</strong>:
output/envs/H3K27ac_E2_E2DHT-differential_binding.RData</li>
</ul>
<!-- end of list -->
<div id="references" class="section level2">
<h2>References</h2>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-HicksQuantro2015" class="csl-entry">
Hicks, S. C., and R. A. Irizarry. 2015. <span>“<span
class="nocase">quantro: a data-driven approach to guide the choice of an
appropriate normalization method</span>.”</span> <em>Genome Biol</em> 16
(June): 117.
</div>
<div id="ref-HicksSQN2017" class="csl-entry">
Hicks, Stephanie C, Kwame Okrah, Joseph N Paulson, John Quackenbush,
Rafael A Irizarry, and Héctor Corrada Bravo. 2017. <span>“<span
class="nocase">Smooth quantile normalization</span>.”</span>
<em>Biostatistics</em> 19 (2): 185–98. <a
href="https://doi.org/10.1093/biostatistics/kxx028">https://doi.org/10.1093/biostatistics/kxx028</a>.
</div>
<div id="ref-IgnatiadisIHW2016" class="csl-entry">
Ignatiadis, N., B. Klaus, J. B. Zaugg, and W. Huber. 2016. <span>“<span
class="nocase"><span>D</span>ata-driven hypothesis weighting increases
detection power in genome-scale multiple testing</span>.”</span> <em>Nat
Methods</em> 13 (7): 577–80.
</div>
<div id="ref-KorotkevichFgsea2019" class="csl-entry">
Korotkevich, Gennady, Vladimir Sukhov, and Alexey Sergushichev. 2019.
<span>“Fast Gene Set Enrichment Analysis.”</span> <em>bioRxiv</em>. <a
href="https://doi.org/10.1101/060012">https://doi.org/10.1101/060012</a>.
</div>
<div id="ref-LawVoom2014" class="csl-entry">
Law, C. W., Y. Chen, W. Shi, and G. K. Smyth. 2014. <span>“<span
class="nocase">voom: <span>P</span>recision weights unlock linear model
analysis tools for <span>R</span><span>N</span><span>A</span>-seq read
counts</span>.”</span> <em>Genome Biol</em> 15 (2): R29.
</div>
<div id="ref-LunSmythF1000" class="csl-entry">
Lun, A. T., and G. K. Smyth. 2015. <span>“<span
class="nocase"><span>F</span>rom reads to regions: a
<span>B</span>ioconductor workflow to detect differential binding in
<span>C</span>h<span>I</span><span>P</span>-seq data</span>.”</span>
<em>F1000Res</em> 4: 1080.
</div>
<div id="ref-LunSmythCsaw2014" class="csl-entry">
Lun, Aaron T L, and Gordon K Smyth. 2014. <span>“<span>D</span>e Novo
Detection of Differentially Bound Regions for
<span>C</span>h<span>I</span><span>P</span>-Seq Data Using Peaks and
Windows: Controlling Error Rates Correctly.”</span> <em>Nucleic Acids
Res.</em> 42 (11): e95.
</div>
<div id="ref-McCarthyTreat2009" class="csl-entry">
McCarthy, Davis J., and Gordon K. Smyth. 2009. <span>“<span
class="nocase">Testing significance relative to a fold-change threshold
is a TREAT</span>.”</span> <em>Bioinformatics</em> 25 (6): 765–71. <a
href="https://doi.org/10.1093/bioinformatics/btp053">https://doi.org/10.1093/bioinformatics/btp053</a>.
</div>
<div id="ref-SubramanianGsea2005" class="csl-entry">
Subramanian, Aravind, Pablo Tamayo, Vamsi K. Mootha, Sayan Mukherjee,
Benjamin L. Ebert, Michael A. Gillette, Amanda Paulovich, et al. 2005.
<span>“Gene Set Enrichment Analysis: A Knowledge-Based Approach for
Interpreting Genome-Wide Expression Profiles.”</span> <em>Proceedings of
the National Academy of Sciences</em> 102 (43): 15545–50. <a
href="https://doi.org/10.1073/pnas.0506580102">https://doi.org/10.1073/pnas.0506580102</a>.
</div>
<div id="ref-YoungGoseq2010" class="csl-entry">
Young, M. D., M. J. Wakefield, G. K. Smyth, and A. Oshlack. 2010.
<span>“<span class="nocase"><span>G</span>ene ontology analysis for
<span>R</span><span>N</span><span>A</span>-seq: accounting for selection
bias</span>.”</span> <em>Genome Biol</em> 11 (2): R14.
</div>
<div id="ref-Zhang18798982" class="csl-entry">
Zhang, Y., T. Liu, C. A. Meyer, J. Eeckhoute, D. S. Johnson, B. E.
Bernstein, C. Nusbaum, et al. 2008. <span>“<span
class="nocase"><span>M</span>odel-based analysis of
<span>C</span>h<span>I</span><span>P</span>-<span>S</span>eq
(<span>M</span><span>A</span><span>C</span><span>S</span>)</span>.”</span>
<em>Genome Biol</em> 9 (9): R137.
</div>
</div>
<br>
<button type="button" class="btn btn-default btn-sessioninfo" data-toggle="collapse" data-target="#sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span>
Session information </button>
</p>
<div id="sessioninfo" class="collapse">
<p><strong>R version 4.2.3 (2023-03-15)</strong></p>
<p><strong>Platform:</strong> x86_64-conda-linux-gnu (64-bit)</p>
<p><strong>locale:</strong> <em>LC_CTYPE=en_AU.UTF-8</em>,
<em>LC_NUMERIC=C</em>, <em>LC_TIME=en_AU.UTF-8</em>,
<em>LC_COLLATE=en_AU.UTF-8</em>, <em>LC_MONETARY=en_AU.UTF-8</em>,
<em>LC_MESSAGES=en_AU.UTF-8</em>, <em>LC_PAPER=en_AU.UTF-8</em>,
<em>LC_NAME=C</em>, <em>LC_ADDRESS=C</em>, <em>LC_TELEPHONE=C</em>,
<em>LC_MEASUREMENT=en_AU.UTF-8</em> and <em>LC_IDENTIFICATION=C</em></p>
<p><strong>attached base packages:</strong> <em>stats4</em>,
<em>stats</em>, <em>graphics</em>, <em>grDevices</em>, <em>utils</em>,
<em>datasets</em>, <em>methods</em> and <em>base</em></p>
<p><strong>other attached packages:</strong> <em>fgsea(v.1.24.0)</em>,
<em>qsmooth(v.1.14.0)</em>, <em>quantro(v.1.32.0)</em>,
<em>metap(v.1.1)</em>, <em>ggraph(v.2.1.0)</em>,
<em>tidygraph(v.1.2.3)</em>, <em>extraChIPs(v.1.5.5)</em>,
<em>goseq(v.1.50.0)</em>, <em>geneLenDataBase(v.1.34.0)</em>,
<em>BiasedUrn(v.2.0.10)</em>, <em>msigdbr(v.7.5.1)</em>,
<em>htmltools(v.0.5.5)</em>, <em>reactable(v.0.4.4)</em>,
<em>GenomicInteractions(v.1.32.0)</em>,
<em>InteractionSet(v.1.26.0)</em>, <em>ggside(v.0.2.2)</em>,
<em>rlang(v.1.1.1)</em>, <em>ggrepel(v.0.9.3)</em>,
<em>IHW(v.1.26.0)</em>, <em>statmod(v.1.5.0)</em>,
<em>magrittr(v.2.0.3)</em>, <em>scales(v.1.2.1)</em>,
<em>edgeR(v.3.40.0)</em>, <em>limma(v.3.54.0)</em>,
<em>csaw(v.1.32.0)</em>, <em>SummarizedExperiment(v.1.28.0)</em>,
<em>Biobase(v.2.58.0)</em>, <em>MatrixGenerics(v.1.10.0)</em>,
<em>matrixStats(v.1.0.0)</em>, <em>ngsReports(v.2.0.0)</em>,
<em>patchwork(v.1.1.2)</em>, <em>cowplot(v.1.1.1)</em>,
<em>Rsamtools(v.2.14.0)</em>, <em>Biostrings(v.2.66.0)</em>,
<em>XVector(v.0.38.0)</em>, <em>rtracklayer(v.1.58.0)</em>,
<em>plyranges(v.1.18.0)</em>, <em>GenomicRanges(v.1.50.0)</em>,
<em>GenomeInfoDb(v.1.34.9)</em>, <em>IRanges(v.2.32.0)</em>,
<em>S4Vectors(v.0.36.0)</em>, <em>BiocGenerics(v.0.44.0)</em>,
<em>yaml(v.2.3.7)</em>, <em>BiocParallel(v.1.32.5)</em>,
<em>pander(v.0.6.5)</em>, <em>glue(v.1.6.2)</em>,
<em>lubridate(v.1.9.2)</em>, <em>forcats(v.1.0.0)</em>,
<em>stringr(v.1.5.0)</em>, <em>dplyr(v.1.1.2)</em>,
<em>purrr(v.1.0.1)</em>, <em>readr(v.2.1.4)</em>,
<em>tidyr(v.1.3.0)</em>, <em>tibble(v.3.2.1)</em>,
<em>ggplot2(v.3.4.2)</em> and <em>tidyverse(v.2.0.0)</em></p>
<p><strong>loaded via a namespace (and not attached):</strong>
<em>rappdirs(v.0.3.3)</em>, <em>bumphunter(v.1.40.0)</em>,
<em>minfi(v.1.44.0)</em>, <em>bit64(v.4.0.5)</em>,
<em>knitr(v.1.43)</em>, <em>DelayedArray(v.0.24.0)</em>,
<em>data.table(v.1.14.8)</em>, <em>rpart(v.4.1.19)</em>,
<em>GEOquery(v.2.66.0)</em>, <em>KEGGREST(v.1.38.0)</em>,
<em>RCurl(v.1.98-1.12)</em>, <em>AnnotationFilter(v.1.22.0)</em>,
<em>doParallel(v.1.0.17)</em>, <em>generics(v.0.1.3)</em>,
<em>preprocessCore(v.1.60.2)</em>, <em>GenomicFeatures(v.1.50.2)</em>,
<em>EnrichedHeatmap(v.1.27.2)</em>, <em>lambda.r(v.1.2.4)</em>,
<em>RSQLite(v.2.3.1)</em>, <em>bit(v.4.0.5)</em>,
<em>tzdb(v.0.4.0)</em>, <em>xml2(v.1.3.4)</em>,
<em>viridis(v.0.6.3)</em>, <em>xfun(v.0.39)</em>, <em>hms(v.1.1.3)</em>,
<em>jquerylib(v.0.1.4)</em>, <em>babelgene(v.22.9)</em>,
<em>evaluate(v.0.21)</em>, <em>scrime(v.1.3.5)</em>,
<em>fansi(v.1.0.4)</em>, <em>restfulr(v.0.0.15)</em>,
<em>progress(v.1.2.2)</em>, <em>dbplyr(v.2.3.2)</em>,
<em>igraph(v.1.4.3)</em>, <em>DBI(v.1.1.3)</em>,
<em>htmlwidgets(v.1.6.2)</em>, <em>futile.logger(v.1.4.3)</em>,
<em>reshape(v.0.8.9)</em>, <em>ellipsis(v.0.3.2)</em>,
<em>crosstalk(v.1.2.0)</em>, <em>backports(v.1.4.1)</em>,
<em>annotate(v.1.76.0)</em>, <em>biomaRt(v.2.54.0)</em>,
<em>deldir(v.1.0-9)</em>, <em>sparseMatrixStats(v.1.10.0)</em>,
<em>vctrs(v.0.6.3)</em>, <em>here(v.1.0.1)</em>,
<em>ensembldb(v.2.22.0)</em>, <em>cachem(v.1.0.8)</em>,
<em>withr(v.2.5.0)</em>, <em>ggforce(v.0.4.1)</em>,
<em>Gviz(v.1.42.0)</em>, <em>BSgenome(v.1.66.3)</em>,
<em>checkmate(v.2.2.0)</em>, <em>vroom(v.1.6.3)</em>,
<em>GenomicAlignments(v.1.34.0)</em>, <em>fdrtool(v.1.2.17)</em>,
<em>prettyunits(v.1.1.1)</em>, <em>mclust(v.6.0.0)</em>,
<em>cluster(v.2.1.4)</em>, <em>lazyeval(v.0.2.2)</em>,
<em>crayon(v.1.5.2)</em>, <em>genefilter(v.1.80.0)</em>,
<em>labeling(v.0.4.2)</em>, <em>pkgconfig(v.2.0.3)</em>,
<em>slam(v.0.1-50)</em>, <em>tweenr(v.2.0.2)</em>,
<em>nlme(v.3.1-162)</em>, <em>ProtGenerics(v.1.30.0)</em>,
<em>nnet(v.7.3-19)</em>, <em>lifecycle(v.1.0.3)</em>,
<em>filelock(v.1.0.2)</em>, <em>BiocFileCache(v.2.6.0)</em>,
<em>dichromat(v.2.0-0.1)</em>, <em>VennDiagram(v.1.7.3)</em>,
<em>rprojroot(v.2.0.3)</em>, <em>polyclip(v.1.10-4)</em>,
<em>reactR(v.0.4.4)</em>, <em>rngtools(v.1.5.2)</em>,
<em>base64(v.2.0.1)</em>, <em>Matrix(v.1.5-4.1)</em>,
<em>lpsymphony(v.1.26.0)</em>, <em>Rhdf5lib(v.1.20.0)</em>,
<em>zoo(v.1.8-12)</em>, <em>base64enc(v.0.1-3)</em>,
<em>GlobalOptions(v.0.1.2)</em>, <em>png(v.0.1-8)</em>,
<em>viridisLite(v.0.4.2)</em>, <em>rjson(v.0.2.21)</em>,
<em>bitops(v.1.0-7)</em>, <em>rhdf5filters(v.1.10.0)</em>,
<em>blob(v.1.2.4)</em>, <em>DelayedMatrixStats(v.1.20.0)</em>,
<em>doRNG(v.1.8.6)</em>, <em>shape(v.1.4.6)</em>,
<em>nor1mix(v.1.3-0)</em>, <em>jpeg(v.0.1-10)</em>,
<em>memoise(v.2.0.1)</em>, <em>plyr(v.1.8.8)</em>,
<em>zlibbioc(v.1.44.0)</em>, <em>compiler(v.4.2.3)</em>,
<em>BiocIO(v.1.8.0)</em>, <em>illuminaio(v.0.40.0)</em>,
<em>RColorBrewer(v.1.1-3)</em>, <em>clue(v.0.3-64)</em>,
<em>cli(v.3.6.1)</em>, <em>htmlTable(v.2.4.1)</em>,
<em>formatR(v.1.14)</em>, <em>Formula(v.1.2-5)</em>,
<em>MASS(v.7.3-60)</em>, <em>mgcv(v.1.8-42)</em>,
<em>tidyselect(v.1.2.0)</em>, <em>stringi(v.1.7.12)</em>,
<em>highr(v.0.10)</em>, <em>askpass(v.1.1)</em>,
<em>locfit(v.1.5-9.8)</em>, <em>latticeExtra(v.0.6-30)</em>,
<em>grid(v.4.2.3)</em>, <em>sass(v.0.4.6)</em>,
<em>VariantAnnotation(v.1.44.0)</em>, <em>fastmatch(v.1.1-3)</em>,
<em>tools(v.4.2.3)</em>, <em>timechange(v.0.2.0)</em>,
<em>parallel(v.4.2.3)</em>, <em>circlize(v.0.4.15)</em>,
<em>rstudioapi(v.0.14)</em>, <em>foreach(v.1.5.2)</em>,
<em>foreign(v.0.8-84)</em>, <em>metapod(v.1.6.0)</em>,
<em>gridExtra(v.2.3)</em>, <em>farver(v.2.1.1)</em>,
<em>ComplexUpset(v.1.3.3)</em>, <em>digest(v.0.6.31)</em>,
<em>quadprog(v.1.5-8)</em>, <em>Rcpp(v.1.0.10)</em>,
<em>siggenes(v.1.72.0)</em>, <em>broom(v.1.0.5)</em>,
<em>httr(v.1.4.6)</em>, <em>ggdendro(v.0.1.23)</em>,
<em>AnnotationDbi(v.1.60.0)</em>, <em>biovizBase(v.1.46.0)</em>,
<em>ComplexHeatmap(v.2.14.0)</em>, <em>Rdpack(v.2.4)</em>,
<em>colorspace(v.2.1-0)</em>, <em>XML(v.3.99-0.14)</em>,
<em>splines(v.4.2.3)</em>, <em>graphlayouts(v.1.0.0)</em>,
<em>multtest(v.2.54.0)</em>, <em>plotly(v.4.10.2)</em>,
<em>xtable(v.1.8-4)</em>, <em>jsonlite(v.1.8.7)</em>,
<em>futile.options(v.1.0.1)</em>, <em>R6(v.2.5.1)</em>,
<em>Hmisc(v.5.1-0)</em>, <em>pillar(v.1.9.0)</em>,
<em>fastmap(v.1.1.1)</em>, <em>DT(v.0.28)</em>,
<em>beanplot(v.1.3.1)</em>, <em>codetools(v.0.2-19)</em>,
<em>utf8(v.1.2.3)</em>, <em>sva(v.3.46.0)</em>,
<em>lattice(v.0.21-8)</em>, <em>bslib(v.0.5.0)</em>,
<em>curl(v.5.0.1)</em>, <em>openssl(v.2.0.6)</em>,
<em>GO.db(v.3.16.0)</em>, <em>interp(v.1.1-4)</em>,
<em>survival(v.3.5-5)</em>, <em>rmarkdown(v.2.22)</em>,
<em>munsell(v.0.5.0)</em>, <em>GetoptLong(v.1.0.5)</em>,
<em>rhdf5(v.2.42.0)</em>, <em>GenomeInfoDbData(v.1.2.9)</em>,
<em>iterators(v.1.0.14)</em>, <em>HDF5Array(v.1.26.0)</em>,
<em>gtable(v.0.3.3)</em> and <em>rbibutils(v.2.2.13)</em></p>
</div>
</div>
</div>

<div class="footer">
  Workflow developed by:<br>
  Dr Stephen Pederson<br>
  Dame Roma Mitchell Cancer Research Laboratories<br>
  Adelaide Medical School, University of Adelaide, SA, Australia
</div>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
